{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Setting the PYTHON_VERSION environment variable.\n",
      "Setting the SPARK_MASTER environment variable.\n",
      "Setting the DB_TYPE environment variable.\n",
      "Setting the DB_PORT environment variable.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2017-12-06 11:45:46.595554\n"
     ]
    }
   ],
   "source": [
    "%run _imports.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "NOTEBOOK_NAME = 'load_pmc_xml_tables'\n",
    "os.makedirs(NOTEBOOK_NAME, exist_ok=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Parameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "MANUSCRIPT_DATA_DIR = Path('../downloads/ncbi/pmc/manuscript/').absolute()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Example"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "filelist.csv\t\t PMC0033XXXXX\t\t  PMC0046XXXXX\r\n",
      "PMC0021XXXXX\t\t PMC0034XXXXX\t\t  PMC0047XXXXX\r\n",
      "PMC0022XXXXX\t\t PMC0035XXXXX\t\t  PMC0048XXXXX\r\n",
      "PMC0023XXXXX\t\t PMC0036XXXXX\t\t  PMC0049XXXXX\r\n",
      "PMC0024XXXXX\t\t PMC0037XXXXX\t\t  PMC004XXXXXX.xml.tar.gz\r\n",
      "PMC0025XXXXX\t\t PMC0038XXXXX\t\t  PMC0050XXXXX\r\n",
      "PMC0026XXXXX\t\t PMC0039XXXXX\t\t  PMC0051XXXXX\r\n",
      "PMC0027XXXXX\t\t PMC003XXXXXX.xml.tar.gz  PMC0052XXXXX\r\n",
      "PMC0028XXXXX\t\t PMC0040XXXXX\t\t  PMC0053XXXXX\r\n",
      "PMC0029XXXXX\t\t PMC0041XXXXX\t\t  PMC0054XXXXX\r\n",
      "PMC002XXXXXX.xml.tar.gz  PMC0042XXXXX\t\t  PMC0055XXXXX\r\n",
      "PMC0030XXXXX\t\t PMC0043XXXXX\t\t  PMC0056XXXXX\r\n",
      "PMC0031XXXXX\t\t PMC0044XXXXX\t\t  PMC005XXXXXX.xml.tar.gz\r\n",
      "PMC0032XXXXX\t\t PMC0045XXXXX\r\n"
     ]
    }
   ],
   "source": [
    "!ls ../downloads/ncbi/pmc/manuscript/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "manuscript_files = pd.read_csv('../downloads/ncbi/pmc/manuscript/filelist.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>File</th>\n",
       "      <th>PMCID</th>\n",
       "      <th>PMID</th>\n",
       "      <th>MID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PMC0021XXXXX/PMC2136438.xml</td>\n",
       "      <td>PMC2136438</td>\n",
       "      <td>19023455</td>\n",
       "      <td>NIHMS10936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PMC0021XXXXX/PMC2185066.xml</td>\n",
       "      <td>PMC2185066</td>\n",
       "      <td>18769527</td>\n",
       "      <td>NIHMS31656</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PMC0022XXXXX/PMC2268633.xml</td>\n",
       "      <td>PMC2268633</td>\n",
       "      <td>17661176</td>\n",
       "      <td>NIHMS37031</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PMC0022XXXXX/PMC2287164.xml</td>\n",
       "      <td>PMC2287164</td>\n",
       "      <td>18389087</td>\n",
       "      <td>NIHMS38164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PMC0022XXXXX/PMC2288569.xml</td>\n",
       "      <td>PMC2288569</td>\n",
       "      <td>18650957</td>\n",
       "      <td>NIHMS40219</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          File       PMCID      PMID         MID\n",
       "0  PMC0021XXXXX/PMC2136438.xml  PMC2136438  19023455  NIHMS10936\n",
       "1  PMC0021XXXXX/PMC2185066.xml  PMC2185066  18769527  NIHMS31656\n",
       "2  PMC0022XXXXX/PMC2268633.xml  PMC2268633  17661176  NIHMS37031\n",
       "3  PMC0022XXXXX/PMC2287164.xml  PMC2287164  18389087  NIHMS38164\n",
       "4  PMC0022XXXXX/PMC2288569.xml  PMC2288569  18650957  NIHMS40219"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "manuscript_files.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pmc_tables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "# import xml.etree.ElementTree as ET\n",
    "# import re\n",
    "\n",
    "# count = 0\n",
    "\n",
    "\n",
    "# def get_tables(root):\n",
    "#     table_wraps_all = []\n",
    "#     tables_all = []\n",
    "#     for child in root:\n",
    "#         table_wraps = child.findall('*/table-wrap')\n",
    "#         tables = child.findall('**/table')\n",
    "#         assert len(table_wraps) == len(tables), (len(table_wraps), len(tables))\n",
    "#         table_wraps_all.extend(table_wraps)\n",
    "#         tables_all.extend(tables)\n",
    "#     return table_wraps_all, tables_all\n",
    "\n",
    "\n",
    "# def caption_to_string(element):\n",
    "#     if element is None:\n",
    "#         return \"\"\n",
    "#     element_text = ET.tostring(element, encoding='utf8', method='text').decode('utf-8')\n",
    "#     element_text = element_text.strip().replace('\\n', ' ')\n",
    "#     element_text = re.sub(' +', ' ', element_text)\n",
    "#     return element_text\n",
    "\n",
    "\n",
    "# for i, row in enumerate(manuscript_files.itertuples()):\n",
    "#     print('\\n' + f\"{row.File}\")\n",
    "\n",
    "#     results = {}\n",
    "    \n",
    "#     tree = ET.parse(MANUSCRIPT_DATA_DIR.joinpath(row.File))\n",
    "#     tables = tree.findall('.//table')\n",
    "#     table_wraps = tree.findall('.//table-wrap')\n",
    "    \n",
    "#     for table_wrap in table_wraps:\n",
    "#         print(\"  \" + \"-\" * 40)\n",
    "#         id_ = table_wrap.get('id')\n",
    "#         assert id_ is not None\n",
    "#         print(f\"  id: {id_}\")\n",
    "#         # Attrib\n",
    "#         attrib_set = set(table_wrap.attrib)\n",
    "#         if attrib_set - {'id', 'position'}:\n",
    "#             print(f\"tw({id_}): {attrib_set}\")\n",
    "#         # Label\n",
    "#         label = table_wrap.find('label')\n",
    "#         label_text = label.text if label is not None else \"\"\n",
    "#         print(f\"  label_text: {label_text}\")\n",
    "#         # Caption\n",
    "#         caption = table_wrap.find('caption')\n",
    "#         caption_text = caption_to_string(caption)\n",
    "#         print(f\"  caption_text: {caption_text}\")\n",
    "#         # Footer\n",
    "#         footer = table_wrap.find('table-wrap-foot')\n",
    "#         footer_text = caption_to_string(footer)\n",
    "#         print(f\"  footer_text: {footer_text}\")\n",
    "#         # Other children\n",
    "#         other_children = [\n",
    "#             c.tag for c in table_wrap.getchildren() \n",
    "#             if c.tag not in ['label', 'caption', 'table', 'table-wrap-foot']\n",
    "#         ]\n",
    "#         if other_children:\n",
    "#             print(other_children)\n",
    "        \n",
    "#         # Tables\n",
    "#         tables = table_wrap.findall('table')\n",
    "#         for i, table in enumerate(tables):\n",
    "#             unique = f\"{row.PMCID}/{op.basename(row.File)}::{id_}-{i}\"\n",
    "#             assert unique not in results\n",
    "#             print(f\"    unique: {unique}\")\n",
    "#             # Attrib\n",
    "#             attrib_set = {a for a in table.attrib if a not in ['frame', 'rules']}\n",
    "#             if attrib_set:\n",
    "#                 print(attrib_set)\n",
    "#             # Children\n",
    "#             children_set = {c.tag for c in table.getchildren() if c.tag not in ['thead', 'tbody']}\n",
    "#             if children_set:\n",
    "#                 print(children_set)\n",
    "#             # DataFrame\n",
    "#             table_bytes = ET.tostring(table)\n",
    "#             display(HTML(table_bytes.decode('utf-8')))\n",
    "#             table_dfs = pd.read_html(table_bytes, na_values=['', '-', '–'])\n",
    "#             assert len(table_dfs) == 1\n",
    "#             table_df = table_dfs[0]\n",
    "#             display(table_df)\n",
    "#             #\n",
    "#             table_df_dev = pmc_tables.read_html(table_bytes, na_values=['', '-', '–'])[0]\n",
    "#             table_df_dev = pmc_tables.process_dataframe(table_df_dev)\n",
    "#             display(table_df_dev)\n",
    "                \n",
    "#     root = tree.getroot()\n",
    "#     a, b = get_tables(root)\n",
    "    \n",
    "#     count += 1\n",
    "#     if count > 20:\n",
    "#         break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import xml.etree.ElementTree as ET\n",
    "import re\n",
    "\n",
    "count = 0\n",
    "\n",
    "\n",
    "def get_tables(root):\n",
    "    table_wraps_all = []\n",
    "    tables_all = []\n",
    "    for child in root:\n",
    "        table_wraps = child.findall('*/table-wrap')\n",
    "        tables = child.findall('**/table')\n",
    "        assert len(table_wraps) == len(tables), (len(table_wraps), len(tables))\n",
    "        table_wraps_all.extend(table_wraps)\n",
    "        tables_all.extend(tables)\n",
    "    return table_wraps_all, tables_all\n",
    "\n",
    "\n",
    "def caption_to_string(element):\n",
    "    if element is None:\n",
    "        return \"\"\n",
    "    element_text = ET.tostring(element, encoding='utf8', method='text').decode('utf-8')\n",
    "    element_text = element_text.strip().replace('\\n', ' ')\n",
    "    element_text = re.sub(' +', ' ', element_text)\n",
    "    return element_text\n",
    "\n",
    "\n",
    "def extract_tables_from_xml(xml_file):\n",
    "    tree = ET.parse(xml_file)\n",
    "    tables = tree.findall('.//table')\n",
    "    table_wraps = tree.findall('.//table-wrap')\n",
    "\n",
    "    data = {}\n",
    "    \n",
    "    for table_wrap in table_wraps:\n",
    "        # Id\n",
    "        id_ = table_wrap.get('id')\n",
    "        assert id_ is not None\n",
    "        logger.debug(f\"  id: {id_}\")\n",
    "        # Attrib\n",
    "        attrib_set = set(table_wrap.attrib)\n",
    "        if attrib_set - {'id', 'position'}:\n",
    "            print(f\"tw({id_}): {attrib_set}\")\n",
    "        # Label\n",
    "        label = table_wrap.find('label')\n",
    "        label_text = label.text if label is not None else \"\"\n",
    "        logger.debug(f\"  label_text: {label_text}\")\n",
    "        # Caption\n",
    "        caption = table_wrap.find('caption')\n",
    "        caption_text = caption_to_string(caption)\n",
    "        logger.debug(f\"  caption_text: {caption_text}\")\n",
    "        # Footer\n",
    "        footer = table_wrap.find('table-wrap-foot')\n",
    "        footer_text = caption_to_string(footer)\n",
    "        logger.debug(f\"  footer_text: {footer_text}\")\n",
    "        # Other children\n",
    "        other_children = [\n",
    "            c.tag for c in table_wrap.getchildren() \n",
    "            if c.tag not in ['label', 'caption', 'table', 'table-wrap-foot']\n",
    "        ]\n",
    "        if other_children:\n",
    "            logger.info(other_children)\n",
    "        # Tables\n",
    "        tables = table_wrap.findall('table')\n",
    "        for i, table in enumerate(tables):\n",
    "            unique = f\"{op.basename(row.File)}::{id_}-{i}\"\n",
    "            assert unique not in results\n",
    "            logger.debug(f\"    unique: {unique}\")\n",
    "            # Attrib\n",
    "            attrib_set = {a for a in table.attrib if a not in ['frame', 'rules']}\n",
    "            if attrib_set:\n",
    "                logger.info(attrib_set)\n",
    "            # Children\n",
    "            children_set = {c.tag for c in table.getchildren() if c.tag not in ['thead', 'tbody']}\n",
    "            if children_set:\n",
    "                logger.info(children_set)\n",
    "            # DataFrame\n",
    "            table_bytes = ET.tostring(table)\n",
    "            # display(HTML(table_bytes.decode('utf-8')))\n",
    "            # table_dfs = pd.read_html(table_bytes, na_values=['', '-', '–'])\n",
    "            # assert len(table_dfs) == 1\n",
    "            # table_df = table_dfs[0]\n",
    "            # display(table_df)\n",
    "            #\n",
    "            data[unique] = (table_df_dev, {\n",
    "                'id': id_,\n",
    "                'file_name': row.File,\n",
    "                \n",
    "            })\n",
    "            try:\n",
    "                table_df_dev = pmc_tables.read_html(table_bytes, na_values=['', '-', '–'])[0]\n",
    "                table_df_dev = pmc_tables.process_dataframe(table_df_dev)\n",
    "                # display(table_df_dev)\n",
    "                store.put('/' + unique, table_df_dev)\n",
    "            except:\n",
    "                pass\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [],
   "source": [
    "store.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0021XXXXX/PMC2136438.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean absolute error (MAE) of prediction for four alternate predictors. The average, minimum and maximum value calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the MAE for the geostatistical methods is lower than for the local empirical Bayes smoother.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Average of the reference risk values and the estimates obtained using four alternate approaches. The average, minimum and maximum value of the statistic calculated over 25 simulated rate maps are reported.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean absolute error (MAE) of prediction of the boundary statistic by five alternate estimators. The average, minimum and maximum MAE calculated over 25 simulated rate maps are reported. The last column gives the percentage of realizations where the MAE for the geostatistical methods is lower than for the local empirical Bayes smoother.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Average of the reference boundary statistic values and the estimates obtained using five alternate approaches. The average, minimum and maximum value of the statistic calculated over 25 simulated rate maps are reported.\n",
      "  footer_text: \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py:809: FutureWarning: The 'tupleize_cols' argument has been deprecated and will be removed in a future version. Column tuples will then always be converted to MultiIndex.\n",
      "\n",
      "\n",
      "  self.options, self.engine = self._clean_options(options, engine)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    unique: PMC2136438/PMC2136438.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Average proportion of false positives necessary to the detection of all 67 target edges using three types of neutral models and four estimators for the boundary statistic. The average, minimum and maximum proportions calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the particular method yields the smallest proportion of false positives.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Goodness statistic measuring the agreement between the actual proportion of false positives and the significance level used in tests based on three types of neutral models and four estimators for the boundary statistic. The average, minimum and maximum values calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the particular method yields the largest goodness statistic.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Average p-value for the tests conducted over the sets of target and non-target edges. The average, minimum and maximum values calculated over 25 simulated rate maps are reported. The last column reports the ratio of average p-values for non-target versus target edges.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T7-0\n",
      "\n",
      "PMC0021XXXXX/PMC2185066.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Estimation Performance of the Rate Parameters When Different Noise Levels cin Are Added to the Input Function. The Results Are Calculated From 100 Simulation Runs\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE II\n",
      "  caption_text: Estimation Performance of the Rate Parameters When Different Noise Level Is Added to the Voxel TACs. The Results Are Calculated From 100 Simulation Runs\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2136438.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2185066.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2185066.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2185066.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE III\n",
      "  caption_text: Performance of Estimating the Factor Images, in Terms of CC and PM. in Each Parenthesis, the First Number Is the Mean, and the Second Number Is the Corresponding Standard Deviation of the Performance Measure CC or PM. Here, the Noise Level σ2 = 30\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T3-0\n",
      "\n",
      "PMC0022XXXXX/PMC2268633.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Distribution of deaths by sex and place of birth, NHW residents, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: * Chronic obstructive pulmonary disease\n",
      "    unique: PMC2268633/PMC2268633.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T7-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T8-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table 8\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T8-0\n",
      "  ----------------------------------------\n",
      "  id: T9\n",
      "  label_text: Table 9\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T9-0\n",
      "  ----------------------------------------\n",
      "  id: T10\n",
      "  label_text: Table 10\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T10-0\n",
      "\n",
      "PMC0022XXXXX/PMC2287164.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sequence variants\n",
      "  footer_text: \n",
      "    unique: PMC2287164/PMC2287164.xml::T1-0\n",
      "\n",
      "PMC0022XXXXX/PMC2288569.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Oral health characteristics of cohort at age 32 years, by sex and SES\n",
      "  footer_text: a One participant unable to be classified for the age-32 SES variable. * P < 0.05; ** P < 0.01; *** P < 0.001.\n",
      "    unique: PMC2288569/PMC2288569.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Distribution of responses to individual OHIP-14 items and mean item scores (n = 924)\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T9-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2268633.xml::T10-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2287164.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Prevalence, extent and severity of impacts by OHIP-14 subscale and total score (n = 924)\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Prevalence of impacts (fairly/very often), by sex, oral disease prevalence, self-rated oral health and usual reason for visiting a dentist\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Prevalence of impacts (fairly/very often), by SES at age 32 years, oral disease prevalence, self-rated oral health and usual reason for visiting a dentist\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Logistic regression model for prevalence of impacts (‘fairly often’ or ‘very often’) at age 32 years (n = 923) Coeff. Odds ratio 95% CI for OR P-value\n",
      "  footer_text: −2 Log-likelihood = 896.69; Cox & Snell R2 = 0.11; Nagelkerke R2 = 0.16; Hosmer and Lemeshow chi-squared test = 3.01, d.f. = 8, P = 0.93.\n",
      "    unique: PMC2288569/PMC2288569.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Poisson regression models for the extent and severity of impacts (n = 923)\n",
      "  footer_text: a Ratio of geometric mean values.\n",
      "    unique: PMC2288569/PMC2288569.xml::T7-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2288569.xml::T8-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table 8\n",
      "  caption_text: International comparisons of prevalence, extent and severity of overall impacts and selected dimensions among dentate adults\n",
      "  footer_text: a Slade et al. (10). b Estimated for the UK and Australian samples by scrutinizing the data distributions.\n",
      "    unique: PMC2288569/PMC2288569.xml::T8-0\n",
      "\n",
      "PMC0022XXXXX/PMC2293303.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Lifetime prevalence associations of gambling types with lifetime onset and persistence of CIDI/DSM-IV pathological gambling (PG)\n",
      "  footer_text: * Significant at the .05 level, two-sided test. 1 Multivariate model controlling for sex, race-ethnicity, age of first gambling, years since first gambling, and 11 gambling types. 2 Multivariate model controlling for sex, race-ethnicity, aoo of first gambling, aoo of first problem, years since first problem, and 11 gambling types\n",
      "    unique: PMC2293303/PMC2293303.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Lifetime comorbidity of CIDI/DSM-IV pathological gambling (PG) with other lifetime CIDI/DSM-IV disorders and temporal priorities in age of onset (AOO)\n",
      "  footer_text: * Significant at the .05 level, two-sided test. 1 The percent of respondents with PG who also meet criteria for the other disorder. 2 The ORs are calculated in the total sample, including respondents who never gambled. The ORs were calculated controlling for age at interview so as to avoid confounding due to time at risk.\n",
      "    unique: PMC2293303/PMC2293303.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Reciprocal time-lagged associations of temporally primary problem gambling predicting the subsequent first onset of other disorders and of temporally primary other disorders predicting the subsequent first onset of CIDI/DSM-IV PG1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2293303.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2293303.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2293303.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2323355.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  footer_text: * Significant at the .05 level, two-sided test. 1 Models are bivariate for each disorder and controlling for age, sex, and race-ethnicity. 2 There are no onsets of ADHD, ODD, CD that came after the PG onsets\n",
      "    unique: PMC2293303/PMC2293303.xml::T3-0\n",
      "\n",
      "PMC0023XXXXX/PMC2323355.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected genes identified by comparison of NE from patients with EAC (N with T) vs. without EAC (N without T). Threshold value set at 2.7; N with T: gene score in the group of patients with esophageal adenocarcinoma; N without T: gene score in the group of patients without esophageal adenocarcinoma. Gene identifiers and gene names are shown in the two leftmost columns.\n",
      "  footer_text: \n",
      "    unique: PMC2323355/PMC2323355.xml::T1-0\n",
      "\n",
      "PMC0023XXXXX/PMC2365750.xml\n",
      "\n",
      "PMC0023XXXXX/PMC2396881.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2423936.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of mouse lines, their genotypes and designations\n",
      "  footer_text: a Wild type mice were homozygous for Trsp and albumin Cre. b Trsp liver knockout mice lacked Trsp in their liver. c A34 transgenic mice carrying 2 copies of the T34→A34 mutant transgene. d G37 transgenic mice carrying 2 copies of the A37→G37 mutant transgene. e G37 transgenic mice carrying 16 copies of the A37→G37 mutant transgene. f Designations used in the text to denote each mouse line.\n",
      "    unique: PMC2423936/PMC2423936.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primers for assessing real-time PCRa\n",
      "  footer_text: a Primers designed for determining real-time PCR of each mRNA examined are shown in the table.\n",
      "    unique: PMC2423936/PMC2423936.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Transcripts in elevated levels within liver of knockout mice relative to wild type mice a\n",
      "  footer_text: a Genes elevated ≥ 2.0 fold in ΔTrsp mice compared to Trsp mice with a P-value ≤ 0.05 were assessed and shown in the table. The corresponding transcript levels were also analyzed in A34, G37L and G37H replacement mice relative to Trsp mice. Gene Unigene Accession Number, gene description, fold change with P value, and gene function(s) are shown. Four control and four experimental animals were used for comparing ΔTrsp, A34 and G37L with Trsp, while three control and three experimental animals were used for comparing G37H with Trsp. b Genes are grouped into classes according to function.\n",
      "    unique: PMC2423936/PMC2423936.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2423936.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2423936.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2423936.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2423936.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Transcripts in reduced levels within liver of knockout mice relative to wild type mice a\n",
      "  footer_text: a Genes depleted ≥ 2.0 fold in ΔTrsp mice as compared to Trsp mice with a P-value ≤ 0.05 were assessed and shown in the table. The corresponding transcript levels were also analyzed in A34, G37L and G37H replacement mice relative to Trsp mice. Gene Unigene Accession Number, gene description, fold change with P value, and gene function(s) are shown. Four control and four experimental animals were used for comparing ΔTrsp, A34 and G37L with Trsp, while three control and three experimental animals were used for comparing G37H with Trsp. b Genes are grouped into classes according to function.\n",
      "    unique: PMC2423936/PMC2423936.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2424084.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of the Study Sample: Community-dwelling Framingham Heart Study Original Cohort\n",
      "  footer_text: * Cardiovascular disease= coronary heart disease, stroke, transient ischemic attack, congestive heart failure, or intermittent claudication. Hypertension= blood pressure ≥ 140/90 or on antihypertensive medication treatment. Diabetes=causal blood glucose of 200 mg/dl or greater or treatment with oral hypoglycemic agent or insulin. Depressive symptoms assessed at examinations 22 and 25 with the Center for Epidemiologic Studies Depression Scale (CES-D). A score of 16 or greater defined the presence of depressive symptoms. Medical conditions included any of the following five conditions: diabetes, cardiovascular disease, cancer, chronic obstructive pulmonary disease, and hip fracture. NA= not available as data not collected. Rx=medication treatment\n",
      "    unique: PMC2424084/PMC2424084.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424084.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424084.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424084.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424084.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Self-reported Prevalence of Functional Limitation or Physical Disability: Framingham Heart Study Original Cohort, Mean Age 82 years\n",
      "  footer_text: Nagi physical performance scale limitation= some difficulty, a lot of difficulty, unable to do, do not do on physician order Rosow-Breslau functional health scale disability= unable to do. Modified Katz ADL scale disability=human assistance needed, minimally dependent; dependent † At exam 25 this question was asked as part of the modified Katz ADL Scale ‡ At Exams 20 and 25 the question was changed to transferring (getting in and out of a chair) and walking on a level surface about 50 yards\n",
      "    unique: PMC2424084/PMC2424084.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Sex-specific Trends in Self-reported Functional Limitations and Physical Disability Adjusted for Age\n",
      "  footer_text: \n",
      "    unique: PMC2424084/PMC2424084.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Appendix 1\n",
      "  caption_text: Study Sample: Framingham Original Cohort, Aged 79 to 88, Attending Index Examinations 15, 20, and 25\n",
      "  footer_text: † Examinations off-site in participants’ homes or nursing homes were not available at examination 15. ‡ The Framingham Disability Study collected physical function data at examination 15.(19;43)\n",
      "    unique: PMC2424084/PMC2424084.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2424085.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of Participants, by Age Group: Health and Retirement Study, 1998\n",
      "  footer_text: Note. ADL = activities of daily living. IADL = instrumental activities of daily living. ADL include bathing, dressing, eating, transferring, and toileting. IADL include shopping, preparing meals, using the telephone, managing medications, and managing finances.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424085.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424085.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424090.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424090.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2424090.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    unique: PMC2424085/PMC2424085.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Harrell's C Statistic of Cox Mortality Models Including Different Risk Factors by Age Groups: Health and Retirement Study, 1998\n",
      "  footer_text: a Permutation test of whether slope of regression between age group and C statistic = 0.\n",
      "    unique: PMC2424085/PMC2424085.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2424090.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of those who died compared to those who did not die during follow-up.\n",
      "  footer_text: a CES-D = Center for Epidemiologic Studies Depression Scale – Short Form\n",
      "    unique: PMC2424090/PMC2424090.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Relative risk of mortality associated with perceived discrimination\n",
      "  footer_text: Hazard ratios and 95% confidence interval; Hazard ratios are estimated from separate models adjusted for age, sex, race, race*age, and lifetime SES.\n",
      "    unique: PMC2424090/PMC2424090.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Relative risk of mortality associated with unfair treatment and personal rejection\n",
      "  footer_text: Hazard ratios and 95% confidence interval; Hazard ratios are estimated from separate models adjusted for age, sex, race, race*age, and lifetime SES.\n",
      "    unique: PMC2424090/PMC2424090.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2430055.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Peripheral blood cell counts in 10-month post-transplant animals\n",
      "  footer_text: Data shown are mean (±SD) values from three to five mice per genotype * p<0.05, ** p<0.005 when compared to wild type VEX\n",
      "    unique: PMC2430055/PMC2430055.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Differential counts of sorted VEX (+) myeloid cells from 10-month post-transplant animals\n",
      "  footer_text: Data shown are percentages of >250 cells counted per sample. Blast/Pro: myeloblast and promyelocytes; Myelo: myelocytes; Band: band-form granulocytes; Eos: eosinophils; Mono: monocytes\n",
      "    unique: PMC2430055/PMC2430055.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2430763.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2435206.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2435220.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Counts of edge classification results\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Accuracy and error statistics for boundary classification using the little b and polygon wombling approaches\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Counts from classification table for Big B, Ladder and Local Moran methods\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Specificity, Type I and Type II error, and sensitivity of the Big B, Ladder and Local Moran methods\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2435384.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Average ± SD values (%) of degree of substitution (DS) and purity of the MPS-peptide-dextran conjugates (n = 3).\n",
      "  footer_text: \n",
      "    unique: PMC2435384/PMC2435384.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Inter-run accuracy (%Error) and precision (%CV) of the reversed-phase assay (n =5)\n",
      "  footer_text: \n",
      "    unique: PMC2435384/PMC2435384.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean ± SD values of degradation half lives (h) of DMP conjugates in pH 7.4 buffer and rat blood (n = 3).\n",
      "  footer_text: a–j Means with similar letters are significantly different each other: ANOVA with post-hoc Scheffe test..\n",
      "    unique: PMC2435384/PMC2435384.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2430055.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2430055.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435220.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435220.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435220.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435220.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435384.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435384.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435384.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435391.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2435391.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Human IBD patient description\n",
      "  footer_text: NOTE. CD; Crohn’s disease, UC; ulcerative colitis, Sm Intest; small intestine.\n",
      "    unique: PMC2435391/PMC2435391.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Gene expression analysis in the colon of Hnf4αF/F and Hnf4αΔIEpC mice\n",
      "  footer_text: NOTE. Quantitative real-time PCR of colon mRNA from 2-month-old mice (n=6). All the values are means ± SD. Significant difference compared to Hnf4αF/F mice: * , p< 0.05 ** , p< 0.01 *** , p< 0.001.\n",
      "    unique: PMC2435391/PMC2435391.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2435511.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2440269.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Percent of Potentially Eligible Subjects Rejected by Reason for Exclusion\n",
      "  footer_text: \n",
      "    unique: PMC2440269/PMC2440269.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2440297.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2440637.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2440700.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of mean lifespan and its components in all experimental groups\n",
      "  footer_text: \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2435391.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2440700.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2440700.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    unique: PMC2440700/PMC2440700.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cox regression results (hazard ratio and Wald statistics) of potentially influential factors for lifespan, AFF* and flightspan*\n",
      "  footer_text: * limited sampling in experiment 4 precluded meaningful calculations\n",
      "    unique: PMC2440700/PMC2440700.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mortality models based on the experimental data (λ: initial mortality, γ: rate of exponential mortality increase, s: mortality deceleration)\n",
      "  footer_text: \n",
      "    unique: PMC2440700/PMC2440700.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Brood production (relative brood production) during the initial three experimental weeks\n",
      "  footer_text: \n",
      "    unique: PMC2440700/PMC2440700.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Comparison of mean lifespan, AFF, and flightspan relative to brood production in all experimental groups\n",
      "  footer_text: \n",
      "    unique: PMC2440700/PMC2440700.xml::T5-0\n",
      "\n",
      "PMC0024XXXXX/PMC2441880.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographics of Asian-American current and former smokers, California Health Interview Study, 2003\n",
      "  footer_text: a Significance for women * p≤0.0001; ** p≤0.01; *** p≤0.05; **** p≤0.001 cpd, cigarettes per day; div, divorced; HS, high school; sep, separated; wid, widowed\n",
      "    unique: PMC2441880/PMC2441880.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2440700.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2440700.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2440700.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2441880.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Multivariate logistic analysis of factors associated with former or lighter smoking among Asian Americans, California Health Interview Study, 2003\n",
      "  footer_text: a The p-value for interaction comparing effects of smoke-free home among long-term residents vs smoke-free home among recent immigrants for (1) former vs current smoker outcome is p=0.02 and (2) lighter vs heavier smoker outcome is p=0.03. Reference group for each category reflects having smoke-free home for the respective category of years in U.S. cpd, cigarettes per day; HS, high school; yrs, years\n",
      "    unique: PMC2441880/PMC2441880.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2441916.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Quantitation of Th1 and Th2 IS formed on supported planar bilayersa\n",
      "  footer_text: a Quantitation of data from Figs. 1 and 2 (100 μM) and Fig. 5 (10 and 1 μM).\n",
      "    unique: PMC2441916/PMC2441916.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2443692.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2443732.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Organic Acids Sulfated In Situ to Identify the Urinary Negative Ion of 230.996 m/z\n",
      "  footer_text: \n",
      "    unique: PMC2443732/PMC2443732.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2441880.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2441916.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2443732.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2443732.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Identification of Mouse Urinary Biomarkers after 3 Gy γ Radiation\n",
      "  footer_text: \n",
      "    unique: PMC2443732/PMC2443732.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Identification of Mouse Urinary Biomarkers after 8 Gy γ Radiation\n",
      "  footer_text: \n",
      "    unique: PMC2443732/PMC2443732.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Historical Biomarkers of Radiation Exposure\n",
      "  footer_text: Notes. 1[M-H]−(± 10 ppm) either not detected or not statistically significantly different (P ≥ 0.05) according to exposure status; 2[M-H]−(± 10 ppm) elevated (P < 0.05) in urine from mice exposed to 3 Gy; 3[M-H]−(± 10 ppm) elevated (P < 0.05) in urine from mice exposed to 8 Gy; 4[M-H]−(± 10 ppm) elevated (P < 0.05) in urine from mice both exposed to 3 and 8 Gy. All comparisons made with mean creatinine-normalized relative concentrations by t test with α = 0.05.\n",
      "    unique: PMC2443732/PMC2443732.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2444046.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Nine CRM community readiness stages [9]\n",
      "  footer_text: \n",
      "    unique: PMC2444046/PMC2444046.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2443732.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2443732.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2444046.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2444046.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2444046.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Aggregate community readiness results\n",
      "  footer_text: Note: All tests of significance are significant, p < .05, two-tailed. All tests of significance are based on 60 dfs. Effect Size: Effect sizes are reported as Cohen’s D. Cohen [35] has suggested the following guidelines for interpreting the magnitude of D: .20=Small; .50=Medium; .80=Large.\n",
      "    unique: PMC2444046/PMC2444046.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Community readiness scores by community, pre- and post-intervention\n",
      "  footer_text: * Note: Following the CRM protocol [9], overall readiness stages are truncated or rounded down to the lower integer.\n",
      "    unique: PMC2444046/PMC2444046.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2446451.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: A full factorial design for four components (factors) each at two levels (four columns A–D) and their interactions\n",
      "  footer_text: \n",
      "    unique: PMC2446451/PMC2446451.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number of groups in a two-level full factorial design as the number of factors increase\n",
      "  footer_text: \n",
      "    unique: PMC2446451/PMC2446451.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: FFD for the Guide to Decide Project\n",
      "  footer_text: \n",
      "    unique: PMC2446451/PMC2446451.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446451.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446451.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446451.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446451.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446469.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446469.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Analysis of participants’ knowledge scores*\n",
      "  footer_text: * Analysis controls for participants’ numeracy score. The main effect “Pictograph” is included even though it is not important because it has significant interactions with other factors.\n",
      "    unique: PMC2446451/PMC2446451.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2446469.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Participant Characteristics by Wave: Baltimore Epidemiological Catchment Area Study, Baltimore, MD, 1981–2005\n",
      "  footer_text: Note. MDD = major depressive disorder. Because of missing data, n = 3332 at wave 1, n = 2743 at wave 2, n = 1801 at wave 3, and n = 1037 at wave 4. P values are from the χ2 (categorical variables) or t test (continuous variables) for MDD vs no MDD within each wave. *** P < .01; † P < .001.\n",
      "    unique: PMC2446469/PMC2446469.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Incidence of Type 2 Diabetes per 1000 Person-Years, by Age Group at Wave 3 and Depression Status: Baltimore Epidemiological Catchment Area Study, Baltimore, MD, 1993–1996\n",
      "  footer_text: Note. MDD = major depressive disorder.\n",
      "    unique: PMC2446469/PMC2446469.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Association of Depression and Risk of Type 2 Diabetes: Baltimore Epidemiological Catchment Area Study, Baltimore, MD, 1981–2005\n",
      "  footer_text: Note. MDD = major depressive disorder; HR = hazard ratio; CI = confidence interval. Reference category for all models was never diagnosed with MDD. a Model 1 adjusted for age, race, gender, education, and body mass index. Model 2 added family history of diabetes to model 1. Model 3 added smoking status, alcohol use, stairs climbed per day, and frequency of eating balanced meals to model 2. Model 4 added frequency of social contact with relatives to model 3. b Model 1 adjusted for age, race, gender, and education. Model 2 added smoking status and alcohol use to model 1. Model 3 added social network size to model 2. Model 4 added antidepressant use to model 3. * P < .10; ** P < .05.\n",
      "    unique: PMC2446469/PMC2446469.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2453010.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2453029.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2446469.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453029.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453029.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453029.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "  caption_text: Intake of soy isoflavones and breast cancer risk\n",
      "  footer_text: a Adjusted for age, years of interview, dialect, education, family history of breast cancer, parity, age when period became regular, menopausal status (if applicable), body mass index, and n-3 fatty acid.\n",
      "    unique: PMC2453029/PMC2453029.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Intake of soy isoflavones and breast cancer risk in postmenopausal women by body mass index\n",
      "  footer_text: a Adjusted for age, years of interview, dialect, education, family history of breast cancer, parity, age when period became regular, body mass index, and n-3 fatty acid.\n",
      "    unique: PMC2453029/PMC2453029.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Intake of soy isoflavones and breast cancer risk (RR, 95% CI)a in postmenopausal women by oestrogen receptor (ER) and progesterone receptor (PR) status\n",
      "  footer_text: a Adjusted for age, years of interview, dialect, education, family history of breast cancer, parity, age when period became regular, body mass index, and n-3 fatty acid. b We did not include breast cancers that were ER+ PR− (n = 57), ER+PR unknown (n = 1), ER−PR+ (n = 4) or ER − PR unknown (n = 2) in the analysis.\n",
      "    unique: PMC2453029/PMC2453029.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Intake of soy isoflavones and breast cancer risk by the length of follow-up\n",
      "  footer_text: a Adjusted for age, years of interview, dialect, education, family history of breast cancer, parity, age when period became regular, menopausal status (if applicable), body mass index, and n-3 fatty acid.\n",
      "    unique: PMC2453029/PMC2453029.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2453069.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2453243.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453029.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2453677.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics and risk factors for cardiovascular disease and/or diabetes of study participants randomized to ILS or placebo\n",
      "  footer_text: Data are medians ± interquartile range unless otherwise indicated. Twenty Pacific Islanders were included in the “Asian” group. Physical activity data are based on responses to the Modifiable Activity Questionnaire. MET-hours represent the average amount of time engaged in specified physical activities multiplied by the MET value of each activity. BP, blood pressure; HRT, hormone replacement therapy; TG, triglyceride. * P < 0.01, men vs. women † P < 0.05, men vs. women.\n",
      "    unique: PMC2453677/PMC2453677.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Percent change in risk factors for cardiovascular disease and diabetes over year 1 in those randomized to placebo\n",
      "  footer_text: Data are medians ± interquartile range. BP, blood pressure; TG, triglyceride. † P < 0.05, men vs. women.\n",
      "    unique: PMC2453677/PMC2453677.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Percent change in risk factors for cardiovascular disease and diabetes over year 1 in those randomized to ILS\n",
      "  footer_text: Data are medians ± interquartile range. BP, blood pressure; TG, triglyceride. * P < 0.01, men vs. women † P < 0.05, men vs. women.\n",
      "    unique: PMC2453677/PMC2453677.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453677.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453677.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453677.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453683.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2453683.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of study participants within different BMI categories. Data shown as mean ± standard deviation for continuous variables, and n (percent) for dichotomous variables\n",
      "  footer_text: Abbreviations: BMI, body mass index; SD, standard deviation, LDL, low-density lipoprotein; HDL, high-density lipoprotein; WC, waist circumference SI conversion factors: To convert glucose to mmol/L, multiply mg/dL values by 0.0555; total, LDL and HDL cholesterol to mmol/L, multiply mg/dL values by 0.0259; triglycerides to mmol/L, multiply mg/dL values by 0.0113 * Global GEE age- and sex-adjusted p-value, except for age which is sex-adjusted and sex, which is age-adjusted\n",
      "    unique: PMC2453683/PMC2453683.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Age-and sex-adjusted rates of hypertension, elevated levels of low lipoprotein cholesterol, type 2 diabetes, treatment and control among BMI categories\n",
      "  footer_text: Abbreviations: CI, confidence interval; BMI, body mass index; LDL-C, low-density lipoprotein cholesterol; Hb A1c, Hemoglobin A1c level; FPG, fasting plasma glucose SI conversion factor: To convert fasting plasma glucose to mmol/L, multiply mg/dL values by 0.0555 * Global GEE p-value † Data only available for Offspring cohort\n",
      "    unique: PMC2453683/PMC2453683.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Online Appendix Table 1\n",
      "  caption_text: Age-and sex-adjusted rates of hypertension, elevated levels of low-density lipoprotein cholesterol, type 2 diabetes, treatment and control among WC categories\n",
      "  footer_text: Abbreviations: CI, confidence interval; WC, waist circumference; LDL-C, low-density lipoprotein cholesterol; Hb A1c, Hemoglobin A1c level; FPG, fasting plasma glucose SI conversion factor: To convert fasting plasma glucose to mmol/L, multiply mg/dL values by 0.0555 * Global GEE p-value † Data only available for Offspring cohort\n",
      "    unique: PMC2453683/PMC2453683.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Online Appendix Table 2\n",
      "  caption_text: Sex-adjusted rates of hypertension, elevated levels of low lipoprotein cholesterol, type 2 diabetes, treatment and control among BMI categories, stratified by age\n",
      "  footer_text: Abbreviations: CI, confidence interval; BMI, body mass index; LDL-C, low-density lipoprotein cholesterol; Hb A1c, Hemoglobin A1c level; FPG, fasting plasma glucose SI conversion factor: To convert fasting plasma glucose to mmol/L, multiply mg/dL values by 0.0555 * Global GEE p-value † Among participants younger than 50 years of age with a BMI 18.5-<25, there were only 5 individuals with diabetes mellitus. Of those, 5 received hypoglycemic treatment and 3 had FPG <126 mg/dL ‡ Interpret p-value with caution as there are only small number in the cells of participants younger than 50 years of age with a BMI 18.5-<25\n",
      "    unique: PMC2453683/PMC2453683.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453683.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453683.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453683.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2453683.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Online Appendix Table 3\n",
      "  caption_text: Age-adjusted rates of hypertension, elevated levels of low-density lipoprotein cholesterol, type 2 diabetes, treatment and control among BMI categories, stratified by sex\n",
      "  footer_text: Abbreviations: CI, confidence interval; BMI, body mass index; LDL-C, low-density lipoprotein cholesterol; Hb A1c, Hemoglobin A1c level; FPG, fasting plasma glucose SI conversion factor: To convert fasting plasma glucose to mmol/L, multiply mg/dL values by 0.0555 * Global GEE p-value † Global ANOVA p-value\n",
      "    unique: PMC2453683/PMC2453683.xml::T5-0\n",
      "\n",
      "PMC0024XXXXX/PMC2459142.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic characteristics of the resident-level sample population and by gender\n",
      "  footer_text: a Indicates a test performed using either independent t-test for gender differences in means or a test in differences in proportions, p<0.05 (2-tailed). b This is measured out of nine possible common medical conditions (e.g., diabetes, hypertension, depression). * p<0.05\n",
      "    unique: PMC2459142/PMC2459142.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Neighborhood-level random components from multilevel Poisson regression models\n",
      "  footer_text: a Random coefficient (variance) at the neighborhood level-variance in Model 0 represents random variation in neighborhoods whereas variance in Model 1 through Model 3 denote residual variance conditional on model covariates/predictors. b Intraclass correlations (ICC) is defined as a ratio of between-neighborhood variability/(between-neighborhood variability + within-neighborhood variability). c Model 0: unconditional model (i.e., no level-specific predictors/covariates); Model 1: model with only individual-level covariates; Model 2: model with both individual- and neighborhood-level covariates; Model 3: model with both individual- and neighborhood-level covariates and key built-environment predictors. d This is the proportion of variance explained at neighborhood level after addition of the neighborhood-level key built-environment variables, calculated as: {variance (Model 2)–variance (Model 3)/variance (Model 2), where variance (Model 2) represents the total between-neighborhood-level variance that can be explained by addition of the between-neighborhood variables. * p<0.05; ** p<0.001\n",
      "    unique: PMC2459142/PMC2459142.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2459142.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2459142.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2459142.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Adjusted estimates of neighborhood-level Poisson regression model analyses: associations between built-environment factors, overweight/obese, and various forms of physical activitya\n",
      "  footer_text: a Adjusting for neighborhood-level covariates of residential density, median household income, percentage of African-American and Hispanic residents, and resident-level covariates of age, gender, race/ethnicity, employment status, home ownership, household income, health status, fruit and vegetable intake, fried-food consumption, and BMI. b Prevalence=exp (coefficient).\n",
      "    unique: PMC2459142/PMC2459142.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2459287.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2464454.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2464569.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2475592.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2475600.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: G6PC mutations accounting for GSD-Ia\n",
      "  footer_text: Numbering according to cDNA sequence in Lei et al. [1993], GeneBank Accession Number U01120. Nucleotide +1 is the A of the ATG initiation codon (which is codon +1). The domains for missense and nonsense mutations are based on the finding that human G6PC is anchored in the ER by 9 transmambrane helices [Pan et al., 1998a, 1998c]. NH2, amino terminal domain; H, helical transmembrane domain; C, cytoplasmic loop; L, luminal loop; AS, active site; COOH, carboxyl terminal domain.\n",
      "    unique: PMC2475600/PMC2475600.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2475600.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2475600.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Prevalent G6PC mutations in GSD-Ia patients\n",
      "  footer_text: \n",
      "    unique: PMC2475600/PMC2475600.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2475702.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2475706.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2478639.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Consensus radiation dose-response signature\n",
      "  footer_text: * not in time-independent dose signature - overlap of 6hr and 24hr signatures only. 1 Data from (6); X= gene not present on array; no=no change in prior experiment; YES= same direction change in prior experiment. 2 Data from (8); X= gene not present on array; no=no change in prior experiment; YES= same direction change in prior experiment. 3 Data from (15); YES= reported in human radiation signature. (All genes presumed to be tested since whole genome platform was used.) 4 Literature search for TP53 regulation of genes using MedMiner (31). No= no reported relationship found; YES= reported regulation by p53.\n",
      "    unique: PMC2478639/PMC2478639.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2478639.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2478639.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2480524.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2480524.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2480524.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Performance of the 74-gene classifier in predicting all five doses for samples assayed six and 24 hours after exposure (upper) or for predicting four dose ranges for the same samples (lower).\n",
      "  footer_text: \n",
      "    unique: PMC2478639/PMC2478639.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2478698.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2480524.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means (and standard deviations) of numbers of items observed eaten (in any amount more than none), and numbers of items reported eaten (in any amount more than none), for two school meals (breakfast, lunch), over children in the four groups 1,2\n",
      "  footer_text: 1 F-MPD interviews used Forward-order prompts (morning-to-evening) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; R-MPD interviews used Reverse-order prompts (evening-to-morning) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; F-ESD interviews used Forward-order prompts for meals/snacks eaten and were conducted in the Evening about that Same Day’s intake. 2 Means on the same row with different superscript letters differed significantly. 3 Analysis of variance failed to detect a significant difference in the numbers of items observed eaten between the four groups, F (3, 282) = 0.63, p = 0.5946. 4 Analysis of variance detected a significant difference in the numbers of items reported eaten between the four groups, F (3, 282) = 3.94, p = 0.0089. Post-hoc two-sample t tests indicated that significantly more items were reported eaten for F-MPD1 interviews than for R-MPD interviews, t(161) = 2.9628, p = 0.0035, and for F-ESD interviews than for R-MPD interviews t(125) = 2.7956, p = 0.0060.\n",
      "    unique: PMC2480524/PMC2480524.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number (and percent) of types of intrusion1 – stretch and confabulation2 – for each of four groups3 by meal component\n",
      "  footer_text: 1 An intrusion is an item that was reported eaten but was not observed eaten at that school meal. Examples of items reported eaten that were classified as intrusions because they differed from items observed eaten included fruit juices (e.g., apple observed, orange reported), milk flavors (e.g., white observed, chocolate reported), dry cereals (e.g., flake-shaped observed, donut-shaped reported), and vegetables (e.g., broccoli observed, green beans reported). Examples of items reported eaten that were classified as matches because they were similar to items observed eaten included all types of pizza (e.g., cheese, pepperoni) and all types of white milk (e.g., whole, 1%, skim). 2 An intrusion was further classified either as a stretch if it referred to an item that was observed on the child’s tray for that meal, but for which the amount observed eaten was none, or as a confabulation if it referred to an item that was not observed on the child’s tray for that meal. 3 F-MPD interviews used Forward-order prompts (morning-to-evening) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; R-MPD interviews used Reverse-order prompts (evening-to-morning) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; F-ESD interviews used Forward-order prompts for meals/snacks eaten and were conducted in the Evening about that Same Day’s intake.\n",
      "    unique: PMC2480524/PMC2480524.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Predicted probabilities (and standard errors) from logistic regression models of the likelihood that reported items were intrusions, confabulations, and stretches, and the likelihood that intrusions were confabulations, for each of the four groups 1,2,3\n",
      "  footer_text: 1 F-MPD interviews used Forward-order prompts (morning-to-evening) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; R-MPD interviews used Reverse-order prompts (evening-to-morning) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; F-ESD interviews used Forward-order prompts for meals/snacks eaten and were conducted in the Evening about that Same Day’s intake. Intrusions were items reported as eaten that had not been observed eaten at that meal. Confabulations were intrusions that were not observed on the child’s meal tray. Stretches were intrusions that were observed on the child’s meal tray 2 For each school meal, observed amounts and reported amounts for each item were recorded in terms of servings of standardized school portions. A reported item was an intrusion if the amount reported eaten was any amount more than none when the observation indicated that none of the item was eaten. 3 Predicted probabilities on the same row with different superscript letters differed significantly. 4 F-MPD1 versus F-ESD (p < 0.001); F-MPD2 versus F-ESD (p < 0.001); R-MPD versus F-ESD (p = 0.002); F-MPD1 versus R-MPD (p = 0.0102). 5 F-MPD1 versus F-ESD (p < 0.001); F-MPD2 versus F-ESD (p < 0.001); R-MPD versus F-ESD (p < 0.001); F-MPD1 versus R-MPD (p = 0.0165). 6 No significant differences. 7 F-MPD1 versus F-ESD (p < 0.001); F-MPD2 versus F-ESD (p = 0.001); R-MPD versus F-ESD (p = 0.011). Furthermore, the likelihood that an intrusion was a confabulation was significantly higher for White children than for Black children (p = 0.032).\n",
      "    unique: PMC2480524/PMC2480524.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Number (and percent) of liking ratings1 for each of the four groups2 by item classification3 (match, omission, stretch, confabulation)\n",
      "  footer_text: 1 Refer to the Appendix for the specific prompts used to obtain liking ratings for each study. 2 F-MPD interviews used Forward-order prompts (morning-to-evening) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; R-MPD interviews used Reverse-order prompts (evening-to-morning) for meals/snacks eaten and were conducted in the Morning about the Previous Day’s intake; F-ESD interviews used Forward-order prompts for meals/snacks eaten and were conducted in the Evening about that Same Day’s intake. 3 A match was an item that was observed eaten (in any amount more than none) and reported eaten (in any amount more than none) at a meal. An omission was an item that was observed eaten (in any amount more than none) but not reported eaten at a meal. An intrusion (an item reported eaten [in any amount more than none] that was not observed eaten) was further classified either as a stretch if it referred to an item that was observed on the tray for that meal, but for which the amount observed eaten was none, or as a confabulation if it referred to an item that was not observed on the tray for that meal. 4 A liking rating was not obtained for 12 matches, 88 omissions, 1 stretch, and 12 confabulations.\n",
      "    unique: PMC2480524/PMC2480524.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2480524.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2480524.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2481383.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2481507.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2481507.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: \n",
      "  caption_text: Details by study relevant to the current analyses\n",
      "  footer_text: 1 For inter-reviewer reliability (IRR), one dietitian interviewed a child while the other dietitian(s) sat behind the child and completed an interview form according to what was heard. At the end of the interview, the listening dietitian(s) could ask the child questions. Results on 8 children indicated that only one additional question would have been asked if a different dietitian had conducted the interview. (These 8 interviews were used for IRR only.) In addition, the Principal Investigator (PI) assessed QCI by reviewing the interview form, audio recording, and typed transcript for a random 5% of each dietitian’s interviews and verified adherence to the interview protocol across dietitians. 2 The interview form, audio recording, and typed transcript from one interview conducted each week by each interviewer was randomly selected and reviewed by another interviewer who completed a QCI checklist by marking adequate, needs improvement, or not applicable for each of 53 items. Each QCI checklist was reviewed by the PI and interviewer. Across 51 QCI checklists (21% of the 242 interviews), 95% of items (2,133 of 2,249) that were not rated not applicable were rated adequate; this indicates that interviewers adequately followed the interview protocols. 3 The interview form, audio recording, and typed transcript from one interview conducted each evening was randomly selected and reviewed by another dietitian, who completed a QCI checklist by marking adequate, needs improvement, or not applicable for each of 56 items. Each QCI checklist was reviewed by the PI and interviewer. Across 22 QCI checklists (32% of the 69 interviews), 97% of items (937 of 970) that were not rated not applicable were rated adequate; this indicates that interviewers adequately followed the interview protocols. 4 Omission rate = (omissions / omissions + matches)× 100; it is the proportion of items observed eaten but not reported eaten. Values range from 0% to 100%, with higher rates indicating more omissions. 100% − omission rate = Match rate. 5 In the original publications of the three validation studies, items were given meal-component weights (combination entrée = 2, condiment = 0.33, remaining meal components = 1) so that errors for condiments (e.g., ketchup) counted less than errors for most meal components, and errors for combination entrées (e.g., cheeseburger) counted more. 6 Intrusion rate = (intrusions / intrusions + matches)× 100; it is the proportion of items reported eaten but not observed eaten. Values range from 0% to 100%, with higher rates indicating more intrusions.\n",
      "    unique: PMC2480524/PMC2480524.xml::T5-0\n",
      "\n",
      "PMC0024XXXXX/PMC2481383.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Pediatric modified RIFLE (pRIFLE) criteria\n",
      "  footer_text: \n",
      "    unique: PMC2481383/PMC2481383.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2481507.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Coding of fields\n",
      "  footer_text: \n",
      "    unique: PMC2481507/PMC2481507.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Concordance between clinicians and interviewers\n",
      "  footer_text: a Calculated using only cases with entries from both data sources. b Excludes patients classified as “other” or “multi-ethnic”.\n",
      "    unique: PMC2481507/PMC2481507.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2481516.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Strata of the distal medulla and their lamina cell terminals\n",
      "  footer_text: Data from Fischbach and Dittrich (1989) and Bausenwein et al. (1992)\n",
      "    unique: PMC2481516/PMC2481516.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Numbers of pre- and postsynaptic sites at medulla terminals\n",
      "  footer_text: 1 Difference between highest and lowest number, as percentage of mean. All seven cells were located in all three columns. We had difficulty however to find the processes of remaining cells, especially C2, in all columns. As a result it remains a formal possibility that these cells are not present in all columns.\n",
      "    unique: PMC2481516/PMC2481516.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2483306.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Estimation for parameters in the Markov model\n",
      "  footer_text: IDU, injection drug user; HCC, hepatocellular carcinoma\n",
      "    unique: PMC2483306/PMC2483306.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2483306.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2483306.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2483306.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2488963.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Medical costs of hepatitis B-associated illnesses and vaccination costs\n",
      "  footer_text: a All medical costs are expressed in 2003 U.S. dollars. b Only medical costs for acute infection are one-time costs.\n",
      "    unique: PMC2483306/PMC2483306.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The baseline cost–benefit result of the syringe exchange program-delivered hepatitis B vaccination involving 1964 IDUs in Chicago IL and Hartford and Bridgeport CT\n",
      "  footer_text: HBV, hepatitis B virus; IDU, injection drug user; QALY, quality-adjusted life year\n",
      "    unique: PMC2483306/PMC2483306.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2484123.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2488960.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2488962.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2488963.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: BHandHLYP/6-311G**, CCSD(T)/aug-cc-pVDZ, CCSD(T)/aug-cc-pVTZ and BD(T)/aug-cc-pVDZ Changes in Energy.1\n",
      "  footer_text: 1 Energies computed using BHandHLYP/6-311G** optimized geometries. All values are in kcal·mol−1. 2 Zero-point vibration corrected electronic energy. 3 Energies are relative to isoprene’s global minima, the s-trans conformation. 4 Energies are relative to isoprene’s s-gauche conformation.\n",
      "    unique: PMC2488963/PMC2488963.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: CCSD(T)/aug-cc-pVDZ and BD(T)/aug-cc-pVDZ Changes in Energy from Reactants to Pre-reactive Complexes.1\n",
      "  footer_text: 1 Energies computed using BHandHLYP/6-311G** optimized geometries. All values are in kcal·mol−1. 2 Zero-point vibration corrected electronic energy.\n",
      "    unique: PMC2488963/PMC2488963.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2488963.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2488963.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2488963.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491345.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: CCSD(T)/aug-cc-pVDZ and BD(T)/aug-cc-pVDZ Energies of Activation, relative to the reactants.1\n",
      "  footer_text: 1 Energies computed using BHandHLYP/6-311G** optimized geometries. All values are in kcal·mol−1. 2 Zero-point vibration corrected electronic energy. 3 Calculated using Transition State Theory where Ea = ΔH°‡ + 2RT.\n",
      "    unique: PMC2488963/PMC2488963.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: CCSD(T)/aug-cc-pVDZ and BD(T)/aug-cc-pVDZ Energies of Reaction, relative to reactants.1\n",
      "  footer_text: 1 Energies computed using BHandHLYP/6-311G* * optimized geometries. All values are in kcal·mol−1. 2 Zero-point vibration corrected electronic energy.\n",
      "    unique: PMC2488963/PMC2488963.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2491345.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "tw(T1): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 1\n",
      "  caption_text: Methylation of Polycomb target genes during inflammation, aging and cancer\n",
      "  footer_text: a Genes hypermethylated in the ileum of 8-month-old Gpx-1/2-KO mice versus non-DKO controls b Genes hypermethylated in the ileum of 8-month-old control mice versus 28-day-old control mice c Genes hypermethylated in tumors of B6;129 Gpx-1/2-KO mice d Genes affected by DNA methylation only in tumors e No expression, zero EST counts; low expression, 1–2 EST counts; Yes, 3 or more EST counts in mouse intestinal tissue. The total numbers in this Table were adjusted since for several genes Polycomb binding or expression data were not available. PcG data are from Boyer et al. (ref. 11).\n",
      "    unique: PMC2491345/PMC2491345.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2491491.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Descriptive Statistics for Seattle and Tahoma High Capacity School Buses in the 2005−2006 School Year (>50 Seats)\n",
      "  footer_text: Continuous data are presented as the overall mean (SD). a Summarized values for subject buses represent the students' typical buses although substitute buses are occasionally used during periods of servicing or vehicle inspection. b Descriptive statistics presented for buses with valid data only. c 25% and 27% of all Seattle buses and Seattle subject buses, respectively, were retrofit with aftermarket DOCs. All other DOCs were original equipment installed by the manufacturer. d Remaining buses have unknown engine types.\n",
      "    unique: PMC2491491/PMC2491491.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491491.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491491.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491491.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491491.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491491.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: PM2.5 Concentrations (μg/m3) Measured Aboard the Bus, Lead Vehicle, and Estimates of Self Pollution\n",
      "  footer_text: a Paired trips imply that a bus and lead vehicle drove the same route simultaneously.\n",
      "    unique: PMC2491491/PMC2491491.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Mean (SD) Characteristics of Ambient and Trip Conditions During Monitored Bus Trips\n",
      "  footer_text: \n",
      "    unique: PMC2491491/PMC2491491.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Ordinary Least Squares Multivariate Regression Results for Roadway Concentrations (CLV, μg/m3)\n",
      "  footer_text: \n",
      "    unique: PMC2491491/PMC2491491.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Ordinary Least Squares Multivariate Regression Results For In-Cabin Concentrations (Cbus, μg/m3\n",
      "  footer_text: \n",
      "    unique: PMC2491491/PMC2491491.xml::T5-0\n",
      "\n",
      "PMC0024XXXXX/PMC2491716.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Subject characteristics\n",
      "  footer_text: a FEV1: forced expiratory volume in 1 s. b PC20: provocative dose of methacholine causing a 20% fall in FEV1.\n",
      "    unique: PMC2491716/PMC2491716.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2491717.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Prevalence of demographic characteristics, high depressive symptomatology, and sexual debut timing in the study population\n",
      "  footer_text: a High depressive symptomatology was based on scoring at or above the proportionately adjusted cutpoint on the modified Center for Epidemiologic Studies Depression scale (range 0–27). The cutpoints are 10 for males and 11 for females. b Sexual debut prior to age sixteen was classified as early, debut between ages 16 and 18 was classified as typical, and debut after age 18 (including those who did not debut by Wave III interview) was classified as late.\n",
      "    unique: PMC2491717/PMC2491717.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Sexual debut timing and emerging adult depressive symptomatology: Logistic regression results, Female Participants\n",
      "  footer_text: a High depressive symptomatology was based on scoring at or above the proportionately adjusted depression cutpoint on the modified Center for Epidemiologic Studies Depression scale (range 0–27). The cutpoints are 10 for males and 11 for females. b Sexual debut prior to age sixteen was classified as early, debut between ages 16 and 18 was classified as typical, debut after age 18 (including those who did not debut by Wave III interview) was classified as late. c For multi-category predictor variables, p-values refer to joint tests of statistical significance\n",
      "    unique: PMC2491717/PMC2491717.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Sexual debut timing and emerging adult depressive symptomatology: Logistic regression results, Male Participants\n",
      "  footer_text: a High depressive symptomatology was based on scoring at or above the proportionately adjusted depression cutpoint on the modified Center for Epidemiologic Studies Depression scale (range 0–27). The cutpoints are 10 for males and 11 for females. b Sexual debut prior to age sixteen was classified as early, debut between ages 16 and 18 was classified as typical, debut after age 18 (including those who did not debut by Wave III interview) was classified as late. c For multi-category predictor variables, p-values refer to joint tests of statistical significance\n",
      "    unique: PMC2491717/PMC2491717.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491716.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491717.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491717.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491717.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2491719.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Traditional DMARDs in clinical application.\n",
      "  footer_text: \n",
      "    unique: PMC2491719/PMC2491719.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Presently used biologic DMARDs\n",
      "  footer_text: \n",
      "    unique: PMC2491719/PMC2491719.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2492751.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493046.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493414.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Intercorrelations Among Maternal Depressed Mood, Other Risk Factors, and Covariates\n",
      "  footer_text: * p < .05\n",
      "    unique: PMC2493414/PMC2493414.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlations Between Risk Factors and Categorical Substance Use by Grade\n",
      "  footer_text: * p < .05\n",
      "    unique: PMC2493414/PMC2493414.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491719.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2491719.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493414.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493414.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493414.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493414.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Effects of Risk Factors on Growth in Adolescent Alcohol Use from Grade 7 through Grade 11 (N = 792)\n",
      "  footer_text: Note. Coeff. = Unstandardized coefficient; S.E. = Standard error; df = degrees of freedom * p < .05. Table 3\n",
      "    unique: PMC2493414/PMC2493414.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Effects of Risk Factors on Growth in Adolescent Cigarette Use from Grade 7 through Grade 11 (N = 792)\n",
      "  footer_text: Note. Coeff. = Unstandardized coefficient; S.E. = Standard error * p < .05 + < .10\n",
      "    unique: PMC2493414/PMC2493414.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Effects of Risk Factors on Growth in Adolescent Marijuana Use from Grade 7 through Grade 11 (N = 785)\n",
      "  footer_text: Note. Coeff. = Unstandarized coefficient; S.E. = Standard error * p < .05\n",
      "    unique: PMC2493414/PMC2493414.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493414.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493415.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493415.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2493415.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: An illustration of word-pair stimulus composition for the study and test phases in relation to the level of fan during study and the stimulus or foil type at test\n",
      "  footer_text: \n",
      "    unique: PMC2493415/PMC2493415.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean proportion of responses to each of the four word-pair types in the word-pair recognition test as a function of the number of associates (fan) for word1 and word2 and whether the pair was repeated in Experiment 1\n",
      "  footer_text: Correct responses are shown in bold. Variability is listed in parentheses as 95% confidence intervals.\n",
      "    unique: PMC2493415/PMC2493415.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean proportion of responses to each of the four word-pair types in the word-pair recognition test as a function of the number of associates (fan) for word1 and word2 and whether the pair was repeated in Experiment 2\n",
      "  footer_text: Correct responses are shown in bold. Variability is listed in parentheses as 95% confidence intervals.\n",
      "    unique: PMC2493415/PMC2493415.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493415.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493415.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493415.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493420.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table A\n",
      "  caption_text: Mean median correct response latency to each of the four word-pair types in the word-pair recognition test as a function of the number of associates (fan) for word1 and word2 in Experiment 1\n",
      "  footer_text: Correct responses in bold. Variability is listed in parentheses as 95% confidence intervals.\n",
      "    unique: PMC2493415/PMC2493415.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table B\n",
      "  caption_text: Mean median correct response latency to each of the four word-pair types in the word-pair recognition test as a function of the number of associates (fan) for word1 and word2 in Experiment 2\n",
      "  footer_text: Correct responses in bold. Variability is listed in parentheses as 95% confidence interval.\n",
      "    unique: PMC2493415/PMC2493415.xml::T5-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493416.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493420.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Preflight growth in CeMM in type I cassettes\n",
      "  footer_text: \n",
      "    unique: PMC2493420/PMC2493420.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Growth in flight\n",
      "  footer_text: \n",
      "    unique: PMC2493420/PMC2493420.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493420.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493420.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493421.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Molting in flight\n",
      "  footer_text: \n",
      "    unique: PMC2493420/PMC2493420.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493421.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Table of Means (Standard Errors) for Behavioral Anxiety as a Function of Dyad Type, Person, and Discussion Topic\n",
      "  footer_text: Note. Means sharing a common subscript within each Dyad Type (Interracial and Same-race) do not differ significantly.\n",
      "    unique: PMC2493421/PMC2493421.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493442.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493443.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493452.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493464.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493467.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Experimental electrolyte solutions and applied voltages\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Pore size of AAO tubular membranes produced by 2.7 wt % oxalic acid at 30 and 40 V at 0°C\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Pore size of AAO tubular membranes produced by 3 wt %, 5 wt %, 7 wt %, 10 wt % and 20 wt % sulfuric acid at 12.5, 15 and 20 V at 0°C\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Interpore distance of AAO tubular membranes produced by 3 wt %, 5 wt %, 7 wt %, 10 wt % and 20 wt % sulfuric acid at 12.5, 15 and 20 V at 0°C\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Interpore distances for AAO tubular membranes anodized by 2.7% oxalic acid at 30 and 40 V at 0°C\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: AAO Growth rate for each sulfuric acid electrolyte concentration\n",
      "  footer_text: \n",
      "    unique: PMC2493467/PMC2493467.xml::T6-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493470.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant characteristics at baseline.\n",
      "  footer_text: * 1 female control, 1 male control, and 2 male exercisers were missing baseline DEXA. † T-test comparing baseline characteristics for female exercisers versus controls. ‡ T-test comparing baseline characteristics for male exercisers versus controls.\n",
      "    unique: PMC2493470/PMC2493470.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Geometric means (95% confidence intervals) for the effect of exercise on CRP: differences at baseline and after 12 months of exercise intervention.\n",
      "  footer_text: † p-value for exercisers versus controls at baseline; ‡ p-value for changes from baseline to 12months, among exercisers versus controls, with adjustment for baseline values.\n",
      "    unique: PMC2493470/PMC2493470.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: CRP levels at baseline and 12 months in exercise intervention and control participants: stratified by baseline BMI.\n",
      "  footer_text: † P-value for difference in CRP changes from baseline to twelve months in exercise intervention versus control stratified by BMI group, controlling for baseline CRP values.\n",
      "    unique: PMC2493470/PMC2493470.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493467.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493470.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493470.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493470.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2493538.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Correlations Among B-YAACQ Scores, Measures of Past-Month Alcohol Consumption, and AUDIT Scores At Baseline and 6-Week Follow-up\n",
      "  footer_text: Note. All correlations significant at p < .01. Correlations in bold represent concurrent correlations. ns range from 279 to 291 due to missing data. * defined as 5+ drinks per occasion for men; 4+ drinks per occasion for women.\n",
      "    unique: PMC2493538/PMC2493538.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493539.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Data processing and structure refinement statistics\n",
      "  footer_text: 1 Highest resolution shell (2.39-2.31 Å). 2 Rsymm = Σ |Ih - <Ih>|/ Σ Ih, where <Ih> is the average intensity over symmetry equivalents. 3 Rwork and Rfree = Σ | | Fo | - | Fc | |/ Σ | Fo | for the working set and test set (6%) of reflections.\n",
      "    unique: PMC2493539/PMC2493539.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493570.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Kinetic Constantsa for Clarithromycin Transport by SCC-25 Cells and Gingival Fibroblasts\n",
      "  footer_text: a Km (Michaelis constant) and Vmax (maximum transport velocity) were determined by Lineweaver-Burk analysis of transport activity assayed in the presence of several different concentrations of clarithromycin (8 to 50 μg/ml) during the rapid initial phase of uptake (first 3 minutes). The Cellular/Extracellular concentration ratio was determined in medium containing 2 μg/ml clarithromycin. The data represent the mean ± SEM of four experiments with SCC-25 cells and three experiments with gingival fibroblasts. b Values within columns are significantly different (P < 0.05, t-test) c Values within columns are significantly different (P < 0.05, Mann-Whitney rank sum test)\n",
      "    unique: PMC2493570/PMC2493570.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493571.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493613.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2493614.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493538.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493539.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493570.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493614.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Metalloproteinase-independent biological activity of TIMPs\n",
      "  footer_text: \n",
      "    unique: PMC2493614/PMC2493614.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2493615.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinical characteristics of 127 cases and 214 controls.\n",
      "  footer_text: Chi-square likelihood ratio p-values from conditional logistic regression analysis, adjusted for age at index biopsy, date of index biopsy, and duration of follow-up and weighted based on the batch-quota and counter-matched sampling. For progression interval and follow-up biopsies, p-value is from t-tests. * 413 controls were originally matched to cases on age at EH diagnosis, date of EH diagnosis, and duration of follow-up, based on the original community diagnoses of EH. Because the 172 controls who were ineligible based on the pathology panel diagnoses are not included above, the distribution of age, date, and follow-up interval among the 241 eligible controls above differs slightly from the distribution among all 413 potential controls. The conditional logistic regression models retained individual matching on age at index biopsy, date of index biopsy, and duration of follow-up interval. ** Not mutually exclusive.\n",
      "    unique: PMC2493615/PMC2493615.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: PTEN expression at index biopsy for 115 cases and 193 controls*.\n",
      "  footer_text: * PTEN expression status at index biopsy was unavailable for 11 cases and 27 controls. IHC was unsuccessful for 12 cases (9.4%) and 21 controls (9.8%). P-value for case-control difference: 0.85.\n",
      "    unique: PMC2493615/PMC2493615.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: PTEN expression by panel WHO EH classification at index biopsy for 115 cases and 193 controls.\n",
      "  footer_text: PTEN expression status at index biopsy was unavailable for 11 cases and 27 controls. IHC was unsuccessful for 12 cases (9.4%) and 21 controls (9.8%). * P-value based on Chi-square test (PTEN status vs. panel WHO classification) among cases and controls separately.\n",
      "    unique: PMC2493615/PMC2493615.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2493615.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Rate ratios of developing endometrial carcinoma after a diagnosis of EH according to PTEN expression (normal vs. null) at index biopsy, combined with panel WHO classification.\n",
      "  footer_text: EH = endometrial hyperplasia; DPEM = disordered proliferative endometrium; SH = simple hyperplasia; CH = complex hyperplasia; AH = atypical hyperplasia; RR = rate ratio; CI = confidence interval. Rate ratios are based on conditional logistic regression analysis adjusted for age at index biopsy, date of index biopsy, interval between EH and carcinoma (or censoring, for controls), and weighted based on the batch-quota and counter-matched sampling. All controls were diagnosed with EH at the same age and date as the cases and remained at risk (i.e., no hysterectomy or uterine cancer) for at least as long as the progression interval of the cases. *Adjusted for age at index biopsy, date of index biopsy, progression interval, BMI at index biopsy, follow-up biopsies, and MPA-based treatment for EH. PTEN expression status at index biopsy was unavailable for 11 cases and 27 controls (RR=0.81, 95% CI, 0.23-2.86). IHC was unsuccessful for 12 cases and 21 controls (RR=0.67, 95% CI, 0.21-2.14). IHC was unsuccessful for 3 cases and 13 controls with DPEM (RR=0.27, 95% CI, 0.05-1.48), 7 cases and 8 controls with SH or CH (RR=0.79, 95% CI, 0.15-4.12), and 2 cases with AH (RR N/A).\n",
      "    unique: PMC2493615/PMC2493615.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: PTEN expression at index biopsy vs. hysterectomy for 104 cases with successful IHC in both specimens.\n",
      "  footer_text: PTEN expression status was unavailable for hysterectomy specimens from 3 cases, index biopsy specimens from 4 cases, and both index biopsy and hysterectomy specimens from 7 cases. IHC was unsuccessful for index biopsy specimens from 11 cases, for hysterectomy specimens from 8 cases, and both index biopsy and hysterectomy specimen for 1 case. P-value is based on Chi-square McNemar’s test.\n",
      "    unique: PMC2493615/PMC2493615.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: PTEN mutations in paired index biopsy – cancer specimens from 4 cases with PTEN-null glands in both their index biopsies and hysterectomy specimens.\n",
      "  footer_text: Dx, diagnosis; CH, complex hyperplasia; AH, atypical hyperplasia; DPEM, disordered proliferative endometrium; AC, endometrioid adenocarcinoma.\n",
      "    unique: PMC2493615/PMC2493615.xml::T6-0\n",
      "\n",
      "PMC0024XXXXX/PMC2494536.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2494579.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and anthropometric data of 18 GHRH-R mutant (GHRH-R mut.) individuals and two control groups. Control B was used only for liver size\n",
      "  footer_text: * P < 0·0001 in comparison to GHRH-R mutant group. Values are means ± SD. BMI, body mass index; BSA, body surface area.\n",
      "    unique: PMC2494579/PMC2494579.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Absolute sizes of organs in GHRH-R mutant (GHRH-R mut.) and control subjects\n",
      "  footer_text: Values are means ± SD. Significant correlations are in bold characters.\n",
      "    unique: PMC2494579/PMC2494579.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Abdominal sizes of organs corrected by stature (S), weight (W) and body surface area (BSA) in patients with GHRH-R mutant (GHRH-R mut.) and control subjects\n",
      "  footer_text: Significant correlations are in bold characters.\n",
      "    unique: PMC2494579/PMC2494579.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494579.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494579.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494579.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494579.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Correlation between sizes of organs and height; sizes of organs and body surface area (BSA) of all the subjects (pooled GHRH-R mutant and control subjects)\n",
      "  footer_text: Values are means ± SD. Significant correlations are in bold characters.\n",
      "    unique: PMC2494579/PMC2494579.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2494595.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics of female and male participants*\n",
      "  footer_text: * Values for continuous variables are means ± SEM, and for categorical variables, percentages † Servings were calculated based on the standardized serving sizes from the Dietary Guidelines for Americans ‡ We tested whether the baseline data was statistically different by sex, by computing two-sample T-tests for continuous variables and the Fisher's exact test for the race variable; dietary intakes were log-transformed prior to hypothesis testing to improve normality. P<0.05 was considered statistically significant.\n",
      "    unique: PMC2494595/PMC2494595.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effect of experimental diets on serum β-glucuronidase activity, stratified by sex and day on experimental diet\n",
      "  footer_text: Abbreviations: F&V, fruit and vegetable diet; CI, confidence interval * Back-log-transformed least-square means ±SE of serum β-glucuronidase activity. Means are adjusted for: sex, days on experimental diet, UGT1A1 genotype, order of assigned experimental diet, carryover effect, β-glucuronidase activity during the pre-intervention period, 2-way interactions between sex, days on experimental diet, UGT1A1 genotype, and experimental diet † Ratio was calculated by exponentiating the difference of the log-transformed adjusted, least-square means ‡ Serum β-glucuronidase activity was significantly higher during F&V than during basal diet, P<0.05 § β-glucuronidase activity averaged over days 8 and 15 of experimental diet\n",
      "    unique: PMC2494595/PMC2494595.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Serum β-glucuronidase activity during pre-intervention and experimental periods, stratified by sex and day on experimental diet\n",
      "  footer_text: Abbreviations: F&V, fruit and vegetable diet; CI, confidence interval * Geometric means ± SE of β-glucuronidase activity; means were unadjusted because paired t-tests were used to compare values † Experimental diet versus Pre-intervention ratios were calculated by exponentiating the difference of the log-transformed (unadjusted) geometric means. Paired t-tests, inherently adjusting for within-person characteristics, were used to calculate the significance of the ratios. ‡ Serum β-glucuronidase activity was significantly different for the pre-intervention than the experimental diet, P<0.05 § For experimental diets, data stratified by sex represent values averaged over days 8 and 15 of experimental diet ∥ Pre-intervention β-glucuronidase activity was sample at one time point\n",
      "    unique: PMC2494595/PMC2494595.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2494699.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494595.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494595.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494595.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494699.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Plasma metabolite and insulin concentrations in 12 week old and 22 week old control and diabetic rats (n = 7).\n",
      "  footer_text: * p < 0.05 between control and diabetic group. ‡ p < 0.05 between young and old groups.\n",
      "    unique: PMC2494699/PMC2494699.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Plasma metabolite and insulin concentrations in 12 week old control and diabetic rats treated with a low or high dose of the PPARα agonist WY14,643 (n = 4–7).\n",
      "  footer_text: * P < 0.05 between control and diabetic group. # P < 0.05 between treated and untreated groups.\n",
      "    unique: PMC2494699/PMC2494699.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: mRNA levels in 12 week old and 22 week old control and diabetic rats and 12 week old rats treated with WY14,643 measured by QPCR (n = 3).\n",
      "  footer_text: * p < 0.05 between control and diabetic groups. ‡ p < 0.05 between young and old groups. # p < 0.05 between treated and untreated groups.\n",
      "    unique: PMC2494699/PMC2494699.xml::T3-0\n",
      "\n",
      "PMC0024XXXXX/PMC2494872.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2494875.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2495052.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2495056.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494699.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2494699.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Rotated factor pattern (multiplied by 100 and rounded to the nearest integer) for first 3 factors from principal components analysis of all line items from the NIH-AARP Diet and Health Study food frequency questionnaire.\n",
      "  footer_text: \n",
      "    unique: PMC2495056/PMC2495056.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Baseline characteristics by quintile of factor scores for men and for women (all values are totals, percents, or means).\n",
      "  footer_text: a p for linear trend < 0.05 b p for linear trend < 0.01 c p for linear trend < 0.001 d p for linear trend <0.0001 e Chi square test of independence of factor quintile against level of categorical variable < 0.0001 for all categorical variables for each of the three factors.\n",
      "    unique: PMC2495056/PMC2495056.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2495056.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2495056.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Age and multivariate-adjusted1 hazard ratios and 95% confidence intervals (CI) for colorectal cancer (2,151 cases in men and 959 in women) in the AARP Diet and Health Study by quintiles of factor scores.\n",
      "  footer_text: 1 Multivariate models adjusted for ethnicity, tobacco use, physical activity, body mass index, education, family history of colon cancer, regular NSAIDs use, and, in the case of women, use of menopausal hormone therapy. 2 Relative risks estimated using Cox proportional hazards regression.\n",
      "    unique: PMC2495056/PMC2495056.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Multivariate-adjusted1 hazard ratios and 95% confidence intervals (CI) for cancer of the proximal colon (826 cases in men, 459 in women), distal colon (646 cases in men, 230 in women), and rectum (631 cases in men, 258 in women) in the AARP Diet and Health Study by quintiles of factor scores.\n",
      "  footer_text: 1 Multivariate models adjusted for ethnicity, tobacco use, physical activity, body mass index, education, family history of colon cancer, regular NSAIDs use, and, in the case of women, use of menopausal hormone therapy. 2 Relative risks estimated using Cox proportional hazards regression.\n",
      "    unique: PMC2495056/PMC2495056.xml::T4-0\n",
      "\n",
      "PMC0024XXXXX/PMC2496967.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2495056.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2495056.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2496968.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497421.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Main events in the evolution of the disease.\n",
      "  footer_text: \n",
      "    unique: PMC2497421/PMC2497421.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2497431.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497432.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497437.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497438.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected characteristics of cervical and vulvar squamous cell carcinoma cases\n",
      "  footer_text: * Represents the percentage of tested individuals\n",
      "    unique: PMC2497438/PMC2497438.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: IL2 tagSNP characteristics, smoking prevalence, and results from tests of independence of IL2 tagSNPs and cigarette smoking in controls\n",
      "  footer_text: * rs number refers to the National Center for Bioinformatics (NCBI) dbSNP build 127 reference sequence number. † Locations are with respect to the first nucleotide position in the NCBI GenBank entry: accession number AF359939. ‡ Location of tagSNP within gene: 5’flanking is upstream of the first exon of the gene, exon is in the coding region of the gene, intron is between coding regions of the gene. § Prevalence of current cigarette smoking among all controls (n=236) who carried at least one copy of the variant allele for each tagSNP ∥ OR, odds ratio; CI, confidence interval. The ORs are the exponentiated joint effect parameters for cigarette smoking and tagSNP genotype among controls from the full model (39) assuming a log-additive genetic model. These can be interpreted as the association between IL2 tagSNP genotypes and cigarette smoking among controls. ** Likelihood ratio test p value (39)\n",
      "    unique: PMC2497438/PMC2497438.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Interaction odds ratios between IL2 genotypes and cigarette smoking on cervical and vulvar cancer risk\n",
      "  footer_text: * rs number refers to the National Center for Bioinformatics (NCBI) dbSNP build 127 reference sequence number. † IOR, interaction odds ratio; CI, confidence interval. ‡ Dominant genetic model. § Recessive genetic model.\n",
      "    unique: PMC2497438/PMC2497438.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Interaction odds ratios between IL2 haplotypes and cigarette smoking on cervical and vulvar cancer risk based on a log-additive model\n",
      "  footer_text: * Alleles in each haplotype are listed from 5′ to 3′ (rs2069762, rs2069763, rs2069777, and rs2069778). Variant alleles are underlined. † IOR, interaction odds ratio; CI, confidence interval. Calculated assuming a log-additive genetic model.\n",
      "    unique: PMC2497438/PMC2497438.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Interaction odds ratios between IL2 diplotypes and cigarette smoking on cervical and vulvar cancer risk\n",
      "  footer_text: * The two haplotypes carried on each chromosome are separated by the “/”. Alleles in each haplotype are listed from 5′ to 3′ (rs2069762, rs2069763, rs2069777, and rs2069778). Variant alleles are underlined. † IOR, interaction odds ratio; CI, confidence interval.\n",
      "    unique: PMC2497438/PMC2497438.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Main effect of each tagSNP on cervical and vulvar cancer risk\n",
      "  footer_text: * rs number refers to the National Center for Bioinformatics (NCBI) dbSNP build 127 reference sequence number. † OR, odds ratio; CI, confidence interval. All controls were seropositive for HPV16 or HPV18 L1 antibodies.\n",
      "    unique: PMC2497438/PMC2497438.xml::T6-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497438.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497442.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2497439.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497442.xml\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Peak principal bone strains (Mean ± SD, in με) during mastication\n",
      "  footer_text: Max= tension; Min = compression; Shear = Max + Min (absolute value); Direc: direction of tension relative to the midline line (PM and MI) or the occlusal plane (MM, angle in degrees) * Independent t tests (Sham-Reduction comparison); # ANOVA/Tukey tests (Gauge-site comparision).\n",
      "    unique: PMC2497442/PMC2497442.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2497445.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497446.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497447.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2497450.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient Characteristics\n",
      "  footer_text: Abbreviations: ESFT, Ewing sarcoma family of # Age at time of apheresis * Given between apheresis and immunotherapy ¤ for surviving patients\n",
      "    unique: PMC2497450/PMC2497450.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Vaccine Peptide Sequences\n",
      "  footer_text: a t(11;22)(q24;q12) Ewing’s Sarcoma Family of Tumors (ESFT) b t(2;13)(q35;q14) Alveolar Rhabdomyosarcoma (AR)\n",
      "    unique: PMC2497450/PMC2497450.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Characteristics of surviving patients\n",
      "  footer_text: Abbreviation: dx, diagnosis; NA, not applicable; LN, lymph node; SX, surgery; SCT, stem cell transplant; XRT, radiation treatment; CR, complete response; PR, partial response; PD, progression of disease; IL-2, interleukin 2; mod, moderate; OS, overall survival; AWD, alive with disease; NED, no evidence of disease. * NED since initiating immunotherapy\n",
      "    unique: PMC2497450/PMC2497450.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497450.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497450.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497450.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2497451.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: IC50 and IC90 values for competitive antagonism of human nAChRs by various neuromuscular blocking agents. Macroscopic currents were activated by 300 μM ACh, −50 mV. CE50 values are estimates of the clinical concentrations of the antagonists that decrease muscle tension by 50% based on pharmacokinetic/pharmacodynamic modeling (32,33). They are not corrected for the free fraction of drug after accounting for protein binding which ranges from 0.31 for vecuronium to 0.63 for atracurium (33).\n",
      "  footer_text: \n",
      "    unique: PMC2497451/PMC2497451.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Fractional analysis (Equation 3) of the extent of synergy of between combinations of competitive antagonists. Synergy is indicated by fractional values significantly less then 1.0.\n",
      "  footer_text: \n",
      "    unique: PMC2497451/PMC2497451.xml::T2-0\n",
      "\n",
      "PMC0024XXXXX/PMC2497455.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Frequency of degeneration among unmyelinated axons.\n",
      "  footer_text: [From Kissin et al. (66)]\n",
      "    unique: PMC2497455/PMC2497455.xml::T1-0\n",
      "\n",
      "PMC0024XXXXX/PMC2497466.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2502065.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497451.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497451.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2497455.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2502065.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Effect of adolescent fluoxetine treatment on the number of BrdU-labeled cells in SFR mice\n",
      "  footer_text: 1 Dose of fluoxetine (Fluox) in mg/kg/day. Data are mean ± SD and were compared by one-way ANOVA. Statistical differences (16 versus 25 mg/kg of Fluox) were resolved post hoc using Tukey Kramer Multiple Comparisons tests. *** p < 0.001 compared with adolescent Fluox (16).\n",
      "    unique: PMC2502065/PMC2502065.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Effect of adult fluoxetine treatment on the number of BrdU-labeled cells in SFR mice\n",
      "  footer_text: 1 Dose of fluoxetine (Fluox) in mg/kg/day. Data are mean ± SD and were compared by one-way ANOVA. Statistical differences were resolved post hoc using Tukey Kramer Multiple Comparisons tests. No statistically significant differences were observed.\n",
      "    unique: PMC2502065/PMC2502065.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Effect of adolescent fluoxetine treatment on the number of BrdU-labeled cells in IMS mice\n",
      "  footer_text: 1 Dose of fluoxetine (Fluox) in mg/kg/day. Data are mean ± SD and were compared by one-way ANOVA. Statistical differences were resolved post hoc using Tukey Kramer Multiple Comparisons tests. No statistically significant differences were observed. nd: not determined.\n",
      "    unique: PMC2502065/PMC2502065.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504020.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504029.xml\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: \n",
      "  caption_text: Food groups used to calculate the Comprehensive Healthy Dietary Pattern score1\n",
      "  footer_text: 1 Range in parentheses; n in brackets. Points were assigned to each category of intake when the dietary pattern scores were calculated. 2 Denotes food groups included in the Simplified Healthy Dietary Pattern. 3 Denotes food groups not included in either the Comprehensive or the Simplified Healthy Dietary Pattern.\n",
      "    unique: PMC2504029/PMC2504029.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2502065.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2502065.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504029.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504029.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Participant characteristics and nutrient intakes across quintiles (Q) of an a priori—defined Healthy Dietary Pattern in the Multi-Ethnic Study of Atherosclerosis1\n",
      "  footer_text: 1 Values are unadjusted x̄ ± SE or percentages, except for macro- and micronutrients, which are adjusted for total energy intake (kcal/d). 2 P for trend across quintiles calculated from linear regression with the a priori Healthy Dietary Pattern modeled continuously (score). 3 Chi-square test for difference among race-ethnic groups. 4 MET, metabolic equivalents.\n",
      "    unique: PMC2504029/PMC2504029.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Geometric mean or odds ratios for select cardiovascular disease risk markers across quintiles (Q) of an a priori—defined Healthy Dietary Pattern in the Multi-Ethnic Study of Atherosclerosis1\n",
      "  footer_text: 1 CRP, C-reactive protein; IL-6, interleukin-6; CAC, coronary artery calcification; ABI, ankle brachial index; LAE, large artery elasticity; SAE, small artery elasticity; ACR, albumin to creatinine ratio; IMT, intima-media thickness; CC, common carotid artery; IC, internal carotid artery. 2 P for trend across quintiles calculated with the Healthy Dietary Pattern modeled continuously (score). 3 All values are geometric x̄; 95% CI in parentheses. 4 Adjusted for energy intake (kcal/d), age (y), race-ethnicity (white, black, Hispanic, Chinese), and study center. 5 Adjusted as for model 1 plus education (<high school, high school, >high school), physical activity (active and inactive leisure in metabolic equivalents per min/wk), smoking (current or not current and pack-years), and supplement use (≥ weekly). Data for LAE and SAE also adjusted for height, weight, systolic blood pressure, and pulse rate. 6 Adjusted as for model 2 plus waist circumference (cm). 7 All values are odds ratios; 95% CI in parentheses. 8 Values calculated in 2380 participants with an Agatston score > 0. 9 All values are x̄ ± SD.\n",
      "    unique: PMC2504029/PMC2504029.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504029.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504029.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504253.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Regression coefficients (or odds ratios) for the relation between select cardiovascular disease (CVD) risk markers and a priori Comprehensive and Simplified Healthy Diet Patterns in the Multi-Ethnic Study of Atherosclerosis1\n",
      "  footer_text: 1 CRP, C-reactive protein; IL-6, interleukin-6; IMT, intima-media thickness; CAC, coronary artery calsification; ABI, ankle brachial index; LAE, large artery elasticity; SAE, small artery elasticity; ACR, albumin to creatinine ratio. 2 Adjusted for energy intake (kcal/d), age (y), race-ethnicity (white, black, Hispanic, Chinese), study center, education (<high school, high school, >high school), physical activity (active and inactive leisure in metabolic equivalents per min/wk), smoking (current or not current and pack-years), and supplement use (≥weekly). 3 Model includes the covariates in footnote 2 and both the difference between scores (Healthy Dietary Pattern score — Simplified Healthy Dietary Pattern score) and the Simplified Healthy Dietary Pattern score. 4 Data were analyzed on the natural log scale. 5 β-coefficient ± SE per 1 SD (10.2) for the Comprehensive Healthy Dietary Pattern score and per 1 SD (3.52) for the Simplified Healthy Dietary Pattern score. 6 P < 0.05. 7 Odds ratio; 95% CI in parentheses (all such values). 8 Estimated from 2380 participants with an Agatston score >0. 9 Data for LAE and SAE also adjusted for height, weight, systolic blood pressure, and pulse rate. Greater elasticity values are favorable.\n",
      "    unique: PMC2504029/PMC2504029.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504253.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: In vitro biofilm biomass\n",
      "  footer_text: a Biomass is given in grams per cm2 of surface area. Values are the mean ± standard deviation for four determinations. b P-value <0.05 compared to the als1/als1 als3/als3 strain.\n",
      "    unique: PMC2504253/PMC2504253.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504411.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinical features in 1,033 consecutive adults with ALF in the United States (1998-2007)\n",
      "  footer_text: APAP= Acetaminophen, HAV= Hepatitis A virus, HBV= Hepatitis B virus, ALF = acute liver failure, ALT = alanine aminotransferase (Data provided by William Lee, MD, and the US ALFSG, July 2007)\n",
      "    unique: PMC2504411/PMC2504411.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504411.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504411.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504411.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Treatable causes of acute liver failure\n",
      "  footer_text: HELLP – hemolysis, elevated liver enzymes, and low platelet count\n",
      "    unique: PMC2504411/PMC2504411.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Diagnosis and management of acetaminophen overdose\n",
      "  footer_text: \n",
      "    unique: PMC2504411/PMC2504411.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Acetaminophen content of selected narcotic analgesics and over-the-counter medications\n",
      "  footer_text: * Contact Poison Control Center for exact dosage of other constituents. Note: This list does not contain all acetaminophen-containing products. ACM - acetaminophen\n",
      "    unique: PMC2504411/PMC2504411.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Management of cerebral edema in acute liver failure\n",
      "  footer_text: ICU – intensive care unit, CT – computerized tomography, ICP – intracranial pressure, INR – international normalized ratio, ALF – acute liver failure\n",
      "    unique: PMC2504411/PMC2504411.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Management of coagulopathy in acute liver failure\n",
      "  footer_text: * Not FDA approved for use in this setting DIC – disseminated intravascular coagulation, PT – prothrombin time, INR – international normalized ratio, PTT – partial thromboplastin time, CBC – complete blood count, ALF – acute liver failure, GI – gastrointestinal, H2 – Histamine-2, FFP – fresh frozen plasma, ICP – intracranial pressure, DVT – deep venous thrombosis, PE – pulmonary embolus.\n",
      "    unique: PMC2504411/PMC2504411.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504492.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic Variable Descriptive Statistics Overall and by Nation\n",
      "  footer_text: \n",
      "    unique: PMC2504492/PMC2504492.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Shift Variables Descriptive Statistics Overall and by Nation\n",
      "  footer_text: \n",
      "    unique: PMC2504492/PMC2504492.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Study Variable Descriptive Statistics and Correlations\n",
      "  footer_text: * Significant at the.05 level (two-tailed) ** Significant at the.01 level (two-tailed) N= 906\n",
      "    unique: PMC2504492/PMC2504492.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Regression of Off-shift Well-being Outcomes on Statistical Controls and Shift Characteristics\n",
      "  footer_text: a Step 1 df = 5, 814; Step 2 df = 14, 805. b Step 1 df = 5, 815; Step 2 df = 14, 806. c Step 1 df = 5, 807; Step 2 df = 14, 798. * p <.05 ** p <.01 *** p <.001\n",
      "    unique: PMC2504492/PMC2504492.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504492.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504492.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504492.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504492.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2504519.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504524.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504689.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Anti-TB activities, cytotoxicity (IC50) and selectivity indices (SIs) of extract partitions and the isolated polyynes 1-5 from A. sinensis\n",
      "  footer_text: a Minimum inhibitory concentration defined as the concentration at which the samples exhibit an inhibition of ≥ 90% in the MABA assay. b Average ± SD of triplicate determinations. c Selectivity index (SI = IC50/MIC). d Positive control (Rifampin); negative control was solvent (DMSO). n/a = not applicable.\n",
      "    unique: PMC2504689/PMC2504689.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: 1H-NMR data of polyyne 1-5 from A. sinensis [CDCl3, 400 MHz, δH (J Hz)]\n",
      "  footer_text: \n",
      "    unique: PMC2504689/PMC2504689.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The 13C-NMR spectral data of polyynes 1-5, isolated from A. sinensis (CDCl3, 100 MHz)\n",
      "  footer_text: \n",
      "    unique: PMC2504689/PMC2504689.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504695.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504689.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504689.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504689.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2504728.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean (SD) Neuropsychological Test Performance for the Three Age Groups in Experiment 1\n",
      "  footer_text: Note. CVLT = California Verbal Learning Test * = significant group differences † = significant group differences\n",
      "    unique: PMC2504728/PMC2504728.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Factor Loadings for 227 Older Adults and 96 Young Adults\n",
      "  footer_text: Note. CVLT = California Verbal Learning Test\n",
      "    unique: PMC2504728/PMC2504728.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean and (SDs) of FL and MTL factor scores in young, old-young, and older adults\n",
      "  footer_text: Note. FL = frontal lobe; MTL = medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Characteristics of Young and Older Adult Groups in Experiment 2\n",
      "  footer_text: Note. FL = frontal lobe; MTL = medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T4-0\n",
      "    unique: PMC2504728/PMC2504728.xml::T4-1\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Mean Proportion Correct (SD) on Source and Item Memory Tasks in Young Adults in Experiment 2\n",
      "  footer_text: Note. FL = frontal lobe; MTL= medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Mean Proportion Correct (SD) on Source and Item Memory Tasks in Older Adults in Experiment 2\n",
      "  footer_text: Note. FL = frontal lobe; MTL= medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T6-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T4-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504728.xml::T8-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504735.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Characteristics of Young Adults in Experiment 3\n",
      "  footer_text: Note.FL = frontal lobe; MTL= medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T7-0\n",
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table 8\n",
      "  caption_text: Mean Proportion Correct (SD) on Source Memory Task in Young Adults in Experiment 3\n",
      "  footer_text: Note. FL = frontal lobe; MTL= medial temporal lobe\n",
      "    unique: PMC2504728/PMC2504728.xml::T8-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504735.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Contaminants Discovered in the Bay Street Neighborhood.\n",
      "  footer_text: Note: RDEC is the Residential Direct Exposure Criteria established by the Rhode Island Department of Environmental Management.\n",
      "    unique: PMC2504735/PMC2504735.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504736.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504739.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Ototoxicity Grading Criteria\n",
      "  footer_text: A loss is defined as a change from baseline at any one frequency * Patient requires hearing aids\n",
      "    unique: PMC2504739/PMC2504739.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Patient Characteristics\n",
      "  footer_text: † PFRT = Posterior Fossa Irradiation, TBRT= Tumor Bed Irradiation\n",
      "    unique: PMC2504739/PMC2504739.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Frequency of Grade 3 or 4 Ototoxicity by Amifostine Administration and Study Cohort\n",
      "  footer_text: † PFRT = Posterior Fossa Irradiation, TBRT= Tumor Bed Irradiation\n",
      "    unique: PMC2504739/PMC2504739.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504739.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504739.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504739.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Grade 3 and 4 Adverse Events Occurring Within 2 Days of Amifostine Administration and Considered At Least Possibly Attributable to Amifostine\n",
      "  footer_text: † Reported is the frequency observed in 62 patients receiving amifostine over 235 courses of treatment.\n",
      "    unique: PMC2504739/PMC2504739.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504742.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504743.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of Tracer Injections in MI Cortex\n",
      "  footer_text: a Values in parentheses represent number of injection penetrations b Wh indicates whiskers, Fp indicates forepaw\n",
      "    unique: PMC2504743/PMC2504743.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504743.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504749.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504751.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504751.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2504749.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant Characteristics\n",
      "  footer_text: Note. PPVT-III = Peabody Picture Vocabulary Test-III (standard score); EVT = Expressive Vocabulary Test (standard score); TELD-3 = Test of Early Language Development-3 (standard score); GFTA-2 = Goldman-Fristoe Test of Articulation-2 (standard score); TSO = parent-reported time since initial onset of stuttering (months); %SLD = mean frequency of stuttering-like disfluencies (percent) per 100 words; SSI-3 = Stuttering Severity Instrument-3 (total score); = test scores could not be obtained, because the participant refused to cooperate with further testing.\n",
      "    unique: PMC2504749/PMC2504749.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504751.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Co-infections in breast milk are not associated with clinical or subclinical mastitis\n",
      "  footer_text: a No infectious agent had a statistically significant difference in prevalence between specimens with and without either clinical or subclinical mastitis, using GEE. b Clinical mastitis defined by report of any symptoms of breast inflammation using a standardized questionnaire. c Subclinical mastitis defined by milk Na+ >12 mM [3].\n",
      "    unique: PMC2504751/PMC2504751.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Plasma HIV-1 RNA load, mastitis parameters and milk co-infections as related to detection of HIV-1 RNA or DNA in breast milk\n",
      "  footer_text: a Detection of HIV-1 RNA at ≥ 50 copies/mL lactoserum, and HIV-1 DNA at ≥ 1−10 copies/cell pellet from 1 to 14 mL whole milk [3]. b GEE logistic model adjusted for all variables in the first column, except mycoplasma and mycobacteria. Mycoplasmal detection was too rare to include in the multivariable modeling. Mycobacteria were not included because their detection likely represents contamination from the skin rather than infection within the breast tissue. c Model does not converge due to a proportion estimate of 0.0 or 1.0 in one group.\n",
      "    unique: PMC2504751/PMC2504751.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504754.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504759.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Means and S.D. of the participant groups for age, gender, and full-scale IQ.\n",
      "  footer_text: \n",
      "    unique: PMC2504759/PMC2504759.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504759.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504761.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504761.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2504761.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: PCR primer sequences and GeneBank Accession Numbers\n",
      "  footer_text: \n",
      "    unique: PMC2504761/PMC2504761.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Flow cytometric analysis of peripheral blood cell from wild-type, Fli-1ΔCTA/Fli-1+ heterozygous and Fli-1ΔCTA/Fli-1ΔCTA mice\n",
      "  footer_text: The percentages of B220+ cells were assessed by flow cytometry. Values represent mean ± SD. ‡ indicates p<0.0001 when compared to wild-type. * indicates p<0.05 when compared to Fli-1ΔCTA/Fli-1ΔCTA homozygous mice.\n",
      "    unique: PMC2504761/PMC2504761.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2504765.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2504766.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Studies of the epidemiology of viral-associated wheezing illnesses in infancy and early childhood\n",
      "  footer_text: \n",
      "    unique: PMC2504766/PMC2504766.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Studies of virus detection associated with acute asthma exacerbations in children and adults\n",
      "  footer_text: \n",
      "    unique: PMC2504766/PMC2504766.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2505184.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2505321.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean and standard error of 2-DG (nCi/g) uptake\n",
      "  footer_text: a Indicates significant differences (p < 0.05) between right and left side within groups, and between the right side of the muscimol and saline groups. b Indicates significant differences (p < 0.05) between the right side of the muscimol and saline groups.\n",
      "    unique: PMC2505321/PMC2505321.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504766.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2504766.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2505321.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2505342.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2505356.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Correlations between GZTS Scales and NEO-PI-R Factors and Most Strongly Related Facet.\n",
      "  footer_text: Note. N = 757 to 854 from a BLSA sub-sample of 900 individuals who were administered both tests during the same visit. Last two columns report the specific NEO-PI-R facet with which each GZTS scale is most highly correlated. Correlations over ± .40 are given in bold. GZTS = Guilford-Zimmerman Temperament Survey; NEO-PI-R = Revised NEO Personality Inventory; N = Neuroticism; E = Extraversion; O = Openness to Experience; A = Agreeableness; C = Conscientiousness. * Conscientiousness was estimated using the mean of General Activity and Restraint. Table adapted from Terracciano et al. (39)\n",
      "    unique: PMC2505356/PMC2505356.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Personality predictors of all-cause mortality\n",
      "  footer_text: Note. For all personality scales tested, age and sex were significant covariates. HR = Hazard ratio. * Conscientiousness was estimated using the mean of General Activity and Restraint.\n",
      "    unique: PMC2505356/PMC2505356.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2507862.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2507878.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2507881.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2505356.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2505356.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2507881.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Summary of cell surface expressed m157 mutations in C1498 cells\n",
      "  footer_text: a only five (of 630) clones survived tertiary screening; clones1E8 and DA10 contained multiple mutations. b only those mutations resulting in a nonconservative, in-frame amino acid substitution are listed. Each unique m157 mutation (shown in bold) and its corresponding alanine substitution variant was regenerated by site-directed PCR mutagenesis. c each unique m157 mutation was transduced into C1498 cells and subsequently sorted by flow cytometry for high-level m157 expression; mean fluorescence intensity (MFI) values are listed after subtracting background MFI from parental C1498 cells. MFI for control C1498-m157 cells (wt-m157) is also indicated as a reference. d represents percentage of wt-m157 activity for each mutant after subtracting the background from parental C1498 cells. e positions mapped using PyMOL analysis software and the published PDB structure file for m157 (ID code: 2NYK); see also Figure 8.\n",
      "    unique: PMC2507881/PMC2507881.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2507886.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Human NAT2 alleles\n",
      "  footer_text: Adapted from http://N-acetyltransferasenomenclature.louisville.edu [56]. a Human NAT2 alleles should be written in upper case and italicized. Protein products of the alleles are also upper case but not italicized and the asterisk is omitted. For example, the allele NAT2*4 encodes the protein NAT2 4. NAT2*4 has historically been designated “wildtype”. Since it is the most common occurring alleles in some but not all ethnic groups the designation of “wildtype” allele is somewhat arbitrary. b Reference gene sequence published in Genbank Assession Number X14672. The frequency of individual SNPs and alleles in various ethnic groups is available online from the National Cancer Institute’s SNP500Cancer database. RS refers to reference SNP. Signature SNP for each cluster is italicized. Although additional SNPs have been identified outside the open reading frame [42,43,57], they will not be named until a functional effect is observed. SNPs should be identified by designating “A” of the ATG translation initiation codon as number 1. SNPs upstream of this site are designated by negative numbers and SNPs downstream of this site are designated by positive numbers [56]. c Phenotype assignments reflect most current research but are not necessarily a consensus of all research papers. Evidence exists for heterogeneity within the “slow” acetylator phenotype [58–62].\n",
      "    unique: PMC2507886/PMC2507886.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2507886.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2507886.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of Non-Synonymous SNP-Induced Changes in NAT2 Function & Structure\n",
      "  footer_text: \n",
      "    unique: PMC2507886/PMC2507886.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2512965.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2512970.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2515386.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Crystallographic Data Statistics for the Gal4(1-100)/DNA Complex\n",
      "  footer_text: rmsd = root-mean-square deviation. Values for the outer resolution shell are given in parentheses. Rmerge= ΣΣi | II − <I>Σ/ | <I>, where <I> is the mean intensity of the N reflections with intensities Ii and common indices h,k,l for the native and derivative crystals, respectively. R factor = Σhkl | | Fobs − | k | Fcal/ | | Σhkl | Fobs |, where Fobs and Fcal Are observed and calculated structure factors,respectively. For Rfree the sum is extended over a subset of reflections (10%) that were excluded from all stages of refinement.\n",
      "    unique: PMC2515386/PMC2515386.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: DSC analysis of Gal4(1-100) and mutants\n",
      "  footer_text: \n",
      "    unique: PMC2515386/PMC2515386.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Sedimentation coefficients of Gal4 constructs and DNA complex\n",
      "  footer_text: \n",
      "    unique: PMC2515386/PMC2515386.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515561.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Fold Decrease in Urokinase Plasminogen Activated Receptor (uPAR) Expression in Hkh2 Cells as Compared to HCT116 Cells\n",
      "  footer_text: \n",
      "    unique: PMC2515561/PMC2515561.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515566.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515386.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515386.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515386.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515561.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515566.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Phase-I goals and lifestyle guidelines given to participants\n",
      "  footer_text: DASH, dietary approaches to stop hypertension\n",
      "    unique: PMC2515566/PMC2515566.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Baseline characteristics\n",
      "  footer_text: \n",
      "    unique: PMC2515566/PMC2515566.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Attendance, adherence, and weight loss by race and gender\n",
      "  footer_text: a With missing data imputed using multiple imputation procedures.\n",
      "    unique: PMC2515566/PMC2515566.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Baseline measures and adherence indicators as predictors of change in weight\n",
      "  footer_text: a A negative parameter estimate indicates that increase in the measure predicts greater weight loss (weight loss is a negative change score)\n",
      "    unique: PMC2515566/PMC2515566.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515624.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515566.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515566.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515624.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515624.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: A list of major components of the Hh pathway in vertebrates\n",
      "  footer_text: \n",
      "    unique: PMC2515624/PMC2515624.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of Hh signaling activation in human cancers\n",
      "  footer_text: \n",
      "    unique: PMC2515624/PMC2515624.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of Hh signaling inhibitors\n",
      "  footer_text: \n",
      "    unique: PMC2515624/PMC2515624.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515713.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Descriptive information regarding study\n",
      "  footer_text: \n",
      "    unique: PMC2515713/PMC2515713.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Parameter Estimates for Mean Change in Cortisol\n",
      "  footer_text: *p < .05 **p < .01 *** p < .001 1 Although not pertinent to the hypotheses of the present study, group intercept estimates are reported for completeness.\n",
      "    unique: PMC2515713/PMC2515713.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Association between Aggression and Mean Change in Cortisol\n",
      "  footer_text: † p < .10 * p < .05 **p < .01 *** p < .001 1 Although not pertinent to the hypotheses of the present study, scaling estimates for the intercept, linear slope, and quadratic change are reported for completeness.\n",
      "    unique: PMC2515713/PMC2515713.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515624.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515713.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515713.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515713.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515713.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2515822.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: The Association between Aggression and Mean Change in Cortisol in Maltreated and Nonmaltreated Children\n",
      "  footer_text: † p < .10 * p < .05 ** p < .01 *** p < .001 1 Although not pertinent to the hypotheses of the present study, scaling estimates for the intercept, linear slope, and quadratic change are reported for completeness.\n",
      "    unique: PMC2515713/PMC2515713.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515822.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Oligonucleotide primer pairs and PCR parameters\n",
      "  footer_text: a The top primer is the sense primer and the bottom primer is the antisense primer\n",
      "    unique: PMC2515822/PMC2515822.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2515935.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2516078.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: The effects of amino acid changes on the λmax shift\n",
      "  footer_text: Amino acid residue numbers are those of bovine RH1 pigment. Residues 164, 261, and 269 correspond to 180, 277, and 285 in the human M/LWS pigments, respectively. a Forward amino acid replacements have been defined previously (Yokoyama 2000a,b; Shi et al. 2001; Yokoyama and Tada 2003; Takahashi and Yokoyama 2005). b Takahashi and Ebrey (2003). c Sugawara et al. (2005). d Yokoyama et al. (2005). e Zhukovsky and Oprian (1989). f Sakmar et al. (1989). g Babu et al. (2001). h Yokoyama and Tada (2003, references therein).\n",
      "    unique: PMC2516078/PMC2516078.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516078.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516078.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516078.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: The λmax’s of M/LWS pigments measured using the in vitro assay\n",
      "  footer_text: The λmax’s that are underlined were measured by subtracting a spectrum measured after photobleaching from a spectrum measured before light exposure (difference spectra), and the λmax’s that are not underlined were measured before light exposure (dark spectra). The “Ancestor d” pigments in group 1 have identical amino acid sequences, but their λmax’s, evaluated by different investigators, differed by 2 nm, while those in the other groups have different amino acids at 180, 197, 277, 285, and 308 (see column 1). Sq. monkey, squirrel monkey. a Yokoyama and Radlwimmer (2001). b Matsumoto et al. (2006). c Yokoyama et al. (2005). d Sun et al. (1997). e Asenjo et al. (1994). f Merbs and Nathans (1993). g Merbs and Nathans (1992). h Hiramatsu et al. (2005). i Davies et al. (2007). j Hiramatsu et al. (2004, references therein).\n",
      "    unique: PMC2516078/PMC2516078.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: The effects of amino acid replacements on the λmax shift obtained by Yokoyama and Radlwimmer (2001) and by our mutagenesis method\n",
      "  footer_text: Some θ’s in Yokoyama and Radlwimmer (2001) were close to zero (—) or could not be determined (?).\n",
      "    unique: PMC2516078/PMC2516078.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2516917.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Candidate prostate cancer genes (decreased in prostate vs. seminal vesicle)\n",
      "  footer_text: * Genes upregulated in cultured prostate cancer cells following 5-aza-dC treatment. # Cytobands coinciding with prostate cancer susceptibility loci. ≠ Average ratio of expression in seminal vesicle compared to normal prostate\n",
      "    unique: PMC2516917/PMC2516917.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516917.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516917.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516963.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516963.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516963.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Candidate prostate cancer genes (increased in prostate vs. seminal vesicle)\n",
      "  footer_text: ≠ Average ratio of expression in normal prostate compared to seminal vesicle.\n",
      "    unique: PMC2516917/PMC2516917.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2516963.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Patient Characteristics in the Two Cohorts*\n",
      "  footer_text: * Missing values were present for some variables as follows. In the development cohort, race data (missing n=178), marital status (n=11); in the validation cohort, race data (n=151), marital status (n=9). † No statistically significant differences in baseline characteristics between the two cohorts.\n",
      "    unique: PMC2516963/PMC2516963.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Variables Considered as Risk Factors for Hospitalization in the Development Cohort*\n",
      "  footer_text: * Prev=prevalence; RR=relative risk; CI=confidence interval. Missing values were present for some variables as follows: race data (missing n=178), marital status (n=11). † Abnormal laboratory results included hematocrit < 30 ml/dL, serum albumin < 3.0 mg/dL, or serum creatinine > 1.5 mg/dL in previous year. The most abnormal result during the previous year was used to score this variable.\n",
      "    unique: PMC2516963/PMC2516963.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Independent Risk Factors for Hospitalization*\n",
      "  footer_text: * RR=relative risk; CI=confidence interval. n=number of patients with the risk factor present. † Adjusted relative risk derived from multivariable models in PROC GENMOD. ‡ _Includes imputed values for 11 missing subjects.\n",
      "    unique: PMC2516963/PMC2516963.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Performance of the Predictive Model in the Two Cohorts*\n",
      "  footer_text: * RR=relative risk; CI=confidence interval. Includes imputed values for marital status for 11 subjects in development cohort and 9 subjects in validation cohort. † χ2trend = 115.6, P<0.0001 ‡ χ2trend = 1151.3, P<0.0001\n",
      "    unique: PMC2516963/PMC2516963.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2516963.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517055.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517055.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517055.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2517055.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Individual and Mean (±SEM) ED50 (mg/kg, i.m.) Values for Zolpidem Generalization From Ethanol\n",
      "  footer_text: Values from sessions with vehicle (i.m.) pretreatment are shown in parentheses but not included in calculation of the mean. NC, not calculable because maximum ethanol-appropriate responding <50%. a Maximum ethanol-appropriate responding ≤79% (i.e., partial generalization).\n",
      "    unique: PMC2517055/PMC2517055.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Individual and Mean Apparent pKB (95% confidence interval) for Antagonism of Zolpidem Generalization From 1.0 and 2.0 g/kg Ethanol by Ro15-4513 in Cynomolgus Monkeys\n",
      "  footer_text: \n",
      "    unique: PMC2517055/PMC2517055.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Test Drugs Generalizing From Ethanol in Cynomolgus Monkeys From Published Reports and the Current Data Set (Zolpidem)\n",
      "  footer_text: ■, complete generalization (≥80% ethanol-appropriate responding for the entire session); partial generalization (≥20 and ≤79% ethanol-appropriate responding for the entire session); □, no generalization (<20% ethanol-appropriate responding for the entire session); NC, data not collected. a Grant et al., 2000 b Vivian et al., 2002.\n",
      "    unique: PMC2517055/PMC2517055.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2517140.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2517158.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of Study Subjects assessed in 1998/99 as a Function of their Diagnostic Classification in 1998/99 and 2002/2003\n",
      "  footer_text: 1 Number (percent) with at least on E*4 allele 2 Number (Percent) of participants in the group meeting CHS criteria for the condition 3 Number (percent) scoring 8 or higher on the Center for Epidemiological Studies Depression Scale (28) 4 Neuropsychological Composite Variables expressed as T-scores using age-and education-adjusted norms. 5 p < .001\n",
      "    unique: PMC2517158/PMC2517158.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Coordinates (in MNI Space) of Peak Voxels in Regions Showing Significant Differences from Normal Control Subjects\n",
      "  footer_text: \n",
      "    unique: PMC2517158/PMC2517158.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Volume of Region of Interest Extracted using SPM2 as a Function of Diagnostic Group by Year of Study\n",
      "  footer_text: Note: The volume unit is the mean volume (mL) per voxel within the region of interest, adjusted for the total brain volume, and the Gaussian smoothing filter.\n",
      "    unique: PMC2517158/PMC2517158.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517158.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517158.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517158.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2517215.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2517226.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Cluster of C. elegans CED-1 regulated genes\n",
      "  footer_text: Shown are the mean ± error of expression levels in wild-type (n = 3) and ced-1(e1735) (n = 2) animals. Eggs were placed in S basal to hatch overnight, causing growth arrest in L1. Synchronized L1 wild-type and ced-1(e1735) animals were grown on E. coli at 25°C for 26 hours and RNA was isolated. Expression data were normalized and cluster of CED-1 regulated genes was performed as described in Experimental Procedures.\n",
      "    unique: PMC2517226/PMC2517226.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517226.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517235.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517235.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517235.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2517235.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Percent correct classification of within-form and cross-form discriminant analyses based on spectral parameters alone (F1/F2 in Bark values) shown in the left column, and spectral parameters plus vocalic duration shown in the right column.\n",
      "  footer_text: \n",
      "    unique: PMC2517235/PMC2517235.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE II\n",
      "  caption_text: Acoustic similarity (F1/F2 in Bark values) of Japanese and American English (AE) vowels: hVb(a) syllables produced in citation.\n",
      "  footer_text: \n",
      "    unique: PMC2517235/PMC2517235.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE III\n",
      "  caption_text: Acoustic similarity (F1/F2 Bark values) of Japanese and American English (AE) vowels: hVb(a) syllables produced in sentence.\n",
      "  footer_text: \n",
      "    unique: PMC2517235/PMC2517235.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE IV\n",
      "  caption_text: Perceptual Assimilation Patterns. Categorization responses, expressed as percentages of total responses summed over speakers and listeners for citation-form (A) and sentence-form (B) materials. Bold = modal perceptual classification; boxed = modal acoustic classification.\n",
      "  footer_text: \n",
      "    unique: PMC2517235/PMC2517235.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: APPENDIX\n",
      "  caption_text: AVERAGE FORMANT FREQUENCIES AND DURATIONS OF JAPANESE (J) AND AMERICAN ENGLISH (AE) VOWELS.\n",
      "  footer_text: \n",
      "    unique: PMC2517235/PMC2517235.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517235.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517268.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517268.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2517268.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Questionnaire Items Used as Independent Variables and Covariates with Respective Response Options\n",
      "  footer_text: \n",
      "    unique: PMC2517268/PMC2517268.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Characteristics of Respondents and Health Centers with Univariate Regression Coefficients Predicting Change in Morale and Burnout among HDC Team Members (n = 622)\n",
      "  footer_text: † Change in response on 1-5 scale. + p ≤ .10. * p ≤ .05. ** p ≤ .01. ‡ “During calendar year 2003, to what degree did the Collaborative worsen or improve staff morale at your center?”: 1 = “greatly worsened”; 2 = “somewhat worsened”; 3 = “no change”; 4 = “somewhat improved”; 5 = “greatly improved.” § “Team members became burned out because of their Collaborative duties.” 1 = “strongly disagree”; 2 = “disagree”; 3 = “neither agree nor disagree”; 4 = “agree”; 5 = “strongly agree.” ¶ Range = 1-3: Low to high index of HDC outcomes. ∥ Range = 1-5: Low to high scoring of HDC performance. †† 1 = “none”; 2 = “a little bit”; 3 = “a moderate amount”; 4 = “quite a bit”; 5 = “a great deal.” ‡‡ Data from the Uniform Data System: aggregate summary for each health center, averaged over health centers. §§ Range = 0-4: “no decrease in provider participation” to “decrease in participation” across four provider types. ¶¶ 1 = “strongly disagree”; 2 = “disagree”; 3 = “neither agree nor disagree”; 4 = “agree”; 5 = “strongly agree.” ∥∥ Range = 0-8: Support in up to eight areas including training, information systems, funding for quality improvement was reported as “very useful” or “critically useful.” HDC, Health Disparities Collaboratives; HC, health center; FTE, full-time equivalent; CEO, Chief Executive Officer; QI, quality improvement.\n",
      "    unique: PMC2517268/PMC2517268.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Predictors of Staff Morale and Burnout Associated with the Health Disparities Collaboratives*\n",
      "  footer_text: * Mixed linear regression estimates † “During calendar year 2003, to what degree did the Collaborative worsen or improve staff morale at your center?” 1 = “greatly worsened”; 2 = “somewhat worsened”; 3 = “no change”; 4 = “somewhat improved”; 5 = “greatly improved.” ‡ Final models identical regardless of inclusion of HDC outcome variables. § “Team members became burned out because of their Collaborative duties.” 1 = “strongly disagree”; 2 = “disagree”; 3 = “neither agree nor disagree”; 4 = “agree”; 5 = “strongly agree.” HDC, Health Disparities Collaboratives; HC, health center.\n",
      "    unique: PMC2517268/PMC2517268.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517268.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517272.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517272.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517272.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517272.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2517272.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of the Patient Sample, N= 25,957\n",
      "  footer_text: Note. In subsequent analyses, patients without admission source and admission severity score were excluded, with a resulting analytic sample of 24,618.\n",
      "    unique: PMC2517272/PMC2517272.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Nursing Organization Characteristics, Hospital Level (n = 164)\n",
      "  footer_text: PES-NWI, Practice Environment Scale of the Nursing Work Index. PES-NWI scores are scored from 1 = strongly disagree; 2 = disagree; 3 = agree, and; 4 = strongly agree that the characteristic is present in their current job.\n",
      "    unique: PMC2517272/PMC2517272.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Unadjusted Outcome Rates\n",
      "  footer_text: \n",
      "    unique: PMC2517272/PMC2517272.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Unadjusted Outcome Rates (%) by Organizational Characteristics, (n= 24,618)\n",
      "  footer_text: * Differences across groups significant at p< .05. ** Differences across groups significant at p<.01. NCI = National Cancer Institute\n",
      "    unique: PMC2517272/PMC2517272.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Predictors of Patient Outcomes, Controlling for Hospital and Patient Characteristics\n",
      "  footer_text: Note: D, 30-day mortality; C, Complications; FTR, Failure to Rescue; NCI, National Cancer Institute; CI, Confidence interval. All models include 25 patient characteristics, bed size, teaching status, and performance of advanced procedures (output suppressed). Marginal analyses consider these patient and hospital characteristics, plus each one of the predictor variables. Partial analyses consider all patient and hospital characteristics, as well as all predictor variables simultaneously. Details on variables and measurement are available in Appendix S1.\n",
      "    unique: PMC2517272/PMC2517272.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2517273.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Descriptive statistics and comparison of survey responders and non-responders. (Standard deviations for continuous variables in parentheses)\n",
      "  footer_text: a *** p < 0.001 ** 0.001 ≤ p < 0.01 * 0.01 ≤ p < 0.05 b Sample size for each QM was different because not all facilities had a score reported.\n",
      "    unique: PMC2517273/PMC2517273.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Impact of publication of report card on the QMs: A pre/post analyses\n",
      "  footer_text: Based on linear combinations of coefficients from estimated model: QMi,t=α+βt+γPi+δPit+∑iφiDi where P =1 post publication and D is a vector of fixed facility effects. Regression coefficients are shown in online appendix table 1. *** p≤0.001 ** 0.001<p≤0.01 * 0.01<p<0.05\n",
      "    unique: PMC2517273/PMC2517273.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517272.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517273.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517273.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517273.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Analysis of the impact of number of actions (N) taken\n",
      "  footer_text: Based on linear combinations of coefficients from estimated model: QMit=α+βt+γPi+δ1PiNi+δ2PiNi2+∑iφiDi where P = 1 post publication, N = number of actions taken in response to publication, and D is a vector of fixed facility effects. Regression coefficients are shown in online appendix table 2. *** p≤0.001 ** 0.001<p≤0.01 * 0.01<p<0.05\n",
      "    unique: PMC2517273/PMC2517273.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Analysis of the impact of specific actions\n",
      "  footer_text: Based on linear combinations of coefficients from estimated model: QMit=α+βt+γPi+∑k=120δkPiAik+∑iφiDi where P = 1 post publication, Ak = 1 if action k was taken, and D is a vector of fixed facility effects. Regression coefficients are shown in online appendix table 3. *** p≤0.001 ** 0.001<p≤0.01 * 0.01<p<0.05\n",
      "    unique: PMC2517273/PMC2517273.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2517420.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Demographic and clinical characteristics of patients with preterm labor and intact membranes (PTL) and patients with preterm prelabor rupture of membranes (PROM)\n",
      "  footer_text: IAI: intra-amniotic infection; Values are expressed as median (range) or numbers (percent); * p<0.05; a p comparison between PTL who delivered preterm without IAI and PTL with IAI; b p comparison between PTL who delivered at term without IAI and PTL with IAI; c p comparison between preterm PROM with and without IAI\n",
      "    unique: PMC2517420/PMC2517420.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517273.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517420.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517420.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Demographic and clinical characteristics of women at midtrimester, women at term not in labor and women at term in labor\n",
      "  footer_text: Values are expressed as median (range) or numbers (percent)\n",
      "    unique: PMC2517420/PMC2517420.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2517611.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2517849.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Percent of R01 clinical applications\n",
      "  footer_text: Abbreviations: PI, principals investigator; A0, initial application, A1, first revision; A2, second revision of application; T1, new application; T2, competing renewal application\n",
      "    unique: PMC2517849/PMC2517849.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Rate of submission of initial and revised applications.\n",
      "  footer_text: Abbreviations: PI, principals investigator; A0, initial application, A1, first revision; A2, second revision of application; T1, new application; T2, competing renewal application\n",
      "    unique: PMC2517849/PMC2517849.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518063.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Demographic and Metabolic Characteristics*\n",
      "  footer_text: * Means ± standard deviations presented; for those without normal distributions, geometric means and standard deviations are presented. Exercise training adherence = Completed/prescribed miles per week*100 † Differs from other groups with Tukey’s post-hoc testing p<0.05 ** Differs from men with Tukey’s post-hoc testing p<0.05\n",
      "    unique: PMC2518063/PMC2518063.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517849.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2517849.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518063.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518063.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlation Coefficients for Change in Insulin Sensitivity (SI [mU/L/min])\n",
      "  footer_text: Data are presented as correlation coefficients (r) for the univariate relationship between the indicated variable and insulin sensitivity change. * P<0.05 † P<0.005\n",
      "    unique: PMC2518063/PMC2518063.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518098.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2518124.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Association of SNPs for Mayo Clinic breast cancer cases and controls, stratified on menopausal status. The three SNPs that were genotyped in the two replication studies are “boxed”.\n",
      "  footer_text: (1) = Adjusted for matching variables of age and area. (2) = Adjusted for age, area, parity/age at first child birth, and physical activity.\n",
      "    unique: PMC2518124/PMC2518124.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association of SNPs in the two replication studies, stratified on menopausal status.\n",
      "  footer_text: (1) = Adjusted for HRT use and physical activity. (2) = Adjusted for physical activity and parity/age at first child birth.\n",
      "    unique: PMC2518124/PMC2518124.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518165.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518124.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518165.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518205.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Esophageal cancer incidence among whites, nine Surveillance, Epidemiology, and End Results registries, 1975−79 to 2000−04\n",
      "  footer_text: * Rates are per 100,000 person-years, age-adjusted to the 2000 US Standard Population. CI = Confidence Interval (Tiwari mod) are 95% for rates; NOS = not otherwise specified; — = Statistic could not be calculated because the rate was based on fewer than 10 cases.\n",
      "    unique: PMC2518165/PMC2518165.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518205.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Features of Alzheimer’s Disease explained by the Metal Theory\n",
      "  footer_text: \n",
      "    unique: PMC2518205/PMC2518205.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518316.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2518320.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Phenotypic Data for Members of a Family with Autosomal Dominant Atrial Fibrillation.*\n",
      "  footer_text: * AF denotes atrial fibrillation, JR junctional escape rhythm, LAE left atrial enlargement, LVDD left ventricular diastolic dimension, LVSD left ventricular systolic dimension, LVEF left ventricular ejection fraction, LVH left ventricular hypertrophy, NA not available, NSR normal sinus rhythm, PAF paroxysmal atrial fibrillation, SB sinus bradycardia, and ULN upper limit of the normal range for that patient in millimeters. Dashes denote not applicable. † Echocardiographic measurements in affected subjects were performed during effective ventricular rate control when atrial fibrillation was present. ‡ The normal value is 50% or more. § Echocardiography was performed before the onset of atrial fibrillation. ¶ Normal sinus rhythm was not documented but was reported in the subject’s medical history. || Subject III-8 had a ventricular pacemaker.\n",
      "    unique: PMC2518320/PMC2518320.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518323.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Lipid peroxidation and protein oxidation in sciatic nerves from control and galactose-fed rats.\n",
      "  footer_text: Data are mean ± SEM (N = 7–11 per group) and were analyzed with unpaired, two-tailed t-tests. Differences are not significant.\n",
      "    unique: PMC2518323/PMC2518323.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Body weight, blood glucose and nerve sugar levels in control rats, diabetic rats and diabetic rats treated with the aldose reductase inhibitor, ICI 222155, or insulin for the last 4 weeks of an 8-week period of diabetes.\n",
      "  footer_text: Data are presented as mean ± SEM (N = 8–16 per group) and were analyzed by one-way ANOVA, after which multiple comparisions were made with Dunnett’s post-hoc test. * p<0.05 vs untreated diabetic group.\n",
      "    unique: PMC2518323/PMC2518323.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2518404.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518320.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518323.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518323.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of individuals in the Polyp Prevention Trial of Aspirin and Folate who completed first year follow-up\n",
      "  footer_text: ¥ 81 and 325 mg/day aspirin treatment groups combined * Total folate and total B6 = dietary + supplements † Total B2 from dietary sources only ‡ Total B12 from supplements only § using standard definitions by Polyp Prevention Study Group (28, 29) ** chi-square test # two sample t-test\n",
      "    unique: PMC2518404/PMC2518404.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association of vitamin B2, B6 and B12 baseline plasma levels and risk of colorectal adenomas\n",
      "  footer_text: * Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, and multivitamin use ¥ Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, multivitamin use, and plasma B2, B6, B12 and folate as appropriate ¶ p-value for trend.\n",
      "    unique: PMC2518404/PMC2518404.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Association of baseline intakes of vitamins B2, and B6 with risk of colorectal adenomas\n",
      "  footer_text: * Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, and multivitamin use ¥ Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, multivitamin use, and intake of B2, B6 and folate as appropriate ¶ p-value for trend ** Ranges are based on 2000 calories/day\n",
      "    unique: PMC2518404/PMC2518404.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518404.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518404.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518404.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Association of vitamin B2, B6 (PLP) and B12 baseline plasma levels and the risk of colorectal adenomas by folate treatment group, multivitamin and alcohol use at baseline\n",
      "  footer_text: * Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, multivitamin use and plasma B2, B6, B12 and folate as appropriate † Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, and plasma B2, B6, B12 and folate as appropriate ** Quartiles for vitamin B2 (riboflavin): Q1: 2.1-8.9 nmol/L; Q2: 9.0-15.0 nmol/L; Q3: 15.1-27.5 nmol/L; Q4: 27.6-479.0 nmol/L *** Quartiles for vitamin B6 (PLP): Q1: 4.7-35.6 nmol/L; Q2: 35.7-53.3 nmol/L; Q3: 53.4-84.3 nmol/L; Q4: 84.7-857.0 nmol/L **** Quartiles for vitamin B12: Q1: 21.4-235.9 pmol/L; Q2: 236.4-300.3 pmol/L; Q3: 300.5-388.8 pmol/L; Q4: 389.2-2479 pmol/L ¥ p-value for interaction ¶ p-value for trend\n",
      "    unique: PMC2518404/PMC2518404.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Selected gene-vitamin B interactions and risk of colorectal adenomas\n",
      "  footer_text: * Adjusted for age, sex, center, duration of follow-up, aspirin treatment group, multivitamin use and plasma B2, B6, B12 and folate as appropriate ¥ p-value for interaction\n",
      "    unique: PMC2518404/PMC2518404.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518404.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518404.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2518644.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Descriptive statistics of HHFS, morbidity, and feeding practices of infants at different ages\n",
      "  footer_text: 1 Only infant feeding scales were standardized and expressed as a percentage of the value of the upper range of theoretical scales.\n",
      "    unique: PMC2518644/PMC2518644.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Lagged model effects of HHFS and other infant, maternal, and household factors on the changes in IFP from 3–6, 6–9, and 9–12 mo of age\n",
      "  footer_text: 1 Lagged morbidity, lagged feeding, birth weight, and sex of infants. 2 Lagged morbidity, lagged IFP, birth weight, and sex of infants, and maternal age, weight, education, and wealth index.\n",
      "    unique: PMC2518644/PMC2518644.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Summary of lagged, dynamic, and difference model effects of HHFS and other infant and maternal factors on the changes in IFP from 3–6, 6–9, and 9–12 mo of age\n",
      "  footer_text: 1 –, Variables were not included in the models.\n",
      "    unique: PMC2518644/PMC2518644.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518644.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518644.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518644.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518983.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2518746.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2518964.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2518983.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Simulated surface tensions of aqueous solutions of NaCl and methanol.\n",
      "  footer_text: All simulations were performed at 300 K for a total simulation time of Tsim. Symbols are: nio, the total number of solute and water molecules used in the simulations; Cso (x2o), the initial bulk solute concentrations; cs (x2), the final bulk solute concentrations. Typical estimated errors for the surface tension values are ±0.3 mN/m.\n",
      "    unique: PMC2518983/PMC2518983.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Simulated and experimental surface adsorption data for aqueous solutions of NaCl and methanol.\n",
      "  footer_text: All data correspond to 300 K. See text for symbol definitions. In the case of ∂γ/∂y, y = cs for NaCl and y = x2 for methanol. The values of the relative surface adsorptions correspond to those obtained from the fitted surface tension changes and bulk solution properties (a22 and η). Units are cs (M), η (M), ∂γ/∂y (mN/m/M for NaCl and mN/m for methanol), and Γ2,1 in ions or molecules/nm2.\n",
      "    unique: PMC2518983/PMC2518983.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2519027.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2519048.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2519054.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2519120.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2519143.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2519172.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2518983.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519612.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519612.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2519612.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Chemopreventive effects of methylselenocysteine on NMU-induced mammary carcinogenesis in female F344 rats.\n",
      "  footer_text: † Tumor incidence was defined by number of tumor-bearing rats divided by total number of rats. †† Both tumor incidence and total number of tumors presented in this table are histologically confirmed adenocarcinomas. ‡ Multiplicity was defined by the total number of tumors divided by total number of tumor-bearing rats. § Tumor inhibition was calculated based on total number of tumors. * P<0.05\n",
      "    unique: PMC2519612/PMC2519612.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Results of GoMiner Gene enrichment analysis of differentially expressed genes by selenium.\n",
      "  footer_text: \n",
      "    unique: PMC2519612/PMC2519612.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Pair-wised comparison of Per2 and DBP mRNA levels between mammary carcinomas and their adjacent normal tissues.\n",
      "  footer_text: \n",
      "    unique: PMC2519612/PMC2519612.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2519872.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Group Characteristics\n",
      "  footer_text: * mean ± standard error\n",
      "    unique: PMC2519872/PMC2519872.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of Statistical Significances Using Limited and Complete Models\n",
      "  footer_text: Hypothesis testing: Intent to treat responses were used. Both models were repeated measures analysis of variance. Effects in the limited model were diet and visit (Month 1, 2, 3). Effects in the complete model were visit (Month 1, 2, 3), diet, gender, age, baseline measure of index being evaluated. Interactions of Visit × Diet and Diet × Gender. # Month 2 was different (Tukey’s test, P < 0.05) from month 1 and 3 * Linear effect from month 1 to month 3 was stronger (p=.0001) than using discrete time points (visits) (p=.0007)\n",
      "    unique: PMC2519872/PMC2519872.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Clinical measures over time\n",
      "  footer_text: \n",
      "    unique: PMC2519872/PMC2519872.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519612.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Clinical measures over time by gender\n",
      "  footer_text: \n",
      "    unique: PMC2519872/PMC2519872.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Distribution of lost ligatures\n",
      "  footer_text: \n",
      "    unique: PMC2519872/PMC2519872.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Clinical measures under the assumption of intent to treat (Control n=32, experimental n=23)\n",
      "  footer_text: \n",
      "    unique: PMC2519872/PMC2519872.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2519956.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of EXAFS Fitting Results for Copper RBP (Best Fit in Bold)\n",
      "  footer_text: a Independent metal–ligand scattering environment. b Scattering atoms: O (Oxygen), N (Nitrogen), C (Carbon). c Metal–ligand bond length in Å. d Metal–ligand coordination number. e Debye–Waller factor in Å2 × 103. f Number of degrees of freedom weighted mean square deviation between data and fit. g Fit using only single scattering Feff 7 theoretical models. h Fit using both single scattering Feff 7 model with an additional multiple scattering Fe–N(Imidazole) model, generated based on crystallographic coordinates listed in ref 17,17 and labeled NIm in table atom designation. Details of fitting are described in the Experimental Section.\n",
      "    unique: PMC2519956/PMC2519956.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519872.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2519956.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2522268.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2522309.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2522314.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2522366.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample of Questions Related to Diabetes and Diabetic Eye Disease\n",
      "  footer_text: \n",
      "    unique: PMC2522366/PMC2522366.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Demographic Characteristics by Diabetes Status\n",
      "  footer_text: a Differences significant at P<.05. b Question was not asked of the diabetic cohort.\n",
      "    unique: PMC2522366/PMC2522366.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Diabetes Duration and Type of Treatment in Participants With Diabetes\n",
      "  footer_text: a Refers to primary mode of treatment. b P<.05.\n",
      "    unique: PMC2522366/PMC2522366.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Diabetes Knowledge by Level of Experience With Diabetesa\n",
      "  footer_text: a Proportion of correct answers to the question, “What can a person with diabetes do to treat and control diabetes?”\n",
      "    unique: PMC2522366/PMC2522366.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Predictors of Knowledge That Uncontrolled Diabetes Could Cause Eye Disease\n",
      "  footer_text: Abbreviations: CI, confidence interval; OR, odds ratio. a P<.05.\n",
      "    unique: PMC2522366/PMC2522366.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Predictors of Having a Dilated Eye Examination in the Past 2 Years in Persons With Diabetes\n",
      "  footer_text: Abbreviations: CI, confidence interval; OR, odds ratio. a P<.05.\n",
      "    unique: PMC2522366/PMC2522366.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2522368.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: KD of the talin RM or smaller fragments binding to talin-PTB and talin-PTB mutants\n",
      "  footer_text: a All binding affinities were calculated by two independent measurements using the BIAcore 3000 evaluation software (Biacore AB, Uppsala, Sweden). χ2 <10 were obtained for all the data, indicating good fit.\n",
      "    unique: PMC2522368/PMC2522368.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2522372.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522366.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2522368.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525452.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2525452.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Bias and Variance of Liver R1, R2 Calibration Curves\n",
      "  footer_text: Bold type indicates statistically significant difference\n",
      "    unique: PMC2525452/PMC2525452.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Bias and Variance of Heart R1, R2 Calibration Curves\n",
      "  footer_text: Bold type indicates statistically significant difference\n",
      "    unique: PMC2525452/PMC2525452.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Variability of Tissue Iron and Tissue NMR measurements\n",
      "  footer_text: \n",
      "    unique: PMC2525452/PMC2525452.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2525512.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2525738.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525452.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525452.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T1-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2525795.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Inclusion and exclusion criteria for MOCOCS study\n",
      "  footer_text: \n",
      "    unique: PMC2525795/PMC2525795.xml::T1-0\n",
      "    unique: PMC2525795/PMC2525795.xml::T1-1\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Sequence and description of color measurements of subjects’ craniofacial structures\n",
      "  footer_text: \n",
      "    unique: PMC2525795/PMC2525795.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Summaries of color parameters of central incisors of 120 teeth (CIELAB units)\n",
      "  footer_text: \n",
      "    unique: PMC2525795/PMC2525795.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table IV\n",
      "  caption_text: ANOVA summary for multiple linear regression of L*a*b* for central incisor\n",
      "  footer_text: \n",
      "    unique: PMC2525795/PMC2525795.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table V\n",
      "  caption_text: Associated parameter estimates for central incisor of L*a*b*\n",
      "  footer_text: ** Parameter estimates were developed using only statistically significant factors found in full model, as noted in text. *** For null hypothesis, population value = 0.\n",
      "    unique: PMC2525795/PMC2525795.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2525799.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2526058.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2525795.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2526058.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2526059.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Histologic Grading\n",
      "  footer_text: Histologic evaluation of each study group. Note the histologic score was performed using standard Hematoxylin and Eosin staining. The Fibrosis score was a ranking of the density of fibrosis after Mason’s trichrome staining based on an arbitrary range of 0 to 5 for each of 4 quadrants of a cross-section of the distal colonic wall. The collagen density values represent the digitized areas of collagen staining as described in the Methods section. Note that ACE-I treatment resulted in a significant decrease in the amount of fibrosis. Abbreviations: DSS, dextran sulfate sodium. ACE-I, angiotensin converting enzyme inhibitor. * P<.05 (ANOVA), Control (Naïve mice, no DSS treatment) versus DSS+Placebo group # P<.05 (ANOVA), DSS+Placebo versus DSS+ACE-I treated group. Data are mean ± SD.\n",
      "    unique: PMC2526058/PMC2526058.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2526059.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Hepatic Gene Expression Analysis of Wild-type Compared to Fah−/− or Fah−/−, p21−/−Mice off NTBC\n",
      "  footer_text: Gene expression levels on a logarithmic scale: 10E(number of ●, with ◖= ~0.5). Genes differentially expressed between Fah−/− and Fah−/−, p21−/− mice such as the mitosis regulators Cyclin A2 and B1 are highlighted in bold. The effect of NTBC on hepatic gene expression is restricted to Fah deficiency (data not shown). Hence, gene expression levels in wild-type mice on water represent findings in wild-type, Fah−/− and Fah−/−, p21−/− mice on NTBC. Raw microarray data is provided in Supplemental Figure 1A.\n",
      "    unique: PMC2526059/PMC2526059.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2526181.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sociodemographic and Health Related Profile and Cancer Screening Rates for Study Sample, Pooled 2003 and 2005 CHIS data, Unweighted, women aged 40-75, men aged 50-75\n",
      "  footer_text: * Self-reported race/ethnic identity using UCLA CHPR categories; white, non-Hispanic excluded. † Respondents answering “yes” to question: “Was there ever a time when you would have gotten better medical care if you had belonged to a different race or ethnic group?” ‡ Endoscopy: sigmoidoscopy or colonoscopy.\n",
      "    unique: PMC2526181/PMC2526181.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2526181.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2526181.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Weighted Colorectal and Breast Cancer Screening Rates by Ethnicity and Medical Discrimination*\n",
      "  footer_text: * Unless otherwise indicated, all the comparisons between no reported vs. reported discrimination are statistically significant (p < 0.001). † Not statistically significant ‡ Predictor variables included in the sociodemographics model were age, education, household income, ethnicity and survey year. § Predictor variables included in the sociodemographics and health factors model were age, education, household income, ethnicity, survey year, health insurance, health status, having a usual source of care, and having a previous diagnosis of cancer.\n",
      "    unique: PMC2526181/PMC2526181.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527026.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample characteristics of participants\n",
      "  footer_text: + For HRV measures, geometric means are presented with SD of the logged values\n",
      "    unique: PMC2527026/PMC2527026.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cross sectional associations between HRV (1 SD decrease) and cognitive function (odds of lowest quintile).\n",
      "  footer_text: + Odds ratios are adjusted for age, sex and highest educational attainment\n",
      "    unique: PMC2527026/PMC2527026.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Prospective association between HRV at phase 5 (1 SD decrease) and cognitive function at phase 7 (odds of lowest quintile).\n",
      "  footer_text: + Odds ratios are adjusted for age, sex and highest educational attainment\n",
      "    unique: PMC2527026/PMC2527026.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527026.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Prospective association between HRV at phase 5 (1 SD decrease) and decline in cognitive function between Phase 5 and Phase 7 (odds of lowest quintile of change).\n",
      "  footer_text: + Odds ratios are adjusted for age, sex and highest educational attainment and time interval between Phase 5 and Phase 7\n",
      "    unique: PMC2527026/PMC2527026.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Cross sectional associations between HRV (1 SD decrease) and cognitive function (odds of lowest quintile) among participants with coronary heart disease.\n",
      "  footer_text: + Odds ratios are adjusted for age, sex and highest educational attainment\n",
      "    unique: PMC2527026/PMC2527026.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Prospective association between HRV at phase 5 (1 SD decrease) and cognitive function at phase 7 (odds of lowest quintile) among participants with coronary heart disease.\n",
      "  footer_text: + Odds ratios are adjusted for age, sex and highest educational attainment and time interval between Phase 5 and Phase 7\n",
      "    unique: PMC2527026/PMC2527026.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527051.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: General characteristics of a sample of normal-weighta seventh-grade students participating in the Teens Eating for Energy and Nutrition at Schools (TEENS) study, by sex\n",
      "  footer_text: a Defined as body mass index between 5th and <85th percentile. b Defined as any past 30 days use of diet pills, vomiting, laxatives, or diuretics.\n",
      "    unique: PMC2527051/PMC2527051.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527051.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527051.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527051.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527051.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527055.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Unadjusted associations between seventh grade predictorsa and overweight status during eighth grade, for participants in the Teens Eating for Energy and Nutrition at Schools (TEENS) study, by sex (n = 1,728)\n",
      "  footer_text: a Based on data collected from classroom surveys. Detailed description of scales and items used in the TEENS study are provided elsewhere (24). b NA = not applicable. The sample size was too small for analysis. * P<0.05.\n",
      "    unique: PMC2527051/PMC2527051.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Predictors of becoming overweight during eighth gradea, based on data collected as part of the Teens Eating for Energy and Nutrition at Schools (TEENS) study (n = 1,728)\n",
      "  footer_text: a Adjusted for socioeconomic status variables, race, and intervention condition. Detailed description of scales and items used in the TEENS study are provided elsewhere (24). * P<0.05.\n",
      "    unique: PMC2527051/PMC2527051.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Predictors of becoming overweight during eighth gradea among boys and girls in the fifth to 75th percentiles for body mass index (n=1,456) in the Teens Eating for Energy and Nutrition at Schools (TEENS) study (n=1,456)\n",
      "  footer_text: a Adjusted for socioeconomic status variables, race, and intervention condition. Detailed description of scales and items used as predictors in the TEENS study are provided elsewhere (24). * P<0.05.\n",
      "    unique: PMC2527051/PMC2527051.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527055.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Radiation Chemical Yields in nmol/J for Free Base Release in pUC18 Plasmid DNA Films and Calculation of the Multiplicity of Single-Strand Breaks\n",
      "  footer_text: a G(sc-loss) values were reported under G total(ssb) in Table I of ref. (13). b Free base release multiplicity = m(fbr) = G(fbr)/G(sc-loss).\n",
      "    unique: PMC2527055/PMC2527055.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Comparative Radiation Chemical Yields in nmol/J for Trapped Deoxyribose Radicals, Free Base Release and Direct SSBs\n",
      "  footer_text: a A yield of 70 nmol/J (5) was applied to the water target mass, which consists of the water lying beyond the hole transfer layer: Γ-10 = target mass = 12.5 waters/nucleotide for Γ = 10. b Gdir(fbr) ≥ G(fbr) − G(●OH) and ≤G(fbr). c Values from ref. (12). d Gdir(ssb) = G(fbr) − 2 × G(dsb), assuming each SSB of the DSB gives a free base release. e SSB multiplicity = m(ssb) = Gdir(ssb)/G(sc-loss). f GdRib(fr) values were reported in Table I of ref. (13). g Shortfall = G(diff) = Gdir(ssb) − GdRib(fr).\n",
      "    unique: PMC2527055/PMC2527055.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527207.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2527209.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Quantitative mass spectrometry of purified ribosome-channel complexes\n",
      "  footer_text: The samples were divided into 6 aliquots which were analyzed and averaged. The appropriate tryptic peptide fragment for proteins marked with an asterisk (*) could only be detected reliably in 5 of the runs.The range of values for each component in the experiments is shown on the right in parenthesis.\n",
      "    unique: PMC2527209/PMC2527209.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527442.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Crystal Data and Summary of Data Collection and Refinement for Clusters 3, 4, and 5a\n",
      "  footer_text: a Collected using Mo Kα radiation (λ = 0.71073 Å) at 193 K. b R1 = Σ [(Fo − Fc)]/Σ (Fo). c wR2 = {Σ [w(Fo2 − Fc2)2/Σ[w(Fo2)2]}1/2. d I >2σ(I). e All data.\n",
      "    unique: PMC2527442/PMC2527442.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527055.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527209.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527442.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Selected Interatomic Distances (Å)a and Angles (deg) of [Fe7S6(PEt3)5Cl2]\n",
      "  footer_text: a Averaged under idealized Cs symmetry. b Bond distance notation: Fe(m)–S(x,y) is the mean of Fe(m)–S(x) and Fe(m)–S(y); Fe(m,n)–S(x,y) is the mean of Fe(m)–S(x) and Fe(n)–S(y). c Displacement (Å) of Fe atom from S3 plane.\n",
      "    unique: PMC2527442/PMC2527442.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Selected Interatomic Distances (Å) and Angles (deg) of [Fe8S8(Pri2NHCMe2)6]\n",
      "  footer_text: a Fe4S4, range of 12. b S(2,3,4) or Fe(2,3,4), range of 9.\n",
      "    unique: PMC2527442/PMC2527442.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: 57Fe Mössbauer Parameters for [Fe4S4]0 and [Fe8S8]0 Clusters\n",
      "  footer_text: a Referenced to Fe metal at room temperature; ±0.02 mm/s. b ±0.03 mm/s. c Reference 23. d Reference 22. e References 6, 9. f Spectrum analyzed as four equally intense doublets with the same isomer shift.\n",
      "    unique: PMC2527442/PMC2527442.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Selected Interatomic Distances (Å) and Angles (deg) of [Fe4S4(Pri2NHCMe2)4]\n",
      "  footer_text: \n",
      "    unique: PMC2527442/PMC2527442.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Comparison of Structural Parameters of Cubane-Type [Fe4S4]0 Clusters\n",
      "  footer_text: a Reference 23. b Reference 10. c Reference 8.\n",
      "    unique: PMC2527442/PMC2527442.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527485.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527442.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527442.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527485.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527485.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Primers used in this study\n",
      "  footer_text: * Genomic DNA was used as the template † Plasmid containing Rv1086 was used as the template\n",
      "    unique: PMC2527485/PMC2527485.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: X-ray data\n",
      "  footer_text: * These data were twinned, for the APO the twin fraction was refined for CITPP the data were detwinned. The twined data have been deposited as well as the de-twinned. † The completeness was limited by radiation damage. Thus the resolution limits for these data sets are nominal.\n",
      "    unique: PMC2527485/PMC2527485.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of protein activity\n",
      "  footer_text: \n",
      "    unique: PMC2527485/PMC2527485.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527511.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Affect of RNAi Depletion of Different Factors on the Chromatin Architecture at Hsp70\n",
      "  footer_text: 1 Knockdown confirmed by Western to be 5–20% of LacZ RNAi levels. 2 Nucleosomes on the body of the gene are better positioned. 3 DNA at the 1st and 2nd nucleosomes after the TSS is less protected. 4 Resembles a NHS profile. 5 Resembles a 1’ HS profile. 6 DNA at the 1st nucleosome after the TSS is more protected.\n",
      "    unique: PMC2527511/PMC2527511.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527598.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527485.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527511.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected statistical data from the four main groups of mice\n",
      "  footer_text: \n",
      "    unique: PMC2527598/PMC2527598.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527650.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Prevalence of Selected Axis I Lifetime Diagnoses Among G1 and G2 Participants\n",
      "  footer_text: Note: G1 = generation 1; G2 = generation 2; MDD = major depressive disorder; ANX = any anxiety disorder; SUD = any substance use disorder. a G1 n = 260; G2 n = 84. b G1 n = 267; G2 n = 183. c G1 N = 527; G2 N = 267. * χ2 significant gender difference at p < .001.\n",
      "    unique: PMC2527650/PMC2527650.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Univariate and Multivariate Associations Between G2 MDD and G3 CBCL Scales\n",
      "  footer_text: Note: N = 267. All scores are t scores. G2 = generation 2; MDD = major depressive disorder; G3 = generation 3; CBCL = Child Behavior Checklist; sr = semipartial correlation. a Adjusted for G2 CBCL rater status and G2 and G3 age and sex (n = 267, df = 261). b Adjusted for G2 Marital Status, Education, and Income (n = 254, df = 245). c Adjusted for G2 Dyadic Adjustment, O'Leary Porter, Major Life Events and Daily Hassles (n = 206, df = 193). † p = .06. * p < .05. ** p < .01.\n",
      "    unique: PMC2527650/PMC2527650.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527731.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2527781.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Odds of reporting arthritis, diabetes, or hypertension in older Mexican women compared to men. (n = 7,792)\n",
      "  footer_text: † ≤.10 * ≤ .05 ** ≤ .01 *** ≤ .001\n",
      "    unique: PMC2527781/PMC2527781.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527598.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527650.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527650.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527781.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527817.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527817.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2527817.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Prevalence of malnutrition at the national level in children with cystic fibrosis1\n",
      "  footer_text: 1 HTp, height percentile; BMIp, BMI percentile; %IBW, percentage of ideal body weight; CFF, Cystic Fibrosis Foundation. 2 Weight-for-height percentile was used instead of BMIp for children <2 y old. 3 Corrected classification refers to the elimination of %IBW as an indicator of underweight.\n",
      "    unique: PMC2527817/PMC2527817.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Nutritional failure rates in children with cystic fibrosis (CF) among 113 CF centers1\n",
      "  footer_text: 1 CFF, Cystic Fibrosis Foundation; HTp, height percentile; BMIp, BMI percentile; %IBW, percentage of ideal body weight. 2 x̄ ± SD (all such values). 3 Criterion: HTp < 5th, BMIp < 10th, or %IBW < 90. Weight-for-height percentile was used instead of BMIp for children <2 y old. 4 Correction: %IBW < 90 was eliminated as an indicator of underweight. 5 Criterion: HTp < 5th or BMIp < 10th. Weight-for-height percentile was used instead of BMIp for children <2 y old.\n",
      "    unique: PMC2527817/PMC2527817.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527838.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Frequency and proportion of cases of breast cancer recurrence and matched controls within group strata (I) expressing the estrogen receptor and receiving at least one year of tamoxifen therapy (ERP+/TAM+), or (II) not expressing the estrogen receptor, never receiving tamoxifen therapy, and surviving at least one year after diagnosis (ERP−/TAM−)\n",
      "  footer_text: † Variable included in risk set sampling to match controls to cases\n",
      "    unique: PMC2527838/PMC2527838.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527838.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527838.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2527838.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number of cases and controls receiving any prescription for each SSRI, and total number of prescriptions for each SSRI within group strata (I) expressing the estrogen receptor and receiving at least one year of tamoxifen therapy (ERP+/TAM+), or (II) not expressing the estrogen receptor, never receiving tamoxifen therapy, and surviving at least one year after diagnosis (ERP−/TAM−)\n",
      "  footer_text: † In the analysis, we defined citalopram exposure as any prescription for Citalopram (N06AB04) or its s-stereoisomer Escitalopram (N06AB10).\n",
      "    unique: PMC2527838/PMC2527838.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Association between SSRI prescription and breast cancer recurrence within strata of (a) Group I, women with tumors that expressed the estrogen receptor and who received at least one year of tamoxifen therapy (ERP+/TAM+), or (b) Group II, women with tumors that did not express the estrogen receptor, and who never received tamoxifen therapy, and who survived at least one year after diagnosis (ERP−/TAM−).\n",
      "  footer_text: † Adjusted for age category and other CYP2D6 inhibiting medications (see the Supplementary On-line Material for a complete list of these medications and the frequency of their use in the study population).\n",
      "    unique: PMC2527838/PMC2527838.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2527972.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528022.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528059.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528060.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528248.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528831.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528843.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2528953.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Blood Ethanol Elimination Rates\n",
      "  footer_text: \n",
      "    unique: PMC2528953/PMC2528953.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2528955.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Subject Characteristics\n",
      "  footer_text: Data are mean ± S.D. BMI is body mass index. FM is fat mass. FFM is fat-free mass. WC is waist circumference. *** represents P < 0.001.\n",
      "    unique: PMC2528955/PMC2528955.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Metabolic data\n",
      "  footer_text: Data are mean ± S.D. Younger, n = 10; Older, n = 10. RQ is respiratory quotient. REE is resting energy expenditure. Cox is basal carbohydrate oxidation. Fox is basal fat oxidation. Pox is basal protein oxidation. VO2max is maximal oxygen uptake during exhaustive exercise. HRmax is maximum heart rate recorded during exhaustive exercise. * represents P < 0.05 *** represents P < 0.001.\n",
      "    unique: PMC2528955/PMC2528955.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Blood chemistry\n",
      "  footer_text: Data are mean ± S.D. FPG is fasting plasma glucose; divide mg/dl by 18 to convert to mmol/l. FSI is fasting serum insulin. HOMA-IR is insulin resistance according to Matthews et al. (1985). TG is triglycerides. T-Chol is total cholesterol.\n",
      "    unique: PMC2528955/PMC2528955.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Bivariate correlations between age and other variables\n",
      "  footer_text: Abbreviations and units of measurement follow the same legend as for previous tables. ** represents P < 0.01 *** represents P < 0.001.\n",
      "    unique: PMC2528955/PMC2528955.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Bivariate correlations between fat oxidation and other variables\n",
      "  footer_text: Abbreviations and units of measurement follow the same legend as for previous tables. * represents P < 0.05 ** represents P < 0.01.\n",
      "    unique: PMC2528955/PMC2528955.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528953.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528955.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528955.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528955.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528955.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2528955.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2529146.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2529228.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Plasma Levels of TSP1 and Cytokines in Patients with RA\n",
      "  footer_text: Italicized value represents statistical trends (when p value is between 0.1 and 0.05)\n",
      "    unique: PMC2529228/PMC2529228.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2529257.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics of hemodialysis patients by serum phosphate quartile at baseline (n=707)\n",
      "  footer_text: * P Value is from χ2 tests for each categorical variable and serum phosphate quartiles.\n",
      "    unique: PMC2529257/PMC2529257.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529228.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529257.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529257.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cross-sectional association of baseline serum phosphate (per 1 mg/dl) with baseline systolic, diastolic and pulse blood pressure in all patients and patient subgroups, adjusted for patient characteristics in the subpopulation of patients with compete data on all variables (n= 669)\n",
      "  footer_text: * β is mmHg higher of blood pressure per 1 mg/dl higher serum phosphate at baseline adjusted for all characteristics in table as well as serum albumin, log(C-reactive protein), dialysis non-adherence, and any β-blocker, Ca channel blocker or ACE inhibitor use. # Adjusted for the above patient characteristics except for the indicated variable.\n",
      "    unique: PMC2529257/PMC2529257.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Association of 1 mg/dl higher in serum phosphate at baseline with blood pressure at different follow-up intervals, adjusted for patient characteristics.\n",
      "  footer_text: * β is mmHg higher of blood pressure per 1 mg/dl higher serum phosphate at baseline adjusted for age, sex, race, baseline blood pressure, ICED score, albumin, log(C-reactive protein), DM, baseline CVD, dialysis non-adherence, and any β-blocker, Ca channel blocker or ACE inhibitor use.\n",
      "    unique: PMC2529257/PMC2529257.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Association of 1 mg/dl higher average serum phosphate measurements up to 3 months with blood pressure at different follow-up intervals, adjusted for patient characteristics.\n",
      "  footer_text: * β is mmHg increase of blood pressure per 1 mg/dl higher serum phosphate at baseline adjusted for age, sex, race, baseline blood pressure, ICED score, albumin, log(C-reactive protein), DM, baseline CVD, dialysis non-adherence, and any β-blocker, Ca channel blocker or ACE inhibitor use.\n",
      "    unique: PMC2529257/PMC2529257.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Association of different percentage of serum phosphate measurements greater than 5.5 mg/dl from 0 to 3 months with blood pressure at different follow-up intervals using 0% of serum phosphate >5.5mg/dl as reference, adjusted for patient characteristics.\n",
      "  footer_text: * β is mmHg increase of blood pressure comparing to the category that no phosphate measurement is above 5.5 mg/dl at 0–3 months adjusted for age, sex, race, baseline blood pressure, ICED score, albumin, log(C-reactive protein), DM, baseline CVD, dialysis non-adherence, and any β-blocker, Ca channel blocker or ACE inhibitor use. + none = no serum phosphate measurements >5.5 mg/dl; some = >0%,<100% of serum phosphate measurements >5.5 mg/dl; all =100% of serum phosphate measurements >5.5 mg/dl within the first 3 months of enrollment.\n",
      "    unique: PMC2529257/PMC2529257.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529257.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529257.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2529257.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2531245.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2531069.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2531214.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2531245.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of the angles and axial spacings/separations between the three levels in fish skeletal and the rabbit heart myosin filaments\n",
      "  footer_text: The rabbit values in the current work are the averages of the values plotted in Fig. 4, whereas those for fish (AL-Khayat et al., 2006) are based on single projections.\n",
      "    unique: PMC2531245/PMC2531245.xml::T1-0\n",
      "    unique: PMC2531245/PMC2531245.xml::T1-1\n",
      "\n",
      "PMC0025XXXXX/PMC2531249.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2531266.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of crystallographic data for anthrax-P4H\n",
      "  footer_text: R sym = [ ( ∑ h ∑ i ∣ I h i − 〈 I h 〉 ∣ ) ∕ ∑ h ∑ i 〈 I h 〉 ] × 100 % * The numbers in parentheses represent highest resolution shell data.\n",
      "    unique: PMC2531266/PMC2531266.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2531280.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: X-ray data collection statistics for HADH. Values in parentheses are for the last resolution shell\n",
      "  footer_text: \n",
      "    unique: PMC2531280/PMC2531280.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2532495.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2532513.xml"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2531245.xml::T1-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2531266.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "PMC0025XXXXX/PMC2532521.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2532600.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2532757.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Characteristics of Study Participants at Randomizationa\n",
      "  footer_text: Abbreviations: BMI, body mass index, calculated as weight in kilograms divided by height in meters squared; NRTI, nucleoside reverse transcriptase inhibitor; NNRTI, nonnucleoside reverse transcriptase inhibitor. a Data are reported as mean (95% confidence interval) unless otherwise indicated. P > .05 for all baseline comparisons by t test for continuous variables and χ2 test for categorical variables. b For growth hormone group, n = 26. c Patients were self-identified as current smokers but did not provide information on packs per day.\n",
      "    unique: PMC2532757/PMC2532757.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532757.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532757.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532757.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effects of Growth Hormone vs Placebo on Body Composition, Lipid Profiles, Biochemical Parameters, and Cardiovascular and Immune Functiona\n",
      "  footer_text: Abbreviations: BMI, body mass index, calculated as weight in kilograms divided by height in meters squared; BP, blood pressure; CI, confidence interval; HDL, high-density lipoprotein; HIV, human immunodeficiency virus; IGF-1, insulinlike growth factor 1; IMT, intima-media thickness; MOS-HIV, Medical Outcomes Study HIV Health Survey; NA, not applicable; SAT, subcutaneous adipose tissue; VAT, visceral adipose tissue. SI conversions: To convert triglycerides to mmol/L, multiply by 0.0113; to convert total and HDL cholesterol to mmol/L, multiply by 0.0259; to convert IGF-1 to nmol/L, multiply by 0.131; to convert glucose to mmol/L, multiply by 0.0555; to convert insulin to pmol/L, multiply by 6.945. a Baseline results are reported for all participants for whom at least 1 follow-up result was obtained. Treatment effects are reported as net difference between the change in growth hormone and placebo. For triglycerides, results are reported as median (interquartile range) using the Wilcoxon rank sum test. The 95% CI could not be calculated for triglycerides given the use of a nonparametric analysis. All baseline comparisons are P > .05 by t test (analysis of variance), except for diastolic BP. b Aggregate change over 18 months for growth hormone vs placebo. See the “Statistical Analysis” section for description of statistical methods used. c Number of patients analyzed was 1 less than the total number (ie, n = 20 for VAT with growth hormone at month 18). d n = 23. e Scale of 0 to 100, with higher number indicating better quality of life. For growth hormone group, n = 23 at baseline and month 6; n = 19 at month 12; n = 17 at month 18; and for placebo group, n = 25 at baseline and month 6; and n = 24 at months 12 and 18.\n",
      "    unique: PMC2532757/PMC2532757.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of Adverse Events\n",
      "  footer_text: Abbreviation: CI, confidence interval. a By χ2 test.\n",
      "    unique: PMC2532757/PMC2532757.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2532844.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2532854.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant Characteristics and Correlations With Family History Density (n = 72)\n",
      "  footer_text: * p < 0.05.\n",
      "    unique: PMC2532854/PMC2532854.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Neurocognitive Scores and Correlations With Family History Density (n = 72)\n",
      "  footer_text: \n",
      "    unique: PMC2532854/PMC2532854.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Regions Activated to SWM Relative to Vigilance Conditions (Clusters > 943 μl, p < 0.025, n = 72)\n",
      "  footer_text: + , clusters activated preferentially to the SWM condition; − , clusters activated preferentially to the vigilance condition. a Divided from the largest cluster.\n",
      "    unique: PMC2532854/PMC2532854.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2532855.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532854.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532854.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532854.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532855.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Participant characteristics at baseline and year 31\n",
      "  footer_text: 1 ucOC, percentage of undercarboxylated osteocalcin; CRP, C-reactive protein; OPG, osteoprotegerin. 2 x̄ ± SD; range in parentheses (all such values). 3 Significantly different between groups at baseline, P < 0.05 (independent samples t test). 4 Significant treatment × time effect, P < 0.001 (repeated-measures ANOVA).\n",
      "    unique: PMC2532855/PMC2532855.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Cross-sectional associations at baseline between measures of vitamin K status and circulating cytokines in men and women aged 60-80 y (n = 379, 59% F)1\n",
      "  footer_text: 1 ucOC, percentage of undercarboxylated osteocalcin. 2 Also adjusted for triacylglycerols. 3 Adjusted for sex and age. 4 Adjusted for sex, age, statin use, and antiinflammatory use. 5 Adjusted for sex, age, statin use, antiinflammatory use, BMI (in kg/m2), physical activity, and smoking.\n",
      "    unique: PMC2532855/PMC2532855.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Correlations between change in bone mineral density (BMD) and change in circulating cytokine concentration in men and women aged 60-80 y (n = 379) after 3 y of follow-up1\n",
      "  footer_text: 1 Partial correlation coefficients were adjusted for age, sex, treatment, baseline cytokine concentration, baseline BMD, statin medication use, antiinflammatory medication use, BMI (in kg/m2), smoking, and physical activity. IL-6, interleukin-6; CRP, C-reactive protein; OPG, osteoprotegerin.\n",
      "    unique: PMC2532855/PMC2532855.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2533126.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: 28 SNPs from IL1A and IL1B were genotyped in 1314 NARAC RA cases and 1103 controls. The allele that is more frequent in cases is shown in bold capital letters and the allele frequency and odds ratio are given for this allele. Genotype counts: (homozygotes of the allele more frequent in cases, heterozygotes, homozygotes of the alternate allele). P-values are from chi-squared analysis performed on the allele counts for single markers in RA patients versus controls. Allelic Polymorphism in IL1 in Cases vs. Controls: NARAC\n",
      "  footer_text: \n",
      "    unique: PMC2533126/PMC2533126.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532855.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2532855.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533126.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533126.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: RA3 and RA4 were genotyped in two independent replication case-control sample sets. First, they were genotyped in 948 CCP-positive cases from the WRDDB/NICRAP/SONORA cohort and 1455 controls from the NYCP. Next, they were genotyped in 525 RA cases and 504 controls from Northern Spain. Frequencies and odds ratios are shown for the allele that is capitalized and in bold. Genotype counts: (homozygotes of the allele capitalized and in bold, heterozygotes, homozygotes of the alternate allele). Chi-squared analysis of allele counts was performed for cases versus controls. Allelic Polymorphism in RA3 and RA4 in Cases vs. Controls: Replication Sets\n",
      "  footer_text: \n",
      "    unique: PMC2533126/PMC2533126.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: 24 SNPs from IL1A and IL1B were genotyped in 712 RA patients. A chi-squared analysis was performed comparing IL1 allele counts in the 403 of these patients that had erosions on plain radiographs of the hands versus the 309 that did not. Frequencies and odds ratios are shown for the allele that is capitalized and in bold. Genotype counts: (homozygotes of the allele capitalized and in bold, heterozygotes, homozygotes of the alternate allele). Chi-squared analysis of allele counts was performed for cases versus controls. Allelic Polymorphism in IL1 in Patients With vs. Without erosions: BRASS Cohort\n",
      "  footer_text: \n",
      "    unique: PMC2533126/PMC2533126.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: All of the SNPs within the 20kb region of IL1A were used to construct haplotypes for the NARAC cases and controls. Haplotypes of a frequency > 0.005 are shown. SNPs are ordered as in Table 1. OR = odds ratio and 95% CI = 95% confidence interval of the OR. IL1A Haplotype Association with NARAC Cases vs. Controls\n",
      "  footer_text: \n",
      "    unique: PMC2533126/PMC2533126.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: All of the SNPs genotyped from IL1A and IL1B were used to construct haplotypes for the NARAC cases and controls. Haplotypes with a frequency of > 0.01 are shown. SNPs are ordered as in Table 1. OR = odds ratio and 95% CI = 95% confidence interval of the OR. Extended IL1A-IL1B Haplotype Association with NARAC Cases vs. Controls\n",
      "  footer_text: \n",
      "    unique: PMC2533126/PMC2533126.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2533156.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533126.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533126.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary Statistics.\n",
      "  footer_text: Note: Percentages may not sum to 100% due to rounding error. N = 2,034 couples\n",
      "    unique: PMC2533156/PMC2533156.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Beliefs, Attitudes, and Practices Regarding Relationship Behaviors, by Religious Participation.\n",
      "  footer_text: Note: Means are significantly (p < .05) different from those in columns denoted by subscripted letters. N = 2,034 couples\n",
      "    unique: PMC2533156/PMC2533156.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Seemingly Unrelated Regressions of Union Satisfaction on Religious Attendance.\n",
      "  footer_text: Note: Model controls for age, education, race, and family structure at age 15. N = 2,034 couples * p < .05 ** p < .01 *** p < .001 (2 tail tests)\n",
      "    unique: PMC2533156/PMC2533156.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Seemingly Unrelated Regressions of Union Satisfaction on Religious Attendance and Relationship Attitudes.\n",
      "  footer_text: Note: Model controls for age, education, race, and family structure at age 15. N = 2,034 couples * p < .05 ** p < .01 *** p < .001 (2 tail tests)\n",
      "    unique: PMC2533156/PMC2533156.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Seemingly Unrelated Regressions of Union Satisfaction on Religious Attendance and Adherence to Values.\n",
      "  footer_text: Note: Model controls for age, education, race, and family structure at age 15. N = 2,034 couples * p < .05 ** p < .01 *** p < .001 (2 tail tests)\n",
      "    unique: PMC2533156/PMC2533156.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Seemingly Unrelated Regressions of Union Satisfaction on Religious Attendance and Behaviors Conducive to Relationships.\n",
      "  footer_text: Note: Model controls for age, education, race, and family structure at age 15. N = 2,034 couples * p < .05 ; ** p < .01 ; *** p < .001 (2 tail tests)\n",
      "    unique: PMC2533156/PMC2533156.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Seemingly Unrelated Regressions of Union Satisfaction on Religious Attendance, Attitudes about Relationships, Adherence to Values, and Behaviors Conducive to Relationships.\n",
      "  footer_text: Note: Model controls for age, education, race, and family structure at age 15. N = 2,034 couples * p < .05 ** p < .01 *** p < .001 (2 tail tests)\n",
      "    unique: PMC2533156/PMC2533156.xml::T7-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533156.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533269.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533269.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533269.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2533269.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean OPTION Scores by Study Variable Groups\n",
      "  footer_text: \n",
      "    unique: PMC2533269/PMC2533269.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Regression Model for Physician, Practice Setting, and Visit Characteristics Predicting Shared Decision Making Behavior (OPTION Score)\n",
      "  footer_text: Note: Group practice is the reference group for practice setting. Independent variables include the following: physician age, physician ethnicity, physician gender, practice setting (HMO, solo or private practice, academic, group), medical specialty (Internal Medicine, Family Practice), number of patients seen during a typical half day, and the duration of the office visit. The dependent variable is shared decision making behavior.\n",
      "    unique: PMC2533269/PMC2533269.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Regression Model for Experimental Variables Predicting Shared Decision Making Behavior (OPTION Score) Controlling for Physician, Practice Setting, and Visit Characteristics\n",
      "  footer_text: Notes: Variables included in the model include request type (brand-specific, general, no request), medical condition (major depression, adjustment disorder), and interaction terms (generated by crossing request type and medical condition); adjusted for physician, practice, and visit characteristics. The dependent variable is shared decision making behavior. Reference groups are no request (Request Type) and adjustment disorder with depressed mood (Medical Condition). Information about the physician, practice setting, and visit characteristics are not included in this Table.\n",
      "    unique: PMC2533269/PMC2533269.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2533376.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics* of US adults with and without diabetes by time period - NHIS 1997–2005 (N=255,966).\n",
      "  footer_text: * Values are weighted percentages (Standard Error) † Based on family income and family size using the US Census Bureau’s poverty thresholds for the previous calendar year. “Poor” persons are defined as below the poverty threshold. “Near poor” persons have incomes of 100% to less than 200% of the poverty threshold. “Not poor” persons have incomes that are 200% of the poverty threshold or greater.\n",
      "    unique: PMC2533376/PMC2533376.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Educational disparities in the prevalence of self-reported CVD over time among US adults with and without diabetes. NHIS 1997–2005.\n",
      "  footer_text: RII: Relative Index of Inequality, PRR: Prevalence Rate Ratio, 95% CI: 95% confidence interval * Adjusted for age, sex, race/ethnicity, poverty status, comorbidity and survey year ¶ Interaction between education and diabetes status in the regression model: p<0.05\n",
      "    unique: PMC2533376/PMC2533376.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533376.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533376.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2533517.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2533528.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2533638.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2533688.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic, CMR, and inter-rater variability data\n",
      "  footer_text: Abbreviations: COV = coefficient of variability; EDV = end-diastolic volume; ESV = end-systolic volume; LV = left ventricle; RV = right ventricle; SD = standard deviation; SV = stroke volume\n",
      "    unique: PMC2533688/PMC2533688.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison between intra-class correlation coefficients of right and left ventricular measurements\n",
      "  footer_text: Abbreviations: ICC = intra-class correlation coefficient; LV = left ventricle; RV = right ventricle; SD = standard deviation\n",
      "    unique: PMC2533688/PMC2533688.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533688.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533688.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533688.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of intra-class correlations between patients with normal right ventricle and those with atrial septal defect or repaired tetralogy of Fallot\n",
      "  footer_text: Abbreviations: ASD = atrial septal defect; ICC = intra-class correlation coefficient; TOF = tetralogy of Fallot\n",
      "    unique: PMC2533688/PMC2533688.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Comparison between inter-observer coefficient of variations in this study and in selected published reports\n",
      "  footer_text: Abbreviations: ASD = atrial septal defect; CHF = congestive heart failure; EDV = end-diastolic volume; EF = ejection fraction; ESV = end-systolic volume; FLASH = fast low-angle shot; LVH = left ventricular hypertrophy; SSFP= steady-state free precession; TOF = tetralogy of Fallot\n",
      "    unique: PMC2533688/PMC2533688.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2533707.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2533688.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535806.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535844.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2533764.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535758.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535806.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: RPE cell culture characteristics\n",
      "  footer_text: \n",
      "    unique: PMC2535806/PMC2535806.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535844.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Radial formation in human cells depleted of HR and NHEJ proteins Average percentage of GM639 cells containing radials following depletion of listed proteins and MMC treatment. Average percentage represents three independent trials. “None” represents a transfection with scrambled siRNA.\n",
      "  footer_text: \n",
      "    unique: PMC2535844/PMC2535844.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of protein deficiencies leading to an increase in radial formation Loss of function or siRNA depletion of proteins listed here causes increased radial formation either spontaneously, following ICL-damage, or both. Five of these have been shown in the depletions in this manuscript. BLM is Bloom syndrome protein. DNA-PK is for the mouse Prkdc protein, the catalytic subunit of DNA-dependent protein kinase.\n",
      "  footer_text: 1 (Schroeder et al. 1964; reviewed in D’Andrea and Grompe 2003) 2 shown in this manuscript. 3 (Bruun et al. 2003). 4 (Hemphill et al. 2008). 5 (German 1997). 6 (Niedernhofer et al. 2004). 7 (Houghtaling et al. 2005). * spontaneous radials without clastogen exposure.\n",
      "    unique: PMC2535844/PMC2535844.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535846.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Characteristics for Participants and Non-Participants After 9 Years\n",
      "  footer_text: * p<0.05 for participants vs non-participants † p<0.05 for participants vs non-participants from logistic regression analyses after adjusting for age and sex ** Non-participants included those who died or did not participate for other reasons Hypertension: SBP≥140 or DBP≥90 or Hx of treatment Diabetes: GHb>10 or self-reported hx of diabetes\n",
      "    unique: PMC2535846/PMC2535846.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Age and Sex-Specific 9-Year Incidence of Hypertension and Diabetes\n",
      "  footer_text: * Hypertension: SBP≥140 or DBP≥90 or Hx of treatment Diabetes: GHb>10 or self-reported hx of diabetes\n",
      "    unique: PMC2535846/PMC2535846.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Relative Risks* for the 9-year Incidence of Hypertension\n",
      "  footer_text: * Adjusted relative risk based on logistic regression model, including variables presented in this table. † p = 0.06 BMI = Body mass index; WHR = Waist-hip ratio; BP = Blood pressure BMI quartiles (< 23.3, ≥ 23.3 and < 26.3, ≥ 26.3 and < 29.8, ≥ 29.8) WHR quartiles (<0.87, ≥ 0.87 and < 0.92, ≥ 0.92 and < 0.97, ≥ 0.97)\n",
      "    unique: PMC2535846/PMC2535846.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Relative Risks* for the 9-year Incidence of Diabetes\n",
      "  footer_text: * Adjusted relative risk based on logistic regression model, including variables presented in this table. BMI = Body mass index; WHR = Waist-hip ratio BMI quartiles (< 23.3, ≥ 23.3 and < 26.3, ≥ 26.3 and < 29.8, ≥ 29.8) WHR quartiles (<0.87, ≥ 0.87 and < 0.92, ≥ 0.92 and < 0.97, ≥ 0.97)\n",
      "    unique: PMC2535846/PMC2535846.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535850.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Descriptive Statistics\n",
      "  footer_text: \n",
      "    unique: PMC2535850/PMC2535850.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: General Linear Mixed Model Analysis of Estimated Marginal Means (EMM), Standard Errors (SE), and Simple Effects Analyses of the Effect of Time within Each Group\n",
      "  footer_text: Note: N's represent the number of independent units of observation in each scale analyzed; MDI = metered dose inhaler skill.\n",
      "    unique: PMC2535850/PMC2535850.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535846.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535846.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535846.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535846.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535850.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535850.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535850.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Results of Tests of Fixed Effects in General Linear Mixed Effect Model Analyses for Each Measure\n",
      "  footer_text: \n",
      "    unique: PMC2535850/PMC2535850.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535851.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of study participants according to quartiles of systolic blood pressure response to cold pressor test\n",
      "  footer_text: Mean ± standard deviation\n",
      "    unique: PMC2535851/PMC2535851.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Multivariate regression analyses of blood pressure responses to dietary interventions according to quartile of blood pressure responses to cold pressor test\n",
      "  footer_text: See Figure 2 legend for the cut-points of area-under-curve quartiles of blood pressure responses to the cold pressor test\n",
      "    unique: PMC2535851/PMC2535851.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multivariate-adjusted odds ratio and 95% confidence intervals of high salt-sensitivity and potassium-sensitivity according to quartiles of blood pressure responses to cold pressor test\n",
      "  footer_text: See Figure 2 legend for the cut-points of area-under-curve quartiles of blood pressure responses to the cold pressor test\n",
      "    unique: PMC2535851/PMC2535851.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535852.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535853.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Household, women and infant characteristics by maternal age categorya\n",
      "  footer_text: a Estimates represent mean (SD) for continuous variables and % for categorical variables. b Chi squared tests were used for comparison of categorical variables by age group and analysis of variance for continuous variables. c A total of 9 were missing latrine data d A total of 229 were missing electricity data e A total of 236 were missing cattle ownership data f A total of 229 were missing ethnicity data g A total of 220 were missing caste data h A total of 4 were missing literacy data i A total of 9 were missing occupation data j A total of 598 were missing night blindness data k A total of 1668 were missing birth weight data l A total of 80 were missing gestational age data m A total of 1716 were missing small for gestational age data\n",
      "    unique: PMC2535853/PMC2535853.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535851.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535851.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535851.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535853.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535853.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535853.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535853.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Neonatal Mortality Rate (per 1000 live births) by maternal age and parity\n",
      "  footer_text: \n",
      "    unique: PMC2535853/PMC2535853.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Neonatal Mortality Rates (NMR), crude and adjusted Odds Ratios (OR) and their 95% Confidence Intervals, for neonatal mortality by maternal age category and other maternal characteristics\n",
      "  footer_text: i A total of 598 were missing night blindness data a Adjusted for treatment received, maternal literacy, ethnic group, caste, latrine ownership, maternal occupation, electricity in the home, cattle ownership, parity and maternal nightblindness during pregnancy b A total of 229 were missing ethnicity data c A total of 220 were missing caste data d A total of 4 were missing literacy data e A total of 9 were missing latrine data f A total of 9 were missing occupation data g A total of 229 were missing electricity data h A total of 236 were missing cattle ownership data j A total of 1668 were missing birth weight data k A total of 80 were missing gestational age data l A total of 1716 were missing small for gestational age data\n",
      "    unique: PMC2535853/PMC2535853.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Crude and adjusted Odds ratios (OR) and 95% Confidence Intervals for adverse birth outcomes by maternal age category\n",
      "  footer_text: a Adjusted for treatment received, maternal literacy, ethnic group, caste, latrine ownership, maternal occupation, electricity in the home, cattle ownership, parity and maternal nightblindness during pregnancy b Same as a plus adjustment for low birth weight c Same as a plus adjustment for preterm birth d Same as a plus adjustment for small for gestational age e Same as a plus adjustment for preterm and small for gestational age f Same as a plus adjustment for 4 preterm birth and small for gestational age categories (term & not SGA, term & SGA, preterm & not SGA, both preterm & SGA)\n",
      "    unique: PMC2535853/PMC2535853.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535854.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Strengthening conclusions to support estimates of treatment or intervention effects when assuming all “important” confounders are measured. Confounders are variables related to the probability of treatment assignment and to the outcomes of interest.\n",
      "  footer_text: ¶ Assuming full compliance and no missing data.\n",
      "    unique: PMC2535854/PMC2535854.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Assumptions needed when conducting an instrumental variables analysis. Applied to Ryan et al study Reference 14). Confounders are variables related to the probability of treatment assignment and to the outcomes of interest.\n",
      "  footer_text: \n",
      "    unique: PMC2535854/PMC2535854.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mechanisms and assumptions for missing data. Confounders are variables related to the probability of missing data and to the outcomes of interest.\n",
      "  footer_text: \n",
      "    unique: PMC2535854/PMC2535854.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535855.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535900.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Electrode placement in EIM.\n",
      "  footer_text: * Measurements were obtained on patients lying supine, with the arms lying passively at the sides of the body and a small bolster or pillow under the knees.\n",
      "    unique: PMC2535900/PMC2535900.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Reference values for phase measured at 50 kHz\n",
      "  footer_text: All measurements are in degrees (°)\n",
      "    unique: PMC2535900/PMC2535900.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535904.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535854.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535854.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535900.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535900.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2535913.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample Details Fifty-eight pre-treatment tumor samples from stage 4 without MYCN amplification were used. Of those, 29 survived without event for at least 3 years (denoted by a “+” in the outcome column) and 29 died of the disease (“−”). Patients were diagnosed between 1991 and 2002 (Diagnosis date), the age at date of diagnosis is given in days. Column “BP17” indicates presence (“yes”) or absence (“no”) of a breakpoint leading to a gain on chromosome 17; column “11q−BP” indicates a breakpoint leading to loss of genomic material on the q-arm of chromosome 11. BP, a chromosomal breakpoint. Columns “1” to “22” and “X”, “Y” indicate gain (“+”), loss (“−”) or no (“0“) whole chromosome change of the corresponding chromosome.\n",
      "  footer_text: \n",
      "    unique: PMC2535913/PMC2535913.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association Analysis Association of localized loss of genomic material on 11q, localized gain on 17q and whole chromosome gains/losses with patient survival for (A) all samples and (B) the subset of samples from patients older than 500 days at time of diagnosis. Each column shows the number of samples with the corresponding characteristic separated for deceased/survived patients. In each column a p-value for association of the observed distribution with the patient’s death pdeceased or survival psurvived is given. BP, a chromosomal breakpoint.\n",
      "  footer_text: \n",
      "    unique: PMC2535913/PMC2535913.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535917.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Evolution of JFH-1 in chimpanzees\n",
      "  footer_text: A Compared to the consensus JFH-1 sequences.\n",
      "    unique: PMC2535917/PMC2535917.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535913.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535913.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535917.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535917.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Specific infectivity and virus secretion in Huh7-25 cells\n",
      "  footer_text: The data are from day 3 after HCV RNA transfection of the Huh7-25 cells A P < 0.05 comparing JFH-1 WT and G838R\n",
      "    unique: PMC2535917/PMC2535917.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535918.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535920.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2535925.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2535925/PMC2535925.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2535930.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536515.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536528.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Summary of patient characteristics in each of the 3 groups1\n",
      "  footer_text: 1 TPN, total parenteral nutrition; PaCO2, arterial carbon dioxide tension; PRISM, Pediatric Risk Mortality. The chi-square test indicated no significant group differences with respect to sex. 2 ANOVA indicated no significant group differences according to the overall F test for age, weight, BMI, pH, or PRISM scores and no significant differences between any 2 groups on the basis of post hoc Tukey tests (P > 0.10 for all). 3 x̄ ± SD (all such values). 4 PaCO2 production was significantly different in the TPN (P < 0.01) and enteral (P = 0.04) groups than in the glucose-electrolytes group.\n",
      "    unique: PMC2536528/PMC2536528.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2535925.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536528.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536528.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536528.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Protein and energy intakes of study patients1\n",
      "  footer_text: 1 TPN, total parenteral nutrition. 2 ANOVA indicated significant group differences according to the overall F test between the TPN and enteral feeding groups compared with the glucose-electrolytes group, P<0.001 (post hoc Tukey test). No significant differences were found between the TPN and enteral groups for protein (P=0.40) or energy (P = 0.35) intake. 3 x̄ ± SD (all such values).\n",
      "    unique: PMC2536528/PMC2536528.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Rates of appearance of carbon dioxide (RaCO2), fractional recovery of 13CO2, and bicarbonate-predicted and measured rates of carbon dioxide production (V̇ CO2) in critically ill children receiving parenteral feedings, enteral feedings, or glucose-electrolytes1\n",
      "  footer_text: 1 TPN, total parenteral nutrition. 2 ANOVA indicated no significant group differences according to the overall F test and no significant differences between any 2 groups according to post hoc Tukey tests (P > 0.60 for all). 3 x̄ ± SD (all such values).\n",
      "    unique: PMC2536528/PMC2536528.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536564.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536567.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Parameter estimates for intervention effects on scale change scores—caregiver data\n",
      "  footer_text: \n",
      "    unique: PMC2536567/PMC2536567.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Parameter estimates for intervention effects on scale change scores—child data\n",
      "  footer_text: \n",
      "    unique: PMC2536567/PMC2536567.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536601.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic Data\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536567.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536567.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536601.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536601.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  footer_text: Mean (SD) unless stated otherwise. SCZ=schizophrenia; WAIS-FSIQ=Wechsler Adult Intelligence Scale-revised, full-scale intelligence quotient; GAF=Global Assessment of Functioning Scale; Prodromal Age=age of onset of prodromal symptoms of schizophrenia * p≤0.05.\n",
      "    unique: PMC2536601/PMC2536601.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Neuropsychological Assessment of Probands with and without Enuresis\n",
      "  footer_text: SCZ=schizophrenia; WAIS-FSIQ=Wechsler Adult Intelligence Scale, full scale IQ; WRAT=Wide Range Achievement Test; WCST=Wisconsin Card Sorting Test; %PE=percentage of perseverative errors; CVLT=California Verbal Learning Test * p≤0.05.\n",
      "    unique: PMC2536601/PMC2536601.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Voxel-wise Statistical Analysis of GM Maps of Decreased Gray Matter in Probands with (n=16) vs. without Enuresis (n=66), and in Healthy Controls with (n=11) and without Enuresis (n=91)\n",
      "  footer_text: Peak coordinates listed rostral to caudal. * p value is p<0.001. kE is the number of voxels in the significant cluster.\n",
      "    unique: PMC2536601/PMC2536601.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536612.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536601.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536612.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536612.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536612.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536613.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Number of rats used in the experimental paradigms\n",
      "  footer_text: \n",
      "    unique: PMC2536612/PMC2536612.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Complete information and immunogen of primary antibodies applied\n",
      "  footer_text: \n",
      "    unique: PMC2536612/PMC2536612.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Assessment of BDA volume, rostro-caudal diffusion and analyzed gray matter area of the spinal cord\n",
      "  footer_text: Data expressed as mean ± SD\n",
      "    unique: PMC2536612/PMC2536612.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536613.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Methods for assessment of diet/nutrition in epidemiologic studies\n",
      "  footer_text: \n",
      "    unique: PMC2536613/PMC2536613.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Selected epidemiologic studies relating dietary factors with lung function or airway hyperresponsiveness\n",
      "  footer_text: NHANES: National Health and Nutrition Examination Survey CARET: Carotene and Retinol Efficacy Trial MORGEN: Monitoring Project on Risk factors for Chronic Diseases EPIC: European Prospective Investigation into Cancer and Nutrition Study\n",
      "    unique: PMC2536613/PMC2536613.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536613.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536613.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536613.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536704.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536708.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Selected epidemiologic studies relating dietary factors with asthma, asthma-related symptoms, and atopy\n",
      "  footer_text: RHINE – Respiratory Health in Northern Europe ISAAC – International Study of Asthma and Allergies in Childhood\n",
      "    unique: PMC2536613/PMC2536613.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Studies of the association between maternal diet and nutrient status in pregnancy and immune markers, asthma, and asthma-related outcomes in childhood\n",
      "  footer_text: SEATON – Study of Eczema and Asthma To Observe the effects of Nutrition ALSPAC – Avon Longitudinal Study of Pregnancy and Childhood\n",
      "    unique: PMC2536613/PMC2536613.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536704.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characterization and proposed classification of BRCA2 SyVaL clones\n",
      "  footer_text: NOTE: Numbers in parentheses represent mean percentage of untreated/treated cells with more than five RAD51 foci ± SE. Fold Δ refers to the ratio of the IC50 of cells having the hemizygous wt allele to the IC50 of cells having the mutated allele. Abbreviations: Pos, positive; Neg, negative; —, not done; Δ, change. * The extreme rarity of knockout clones may indicate an especially severe deleterious effect (21).\n",
      "    unique: PMC2536704/PMC2536704.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536708.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Nucleotide sequences of the primers used for PCR\n",
      "  footer_text: \n",
      "    unique: PMC2536708/PMC2536708.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536719.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of E. coli inactivation of several compounds in aqueous solution\n",
      "  footer_text: a Obtained from Sigma-Aldrich Co. b Oxidized for 2 hr under air saturation c Unless otherwise specified: pH = 8.0 (carbonate buffer), T = 21±0.5°C, air saturation indicates open system to atmosphere d From the data of Figure 2 e Prepared by photoreduction of silver ion in aqueous solution containing poly(ethylene oxide)–poly(propylene oxide)–poly(ethylene oxide) block copolymers (Supporting Information, S3)\n",
      "    unique: PMC2536719/PMC2536719.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536720.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536721.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Locomotor activity rhythms of cry excision strains under LL conditions. Period and power were calculated for all rhythmic animals as described in Materials and Methods. Activity was calculated for all flies that survived to the end of the experiment, as described in Materials and Methods. Homozygous deletions are shown in bold. n.a.; not applicable.\n",
      "  footer_text: \n",
      "    unique: PMC2536721/PMC2536721.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Locomotor activity rhythms of cry excision strains under DD conditions. Period and power were calculated for all rhythmic animals as described in Materials and Methods. Activity was calculated for all flies that survived to the end of the experiment, as described in Materials and Methods. Homozygous deletion strains are shown in bold.\n",
      "  footer_text: \n",
      "    unique: PMC2536721/PMC2536721.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536746.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536719.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536721.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536721.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Characteristics1 of 698 prostate cancer cases and 698 matched controls nested in the health professionals follow-up study\n",
      "  footer_text: 1 Cases and controls were matched on age, time of day and year of blood draw, and PSA test prior to blood draw. Controls all had a PSA test after blood draw. Unless otherwise indicated, values are from the 1994 follow-up questionnaire. Red meat and fish intake are averages of reports in 1986, 1990, and 1994. Diabetes mellitus includes cases through 1994. 2 High grade was defined as Gleason sum ≥ 7. Low grade was defined as Gleason sum < 7. Gleason sum was not available for 65 cases. 3 For the hypothesis test of no difference in means (paired t-test) or proportions (McNemar's test) between total prostate cancer cases and controls. All tests are 2-sided. 4 For the hypothesis test of no difference in means (t-test) or proportions (Chi-square test) between high- and low-grade cases. All tests are 2-sided.\n",
      "    unique: PMC2536746/PMC2536746.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Mean plasma cholesterol concentration, dietary cholesterol intake, and history of elevated cholesterol in prostate cancer cases and matched controls nested in the health professionals follow-up study\n",
      "  footer_text: 1 High grade was defined as Gleason sum ≥ 7. Low grade was defined as Gleason sum < 7. Gleason sum was not available for 65 cases. 2 For the hypothesis test of no difference in means (paired t-test) or proportions (McNemar's test) between total prostate cancer cases and controls. All tests are 2-sided. 3 For the hypothesis test of no difference in means (t-test) or proportions (Chi-square test) between high- and low-grade cases. All tests are 2-sided. 4 Case-control pairs were run in 3 batches; the same assay method was used for the 1996 and 1998 batches and a different assay method was used for the 2000 batch. Means ± standard deviations in the cases and controls by batch were: 1996: case 218.5 ± 37.8, control 217.9 ± 39.8 mg/L; 1998: case 211.7 ± 38.0, control 213.9 ± 41.9 mg/L; and 2000 case 166.9 ± 32.1, control 166.1 ± 38.1 mg/L.\n",
      "    unique: PMC2536746/PMC2536746.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Odds ratio of prostate cancer by plasma cholesterol concentration, 698 matched pairs nested in the health professionals follow-up study\n",
      "  footer_text: 1 Estimated from a conditional logistic regression model. 2 Estimated from a conditional logistic regression model and adjusted for family history of prostate cancer, height, vigorous physical activity, diabetes mellitus, vasectomy, cigarette smoking in the past 10 years, intake of energy, red meat, fish, and alcohol, intake of energy-adjusted lycopene, calcium, fructose, and α-linolenic acid, and supplemental vitamin E and selenium. 3 Estimated from a logistic regression model after excluding men who ever used cholesterol-lowering drugs and adjusted for age, PSA test before blood draw, family history of prostate cancer, height, vigorous physical activity, diabetes mellitus, vasectomy, cigarette smoking in the past 10 years, intake of energy, red meat, fish, and alcohol, intake of energy-adjusted lycopene, calcium, fructose, and α-linolenic acid, and supplemental vitamin E and selenium. 4 Cutpoints differed by analytical batch (see Material and methods for details): 1996 192.8 mg/dL, 1998 185.7 mg/dL, and 2000 140.1 mg/dL).\n",
      "    unique: PMC2536746/PMC2536746.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536746.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536746.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536746.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2536749.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline clinical characteristics\n",
      "  footer_text: NA intestinal or colonic transit could not be measured because of technical limitations, N normal,↑ increased, ↓ decreased\n",
      "    unique: PMC2536749/PMC2536749.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effects of drugs on bowel symptoms assessed by weekly diaries\n",
      "  footer_text: a Worsening or improvement was defined as ≥1 point deterioration or improvement in symptom severity respectively on 5-point symptom severity scales for each symptom. Missing data for 1 subject who did not complete 8-week questionnaires\n",
      "    unique: PMC2536749/PMC2536749.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Colonic motility findings\n",
      "  footer_text: Prhalf pressure corresponding to half maximum volume during the pressure-volume relationship. Neostigmine increased the Prhalf, reflecting lower compliance, or a stiffer colon\n",
      "    unique: PMC2536749/PMC2536749.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536753.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: NDP-MSH-Stimulated Signaling and Ligand Binding of WT and Mutant hMC3Rs\n",
      "  footer_text: * ND, could not be detected. a significantly different from WT hMC3R, p<0.05 b significantly different from WT hMC3R, p<0.01 c significantly different from WT hMC3R, p<0.001 The data are expressed as the mean ± SEM of three or four independent experiments. The maximal response under NDP-MSH stimulation was 2308 ± 362 pmol cAMP/106 cells for WT hMC3R. Binding is shown as receptor occupancy (RO) that is a crude estimate of the relative maximal binding level.\n",
      "    unique: PMC2536753/PMC2536753.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536749.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536749.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536749.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536753.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536753.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Ligand Binding and D-Trp8-γ-Stimulated cAMP Production of WT and Mutant hMC3Rs\n",
      "  footer_text: a significantly different from WT hMC3R, p<0.05 b significantly different from WT hMC3R, p<0.01 c significantly different from WT hMC3R, p<0.001 The data are expressed as the mean ± SEM of three or four independent experiments. The maximal response under d-Trp8-γ-MSH stimulation was 2162 ± 338 pmol cAMP/106 cells for WT hMC3R.\n",
      "    unique: PMC2536753/PMC2536753.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Ligand Binding and α-MSH-Stimulated cAMP Production of WT and Mutant hMC3Rs\n",
      "  footer_text: a significantly different from WT hMC3R, p<0.05 b significantly different from WT hMC3R, p<0.01 c significantly different from WT hMC3R, p<0.001 The data are expressed as the mean ± SEM of three to five independent experiments. The maximal response under 〈-MSH stimulation was 1356 ± 180 pmol cAMP/106 cells for WT hMC3R.\n",
      "    unique: PMC2536753/PMC2536753.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Ligand Binding and γ-MSH-Stimulated cAMP Production of WT and Mutant hMC3Rs\n",
      "  footer_text: * ND, could not be detected. a significantly different from WT hMC3R, p<0.05 b significantly different from WT hMC3R, p<0.01 c significantly different from WT hMC3R, p<0.001 The data are expressed as the mean ± SEM of 3-5 independent experiments. The maximal response under γ-MSH stimulation was 1324 ± 309 pmol cAMP/106 cells for WT hMC3R.\n",
      "    unique: PMC2536753/PMC2536753.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536754.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of the power functions under various hypotheses between the Poisson-based test ZPOIS and the weighted-least-squares based test ZWLS for two overlapping regions over disjoint time intervals, Region 1 (1980–1989) vs Region 2 (1990–1999). APC1 and APC2 are the annual percent changes in Regions 1 and 2, respectively.\n",
      "  footer_text: \n",
      "    unique: PMC2536754/PMC2536754.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536753.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536753.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536754.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536754.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of the power functions between the Poisson based ZPOIS and the weighted-least-squares based ZWLS for two overlapping regions over roughly the same time intervals, Region 1 (1980–1989) vs Region 2 (1983–1992). APC1 and APC2 are the annual percent changes in Regions 1 and 2, respectively.\n",
      "  footer_text: \n",
      "    unique: PMC2536754/PMC2536754.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of the power functions between the Poisson based ZPOIS and the weighted-least-squares based ZWLS for two overlapping regions over slightly overlapping time intervals, Region 1 (1980–1989) vs Region 2 (1987–1996). APC1 and APC2 are the annual percent changes in Regions 1 and 2, respectively.\n",
      "  footer_text: \n",
      "    unique: PMC2536754/PMC2536754.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Comparing APC of Breast Cancer Mortality Between CA and the US\n",
      "  footer_text: \n",
      "    unique: PMC2536754/PMC2536754.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2536759.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536764.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536766.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: A Comparison of Reduction Potentials for Pt(IV) Compounds.\n",
      "  footer_text: en = ethylenediamine; ipa = isopropylamine; a = amine; cha = cyclohexylamine; dach = diaminocyclohexane\n",
      "    unique: PMC2536766/PMC2536766.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536754.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536754.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536766.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2536768.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinicopathological characteristics of study population\n",
      "  footer_text: \n",
      "    unique: PMC2536768/PMC2536768.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association of Demographics with Gene Methylation (PMR) (P values)\n",
      "  footer_text: 1 P values based on Kruskal-Wallis test. 2 P values based on Wilcoxon test. 3 Data grouped into the categories of Ta, CIS, T1, and T2-T4. 4 Data grouped into the categories of low grade (1-2) and high grade (3-4). 5 Data grouped into the categories of male and female. 6 Data grouped into the categories of ≤40, 41-50, 51-60, and 61-65 years at diagnosis. 7 Data grouped into the categories of never/irregular smokers and former/current smokers.\n",
      "    unique: PMC2536768/PMC2536768.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Impact of Sex, Age, Tumor Stage, Tumor Grade, and Smoking History on RUNX3 Methylation (PMR ≥ 10%, N=304)\n",
      "  footer_text: 1 p-value based on likelihood ratio test.\n",
      "    unique: PMC2536768/PMC2536768.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536768.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536768.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2536768.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2536769.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536792.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2536869.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2537462.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2537995.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538376.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538424.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of Filament Length and Filament Extension (Pitch) Measurements\n",
      "  footer_text: All measurements are on M13mp18 cssDNA and lengths are in nm. For filament length measurements n=19 for wt, n=57 for E38K K72R and n=85 for K72R. For pitch measurements 50 individual pitches from 6-10 different molecules were measured for all conditions. * N.A. is not available since RecA K72R did not form filaments under these conditions. N.D. is not determined.\n",
      "    unique: PMC2538424/PMC2538424.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538424.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538424.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538424.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538424.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The Level of SOS Expression before and after UV Treatment\n",
      "  footer_text: Shows the average of 1000-3000 total cells analyzed from three different experiments. N.D. is not determined.\n",
      "    unique: PMC2538424/PMC2538424.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of Sensitivity to UV light and Recombination Phenotypes of RecA Mutants\n",
      "  footer_text: a UV Resistance is measured by % survival of strains after 10J of UV irradiation b Conjugation frequency reported as % transconjugants/100 donors\n",
      "    unique: PMC2538424/PMC2538424.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Strains Used in this Work\n",
      "  footer_text: a JC13509 has the following partial genotype: sulB103- lacMS286 Φ80dIIlacBK1 argE3 his-4 thi-1 xyl-5 mtl-1 rplS31 tsx. The lacMS286 Φ80dIIlacBK1 codes for two partial non-overlapping deletions of the lac operon. See (Konrad, 1977; Zieg and Kushner, 1977). b attλ∷sulApΩgfp-mut2 is abbreviated Ωgfp. c pKD46 expresses the Lambda Red proteins (Datsenko and Wanner, 2000). d P1 transduction with SS4660 as a recipient was carried out as normal with the exception that the strain was grown in Luria broth with ampicillin (50 μg/mL) and induced with 0.2% arabinose 1 hour before harvesting the culture.\n",
      "    unique: PMC2538424/PMC2538424.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538429.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538488.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538554.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Functional roles of conserved catalytic motifs among motor ATPase families. Abbreviations: W-A = Walker A, W-B = Walker B, RF = arginine finger, CE = catalytic glutamate, H = H loop, D = D loop, Q = Q loop, C = C region or ABC signature motif, SI = sensor I, SII = sensor II. All other motifs are listed as roman numerals.\n",
      "  footer_text: \n",
      "    unique: PMC2538554/PMC2538554.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538554.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538563.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538563.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538566.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538575.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2538563.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Fragments Generated from Chymotryptic Digest of CaM+VLRRRRKRVN (D2NI3L)\n",
      "  footer_text: \n",
      "    unique: PMC2538563/PMC2538563.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Fragments Generated from Chymotryptic Digest of CaM+RIREFRQTFR (A2A−1)\n",
      "  footer_text: \n",
      "    unique: PMC2538563/PMC2538563.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538566.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Characteristics\n",
      "  footer_text: * BNP was log transformed for analysis in determining p value. Age, BMI, and creatinine clearance values are expressed as mean (standard deviation). BNP values are expressed as median (interquartile range). All other values are given as frequency (%). BMI= body mass index, BNP= brain natriuretic peptide, COPD= chronic obstructive pulmonary disease, NYHA= New York Heart Association\n",
      "    unique: PMC2538566/PMC2538566.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538575.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: KCNQ1 mutations in study patients\n",
      "  footer_text: * p-value for the comparison between Caucasians and Japanese patients <0.05. † Splice site and in-frame deletion mutation-types were assumed haploinsufficiency.\n",
      "    unique: PMC2538575/PMC2538575.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Characteristics of Caucasian and Japanese LQT1 patients\n",
      "  footer_text: Plus-minus values are means±SD. † Denotes the number of patients who experienced at least one cardiac event during follow-up. Mean QTc values include 87 cases; 13 cases were missing: 6 male and 7 female. ICD = implantable cardioverter defibrillator.\n",
      "    unique: PMC2538575/PMC2538575.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538575.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538575.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538575.xml::T3-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multivariate analysis*\n",
      "  footer_text: * The analysis involved 87 cases; 13 cases were omitted due to missing QTc values; virtually identical results were obtained when analysis were repeating with censoring of patients upon implantation of an ICD. † β-blocker therapy was analyzed as a time-dependent variable in the multivariate model.\n",
      "    unique: PMC2538575/PMC2538575.xml::T3-0\n",
      "    unique: PMC2538575/PMC2538575.xml::T3-1\n",
      "\n",
      "PMC0025XXXXX/PMC2538577.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538583.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538606.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics by race\n",
      "  footer_text: * Abbreviations: DVT=deep vein thrombosis; PE=pulmonary embolism; TIA=transient ischemic attack † P-values comparing characteristics by race\n",
      "    unique: PMC2538606/PMC2538606.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538606.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538606.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Dosing algorithm for Caucasians in the multivariate linear regression model (outcome variable is log warfarin maintenance dose per day).*\n",
      "  footer_text: * Dose (mg/day) = exp[2.74 −(0.01 X age) −(0.40 X VKORC1 1173CT) −(0.82 X VKORC1 1173TT) −(0.21 X CYP2C9 any*2) −(0.32 X CYP2C9 any*3) −(0.18 X BMI <25) −(0.19 X BMI 25 to 30) −(0.08 X number of medications that potentiate warfarin) +(0.03 X most alcoholic drinks on one occasion) +(0.21 X history of deep vein thrombosis) −(0.13 X female)] † Age, number of medications that potentiate warfarin, and most drinks on one occasion were entered as linear terms in the model and dummy variables (0=absent, 1=present) for the other variables.\n",
      "    unique: PMC2538606/PMC2538606.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Dosing algorithm for African Americans in the multivariate linear regression model (outcome variable is log warfarin maintenance dose per day). *\n",
      "  footer_text: * Dose (mg/day) = exp[2.66 −(0.01 X age) −(0.36 X VKORC1 1173CT or 1173TT) −(0.19 X Factor 7 −401GT) −(0.25 X Factor 7 −401TT) −(0.28 X APOE E2/E2 or E2/E3) −(0.21 X APOE E3/E3 or E2/E4) −(0.19 X BMI <25) −(0.25 X BMI 25 to 30) −(0.09 X number of medications that potentiate warfarin)] † Age and number of medications that potentiate warfarin were entered as linear terms in the model and dummy variables (0=absent, 1=present) for the other variables.\n",
      "    unique: PMC2538606/PMC2538606.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Dosing algorithm for African Americans and Caucasians in the multivariate linear regression model (outcome variable is log warfarin maintenance dose per day).*\n",
      "  footer_text: * Dose (mg/day) = exp[2.81 −(0.08 X African American) −(0.01 X age) −(0.39 X VKORC1 1173CT) −(0.82 X VKORC1 1173TT) −(0.11X Factor 7 −401GT) −(0.19X Factor 7 −401TT) −(0.16 X CYP2C9 any*2) −(0.30 X CYP2C9 any*3) −(0.18 X BMI <25) −(0.23 X BMI 25 to 30) –(0.08 X number of medications that potentiate warfarin) −(0.10 X female) +(0.06 X most alcoholic drinks on one occasion) −(0.03 X African American x most drinks on one occasion) +(0.58 X history of deep vein thrombosis) −(0.33 X African American x history of deep vein thrombosis)] † Age, number of medications that potentiate warfarin, and most drinks on one occasion were entered as linear terms in the model and dummy variables (0=absent, 1=present) for the other variables.\n",
      "    unique: PMC2538606/PMC2538606.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Validation results of other algorithms in our cohort and cross validation of our models\n",
      "  footer_text: * Gage et al. algorithm; dose (mg/day) = exp[0.385 −(0.0083 X age) +( 0.498 X BSA) – ( 0.208 X CYP2C9*2) −( 0.350 X CYP2C9*3) −( 0.341 X amiodarone) + ( 0.378 X Target INR) −( 0.125 X simvastatin) −(0.113 X race) −( 0.075 X female)] † Hillman et al. algorithm; dose (mg/week) =exp[1.654 + (0.429 X CYP2C9 *2 or *3) + (−0.274 X CYP2C9*3) + (0.221 X valve replacement) + (−0.035 X age) + (2.886 X BSA) + (0.936 X diabetes mellitus) + (0.0002 X age2) + (−0.604 X BSA2) + (−0.014 X age*diabetes mellitus) + (−0.305 X BSA*CYP2C9 *2 or *3) + (0.129 X diabetes mellitus*CYP2C9 *2 or *3)] ‡ Sconce et al. algorithm; vDose (mg/day)= 0.628 − 0.0135 (Age)−0.240 (CYP*2)− 0.370(CYP*3) − 0.241 (VKOR)+0.0162 (Height) § Gage et al. algorithm; Dose (mg/day) = exp[0.9751 −(0.3238 X VKORC1 −1639G/A) + (0.4317 X BSA) −(0.00745 X age) −(0.4008 X CYP2C9*3) −(0.2066 X CYP2C9*2) + (0.2029 X Target INR) −(0.2538 X amiodarone) + (0.0922 X smokes) −(0.0901 X African American race) + (0.0664 X DVT/PE)] || Anderson algorithm (mg/week); dose =1.64+exp[3.984+CYP2C9*1*1(0)+*1*2(−0.197)+*1*3(−0.360)+*2*3(−0.947)+*2*2(−0.265)+*3*3(−1.892)+Vk-CT(−0.304)+Vk-TT(−0.569)+Vk-CC(0)+age(−0.009)+male sex(0.094)+female sex(0)+weight in kg(0.003)] # See Table 1 ** See Table 2 †† See Table 3 ‡‡ R2 estimate in the race combined cohort\n",
      "    unique: PMC2538606/PMC2538606.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538607.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic characteristics by HAART status of 4093 AIDS subjects included in study\n",
      "  footer_text: † 8 cases missing; ‡ 78 cases missing\n",
      "    unique: PMC2538607/PMC2538607.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538606.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538606.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538606.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538607.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Unadjusted association of mortality and risk factors by HAART status\n",
      "  footer_text: † Opportunistic Infections (OI) include fever, diarrhea, thrush, and oral hairy leukoplakia. NA: the covariate was not considered for proportional hazard models.\n",
      "    unique: PMC2538607/PMC2538607.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Adjusted association of mortality and risk factors by HAART status\n",
      "  footer_text: 1 The initial full model included gender, age at AIDS diagnosis, marriage, education, and on HAART; 2 The initial full model included gender, age at AIDS diagnosis, marriage, education, number of OIs at HAART initiation, and CD4 counts at HAART initiation. 3 The initial full model included gender, age at AIDS diagnosis, marriage, education, and occupation. NA: the covariate was not considered for proportional hazard models. NS: the covariate was either insignificant, not entered into initial full model, or deleted from the model selection procedure.\n",
      "    unique: PMC2538607/PMC2538607.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538608.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Experimental design. Initial letters denote the type of cues used each day. Suffixes denote whether the context was the same or different for days two, three, and four; no suffix is used for the first day because it had no preceding day. On day one, the initial alcohol cue reactivity hypothesis was Control < Single Context = Multiple Context. At the test trial, the alcohol cue reactivity hypothesis was Control > Single Context > Multiple Context.\n",
      "  footer_text: Notes: N = Neutral Cues, A = Alcohol Cues; for day two on, suffix of S = Same as preceding day, suffix of D = different from preceding day.\n",
      "    unique: PMC2538608/PMC2538608.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538607.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538607.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538608.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2538611.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Effect of immunization route and number and identity of priming immunogens on vaccine elicited protection in the rhesus macaque model.\n",
      "  footer_text: # includes a subset of 3, non-Mamu A*01 + elite controllers exhibiting either undetectable or transient low level viremia; ongoing investigation of immune correlates IN = intranasal; IM = intramuscular; IV = intravenous; O = oral; IT = intratracheal; Ivag = intravaginal; IR = intrarectal\n",
      "    unique: PMC2538611/PMC2538611.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538611.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538613.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538613.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538613.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2538613.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic Characteristics and Estimated IQ for Final Sample and Statistical Comparisons of Subsamples\n",
      "  footer_text: Note. Total Sample N = 100; Community Sample n = 26; Student Sample n = 74. Statistics in bold indicate significant differences between the community and subject pool subsamples.\n",
      "    unique: PMC2538613/PMC2538613.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Descriptive Statistics for the Psychopathic Personality Inventory and Statistical Comparisons of Subsamples\n",
      "  footer_text: Note. Total Sample N = 100; Community Sample n = 26; Student Sample n = 74. Statistics in bold indicate significant differences between the community and subject pool subsamples.\n",
      "    unique: PMC2538613/PMC2538613.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean Reaction Time and Total Error Rate by Load and Distractor Compatibility for the Perceptual Load Task and Cognitive Control Task\n",
      "  footer_text: Note. Standard deviations are presented in parentheses. N = 98 for the Perceptual Load Task. N = 97 for the Cognitive Control Task.\n",
      "    unique: PMC2538613/PMC2538613.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Correlations between Reaction Time Interference and PPI factors by Load Condition for Perceptual Load Task and Cognitive Control Task\n",
      "  footer_text: Note. n = 98 for all perceptual load correlations. n = 97 for all working memory load correlations. PPI = Psychopathic Personality Inventory. ** indicates p < .01. * indicates p < .05. The letter a indicates the correlations between the factor scores (PPI-CU and PPI-II) and RT are significantly different, p = .017\n",
      "    unique: PMC2538613/PMC2538613.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538613.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538615.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538615.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538615.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538615.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538615.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2538615.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean Gingival Index values at control and gingivitis sites§\n",
      "  footer_text: § Data are expressed as median and range of eight subjects. Within columns, values that are statistically significant are denoted by * (P = 0.008, Wilcoxon Signed Rank test).\n",
      "    unique: PMC2538615/PMC2538615.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean Plaque Index values at control and gingivitis sites§\n",
      "  footer_text: § The data are reported as median and range of eight subjects. Within columns, values that are statistically significant are denoted by * (P = 0.008, Wilcoxon Signed Rank test).\n",
      "    unique: PMC2538615/PMC2538615.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean gingival crevicular fluid flow rate at control and gingivitis sites§\n",
      "  footer_text: § The data are expressed as mean flow rate (in μl per 45 seconds) ± SEM. Baseline values at control and experimental gingivitis sites were very similar (P = 0.54). On day 24, there was a statistically significant difference between control and experimental gingivitis sites (^ denotes P <0.001). Within rows, significant differences from baseline are denoted (* P = 0.015, # P = 0.018, paired t tests adjusted for multiple comparisons).\n",
      "    unique: PMC2538615/PMC2538615.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Mean clarithromycin levels in serum, healthy control gingiva and inflamed gingiva§\n",
      "  footer_text: § Data are presented as mean ± SEM. Within rows, * denotes values that were significantly different from serum levels, while # indicates data in which all three pairwise comparisons yielded significant differences (P <0.05, Holm-Sidak test).\n",
      "    unique: PMC2538615/PMC2538615.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Ratio of tissue clarithromycin concentration to serum clarithromycin concentration in healthy control gingiva and gingivitis sites§\n",
      "  footer_text: § Data are expressed as mean ± SEM. Within rows, * denotes a significant difference between values (P ≤0.02, paired t test). * Periopaper, Oraflow Inc, Plainview, NY, USA. † Periotron 6000, Oraflow, Inc, Plainview, NY, USA. § Supelcosil, Sigma-Aldrich, St. Louis, MO, USA. ¶ United States Pharmacopeia, Rockville, MD, USA.\n",
      "    unique: PMC2538615/PMC2538615.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538616.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The Calculated MP2 Energies of Models A-F\n",
      "  footer_text: a Model A: extracted from the crystal structure; models B-D: ligand-rotated 90°, 180°, 270° relative to model A; model E: ligand-flipped orientation of model A; model F: ligand-rotated 180° relative to model E. b MP2 energies without BSSE. c ΔE1 and ΔE2 refer to the energy difference (kcal/mol) between model A and other models calculated at MP2/6−31G*, while ΔE3 and ΔE4 refer to the energy difference (kcal/mol) between model A and other models calculated at MP2/6−31G*(0.25). d MP2 energies with BSSE correction.\n",
      "    unique: PMC2538616/PMC2538616.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The Solvation Energies (kcal/mol) from MP2/6−31G*(0.25) and B3LYP/6−31G* Using the PCM Model\n",
      "  footer_text: a ΔGsolvation(MP2) = EPCM-MP2 — EMP2, ΔGsolvation(DFT) = EPCM-DFT — EDFT b ΔEsolvation was the relative solvation energy of the models investigated at the MP2 level. c ΔE was the relative energy in aqueous solution of the models investigated at the MP2 level, which was computed as the sum of the relative energy in gas-phase (ΔE4 in Table 1) and relative solvation energy (ΔEsolvation). d The results were obtained using B3LYP/6−31G* and the UAKS radii scheme. e The results were obtained using B3LYP/6−31G* and the UAHF radii scheme.\n",
      "    unique: PMC2538616/PMC2538616.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The Binding Energies (kcal/mol) of Models A-F\n",
      "  footer_text: a From MP2/6−31G*(0.25) calculations with BSSE. b Ecorr = Eint(MP2) — Eint(HF). c ECT was calculated from HF/6−31G**.\n",
      "    unique: PMC2538616/PMC2538616.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538619.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Antiproliferative and Topoisomerase I Inhibitory Activities of 14-Substituted Aromathecin Analogues\n",
      "  footer_text: a GI50 values are the concentrations corresponding to 50% growth inhibition. b Mean graph midpoint for growth inhibition of all human cancer cell lines successfully tested, ranging from 10−8 to 10−4 M. c Compound-induced DNA cleavage due to Top1 inhibition is graded by the following rubric relative to 1 μM camptothecin: 0, no inhibitory activity; +, between 20% and 50% activity; ++, between 50% and 75% activity; +++, between 75% and 95% of activity; ++++, equipotent. d 22-Hydroxyacuminatine was not tested in the National Cancer Institute assay. e These compounds were not selected for further testing; refer to text for details. f Currently undergoing fivedose testing.\n",
      "    unique: PMC2538619/PMC2538619.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538616.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538616.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538616.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538619.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2538620.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538621.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Variation of membrane and rhodopsin properties as a function of lipid chain length. The average order parameter (ΔSave), estimated hydrophobic thickness (d), rhodopsin molar ellipticity at λ= 222 nm [θ]222, increase in helical content relative to helical content in 14:0d27-14:1-PC (& relative to18:0d35-18:1-PC) , the ratio MII/MI after photoactivation at 37°C, Keq, apparent time constant of MII formation at 37°C, Tapp, and temperature of the midpoint of rhodopsin thermal denaturation, Td, as a function of lipid chain length.\n",
      "  footer_text: * The hydrophobic thickness d, defined as the distance between acyl chain carbonyls of the two apposing monolayers was calculated as in (17). # Temperature was 27°C (all other CD, SRCD experiments were conducted at 21°C).\n",
      "    unique: PMC2538621/PMC2538621.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538623.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538624.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538625.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: TCI-R correlates with putative measures of response distortion\n",
      "  footer_text: Notes: MPQ = Multidimensional Personality Questionnaire; BIDR = Balanced Inventory of Desirable Responding. Ns vary from 651 to 654. *** p < .001, ** p < .01, * p < .05\n",
      "    unique: PMC2538625/PMC2538625.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538626.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538621.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538625.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Pairwise all-atom root-mean-squared deviation values between IRE conformation cluster median snapshots. All values in Å.\n",
      "  footer_text: \n",
      "    unique: PMC2538626/PMC2538626.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Optimal parameters for the best-fit nonpolar solvent radii and associated quality of fit metrics (see text for definitions). Parameter 99% confidence intervals are presented in square brackets. The “SASA only” and related labels refer to the model of the nonpolar cavity creation term and are described in detail the manuscript text.\n",
      "  footer_text: \n",
      "    unique: PMC2538626/PMC2538626.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Parameters for specific nonpolar solvent radii and associated quality of fit metrics (see text for definitions). Parameter 99% confidence intervals are presented in square brackets. Parameters without a * are the best fit results for that σs; parameters with a * are from the previous work of Wagoner et al [47] and included to demonstrate the extent of parameter transferability between systems. The “SASA only” and related labels refer to the model of the nonpolar cavity creation term and are described in detail the manuscript text.\n",
      "  footer_text: \n",
      "    unique: PMC2538626/PMC2538626.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538627.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538674.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538677.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538685.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538687.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538752.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Association of MUC4 expression pattern with types and grades of gastric cancer\n",
      "  footer_text: \n",
      "    unique: PMC2538752/PMC2538752.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538752.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538752.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2538752.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association of MUC4 expression pattern with stages of gastric cancer\n",
      "  footer_text: *** Fisher's Exact test. Information regarding the tumour stage was not available for all the spots.\n",
      "    unique: PMC2538752/PMC2538752.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Incidence of tumours in animals injected with AGS-MUC4 and AGS-pSecTagC cells\n",
      "  footer_text: *** Fisher's Exact test.\n",
      "    unique: PMC2538752/PMC2538752.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2538802.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2538952.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2539063.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2539065.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2542504.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2542582.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2542583.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Number of children (out of seven) who produced fewer than five stops in a category during the middle 10 min of recording, by session.\n",
      "  footer_text: \n",
      "    unique: PMC2542583/PMC2542583.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542583.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542584.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2542584.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Types of collaboration and corresponding leadership characteristics\n",
      "  footer_text: \n",
      "    unique: PMC2542584/PMC2542584.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2542876.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2542881.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2542884.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Experiment 1 Lexicon\n",
      "  footer_text: \n",
      "    unique: PMC2542884/PMC2542884.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Experiment 2 Lexicon\n",
      "  footer_text: \n",
      "    unique: PMC2542884/PMC2542884.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Experiment 3 Lexical Items\n",
      "  footer_text: \n",
      "    unique: PMC2542884/PMC2542884.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2542896.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: NNK content in Quest cigarettes before and after spiking with [pyridine-D4]NNK\n",
      "  footer_text: a Total NNK (the sum of [pyridine-D4]NNK and unlabeled NNK) measured by GC-TEA b Sum of [pyridine-D4]NNK and unlabeled NNK measured separately by GC-MS/MS\n",
      "    unique: PMC2542896/PMC2542896.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary statistics for deuterated biomarkers analyzed in 20 study participants\n",
      "  footer_text: a Free [pyridine-D4]NNAL plus [pyridine-D4]NNAL-Glucs b Urine samples were treated with NaBH4 and analyzed for [pyridine-D4]hydroxy acid\n",
      "    unique: PMC2542896/PMC2542896.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Separate analysis of deuterated and unlabeled keto acid and hydroxy acid in 3 study participants\n",
      "  footer_text: a Urine samples were treated with NaBH4 and analyzed for hydroxy acid b Urine samples were analyzed for hydroxy acid without NaBH4 reduction step. c The amount of keto acid was calculated by subtracting hydroxy acid from the sum of keto acid and hydroxy acid.\n",
      "    unique: PMC2542896/PMC2542896.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542884.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542884.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542884.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542896.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542896.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542896.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Summary statistics for biomarker ratios\n",
      "  footer_text: a [Pyridine-D4]keto acid plus [pyridine-D4]hydroxy acid\n",
      "    unique: PMC2542896/PMC2542896.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Urinary excretion of NNK metabolites in mice, rats, primates, and humans\n",
      "  footer_text: a Data for urinary metabolites excreted at low NNK doses (11) b From ref. 9 c Combined data from ref. 13, 17, and the current study (based on geometric means for deuterated metabolites), and assuming that there are no other NNK metabolic pathways contributing to urinary metabolites. d ND, not detected e NA, not analyzed\n",
      "    unique: PMC2542896/PMC2542896.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2542982.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The frequency of opening each door for all congruent trials, Violation-of-Solidity conflict trials and Stop-Early conflict trials. The first number in each cell indicates the number of times that particular door was chosen. The second number in each cell indicates the number of times that particular door represented the correct choice. The final number in each cell indicates the proportion between the number of times the door was chosen over the number of times the door was correct\n",
      "  footer_text: \n",
      "    unique: PMC2542982/PMC2542982.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2543091.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Serum concentrations (in ng/mL) in 12 Faroese mothers (collected in 2000) and their 5 year old children (collected in 2005)\n",
      "  footer_text: \n",
      "    unique: PMC2543091/PMC2543091.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Serum concentrations (in ng/mL) in the Faroese 7-year-old children (n = 103, collected in 1993–1994) and in the 14-year-old children (n=79, collected in 2000–2001)\n",
      "  footer_text: \n",
      "    unique: PMC2543091/PMC2543091.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean serum concentrations (in ng/mL) in 79 Faroese adolescents at age 14 years (in 2000–2001) in relation to the monthly number of pilot whale dinners (numbers in parenthesis).\n",
      "  footer_text: * p < 0.05; ** p < 0.01\n",
      "    unique: PMC2543091/PMC2543091.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542896.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542896.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2542982.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543091.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543091.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2543121.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Plasma and hepatic lycopene isomer concentrations in rats fed diets with or without alcohol and lycopene for 11 wk1–4\n",
      "  footer_text: 1 Values are means ± SEM, n = 9–10. Means in a row without a common superscript differ, P < 0.05 (1-way ANOVA followed by Tukey’s test for differences between means). 2 Lycopene (Lyc) was not detected in placebo-fed rats with or without alcohol. 3 P-values of main effects and the interaction were determined by 2-way ANOVA. 4 One outlier from each of the alcohol + low Lyc and alcohol + high Lyc groups was excluded from final analyses.\n",
      "    unique: PMC2543121/PMC2543121.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2543933.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2543936.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: CMS schedule\n",
      "  footer_text: * Restricted movements consisted of placing rats into a mouse cage (24W×10L×9H cm) for 1 hr.\n",
      "    unique: PMC2543936/PMC2543936.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2543944.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543091.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543121.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543936.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2543944.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Nomenclatures of genes and gene products analyzed in this study Human and mouse SynCAM proteins are encoded by the genes CADM 1–4 and Cadm 1–4, respectively. Different nomenclatures used for individual gene products are listed.\n",
      "  footer_text: \n",
      "    unique: PMC2543944/PMC2543944.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2544468.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546443.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546465.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546488.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546489.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546495.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The neuropeptide standards used for the off-line CE-MALDI MS experiments\n",
      "  footer_text: CCAP, Crustacean Cardioactive Peptide; AST, Allatostatin; CabAST-B1, C. borealis B-type allatostatin 1; UP, Uremic Pentapeptide; Pk, Pyrokinin; Ag, Angiotensin; Ok, Orcokinin. q is net charge of peptide, M is molecular mass. Mf., the custom defined matching factor of the peptide for observed and predicted migration time in CE. The Mf. (c) and Mf. (d) values resulting from the linear fitting in Figure 2 (c) and (d). CI, confidence interval. a Outlier in statistical analysis due to the extra large Mf. value.\n",
      "    unique: PMC2546495/PMC2546495.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546495.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546495.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546505.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546505.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Representative examples of the peptide assignment via off-line CE-MALDI MS\n",
      "  footer_text: Mass underlined, the unidentified peptides. Samples are from tissue(s) of pericardial organs (PO), Brain (Br), commissural ganglia (CoG), respectively. The columns under PO: the N column, normal sample without isotopic formaldehyde labeling; the (p) column, peptides detected as pairs with+28/+32 Da mass shift upon labeling; the (s) column, peptides detected as unlabeled single peaks. “+” shows that the peptides are detected in the samples. Er., the mass error in ppm. Mf., the custom defined matching factor of the peptide for observed and predicted migration time in CE. AST -A, -B: Allatostatins A and B types. Orco-LP: Orcokinin-like peptide. See the text for the details of these data analysis.\n",
      "    unique: PMC2546495/PMC2546495.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2546505.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic factors associated with herpes simplex virus type-2 (HSV-2) seropositivity among 737 female sex workers (FSWs)\n",
      "  footer_text: * Reference group\n",
      "    unique: PMC2546505/PMC2546505.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: STIs and relevant symptoms associated with HSV-2 seropositivity among 737 FSWs\n",
      "  footer_text: † Reference group\n",
      "    unique: PMC2546505/PMC2546505.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Behavioural characteristics associated with herpes simplex virus type-2 (HSV-2) seropositivity among 737 female sex workers (FSWs)\n",
      "  footer_text: * Reference group\n",
      "    unique: PMC2546505/PMC2546505.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546505.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546505.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546574.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546574.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Multivariate analyses of risk and protective factors of herpes simplex virus type-2 (HSV-2) infection among 737 female sex workers (FSWs), based on the logistic regression model*\n",
      "  footer_text: * To reduce residual confounding, years of education, duration of sex work and number of clients in the preceding week were entered into models as continuous variables. The following odds ratios and 95% confidence intervals were found for these risk factors: per increased year of education: 0.9 (0.87–0.98); per additional client: 1.1 (1.02–1.11) and per additional year in sex work: 1.1 (1.03–1.18)\n",
      "    unique: PMC2546505/PMC2546505.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2546523.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546527.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2546574.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean values of participant characteristics. with standard deviations in parentheses.\n",
      "  footer_text: * p < .05 ** p < .001 *** p < .0001 1 (poor = 1 – excellent = 5) 2 gender was not recorded for two participants. WAIS: Wechsler Adult Intelligence Scale III (Wechsler, 1997a) WMS: Wechsler Memory Scale III (Wechsler, 1997b)\n",
      "    unique: PMC2546574/PMC2546574.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: High frequency triplets for Experiment 3 with their overall joint probability and cue-pair conditional probability.\n",
      "  footer_text: \n",
      "    unique: PMC2546574/PMC2546574.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Best fitting individual stepwise regression coefficients for cue-pair and single-cue conditional probabilities and each participant in Experiment 3.\n",
      "  footer_text: \n",
      "    unique: PMC2546574/PMC2546574.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2546574.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547116.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547125.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2546611.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547116.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Infant (1–11 months) and childhood (5 years) parasite prevalence in Ngerenya from 1993 to 2005\n",
      "  footer_text: Confidence intervals are exact binomial confidence intervals. * Personal communications. † Snow and others,1 August 1994 and March 1995 surveys. ‡ Mwangi and others15 1998–2001. § Personal comunications. ¶ Children 5 years of age (60–71 months) only are included to compare cross-sectional surveys in the lingitudinal cohort. This prevernts any child from appearing in more than one PR estimate. ** May cross-sectional surveys only to avoid any child from contributing to the denominator twice.\n",
      "    unique: PMC2547116/PMC2547116.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547124.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547125.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Molecular markers used to distinguish MN columnar identities\n",
      "  footer_text: * SCIP has also been observed to label LMCm motor pools in the caudal brachial and caudal lumbar spinal cord (Dasen et al., 2005; Luria and Laufer, 2007). To avoid confusion with these LMCm MNs, our analyses examine SCIP expression only in the rostral portion of these spinal cord segments.\n",
      "    unique: PMC2547125/PMC2547125.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547131.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547133.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547134.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Homology of MPV vCCI to CPV vCCI, RPV vCCI, VARV vCCI, and VV COP vCCI.\n",
      "  footer_text: \n",
      "    unique: PMC2547134/PMC2547134.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547144.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547344.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Correlation of clinicopathological variables with TG2 overexpression in patients with invasive ovarian cancer\n",
      "  footer_text: \n",
      "    unique: PMC2547344/PMC2547344.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547354.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2547354/PMC2547354.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547487.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547489.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547134.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547344.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547354.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2547490.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547491.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: CCA Risk Factors\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Molecular Pathology of CCA\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: TNM Classification and UICC Staging System for Intrahepatic CCA\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Memorial Sloan-Kettering Cancer Center Staging System for Hilar CCA\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Exclusion Criteria for Surgical Resection of Hilar CCA\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Mayo Clinic Criteria for Liver Transplantation for Hilar CCA\n",
      "  footer_text: \n",
      "    unique: PMC2547491/PMC2547491.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547494.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Information on RNA probes used for in situ hybridization experiments. The size of the cDNA determines the region of the RNA from which they were derived, as their 3′ end is at the terminus of the transcript; except for ErbB4, where the insert encodes amino acids 326-633.\n",
      "  footer_text: \n",
      "    unique: PMC2547494/PMC2547494.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547491.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547491.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547491.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547494.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547494.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547494.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Information on the antibody reagents; details on the specificity controls are described in the Materials and Methods.\n",
      "  footer_text: \n",
      "    unique: PMC2547494/PMC2547494.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Quantification of numbers of cells that are positive for PLAP, DLX2 and both PLAP & DLX2 immunoreactivity in the E14.5 and E18.5 cortex of Lhx6PLAP/+ and Lhx6PLAP/PLAP, using confocal data from Figure 1. The numbers represent the areal density (positive cells per 1000 μm2 of the box surface area) in cortex subregions. MZ: marginal zone; CP: cortical plate; II–IV: layers II, III and IV; V: layer V; VI: layer VI; SVZ: subventricular zone; VZ: ventricular zone.\n",
      "  footer_text: \n",
      "    unique: PMC2547494/PMC2547494.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Quantification of numbers of cells positive for PLAP histochemistry in different cortical layers of E18.5 Lhx6PLAP/+ and Lhx6PLAP/PLAP, using light microscopic data from Figure 2 (20 μm section plane). Boxes 1, 2 & 3 correspond to different dorsoventral regions of the neocortex as indicated in Fig. 2C&D. The numbers represent the areal density (positive cells per 1000 μm2 of the box surface area) in cortex subregions. Abbreviations: see Table 3.\n",
      "  footer_text: \n",
      "    unique: PMC2547494/PMC2547494.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547848.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Strain differences in c-Fos expression in cortical and amygdaloid regions following extinction recall. 129S1 mice had fewer c-Fos positive cells in IL and BLA and more c-Fos positive cells in CeA than C57BL/6J after extinction recall. No other cortical or amygdaloid region examined differed between strains (n=8/strain). Baseline c-Fos expression did not differ between strains baseline in any brain region examined (n=4/strain). Data are Mean ±SEM c-Fos positive cells per 0.01 mm2. For each region, the abbreviation and plane of section relative to Bregma is given in parenthesis. *P<.01 C57BL/6J vs. 129S1/same condition; #P<.01 extinction recall vs. control.\n",
      "  footer_text: \n",
      "    unique: PMC2547848/PMC2547848.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Strain differences in Zif268 expression in cortical and amygdaloid regions following extinction recall. 129S1 mice had fewer Zif268 positive cells in IL, LA, BLA and Imp than C57BL/6J after extinction recall. No other cortical or amygdaloid region examined differed between strains (n=7−8/strain). Baseline Zif268 expression did not differ between strains in any brain region examined (n=4/strain). Data are Mean ±SEM Zif268 positive cells per 0.01 mm2. For each region, the abbreviation and plane of section relative to Bregma is given in parenthesis. *P<.01 C57BL/6J vs. 129S1/same condition; #P<.01 extinction recall vs. control. a whole nucleus counted.\n",
      "  footer_text: \n",
      "    unique: PMC2547848/PMC2547848.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547494.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547848.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547848.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Absence of strain differences in c-Fos or Zif268 expression in cortical and amygdaloid regions following fear recall. 129S1 and C57BL/6J mice did not differ in c-Fos or Zif268 positive cells in any brain region examined (n=10/strain). Data are Mean ±SEM c-Fos and Zif268 positive cells per 0.01 mm2. For each region, the abbreviation and plane of section relative to Bregma is given in parenthesis. awhole nucleus counted.\n",
      "  footer_text: \n",
      "    unique: PMC2547848/PMC2547848.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547880.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Nutrient composition of the study diets\n",
      "  footer_text: 1 Diet high in monounsaturated fat. 2 Diet high in complex carbohydrate. 3 Complex carbohydrate refers to polysaccharides (starch), such as those in cereal products, vegetables, legumes, and nuts. Simple carbohydrate refers to monosaccharides or disaccharides, such as those in fruit, beverages, and desserts.\n",
      "    unique: PMC2547880/PMC2547880.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2547887.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547984.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2547989.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Menstrual cycles observed in 89 chimpanzees.\n",
      "  footer_text: \n",
      "    unique: PMC2547989/PMC2547989.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547848.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547880.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2547989.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2547996.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548276.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548278.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means and standard deviations for demographics and clinical symptoms among the three groups\n",
      "  footer_text: \n",
      "    unique: PMC2548278/PMC2548278.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Key activation in brain regions labeled in SPM analysis shown in Figure 10. SPM analysis is between-groups and for the attend minus ignore PPI condition differences\n",
      "  footer_text: BA=Brodmann area, L=Left hemisphere, R=Right hemisphere\n",
      "    unique: PMC2548278/PMC2548278.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548279.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548278.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548278.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: PCR primers.\n",
      "  footer_text: \n",
      "    unique: PMC2548279/PMC2548279.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548286.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Proposed Targets of Antiretinal Antibodies\n",
      "  footer_text: Retinal proteins which are proposed to be targets of antiretinal antibodies are shown, as well as the assay method(s) used to detect their corresponding antiretinal antibodies. WB = Western blot, IHC = immunohistochemistry, ELISA = enzyme-linked immunosorbent assay.\n",
      "    unique: PMC2548286/PMC2548286.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548289.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Classification of Iron Overload Syndromes\n",
      "  footer_text: \n",
      "    unique: PMC2548289/PMC2548289.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548291.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Monochlorophenols and dichlorophenols as substrates for hSULT2A1.\n",
      "  footer_text: a Relative activity is expressed as a percentage based on the ratio of the rate of sulfation of the chlorinated phenol at 50 µM to the rate of sulfation of 50 µM DHEA under the same assay conditions.\n",
      "    unique: PMC2548291/PMC2548291.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Tri-, tetra-, and pentachlorophenols as substrates for hSULT2A1.\n",
      "  footer_text: a Relative activity is expressed as a percentage based on the ratio of the rate of sulfation of the chlorinated phenol at 50 µM to the rate of sulfation of 50 µM DHEA under the same assay conditions.\n",
      "    unique: PMC2548291/PMC2548291.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548286.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548291.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548291.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2548298.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548310.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548317.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548318.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Cell cycle progression in nocodazole blocked cells\n",
      "  footer_text: Data represent the means ± SEM of two (PC3, HCT) and three (MDCK) independent experiments.\n",
      "    unique: PMC2548318/PMC2548318.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548319.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Exploring Optimal Conditions for Formation and Cross Coupling of Organoindium Reagents\n",
      "  footer_text: a Refers to isolated yields after column chromatography. b Methanol (10% by volume) was added to the DMF solution of the indium reagent just prior to the cross-coupling reaction.\n",
      "    unique: PMC2548319/PMC2548319.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Reaction of Diverse Aryl Iodides with In•2LiCl: Scope and Cross-Coupling Reactions\n",
      "  footer_text: a For the insertion reaction, conversions were assayed by GC–MS analysis of an aliquot quenched with HCl/H2O. b Refers to isolated yields after column chromatography, unless otherwise indicated. c Insertion reaction performed at 120 °C for 12 h. d Yield determined by GC–MS analysis. e Significant amounts of homodimer 4 also produced. f Catalyst for cross-coupling was Cl2Pd(PPh3)2.\n",
      "    unique: PMC2548319/PMC2548319.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548318.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548319.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548319.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548319.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Reaction of in Situ Generated Organoindium Reagents with Acyl and Allylic Electrophiles.a\n",
      "  footer_text: a Typical reaction conditions: a DMF solution of the indium reagent (~0.81 M) was added to R1-X (1.0 equiv) and palladium catalyst (5–10 mol %) and heated to 80 °C under argon for 2 h, at which time complete consumption of R1-X was observed by TLC. b A= 5 mol % Pd(PPh3)4; B= 10 mol % Pd(dba)2. c Refers to isolated yields after column chromatography.\n",
      "    unique: PMC2548319/PMC2548319.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548320.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Probability Structure of the Weather Prediction Task\n",
      "  footer_text: Note. On any trial, 1 of 14 possible combinations of four cues appeared with the probability indicated above [P(cue combination appearing)]. Each cue was associated with one of the two weather outcomes (1 = Sunshine vs. 2 = Rain) with a fixed probability. Across all 14 combinations of cues, Cue 1 was associated 75% of the time with Outcome 1 and 25% of the time with Outcome 2; Cue 2 was associated 56% of the time with Outcome 1 and 44% of the time with Outcome 2; Cue 3 was associated 44% of the time with Outcome 1 and 56% of the time with Outcome 2; Cue 4 was associated 25% of the time with Outcome 1 and 75% of the time with Outcome 2. Each combination of cues predicted Outcome 1 (Sunshine) with the probability shown above [P(feedback indicating Outcome 1 is correct)] and predicted Outcome 2 (Rain) with a probability of 1 − P.\n",
      "    unique: PMC2548320/PMC2548320.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Artificial Grammar Task Test Items and Chunk Strengths (CS)\n",
      "  footer_text: \n",
      "    unique: PMC2548320/PMC2548320.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Demographic Characteristics for Schizophrenia (n = 59) and Control (n = 43) Groups\n",
      "  footer_text: \n",
      "    unique: PMC2548320/PMC2548320.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Correlations Within the Schizophrenia Group (n = 59)\n",
      "  footer_text: Notes: WTAR: Wechsler Test of Adult Reading; HVLT: Hopkins Verbal Learning Test; PC: Probabilistic Classification; AG: Artificial Grammar; BPRS: Brief Psychiatric Rating Scale; SANS: Scale for the Assessment of Negative Symptoms. * p < .05.\n",
      "    unique: PMC2548320/PMC2548320.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548320.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548320.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548320.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548320.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548321.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2548321.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Protein Solubility in Various Sample Buffers: Soluble (S), Partially Soluble (S/I), and Insoluble (I)\n",
      "  footer_text: \n",
      "    unique: PMC2548321/PMC2548321.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of 2D Separation Methods\n",
      "  footer_text: \n",
      "    unique: PMC2548321/PMC2548321.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of Separation Methods According to the Number of Total Proteins, Unique Peptides, Total Peptides, Average Percent Sequence Coverage, Average Protein Scores (log E), and False Discovery Rate (FDR)\n",
      "  footer_text: \n",
      "    unique: PMC2548321/PMC2548321.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548323.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2548325.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Loss of individual mitochondrial fission or fusion genes does not cause inappropriate survival of cells that normally die in the anterior pharynx.\n",
      "  footer_text: The number of extra cells in the anterior pharynx of the indicated animals was counted as described in Experimental Procedures. s.e.m., standard error of the mean. Unpaired two tailed t-test, compared with N2 animals: 1 P=0.0017, 2 P=0.0011. Others have P values > 0.05.\n",
      "    unique: PMC2548325/PMC2548325.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548321.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548321.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548325.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548325.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548325.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: drp-1 and fis-2 have minor, independent roles during programmed cell death.\n",
      "  footer_text: The number of extra cells in the anterior pharynx of the indicated animals was counted. All data involving the ced-3(n2438) strains were confirmed by multiple independent blind test analyses. s.e.m., standard error of the mean. Unpaired two tailed t-test, compared with ced-3(n2438) animals: 1 P=0.0012, 2 P=3.4×10−5, 3 P=6.3×10−5, 4 P=0.0018, 5 P=2.1×10−11, 6 P=2.7×10−9; compared with ced-4(n2273) animals: 7 P=0.006,, 8 P=0.043,, 9 P=3.0×10−9; compared with control(RNAi) treatment of the same strain: 10 P=8.8×10−5, 11 P=2.8×10−5, 12 P=1.3×10−5, 13 P=9.0×10−8, 14 P=6.3×10−6, 15 P=2.7×10−7, 16 P=8.7×10−6, 17 P=1.1×10−4. Others have P values > 0.05.\n",
      "    unique: PMC2548325/PMC2548325.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: drp-1 and fis-2 function independently of ced-9 during programmed cell death.\n",
      "  footer_text: The number of extra cells in the anterior pharynx of the indicated animals was counted as described in Table 1. All strains carried the dpy-4(e1166) allele. All data were confirmed by multiple independent blind test analyses. s.e.m., standard error of the mean. Unpaired two-tailed t-test, compared with ced-9(n2812); ced-3(n2438) animals: 1 P=0.0084; 2 P=3.3×10−4, 3 P=2.3×10−6.\n",
      "    unique: PMC2548325/PMC2548325.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: drp-1 and fis-2 function downstream of, or in parallel to, egl-1 and ced-3, and independently of cps-6 and wah-1.\n",
      "  footer_text: Phspegl-1 (smIs82) is an integrated transgene containing the Phspegl-1 constructs. EGL-1 expression was activated by heat-shock treatment and the number of cell corpses in the head region of 1.5-fold stage embryos was scored. Pegl-1acCED-3 (smIs111) is an integrated transgene containing the Pegl-1acCED-3 construct (Kokel et al., 2006). acCED-3 is therefore expressed in cells normally destined to die. The number of cell corpses was counted in the head region of the 4-fold stage embryos. RNAi experiments were carried out as described in Experimental Procedures. All strains were in the ced-1(e1735); ced-4(n1162) background and confirmed by multiple independent blind test analyses. s.e.m., standard error of the mean. Unpaired two tailed t-test, compared with Phspegl-1 animals: 1 P = 8.7×10−5; compared with Pegl-1acCED-3 animals: 2 P=1.2×10−6; compared with control(RNAi) treatment of the same strain: 3 P=0.00035, 4 P=0.0063, 5 P=1.7×10−5, 6 P=1.5×10−6. Others have P values > 0.05.\n",
      "    unique: PMC2548325/PMC2548325.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548326.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of significant results emerging from whole-brain LORETA analyses contrasting learners (n = 16) and non-learners (n = 14) in their response to reward feedback after correctly identifying the stimulus associated with more frequent reward in the probabilistic reward task.\n",
      "  footer_text: The anatomical regions, MNI coordinates, and Brodmann areas of extreme t-values are listed. Positive t-values are indicative of stronger current density for the learners than non-learners, and vice versa for negative t-values. The numbers of voxels exceeding the statistical threshold are also reported (p<0.02; minimum cluster size: 5 voxels). Coordinates in mm (MNI space), origin at anterior commissure; (X) = left (−) to right (+); (Y) = posterior (−) to anterior (+); (Z) = inferior (−) to superior (+).\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548325.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548326.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548329.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548403.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548403.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    unique: PMC2548326/PMC2548326.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548329.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Oxidation degree of oxidized dextran determined from 1H NMR and number average of molecular weight (Mn) estimated by GPC.\n",
      "  footer_text: a Calculated by integrating the anomeric proton peak at δ 4.8 ppm and the tert-butyl proton peak at δ 1.3 ppm. b Gelling time for 2.5 % solutions of Odex and 2.5% CEC solutions in 0.01 M PBS at 37 °C.\n",
      "    unique: PMC2548329/PMC2548329.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2548403.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Characteristics of FLOW Cohort of COPD vs. Referent Subjects\n",
      "  footer_text: \n",
      "    unique: PMC2548403/PMC2548403.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Pulmonary Function and Stage of COPD in the FLOW Cohort\n",
      "  footer_text: Mean (sd)\n",
      "    unique: PMC2548403/PMC2548403.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Impact of COPD on Functional Limitation: Lower Extremity Functioning, Exercise Performance, and Balance\n",
      "  footer_text: All analyses control for age, sex, height, race, educational attainment, smoking The third column show results for all subjects. The fourth column shows results for more severe COPD only (GOLD Stage II or greater) vs. referent group (n=742 COPD cases vs. 302 referent subjects) The fifth column shows results for GOLD Stage II or greater vs. referent group, controlling for baseline covariates plus inclusion of cardiovascular comorbidities (congestive heart failure, coronary artery disease, myocardial infarction, or stroke) and diabetes\n",
      "    unique: PMC2548403/PMC2548403.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Impact of COPD on Functional Limitation: Skeletal Muscle Strength\n",
      "  footer_text: Muscle strength is measured in pounds of force; greatest of 3 repetitions on the dominant side All analyses control for age, sex, race, educational attainment, and smoking history The fourth column shows results for more severe COPD only (GOLD Stage II or greater) vs. referent group (n=742 COPD cases vs. 302 referent subjects); last column also controls for cardiovascular comorbidities and diabetes\n",
      "    unique: PMC2548403/PMC2548403.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Impact of COPD on Self-reported Functional Limitation\n",
      "  footer_text: Multivariate logistic regression analysis controlling for age, sex, race, height, smoking, education † “Unable to do” or “severe limitation” in a battery of basic physical activities (see Methods) †† Health limits “a lot” in performing moderate activities, such as moving a table, pushing a vacuum cleaner, bowling or playing golf The fourth column shows results for more severe COPD only (GOLD Stage II or greater) vs. referent group (n=742 COPD cases vs. 302 referent subjects). The fifth column shows the same group with additional statistical control for cardiovascular comorbidities and diabetes.\n",
      "    unique: PMC2548403/PMC2548403.xml::T5-0"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548403.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548403.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2548403.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551310.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "PMC0025XXXXX/PMC2548421.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551310.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Better ear pure-tone average (BE-PTA) thresholds (at 0.5, 1, and 2 kHz) and age of identification for the children with HL\n",
      "  footer_text: Note. The maximum measurable hearing loss (HL) is 120 dB. M = male; F = female; supp. = supplemental; ID = identification.\n",
      "    unique: PMC2551310/PMC2551310.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Adjusted number of utterances (adjusted inversely proportional to the duration of recording) and syllable statistics\n",
      "  footer_text: Note. NH = normal hearing.\n",
      "    unique: PMC2551310/PMC2551310.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Adjusted number of consonants (adjusted inversely proportional to the duration of recording) and consonant statistics\n",
      "  footer_text: Note. Em dashes indicate “not applicable.”\n",
      "    unique: PMC2551310/PMC2551310.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551310.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551310.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551310.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: The number of measurements and standard deviations of the first two formant frequencies, F1 and F2\n",
      "  footer_text: \n",
      "    unique: PMC2551310/PMC2551310.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551315.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551322.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List of chromatin interacting proteins that are regulated by UV-B in B73 and/or mbd101 and chc101 RNAi lines clustered by SOM\n",
      "  footer_text: \n",
      "    unique: PMC2551322/PMC2551322.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551322.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551322.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551328.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551328.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primers used for qRT-PCR\n",
      "  footer_text: a Primers specifically designed to detected decreased transcript accumulation mediated by RNAi transformation\n",
      "    unique: PMC2551322/PMC2551322.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551328.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Data collection, refinement and geometry statistics\n",
      "  footer_text: a ASU - Asymmetric unit b r.m.s.d. - root mean square deviation c Values in parenthesis for the highest resolution bin d Rsym = Σ|I − <I>| /Σ<I>, where I is the observed intensity, and <I> is the average intensity of multiple observations of symmetry-related reflections. e R = Σ ||Fobs|−|Fcalc||/ Σ |Fobs| where Fobs and Fcalc are the observed and calculated structure factors\n",
      "    unique: PMC2551328/PMC2551328.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Kinetic Parameters of the Glycine Hinge Mutations\n",
      "  footer_text: a No detectable protein produced b All mutants are able to be inhibited by serine to < 95%.\n",
      "    unique: PMC2551328/PMC2551328.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Kinetic Parameters of the Serine Binding Residue Mutations\n",
      "  footer_text: a Ki refers to substrate inhibition. b All mutants are able to be inhibited by serine to < 95%.\n",
      "    unique: PMC2551328/PMC2551328.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551328.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551557.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551557.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2551557.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Results from Screen\n",
      "  footer_text: a Number of gynogenetic diploid sib cohorts from F1 females with defective gene expression b Number of rag-1 negative sib cohorts c Number of αe1 negative sib cohorts\n",
      "    unique: PMC2551557/PMC2551557.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Phenotypic characteristics of rag-1 deficient mutants\n",
      "  footer_text: mc: microcephaly mo: microphthalmia\n",
      "    unique: PMC2551557/PMC2551557.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551558.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551618.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551738.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551742.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551745.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Parameter estimates from the application of mixture model methods to HIV data.\n",
      "  footer_text: * Mixture Probablity πi=eΣParameter Estimate1+eΣParameter Estimate ** Centered at Log10(70000)\n",
      "    unique: PMC2551745/PMC2551745.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551751.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Serum corticosterone measurements in wild type (WT) and stress-sensitive (KO) mice showing continued stress response throughout the chronic stress paradigm (N = 6-7). Stressed mice (CVS) of each genotype were compared to unstressed littermates (CTRL). Samples taken prior to onset (Basal) and at completion (Stressed) of a 15 minute restraint stress. Values are expressed as mean ± SEM.\n",
      "  footer_text: \n",
      "    unique: PMC2551751/PMC2551751.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Active and passive cell characteristics in whole cell recordings of cells in ventromedial dorsal raphe (vmDR). Recordings were taken from serotonin (5-HT+) and non 5-HT (5-HT−) cells in wild type (WT) and stress-sensitive (KO) mice. Abbreviations are as follows: Vm, resting membrane potential; R, resistance; AHP, afterhyperpolarization; Amp, amplitude; AP, action potential. Values are expressed as mean ± SEM.\n",
      "  footer_text: * significant difference across 5-HT+and 5-HT− cells, p ≤ 0.05. ‡ significant difference across genotypes, p ≤ 0.05.\n",
      "    unique: PMC2551751/PMC2551751.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551756.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2551759.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: NMR data of calanquinone A (1)a\n",
      "  footer_text: a Measured in CDCl3 (300 and 500 MHz, δ in ppm, J in Hz).\n",
      "    unique: PMC2551759/PMC2551759.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551745.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551751.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551751.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551759.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cytotoxicity of calanquinone A (1) isolated from C. arisanensis\n",
      "  footer_text: a positive control\n",
      "    unique: PMC2551759/PMC2551759.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Cytotoxicity of synthesized 1 and related intermediates\n",
      "  footer_text: NA: no activity up to 20 µg/mL\n",
      "    unique: PMC2551759/PMC2551759.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551761.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Computational predictions of toxin stabilitya\n",
      "  footer_text: a Protein instability data for Pet and for the A chains of ricin, cholera toxin (CTA1), and pertussis toxin (PTS1) were obtained from the ProtParam function of ExPASy-SWISS-PROT. An instability index value greater than 40 is indicative of protein instability. The predicted A chain stabilities were consistent with experimental data from three studies which collectively indicated a relative stability order of ricin A > CTA1 > PTS1 (10-12). It should be noted that an interaction with negatively charged phospholipids is thought to further destabilize ricin A chain before its export from the ER to the cytosol (34).\n",
      "    unique: PMC2551761/PMC2551761.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2551765.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Primers used for library construction\n",
      "  footer_text: The sequences underlined enable amplification or randomization of CDR repertoires. a Sub-groups of human antibody heavy chain genes. b All members in the groups.\n",
      "    unique: PMC2551765/PMC2551765.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551759.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551759.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551761.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551765.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primer pairings used for amplification of gene segments\n",
      "  footer_text: a These products partially cover FRs. b All members in the group of VH germlines c Sub-group within the group of VH germlines.\n",
      "    unique: PMC2551765/PMC2551765.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: CDR diversity of unique phage-displayed VHs selected against B5R after the third round of panning\n",
      "  footer_text: \n",
      "    unique: PMC2551765/PMC2551765.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2552397.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Glide, Liaison and Scoring Function Training and Test Set Predictions for GGPPS Inhibition.\n",
      "  footer_text: a structures of all inhibitors shown in Figure 1 b bold values indicated compounds not included in training set.\n",
      "    unique: PMC2552397/PMC2552397.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551765.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2551765.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552397.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Receptor-Guided Alignment CoMSIA training and Test Set Predictions for GGPPS Inhibition.\n",
      "  footer_text: N = number of components; n = Number of training set compounds a structures of all inhibitors shown in Figure 1 b bold values indicated compounds not included in training set.\n",
      "    unique: PMC2552397/PMC2552397.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552397.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552397.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552397.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Data Collection and Refinement Statistics for GGPPS-Bisphosphonate Crystals.\n",
      "  footer_text: a Values in the parenthesis are the highest resolution shells.\n",
      "    unique: PMC2552397/PMC2552397.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Data collection and refinement statistics for GGPPS-bisphosphonate crystals.\n",
      "  footer_text: a Values in the parenthesis are the highest resolution shells.\n",
      "    unique: PMC2552397/PMC2552397.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2552408.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2552409.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2552411.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: CHARACTERISTICS AND ODDS RATIOS FOR ESCC CASES AND CONTROLS\n",
      "  footer_text: 1 Crude Odds Ratio. 95%CI = 95% confidence interval. ESCC = esophageal squamous cell carcinoma. NA = not applicable. OR = odds ratio. UGI = upper gastrointestinal.\n",
      "    unique: PMC2552411/PMC2552411.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552411.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552411.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552411.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552975.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552975.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: ODDS RATIOS FOR SIGNIFICANT SNPs IN ESCC CASE-CONTROL STUDY (CASES n=300, CONTROLS n=300)\n",
      "  footer_text: 1 SNP coding a nonsynonymous amino acid substitution Lysine 270 to Methionine. 2 SNPs identified from pilot study. 3 SNPs in linkage disequilibrium. 4 Permuted p value GLM (additive model). 5 Permuted Fishers p value (2×3). 6 P value (HWE) derived from 2×3 distributions. 2×3 = 2×3 frequency distribution table. ESCC = esophageal squamous cell carcinoma. GLM = general linear model (additive model). HWE = Hardy-Weinburg equilibrium. SNP = single nucleotide polymorphism. Homozygous common allele in controls is the reference genotype.\n",
      "    unique: PMC2552411/PMC2552411.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: ODDS RATIOS FOR NUMBER OF PROTECTIVE ALLELES IN ESCC CASE-CONTROL STUDY (CASES n=300, CONTROLS n=300)\n",
      "  footer_text: 1 Odds ratio GLM additive mode. 2 P value GLM additive model. SNPs and their corresponding alleles used for calculating combined effects of multiple protective alleles; rs1515366 (allele A), EPHB1; rs4264508 (allele T), PGLYRP2; rs52911 (allele C), PIK3C3; rs16853475 (allele T), SLC9A9. The single most protective SNP from each of our four targeted genes were evaluated to determine the combined effects of multiple protective alleles.Multiple protective alleles were examined in three ways: (i) as a continuous variable; (ii) categorized into four groups approximating quartiles as: group 1 (0-1 protective allele), group 2 (2 protective alleles), group 3 (3 protective alleles), and group 4 (4-6 protective alleles; there were no individuals with 7-8 protective alleles) and analyzed using indicator variables; and (iii) combined into a single ordinal variable to test for trend.\n",
      "    unique: PMC2552411/PMC2552411.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2552975.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Hypothetical Self-ratings for Traits in General and in Five Specific Roles\n",
      "  footer_text: Note. For brevity, only 10 of 40 Big Five traits in the Trait-Role Questionnaire are shown.\n",
      "    unique: PMC2552975/PMC2552975.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean General-Specific Role Correlations and Associated Confidence Intervals for Each Role and Culture\n",
      "  footer_text: Note. C.I. = confidence interval.\n",
      "    unique: PMC2552975/PMC2552975.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Means and Standard Deviations for (In)Consistency (SD) Indices in Six Cultures\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552975.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552975.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  footer_text: Note. Larger values indicate greater cross-role variability (i.e., inconsistency). For each type of data, means with the same superscripts are not significantly different in between-culture comparisons (Scheffé post-hoc t-tests). Partial η2 values are for cultural main effects in analyses of variance (ANOVAs).\n",
      "    unique: PMC2552975/PMC2552975.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Correlations and Standardized Beta Coefficients in Predicting SD Variability Index in Six Cultures\n",
      "  footer_text: ** p<.01; * p<.05.\n",
      "    unique: PMC2552975/PMC2552975.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Prediction of Adjustment Measures\n",
      "  footer_text: Note. Where two values of β or R are separated by a slash (/), the first value is for the original data and the second value is for the ipsatized data. ** p<.01; * p<.05.\n",
      "    unique: PMC2552975/PMC2552975.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2552975.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553012.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553002.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553012.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Location estimates of current sources and current sinks. Actual locations refer to independent measurements of current injection electrode positions. The FWHM is twice the mean distance from a given extrema of the divergence to the half-maximum contour. FWHM = Full-Width at Half-Maximum\n",
      "  footer_text: \n",
      "    unique: PMC2553012/PMC2553012.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553033.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553044.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Examples of pathogens that bind to sialic acids on human cell surfaces\n",
      "  footer_text: \n",
      "    unique: PMC2553044/PMC2553044.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Examples of pathogens that express sialic acids on their surfaces\n",
      "  footer_text: \n",
      "    unique: PMC2553044/PMC2553044.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Sialylation changes associated with malignant transformation and tumor progression\n",
      "  footer_text: Abbreviations: N-CAM, neural cell adhesion molecule; Tn, Siaα2-3GalNAcα-Ser/Thr.\n",
      "    unique: PMC2553044/PMC2553044.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553045.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Numbers of slices and the age range of animals from which they were taken for electrophysiological measurements.\n",
      "  footer_text: \n",
      "    unique: PMC2553045/PMC2553045.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of intrinsic electrical properties in wild type and jerker mice reveal no differences.\n",
      "  footer_text: Values of resting potential (Vrest), input resistance (Rin), and membrane time constant (τ) are given as means ± SE. The Rin and τ were measured near the resting potential.\n",
      "    unique: PMC2553045/PMC2553045.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553044.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553044.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553044.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553045.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553045.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553221.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553239.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patterns of use of tobacco among medical students in Damascus according to year of study at university (1st versus 5th)\n",
      "  footer_text: * p<0.05 for comparisons across year of study (1st versus 5th)\n",
      "    unique: PMC2553239/PMC2553239.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlates of smoking among medical students in Damascus, Syria\n",
      "  footer_text: * increment of 1 year in age\n",
      "    unique: PMC2553239/PMC2553239.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multivariate logistic regression analysis for predictors of tobacco use among medical students in Damascus, Syria\n",
      "  footer_text: * because of multi-colinearity, year at university and age were entered in the models separately.\n",
      "    unique: PMC2553239/PMC2553239.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Beliefs & attitudes towards tobacco use among medical students in Damascus, Syria\n",
      "  footer_text: * p < 0.05 for comparisons across smoking status for each tobacco use method\n",
      "    unique: PMC2553239/PMC2553239.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553239.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553239.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553239.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553239.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553272.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553250.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553270.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553272.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Biodistribution of 111In-labeled DO3A-BP in normal Balb/c mice (n = 4). Data are presented as %ID/g ± standard deviation. B/B: Bone/Blood; B/M: Bone/Muscle.\n",
      "  footer_text: \n",
      "    unique: PMC2553272/PMC2553272.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553276.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List of the locations in the visual field that were represented by our injected SC sites across all experiments. The sites are sorted by brain hemisphere as well as by their retinotopic location and polar coordinates, as estimated from the maps of visually guided saccade latencies after muscimol injection (see Materials and Methods). The asterisk in the last column indicates that the site was also used later (after all muscimol experiments were completed) for a saline control injection\n",
      "  footer_text: \n",
      "    unique: PMC2553276/PMC2553276.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553351.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Distribution of MMP-1 (collagenase-1), MMP-8 (collagenase-2), and MMP-13 (collagenase-3)\n",
      "  footer_text: Data reported as the percentage of total collagenase protein in GCF.\n",
      "    unique: PMC2553351/PMC2553351.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553352.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553276.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553351.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553352.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of hamsters fed on regular chow vs. high fructose diet\n",
      "  footer_text: Hamsters were fed regular chow or high fructose diet for 8 weeks, followed by the determination of body weight, fasting blood glucose levels, fasting plasma levels of insulin, FFA, triglyceride, cholesterol, HDL-cholesterol (HDL-c). Non-HDL-cholesterol (non-HDL-c) levels were calculated by subtracting HDL-c from total cholesterol levels in plasma. Glucose tolerance was performed after 7 weeks of fructose feeding for the determination of AUC of blood glucose profiles in response to glucose challenge. Hamsters were sacrificed at the end of study and liver tissues were used for the determination of hepatic lipid content, defined as mg of triglyceride per gram of wet liver tissue. * P<0.05 vs. control by ANOVA.\n",
      "    unique: PMC2553352/PMC2553352.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Hepatic proteins with >2-fold alterations in response to high fructose feeding\n",
      "  footer_text: Proteomic profiling of livers of hamsters fed on high fructose (n=6) and regular chow (n=6) were performed. Determination of changes in protein expression levels was performed on a total of 12 2-D gels from individual hamster livers in control and fructose_groups, using DeCyder software version 5 and calculated from the volume ratios of the normalized fluorescent signals. Protein spots with significant differences of greater than 2 fold (P<0.05) between high fructose- and regular chow-fed hamsters were identified. All identified proteins match the apparent molecular mass and pI values, based on the 2-D gels.\n",
      "    unique: PMC2553352/PMC2553352.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553360.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553361.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of common affects of FASN inhibition on PC-3, MCF-7 and SKOV-3 cancer cells, expressed as % of control\n",
      "  footer_text: * P<0.05 (two-tailed unpaired Student’s t test) a 3 or more repeats b 2 repeats.\n",
      "    unique: PMC2553361/PMC2553361.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553352.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553361.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553434.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553434.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553395.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553425.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553434.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: FMRI image acquisition parameters for 3T and 4T datasets\n",
      "  footer_text: \n",
      "    unique: PMC2553434/PMC2553434.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Similar regions of coherent positive and negative BOLD signal changes in BT and 4T datasets determined by 2dTCA\n",
      "  footer_text: Region number corresponds to numbers indicated in Figures 2A and 3A. MNI coordinates indicate center of one comparable activated duster in this region.\n",
      "    unique: PMC2553434/PMC2553434.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553438.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Ranking experiment results showing the average number of top-10 ranked markers within a given distance of the true disease-predisposing locus.\n",
      "  footer_text: Based on 100 simulated data sets each with 1000 case and 1000 control individuals. Columns denoted χ2 correspond to single-marker χ2-test results, while columns denoted “P” (respectively, “U”) correspond to Blossoc results using m = 5 for phased (respectively, unphased) data. GRR=“Genetic Relative Risk”. WR=“Wildtype Risk”.\n",
      "    unique: PMC2553438/PMC2553438.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Ranking experiment results showing the fraction of data sets with at least one top-10 marker within a given distance of the true disease-predisposing locus.\n",
      "  footer_text: Based on 100 simulated data sets each with 1000 case and 1000 control individuals. Columns denoted χ2 correspond to single-marker χ2-test results, while columns denoted “P” (respectively, “U”) correspond to Blossoc results using m = 5 for phased (respectively, unphased) data. GRR=“Genetic Relative Risk”. WR=“Wildtype Risk”.\n",
      "    unique: PMC2553438/PMC2553438.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Percentage of data sets with the highest scoring marker within distance ε (in Kbp) from the disease-predisposing SNP, which is untyped.\n",
      "  footer_text: Based on 100 simulated data sets for each setting, with 5% wild-type risk and ρ = 40. Columns denoted χ2 correspond to single-marker χ2-test results, while columns denoted “P” (respectively, “U”) correspond to Blossoc results using m = 5 for phased (respectively, unphased) data.\n",
      "    unique: PMC2553438/PMC2553438.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Percentage of data sets with the highest scoring marker within distance ε (in Kbp) from the disease-predisposing SNP, for 1000 case and 1000 control individuals, and GRR= 2.0.\n",
      "  footer_text: Based on simulated data sets from Minichiello and Durbin (2006); each data set was for a 1 Mbp region with ρ = 440 and 300 tagging SNPs. Columns denoted χ2 correspond to single-marker χ2-test results, while columns denoted “P” (respectively, “U”) correspond to results for phased (respectively, unphased) data. We used m = 5 in running Blossoc. The results for Margarita correspond to Figure 2E of Minichiello and Durbin (2006).\n",
      "    unique: PMC2553438/PMC2553438.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553438.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553520.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Top 10 highest scoring SNPs in the Parkinson disease data set (Fung et al., 2006) analyzed using Blossoc.\n",
      "  footer_text: Locations correspond to that of NCBI Build 36.1.\n",
      "    unique: PMC2553438/PMC2553438.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Running times of Blossoc. See the main text for the computer spec.\n",
      "  footer_text: \n",
      "    unique: PMC2553438/PMC2553438.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553514.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553520.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Average (mean) life span (years), maximum life span (years), average age at menopausal onset (years), and proportion of life spent in a nonreproductive state as a function of currently known average or maximum life spans in a variety of primate species.\n",
      "  footer_text: a [128] b [129] c [35] d [130] e [130] f [62, 64, 65] g [131] h [131] i [77] j [132] k [55] l [133] m [134] n [81] o [42] p [135] q [40]\n",
      "    unique: PMC2553520/PMC2553520.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Criterion-based operational definitions and dependent variables used to establish the existence of menopause.\n",
      "  footer_text: \n",
      "    unique: PMC2553520/PMC2553520.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553520.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553624.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553624.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553624.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Correlations, Ranges, Means, and Standard Deviations of Stress, Distress and Demographic Variables\n",
      "  footer_text: a Job Status: 0 = not employed; 1 = part time; 2 = full time. b Education: 1 = <11 years; 2 = high school; 3 = trade/tech school; 4 = college, not graduated; 5 = associate’s degree; 6 = college, graduated; 7 = some grad school, did not finish; 8 = Master’s/Terminal degree. c Income: 1 = less than 15K; 2 = 15K to 25K; 3 = 26K to 50K; 4 = 51K to 75K; 5 = 76K to 100K; 6 = > 100K. d Children: 0 = No; 1 = Yes. e Num. Children: coded as continuous variable. f Marital status: 0 = single, divorced, separated, or widowed; 1 = Married, living as if married, committed, not living together. g Neighborhood: 1 = all share ethnic background; 2 = most share background; 3 = half share background; 4 = very few share background; 5 = none share background. h Student status: 0 = Not a student; 1 = part time student; 2 = full time student. ** p < .01. *** p < .001.\n",
      "    unique: PMC2553624/PMC2553624.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Path Coefficients, Standard Errors, and Significance Levels of Models\n",
      "  footer_text: Note. RRS = Race-related stress; RRSE = race-related stress exposure; RRSA = race-related stress appraisal; GRS = gender-related stress; GRSE = gender-related stress exposure; GRSA = gender-related stress appraisal; GS = generic stress; GSE = generic stress exposure; GSA = generic stress appraisal. All p values are for the ratio of unstandardized estimates/SE. ** p < .01. *** p < .001.\n",
      "    unique: PMC2553624/PMC2553624.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553626.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: ALS Genes and Loci1\n",
      "  footer_text: \n",
      "    unique: PMC2553626/PMC2553626.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: ALS-related Motor Neuron Disorders with upper and lower motor neuron involvement1\n",
      "  footer_text: * No bulbar involvement. Long, slow progression, distal wasting with pyramidal signs and sensory loss, previously called axonal CMT with pyramidal signs. ** This is a disorder that appears to be proximal SMA IV (Finkel type) with some UMN findings.\n",
      "    unique: PMC2553626/PMC2553626.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553682.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Descriptive Statistics by Treatment Condition for Study Outcome Measures\n",
      "  footer_text: Note. CB = Cognitive-behavioral therapy; SD = standard deviation.\n",
      "    unique: PMC2553682/PMC2553682.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Repeated Measures ANCOVA Pairwise Contrasts\n",
      "  footer_text: Note. CB = Cognitive-Behavioral Therapy; Coeff = coefficient; SE = standard error; d = Cohen’s d; CI = confidence interval; p = p-value.\n",
      "    unique: PMC2553682/PMC2553682.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553626.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553626.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553682.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553682.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553692.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553704.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553687.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553690.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553692.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Functional differences between two populations of V1 neurons, difficulty-suppressed and difficulty-enhanced The width of the interspike interval distribution was measured at half of its amplitude and the peak as the time with the maximum value in the distribution (measured with a bin width = 1 ms). The contrast sensitivity was measured from the contrast response function as the contrast that generated half-maximum response or C50. Significance was measured with a Mann-Whitney test\n",
      "  footer_text: \n",
      "    unique: PMC2553692/PMC2553692.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553693.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553704.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of the Study Participants.*\n",
      "  footer_text: * Plus–minus values are means ±SD. BDNF denotes brain-derived neurotrophic factor, and BMI body-mass index. † Nominal P values are shown. ‡ Race or ethnic group was self-reported. § Data are for 33 parents of persons with heterozygous BDNF deletions and 27 parents of persons without BDNF deletions. ¶ There were differences in the sample size for BMI z scores because some participants had incomplete growth records and others were of young age. Multiple observations were obtained from each patient, but only one data point (if available) for each age-category analysis was used. ∥ Data are for 18 patients with heterozygous BDNF deletions and 9 patients without BDNF deletions. ** Data are for 18 patients with heterozygous BDNF deletions and 8 patients without BDNF deletions. †† Data are for 12 patients with heterozygous BDNF deletions and 8 patients without BDNF deletions. ‡‡ Data are for 5 patients with heterozygous BDNF deletions and 5 patients without BDNF deletions. §§ Data are for 19 patients with heterozygous BDNF deletions and 14 patients without BDNF deletions. Covariates for the adjusted current BMI z score were the BMI values of the parents.\n",
      "    unique: PMC2553704/PMC2553704.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553708.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics of the study population by quartile (Q) of serum selenium concentrations (n = 5452)1\n",
      "  footer_text: 1 Adjusted for age, sex, and race-ethnicity. 2 x̄ ± SE (all such values). 3 SE in parentheses (all such values).\n",
      "    unique: PMC2553708/PMC2553708.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Difference (95% CI) in serum lipid and lipoprotein concentrations by quartile (Q) of serum selenium concentrations (n = 5452)1\n",
      "  footer_text: 1 Apo B, apolipoprotein B; apo A-I, apolipoprotein A-I. 2 x̄ within each quartile (all such values). 3 Adjusted for age (continuous), sex (male, female), and race-ethnicity (non-Hispanic white, non-Hispanic black, Mexican American, other). 4 Further adjusted for education (≥12 y of school completed, <12 y of school completed), family income (≥$20 000,<$20 000), postmenopausal status for women (yes, no), cigarette smoking (current, former, never), serum cotinine (continuous), alcohol consumption (current, former, never), physical activity (0, 1-2, ≥3 times/wk), and body mass index (continuous). 5 Further adjusted for cholesterol-lowering medication use (yes, no), vitamin-mineral supplement use (yes, no), glomerular filtration rate (continuous), C-reactive protein (≥1.0 mg/dL, 0.3-0.9 mg/dL, <0.3 mg/dL), diabetes mellitus (yes, no), thyroid-stimulating hormone (continuous), and thyroxine (continuous). 6 Analysis based on subsample of 2589 participants because apo B and apoA-I were only measured during 1988-1991.\n",
      "    unique: PMC2553708/PMC2553708.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Ratio (95% CI) of serum triacylglycerol concentrations by quartile (Q) of serum selenium concentrations (n = 5452)\n",
      "  footer_text: 1 Geometric x within each quartile (all such values). 2 Adjusted for age (continuous), sex (male, female), and race-ethnicity (non-Hispanic white, non-Hispanic black, Mexican American, other). 3 Further adjusted for education (≥12 y of school completed, <12 y of school completed), family income (≥$20 000, <$20 000), postmenopausal status for women (yes, no), cigarette smoking (current, former, never), serum cotinine (continuous), alcohol consumption (current, former, never), physical activity (0, 1-2, ≥3 times/wk), and body mass index (continuous). 4 Further adjusted for cholesterol-lowering medication use (yes, no), vitamin-mineral supplement use (yes, no), glomerular filtration rate (continuous), C-reactive protein (≥1.0 mg/dL, 0.3-0.9 mg/dL, <0.3 mg/dL), diabetes mellitus (yes, no), thyroid-stimulating hormone (continuous), and thyroxine (continuous).\n",
      "    unique: PMC2553708/PMC2553708.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Difference (95% CI) in the ratio of LDL cholesterol to HDL cholesterol (LDL cholesterol:HDL cholesterol) and in the ratio of apolipoprotein B to apolipoprotein A-I (apo B:apo A-I) by quartile (Q) of serum selenium concentrations (n = 5452)\n",
      "  footer_text: 1 x̄ within each quartile (all such values). 2 Adjusted for age (continuous), sex (male, female), and race-ethnicity (non-Hispanic white, non-Hispanic black, Mexican American, other). 3 Further adjusted for education (≥12 y of school completed, <12 y of school completed), family income (≥$20 000, <$20 000), postmenopausal status for women (yes, no), cigarette smoking (current, former, never), serum cotinine (continuous), alcohol consumption (current, former, never), physical activity (0, 1-2, ≥3 times/wk), and body mass index (continuous). 4 Further adjusted for cholesterol-lowering medication use (yes, no), vitamin-mineral supplement use (yes, no), glomerular filtration rate (continuous), C-reactive protein (≥1.0 mg/dL, 0.3-0.9 mg/dL, <0.3 mg/dL), diabetes mellitus (yes, no), thyroid-stimulating hormone (continuous), and thyroxine (continuous). 5 Analysis based on subsample of 2589 participants because apo B and apo A-I were only measured during 1988-1991.\n",
      "    unique: PMC2553708/PMC2553708.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553708.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553708.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553708.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553708.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2553710.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Area under the lives-saved-weighted ROC curve with spores falling in the Class A shape in an urban location and with k = ncases in equation (1) for each of the 7 methods\n",
      "  footer_text: a Mixed effects Poisson models with signal lengths of exactly 1, 3, and 7 days b Space-and-time scan approaches with maximum signal lengths of 1, 3, and 7 days\n",
      "    unique: PMC2553710/PMC2553710.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Area under the lives-saved-weighted ROC curve with spores falling in the Class A shape in an urban location and with the weight function as in equation (1) with k = the median cases saved for each of the 7 methods\n",
      "  footer_text: a Mixed effects Poisson models with signal lengths of exactly 1, 3, and 7 days b Space-and-time scan approaches with maximum signal lengths of 1, 3, and 7 days\n",
      "    unique: PMC2553710/PMC2553710.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553713.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553710.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553710.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553713.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2553713.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Results for parameter robustness checks. For each new parameter value in turn, 100 psychometric functions were generated. The numbers in the columns headed `Threshold' and `Slope' refer to the median of the values from those 100 runs. Values are given in units of % contrast. For details on the meaning of the individual parameters, and the abbreviations used, see the Methods section. For quick comparison, the equivalent results using the standard parameter values as given in the Methods section are included at the top of the table.\n",
      "  footer_text: \n",
      "    unique: PMC2553713/PMC2553713.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of experimental results by subject. Values are the mean of the fitted parameters from the four visual field locations tested, given in units of % contrast, and the estimated false negative rate FN.\n",
      "  footer_text: \n",
      "    unique: PMC2553713/PMC2553713.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2553753.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2553894.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2554670.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Torque and Muscle Activation (Mean ± SD) Produced With Protocols A and B\n",
      "  footer_text: Abbreviations: CSA, cross-sectional area; MVIT, maximum voluntary isometric torque; NMES, neuromuscular electrical stimulation; TTI, torque time integral. * Significant difference between protocols (P<.05).\n",
      "    unique: PMC2554670/PMC2554670.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Torque (Mean ± SD) Produced at the Start and the End of Protocol B*\n",
      "  footer_text: * The decline in “starting” and “ending” torque between the first and tenth contractions was 31% and 50%, respectively. The tenth contraction showed a decline in torque by 24% from the beginning to the end of the contraction.\n",
      "    unique: PMC2554670/PMC2554670.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2554670.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2554670.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555965.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555965.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555965.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555965.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2555965.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The history of developments observed in PBC diagnosis are well representative of the different discovery paces with the early major discoveries in PBC diagnosis and the slow progression of newer techniques for AMA testing. Putative developments for the expected upcoming phases are suggested.\n",
      "  footer_text: \n",
      "    unique: PMC2555965/PMC2555965.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: PBC natural history and treatment remain an unfinished business. The rapid pace of earlier discoveries well represent phase 1 and was followed by slower developments with combination treatments and the prognostic significance of ANA. Future developments in the third phase are hypothesized.\n",
      "  footer_text: \n",
      "    unique: PMC2555965/PMC2555965.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: PBC genetic and environmental bases recognize three bases with an earlier rapid development of descriptive data on familial PBC and twin concordance subsequently followed by a slower phase with unconclusive association studies while recapitulation of the role of sex chromosome, xenobiotics, and bacteria is awaited. Epigenetic studies may constitute the key to a awaited third phase.\n",
      "  footer_text: \n",
      "    unique: PMC2555965/PMC2555965.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: PBC immunobiology was characterized by earlier discoveries leading to the definition of AMA autoantigens and autoreactive T cells while a second phase is well represented by the study of T regulatory cells, apoptosis in cholangiocytes, and innate immune changes. The integration of these lines of evidence is the major awaited development in the third phase.\n",
      "  footer_text: \n",
      "    unique: PMC2555965/PMC2555965.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2555970.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Strains Used in This Study\n",
      "  footer_text: \n",
      "    unique: PMC2555970/PMC2555970.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555970.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2555970.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556036.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556036.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556039.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primers Used in This Study\n",
      "  footer_text: \n",
      "    unique: PMC2555970/PMC2555970.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2555987.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556036.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Unrelated Donor (URD) Searches for African-Americans with Hematologic Malignancies\n",
      "  footer_text: \n",
      "    unique: PMC2556036/PMC2556036.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Preliminary Cord Blood Unit Searches for African-Americans with Hematologic Malignancies\n",
      "  footer_text: \n",
      "    unique: PMC2556036/PMC2556036.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556039.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characterization of splenic erythroblast maturation after Phenylhydrazine induced anemia and Erythropoietin treatment\n",
      "  footer_text: All data are Mean ± SEM. * indicates P<0.05 by post hoc testing for comparison of treatment groups with respective baseline by one way ANOVA. The experimental n for baseline, post EPO and post PHZ, are: (WT)-11, 4 and 5, (Nix-/-)- 8, 4 and 5.\n",
      "    unique: PMC2556039/PMC2556039.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556073.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Analysis of SMN RNA expression The relative ratio of newborn to adult SMN RNA was determined using Multi Gauge (Fugi Film). Each sample was subtracted from background and normalized against β-actin. Error bars reflect the standard deviation. Four independent amplifications of porcine adult and newborn tissues were used in these studies\n",
      "  footer_text: \n",
      "    unique: PMC2556073/PMC2556073.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556079.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556081.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Simulation results with a constant hazard: n is the number of individuals, θ the frailty variance, m the number of individuals with at least one event, T̄ the mean follow-up time, N(T)‒ and median(N(T)) are the mean and median numbers of events for subjects with at least one event; E(U) and V(U) are the sample mean and variance of U for subjects with at least one event, with true values 0 and 1/12=0.083, respectively. In the last two columns the power of the test (5) and (6) at 5% level is given.\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Simulation results with a decreasing hazard: n is the number of individuals, θ the frailty variance, m the number of individuals with at least one event, T̄ the mean follow-up time, N(T)‒ and median(N(T)) are the mean and median numbers of events for subjects with at least one event; E(U) and V(U) are the sample mean and variance of Ui, respectively, for subjects with at least one event. In the last two columns the power of the test (5) and (6) at 5% level is given.\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Simulation results with an increasing hazard: n is the number of individuals, θ the frailty variance, m the number of individuals with at least one event, T̄ the mean follow-up time, N(T)‒ and median(N(T)) are the mean and median numbers of events for subjects with at least one event; E(U) and V(U) are the sample mean and variance of U, respectively for subjects with at least one event. In the last two columns the power of the test (5) and (6) at 5% level is given.\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Results from the TTT method applied to the Swiss clinical study. The number of patients included in the study (n), the number of patients with at least one additional admission during follow-up (m), the average follow-up time in years, the mean and median number of events for the m patients with at least one event during follow-up. The sample mean and variance of Ui and the p-value for testing (5) and (6).\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Results from the TTT method applied to Danish registry data 1994-1999. The number of patients included in the study (n), the number of patients with at least one additional admission during follow-up (m), the mean follow-up time in years (T̄), the mean and median numbers of events for the m patients with at least one event during follow-up. The sample mean and variance of Ui and the p-values for testing (5) and (6).\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Results from a gap-time Γ frailty model stratified by sex, fitted to Danish registry data 1994-1999, further corrected for: calendar time, age at first admission, and polarity. Only patients with at least one additional admission after first discharge are included.\n",
      "  footer_text: \n",
      "    unique: PMC2556081/PMC2556081.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556153.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556171.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of Peptide Modifications Found in 65 High Abundant Glycoproteins in Plasmaa\n",
      "  footer_text: a FETUA_BOVIN was spiked in all samples. Bold-marked amino acids indicate the modification site. When a high number of modified peptides were found, only the total number of modified peptides is listed.\n",
      "    unique: PMC2556171/PMC2556171.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of Modifications Found in SiHa (Cervical Cancer) Cell Line and Comparison of Selected Modifications Found in the SiHa Cell Line Using an Unrestricted and Targeted Searcha\n",
      "  footer_text: a Bold-marked amino acids indicate the modification site. b Spectra with XCorr values greater than 1.9, 2.2, and 3.8 for 1+, 2+, and 3+, respectively, were considered to be correct hits. All peptides found in the unrestricted search could be verified by the targeted search. See text for details.\n",
      "    unique: PMC2556171/PMC2556171.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556081.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556171.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556171.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556171.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Validation of the Unrestricted Search Algorithm for Peptide Pairs in Homologous Proteins in the SiHa Cell Linea\n",
      "  footer_text: a Bold-marked amino acids indicate the modification site. b Modification was detected (Y) or not detected (N) in unrestricted search. See text for more details.\n",
      "    unique: PMC2556171/PMC2556171.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556173.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556205.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556206.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556218.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Expression of daf-1 in Interneurons Is Sufficient to Rescue the Dauer Formation, Feeding Rate, and Fat Storage Phenotypes of daf-1(m40) Mutants daf-1::gfp was expressed under various promoters in daf-1(m40) mutants. Larval dauer formation, adult feeding rate and fat content were measured in each of the transgenic lines. Indicated results were consistent for at least three independent transgenic lines. Promoters used to drive daf-1::gfp are grouped based on anatomical expressions.\n",
      "  footer_text: a % animals that entered dauer when placed on food plates as L1s at 25°C. Phenotypic example shown in Figure S1 b Average pumping rate of well-fed, young gravid adults. Standard deviations are shown. c Young, well-fed wild-type animals retained ~10 eggs on average. This phenotype is denoted as (+). Mutants such as daf-1(m40) retained excess eggs (~24), denoted as (+++). Intermediate egg retention is denoted as (++). Phenotypic examples are shown in Figure S1. d Expression of daf-1(+) by these promoters reduced the excess fat of daf-1(m40) to wild-type levels. Phenotypic examples shown in Figure 1B. e Statistically significant (p <0.001, determined by ANOVA with Bonferroni post-test) increases in feeding rate relative to daf-1(m40).\n",
      "    unique: PMC2556218/PMC2556218.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556220.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556223.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Hit identification through primary HTS and counterscreening\n",
      "  footer_text: a structures of compounds lacking activity in counter-screening are not shown b values represent averages of two experiments ± range; highest concentration assessed 37.5 μM c values represent averages of two experiments ± range; highest concentration assessed 150 μM; ND: not determined d values represent averages of four experiments ± SD; highest concentration assessed 37.5 μM e CC50 not determined (ND) when EC50 >5 μM f CC50 not determined (ND) due to insufficient amount of compound available\n",
      "    unique: PMC2556223/PMC2556223.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556218.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Assessment of accuracy of primary hit identification\n",
      "  footer_text: a unmarked compounds represent the group-defining confirmed hit; (*) bona-fide hits (primary HTS score >1.9, passed visual inspection); (#) excluded (primary HTS score >1.9, excluded at visual inspection); (&) negatives (primary HTS score < 1.9); shading highlights compounds found to be “false negatives” b based on chemical structures c values represent averages of two experiments ± range; highest concentration assessed 37.5 μM d values represent averages of four experiments ± SD; highest concentration assessed 37.5 μM\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-0\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-1\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-2\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-3\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-4\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-2'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-3'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-4'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-5'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-6'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T2-7'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    unique: PMC2556223/PMC2556223.xml::T2-6\n",
      "    unique: PMC2556223/PMC2556223.xml::T2-7\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Assessment of cross-resistance with known MV inhibitors\n",
      "  footer_text: a shading highlights compound identified in pre-screening as fusion inhibitor candidates b MV-Alaska variant adapted to grow in the presence of MV RdRp inhibitor AS-136A (Sun et al. 2007; White et al. 2007); values represent averages of two experiments ± range; highest concentration assessed 37.5 μM c MV-Ibd, naturally resistant to MV fusion inhibitor AS-48 (Plemper et al. 2004) values represent averages of two experiments ± range; highest concentration assessed 37.5 μM; only compounds identified as fusion inhibitor candidates were tested against MV-Ibd (ND: not determined) d for comparison, EC50 values against MV-Alaska (table 1) are shown\n",
      "    unique: PMC2556223/PMC2556223.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Three distinct patterns of target specificity\n",
      "  footer_text: a values represent averages of two experiments ± range; highest concentration assessed 37.5 μM b values represent averages of four experiments ± SD; highest concentration assessed 37.5 μM\n",
      "    unique: PMC2556223/PMC2556223.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556234.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556223.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556234.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556243.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Effect of tyrosine phosphatases inhibitors on PTP1B activity from membrane fraction of HCE cells HCE cells were incubated with the inhibitor at the concentrations indicated for 30 min (bpV(phen), PAO, or SOV) or 60 min (CinnGEL 2Me). One µg of protein from the membrane fraction was used to determine the PTP1B activity as explained in Methods. The percentage of inhibition was calculated with respect to control without inhibitors. Values correspond to average of 12 samples ± SD of three different experiments.\n",
      "  footer_text: \n",
      "    unique: PMC2556234/PMC2556234.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556243.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Multivariate ANOVA results for relations between marijuana use frequency and social anxiety, marijuana-related problems, marijuana effect expectancies.\n",
      "  footer_text: Note. Marijuana effect expectancies were measured using the Marijuana Effect Expectancy Questionnaire (MEEQ) (Aarons et al., 2001), social anxiety with the Social Interaction Anxiety Scale (Mattick & Clarke, 1998), and marijuana problems with the Marijuana Problems Scale (Stephens et al., 2000). Marijuana use frequency grouping: nonusers (those who never used marijuana; n = 105), infrequent users (less than weekly marijuana use; n = 122), and frequent users (weekly or more marijuana use; n = 109). a Means not significantly different from one another (p < .01). * p < .05 ** p < .01 *** p < .001.\n",
      "    unique: PMC2556243/PMC2556243.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Multivariate ANOVA results for marijuana effect expectancies among current users (n = 150).\n",
      "  footer_text: Note. Marijuana effect expectancies were measured using the Marijuana Effect Expectancy Questionnaire (Aarons et al., 2001), social anxiety with the Social Interaction Anxiety Scale (Mattick & Clarke, 1998). * p < .05 ** p < .01 *** p < .001.\n",
      "    unique: PMC2556243/PMC2556243.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Logistic Regressions for Marijuana Use versus Nonuse.\n",
      "  footer_text: Note. Marijuana effect expectancies were measured using the Marijuana Effect Expectancy Questionnaire (Aarons et al., 2001), social anxiety with the Social Interaction Anxiety Scale (Mattick & Clarke, 1998). * p < .05.\n",
      "    unique: PMC2556243/PMC2556243.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556245.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556243.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556243.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556245.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556246.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table\n",
      "  caption_text: Mean weight, energy density, energy content, and macronutrient content of the liquid and solid meal replacement products consumed by older adults in a study of postprandial appetite ratings\n",
      "  footer_text: a Ingredients (from highest to lowest content in the liquid meal replacement product; quantities not provided on the manufacturer's label): water, sugar, corn syrup, maltodextrin (corn), calcium caseinate, high-oleic safflower oil, canola oil, soy protein isolate, whey protein concentrate, and corn oil (also 32 vitamins and mineral compounds). b Ingredients (from highest to lowest content in the solid meal replacement product; quantities not provided on the manufacturer's label): yogurt coating, high-fructose corn syrup, soy protein isolate, raisins, honey, crisp rice, toasted rolled oats, fructose, maltodextrin, fructooligosaccharides, whey protein concentrate, glycerone, guar gum, calcium caseinate, high-oleic sunflower oil, high-oleic safflower oil (also 26 vitamins and mineral compounds). c Data expressed as mean±standard error of the mean.\n",
      "    unique: PMC2556245/PMC2556245.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556246.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Transferred EAE signs in wild type, ICAM-1 × C3−/− and C3−/− mice.\n",
      "  footer_text: A Cumulative Disease Index is the mean of the sum of daily clinical scores observed between days 7 and 30. B Disease onset is defined as the first day of two consecutive days with a clinical score of two or more. C Disease incidence is defined as the percent of mice that displayed any clinical signs of disease. D EAE was significantly different between all three groups of mice as determined by one-way ANOVA (p<0.0001) and by Newman-Keuls multiple comparisons test: WT vs. ICAM-1 × C3−/− (p<0.01), WT vs. C3−/− (p<0.001) and ICAM-1 × C3−/− vs. C3−/− (p=0.05).\n",
      "    unique: PMC2556246/PMC2556246.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cytokine production by CD4 and CD8 T cells derived from spinal cords of wild type, ICAM-1 × C3−/− and C3−/− mice during EAE.\n",
      "  footer_text: A Mean percent cytokine positive cells derived from mice at day 15 post-immunization. Cells were pooled from at least 2 mice per group from 2 independent experiments.\n",
      "    unique: PMC2556246/PMC2556246.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556260.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556278.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556279.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556280.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556284.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556246.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556288.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556288.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2556288.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Experimentally measured local and global movements associated with the transition from the ground state to the activated state of rhodopsin.\n",
      "  footer_text: \n",
      "    unique: PMC2556288/PMC2556288.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Waiting times for the LUMI – META II transition for wild type rhodopsin and mutants. Waiting times tf and tb are for the completion of the forward and backward transitions, respectively. Means and standard errors were estimated from five simulations. As a rough approximation for the equilibrium population shift, we calculate the ratio of waiting times, K=tb/tf, which provides an estimate of the relative stability of the META II state over the LUMI state.\n",
      "  footer_text: \n",
      "    unique: PMC2556288/PMC2556288.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556289.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means (and standard deviations) of predictor and cognitive functioning variables by cocaine exposure and gender\n",
      "  footer_text: Note. BDI = Beck Depression Inventory; FCS = Family Chaos Scale; HOME = Home Observation for Measurement of the Environment; SEI = Social Environment Inventory; TANF = Temporary Assistance for Needy Families. Note. Neonatal medical problems, alcohol exposure, cigarette exposure, and marijuana exposure variables were each log transformed to correct for skew in all subsequent analyses. Note. Means with different superscripts indicate significant between group differences at p <.05. # p ≤ .10. * p ≤ .05. ** p ≤ .01. *** p ≤ .001.\n",
      "    unique: PMC2556289/PMC2556289.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556289.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556289.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556289.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Pearson correlations among predictor and cognitive functioning variables (below diagonal = boys; above diagonal = girls)\n",
      "  footer_text: Note. Cocaine exposure was entered as a dichotomous variable (0 = no exposure; 1 = exposed). # p ≤ .10. * p ≤ .05. ** p ≤ .01. *** p ≤ .001.\n",
      "    unique: PMC2556289/PMC2556289.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Coefficients (and standard error; followed by 95% confidence intervals) from mixed model analyses predicting Stanford-Binet IV IQ and subscales\n",
      "  footer_text: Note. Cocaine exposure was entered as a dichotomous variable (0 = no exposure; 1 = exposed). Age was coded such that 4-years is the referent to which the other ages are compared. # p ≤ .10. * p ≤ .05. ** p ≤ .01. *** p ≤ .001. (2-tailed).\n",
      "    unique: PMC2556289/PMC2556289.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556292.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556299.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Descriptive and Zero-Order Relations among Theoretically Relevant Variables\n",
      "  footer_text: † Note: p < .05 * p < .01 ** p < .001. NA = Negative Affect subscale of PANAS; AS = ASI total; Cigs = Cigarettes smoked per day; DI = DIS total; PA = Panic Attack status during first hyperventilation challenge (yes/no); DT = Distress Tolerance during second hyperventilation challenge (i.e. time in minutes of voluntary hyperventilation)\n",
      "    unique: PMC2556299/PMC2556299.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556306.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Nucleotide percentage identity cut-off values defining genotypes for 11 rotavirus gene segments.\n",
      "  footer_text: \n",
      "    unique: PMC2556306/PMC2556306.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Application of the proposed classification and nomenclature to the structural and non-structural protein encoding genes for known human, bovine, porcine, simian, avian, equine, lapine, murine and ovine rotavirus strains. Green, red, and orange were used for Wa-like, DS-1-like and AU-like gene segments, respectively, while yellow, blue, and purple were used for the avian PO-13-like rotavirus gene segments, some typical porcine VP4, VP7, and VP6 genotypes, and the SA-11-like gene segments, respectively. It should be noted that the genotypes of more than 60% of the isolates in this table are partial, requiring additional work.\n",
      "  footer_text: \n",
      "    unique: PMC2556306/PMC2556306.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556299.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556306.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556306.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Current members of the RCWG\n",
      "  footer_text: \n",
      "    unique: PMC2556306/PMC2556306.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556357.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556377.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556461.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556506.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Town/Gown Education/Communication Activity Summary for the DOVE study.\n",
      "  footer_text: \n",
      "    unique: PMC2556506/PMC2556506.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Number and Percent of Home Visitor Personal Experience with Abuse.\n",
      "  footer_text: * NOTE: Questionnaires with missing data on abuse items were excluded from some analysis\n",
      "    unique: PMC2556506/PMC2556506.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556306.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556506.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556506.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556506.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Descriptive Information from the Home Visitors Completing the Survey.\n",
      "  footer_text: \n",
      "    unique: PMC2556506/PMC2556506.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556508.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: K-ras mutations in tumors from L2G7 and 5G8 treated mice.\n",
      "  footer_text: \n",
      "    unique: PMC2556508/PMC2556508.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556512.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Rotational and Translational Accuracy of Electromagnetic and Optoelectric Systems. Data are reported as (actual value − calculated value)\n",
      "  footer_text: \n",
      "    unique: PMC2556512/PMC2556512.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Bland-Altman test values for precision measurements. All measured differences were less than the measured accuracy of the systems.\n",
      "  footer_text: \n",
      "    unique: PMC2556512/PMC2556512.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556516.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556508.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556512.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556512.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556516.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Primary Antibody/Toxin Information\n",
      "  footer_text: 1 mab: monoclonal 2 Sigma-Aldrich Co., St. Louis, Mo. 3 Molecular Probes-Invitrogen, Carlsbad, Ca. 4 Phosphosolutions, Aurora, Co. 5 Developmental Studies Hybridoma Bank. The 3A10 monoclonal antibody developed by Drs. T. Jessell and J. Dodd was obtained from the Developmental Studies Hybridoma Bank developed under the auspices of the NICHD and maintained by The University of Iowa, Department of Biological Sciences, Iowa City, IA 52242.\n",
      "    unique: PMC2556516/PMC2556516.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556547.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Relapse Between Maximum Expansion (T3) and End of Treatment (T5)\n",
      "  footer_text: The variation of N is explained by the fact that some patients had extraction of teeth so the number of measurements is reduce for those teeth.\n",
      "    unique: PMC2556547/PMC2556547.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556550.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of UV-Melting Temperatures of Duplexes Containing 5 or MedC\n",
      "  footer_text: \n",
      "    unique: PMC2556550/PMC2556550.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556547.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556550.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556550.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556550.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556550.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556562.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effect of Flanking Sequence Mismatches on ICL Formation from 5\n",
      "  footer_text: \n",
      "    unique: PMC2556550/PMC2556550.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: UV-Melting Temperatures of Triplexes\n",
      "  footer_text: \n",
      "    unique: PMC2556550/PMC2556550.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Interstrand Cross-Link (ISC) Yields from 1 and 5 in Triple Helical DNA\n",
      "  footer_text: a % Yields are the average of at least two independent experiments. b NFT = no further treatment after reaction with NaIO4. c 90 °C for 20 min at pH 7.2. d 1.0 M Piperidine, 90 °C, 20 min.\n",
      "    unique: PMC2556550/PMC2556550.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556562.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Responses to Tone\n",
      "  footer_text: Values are means (± se), computed over 50 dB range above tone threshold. * = statistically significant difference (p = 0.05 or less); ns = not significant\n",
      "    unique: PMC2556562/PMC2556562.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Responses to Noise\n",
      "  footer_text: Values are means (± se), computed over 50 dB range above tone threshold. * = statistically significant difference (p = 0.05 or less); ns = not significant\n",
      "    unique: PMC2556562/PMC2556562.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556566.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556562.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556566.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556596.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556596.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Effect of inhibitors on HO activity.\n",
      "  footer_text: Mean HO activity (nmol CO/mg protein/h ± SEM, n = 5-31/condition) in cultures subjected to medium exchange (sham-wash, SW) only or treated for 30 minutes with indicated concentrations of TBB, DMAT, DRB, GF109203X, LY294002 and Wortmannin.\n",
      "    unique: PMC2556566/PMC2556566.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556571.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556596.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of 150 Adolescents Aged 11 to 21 yr Evaluated for a Sexually Transmitted Infection (STI) at an Academic Ambulatory Care Facility\n",
      "  footer_text: * Sexually transmitted infection. † Female greater than male; P value <0.01.\n",
      "    unique: PMC2556596/PMC2556596.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Predictors of HIV Testing Among 150 Adolescents Aged 11 to 21 yr Evaluated For an STI At an Academic Ambulatory Care Facility, Baltimore, July 2003 to June 2004\n",
      "  footer_text: * Adjusted for age, gender and insurance status. † P <0.05.\n",
      "    unique: PMC2556596/PMC2556596.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556598.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Generation of ALS2−/− mice\n",
      "  footer_text: NA = No information available.\n",
      "    unique: PMC2556598/PMC2556598.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556598.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556598.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556598.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556855.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Behavioral phenotypes of ALS2−/− mice\n",
      "  footer_text: NS = Not statistically significant; NA = data not available. 1 May relate to higher anxiety-induced reduction of exploratory activity.\n",
      "    unique: PMC2556598/PMC2556598.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Pathological abnormalities of ALS2−/− mice\n",
      "  footer_text: CSMN = Corticospinal motor neurons; CST = corticospinal track; SMN = spinal motor neuron; VR = ventral root. 1 Small caliber axons. 2 Calbindin-positive.\n",
      "    unique: PMC2556598/PMC2556598.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556850.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556855.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: The frequencies, bandwidths and isotope shifts of extracted resonance Raman bands of substrate-bound P450cam and its isotopicaly substituted analogues.\n",
      "  footer_text: a Normal mode calculations for NiOEP, including all seven atoms for each ethyl group (C2h or S4 molecular symmetry),14,37 indicate the possibility for several modes between 350 and 450 cm−1, designated as δ(CβC1C2) or γ2 (A1u), whose d4 and d16 shifts are reasonably consistent with those observed here.\n",
      "    unique: PMC2556855/PMC2556855.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556856.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556858.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556862.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556863.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556865.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of MCP participants in in-depth interviews (N=18)\n",
      "  footer_text: \n",
      "    unique: PMC2556865/PMC2556865.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556878.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556881.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Dimensions of Channels and Other Microfeatures in the Original Design, the Brass Mold, and the Embossed Plastic Microfluidic Devices\n",
      "  footer_text: \n",
      "    unique: PMC2556881/PMC2556881.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556882.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556865.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556881.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Kappa Coefficients for Strategy Categories\n",
      "  footer_text: Note. PF = problem-focused strategy; EF = emotion-focused strategy. * Strategies included in this category were not included in analyses of strategy type. 1 Based on 20% of the data. 2 Based on an additional 6 cases.\n",
      "    unique: PMC2556882/PMC2556882.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Strategy Measures by Interpersonal and Instrumental Problem Domain: Mean, SD, Range\n",
      "  footer_text: Note. PF= problem-focused strategy; EF = emotion-focused strategy. * Indicates strategy type excluded in analyses. 1 The mean proportion score for this strategy was .07 for older interacting pairs thus, it was included in the analysis of strategy type. 2 The mean proportion scores for younger nominal pairs was .07 and for interacting pairs was .09, thus, this strategy was included in the analysis of strategy type.\n",
      "    unique: PMC2556882/PMC2556882.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Age Differences in Strategy Measures for Interpersonal Problems: Mean, SD, and F Statistics\n",
      "  footer_text: Note. PF= problem-focused strategy; EF = emotion-focused strategy. * p < .05 ** p < .01 *** p < .001.\n",
      "    unique: PMC2556882/PMC2556882.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556882.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Age Differences in Strategy Measures for Instrumental Problems: Mean, SD, and F Statistics\n",
      "  footer_text: Note. PF=problem-focused strategy; EF=emotion-focused strategy. * p < .05 ** p < .01 *** p < .001.\n",
      "    unique: PMC2556882/PMC2556882.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Percent Selecting Strategy as Most Effective by Age Group: Interpersonal Problems\n",
      "  footer_text: Note. PF=problem-focused strategy; EF=emotion-focused strategy. Number of cases = 224 (38 interacting pairs each age group; 74 individuals each age group). * indicates z ≥ 1.96, p ≤ .05\n",
      "    unique: PMC2556882/PMC2556882.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Percent Selecting Strategy as Most Effective by Age Group: Instrumental Problems\n",
      "  footer_text: Note. PF= problem-focused strategy; EF = emotion-focused strategy. Number of cases = 224 (38 interacting pairs each age group; 74 individuals each age group). * indicates z ≥ 1.96, p ≤ .05\n",
      "    unique: PMC2556882/PMC2556882.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556883.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean hit rates (with standard errors) as a function of source, frequency, and age.\n",
      "  footer_text: \n",
      "    unique: PMC2556883/PMC2556883.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556883.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556883.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556883.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556883.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556886.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean non-zero imagination-frequency and performance-frequency responses (with standard errors) for imagined-only actions as a function of age and frequency of imagination.\n",
      "  footer_text: \n",
      "    unique: PMC2556883/PMC2556883.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean proportion of judged-imagined items that were also judged performed as a function of frequency and age (standard errors in parentheses)\n",
      "  footer_text: \n",
      "    unique: PMC2556883/PMC2556883.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Mean non-zero performance-frequency and imagination-frequency responses (with standard errors) for performed-only actions as a function of age and frequency of performance.\n",
      "  footer_text: \n",
      "    unique: PMC2556883/PMC2556883.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556884.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556886.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Robustness of the model-free spectral density\n",
      "  footer_text: a From Eq. (13) b Diffusion in a three identical well cosine potential with 1 kcal/mol barriers. c Torsion refers to the number of rotatable bonds before the methyl. d Stochastic jump model. Ratios in the model column refer to the expected equilibrium g-, g+, t rotamer population ratios calculated from the jump probabilities. e Motional asymmetry is defined through Eq. (17), as the difference between Oaxis2 calculated without and with assumption of azimuthal symmetry.\n",
      "    unique: PMC2556886/PMC2556886.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Average values of structural correlates and their correlation with Oaxis2a\n",
      "  footer_text: a Average values with standard deviation for several simple structural properties for the methyl groups of CaM. Difference represents the average pair-wise difference between the two properties. R2 represents the correlation of Oaxis2 at a site with the structural property at that site.\n",
      "    unique: PMC2556886/PMC2556886.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556886.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556888.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556888.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556888.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556888.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2556888.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means and Standard Deviations for Participant Characteristics\n",
      "  footer_text: ** Matrix Reasoning, Ravens Advanced Progressive Matrices, and Operation Span scores were significantly different between the older and younger adults (p < .01), and were negatively correlated with age between ages 60-95. There were no other significant age differences.\n",
      "    unique: PMC2556888/PMC2556888.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: ANOVA Results from the Complete Release-from-PI Operation Span Task\n",
      "  footer_text: \n",
      "    unique: PMC2556888/PMC2556888.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: \n",
      "  footer_text: * p < .05; ** p < .01\n",
      "    unique: PMC2556888/PMC2556888.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: \n",
      "  caption_text: Stimuli from the Release-from-PI Operation Span\n",
      "  footer_text: \n",
      "    unique: PMC2556888/PMC2556888.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556890.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Xenograft tumor treatment groups.\n",
      "  footer_text: * harvested at early time points (tumor size 200–400 mm3) ** harvested at later time points (tumor size ~1000 mm3)\n",
      "    unique: PMC2556890/PMC2556890.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Gene sets associated with endocrine therapy resistance in MCF7 xenograft mouse models.\n",
      "  footer_text: \n",
      "    unique: PMC2556890/PMC2556890.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556891.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556893.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556897.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556890.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556890.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Zero-Order Correlations between Selected Independent Variables\n",
      "  footer_text: Note. AEMaxPosAr, AEMaxSed = AEMax: Positive Arousing factor, Sedating factor, respectively; AEQGloPos, AEQSoc = AEQ: Global Positive Changes scale, Social scale, respectively; TotalDr ln = Total number of standard drinks over the past 30 days, log-transformed; DrinkOcc ln = Number of drinking occasions over the past 30 days, log-transformed; AveQuant ln = Average number of standard drinks per drinking occasion, log-transformed; HighestDr ln = Highest number of standard drinks per drinking occasion, log-transformed. * p < .05 ** p < .01 *** p < .001\n",
      "    unique: PMC2556897/PMC2556897.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Zero-Order Correlations between Selected Independent Variables and TF2/SF2 (P300) scores\n",
      "  footer_text: Note. P300*Neg, P300*Pos = TF2/SF2 factor scores in response to Alcohol/Negative-Sedating and Alcohol/Positive-Arousing conditions, respectively; AEMaxPosAr, AEMaxSed = AEMax: Positive Arousing factor, Sedating factor, respectively; AEQGloPos, AEQSoc = AEQ: Global Positive Changes scale, Social scale, respectively; TotalDr ln = Total number of standard drinks over the past 30 days, log-transformed; DrinkOcc ln = Number of drinking occasions over the past 30 days, log-transformed; AveQuant ln = Average number of standard drinks per drinking occasion, log-transformed; HighestDr ln = Highest number of standard drinks per drinking occasion, log-transformed. * p < .05 ** p < .01 *** p < .001\n",
      "    unique: PMC2556897/PMC2556897.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2556897/PMC2556897.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556897.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556897.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556897.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2556898.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556899.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Thermodynamic parameters for the interactions of the compounds in Figure 1 binding to nonalternating and alternating sequences at different lengths.\n",
      "  footer_text: a Kmean values are from SPR experiments as described in Materials and methods and ΔG is obtained from the equation: ΔG = −RTlnKmean. b ΔH binding values are from ITC as described in Materials and methods Experimental error limits for SPR K values and ITC ΔH values are both approximately ±10%.\n",
      "    unique: PMC2556899/PMC2556899.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556901.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List of 37 deficits used in the analyses\n",
      "  footer_text: \n",
      "    unique: PMC2556901/PMC2556901.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556956.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556977.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics and body composition of the study participants1\n",
      "  footer_text: 1 All values are x̄ ± SEM. Significance of differences between groups was evaluated by using the Student’s t test for independent samples. IHTG, intrahepatic triglyceride; NAFLD, nonalcoholic fatty liver disease. 2 Data represent values obtained from 9 of the 10 subjects who had NAFLD, because technical problems precluded obtaining images of abdominal fat in 1 subject. 3 Significantly different from overweight subjects with normal IHTG content, P < 0.01. 4 Significantly different from overweight subjects with normal IHTG content, P < 0.001.\n",
      "    unique: PMC2556977/PMC2556977.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Basic metabolic characteristics of the study participants1\n",
      "  footer_text: 1 All values are x̄ ± 3 SEM. Significance of differences between groups was evaluated by using the Student’s t test for independent samples. IHTG, intrahepatic triglyceride; NAFLD, nonalcoholic fatty liver disease. 2–4 Significantly different from overweight subjects with normal IHTG content: 2 P < 0.05, 3 P < 0.001, 4 P < 0.01.\n",
      "    unique: PMC2556977/PMC2556977.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556899.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556901.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556977.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556977.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2556978.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556981.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556983.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556985.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means and Standard Errors for the Study Variables\n",
      "  footer_text: \n",
      "    unique: PMC2556985/PMC2556985.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Percentage Perceiving Incident at Least Once in Past Year\n",
      "  footer_text: \n",
      "    unique: PMC2556985/PMC2556985.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Percentage Perceiving Incident at Least Once in Past Year\n",
      "  footer_text: \n",
      "    unique: PMC2556985/PMC2556985.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Perceived Discrimination as a Predictor of Depressive Symptoms\n",
      "  footer_text: * p < .02, ** p < .01.\n",
      "    unique: PMC2556985/PMC2556985.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556985.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556986.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  label_text: Table 5\n",
      "  caption_text: Perceived Discrimination as a Predictor for Self-esteem\n",
      "  footer_text: * p < .02, ** p < .01.\n",
      "    unique: PMC2556985/PMC2556985.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Discrimination as a Predictor for Life Satisfaction\n",
      "  footer_text: * p < .02, ** p < .01.\n",
      "    unique: PMC2556985/PMC2556985.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556986.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Behavioral risks during Senior Year in High School, and Fall and Spring Semesters of Freshman Year\n",
      "  footer_text: Notes. All variables were assessed over the 3 months prior to the assessment. Drinking variables and number of sexual partners were measured continuously. All other behavioral risks were measured on the following seven-point scale (0 = 0; 1 = 1; 2 = 2; 3 = 3−5; 4 = 6−10; 5 = 11−20; 6 = 21 or more). Means (SD) for multiple sexual partners represent values for students reporting two or more partners in the past 3 months. Means (SD) for all other behavioral risks represent values for those who report any participation in the behavior.\n",
      "    unique: PMC2556986/PMC2556986.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Main Effects of HLM Model for Alcohol Use\n",
      "  footer_text: Note. Coefficients for all residence groups represent contrasts relative to campus dorms, and coefficients for area of residence represent contrasts relative to individuals from suburban high schools. Family structure was coded 0 for two-parent households and 1 for households without two resident parents such that positive coefficients reflect increased risk in households without two resident parents.\n",
      "    unique: PMC2556986/PMC2556986.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2556987.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2556989.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Lyn kinase activity in various tumor cells\n",
      "  footer_text: NOTE: Cell lysates (200 μg) were assayed for Lyn kinase activity by in vitro kinase assay.\n",
      "    unique: PMC2556989/PMC2556989.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556986.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556989.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556990.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2556990.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557022.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2556990.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Description of the physical components of each context.\n",
      "  footer_text: * Walls were modified using paper to cover the clear Plexiglas. + Floors were modified using an insert that was placed over the existing stainless steel bar floor.\n",
      "    unique: PMC2556990/PMC2556990.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Assignment of training, extinction, and renewal test contexts for subjects in Experiment 2a and 2b.\n",
      "  footer_text: \n",
      "    unique: PMC2556990/PMC2556990.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2557022.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Risk factors, demographics, and clinical characteristics, 2000-2006, Nueces County, Texas.\n",
      "  footer_text: * Current Smoker 4 missing values Dicharge GCS, 3 missing values Insurance Status, 1 missing value Discharge Rankin, 2 missing values Length of Stay, 2 missing Time to Presentation, 5 missing values MA=Mexican American, NHW=non-Hispanic white, GCS=Glasgow Coma Scale, IQR=Interquartile range\n",
      "    unique: PMC2557022/PMC2557022.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Seven year cumulative incidence and cumulative incidence risk ratios by age, ethnicity and gender, 2000-2006, Nueces County, Texas.\n",
      "  footer_text: * Test of Homogeneity Across Age Strata, p-value = 0.91 ** Test of Homogeneity Across Age Strata, p=0.08 NHW=non-Hispanic white, MA=Mexican American\n",
      "    unique: PMC2557022/PMC2557022.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Age and gender-specific seven-year cumulative incidence and risk ratios, 2000-2006, Nueces County, Texas.\n",
      "  footer_text: \n",
      "    unique: PMC2557022/PMC2557022.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Poisson regression model, adjusted for age, ethnicity and gender, of seven-year cumulative incidence, 2000-2006, Nueces County, Texas.\n",
      "  footer_text: NHW=non-Hispanic white, MA=Mexican American\n",
      "    unique: PMC2557022/PMC2557022.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557022.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557022.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557022.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557024.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557024.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2557024.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Training methods, indications for use, suggestions for competency assessment, and examples from research program\n",
      "  footer_text: \n",
      "    unique: PMC2557024/PMC2557024.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Evidence-based research team training plan (basic research topics )\n",
      "  footer_text: \n",
      "    unique: PMC2557024/PMC2557024.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Evidence based research team training plan (supplemental topics)\n",
      "  footer_text: Keys: SD-self directed, I-independent, GD-group didactic, GI-group interactive, SB-simulation based, SV-supervisory iteration, TT-two tiered sign off GT-graded test, SM-seminar, CX-cross training, RA-research assistant\n",
      "    unique: PMC2557024/PMC2557024.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2557442.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: en-driven mCD8::GFP expression in antennal lobe glomeruli. Fluorescent glomeruli and their corresponding olfactory receptors are indicated in bold.\n",
      "  footer_text: \n",
      "    unique: PMC2557442/PMC2557442.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557024.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557442.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557553.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2557553.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Values of the parameters used in the creation of synthetic luteinizing hormone concentration-time series.\n",
      "  footer_text: \n",
      "    unique: PMC2557553/PMC2557553.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean (± SEM) number of peaks known to be present in the synthetic luteinizing hormone concentration time series and the number recovered by analysis with AutoDecon and Cluster. For each group studied (i.e., early follicular, late follicular, mid-luteal and post-menopausal), values identified with different superscripts differ significantly (p < 0.001) by analysis of variance with Duncan’s Multiple Range test.\n",
      "  footer_text: \n",
      "    unique: PMC2557553/PMC2557553.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Performance characteristics of AutoDecon and Cluster based upon synthetic data which mimics normal human subjects at different phases of the menstrual cycle.\n",
      "  footer_text: \n",
      "    unique: PMC2557553/PMC2557553.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Mean (± SEM) LH burst mass, burst half-duration and luteinizing hormone (LH) half-life characterizing the synthetic data series and recovered by application of AutoDecon to the synthetic data series. In each case, and for all groups (i.e. early follicular, late follicular, mid-luteal and post-menopausal) AutoDecon returned values that are statistically indistinguishable from the values known to characterize the synthetic data series (p> 0.05).\n",
      "  footer_text: \n",
      "    unique: PMC2557553/PMC2557553.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2558674.xml\n",
      "  ----------------------------------------"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557553.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557553.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2557553.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2558674.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2558674.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Brain areas exhibiting activation during FIXATION (> PICTURE)\n",
      "  footer_text: Height threshold at P < 0.05, FDR-corrected. For each activation foci, laterality (left, right), peak Montreal Neurologic Institute (MNI) atlas coordinates, cluster size (number of contiguous voxels), and Z-score are provided. IPL, Inferior Parietal Lobule; MTG , Middle Temporal Gyrus; STG, Superior Temporal Gyrus; PCC, Posterior Cingulate Cortex; ACC, Anterior Cingulate Cortex; L, left; R, right.\n",
      "    unique: PMC2558674/PMC2558674.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Brain areas in which extent of activation during FIXATION is modulated by prior affective state\n",
      "  footer_text: Brain regions that demonstrate an effect of prior emotional (Positive, Negative, Neutral) epoch on activation during FIXATION. Height threshold at P < 0.05, FDR-corrected. IPL, Inferior Parietal Lobule; FIX, FIXATION.\n",
      "    unique: PMC2558674/PMC2558674.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559463.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and clinical characteristics (mean ± SD) of the sample\n",
      "  footer_text: NS = non-significant AA = African-American; AS = Asian; CC = Caucasian; and HS = Hispanic BDI = Beck Depression Inventory HDRS = Hamilton Depression Rating Scale CGAS = Children’s Global Assessment Scale\n",
      "    unique: PMC2559463/PMC2559463.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Predictors of net peak salivary cortisol concentrations in response to the TSST in depressed adolescents and controls (N = 55)\n",
      "  footer_text: * T = Tolerance (1−R2). A Tolerance value of less than 0.20 indicates significant inter-correlation among variables ** Ridge regression was performed to adjust for multicollinearity HDRS = Hamilton Depression Rating Scale Model 1: k = 0, F = 0.24, df = 3,51, NS; Model 2: k = 1.0, F = 1.35, df = 6,48, NS Model 3: k = 1.0, F = 2.57; df = 9,45, p ≤ .05\n",
      "    unique: PMC2559463/PMC2559463.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Significant predictors of net peak salivary cortisol concentrations in response to the TSST in depressed adolescents and controls (N = 55)\n",
      "  footer_text: * T = Tolerance (1−R2). A Tolerance value of less than 0.20 indicates significant inter-correlation among variables ** Ridge regression was performed to adjust for multicollinearity Model 1: k = 0; F = 4.95; df = 2, 52; p ≤ .05 Model 2: k = 0; F = 8.35; df = 4, 50; p ≤ .0001 Model 3: k = 2.0; F = 4.65; df = 5, 49; p ≤ .001\n",
      "    unique: PMC2559463/PMC2559463.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559855.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559463.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559463.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559463.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559855.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant characteristics.\n",
      "  footer_text: Note. There are no statistically significant differences between abstinent and relapsed women.\n",
      "    unique: PMC2559855/PMC2559855.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Questions and responses on survey of postpartum women who had remained abstinent or relapsed to smoking (N = 36).\n",
      "  footer_text: a Total N for this question is 35. One individual failed to answer this question.\n",
      "    unique: PMC2559855/PMC2559855.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559856.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559857.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559858.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Data collection and refinement statistics of the hKAT II complex with α-oxoglutaric acid GOL, glycerol; 2OG, α-oxooglutaric acid; RMS, root mean square.\n",
      "  footer_text: * The values in parentheses are for the highest resolution shell.\n",
      "    unique: PMC2559858/PMC2559858.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559855.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559858.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559858.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Kinetic parameters of hKAT II towards amino acids and α-oxo acids The Km and kcat for oxo acids were derived by using various concentrations of individual oxo acids in the presence of 10 mM kynurenine; for amino acids, the results were derived by using various concentrations of individual amino acids in the presence of 30 mM glyoxylate. 3-HK, 3-HK, 3-hydroxy-kynurenine; α-KMB, α-oxo-γ-methiol-butyric acid. Results are means±S.E.M. (n=2).\n",
      "  footer_text: \n",
      "    unique: PMC2559858/PMC2559858.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559860.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "tw(T1): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 1\n",
      "  caption_text: Gating asymmetry ratio measurements\n",
      "  footer_text: (Top) Effect of the chloride electrochemical gradient at zero proton electrochemical gradient (pHext 8.5, pHint 7.5, V = −58 mV, [Cl−]int = 120 mM). (Bottom) Effect of the proton electrochemical gradient (at symmetrical chloride concentration [Cl−]int = [Cl−]ext = 120 mM, and pHint 7.50). * One of the four patches with ΔGH = 2.7 kJ mol−1 and one of the eight patches with ΔGH = 4.6 kJ mol−1 was measured at pHint 7.00 and pHext 9.00. a ΔGCl was calculated according to equation 3. b ΔGH was calculated according to equation 2.\n",
      "    unique: PMC2559860/PMC2559860.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559871.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559912.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559947.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: SE analysis of molecular mass as a function of pea PBGS protein concentration, assuming a single ideal species\n",
      "  footer_text: 1 0.1 M BTP-HCl, pH 8.5, 0.1 mM DTT, 10 mM MgCl2 collected at 20°C. 2 The term, “global” refers to global fits across all speeds for a given concentration. The last row of this column represents an global fit to all three speeds and three concentrations simultaneously. 3 All molecular masses are reported in Daltons. The theoretical monomer molecular mass of pea PBGS is 36,693 Da. 4 The row labeled “all conc’ns” contains global fits across all concentrations for a given speed.\n",
      "    unique: PMC2559947/PMC2559947.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559860.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559947.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559947.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559947.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Model analysis of pea PBGS from SE experiments with and without MgCl2, using an unweighted reduced χ2 as a measure of goodness of fit1\n",
      "  footer_text: 1 0.1 M BTP-HCl, pH 8.5, 0.1 mM DTT, ±10 mM MgCl2, 20°C. Model analysis is based on global fitting to the three protein concentrations and three speeds, as reported in Table I. 2 When two values are presented, the first value assumes a non-equilibrating mixture model and the second value assumes an equilibrium model. In each model, the species are held fixed as appropriate multiples of the monomeric molecular mass. 3 Fitting of the data with inclusion of mass balance constraints reveals 65% octamer and 35% hexamer by weight. 4 Fitting of the data with inclusion of mass balance constraints reveals 25% octamer and 75% hexamer by weight.\n",
      "    unique: PMC2559947/PMC2559947.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Molecular mass analysis of pea PBGS from SE experiments testing the effect of magnesium depletion on pea PBGS quaternary structure assemblies, assuming a single ideal species\n",
      "  footer_text: 1 Samples of pea PBGS were dialyzed against 0.1 M BTP-HCl, pH 8.5, 0.1 mM DTT at 4°C prior to SE analysis and run at 20°C. 2 The term, “global” refers to global fits across all speeds for a given concentration. The last row of this column represents an global fit to all three speeds and three concentrations simultaneously. 3 All molecular masses are reported Daltons. 4 The row labeled “all conc’ns” contains global fits across all concentrations for a given speed.\n",
      "    unique: PMC2559947/PMC2559947.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559948.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559949.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The effect of nicotine administration on exploratory behavior during a 10 minute pre-exposure to the training context. Data presented as mean ± SEM.\n",
      "  footer_text: \n",
      "    unique: PMC2559949/PMC2559949.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559950.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "tw(T1): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 1\n",
      "  caption_text: The S. pombe SWI/SNF and RSC chromatin-remodeling complexes\n",
      "  footer_text: Mass-spectrometry results showing the identification SWI/SNF and RSC components for each TAP-tagged subunit preparation. a Protein size in amino acids. b Sequence coverage (percentage of amino acids) for the respective protein. c Protein name indicates the TAP fusion used for purification.\n",
      "    unique: PMC2559950/PMC2559950.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559949.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559950.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559950.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559951.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559951.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "tw(T2): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 2\n",
      "  caption_text: Composition of S. pombe SWI/SNF and RSC complexes compared to those of S. cerevisiae and human\n",
      "  footer_text: a Subunits of the respective complexes are listed in columns, with orthologous proteins grouped horizontally. b Paralogs in the same complex are grouped in the same row of the column.\n",
      "    unique: PMC2559950/PMC2559950.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559951.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant Characteristics\n",
      "  footer_text: Note. WAIS-R = Wechsler Adult Intelligence Scale-Revised vocabulary subtest (Wechsler, 1981), MMSE = Mini Mental State Exam (Folstein et al., 1975), GDS = Geriatric Depression Scale (Yesavage et al., 1983). a Expressed as WAIS-R vocabulary subtest raw scores.\n",
      "    unique: PMC2559951/PMC2559951.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Treatment Descriptions and Mean (± SD) Hormone Levels Pre/Post-Treatment\n",
      "  footer_text: Note. Hormone levels are from the day of cognitive testing before treatment, and again on the day of cognitive testing after approximately 6 weeks of treatment. Tx = treatment group. T-gel = Transdermal Testosterone Gel; a Estradiol measured in pg/mL. b Testosterone measured in ng/dL. c Free Testosterone measured in pmoles/L. * Hormone levels significantly differ from those of treatment group 4 in the older or younger men. † Hormone levels are significantly greater post-treatment compared to pre-treatment.\n",
      "    unique: PMC2559951/PMC2559951.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559953.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Performance evaluation of serological assays used in the study\n",
      "  footer_text: 1 Samples taken one year apart 2 Excluding equivocal results 3 --, concordant negative 4 -+, disconcordant 5 ++, concordant positive 6 Major outer membrane protein (MOMP), Enzyme-linked immunosorbent assay (ELISA), virus-like particle (VLP), HSV-2 specific glycoprotein (gG2)\n",
      "    unique: PMC2559953/PMC2559953.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlation coefficients for prior sexually transmissible infections among nested controls in the PLCO Cancer Screening Trial, 1993-2001\n",
      "  footer_text: 1 Values in regular font, whites; values in italic font, blacks 2 * PChi-Square ≤0.01; ** PChi-Square ≤0.001; *** PChi-Square ≤0.0001\n",
      "    unique: PMC2559953/PMC2559953.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Self-reported baseline and clinical characteristics of prostate cancer cases and nested controls among whites and blacks in the PLCO Cancer Screening Trial, 1993 - 2001\n",
      "  footer_text: * Statistically different from race-specific controls (PPearson-Correlation <0.05 or PChi-Square <0.05)\n",
      "    unique: PMC2559953/PMC2559953.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559953.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559953.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559953.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559953.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559960.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559960.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  label_text: Table 4\n",
      "  caption_text: Prior sexually transmissible infections and risk of prostate cancer among whites and blacks in the nested case-control study, PLCO Cancer Screening Trial, 1993 - 2001\n",
      "  footer_text: 1 Using the total number of past infections, ranging from 0-7\n",
      "    unique: PMC2559953/PMC2559953.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559954.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559956.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559959.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559960.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sociodemographic Characteristics of the Sample (N = 78)\n",
      "  footer_text: a PANSS, Positive and Negative Syndromes Scales b WHODAS-II, World Health Organization Disability Assessment Schedule, Participation in Society subscale\n",
      "    unique: PMC2559960/PMC2559960.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Stepwise Regression of Stigma-Related Disability on Psychiatric and Depressive Symptoms and Cognitive Functioninga\n",
      "  footer_text: a Overall model was significant: (F(3,70) = 19.4, p< .001). b df for F change is as follows: (1) 1,72; (2) 1,71; (3) 1,70\n",
      "    unique: PMC2559960/PMC2559960.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559961.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559962.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Breakdown of ethnicity of IPC patients and in the California population as a whole\n",
      "  footer_text: \n",
      "    unique: PMC2559962/PMC2559962.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559964.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559965.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559966.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Geometric means (95% confidence intervals) of plasma carotenoids across categories of baseline characteristics among 2895 women in Women's Health Study\n",
      "  footer_text: 1 : p < 0.05 compared to the reference category using t-test. 2 : p < 0.01 compared to the reference category using t-test. 3 : p < 0.001 compared to the reference category using t-test.\n",
      "    unique: PMC2559966/PMC2559966.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559962.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559966.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559966.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Adjusted geometric means (95% confidence intervals) of plasma carotenoids across categories of cardiovascular disease related biomarkers among 2895 women in Women's Health Study\n",
      "  footer_text: 1 : Model adjusted for age and race for total cholesterol, additionally adjusted for total cholesterol for the other biomarkers. 2 : Model additionally adjusted for smoking (never, former, current <15 cigarettes/day, current ≥15 cigarettes/day), alcohol use (rarely/never, 1-3 drinks/mo, 1-6 drinks/wk, ≥1 drink/d), vigorous exercise (rarely/never, <1, 1-3, ≥4 times/wk), menopausal status (yes, no, uncertain), post-menopausal hormone use (never, former, current), multivitamin use (never, former, current), history of hypercholesterolemia (yes, no), hypertension (yes, no), and diabetes (yes, no), body mass index (continuous), and dietary intake of respective carotenoid (continuous). 3 : Linear trend was tested using the median values of each category as ordinal variable. 4 : p < 0.05 compared to the reference category using t-test. 5 : p < 0.01 compared to the reference category using t-test. 6 : p < 0.001 compared to the reference category using t-test.\n",
      "    unique: PMC2559966/PMC2559966.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Linear regression coefficients on natural log transformed plasma carotenoids by one unit change in cardiovascular disease related biomarkers among 2895 women in Women's Health Study\n",
      "  footer_text: 1 : Regression coefficients were standardized to the units specified in parenthesis for each biomarker. 2 : Model adjusted for age and race for total cholesterol, additionally adjusted for total cholesterol for the other biomarkers. 3 : The percentage changes presented in text were calculated by taking exponential of these regression coefficients (all such values). 4 : Model additionally adjusted for smoking (never, former, current <15 cigarettes/day, current ≥15 cigarettes/day), alcohol use (rarely/never, 1-3 drinks/mo, 1-6 drinks/wk, ≥1 drink/d), vigorous exercise (rarely/never, <1, 1-3, ≥4 times/wk), menopausal status (yes, no, uncertain), post-menopausal hormone use (never, former, current), multivitamin use (never, former, current), history of hypercholesterolemia (yes, no), hypertension (yes, no), and diabetes (yes, no), body mass index (continuous), and dietary intake of respective carotenoid (continuous).\n",
      "    unique: PMC2559966/PMC2559966.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559966.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559969.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559972.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559972.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2559969.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of reproductive organ weights (mean ± SD) of adult cynomologus macaques, rhesus macaques and baboons\n",
      "  footer_text: \n",
      "    unique: PMC2559969/PMC2559969.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559971.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559972.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of the study population, Singapore Chinese Health Study 1993-2005\n",
      "  footer_text: * Included both never users and former users over lifetime.\n",
      "    unique: PMC2559972/PMC2559972.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Characteristics of patients with histologically confirmed respiratory tract carcinomas, Singapore Chinese Health Study 1993-2005\n",
      "  footer_text: \n",
      "    unique: PMC2559972/PMC2559972.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Hazards ratio (HR) of respiratory tract carcinomas by incense use, Singapore Chinese Health Study 1993-2005\n",
      "  footer_text: * Adjusted for age at recruitment, year of interview, gender, dialect group, level of education, body mass index, number of cigarettes/day, number of years of smoking, time since smoking cessation, alcohol consumption, dietary intakes of isothiocyanate, beta-cryptoxanthin, lutein, lycopene and summed Chinese preserved foods, and for women, number of children (see details in the Materials and Methods). † Including both never and former users of incense.\n",
      "    unique: PMC2559972/PMC2559972.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Hazards ratio (HR) of respiratory tract carcinomas in relation to combined levels of cigarette smoking status, and frequency of incense use, Singapore Chinese Health Study 1993-2005\n",
      "  footer_text: * Adjusted for age at recruitment, year of interview, gender, dialect group, level of education, body mass index, number of cigarettes/day, number of years of smoking, time since smoking cessation, alcohol consumption, dietary intakes of isothiocyanate, beta-cryptoxanthin, lutein, lycopene and summed Chinese preserved foods, and for women, number of children (see details in the Materials and Methods). † Including both never and former users of incense.\n",
      "    unique: PMC2559972/PMC2559972.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Hazards ratio (HR) of respiratory tract carcinoma by histological subtype in relation to frequency of incense use, Singapore Chinese Health Study 1993-2005\n",
      "  footer_text: * Adjusted for age at recruitment, year of interview, gender, dialect group, level of education, body mass index, number of cigarettes/day, number of years of smoking, time since smoking cessation, alcohol consumption, dietary intakes of isothiocyanate, beta-cryptoxanthin, lutein, lycopene and summed Chinese preserved foods, and for women, number of children (see details in the Materials and Methods) † No current use comprise never and ex-users of incense. ‡ The sum of all respiratory tract carcinomas may differ from the total number of cases listed in Tables 2 and 3 since the present table counted the first incident cancer for a given individual. § Non-squamous cell carcinoma included adenocarcinoma, small cell carcinoma, undifferentiated, other and unspecified carcinoma.\n",
      "    unique: PMC2559972/PMC2559972.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559972.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559972.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559972.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559975.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2559974.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559975.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: GAR and AICAR Tfase Inhibition (Ki, μM)\n",
      "  footer_text: a Ref 18\n",
      "    unique: PMC2559975/PMC2559975.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: In vitro Cytotoxic Activity\n",
      "  footer_text: \n",
      "    unique: PMC2559975/PMC2559975.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: In vitro Cytotoxic Activity in Mutant Cell Lines\n",
      "  footer_text: a Ref 18\n",
      "    unique: PMC2559975/PMC2559975.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559976.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559977.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2559978.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559975.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559975.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2559978.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Relative Energies, E(kcal/mol) (Values Thermally Corrected to 298 K° Are Given in Parentheses) for the Acyclic conformers 1a (TTT) and 1b (CCT) of NH=CH−CH=CH−NH−BO and the cyclic conformer 1c (CCC) of (−NH=CH−CH=CH−NH−B−)O at the MP2(FC)/cc-pVDZ, MP2(FC)/aug-cc-pVDZ, and MP2(FC)/cc-pVTZ Computational Levels.\n",
      "  footer_text: \n",
      "    unique: PMC2559978/PMC2559978.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2559979.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2560174.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2560175.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Protection of mice immunized with CAS or RAS against challenge with wild-type sporozoites\n",
      "  footer_text: a Groups of mice were immunized with P. berghei ANKA (P.b.) or P. yoelii 17XNL (P.y.) control (vehicle-treated) sporozoites, CAS, or RAS as indicated, isolated from the same mosquito batches and were immunized at 7 day intervals. b Groups of mice were challenged with P. berghei ANKA or P. yoelii 17XNL wild-type sporozoites c Naïve, age-matched mice were used at the time of all immunizations and challenges. d Statistical analysis using Fisher exact test for comparison of Control and CAS groups gave p<0.002 for groups 1 and 3. e NA, not applicable. f Naïve, age-matched mice developed patent parasitemia 1.5 days p.i. in both the CAS and RAS groups. g All mice showed patent parasitemia 2 days p.i.\n",
      "    unique: PMC2560175/PMC2560175.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2560176.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient characteristics by race/ethnicity.\n",
      "  footer_text: Significance (p < .05) based on chi-square tests and ANOVAs with Bonferroni post-hoc comparisons, reference group = Non-Hispanic white: 1 Non-Hispanic black 2 Hispanic 3 Asian.\n",
      "    unique: PMC2560176/PMC2560176.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560175.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560176.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560176.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560176.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Regression coefficients for race/ethnicity on hip fracture outcomes.\n",
      "  footer_text: * F change p < .001. Reference group: non-Hispanic white. Model 1: race/ethnicity only. Model 2: adjusted for sociodemographic factors (gender, age, married, Medicare coverage). Model 3: adjusted for sociodemographic factors and case severity (sum of comorbidites, CMG tier, admission or discharge FIM, and LOS [except for model with LOS as dependent variable]).\n",
      "    unique: PMC2560176/PMC2560176.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Odds ratios for home discharge based on race/ethnicity.\n",
      "  footer_text: * p < .001 for block addition to model. Reference group: non-Hispanic white. Model 1: race/ethnicity only. Model 2: adjusted for sociodemographic factors (gender, age, married, Medicare). Model 3: adjusted for sociodemographic factors and case severity (sum of comorbidites, CMG tier, discharge FIM, and LOS). Model 4: adjusted for sociodemographic factors, case severity, and pre-hospitalization living setting (home vs. not home).\n",
      "    unique: PMC2560176/PMC2560176.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2560177.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: WHI RCT and GPRD STUDIES IN WOMEN WITH AN INTACT UTERUS TREATED WITH COMBINED HORMONE RECPLACEMENT THERAPY\n",
      "  footer_text: a Reference (5), 21.5% of subjects > age 70 vs. 2.5% in GPRD study. b Reference (8) c Reference (10) d Reference (11) e Number of events f Hazard ratio (95% confidence interval) g Exposed, number of events h Unexposed, number of events i Cox adjusted hazard ratio (95% confidence interval) j Coronary Heart Disease (CHD) a combination of myocardial infarction and coronary death was the primary outcome in the WHI RCT; however the results for myocardial infarction alone were similar. Results in the GPRD studies are all acute myocardial infarctions (MI). k Cerebrovascular accident l Adjusted hazard ratio using Prior event rate ratio (PERR) = 1.40 (0.87 – 2.44) m Adjusted hazard ratio using PERR = 2.63 (1.38 – 7.43) n Adjusted hazard ratio using PERR = 0.77 (0.41–1.42) o Adjusted hazard ratio using PERR = 2.20 (1.12–5.45)\n",
      "    unique: PMC2560177/PMC2560177.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: WHI RCT AND GPRD STUDIES in WOMEN WITH A PRIOR HYSTERECTOMY TREATED WITH ESTROGEN ONLY\n",
      "  footer_text: a Reference (7), 24% of women were older than 70 years versus 5.2% of women in the GPRD study b Reference (8) c Reference (12) d Adjusted HR using PERR = 1.28 (0.69–2.56) e Adjusted HR using PERR = 3.06 (1.39 – 10.31) f Adjusted HR using PERR = .92 (.35–2.79) g Adjusted HR using PERR = 2.32 (.98–7.94)\n",
      "    unique: PMC2560177/PMC2560177.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560177.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560177.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2560179.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2560982.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2560985.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2560987.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic features of US travelers to Mexico and risk factors for acquiring travelers' diarrhea\n",
      "  footer_text: * Student t test: means ± SD.\n",
      "    unique: PMC2560987/PMC2560987.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of PCR followed by genotyping versus ELISA for Cryptosporidium detection\n",
      "  footer_text: * PCR followed by genotype sequencing. † Calculated as [number of true positives/(number of true positives + number of false negatives)]. ‡ Calculated as [number of true negatives/(number of true negatives + number of false positives)]. § Calculated as [number of true positives/(number of true positives + number of false positives)]. ¶ Calculated as [number of true negatives/(number of true negatives + number of false negatives)].\n",
      "    unique: PMC2560987/PMC2560987.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Epidemiologic and clinical features of bacterial TD compared with Cryptosporidium TD\n",
      "  footer_text: * Includes the cases of cryptosporidiosis only and cryptosporidiosis with bacterial co-infection. Cryptosporidiosis cases in which bacterial entero-pathogens were identified were counted once in the cryptosporidiosis group for analysis and were not included in the bacterial infection subgroup. † Student t test: mean (95% CI).\n",
      "    unique: PMC2560987/PMC2560987.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560987.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560987.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2560987.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561069.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561069.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2560992.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561069.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparisons on demographic, smoking history, and affective variables across preinclusion attrition status.\n",
      "  footer_text: * Note: Denotes variables that have been transformed; No significant group differences were identified across demographic, smoking history, affective, or distress tolerance variables.\n",
      "    unique: PMC2561069/PMC2561069.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Spearman Rank Correlations among Physical and Psychological Distress tolerance tasks\n",
      "  footer_text: * p<.05; ** p<.01\n",
      "    unique: PMC2561069/PMC2561069.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561072.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561080.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561171.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Primary analysis: Classification of blinded validation set of sera from asymptomatic colonoscopy-screened persons, based on analysis of spectra generated by a nano-electrospray ionization source using a QSTAR-XL mass spectrometer and the ProteomeQuest® algorithm\n",
      "  footer_text: Sensitivity: 10/12= 83% Specificity: 9/25= 36% Accuracy: 51% (95% CI: 38% to 65%)\n",
      "    unique: PMC2561171/PMC2561171.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Post-hoc analysis: classification of blinded validation set after hemolyzed specimens were excluded\n",
      "  footer_text: Sensitivity: 7/9= 78% (95% CI: 40% to 96%) Specificity: 8/15= 53% (95% CI: 27% to 78%) Accuracy: 63% (95% CI: 53% to 72%)\n",
      "    unique: PMC2561171/PMC2561171.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561180.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561171.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561171.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561180.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Distributions of demographic characteristics and established breast cancer risk factors among controls and luminal, HER2-overexpressing, and triple-negative cases\n",
      "  footer_text: \n",
      "    unique: PMC2561180/PMC2561180.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Age-adjusted odds ratios for body mass index (BMI) and weight by tumor subtype and current HT use\n",
      "  footer_text: \n",
      "    unique: PMC2561180/PMC2561180.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Age-adjusted odds ratios for quartiles of other anthropometric characteristics by tumor subtype\n",
      "  footer_text: * Odds ratio (OR) adjusted for age and reference year. † p<0.05.\n",
      "    unique: PMC2561180/PMC2561180.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561180.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561180.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561190.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561182.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561188.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561190.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of individual studies with information on involuntary smoking in International Head and Neck Cancer Epidemiology consortium pooled data v1.1\n",
      "  footer_text: * Representative publication in which study methods are available. + All studies frequency matched controls to cases, minimally on age and sex. Additional frequency matching factors included center (Central Europe and Latin America studies), ethnicity (Tampa study), and neighborhood (Los Angeles study). ∥ Multicenter study.\n",
      "    unique: PMC2561190/PMC2561190.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Selected characteristics of head and neck cancer cases and controls\n",
      "  footer_text: # Information on ethnicity was not collected in the Central Europe and Latin America studies. In the Central Europe study, all subjects were classified as non-Hispanic White, since the large majority of these populations are expected to be White. In the Latin America study, we categorized subjects as ‘Latin American’. We adjusted for study center in all logistic regression models as a proxy variable for race/ethnicity since each center has an expected predominant ethnic group distribution.\n",
      "    unique: PMC2561190/PMC2561190.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561190.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561190.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561190.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Involuntary smoking among never tobacco users, involuntary smoking among never tobacco users and never alcohol drinkers and the risk of head and neck cancer\n",
      "  footer_text: * Adjusted for centers, age, sex, race/ethnicity, education and alcohol drinking (in drink-years; for never tobacco users only). ** Estimates from random effect models. 1 Included Central Europe, Tampa, Latin America, Los Angeles, and Houston studies. 2 Included Central Europe, Tampa, Latin America, Los Angeles, and Puerto Rico studies. 3 Included Central Europe, Tampa, and Latin America studies. 4 Included Central Europe, Tampa, Latin America, and Los Angeles studies. 5 Included Central Europe and Latin America studies.\n",
      "    unique: PMC2561190/PMC2561190.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Involuntary smoking among never tobacco users and the risk of head and neck cancer, by cancer subsite\n",
      "  footer_text: * Adjusted for centers, age, sex, race/ethnicity, education and alcohol drinking (in drink-years). § Estimates from random effect model. 1 Included Central Europe, Tampa, Latin America, Los Angeles, and Houston studies (no Tampa study for larynx). 2 Included Central Europe, Tampa, Puerto Rico, Latin America, and Los Angeles studies (no Tampa and Puerto Rico studies for larynx). 3 Included Central Europe, Tampa, Latin America, and Los Angeles studies (no Tampa study for larynx).\n",
      "    unique: PMC2561190/PMC2561190.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Involuntary smoking among never tobacco users and never alcohol users and the risk of head and neck cancer, by cancer subsite\n",
      "  footer_text: Note: Adjusted for centers, age, sex, race/ethnicity and education. 1 Included Central Europe, Tampa, Latin America, Los Angeles, and Houston studies (no Tampa study for larynx). 2 Included Central Europe, Tampa, Puerto Rico, Latin America, and Los Angeles studies (no Tampa and Puerto Rico studies for larynx). 3 Included Central Europe, Tampa, Latin America, and Los Angeles studies (no Tampa study for larynx).\n",
      "    unique: PMC2561190/PMC2561190.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561192.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: TMS-induced changes in tangential forces\n",
      "  footer_text: The data are shown for different external torques that required supination or pronation efforts. Δftht, Δfvft, and Δftotalt stand for the TMS-induced changes in the tangential forces of the thumb, virtual finger, and the total tangential force. Averages across subjects with standard errors are presented.\n",
      "    unique: PMC2561192/PMC2561192.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: TMS-induced changes in the moment of force\n",
      "  footer_text: The data are shown for different external torques that required supination or pronation efforts. ΔMn, ΔMt, ΔMtot stand for the changes in the moment of normal force, moment of tangential force and total moment of force. Averages across subjects with standard errors are presented.\n",
      "    unique: PMC2561192/PMC2561192.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Timing of synergy restoration\n",
      "  footer_text: The times of the initiation of the two stages of synergy restoration (in ms) are presented for the three characteristics of the hand mechanical action (means across the subjects and standard errors.)\n",
      "    unique: PMC2561192/PMC2561192.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561190.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561192.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561192.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561192.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561197.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561197.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561196.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561197.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Results from the Geometric Accuracy Test from T1-weighted Images. ACR specifies that lengths should be with ± 2 mm of actual length shown in first column.\n",
      "  footer_text: \n",
      "    unique: PMC2561197/PMC2561197.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: High Contrast Spatial Resolution Results from the T1- and T2-weighted Images from the Lower Right (LR) and Upper Left (UL) Grids at the Three Separate Spatial Resolutions 1.1, 1.0 and 0.9 mm provided in the ACR Phantom. ACR specifies that a 1 mm resolution should be regularly achieved with the settings employed for this test.\n",
      "  footer_text: \n",
      "    unique: PMC2561197/PMC2561197.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: MR QA Summary Statistics for the remaining five imaging tests. The ACR recommended specifications for the various test values are provided in the last column.\n",
      "  footer_text: \n",
      "    unique: PMC2561197/PMC2561197.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561208.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Peptide sequences used in these studies\n",
      "  footer_text: \n",
      "    unique: PMC2561208/PMC2561208.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: DNA immunization with a vector encoding RSV M2 protein lacking the immunodominant CTL epitope induces significant but reduced protection compared to immunization with the wild-type M2 genea\n",
      "  footer_text: a BALB/c mice (n=5) were immunized with one of the immunogens at day 0, 14 and 21, and challenged with live RSV on day 28 (day 7 after the third immunization, the expected peak day of CTL activity). Lung titers were determined on day 4 following challenge (the expected peak day of viral replication). b No significant difference from values obtained for mice immunized with PBS. (P≥0.05) c Significantly different from values obtained for mice immunized with PBS (P<0.01). d Significantly different from values obtained for mice immunized with PBS (P<0.02).\n",
      "    unique: PMC2561208/PMC2561208.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561223.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561224.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561228.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561197.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561208.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561208.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561228.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561228.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561228.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Dissociation constants for 8ec1 at 300 K in absence of calcium.\n",
      "  footer_text: \n",
      "    unique: PMC2561228/PMC2561228.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: 15Nε1 relaxation and dynamics parameters at pH 6.25, 281.5 K, and 14.1 T.\n",
      "  footer_text: NOE and R1 measurements used 0.65 mM 8ec1. R2 measurements are reported for both 0.65 mM and 0.30 mM 8ec1 in the absence of calcium and for 0.3 mM 8ec1 in the presence of 5 mM calcium; R2 results for 0.3 mM 8ec1 are shown in square brackets. Values of S2 were obtained from the NOE and R1 data using τm = 12.8 ± 0.7 ns and 21.6 ± 0.5 ns, for monomer and dimer states, respectively, as discussed in the text.\n",
      "    unique: PMC2561228/PMC2561228.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: 15Nε1 relaxation dispersion parameters at pH 6.25, 281.5 K.\n",
      "  footer_text: Relaxation dispersion parameters obtained from fitting the Carver-Richards equation (Carver and Richards, 1972) to the relaxation dispersion profiles shown in Figure 6. Values reported with (*) refer to parameters fit at 14.1 T. The value of Rex is reported for a τcp = 1 ms CPMG pulsing rate.\n",
      "    unique: PMC2561228/PMC2561228.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561229.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The velocity-amplitude relationships for horizontal vergence and vertical version for all subjects. The correlations were analyzed under logarithmic plots. All relationships are linearly positive (**p<0.0001). The bino-vg: vergence from binocular condition; bino-vs: version from binocular condition; mono-vg: vergence from monocular condition; mono-vs: version from monocular condition; divg: divergence; convg: convergence; vs: version.\n",
      "  footer_text: \n",
      "    unique: PMC2561229/PMC2561229.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Duration-amplitudes relationships for horizontal vergence and vertical saccades for all subjects. The correlations were analyzed under logarithmic plots. All relationships are linear (p<0.0001). The bino-vg: vergence from binocular condition; bino-vs: version from binocular condition; mono-vg: vergence from monocular condition; mono-vs: version from monocular condition; divg: divergence; convg: convergence; vs: version.\n",
      "  footer_text: \n",
      "    unique: PMC2561229/PMC2561229.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Relationships between the time of vergence velocity peaks and time of saccade velocity peaks for all subjects. R2: Correlation coefficients; S: slopes of the regression lines. All relationships of the subjects are linearly positive (**p<0.0001).\n",
      "  footer_text: \n",
      "    unique: PMC2561229/PMC2561229.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561229.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561229.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561229.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561229.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561229.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Parameters for relationships between peak velocity of horizontal vergence and peak velocity of vertical saccades for binocular viewing conditions. R2: Correlation coefficients; I: Intercepts; S: slopes of the regression lines. Positive relationships of the subjects are indicated with stars (* p<0.05; ** p<0.0001).\n",
      "  footer_text: \n",
      "    unique: PMC2561229/PMC2561229.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Relationships between peak velocity of horizontal vergence and peak velocity of vertical saccades for monocular viewing conditions. R2: Correlation coefficients. I: Intercepts. S: slopes of the regression lines. Positive relationships of the subjects are indicated with * and ** (* p<0.05; ** p<0.0001).\n",
      "  footer_text: \n",
      "    unique: PMC2561229/PMC2561229.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561244.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: \n",
      "  footer_text: Prevalence of the indirect and direct pathways for the synthesis of the aa-tRNA species listed in all three domains of life. The table was adapted from [132] and is reprinted with permission from Oxford University Press.\n",
      "    unique: PMC2561244/PMC2561244.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561247.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Association between participant characteristics and nonsteroidal anti-inflammatory drug use among 63 809 participants in the Vitamins and Lifestyle cohort (2000–2005)*\n",
      "  footer_text: * Values of some characteristics do not add up to total because of missing data (<5% of data are missing). NSAID = nonsteroidal anti-inflammatory drug; OR = odds ratio; CI = confidence interval; BMI = body mass index. † NSAID users are defined as participants who regularly used a NSAID (low-dose aspirin, regular- or extra-strength aspirin, or non-aspirin NSAIDs [ibuprofen, naproxen, or celecoxib or rofecoxib, or for women only, other prescription NSAIDs]) at least once a week for 1 year during the 10 years before baseline. ‡ Odds ratios were calculated by use of a logistic regression analysis that was adjusted for age and sex. § Cardiovascular disease variable was defined as report of bypass surgery, angioplasty, heart attack, or angina. || Met-h/wk is the metabolic equivalent hours per week and is also the energy expenditure expressed as kilocalories per kilogram per week. ¶ Smoking categories were defined as follows: never = never smoked daily for at least a year; current or recent = quit less than 10 years from baseline; former = quit 10 or more years from baseline.\n",
      "    unique: PMC2561247/PMC2561247.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561244.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561247.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Association between nonsteroidal anti-inflammatory drug use and incident melanoma in the Vitamins and Lifestyle cohort (2000–2005)\n",
      "  footer_text: * Adjustments were for age at baseline (timescale), sex (female or male), education (high school or fewer than 12 years, some college, or advanced degree), first-degree family history of melanoma (no or yes), personal history of nonmelanoma skin cancer (no or yes), ever had moles removed (no or yes), freckles between ages 10 and 20 years (no or yes), had three or more severe sunburns between ages 10 and 20 years (no or yes), natural red or blond hair between ages 10 and 20 years (no or yes), reaction to 1 hour in strong sunlight (tan or no sunburn, mild burning, painful sunburn, or severe sunburn with blistering), multivitamin use (none, <0 to 3, or ≥4 d/wk), chronic pain in last year (no or yes), arthritis (no or yes), cardiovascular disease (no or yes), and kidney disease or ulcer (no or yes). HR = hazard ratio; CI = confidence interval; NSAID = nonsteroidal anti-inflammatory drug. † Any NSAID use was defined as report of regular use of low-dose aspirin, regular- or extra-strength aspirin, or non-aspirin NSAIDs (ibuprofen, naproxen, or cele-coxib or rofecoxib or, for women only, other prescription NSAIDs). ‡ Nonusers of any NSAID (low-dose aspirin, regular- or extra-strength aspirin, or non-aspirin NSAIDs [ibuprofen, naproxen, or celecoxib or rofecoxib or, for women only, other prescription NSAIDs]) were the common reference for all exposure groups. § Dose for each exposure was computed as (days per week of use) multiplied by (years of use divided by 10 years). || Users of NSAIDs other than regular- or extra-strength aspirin were excluded from the referent group. ¶ Users of aspirin were excluded from the referent group.\n",
      "    unique: PMC2561247/PMC2561247.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561248.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561250.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of cortactin binding proteins that regulate actin cytoskeletal dynamics Summary of cortactin-binding proteins that interact directlvy with the actin cytoskeleton, including binding characteristics to cortactin and actin, as well as functional and physiological effects.\n",
      "  footer_text: \n",
      "    unique: PMC2561250/PMC2561250.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561252.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561247.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561250.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561256.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561253.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561255.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561256.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Custom Internal Size Standard Primers.\n",
      "  footer_text: A single ROX-labeled forward primer and a series of 20 reverse primers were used to generate amplicons of varying length using the pcDNA3.1+ plasmid as a template for PCR. Amplicon lengths are indicated to the right of each reverse primer. PCR products were purified, normalized to 10ng/uL and pooled to generate a custom internal size standard for 3130xL capillary electrophoresis RSCA.\n",
      "    unique: PMC2561256/PMC2561256.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: MHC class I screening panel sequences pooled based on reference strand specific migration rates.\n",
      "  footer_text: The fourteen screening panel sequences exhibit distinct characteristic electrophoretic mobilities when hybridized to each of three reference strand sequences. Based on their relative mobilities, screening panel sequences were divided into two pools per reference strand, increasing the number of sequences that can be identified in a single run.\n",
      "    unique: PMC2561256/PMC2561256.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The concordance of RSCA genotyping for 12 previously screened Chinese rhesus macaques and the frequency of 14 MHC class I alleles in 37 Chinese rhesus macaques.\n",
      "  footer_text: The concordance of RSCA genotyping with cDNA cloning and sequencing was determined for 12 Chinese rhesus macaques (upper panel). The frequencies of the 14 screening panel sequences were determined using an additional 25 Chinese rhesus macaques (bottom row). Nine macaques (ChRh13, 15, 23, 25, 32, 35, 37–39) did not express any of the 14 screening panel clones.\n",
      "    unique: PMC2561256/PMC2561256.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561257.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561258.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561256.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561261.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561268.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Kinetic constants of wild-type and CCPGCC-insertion PTPs assayed with pNPP\n",
      "  footer_text: \n",
      "    unique: PMC2561268/PMC2561268.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Kinetic constants of wild-type and CCPGCC-insertion PTPs in the absence (−) and presence (+) of FlAsH when assayed with pNPP\n",
      "  footer_text: \n",
      "    unique: PMC2561268/PMC2561268.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Apparent inhibition constants (KIapp) of FlAsH for CCPGCC-insertion PTPs\n",
      "  footer_text: \n",
      "    unique: PMC2561268/PMC2561268.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Kinetic constants of wild-Type and CCPGCC-insertion PTPH1 and PTPα in the absence (−) and presence (+) of FlAsH when assayed with DADEpYLIPQQG\n",
      "  footer_text: \n",
      "    unique: PMC2561268/PMC2561268.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561268.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561268.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561268.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561268.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561269.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561272.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The synergy of killing by sorafenib and vorinostat is dependent upon loss of c-FLIP-s expression Pancreatic (MiaPaca2) and hepatoma (HEP3B) cells were infected to express empty vector (CMV) or c-FLIP-s. Twenty four hours after infection cells were plated as single cells (250–1500 cells/well) in sextuplicate and 12h after plating treated with vehicle (VEH, DMSO), sorafenib (Sor., 3.0–6.0 µM) or vorinostat (Vor. 250–500 nM), or with both drugs combined, as indicated at a fixed concentration ratio to perform median dose effect analyses for the determination of synergy. Forty eight hours after drug exposure, the media was changed and cells cultured in drug free media for an additional 10–14 days. Cells were fixed, stained with crystal violet and colonies of > 50 cells / colony counted. Colony formation data were also entered into the Calcusyn program and combination index (CI) and fraction affected (Fa) values determined. A CI value of less than 0.90–1.00 indicates synergy; a CI value of 0.90–1.10 approximates to additive interactions between the drugs; a CI value of above 1.10 indicates antagonism (n = 2 independent studies).\n",
      "  footer_text: \n",
      "    unique: PMC2561272/PMC2561272.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561273.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Transcriptome changes (2-fold or greater; P values ≤ 0.05) associated with rgg nactivation during post-exponential growth\n",
      "  footer_text: a SPy numbers designate open reading frames based on the SF370 S. pyogenes genome annotation (Ferretti et al., 2001). b Change in transcript level of rgg mutant compared to that of the wild-type. c rgg inactivation doesn’t affect transcript level. d Asterisks indicate genes similarly regulated in NZ131 strain (Dmitriev et al., 2006).\n",
      "    unique: PMC2561273/PMC2561273.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561272.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561273.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561273.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlation between results obtained with DNA microarrays and quantitative TaqMan RT-PCR\n",
      "  footer_text: a SPy numbers designate open reading frames based on the SF370 S. pyogenes genome annotation (Ferretti et al., 2001). b Gene designations are in italics, and the corresponding protein function is in parentheses. c Change in transcript level of rgg mutant compared to that of the wild-type.\n",
      "    unique: PMC2561273/PMC2561273.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561277.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Human subject characteristics\n",
      "  footer_text: a The postmortem time was the time lapsed between a subject’s death and the rheometric experiments, except for Subjects 1 and 2 who were total laryngectomy patients.\n",
      "    unique: PMC2561277/PMC2561277.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Results of least-squares regressions for the parametric description of elastic shear modulus (G’), viscous shear modulus (G”), and dynamic viscosity (η’) of the human vocal fold cover according to Eqs. (25) - (27) (n = 7). R2 is the coefficient of determination indicating the goodness of curve-fitting\n",
      "  footer_text: \n",
      "    unique: PMC2561277/PMC2561277.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561279.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561283.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Physico-chemical properties of fluorescent ferumoxides (FL FE)\n",
      "  footer_text: [a] T1 Relaxation rate [b] T2 Relaxation rate\n",
      "    unique: PMC2561283/PMC2561283.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: NMR Relaxivities of FE-Pro complex in a gel\n",
      "  footer_text: [a] T1 Relaxation rate [b] T2 Relaxation rate\n",
      "    unique: PMC2561283/PMC2561283.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561277.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561277.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561283.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561283.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561284.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561285.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Group neurocognitive profile for the overall sample (N=85)\n",
      "  footer_text: * Item in italics represents a score in the borderline range of functioning\n",
      "    unique: PMC2561285/PMC2561285.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Group neurocognitive profile for the globally intact sample (n=54)\n",
      "  footer_text: \n",
      "    unique: PMC2561285/PMC2561285.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Incidence of absolute impairment in the globally intact group for each domain (N=54)\n",
      "  footer_text: \n",
      "    unique: PMC2561285/PMC2561285.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561289.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Pathology of the stomach and spleen, 6 weeks after injection of Th1, Th2 and Th17 effector T cells\n",
      "  footer_text: Legend:parietal cell destruction (PCD) was scored as described above, with grade 5 indicating 25-50% PCD, grade 6 indicating 50-100% PCD. Cells within the stomach infiltrates, as well as fore-stomach and splenic lesions were rated either zero (-) or + to +++ for mild/moderate/high presence. Abreviations: Eos=eosinophils, GC=macrophage giant cells, GCH=germinal center hyperplasia, Ly=lymphocytes, PC=plasmocytosis\n",
      "    unique: PMC2561289/PMC2561289.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561285.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561285.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561285.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561289.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561289.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Succeptibility of Teffector cells to Treg mediated suppression in vivo at 2, 4 and 6 weeks after injection\n",
      "  footer_text: Legend: Δ= delta (Treg-mediated reduction of cell numbers per gLN expressed in %). All other values are given as cells per gLN × 1000.\n",
      "    unique: PMC2561289/PMC2561289.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561294.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic characteristics and prevalence of Hurricane Katrina-related factors.\n",
      "  footer_text: Abbreviation: SD – standard deviation, PTSD – Post-traumatic stress disorder † All p-values (except age, gender, and race) are adjusted for age, gender, and race\n",
      "    unique: PMC2561294/PMC2561294.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Adjusted prevalence ratios of low adherence by different Hurricane Katrina-related factors for CoSMO participants from high and low affected areas.\n",
      "  footer_text: Adjusted for age, race, gender, education and marital status. Abbreviation: PR = Prevalence ratio; CI = Confidence Interval\n",
      "    unique: PMC2561294/PMC2561294.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561300.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Distribution of Less-than-perfect Intentional and Unintentional Medication Adherence by Socio-Demographic Characteristics and Damage to Residence in Sample of Patients (n= 210) in a New Orleans metropolitan area clinic from November 2005 to August 2006\n",
      "  footer_text: \n",
      "    unique: PMC2561300/PMC2561300.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561294.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561294.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561300.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561300.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561305.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Distribution of Post-Katrina Medication Issues by Socio-Demographic Characteristics in Sample of Patients (n= 86) in a New Orleans metropolitan area clinic from November 2005 to August 2006\n",
      "  footer_text: * p< 0.05\n",
      "    unique: PMC2561300/PMC2561300.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561305.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of Different Loading Conditions on the Crack Edge Area (CA) to Total Volume (V) of Multiaxial Confined Compression Specimens of Renew Dental Composite\n",
      "  footer_text: \n",
      "    unique: PMC2561305/PMC2561305.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561309.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Constant, R2, and p-values, and residuals for each of the nine f2 frequencies.\n",
      "  footer_text: \n",
      "    unique: PMC2561309/PMC2561309.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561312.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Biodistribution of different anti-HER2 Affibody conjugates after iv injection in NMRI mice.\n",
      "  footer_text: a data presented as average ± SD (n = 4) of per cent of injected activity per gram (% IA/g); b for intestine (together with content) and carcass, data presented as per cent of injected activity per whole sample (% IA); c significant difference (p < 0.05) with 111In-DOTA-Z2395-C at the same time point; d significant difference (p < 0.05) with 111In-CHX-A” DTPA-Z342 at the same time point;\n",
      "    unique: PMC2561312/PMC2561312.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561309.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561312.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561312.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561312.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Biodistribution of 111In-CHX-A” DTPA-Z2395-C, 111In-DOTA-Z2395-C and 111In-CHX-A” DTPA-Z342 Affibody conjugates 4 h after iv injection in BALB/C nu/nu mice bearing SKOV-3 xenografts\n",
      "  footer_text: a data presented as average ± SD (n = 4) of per cent of injected activity per gram (% IA/g); b for intestine (together with content) and carcass, data presented as per cent of injected activity per whole sample (% IA); c significant difference (p < 0.05) with 111In-DOTA-Z2395-C at the same time point; d significant difference (p < 0.05) with 111In-CHX-A” DTPA-Z342 at the same time point;\n",
      "    unique: PMC2561312/PMC2561312.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Tumor-to-organ ratios in BALB/C nu/nu mice bearing SKOV-3 xenograftsa\n",
      "  footer_text: a Data are presented as an average for four animals ± SD. b Significant difference (p < 0.05) between tumor-to-organ ratios at 1 and 4 h p.i. for 111In-CHX-A” DTPA-Z2395-C c Significant difference (p < 0.05) between tumor-to-organ ratios at 4 and 24 h p.i. for 111In-CHX-A” DTPA-Z2395-C d Significant difference (p < 0.05) between tumor-to-organ ratios 4 h p.i. for 111In-CHX-A” DTPA-Z2395-C and 111In-DOTA-Z2395-C; e Significant difference (p < 0.05) between tumor-to-organ ratios 4 h p.i. for 111In-CHX-A” DTPA-Z2395-C and 111In-CHX-A” DTPA-Z342 f Significant difference (p < 0.05) between tumor-to-organ ratios 4 h p.i. for 111In-DOTA-Z2395-C and 111In-CHX-A” DTPA-Z342\n",
      "    unique: PMC2561312/PMC2561312.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561316.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561321.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: BRIP1 sequence alterations\n",
      "  footer_text: a Numbering based on RefSeq NT_010783.13 (for the intronic variant) and NM_032043 (for all the variants within the coding region). For exonic variants, numbering starts at codon 1. b Frequency of second allele listed.\n",
      "    unique: PMC2561321/PMC2561321.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561336.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561893.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561321.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561893.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561893.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561893.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561893.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  caption_text: Agreement between algorithm and three neonatologists on whether infant received a permissive hypercapnia strategy of care\n",
      "  footer_text: \n",
      "    unique: PMC2561893/PMC2561893.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Infant characteristics and incidence of intraventricular hemorrhage and death by normocapnia, permissive hypercapnia or unclassifiable strategy of care\n",
      "  footer_text: Numbers are mean ± standard deviation or frequency (percent) unless otherwise noted * p < 0.05 for difference between permissive hypercapnia and normocapnia strategy † interquartile range ‡ Score for Neonatal Acute Physiology - II # positive inspiratory pressure § post-menstrual age ** intraventricular hemorrhage\n",
      "    unique: PMC2561893/PMC2561893.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Odds ratios (OR) and 95% confidence intervals (CI) for intraventricular hemorrhage comparing infants with permissive hypercapnia and normocapnia treatment strategies\n",
      "  footer_text: * Adjusted for gestational age, 1-minute Apgar score, sex, outborn status, receipt of antenatal steroids, and revised SNAP-II, p > 0.28 for all models based on Hosmer-Lemeshow goodness of fit test † intraventricular hemorrhage\n",
      "    unique: PMC2561893/PMC2561893.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Odds ratios (OR) and 95% confidence interval (CI) for intraventricular hemorrhage comparing infants with a permissive hypercapnia and normocapnia strategy of care, by one-minute Apgar score\n",
      "  footer_text: * adjusted for gestational age, sex, outborn status, antenatal steroids, and revised SNAP-II † Apgar score was missing for 5 infants\n",
      "    unique: PMC2561893/PMC2561893.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: TABLE 5\n",
      "  caption_text: Mean scores and crude and adjusted differences in mean scores for early childhood behavior and function scores for children with a permissive hypercapnia and a normocapnia strategy of care\n",
      "  footer_text: * adjusted for gestational age, one-minute Apgar score, sex, outborn status, receipt of antenatal steroids, revised SNAP-II, maternal education, race, household income, single-parent household, and interviewer † higher scores indicate more problems ‡ higher scores indicate better function\n",
      "    unique: PMC2561893/PMC2561893.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561895.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Genotype-phenotype correlations in colorectal neoplasia\n",
      "  footer_text: "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561893.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561895.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "    unique: PMC2561895/PMC2561895.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561898.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561900.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561901.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Drugs Receiving Specific Recommendation from the PAC\n",
      "  footer_text: Abbreviations: AM-T = antimicrobial/transplant; AP = allergy/pulmonary; CR = cardiorenal; GI = gastroenterology; M = metabolic; N = neurologic; P = psychiatric; Pa = pain; U = urologic.\n",
      "    unique: PMC2561901/PMC2561901.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Drugs Returned to Routine Monitoring by the PAC\n",
      "  footer_text: Abbreviations: AM-T = antimicrobial/transplant; AP = allergy/pulmonary; CR = cardiorenal; GI = gastroenterology; H = hematologic; M = metabolic; N = neurologic; O = ophthalmologic; Pa = pain; Oc = oncologic; R = reproductive; Rh = rheumatologic; U = urologic.\n",
      "    unique: PMC2561901/PMC2561901.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561901.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561901.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561905.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561905.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561902.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561905.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sequences of Lacl DNA operator ligandsa.\n",
      "  footer_text: a lacO1 is one of the naturally occurring operators found in the lac operon (19) and controls transcription in the in vivo assays described previously (4). lacOsym and lacOdisC are engineered variants (16-18).\n",
      "    unique: PMC2561905/PMC2561905.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Operator binding by LLhP variantsa.\n",
      "  footer_text: a DNA binding data were determined from 3−4 independent measurements, using two different preparations of protein. Reported errors represent one standard deviation. Buffer used for DNA-binding measurements was 10 mM Tris-HCl, pH 7.4, 150 mM KCl, 5% DMSO, 0.1 mM EDTA, and 0.3 mM DTT. DNA concentration was 1.5 × 10−12 M. b For LLhP variants, low affinity conditions were in the absence of hypoxanthine. For Lacl, low affinity conditions were in the presence of 1mM IPTG. c For LLhP variants, high affinity conditions included a saturating concentration of hypoxanthine (see Materials and Methods). For Lacl, no effector was present. d Low affinity binding divided by high affinity binding. e Values from (9). Lacl measurements were also repeated in parallel with LLhP and showed similar results. f NR: No response to the addition of hypoxanthine. g NBD: Very little, if any, binding detected. h The same buffer as solution scattering experiments: 0.12 mM Hepes-KOH, pH 7.6, 200 mM KCl, 5% glycerol, 1 mM EDTA, and 0.3 mM DTT.\n",
      "    unique: PMC2561905/PMC2561905.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of Experimental and Model Structural Parameters for LLhP Chimera and R3: Effects of DNA and Effector Binding\n",
      "  footer_text: a Total quality estimate defined in GNOM (49), equal to 1.0 for an ideal solution. Model parameters are given for b Model of apo-LLhP (see Materials and Methods). c Model of DNA-bound LLhP (see Materials and Methods). d,e,f Parameters taken from reference (23) d is the model of the apo-Lacl dimer taken from the one homodimer of the tetrameric Lacl crystal structure (1lbg chains A and B; (41)) e is the Lacl R3 dimer model optimized by refinement against the scattering data that shows the extended configuration (as opposed to the folded hinge-helix) for the linker sequence connecting the DBD and regulatory domains f is the model for R3-lacO1 taken without modification from the 1LBG crystal structure (1lbg chains A, B, E, and F). Note that this latter structure is missing 4 base pairs from the DNA sequence (17 basepairs instead of 21) and 11 amino acid residues from the protein, resulting in a somewhat higher than ideal χ2 value. g No high-resolution structures are available for the Lacl-lacO1-IPTG complex.\n",
      "    unique: PMC2561905/PMC2561905.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561905.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561905.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561918.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561918.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561918.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561918.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561918.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Operator capture by LLhP and variants (× 10−7 M hypoxanthine)a.\n",
      "  footer_text: a The midpoints of the operator capture experiments were determined from 3−4 independent measurements, using two different preparations of protein. Reported errors represent one standard deviation. Buffer was 10 mM Tris-HCl, pH 7.4, 150 mM KCl, 5% DMSO, 0.1 mM EDTA, and 0.3 mM DTT. The DNA concentration was ∼2 × 10−12 M and hypoxanthine concentrations were varied. Protein concentrations were chosen so that the final conditions were in the range of 70−90% saturation on affinity binding curves determined in the presence of hypoxanthine: concentrations were 1.2 to 3.3 × 10−8 M I48S; 2 × 10−9 to 1.2 × 10−8 M I48V; 1.5 × 10−9 to 2.5 × 10−9 M Q55T; 3 × 10−9 to 3 × 10−8 M Q55V; and 6 × 10−9 M S61C and S61M. b Not determined, because affinity experiments showed no allosteric response and very weak affinity (Table 2).\n",
      "    unique: PMC2561905/PMC2561905.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561909.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561918.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Lowest bending (B), stretching (S) and twisting (T) frequencies in cm−1 for the sequence GACTn, n = 4,6,8,10 with a salt concentration of 0.05M and T=300K.\n",
      "  footer_text: \n",
      "    unique: PMC2561918/PMC2561918.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mechanical constants for various DNA sequences with a salt concentration of 0.05M and T=300K. The values reported in italics and bold correspond to stiffer and more flexible characteristics than random DNA, respectively\n",
      "  footer_text: \n",
      "    unique: PMC2561918/PMC2561918.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Lowest in plane bending, out of plane bending, and twisting frequencies in cm−1 for the 94-base-pair minicircle (TA94) with a salt concentration of 0.05M and T=300K.\n",
      "  footer_text: \n",
      "    unique: PMC2561918/PMC2561918.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Mechanical constants extracted from the results presented in Table 3\n",
      "  footer_text: \n",
      "    unique: PMC2561918/PMC2561918.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Mechanical constants for several 94bp sequences with a salt concentration of 0.05M and T=300K.\n",
      "  footer_text: \n",
      "    unique: PMC2561918/PMC2561918.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561920.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561922.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Expression levels of CD19 and CD22 and the cytotoxicity of immunotoxins\n",
      "  footer_text: * Flow cytometry was conducted with two-step staining. First FMC63 mAb or RFB4 mAb, then goat anti-mouse PE conjugated F(ab)’2. † Cytotoxicity was determined by WST-8 cell viability assay.\n",
      "    unique: PMC2561922/PMC2561922.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Relationship between internalization of RFB4 and intracellular CD22 change\n",
      "  footer_text: * CD22, cell surface CD22. † RFB4, internalized RFB4. † CD22, intracellular CD22. ‡† CD22, the decrease of intracellular CD22 = †CD22- †CD22 (with RFB4).\n",
      "    unique: PMC2561922/PMC2561922.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561923.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Knockout of genes expressed in ectodermal appendages\n",
      "  footer_text: \n",
      "    unique: PMC2561923/PMC2561923.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mutations in human homeobox genes affecting development of ectodermal appendages\n",
      "  footer_text: \n",
      "    unique: PMC2561923/PMC2561923.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561922.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561922.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561923.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561923.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561942.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means and standard deviations for demographic and sleep-related variables\n",
      "  footer_text: a Numbers in parentheses represent standard deviations b Obstructive Apnea Index c Multiple Sleep Latency Test d trend, p = .06 e OAI analyses completed on log data due to skewed distribution, but raw data presented for clarity f Pediatric Sleep Questionnaire (PSQ) subscale * p < .05 ** p < .01 *** p < .001\n",
      "    unique: PMC2561942/PMC2561942.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Means and standard deviations for behavioral and cognitive test scores\n",
      "  footer_text: a Wechsler Abbreviated Scale of Intelligence (WASI) b Numbers in parentheses represent standard deviations c Wechsler Individualized Achievement Test (WIAT) d Trend for significance at p < .08 or better e Children’s Memory Test (CMS) f Integrated Visual and Auditory Continuous Performance Test (IVA), Full Scale Attention Quotient (FSAQ) and Full Scale Response Control Quotient (FSRCQ) g Connors’ Parent Rating Scale: Long Version (CPRS-R:L) * p < .05 ** p < .01 *** p < .001\n",
      "    unique: PMC2561942/PMC2561942.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561946.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561947.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561942.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561942.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Testing Criteria\n",
      "  footer_text: * Percentage of expected positive results based on the experience of the BCCA HDGC program\n",
      "    unique: PMC2561947/PMC2561947.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561951.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Comparison of clinical variables and demographics according to status of the Sentinel Node in patients with thin primary cutaneous melanoma.\n",
      "  footer_text: \n",
      "    unique: PMC2561951/PMC2561951.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Distribution of positive sentinel nodes according to tumor depth and intranodal tumor burden for patients with thin primary cutaneous melanoma.\n",
      "  footer_text: \n",
      "    unique: PMC2561951/PMC2561951.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Patterns of first recurrence for SN negative and SN positive patients with thin primary cutaneous melanoma.\n",
      "  footer_text: \n",
      "    unique: PMC2561951/PMC2561951.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Reported results of Sentinel Lymph Node Biopsy in patients with thin (≤1.0 mm) primary cutaneous melanoma and factors correlated with a tumor-positive Sentinel Node.\n",
      "  footer_text: \n",
      "    unique: PMC2561951/PMC2561951.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561954.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and Alcohol Use Data\n",
      "  footer_text: a The two studies involved different age groups; however, there were no differences whatsoever between the two original control groups on any of the dependent variables; therefore we combined the control groups into a single non-alcoholic control (NAC) group. Measures are reported mean ± standard deviation. N/A: Not applicable b Proportion of first degree relatives who are problem drinkers\n",
      "    unique: PMC2561954/PMC2561954.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561951.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561951.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561951.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561951.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561954.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Psychological Measures and Psychiatric Symptom Counts\n",
      "  footer_text: a summed across anxiety, mood and externalizing domains b mood + anxiety symptoms Effect is significant * p ≤ 0.05 ** p ≤ 0.01 *** p ≤ 0.001\n",
      "    unique: PMC2561954/PMC2561954.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561956.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561954.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2561958.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of the subjects according to the gastric cancer risk area\n",
      "  footer_text: Abbreviations: SD: standard deviation; NAG: non-atrophic gastritis; MAG: multifocal atrophic gastritis without metaplasia; IM: intestinal metaplasia; PCR: polymerase chain reaction a Cases with normal histology were not included. b This category includes s1m1 + s1 and m1/m2 + s1/s2 and m1 c Eight percent of missing values in high gastric cancer risk area.\n",
      "    unique: PMC2561958/PMC2561958.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Eosinophil and mast cell quantification in antral gastric mucosa according to the gastric cancer risk area\n",
      "  footer_text: Abbreviations: HPF: high power field (400x); IQR: interquartile range. a Muscularis mucosae not included\n",
      "    unique: PMC2561958/PMC2561958.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561958.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561958.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561958.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561958.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561958.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Eosinophil density according to histopathological diagnosis by gastric cancer risk area\n",
      "  footer_text: a This value is not a median, it is the eosinophil density of the only case in that category. Abbreviations: NAG: non-atrophic gastritis; MAG: multifocal atrophic gastritis without metaplasia; IM: intestinal metaplasia; IQR: interquartile range.\n",
      "    unique: PMC2561958/PMC2561958.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Odds ratios (OR) and 95% confidence intervals (CI) for advanced gastric lesions (IM and dysplasia) according to selected percentiles of the eosinophil density by gastric cancer risk area\n",
      "  footer_text: a Odds ratios adjusted for H. pylori genotypes.\n",
      "    unique: PMC2561958/PMC2561958.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Odds ratios (OR) and 95% confidence intervals (CI) for advanced gastric lesions (IM and dysplasia) according to selected percentiles of the mast cell density\n",
      "  footer_text: a Odds ratios adjusted for H. pylori genotypes.\n",
      "    unique: PMC2561958/PMC2561958.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561962.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2561993.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Stimulus exemplars used in the attentional set-shifting task\n",
      "  footer_text: Compound discriminations were based on fixed combinations of pairs of exemplars. The exemplar pairs from different dimensions were presented in random combination, but each sequence performed by a control mouse was matched by that of a COMT-Val-tg littermate. Neutral stimuli for the different dimensions were: no outer texture; sawdust bedding (Harlan Teklad); digging medium; and no odor. The digging medium was changed often so the mice would not use odor cues to locate the bowls baited with food pellets. For 500 ml of each medium, 10 ml of crushed food pellets (about 6g) were mixed with the medium to mask the smell of the hidden food pellet. For scented mixtures, an odor (0.7g) was mixed with 500 ml of the medium and crushed pellets. a Aspen bedding, Aquarium gravel, Moss, Kay-Kob, Repti Bark (PetSmart); Alpha-dri (Shepherd Specialty Papers); CareFresh, Ultra White CareFresh (International Absorbents); Crystalit (Pharmacal Research Laboratories); PaperChip (Shepherd Specialty Papers).\n",
      "    unique: PMC2561993/PMC2561993.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2561998.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Designation, sexes and allotypes of experimental rabbits\n",
      "  footer_text: \n",
      "    unique: PMC2561998/PMC2561998.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Leukocyte responses to immunizations among experimental rabbitsa\n",
      "  footer_text: a The mean value of the 3rd,5th and 7th boosts is shown outside the parenthesis and the highest leukocyte response of the 3rd, 5th and 7th boosts is the value within parenthesis (WBC reference range in 1000s 4–10; reference range/μl Monocytes; 0–300, Eosinophils, 0–100; and Basophils, 0–500). Neutrophil and lymphocyte counts remained within the reference ranges (reference range/μl Neutrophils 1,200–7,000 and Lymphocytes 1,200–7,000).\n",
      "    unique: PMC2561998/PMC2561998.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of autoantibody response among experimental rabbitsa\n",
      "  footer_text: a Numbers shown are highest values among the sera collected after 3rd, 5th, and 7th boosts and numbers in parentheses are the means of these values. Mean antibody response OD values were: GR MAP-8, 2.88 +/− 0.11; anti-dsDNA, 1.40 +/− 0.72; RNP, 0.73 +/− 0.49; Sm 0.72 +/− 0.56, and ANA 1.38 +/− 0.66. Shaded cells indicate responder rabbits with a highest OD value 1 standard deviation above mean antibody responses. For ANA IFA immunofluorescence, scores are shown for pre- and post 3rd, 5th, and 7th boosts and shaded areas indicate those sera with at least one 3+ score.\n",
      "    unique: PMC2561998/PMC2561998.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Correlations between leukocyte, immune, and autoantibody responses among immunized rabbits.\n",
      "  footer_text: * significant at 5% level ** Significant at 1% level\n",
      "    unique: PMC2561998/PMC2561998.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561993.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561998.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561998.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561998.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2561998.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562002.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Baseline Characteristics and Testing Data for 72 subjects who completed Keep the Beat program participation\n",
      "  footer_text: \n",
      "    unique: PMC2562002/PMC2562002.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Baseline and 3 month data for 72 subjects who completed participation in Keep the Beat\n",
      "  footer_text: * Women Only (N=58) ** Subset of Participants with Flow Cytometry Data (N=21)\n",
      "    unique: PMC2562002/PMC2562002.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Covariates as predictors of change in brachial artery flow-mediated dilation following 3 months of program participation, initially found significant in the stepwise model, then re-run using the general linear model with Type III sums of squares so that the P-value of each covariate reflects adjustment for all other covariates. All analyses were performed using the SAS statistical analysis package, utilizing the STEPWISE, GLM, and MEANS procedures (SAS User's Guide: Statistics, Version 9 Edition: SAS Institute Inc, Cary, NC). All reported P-values are based on two-sided t-tests.\n",
      "  footer_text: \n",
      "    unique: PMC2562002/PMC2562002.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562003.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562027.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562028.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562002.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562002.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562002.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562028.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of participants at their baseline examination in The Framingham Heart Study, Framingham, MA, 1948-1975\n",
      "  footer_text: \n",
      "    unique: PMC2562028/PMC2562028.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Cumulative incidence of an alcohol use disorder (ICD-9 code for 100% attributable disease) between ages 40 and 79, by sex and birth cohort from The Framingham Heart Study, Framingham, MA, 1948-1975\n",
      "  footer_text: \n",
      "    unique: PMC2562028/PMC2562028.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562037.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562040.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Mean and standard deviations of the kinematic, kinetic and temporal data from all subjects (N = 10) Statistical analysis includes results of Bonferroni corrected post-hoc paired t-tests between natural and altered conditions. When repeated measures ANOVAs testing the main effect of condition were not significant (p>0.05), post-hoc tests were not conducted (NS). Bold text indicates significant differences (p<0.05).\n",
      "  footer_text: \n",
      "    unique: PMC2562040/PMC2562040.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562042.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562028.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562040.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562162.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Developmental phenotypes of Sphk and S1P receptor null mice\n",
      "  footer_text: \n",
      "    unique: PMC2562162/PMC2562162.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562164.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample Characteristics\n",
      "  footer_text: N = 25,957\n",
      "    unique: PMC2562164/PMC2562164.xml::T1-0\n",
      "    unique: PMC2562164/PMC2562164.xml::T1-1\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Complications and Death Rates\n",
      "  footer_text: a Patients may have suffered from more than one complication. Note. Because of incomplete or missing data, N = 24,618.\n",
      "    unique: PMC2562164/PMC2562164.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562162.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562164.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562164.xml::T1-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562164.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Most Common Complications and Death Rates by Tumor Type\n",
      "  footer_text: Note. Because of incomplete or missing data, N = 24,618.\n",
      "    unique: PMC2562164/PMC2562164.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Selected Complication Rates by Emergency Department Source of Admission\n",
      "  footer_text: a p < 0.001 for all complications Note. Because of incomplete or missing data, N = 24,618.\n",
      "    unique: PMC2562164/PMC2562164.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562167.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562168.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562170.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562164.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562164.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient demographics (N = 133)\n",
      "  footer_text: * Other includes Hispanic and Asian\n",
      "    unique: PMC2562170/PMC2562170.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Location of tumors\n",
      "  footer_text: N/A = Not applicable.\n",
      "    unique: PMC2562170/PMC2562170.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Characteristics of Chemotherapy [Median- 3 cycles (range: 1–24 cycles)]\n",
      "  footer_text: * Cyclophosphamide, vincristine, adriamycin, predinsone, Rituxmab\n",
      "    unique: PMC2562170/PMC2562170.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562171.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562173.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Demographic and clinical characteristics of 116 HIV-positive children at enrollment in an observational study of macronutrient intake1\n",
      "  footer_text: 1 Enrollment is defined as the first qualifying dietary intake. CDC, Centers for Disease Control and Prevention; HAART, highly active antiretroviral therapy. 2 The initial CDC stage was missing for one child from the Boston cohort. 3 Medians; range in parentheses (all such values). 4 Years after the beginning of the study (14 February 1995) that a patient’s first qualifying dietary intake occurred. 5 n = 79 for the Boston cohort, and n = 12 for the Rochester cohort.\n",
      "    unique: PMC2562173/PMC2562173.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Regression estimates of anthropometric and dietary characteristics among 116 HIV-positive children in 19951\n",
      "  footer_text: 1 All values are means unless designated as proportions and expressed as percentages (see footnote 3); 95% CIs in parentheses (all such values). For some variables, the number of male/female subjects (observations) was different from that indicated in the table header. CD4: 45/60 (275/353); sugar: 46/62 (216/273); monounsaturated and saturated fat: 47/66 (284/341); polyunsaturated fat: 45/65 (275/331); dietary fiber: 48/66 (287/350); and cholesterol: 48/66 (290/353). Sex-specific means or proportions at the beginning of the study period with 50% weight on each center and their 95% CIs were estimated with generalized linear models including center, sex, years since the beginning of the study (14 February 1995), and the interaction between years and sex. HAART, highly active antiretroviral therapy; AMC, arm-muscle circumference; EER, estimated energy requirement; RDA, Recommended Dietary Allowance; AI, adequate intake; USDA, US Department of Agriculture. 2 Estimates were obtained by using generalized estimating equations with an exchangeable working correlation structure to account for potential correlations between repeated measurements on the same individuals. The estimated proportion was calculated using the logit link and applying backtransformation to the sex-specific intercepts. A working independence correlation structure was used to model the proportion of HAART-exposed children because of convergence problems with an exchangeable working correlation structure. P value for the 2-sided Wald test of no sex difference in the estimated mean or proportion at the beginning of the study period (no sex difference in intercepts in the generalized linear model). 3 Estimated proportion from the generalized linear model expressed as a percentage.\n",
      "    unique: PMC2562173/PMC2562173.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Regression estimates of anthropometric and dietary characteristics among 116 HIV-positive children in 20041\n",
      "  footer_text: 1 All values are means unless designated as proportions and expressed as percentages (see footnote 3); 95% CIs in parentheses (all such values). For some variables, the number of male/female subjects (observations) was different from that indicated in the table header. CD4: 45/60 (275/353); sugar: 46/62 (216/273); monounsaturated and saturated fat: 47/66 (284/341); polyunsaturated fat: 45/65 (275/331); dietary fiber: 48/66 (287/350); and cholesterol: 48/66 (290/353). Sex-specific means or proportions at the end of the study period with 50% weight on each center and their 95% CIs were estimated with generalized linear models including center, sex, years since the beginning of the study (14 February 1995), and the interaction between years and sex. HAART, highly active antiretroviral therapy; AMC, arm-muscle circumference; EER, estimated energy requirement; RDA, Recommended Dietary Allowance; AI, adequate intake; USDA, US Department of Agriculture. 2 Estimates were obtained by using generalized estimating equations with an exchangeable working correlation structure to account for potential correlations between repeated measurements in the same individuals. The estimated proportion was calculated by using the logit link and applying backtransformation to the sex-specific linear predictors at the end of the study period.A working independence correlation structure was used to model the proportion of HAART-exposed children because of convergence problems with an exchangeable working correlation structure. P value for the two-sided Wald test of no sex difference in the estimated mean or proportion at the end of the study period (no sex difference in linear predictors from the generalized linear model at the end of the study period). 3 Estimated proportion from the generalized linear model expressed as a percentage.\n",
      "    unique: PMC2562173/PMC2562173.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562173.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562173.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562173.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562173.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE 4\n",
      "  caption_text: Regression estimates of per-annum changes in anthropometric and dietary characteristics among 116 HIV-positive children from 1995 to 20041\n",
      "  footer_text: 1 For some variables, the number of male/female subjects (observations) was different from that indicated in the table header. CD4 (%): 45/60 (275/353); sugar: 46/62 (216/273); monounsaturated and saturated fat: 47/66 (284/341); polyunsaturated fat: 45/65 (275/331); dietary fiber: 48/66 (287/350); cholesterol: 48/66 (290/353). OR, odds ratio; HAART, highly active antiretroviral therapy; AMC, arm-muscle circumference; EER, estimated energy requirement; RDA, Recommended Dietary Allowance; AI, adequate intake; USDA, US Department of Agriculture. 2 Sex-specific per-annum changes or ORs and their 95% CIs were estimated with generalized linear models including center, sex, years since the beginning of the study (14 February 1995), and the interaction between years and sex. Estimates were obtained by using generalized estimating equations with an exchangeable working correlation structure to account for potential correlations between repeated measurements in the same individuals. The logit link was used for proportions. A working independence correlation structure was used to model the proportion of HAART-exposed children because of convergence problems with an exchangeable working correlation structure. 3 P value for the 2-sided Wald test of no per-annum trend (zero sex-specific years effect in the generalized linear model). 4 P value for the 2-sided Wald test of no sex difference in per-annum trends (no interaction between years and sex in the generalized linear model). 5 Estimated per-annum OR from the generalized linear model.\n",
      "    unique: PMC2562173/PMC2562173.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562174.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562195.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Migratory behavior of myeloid cells varies with the tumor microenvironment\n",
      "  footer_text: Velocity of c-fms+ cells were computed from four 1-hour-long time-lapse movies of 200×200 μm fields per microenvironment. Fields with normal epithelium and hyperplasia also contained some of the adjacent stroma. The distribution of myeloid cell velocities differed significantly between the microenvironments (P<0.0001, chi-squared test).\n",
      "    unique: PMC2562195/PMC2562195.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Migratory behavior varies with dextran ingestion\n",
      "  footer_text: Velocity of c-fms+, and 10 kDa and 2 MDa dextran-ingesting cells were computed from five fields of tumor-stroma border areas (hyperplasia, early and late carcinomas) in 1-hour-long time-lapse movies. The distribution of cell velocities differed significantly between the cell types (P<0.0001, chi-squared test).\n",
      "    unique: PMC2562195/PMC2562195.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562217.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562195.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562195.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562217.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Percentage of retrogradely labeled neurons in different regions of the VNLL after BDA injections in the CNIC.\n",
      "  footer_text: 1 Cases were classified as low-, mid- or high-frequency according to the location of retrogradely labeled cells in the contralateral cochlear nucleus (see Cant and Benson, 2006b for details).\n",
      "    unique: PMC2562217/PMC2562217.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562219.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562222.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562225.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562226.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Growth Inhibition of Pancreatic Cancer Cell Lines by PP and PSP Analogues Compared with Untreated Control\n",
      "  footer_text: \n",
      "    unique: PMC2562226/PMC2562226.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562228.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562230.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562231.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562237.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562238.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562226.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562239.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562242.xml\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: APPENDIX 1\n",
      "  caption_text: ICD-9 CODES USED FOR IDENTIFICATION OF SUBJECTS\n",
      "  footer_text: \n",
      "    unique: PMC2562242/PMC2562242.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of Admissions for Children With Severe TBI in the 6-State Cohort (N = 2117)\n",
      "  footer_text: \n",
      "    unique: PMC2562242/PMC2562242.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: State and HRR Characteristics\n",
      "  footer_text: \n",
      "    unique: PMC2562242/PMC2562242.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Regionalization of Severe Pediatric TBI According to Presence of High-Level or Level II Centers in HRRs\n",
      "  footer_text: \n",
      "    unique: PMC2562242/PMC2562242.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562242.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562242.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562242.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562244.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562245.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Bland Altman comparison of 2D breath-hold and free-breathing measurements of left ventricular function.\n",
      "  footer_text: SD = standard deviation, FB = free-breathing, BH = breath-hold, EDV = end-diastolic volume, ESV = end-systolic volume, EDM = end-diastolic mass, EF = ejection fraction.\n",
      "    unique: PMC2562245/PMC2562245.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562246.xml\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2562246/PMC2562246.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Photophysical and physiological parameters used in the simulation\n",
      "  footer_text: * On the basis of published reports 4, 5,16, we estimate that the mTHPC concentrations in murine tumors at 3, 6, 24 and 96 h following i.v. injection (0.3 mg kg-1) are 0.3, 0.42, 0.54, and 0.18 μg ml-1, assuming a tissue density of 1 g cm-3. Using these data and methods described in Wang et al.8, we calculate βPDT at 650 nm at the capillary wall to be 0.063, 0.045, 0.021 and 0.006 for the cases of nonuniform mTHPC distribution at 3, 6, 24 and 96 h drug-light intervals, respectively.\n",
      "    unique: PMC2562246/PMC2562246.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562245.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562246.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562246.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562246.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: \n",
      "  footer_text: Mean recurrence-free survival of mTHPC- PDT-treated tumors illuminated with 100 mW cm-2 and 30 J cm-2 reproduced from Fig. 3 of Cramers et al.4 and Fig. 2A of Triesscheijn et al.5. The recurrence-free interval is defined by the time taken for a tumor diameter to increase 2 mm from its treatment size. Tumors that had not regrown within 120 days were defined as cured.\n",
      "    unique: PMC2562246/PMC2562246.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562247.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean Percentage of Presented faces (SD in parentheses) Resulting in Name Errors (Occupation Correctly Remembered but Not Name) and Occupation Errors (Name Correctly Remembered but Not Occupation) for Young and Older Adults in Experiments 1 and 2\n",
      "  footer_text: \n",
      "    unique: PMC2562247/PMC2562247.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean Percentage Correct (SD in parentheses) for Names and Occupations for Young and Older Adults in the Initial Testing Round for Experiments 1 and 2\n",
      "  footer_text: \n",
      "    unique: PMC2562247/PMC2562247.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562248.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Exogenous IGF-1 does not alter surface expression of CD80, CD86 or CD23 by GD or control B cells\n",
      "  footer_text: PBMC were isolated and cultured with or without CpG (2 μg/ml) and IGF-1 (10 ng/ml) for 48 hours and assayed for expression of CD80, CD86, CD23 Each datum point represents percent positive-staining CD19+ B cells ± S.E.M. All comparisons of treatments compared to their respective controls were not significant\n",
      "    unique: PMC2562248/PMC2562248.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562247.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562247.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562248.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562250.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562252.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562253.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562255.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table\n",
      "  caption_text: Distinguishing features of chromosomal translocations that activate cellular oncogenes or generate novel fusion genes\n",
      "  footer_text: \n",
      "    unique: PMC2562255/PMC2562255.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562256.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List of measured INEE values for various detector systems and the normalized INEE for 150 μm × 150 μm pixel area.\n",
      "  footer_text: \n",
      "    unique: PMC2562256/PMC2562256.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562257.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562255.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562256.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562257.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562262.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Association studies of STAT4 with autoimmune diseases. Allele frequencies and association statistics are given for the disease-associated SNP rs7574865\n",
      "  footer_text: * The 1039 SLE cases reported in Remmers et al. are a subset of the 1380 cases reported in Hom et al.\n",
      "    unique: PMC2562257/PMC2562257.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562258.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562262.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mass and fluoride concentrations (mean ± standard error) in saliva samples obtained overnight after use of a calcium lactate pre-rinse followed by a 228 μg/g F rinse, a 228 or 912 μg/g F rinse alone (with no pre rinse), or a distilled water rinse.\n",
      "  footer_text: For saliva fluoride, means followed by different superscript letters have a statistically significant difference (Holm-Sidak pairwise multiple comparison test, p < 0.05). The 912 μg/g F rinse data was obtained 6 months later and hence was not included in the statistical analysis. * Data were obtained from centrifuged saliva.\n",
      "    unique: PMC2562262/PMC2562262.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562263.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Redox Isomerization under Indium Trichloride Cocatalysis\n",
      "  footer_text: Reactions were performed on 1 mmol scale at 0.25M with 5% catalyst 2 and 40% InCl3 in THF at reflux. For primary alcohols 5% NH4PF6 and 5% Et3NHPF6 was added. For secondary alcohols 10% Et3NHPF6 was added.\n",
      "    unique: PMC2562263/PMC2562263.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Isomer ization under Indium Triflate Cocatalysis\n",
      "  footer_text: a) Reactions were performed on 1 mmol scale in THF at reflux. Yield refers to isolated yields. b) The product in this case is 4- keto-n-decanal.\n",
      "    unique: PMC2562263/PMC2562263.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562263.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562263.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562265.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562264.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562265.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: 13C and 1H NMR data for compounds 1 and 2, δ in ppm (quantity, multiplicity and J/Hz).\n",
      "  footer_text: a d6-acetone, 100.6 MHz. b d6-acetone, 400 MHz.\n",
      "    unique: PMC2562265/PMC2562265.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Cytotoxicity assay of moromycins A (1) and B (2) in comparison with saquayamycin B (3) against human lung H-460 and human breast MCF-7 cancer cell lines (given are GI50, TGI and LC50 values in µmol). The data represent the average of three measurements.\n",
      "  footer_text: GI50= 50% inhibition of cell growth; TGI=total growth inhibition; LC50=50% lethal concentration.\n",
      "    unique: PMC2562265/PMC2562265.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562266.xml\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: \n",
      "  caption_text: Appendix Word lists used in the baseline and experimental categorization tasks. Each list contained equal numbers of words referring to people, food, and animals\n",
      "  footer_text: \n",
      "    unique: PMC2562266/PMC2562266.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562265.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562266.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562266.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562266.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Individual hearing thresholds (in dB HL) for the children with minimal HL as well as the average hearing thresholds (and standard deviation) for both the children with HL and the children with NH. The values in parentheses were not included in the calculation of averages. The amplification status of the children with minimal HL is also given (B=both ears, R=Right ear, L=Left ear, N=No amplification). Hearing losses are listed from mild to high-frequency to unilateral. The hearing losses of a few children were considered combinations of two configurations and are listed at the boundaries of each category.\n",
      "  footer_text: \n",
      "    unique: PMC2562266/PMC2562266.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Average baseline and experimental dot rates (dots per minute) for each multitasking condition for the children with normal hearing (NH) and the children with minimal hearing loss (HL). Values in parentheses are standard error. Numbers in italics are the result of paired samples t-tests (t, p, and Cohen’s d). Note that the same games were played in each listening condition and that the baseline dot rates were not obtained in noise.\n",
      "  footer_text: * Significantly different from baseline as indicated by paired samples t-test (p ≤ 0.017, Bonferroni adjustment).\n",
      "    unique: PMC2562266/PMC2562266.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Average word categorization (percent correct) for the baseline and multitasking conditions for the children with normal hearing (NH) and the children with minimal hearing loss (HL). Values in parentheses are standard error. Numbers in italics are the result of paired samples t-tests (t, p, and Cohen’s d)\n",
      "  footer_text: * Significantly different from baseline as indicated by paired samples t-test (p ≤ 0.017, Bonferroni adjustment).\n",
      "    unique: PMC2562266/PMC2562266.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562271.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562266.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562271.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562271.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562271.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562271.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Results of switching from injection to CSII therapy in non-randomized pediatric studies\n",
      "  footer_text: \n",
      "    unique: PMC2562271/PMC2562271.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Results of Randomized clinical trials comparing CSII and MDI in patients with type 1 DM\n",
      "  footer_text: * p <0.05\n",
      "    unique: PMC2562271/PMC2562271.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Indications for use of CSII in Pediatrics (adapted from Reference 66)\n",
      "  footer_text: \n",
      "    unique: PMC2562271/PMC2562271.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Pump Options and Features\n",
      "  footer_text: \n",
      "    unique: PMC2562271/PMC2562271.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Correction Factors and Carbohydrate Ratio Based on Age for Initial Pump Settings\n",
      "  footer_text: * Often a lower carbohydrate ratio is needed for breakfast (-2 grams for breakfast carb ratio). For example for children <3, start breakfast carbohydrate ratio at 43 grams.\n",
      "    unique: PMC2562271/PMC2562271.xml::T5-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562272.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562273.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics of all patients.\n",
      "  footer_text: Abbreviations: ECOG, Eastern Cooperative Oncology Group; UWCCC, University of Wisconsin Paul P. Carbone Comprehensive Cancer Center; WON, Wisconsin Oncology Network.\n",
      "    unique: PMC2562273/PMC2562273.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Maximal severity of toxicities per phase I patient and type regardless of attribution to study drugs occurring in ≥ 20% patients.\n",
      "  footer_text: * National Cancer Institute Common Toxicity Criteria version 2.0 Abbreviations: BID, twice daily; n, number; q, every; AM, morning; PM, evening.\n",
      "    unique: PMC2562273/PMC2562273.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Maximal severity of toxicities per phase II patient and type regardless of attribution to study drug occurring in ≥ 20% of patients (n=30).\n",
      "  footer_text: * National Cancer Institute Common Toxicity Criteria, version 2.0. ^ Two patients had cataracts & One patient was status/post cerebrovascular accident and another patient had cataracts. Abbreviations: n, number.\n",
      "    unique: PMC2562273/PMC2562273.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562273.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562273.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562273.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562273.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Efficacy.\n",
      "  footer_text: Abbreviations: TTP, time-to-progression; OS, overall survival; CI, confidence interval.\n",
      "    unique: PMC2562273/PMC2562273.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562281.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562285.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic characteristics of 180 syphilis patients and of the overall clinical cohort.\n",
      "  footer_text: NOTE. Data are no. (%) of patients, unless otherwise indicated.\n",
      "    unique: PMC2562285/PMC2562285.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Clinical characteristics of 231 cases of syphilis.\n",
      "  footer_text: NOTE. Data are no. (%) of cases, unless otherwise indicated. IQR, interquartile range; RPR, rapid plasma reagin. a Includes primary syphilis, secondary syphilis, and early latent syphilis. b At the time of syphilis diagnosis. c Any time during follow-up.\n",
      "    unique: PMC2562285/PMC2562285.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multivariable predictors of serologic failure overall and by type of failure (4-fold increase in titers vs. lack of 4-fold decrease and early failure vs. late failure) among 180 HIV-infected patients.\n",
      "  footer_text: NOTE. The multivariable model adjusted for sex, race, previous history of syphilis, baseline RPR titer, and type of syphilis therapy received. The Andersen-Gill method was used to adjust for the multiple failure-instance data (180 subjects had 231 cases of syphilis and 71 instances of serologic failure overall). NA, not applicable; Ref, reference; RPR, rapid plasma reagin test. a Early failure is defined as serologic failure occurring ≤365 days after therapy for early stage syphilis and ≤730 days after therapy for the late stages. b Late failure is defined as serologic failure occurring >365 days after therapy for early stage syphilis and >730 days after therapy for the late stages. c Defined as <400 copies/mL. d Defined as >50% increase in CD4 cell count from baseline measured as a continuous change during follow-up. e Ninety-seven percent of macrolide therapy involved the administration of azithromycin (mean dosage, 1100 mg/week); 3% involved clarithromycin (mean dosage, 7000 mg/week). f NA denotes a lack of 4-fold decrease, which, by definition, precludes seroreversion.\n",
      "    unique: PMC2562285/PMC2562285.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562285.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562285.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562285.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562285.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562288.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Multivariable predictors of rapid plasma reagin (RPR) seroreversion in the HIV cohort.\n",
      "  footer_text: NOTE. The multivariable model adjusted for sex, race, previous history of syphilis, type of syphilis therapy received. Ref, reference. a Early syphilis includes primary, secondary, and early latent stages of syphilis. b Defined as <400 copies/mL. c Ninety-seven percent of macrolide therapy involved the administration of azithromycin (mean dosage, 1100 mg/week); 3% involved clarithromycin (mean dosage, 7000 mg/week).\n",
      "    unique: PMC2562285/PMC2562285.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562288.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Subjects and results for the current experiment.\n",
      "  footer_text: * Independent sample t-test, two-tailed\n",
      "    unique: PMC2562288/PMC2562288.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562290.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinical characteristics of affected men (n=542)\n",
      "  footer_text: * Column subtotals do not sum to 542 due to missing data. † Median and [interquartile range]. ‡ Number and (percentage) of men with prostate cancer who underwent radical prostatectomy.\n",
      "    unique: PMC2562290/PMC2562290.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Major allele frequencies and FBAT results (n=403 families)\n",
      "  footer_text: Note: rs4430796 and rs7501939 are in strong linkage disequilibrium (r2=0.68) * Based on the combined sample of men with and without prostate cancer and an additive genetic model for the major allele. † Number of informative families.\n",
      "    unique: PMC2562290/PMC2562290.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of odds ratios (ORs) for 17q SNPs tested in the University of Michigan (UM), Gudmundsson et al., and the Cancer Genetic Markers of Susceptibility (CGEMS) studies\n",
      "  footer_text: Note: Odds ratios were computed under an additive genetic model. * Before conducting the conditional logistic regression analyses, we excluded 38 men (from 10 families) who were not brothers of the index case, resulting in a reduced sample size of 977 men and 604 DSPs. † Logistic regression as reported by Gudmundsson et al. (2007), all samples combined (1,501 cases and 11,289 controls) ‡ Logistic regression, unadjusted for covariates (1,155 cases and 1,106 controls)\n",
      "    unique: PMC2562290/PMC2562290.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562290.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562290.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562290.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562291.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562291.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562291.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562291.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Recurrent versus non-recurrent diarrhea in relation to height and weight gains.\n",
      "  footer_text: * >30% prevalence over subsequent 2 cm ** p=0.1 (21% less) *** p=0.01 (43% less) Data from Schorling and Guerrant (1990)29 and Schorling et al. (1990)39\n",
      "    unique: PMC2562291/PMC2562291.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Emerging causes of persisting diarrhea in shantytowns in Fortaleza northeastern Brazil.\n",
      "  footer_text: * p<0.05 ** p<0.02 Data from Steiner et al. (1998),12 Lima et al. (2000),61 Fang et al. (1995),195 and Moore et al. (2000)196\n",
      "    unique: PMC2562291/PMC2562291.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Effect of malnutrition on both the incidence and duration of diarrhea.\n",
      "  footer_text: * Wilcoxon rank sum test. Also valid for moderate MN (<90% HAZ) Data from Guerrant et al. (1992)11 and Schorling et al. (1990)39\n",
      "    unique: PMC2562291/PMC2562291.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Effects from early childhood diarrhea/enteric infections on lasting disability.\n",
      "  footer_text: Abbreviations: AASS, age at starting school; AFG, age for grade; EAEC, enteroaggregative E. coli; HAZ-2, height for age Z score at 2 years; HST, Harvard Step Test scores; TONI, Test of Nonverbal Intelligence; WISC, Wechsler Intelligence Scale for Children; →, predisposes to.\n",
      "    unique: PMC2562291/PMC2562291.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562292.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562294.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562291.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562295.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562299.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562295.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table\n",
      "  caption_text: Disaccharide composition analysis of CS-A, CS-C, and chondro-4-sulfatase-treated CS-A Unsaturated disaccharides generated by digestion with cABC were analyzed by anionexchange HPLC after labeling with the fluorophore 2AB as described in Materials and Methods section. The values obtained from 3 independent experiments were used to calculate the percentage of each unsaturated disaccharide (mean ± SD).\n",
      "  footer_text: a ΔDi-0S=DHexUA-GalNAc; ΔDi-6S=DHexUA-GalNAc(6-O-sulfate); ΔDi-4S=DHexUA-GalNAc(4-O-sulfate); ΔDi-2,6S=DHexUA(2-O-sulfate)-GalNAc(6-O-sulfate); ΔDi-4,6S=DHexUA-GalNAc(4,6-O-disulfate); ΔDi-2,4,6S=DHexUA(2-Osulfate)-GalNAc(4,6-O-sulfate).\n",
      "    unique: PMC2562295/PMC2562295.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562297.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562299.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Effects of Eribulin and ER-076349 on Centromere Dynamics\n",
      "  footer_text: Between 6 and 11 pairs of centromeres from between 5 and 7 cells were measured for each condition. *, **, *** Values were significantly different from control values at the 95%, 98%, or 99% confidence level, respectively (Student’s t test). § The mean for all tracings of the two maximal separations (peaks) or the two minimal separations (valleys) during each 5-min tracing for each condition.\n",
      "    unique: PMC2562299/PMC2562299.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562300.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: The time scales τi (ns), for models with 120, 6 and 14 states, calculated using τ = 100 ps.\n",
      "  footer_text: \n",
      "    unique: PMC2562300/PMC2562300.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562300.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562300.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562301.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE II\n",
      "  caption_text: The characterization of the Markov models with 2, 4 and 6 states in terms of the dihedral angle conformations. The lifetimes of the states, from eq (32), are in bold.\n",
      "  footer_text: a This macrostate contains two microstates (the two black points in Figures 3a and 3b), which have very similar values for all the five dihedrals, μi ≈ (−170°, 160°, −95°, 75°, −100°).\n",
      "    unique: PMC2562300/PMC2562300.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE III\n",
      "  caption_text: Populations (%) and lifetimes (ns) of the 14-state Markov model, normalized separately for conformations with χ3 ≈ −90° (states 1 to 7), and χ3 ≈ 90° (states 8 to 14). The states with χ1 ≈ 60° are indicated with a star.\n",
      "  footer_text: a Used in Figures 3 and 4.\n",
      "    unique: PMC2562300/PMC2562300.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: TABLE IV\n",
      "  caption_text: Parameters used in the simulation of the ESR spectra from the MD and the Markov chain trajectories.\n",
      "  footer_text: a Twice as many points were used with the Markov trajectories. b Four times more points were used with the Markov trajectories.\n",
      "    unique: PMC2562300/PMC2562300.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562301.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Relationship Between PPy Film Thickness and Charge Passed per Area\n",
      "  footer_text: * Calculated from Equation [4] (Stankovic, et al. 1994).\n",
      "    unique: PMC2562301/PMC2562301.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562302.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562305.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562314.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562316.xml\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table A1\n",
      "  caption_text: Substance use patterns, by substance\n",
      "  footer_text: Moderate/heavy alcohol use = 1 if respondent reported being drunk several times or more often in the last 12 months. Moderate/heavy marijuana (or other illicit drug) use = 1 if caregiver reported using marijuana (or other illicit drug) several times or more often in the last 12 months.\n",
      "    unique: PMC2562316/PMC2562316.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table A2\n",
      "  caption_text: Relative Risk Ratios from Models in Table 2\n",
      "  footer_text: a -Numbers in [] show the marginal effect compared to women with light/no use in both waves. Thus, women with increased use are 8 percentage points more likely to be detached compared to women with light/no use in both waves. * = p < .10 ** = p < .05 *** =p<.01\n",
      "    unique: PMC2562316/PMC2562316.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table A3\n",
      "  caption_text: Odds Ratios from models in Table 3\n",
      "  footer_text: a Numbers in [] show the marginal effect compared to women with light/no use in both waves. Thus, women who reduced substance use between waves are 22 percentage points more likely to lose insurance compared to women with light/no use in both waves. * = p < .10 ** = p < .05 *** =p<.01\n",
      "    unique: PMC2562316/PMC2562316.xml::T7-0\n",
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table A4\n",
      "  caption_text: Odds Ratios from models in table 4\n",
      "  footer_text: BSI=Brief Symptom Inventory, CBCL=Child Behavior Checklist, TANF=Temporary Assistance to Needy Families. a Numbers in [] show the marginal effect compared to women with light/no use in both waves. Thus, women who increased substance use between waves are 18 percentage points less likely to report that child behavior improved between waves. * = p < .10 ** = p < .05 *** =p<.01\n",
      "    unique: PMC2562316/PMC2562316.xml::T8-0\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of Female Caregivers at Wave 1 and Wave 2\n",
      "  footer_text: a Weights make population representative of Boston, Chicago, and San Antonio neighborhoods sampled. b Races other than Black or White accounted for 2.7% of the sample, and no women in the study were of black race and Hispanic ethnicity. c Education is reported for wave 2 only because measured education is not directly comparable between waves 1 and 2, as described in footnote 17 of the survey documentation publication, “Welfare, Families, and Children: A Three-City Study,” Fomby, P., Estacion, A. and Moffit, R., Johns Hopkins University, 2003, (available at http://web.jhu.edu/threecitystudy/images/CharacteristicsReport.pdf). d BSI=Brief Symptom Inventory, CBCL=Child Behavior Check List.\n",
      "    unique: PMC2562316/PMC2562316.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T8-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Wave 2 work and income assistance according to change in substance use between waves 1 and 2\n",
      "  footer_text: * = p < .10 ** p < .05, ***p<.01 for test of share in this category compared with the “Light/no use” category in multinomial logit models controlling for black race, Hispanic ethnicity, and high school degree, marital and cohabitation status, and age (<30 or 40+) at wave 2 interview and wave 1 work and TANF status.\n",
      "    unique: PMC2562316/PMC2562316.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Health insurance and patterns of substance use between waves 1 and 2\n",
      "  footer_text: * = p < .10 ** p < .05 *** p<.01 for test of difference between this category compared with the “No change No/light use” category in logistic regression models controlling for high school degree, marital and cohabitation status, black race, Hispanic ethnicity, and age (<30 or 40+) at wave 2 and wave 1 work and TANF status. Estimate of “lost health insurance” applies to 1150 caregivers with health insurance in wave 1, while “gained health insurance” model applies to 473 caregivers without health insurance in wave 1.\n",
      "    unique: PMC2562316/PMC2562316.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Well-being and patterns of substance use between waves 1 and 2\n",
      "  footer_text: * = p < .10 ** = p < .05 *** =p<.01 for test of difference between this category compared with the “No change No/light use” category in logistic regression models controlling for high school degree, marital and cohabitation status, black race, Hispanic ethnicity, and age (<30 or 40+) at wave 2 interview and wave 1 work and TANF status.\n",
      "    unique: PMC2562316/PMC2562316.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562318.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means and Standard Deviations for the Big Five T Scores by Age Categories\n",
      "  footer_text: Note: BHPS = British Household Panel Study; GSEOP = German Socio-Economic Panel Study; T scores were created by standardizing scores to the mean and SD for individuals ages 30 to 34 within each sample.\n",
      "    unique: PMC2562318/PMC2562318.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Regression Models Linking Age to the Big Five\n",
      "  footer_text: Note: Unstandardized coefficients reported. Standard errors reported inside parentheses. Age was mean-centered within each sample (Age in BHPS = 45.50; Age in GSOEP = 46.09). More complicated models involving age polynomials were selected only when the inclusion of the higher order term improved overall model fit at F > 25.00. Coefficients for age were reported from the final selected model within each sample. See text for complete details and justification of the model selection strategy.\n",
      "    unique: PMC2562318/PMC2562318.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562316.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562318.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562318.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562318.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562322.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Effect Sizes for Gender and Education Differences by Age Categories\n",
      "  footer_text: Note: BHPS = British Household Panel Study; GSEOP = German Socio-Economic Panel Study. Effect sizes for gender were calculated so that positive scores indicated that women scored higher than men. Effect sizes for education were calculated so that positive scores indicated that more educated individuals scored higher than less educated individuals.\n",
      "    unique: PMC2562318/PMC2562318.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562322.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Kinetic and thermodynamic fitting parameters for wild-type E:THF:NADP+ 15N R2 relaxation dispersion curves at different pH and temperatures\n",
      "  footer_text: 1 not determined; although conformational exchange is still apparent in the cofactor binding cleft, the R2 relaxation dispersion curves are not of sufficient quality to determine Δw, kex or pApB values\n",
      "    unique: PMC2562322/PMC2562322.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562323.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562325.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Leakage due to the voxel-shift operation (see Figure 4). The percentages represent the relative deviation of the reconstructed peak-areas from their nominal values based on the spatial overlap of a center voxel of the imaging grid with the phantom partitions. The “–” sign indicates “leakage out” and the “+” —“ leakage in”.\n",
      "  footer_text: \n",
      "    unique: PMC2562325/PMC2562325.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Leakage due to interpolation and voxel-shift operations (see Figure 5). The percentages represent the relative deviation of the reconstructed peak-areas from their nominal values based on the spatial overlap of a center voxel of the imaging grid with the phantom partitions. The “–” sign indicates “leakage out” and the “+” —“ leakage in”.\n",
      "  footer_text: \n",
      "    unique: PMC2562325/PMC2562325.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562329.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562332.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562325.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562325.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562336.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562336.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562336.xml::TA1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562336.xml::TA2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562336.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List 2 recall by age group, condition, and experiment. Values in brackets represent standard deviations\n",
      "  footer_text: \n",
      "    unique: PMC2562336/PMC2562336.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Intrusion Errors as a Function of List, Age and Cue\n",
      "  footer_text: \n",
      "    unique: PMC2562336/PMC2562336.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: TA1\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2562336/PMC2562336.xml::TA1-0\n",
      "  ----------------------------------------\n",
      "  id: TA2\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2562336/PMC2562336.xml::TA2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562337.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562338.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562339.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562340.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample Demographic and Diabetes-Related Characteristics (N=81)\n",
      "  footer_text: Note: For diabetes quality of life, parenting style, conflict, and parent responsibility-sharing, higher scores indicate higher levels of that particular dimension; for example, greater quality of life, greater conflict, greater parent responsibility.\n",
      "    unique: PMC2562340/PMC2562340.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlation of demographic and disease characteristics, diabetes-specific parent-child behaviors, and diabetes quality of life\n",
      "  footer_text: * P ≤ .05; ** P < .01; † P < .10\n",
      "    unique: PMC2562340/PMC2562340.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multiple regression of diabetes quality of life at baseline and 12-month follow-up on baseline parenting style\n",
      "  footer_text: ‡ Standardized beta coefficients.\n",
      "    unique: PMC2562340/PMC2562340.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562341.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562340.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562340.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562340.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and context data for 305 persons who lived and/or worked around nonhuman primates, Asia*\n",
      "  footer_text: * SFV+, simian foamy virus positive. † Predominant form of human-nonhuman primate contact at the time of the study.\n",
      "    unique: PMC2562341/PMC2562341.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Persons at high risk for SFV, Asia*\n",
      "  footer_text: * WB, Western blot; SFV, simian foamy virus; NA, not applicable. † 65/213 serum samples were bioplex reactive and further screened with glutathione S-transferase-Gag ELISA.\n",
      "    unique: PMC2562341/PMC2562341.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Exposure characteristics of SFV-positive persons who had had contact with nonhuman primates, Asia*\n",
      "  footer_text: * SFV, Simian foamy virus; NA, not applicable. All persons were confirmed SFV positive by Western blot.\n",
      "    unique: PMC2562341/PMC2562341.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562342.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Numbers and characteristics of monkeys included in the study at each site\n",
      "  footer_text: CR: Caloric restriction group; AL: Ad libitum control group\n",
      "    unique: PMC2562342/PMC2562342.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562343.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562341.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562341.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562341.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562342.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Concentrations of Chemokines and Cytokines in Conditioned Media Samples from Mono- and Co-cultures Based on Multiplexed Antibody Bead Assays\n",
      "  footer_text: The illustrated results are from one experiment and are representative of results from three experiments. Values are the means of triplicate samples. (−) designates undetectable levels. (**) designates p value<0.01. (*) designates p value<0.05.\n",
      "    unique: PMC2562343/PMC2562343.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562344.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patterns of Correspondence Across Scores (Kendall’s tau, N = 17)\n",
      "  footer_text: ~ trend (p < .10), * p ≤ .05 level, ** p ≤ .01 level\n",
      "    unique: PMC2562344/PMC2562344.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562345.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562346.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562347.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562343.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562344.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562347.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562348.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Spectral karyotyping analysis of neu-expressing cell lines TUBO, Bam1a, Bam IR-5, D2F2/neu, and D2F2. Clonal chromosome aberrations (CCAs), non-clonal chromosome aberrations (NCCAs), and chromosome fragments were summarized\n",
      "  footer_text: t = translocation i = isochomosome\n",
      "    unique: PMC2562347/PMC2562347.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562348.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of the patient demographics, treatment, tumor site, histopathology, and culture information. All 13 tissue samples were adenocarcinomas isolated from patients with resectable colorectal hepatic metastases. Long-term cultures were established from most tumor cell samples. Long-term cultures were those ranging from several passages to more than 10 passages. In vitro differentiation was defined as the presence of pleiomorphic maturation and the expression of lineage markers. ND: not determined and chemo: chemotherapy treatment.\n",
      "  footer_text: \n",
      "    unique: PMC2562348/PMC2562348.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Percentage of cells exhibiting the chromosomal aberrations found in parental and clonal cultures of Tu-12 and Tu-21. Twenty metaphase cells were analyzed per tumor culture. Structural changes that were seen more than once, numerical gains of chromosomes at least seen twice, and numerical chromosome loss seen three or more times were reported in this table. Add, additional unidentifiable chromatin material attached to and replacing the chromatin distal to that breakpoint; del, deletion of a chromosome segment; i, isochromosome; \"+1~5 mar\", at least two of the cells studied from that sample contained between one and five unidentifiable marker chromosomes. Clonally abnormal chromosomes in all three related cultures confirm the relationship between the parent and her descendants. These would be the chromosomal aberrations found in all three samples. While the nomenclature describing the del(5q) and the marker chromosomes appear the same for the two unrelated tumors, we do not believe that they are actually identical abnormalities in the two tumors at the molecular level (See Supplementary Material for more information).\n",
      "  footer_text: \n",
      "    unique: PMC2562348/PMC2562348.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562348.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562352.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562349.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562350.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562352.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Antiviral Activities vs. Yellow Fever Virus and Cytotoxicities of the Original Hits and Second Generation Compounds\n",
      "  footer_text: a Note: The EC50 is the concentration of the compound resulting in a 50% inhibition in virus production. b The CC50 is the concentration of the compound causing a 50% growth inhibition of uninfected BHK cells. c TI is the therapeutic index and is equal to CC50/EC50. d A short dash indicates that the value was not determined.\n",
      "    unique: PMC2562352/PMC2562352.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562353.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562354.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562356.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Significant genes with expression that was associated with gemcitabine or AraC cytotoxicity (IC50 values). R values represent correlation coefficients. Q values represent the false discovery rate.\n",
      "  footer_text: \n",
      "    unique: PMC2562356/PMC2562356.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Intracellular metabolites determined by HPLC in lymphoblastoid cells after incubation with (A) AraC or (B) gemcitabine. Metabolites are μM/5 × 106 cells.\n",
      "  footer_text: * = indicates p < 0.05 between sensitive and resistant cell lines.\n",
      "    unique: PMC2562356/PMC2562356.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562419.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of isolated clones and overexpression phenotypes. Phenotype categories: AD: Anterior Defects, CED: Convergent Extension Defects, DD: Dorsal Defects, GD: Gastrulation Defects, K: Kinked, NM: Neural Mispatterning, PD: Pigmentation Defects, SA: Secondary Axis, SB: Spina Bifida, TD: Trunk Defects, V: Ventralized.\n",
      "  footer_text: \n",
      "    unique: PMC2562419/PMC2562419.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562356.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562356.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562419.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562423.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562426.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562427.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Arrestin levels in mutant backgrounds Values determined by densitometry of Western blots (mean ± SD, normalised to levels in wild type using tubulin as standard), see methods and Supplemental Figure 1 for further details.\n",
      "  footer_text: \n",
      "    unique: PMC2562427/PMC2562427.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mutants used\n",
      "  footer_text: \n",
      "    unique: PMC2562427/PMC2562427.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562427.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562427.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562438.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562438.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562431.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562438.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Primers used in Vamp1 PCR experiments\n",
      "  footer_text: Forward and reverse primers used for the identification of Vamp1 variants in the brain and cochlear nucleus. The rat Vamp1nv (specific) primer set spanned introns and generated a 110bp sequence unique to this variant.\n",
      "    unique: PMC2562438/PMC2562438.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primer-Probe Sequences used in Vamp1 Splice Variant Taqman® Real-Time RT-PCR experiments\n",
      "  footer_text: Primer-probe sets used for real-time RT-PCR experiments. These were used to identify the relative levels of each variant among the subdivisions. Primers were designed around the unique splice sites to be specific for each variant.\n",
      "    unique: PMC2562438/PMC2562438.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562451.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562456.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Ar-Bb interaction energies (kJ·mol-1)\n",
      "  footer_text: † interaction energies between the aromatic side chain and the Ac-N1H group (Figure 1a, Scheme 1S). ‡ interaction energies between the aromatic side chain and the Gly-Gly-NHMe and the C1=O groups (Figure 1a, Scheme 1S).\n",
      "    unique: PMC2562456/PMC2562456.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562456.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562456.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562456.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562504.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Bb-Bb interaction energies (kJ·mol-1)\n",
      "  footer_text: † the peptide bonds are modeled as acetamide (Figure 1b). ‡ interaction energies of H-bonds are shown in bold. § a bifurcated H-bond where the C0 carbonyl oxygen is the acceptor atom and the N3 and N4 hydrogens are the donor atoms; a special fragmentation was used (Supplementary material, Figure 2S).\n",
      "    unique: PMC2562456/PMC2562456.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Intramolecular interaction energies (kJ·mol-1) in FGG, WGG and YGG\n",
      "  footer_text: † weakly polar interaction energy ‡ includes the energy of a Pb1-Pb2 H-bond calculated by the rotation method (Supplementary material, Figure 3S). § includes the energy of a bifurcated H-bond calculated by a special fragmentation (Supplementary material, Figure 2S).\n",
      "    unique: PMC2562456/PMC2562456.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562504.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Prediction and conserve of sumoylation residues in papillomavirus E2\n",
      "  footer_text: \n",
      "    unique: PMC2562504/PMC2562504.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562522.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic Characteristics stratified by Source of Recipient Data (Number in parenthesis is %, unless otherwise noted)\n",
      "  footer_text: \n",
      "    unique: PMC2562522/PMC2562522.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Donor Characteristics (n=512)\n",
      "  footer_text: 1 ECD defined as donor age >60; or donor age >50 with any 2 of the following donor criteria: (1) terminal serum creatinine >1.5 mg/dl, (2) hypertension, or (3) death due to CVA.\n",
      "    unique: PMC2562522/PMC2562522.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Outcomes post-transplantation By Source of Recipient Information\n",
      "  footer_text: \n",
      "    unique: PMC2562522/PMC2562522.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Adjusted and unadjusted association with DGF for inflammation and apoptosis related Polymorphisms [ND1]\n",
      "  footer_text: Adjusted for cold ischemia time, recipient race, extended criteria donor, donor cause of death, donor race, and source of DGF information (medical record or CMS claims or UNOS forms). The (GT)n SNP is categorized as short (S for ≤ 27 repeats) vs long repeats (L for >27 repeats * denotes p< 0.05 † rs3091257 has been merged into rs1800628.[ND3]\n",
      "    unique: PMC2562522/PMC2562522.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562525.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562531.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562532.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562533.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562544.xml"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562522.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562522.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562522.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "PMC0025XXXXX/PMC2562546.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Somatic signs of morphine withdrawal in Narp KO and WT mice Three days after morphine pump implantation, Narp KO and WT mice were administered a single dose of saline in a novel environment. The following day mice were administered a single dose of naltrexone 250ug/kg sc in the same chamber and were monitored for 30 minutes. The table shows the mean number of somatic morphine withdrawal signs for each genotype after saline and naltrexone administrations\n",
      "  footer_text: Note: 1 Based on 2-way ANOVA that included Injection (Naltrexone versus Saline) and Genotype (WT versus KO) as within and between comparisons. Significant when p <0.05.\n",
      "    unique: PMC2562546/PMC2562546.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562547.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562548.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Risk of developing second primary breast cancer associated with ATM gene carrier status.\n",
      "  footer_text: * Adjusted for exact age at diagnosis of the first primary and counter-matching weight. Common and Rare models also adjusted for carriers of other remaining ATM variants. † 95% CI is 95% confidence interval. ‡ Common variants are defined as those carried by greater than or equal to 1% of the WECARE Study participants. Rare variants are those carried by fewer than 1% of the participants.\n",
      "    unique: PMC2562548/PMC2562548.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Risk of developing second primary breast cancer associated with rare ATM variants.\n",
      "  footer_text: * Adjusted for exact age at diagnosis of the first primary, countermatching weight and for carriers of the other remaining ATM variants. † 95% CI is 95% confidence interval. ‡ Meeting one or more of the following criteria: (1) changes predicted to result in truncation of the ATM protein whether by direct termination or frameshifting, (2) changes affecting the two highly conserved nucleotides flanking exons that direct splicing, (3) changes predicted to result in amino acid substitutions for which there is documented evidence of both a deleterious effect on ATM function and identification in diagnosed A-T patients, or (4) changes documented as AT causing in the ATM Mutation Database. § Defined as by SIFT: Variants with normalized probabilities less than 0.05 are predicted to be deleterious, while those greater than or equal to 0.05 are predicted to be tolerated.. Results for missense variants are adjusted for other variants.\n",
      "    unique: PMC2562548/PMC2562548.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562546.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562548.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562548.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562548.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Risk of developing second primary breast cancer associated with common ATM variants*\n",
      "  footer_text: * Variants carried by more than 1% of the WECARE Study subjects. † Variants indicated relative to the reference sequence for the ATM Mutation Database (http://chromium.liacs.nl/lovd/refseq/ATM_codingDNA.html). Nomenclature as recommended by the Human Variome Project. ‡ rs numbers are provided for those SNPs currently listed in dbSNP. § Adjusted for exact age at diagnosis of the first primary, countermatching weight and for carriers of the other remaining ATM variants so that the rate ratio is relative to those for wildtype for ATM variants. ║ 95% CI is 95% confidence interval.\n",
      "    unique: PMC2562548/PMC2562548.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562550.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Number (M of Four Tests ± SEM) of Contacts and Anogenital Sniffs\n",
      "  footer_text: Note. Conducted in 45-min tests with WT controls and SF-1 KO animals without sex steroid replacement tested with stimulus females. WT = wild type; f = female; m = male; KO = knockout. * Significantly different between genotypes: p < .01. ** Significantly different between genotypes: p < .001.\n",
      "    unique: PMC2562550/PMC2562550.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number (M of Four Tests ± SEM) of Anogenital Sniffs\n",
      "  footer_text: Note. Conducted in tests adjusted to 15 min with TP treated WT controls and SF-1 KO animals. WT = wild type; f = female; m = male; KO = knockout. * Significantly different between genotypes: p < .05. ** Significantly different between genotypes: p < .01. *** Significantly different between genotypes: p < .001.\n",
      "    unique: PMC2562550/PMC2562550.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562556.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Meta-analyses of allelic association studies for 12 schizophrenia candidate genes\n",
      "  footer_text: a Polymorphsims and their minor alleles were annotated on the SchizophreniaGene of the Schizophrenia Research Forum (http://www.schizophreniaforum.org/res/sczgene/default.asp). b Meta-analyses were performed for samples from all populations, Europeans, and Asians, respectively. c Numbers of Family-based (Fam, numbers of families are shown in the parentheses) and Case-Control (CC) association studies included in the meta-analyses. d Minor allele frequencies based on case-control association data, numbers of cases and controls are shown in the parentheses. e I2 was employed to assess between-study heterogeneity. I2 > 50% indicates evidence for significant between-study heterogeneity. f Overall Odds Ratios (OR) and their 95% Confidence Intervals (CI) were based on the random-effects model if I2 = 50%, otherwise based on the fix-effects model (I2 < 50%). g P(Z), Z test was used to detect the significance of the overall OR. P-values < 0.05 are shown in bold type. h Associations remain significant after correction for the number of meta-analyses for each gene. Underlined are obvious different results (significant gene-wide associations appear or disappear) of meta-analyses including or excluding family-based association data in Caucasian and all populations. The results excluding family data are similar to those of \"all excluding HWE deviations\" in the Schizophrenia Research Forum. Note that meta-analyses were performed for polymorphisms with at least one family-based association study, except for those in the DAOA gene, which have been reported recently (Shi et al., Schizophr. Res. 2008; 98: 89–97). NA: not applicable.\n",
      "    unique: PMC2562556/PMC2562556.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562550.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562550.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562556.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562557.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562558.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562560.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562562.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline characteristics of participants in the Women’s Health Study according to MTHFR 677C>T genotype (N=25,001) *\n",
      "  footer_text: * data are expressed as percentages unless otherwise stated. † p-values for chi-square test for categorical variables, and Kruskal-Wallis test for continuous variables; comparing no history of migraine vs. any history of migraine. IQR: interquartile range. Proportions may not add up to 100 due to rounding or missing values.\n",
      "    unique: PMC2562562/PMC2562562.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562562.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562562.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562562.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562562.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Age- and multivariable adjusted odds ratios (OR) and 95% confidence intervals (95% CI) for migraine according to MTHFR 677C>T polymorphism\n",
      "  footer_text: * controlling for: age, body mass index, diabetes, physical activity, hormone replacement therapy, oral contraceptive use, history of hypertension, alcohol consumption, smoking categories, and family history of myocardial infarction. † controlling for: age, body mass index, diabetes, physical activity, hormone replacement therapy, oral contraceptive use, history of hypertension, alcohol consumption, smoking categories, family history of myocardial infarction, serum homocysteine levels, folate intake, vitamin B2 intake, vitamin B6 intake, and vitamin B12 intake.\n",
      "    unique: PMC2562562/PMC2562562.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Multivariable-adjusted* hazard ratios (HR) and 95% confidence intervals (95% CI) for ischemic vascular events according to MTHFR 677C>T polymorphism (n=25,001)\n",
      "  footer_text: * controlling for: age, body mass index, diabetes, physical activity, hormone replacement therapy, oral contraceptive use, history of hypertension, alcohol consumption, smoking categories, family history of myocardial infarction, serum homocysteine levels, folate intake, vitamin B2 intake, vitamin B6 intake, and vitamin B12 intake.\n",
      "    unique: PMC2562562/PMC2562562.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Multivariable-adjusted Hazard Ratios for Cardiovascular Events according to Migraine Status Stratified by MTHFR 677C>T genotype in the Women’s Health Study (N=25,001)\n",
      "  footer_text: * defined as the first of any of these events: nonfatal ischemic stroke, nonfatal myocardial infarction, or death due to ischemic cardiovascular cause. † controlling for: age, body mass index, diabetes, physical activity, hormone replacement therapy, oral contraceptive use, history of hypertension, alcohol consumption, smoking categories, family history of myocardial infarction, serum homocysteine levels, folate intake, vitamin B2 intake, vitamin B6 intake, and vitamin B12 intake.\n",
      "    unique: PMC2562562/PMC2562562.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562567.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Glossina fuscipes fuscipes mitochondrial haplotype frequency distribution.\n",
      "  footer_text: \n",
      "    unique: PMC2562567/PMC2562567.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562567.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562567.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562567.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562568.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562578.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number of haplotypes shared (upper diagonal) and genetic differentiation (FST) among samples (lowest diagonal).\n",
      "  footer_text: CAR-lab, Central African Republic laboratory strain.\n",
      "    unique: PMC2562567/PMC2562567.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Indices of molecular variation in Glossina fuscipes fuscipes including haplotype (HS) and nucleotide (πS) diversities. Kenyan samples were pooled for tests for selective neutrality (R2) and demographic equilibrium (Fs).\n",
      "  footer_text: * Test according to Ramos-Onsins & Rozas (2002). † Test according to Fu (1997). ‡ Between field-collected samples: Student’s t = 13.12, P < 0.006. § Between field-collected samples: t = 1.99, P = 0.19. ¶ P < 0.001. CAR-lab, Central African Republic laboratory strain.\n",
      "    unique: PMC2562567/PMC2562567.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562568.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Coculture of OCI-AML3 cells with MSC feeder layers increases the apoptotic EC50 of traditional chemotherapeutic drugs.\n",
      "  footer_text: \n",
      "    unique: PMC2562568/PMC2562568.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562576.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562578.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient samples used in the comparison of the colon crypt proteomes of FAP patients and controls (distal mucosa from sporadic cancer cases).\n",
      "  footer_text: \n",
      "    unique: PMC2562578/PMC2562578.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562581.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562587.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sample Characteristics Overall and by Race/Ethnicity/Language (N=1137)\n",
      "  footer_text: Chi-square test for association; age (p=0.003), number of co-morbidities 0.001, breast cancer stage p=0.186, surgery (p=0.007), radiation (p=0.177), all else <0.001 percents in the table are unweighted\n",
      "    unique: PMC2562587/PMC2562587.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Percent of Women Who Received Treatment and Survivorship-Related Information, and Wanting More Information by Race/Ethnicity\n",
      "  footer_text: * Restricted to those who received radiation therapy † Restricted to those who received chemotherapy ‡ Restricted to those who are married or living with someone significant p-values: received information: recovery (p=0.047), nutrition (p=0.004), relationship (p<0.001), sexual function (p=0.002) wanted more information: all p’s <0.001 percents are weighted to account for differential selection by race, ethnicity, and nonresponse.\n",
      "    unique: PMC2562587/PMC2562587.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Adjusted Odds of Wanting More Information by Sociodemographic Characteristics, Controlling for Clinical, and Treatment Related Factors*\n",
      "  footer_text: * Adjusted for age, race, education, marital status, # of comorbidities, breast cancer stage and treatment; † Restricted to women who received radiation therapy; ‡ Restricted to women who received chemotherapy percents are weighted to account for differential selection by race, ethnicity, and nonresponse. N/A re: desire for more information on “relationships,” was restricted to those who were married or with a partner\n",
      "    unique: PMC2562587/PMC2562587.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562587.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562587.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562587.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562590.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562590.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Odor sets used for the habituation/spontaneous discrimination task. Numerical values depict vol/vol liquid dilutions in mineral oil theoretically producing a vapor phase partial pressure of 1.0 Pa.\n",
      "  footer_text: \n",
      "    unique: PMC2562590/PMC2562590.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562591.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mouse strains, genotypes, and disease status.\n",
      "  footer_text: \n",
      "    unique: PMC2562591/PMC2562591.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Expression of selected genes in Fli-1+/+ (n = 5) versus Fli-1+/− (n = 5) MRL/lpr B cells. One triplicate experiment per gene.\n",
      "  footer_text: \n",
      "    unique: PMC2562591/PMC2562591.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562592.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Spearman correlation coefficients between estrogens and estrone metabolites among controls*\n",
      "  footer_text: * N ranges from 460 (estrone) to 675 (2-OH, 16α-OH, 2:16α-OH ratio)\n",
      "    unique: PMC2562592/PMC2562592.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562591.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562591.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562592.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562592.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562592.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Characteristics (Mean (SD) or %) of breast cancer cases and matched controls, Nurses' Health Study\n",
      "  footer_text: * Among parous women only ** Among women with natural menopause or bilateral oophorectomy only\n",
      "    unique: PMC2562592/PMC2562592.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Median (range*) of hormones among breast cancer cases and matched controls\n",
      "  footer_text: * From the median of the bottom quartile (12.5 percentile) to the median of the top quartile (87.5 percentile)\n",
      "    unique: PMC2562592/PMC2562592.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Multivariate* relative risks and 95% confidence intervals of breast cancer according to quartiles of estrogen metabolites\n",
      "  footer_text: * Multivariate models controlled for body mass index at age 18 y (<21, 21-22.9, 23-24.9, or ≥25 kg/m2), family history of breast cancer (yes or no), age at menarche (<12, 12, 13, or ≥14 y), age at first birth and parity (nulliparous; 1-4 children, first birth <25 y; 1-4 children, first birth 25-29 y; 1-4 children, first birth ≥30 y; ≥5 children, first birth <25 y; or ≥5 children, first birth ≥25 y), age at menopause (<46, 46-50, 51-55, or ≥56 y), and duration of past postmenopausal hormone use (continuous); unconditional multivariate models (for ER/PR analyses) additionally controlled for matching factors (age [<55, 55-59, 60-64, ≥65], date of blood draw [6-m groups], time of blood draw [1-8am, 9am-12pm, 1pm-12am], fasting status [<10 versus ≥10 h]). ** p-heterogeneity between ER+/PR+ and ER-/PR- were: 0.02 for 2-OH estrone, 0.81 for 16α-OH estrone, and 0.005 for 2:16α OH estrone ratio\n",
      "    unique: PMC2562592/PMC2562592.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562593.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562592.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562594.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: BARRIERS TO STUDY PARTICIPATION*\n",
      "  footer_text: * Both studies include interviews, neuropsychological testing, MRI brain scans, and phlebotomy † MRI= magnetic resonance imaging without intravenous contrast ‡ Defined as 90 minutes or longer\n",
      "    unique: PMC2562594/PMC2562594.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: MOTIVATORS FOR STUDY PARTICIPATION*\n",
      "  footer_text: * Both studies include interviews, neuropsychological testing, MRI brain scans, and phlebotomy. Includes all participants who did not refuse study participation\n",
      "    unique: PMC2562594/PMC2562594.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: SOLUTIONS TO MAXIMIZE STUDY PARTICIPATION (N=50)\n",
      "  footer_text: \n",
      "    unique: PMC2562594/PMC2562594.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562596.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Body Weight and Fluid Intake\n",
      "  footer_text: * indicate significant main effects of Age ** indicates significant Age × Solution interaction\n",
      "    unique: PMC2562596/PMC2562596.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562597.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562598.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562603.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562594.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562594.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562594.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Origins of the alternans patterns in Fig. 4.\n",
      "  footer_text: \n",
      "    unique: PMC2562603/PMC2562603.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562606.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562609.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562610.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Means for Suicidal Ideation (Equivalent to Percent Reporting Suicidal Ideation¹) Within Observed Combinations of Spell and Period\n",
      "  footer_text: Note: ns are reported in parenthesizes following the means. a For example, of the 23 men in Spell 3, Period 8 (i.e., 2 reported occurrences of suicidal ideation, and 7 years since it was last reported), 9% reported suicidal ideation in the past week. b Complete data are not shown for the one individual represented in the Spell 10, Period 1 cell as he progressed to Spell 13, Period 3.\n",
      "    unique: PMC2562610/PMC2562610.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Results of Multiple-Spell Discrete-Time Event-History Analysis: Prediction of Suicidal Ideation from Period, Spell, Parental Depressive Symptoms, and Participants’ Alcohol/Substance Use and Depressive Symptoms (N = 3347)\n",
      "  footer_text: Notes. Exp(B) = exponentiated B; Parent Dep = parental depressive symptoms; Alc/Sub = 1-year lagged alcohol/substance use; Dep = 1-year lagged depressive symptoms; Dep2 = 1-year lagged quadratic depressive symptoms; −2LL = −2 log likelihood; R2 refers to Nagelkerke R2. † p < .10. * p < .05. ** p < .01. *** p < .001.\n",
      "    unique: PMC2562610/PMC2562610.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562610.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562610.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562611.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562611.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean (± standard error of the mean) plasma concentrations of corticosterone (B), estradiol (E2), progesterone (P4) and allopregnanolone (3α,5α-THP) of ovariectomized wildtype (WT) or estrogen receptor β knockout (βERKO) mice administered subcutaneous vehicle (far left), E2 (middle), and DPN (far right). NB: Because it is not clear if the E2 antibody utilized binds DPN, analyses of E2 levels from mice treated with DPN are not included.\n",
      "  footer_text: * indicates a significant (P ≤ 0.05) effect of treatment, which is attributable to higher levels vs. vehicle.\n",
      "    unique: PMC2562611/PMC2562611.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Behavior (mean ± sem) of vehicle-administered WT (n=12) or estrogen receptor β knockout (βERKO; n=18) mice, E2-administered WT (n=16) or βERKO (n=14) mice, or DPN-administered WT (n=18) or βERKO (n=12) mice.\n",
      "  footer_text: * indicates a significant (P ≤ 0.05) effect of treatment, which is attributable to higher levels vs. vehicle. ^ indicates a significant (P ≤ 0.05) effect of genotype. ** indicates a significant (P ≤ 0.05) interaction.\n",
      "    unique: PMC2562611/PMC2562611.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562611.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562612.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562616.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562616.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562612.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Absolute and relative comparison of energy dissipated, maximum force, and pulling length in Na buffer, Ca buffer and La buffer, corresponding to the data presented in figure 3.\n",
      "  footer_text: \n",
      "    unique: PMC2562612/PMC2562612.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562615.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562616.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient characteristics and asthma related function of total sample and risk groups\n",
      "  footer_text: Asterisks denote the statistical significance of the comparison between high- and low-risk groups for each by Chi Square or T-test: *** p < 0.001 ** p < 0.01 * p < 0.05. In the comparison of high-risk groups from each definition, p-values are shown or NS denotes the difference is not significant at the level of p<0.05. CHS-A = Child Health Survey — Asthma.\n",
      "    unique: PMC2562616/PMC2562616.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Number of youth captured by high-risk definition criteria (N=769)\n",
      "  footer_text: ED: ≥1 emergency department visits for asthma Inpatient: ≥1 inpatient hospitalizations for asthma OS: ≥1 oral steroid prescriptions for asthma Rx: ≥4 prescriptions for asthma medication Visits + Rx: ≥4 ambulatory visits for asthma and ≥2 prescriptions for asthma medication\n",
      "    unique: PMC2562616/PMC2562616.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Health care cost and utilization for total sample and risk groups\n",
      "  footer_text: Asterisks denote the statistical significance of the comparison between high- and low-risk groups within each definition by T-test: In the comparison of high-risk groups from each definition, p-values are shown or NS denotes the difference is not significant at the level of p<0.05 *** p < 0.001 ** p < 0.01 * p < 0.05\n",
      "    unique: PMC2562616/PMC2562616.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562616.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562642.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562620.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562630.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562634.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562635.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562638.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562640.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562642.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of different false positive reduction scheme\n",
      "  footer_text: \n",
      "    unique: PMC2562642/PMC2562642.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562647.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562649.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Clinical Phenotype, Including Behavior Problems, Ataxia, and Seizures, in 33 Adolescent and Adult SSADH Deficiency Patients (mean age: 12.1±7.6 years, range: 2.2-39.6 years; 48% females)\n",
      "  footer_text: SSADH=succinic semialdehyde dehydrogenase.\n",
      "    unique: PMC2562649/PMC2562649.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Follow-up Assessment of Language, Cognition, and Gross and Fine Motor Performance in 24 Adolescent and Adult SSADH Deficiency Patients (mean age: 18.1±7.0 years, age range: 10.6–40.0 years; 50% females)\n",
      "  footer_text: The mean follow-up period was 11. ±4.6 years. (range: 5.3-21.4 years.) All performances were classified by an experienced physician in relative units from 1 (severe deficit) to 5 (normal). SSADH=succinic semialdehyde dehydrogenase.\n",
      "    unique: PMC2562649/PMC2562649.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562655.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Number of classes, number of images, format, size and microscopy of the datasets of IICBU-2008.\n",
      "  footer_text: \n",
      "    unique: PMC2562655/PMC2562655.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562658.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562668.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562649.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562655.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: List of strains used in the MLST analysis; their toxin subtype designation, sequencing types and alleles for 73 strains for the 7 selected housekeeping genes\n",
      "  footer_text: a * represents the initial 30 strains used in housekeeping genes screening study. b Strain names refer to Eric A. Johnson’s laboratory’s stocks. c ST: Sequencing type. d A1 has BoNT/A1 gene (see NCBI publication number AF461539); A2 has the BoNT/A2 gene (see NCBI publication number AY953275), A(B) has BoNT/A1 gene and a silent BoNT/B gene (see NCBI publication number NCTC 2916), A3 has the BoNT/A3 gene (see NCBI publication number DQ185900), and A4 has the BoNT/A4 gene (see NCBI publication number DQ185901) and BoNT/B genes (see NCBI publication number EU341304). ha−/orfx+ A1 produces BoNT/A1 but with an ORFX cluster, sporo: Clostridium sporogenes\n",
      "    unique: PMC2562668/PMC2562668.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562668.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562668.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562668.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562681.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562681.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Details on primer sequences and the PCR conditions of 7 housekeeping genes used in MLST analysis\n",
      "  footer_text: a F: forward; R: reverse; aroE: shikimate dehydrogenase gene; hsp6: heat shock gene; mdh: malate dehydrogenase gene; recA: RecA gene; aceK: isocitrate dehydrogenase gene; rpoB: RNA polymerase subunit B gene; oppB: oligopeptide/dipeptide ABC transporter gene.\n",
      "    unique: PMC2562668/PMC2562668.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Genetic variability of 7 genes in 24 STs. The number of variable sites ranges from 39 to 73. The synonymous rate of substitution (dS) and non-synonymous rate of substitution (dN) range among alleles and average to 5.40 and 0.69 among all sites. Estimates of dN /dS and CI were calculated by method of Pond and Frost (2005). Both oppB and aroE have larger dN/dS suggestion these alleles are under more relaxed selection.\n",
      "  footer_text: \n",
      "    unique: PMC2562668/PMC2562668.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562671.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562681.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Possible causes and solutions of the young driver problem and strength of evidence of efficacy of the proposed solutions\n",
      "  footer_text: Note. This table was created using the references reviewed by the authors in the text.\n",
      "    unique: PMC2562681/PMC2562681.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Descriptions of the Checkpoints and the Safe Drivers Wanted parent management programs\n",
      "  footer_text: DMV = department of motor vehicles.\n",
      "    unique: PMC2562681/PMC2562681.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562683.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The dependence of HF on θF: parameters of the best-fit sigmoid functions (given by Expression 2).\n",
      "  footer_text: f, spatial frequency in cycles/°; B, offset in degrees; Amax, amplitude in degrees; Pc, center point of curve in degrees; K, decay constant in degrees−1; r2, coefficient of determination; 90%, value of θF at which HF was 90%.\n",
      "    unique: PMC2562683/PMC2562683.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The dependence of VF on θF: parameters of the best-fit exponential functions (given by Expression 3).\n",
      "  footer_text: f, spatial frequency in cycles/°; B, offset in degrees; S, slope in degrees; τ, decay constant in degrees−1; r2, coefficient of determination.\n",
      "    unique: PMC2562683/PMC2562683.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The dependence of HF on log spatial frequency for various values of θF: parameters of the best-fit Gaussian functions.\n",
      "  footer_text: Cardinal axis (horizontal), indicates that spatial frequency is specified with respect to the horizontal; Orthogonal axis, indicates that spatial frequency is specified orthogonal to the orientation of the grating; θF, folded direction of the orthogonal disparity vector; Apeak, amplitude of the peak in degrees; fo, spatial frequency of the peak in cycles/°; σ, standard deviation in °/cycle (log units to the base 10); r2, coefficient of determination.\n",
      "    unique: PMC2562683/PMC2562683.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: The dependence of VF on log spatial frequency for various values of θF: parameters of the best-fit Gaussian functions.\n",
      "  footer_text: Cardinal axis (vertical), indicates that spatial frequency is specified with respect to the vertical. Other abbreviations as for Table 3.\n",
      "    unique: PMC2562683/PMC2562683.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562683.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562683.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562683.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562683.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562686.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: One mole of ApbC binds two moles of iron and two moles sulfur atoms.a\n",
      "  footer_text: a ApbC was reconstituted with a five-fold excess of S and Fe as described in material and methods. Iron and sulfide were quantified as described elsewhere (35). Values shown for individual reconstitutions are the average two independent measurements. Errors are reported as standard deviations. b Below Detectable Limit\n",
      "    unique: PMC2562686/PMC2562686.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562688.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Effects of cdh6MOs injection on zebrafish development\n",
      "  footer_text: * Only one of the seven embryos showed thorax edema and curved body, and the gross defects in the remaining six embryos (much smaller and truncated body) were different from the cdh6 morphants.\n",
      "    unique: PMC2562688/PMC2562688.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effects of cdh6MOs injection on expression of transcription factors and notch-delta genes in the retina\n",
      "  footer_text: n, number of eyes examined; %, percentages of greatly reduced or altered staining (staining area and/or staining intensity), compared to the majority of control embryos. For cdh6 morphants, only those with obvious gross morphological defects were used. Abbreviation: ND, analysis not done.\n",
      "    unique: PMC2562688/PMC2562688.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Effects of cdh6MOs injection on zebrafish retinal development revealed by immunostaining\n",
      "  footer_text: n, number eyes examined; n1 and n2, the numbers of eyes from embryos injected with cdh6MO1 and cdh6MO2, respectively. %, percentages of greatly reduced staining (staining area and/or intensity) compared to the majority of control embryos. * The retinal defects in these embryos were milder than those from morphants with obvious gross morphological defects. Abbreviations: Hu, anti-HuC/HuD immunostaining; Hu & zpr-1, anti-HuC/HuD and zpr-1 double immunostaining; ND, analysis not done; Parv, anti-parvalbumin immunostaining; Pax6, anti-Pax6 immunostaining; tub, anti-acetylated tubulin immunostaining; zn5, zn5 immunostaining.\n",
      "    unique: PMC2562688/PMC2562688.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562689.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Descriptive statistics for WS and TD groups\n",
      "  footer_text: 1 Data for one child missing from analysis.\n",
      "    unique: PMC2562689/PMC2562689.xml::T1-0"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562686.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562688.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562688.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562688.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562689.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: TRC raw score and percent correct on spatial items for WS and TD groups matched for relational vocabulary size at various levels of stringency\n",
      "  footer_text: \n",
      "    unique: PMC2562689/PMC2562689.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562694.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Change in WHEL dietary components and plasma carotenoids from baseline to one year by cluster\n",
      "  footer_text: \n",
      "    unique: PMC2562694/PMC2562694.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562689.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562694.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Intensity and frequency of the first phase of the WHEL tailored intervention\n",
      "  footer_text: \n",
      "    unique: PMC2562694/PMC2562694.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Differences in cluster membership by baseline characteristics and health behaviors\n",
      "  footer_text: \n",
      "    unique: PMC2562694/PMC2562694.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Differences in cluster membership by introductory call variables\n",
      "  footer_text: \n",
      "    unique: PMC2562694/PMC2562694.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562695.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Variance (in pixels) associated with each category in the experiment\n",
      "  footer_text: \n",
      "    unique: PMC2562695/PMC2562695.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean accuracy in Experiment 1 across all participants for each of the four conditions. Accuracy is given by percent correct across all 10 experimental blocks\n",
      "  footer_text: \n",
      "    unique: PMC2562695/PMC2562695.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Goodness-of-fit values (-2ln L) for the prototype and exemplar models in Experiment 1\n",
      "  footer_text: \n",
      "    unique: PMC2562695/PMC2562695.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Mean accuracy in Experiment 2 across all participants for each of the four conditions. Accuracy is given by percent correct across all 10 experimental blocks\n",
      "  footer_text: \n",
      "    unique: PMC2562695/PMC2562695.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562697.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562694.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562694.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562694.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562695.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562695.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562695.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562695.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562699.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The Ten-Point Surgical Apgar Score The Surgical Apgar Score is calculated at the end of any general or vascular surgery operation, from the estimated blood loss, lowest mean arterial pressure and lowest heart rate entered in the anaesthesia record during the operation. The score is the sum of the points from each category.\n",
      "  footer_text: * Occurrence of pathologic bradyarrhythmia, including sinus arrest, atrioventricular block or dissociation, junctional or ventricular escape rhythms, and asystole also receive 0 pts for lowest heart rate\n",
      "    unique: PMC2562699/PMC2562699.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Characteristics of patients, procedures, and outcomes, by Surgical Apgar Score\n",
      "  footer_text: * Hypothesis testing employed Cochrane-Armitage chi-square trend tests for categorical variables, and Spearman correlation coefficients for continuous variables, except for length of stay, for which we used the Kruskal-Wallis test.\n",
      "    unique: PMC2562699/PMC2562699.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562699.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562699.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Postoperative outcomes, by preoperative risk stratum and Surgical Apgar Score Using a detailed preoperative risk prediction model, patients were stratified by quintiles of likelihood of a major postoperative complication. The predicted and observed incidence of major postoperative complications are presented.\n",
      "  footer_text: * Cochrane-Armitage chi-square trend test\n",
      "    unique: PMC2562699/PMC2562699.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Complication rates and adjusted and unadjusted likelihood ratios for levels of the Surgical Apgar Score\n",
      "  footer_text: \n",
      "    unique: PMC2562699/PMC2562699.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562701.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of the equilibrium and kinetic parameters [42]\n",
      "  footer_text: L=[ET]/[ER] is a unitless parameter. The K values are in μM and S, I and A are substrate (PEP), inhibitor (Phe) and activator (fructose-bis-phosphate, FBP) respectively. The k values are expressed as relative activity, 1.0 being equal Vmax.\n",
      "    unique: PMC2562701/PMC2562701.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562705.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Data Collection and Refinement Statistics\n",
      "  footer_text: a Numbers in parentheses indicate values for the highest resolution shell.\n",
      "    unique: PMC2562705/PMC2562705.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562699.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562699.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562701.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562705.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Kinetic Constants for the RhamD Reactiona\n",
      "  footer_text: a Assay conditions are given in the Materials and Methods section.\n",
      "    unique: PMC2562705/PMC2562705.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Kinetic Constants for the 2-Keto-3-Deoxy-L-Rhamnonate Aldolase Reactiona\n",
      "  footer_text: a Assay conditions are given in the Materials and Methods section.\n",
      "    unique: PMC2562705/PMC2562705.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Kinetic constants of Mutants for the RhamD Reactiona\n",
      "  footer_text: a Assay conditions are given in the Materials and Methods section. b No detectable activity\n",
      "    unique: PMC2562705/PMC2562705.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562707.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562708.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2562708/PMC2562708.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: \n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2562708/PMC2562708.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562705.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562705.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562705.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562708.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562708.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562708.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562709.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "tw(T3): {'id', 'orientation', 'position'}\n",
      "  label_text: TABLE I\n",
      "  caption_text: SNR AND CNR VERSUS POD.\n",
      "  footer_text: \n",
      "    unique: PMC2562708/PMC2562708.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562709.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Neisseria gonorrhoeae strains used in this study.\n",
      "  footer_text: * Pro: requires proline; Proto; Prototrophic; PAU: requires proline, arginine and uracil; PAHU: requires proline, arginine, hypoxanthine and uracil; AHU: requires arginine, hypoxanthine and uracil\n",
      "    unique: PMC2562709/PMC2562709.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562710.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562711.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562714.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Data\n",
      "  footer_text: \n",
      "    unique: PMC2562714/PMC2562714.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562714.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562714.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562714.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562714.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562726.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Correlation Matrix of Behavioral Antecedents and Outcome Behaviors\n",
      "  footer_text: * Note: p < .05; ** p > .01\n",
      "    unique: PMC2562714/PMC2562714.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Predictors for Protected Vaginal Sex\n",
      "  footer_text: \n",
      "    unique: PMC2562714/PMC2562714.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Predictors for Unprotected Vaginal Sex\n",
      "  footer_text: \n",
      "    unique: PMC2562714/PMC2562714.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562726.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Effect of ethanol pretreatment of P-selectin-coated Protein A beads All measurements at 100 s-1 with untreated neutrophils.\n",
      "  footer_text: \n",
      "    unique: PMC2562726/PMC2562726.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Neutrophil rolling on P-selectin-coated surface. (N=3 donors)\n",
      "  footer_text: * p<0.001 comparing untreated vs ethanol-treated samples.\n",
      "    unique: PMC2562726/PMC2562726.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Neutrophil rolling on IL-1 stimulated endothelial cells. (N=3 donors)\n",
      "  footer_text: * p<0.001 comparing untreated vs ethanol-treated samples.\n",
      "    unique: PMC2562726/PMC2562726.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562728.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Results of analyses for antibodies to West Nile virus in horse sera from Connecticut and New York State by plaque reduction neutralization tests (PRNT) and a polyvalent enzyme-linked immunosorbent assay (ELISA) containing recombinant envelope protein antigen.\n",
      "  footer_text: * Positive by PRNT when at least 95% neutralization of plaque-forming units was detected. Positive ELISA titers (≥ 1 in 160). Note: unknown history refers to blood samples drawn from ill horses in 1995−1996 prior to the 1999 outbreak of human and horse cases of West Nile encephalitis in northeastern United States. All tests were performed on one serum sample per animal at The Connecticut Agricultural Experiment Station.\n",
      "    unique: PMC2562728/PMC2562728.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Frequency distributions of serum antibody titers for horse sera positive by a polyvalent enzyme-linked immunosorbent assay (ELISA) or in plaque reduction neutralization tests (PRNT).\n",
      "  footer_text: * Positive antibody titers by an ELISA were 1 ≥ 160. Seropositive PRNT indicate 95% neutralization of plaque-forming units at all specified serum dilutions but not tested beyond 1 in 320. NA = not applicable. All tests were performed at The Connecticut Agricultural Experiment Station.\n",
      "    unique: PMC2562728/PMC2562728.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of serologic test results for 43 horse sera (group C) from subjects with undiagnosed illnesses prior to 1999 and tested for antibodies to West Nile virus, Anaplasma phagocytophilum, and/or Borrelia burgdorferi by separate polyvalent enzyme-linked immunosorbent assays.\n",
      "  footer_text: * Recombinant envelope E protein antigen was used in an ELISA for West Nile virus (WNV) antibodies, whereas recombinant protein 44 and whole-cell antigens of A.. phagocytophilum (AP) and B. burgdorferi (BB), respectively, were incorporated separately in an ELISA. All tests were performed at The Connecticut Agricultural Experiment Station.\n",
      "    unique: PMC2562728/PMC2562728.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562732.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562726.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562726.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562728.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562728.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562728.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  label_text: Table 1\n",
      "  caption_text: Measurements of Postmortem Tongues\n",
      "  footer_text: Asteriks indicate significant differences between groups A and B and between groups C and D (non-paired t tests). * p < 0.05 ** p < 0.01 *** p < 0.001.\n",
      "    unique: PMC2562732/PMC2562732.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Measurements of Tongue Casts\n",
      "  footer_text: * Comparisons before and after the surgery (paired t tests) * p < 0.05 ** p < 0.01 *** p < 0.001 # Comparisons between the two groups (non-paired t tests). # p < 0.05 ## p < 0.01 ### p < 0.001\n",
      "    unique: PMC2562732/PMC2562732.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562733.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline predictions for moderate glucose control and expected benefits of intensive glucose control in older patients with no comorbid illnesses or functional impairments.*\n",
      "  footer_text: * Moderate glucose control= glycosylated hemoglobin 7.9%. Intensive glucose control=glycosylated hemoglobin 7.0%. Inc.= remaining lifetime incidence. ARR= absolute risk reduction. QAD= quality-adjusted days. CI= confidence interval. Virtual patients with duration of diabetes in ranges 0–5 years and 5–10 years had results that were intermediate between patients with new onset diabetes and 10–15 years of diabetes at the start of simulations.\n",
      "    unique: PMC2562733/PMC2562733.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562736.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: TGCT incidence in Kitl (Sl) and Kit (W) heterozygous mutant mice*\n",
      "  footer_text: * L.C. Stevens and D.S. Varnum (Ref. 6; Sl/+ data, personal communication). † The relative numbers of wild-type and mutant mice were not published or available for these crosses. Therefore, statistical significance was not calculated.\n",
      "    unique: PMC2562736/PMC2562736.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Increased incidence of spontaneous TGCTs in grizzle-belly (Slgb) males\n",
      "  footer_text: \n",
      "    unique: PMC2562736/PMC2562736.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562732.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562733.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562736.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562736.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562736.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Parent-of-origin effect revealed by Slgb reciprocal crosses\n",
      "  footer_text: * NS = not significant\n",
      "    unique: PMC2562736/PMC2562736.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562746.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: The main effects for and interactions between calibration method, stimulus level and frequency on the absolute change in DPOAE level between deep and shallow insertions after correcting for the incidental difference due to a change in ear-canal volume. The asterisk (*) denotes statistical significance for p < 0.05.\n",
      "  footer_text: \n",
      "    unique: PMC2562746/PMC2562746.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Means and standard deviations of the absolute change in DPOAE level between insertions after correcting for the expected, incidental change. The first five rows are averaged across frequency for each calibration method, and the sixth row is also averaged across level.\n",
      "  footer_text: \n",
      "    unique: PMC2562746/PMC2562746.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562749.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562756.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562759.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562746.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562746.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Samples used for analysis The table shows the source of samples analyzed (Clinical Study), the mutations detected by the ViroSeq system, and the HIV-1 subtype (pol region).\n",
      "  footer_text: * Note that the mutations K103R and V179D are associated with NVP resistance only when they occur together.\n",
      "    unique: PMC2562759/PMC2562759.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: NVP resistance mutations detected in clones Fifty HIV-1 clones were analyzed by AmpliSeq from each infant sample. The number of clones with 0, 1, or 2 mutations is shown. Mutations shown in bold were detected using the ViroSeq HIV-1 Genotyping System (population sequencing). The following amino acid polymorphisms (not shown in Table 2) were also detected in the samples at codons in HIV-1 reverse transcriptase that are the sites of NVP resistance mutations: 147770: V179A (1 clone), V106S+V108S (1 clone). 147858: V181H (1 clone). 274581: L100V+A98C (1 clone). 288407: K103R (50 clones), V179E (26 clones). 289005: V179I (49 clones), V179M (1 clone), K103E+A98S (2 clones). 324698: K101R (1 clone).\n",
      "  footer_text: * Note that the mutations K103R and V179D are associated with NVP resistance only when they occur together.\n",
      "    unique: PMC2562759/PMC2562759.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Codons detected in clones at positions in HIV-1 reverse transcriptase associated with NVP resistance Fifty HIV-1 clones were analyzed from each of the six infant samples (300 clones total). Sample numbers are indicated on the left. NVP resistance mutations occur at codons 98, 100, 101, 103, 106, 108, 179, 181, 188, and 190 in HIV-1 reverse transcriptase. The table shows the nucleotide sequences and corresponding amino acids detected in clones from each infant sample at each position. The number of clones with each nucleotide sequence is indicated. Reference sequences / amino acids (those found in reference strain HXB2) are shaded. Sequences / mutations associated with NVP resistance are shown in bold.\n",
      "  footer_text: * Note that the mutations K103R and V179D are associated with NVP resistance only when they occur together.\n",
      "    unique: PMC2562759/PMC2562759.xml::T3-0\n",
      "    unique: PMC2562759/PMC2562759.xml::T3-1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562759.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562759.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562759.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562759.xml::T3-1'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562761.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562761.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562774.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562761.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of Mean Stiffness and Hardness in Mytilus Cuticle, the Byssal Thread Core, and Synthetic Epoxies\n",
      "  footer_text: \n",
      "    unique: PMC2562761/PMC2562761.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Repeat Sequence Motifs from mfp1s of Seven Marine Mussel Species (Italicized P and Y denote Pro and Tyr residues targeted for hydroxylation)\n",
      "  footer_text: \n",
      "    unique: PMC2562761/PMC2562761.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562772.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562774.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected Bond Lengths (Å) and Angles (°) for Fe3(SPh)6(CO)6,24 Fe3(S2C3H6)3(CO)4(PEt3)2, and PBnPh3[Fe3(S2C3H6)4(CO)4] (Fe(1) is Terminal and Fe(2) is Central).\n",
      "  footer_text: \n",
      "    unique: PMC2562774/PMC2562774.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Selected Crystallographic Data for PBnPh3[Fe3(S2C3H6)4(CO)4]•3THF and Fe3(S2C3H6)3(CO)4(PEt3)2.\n",
      "  footer_text: \n",
      "    unique: PMC2562774/PMC2562774.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562775.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic data on child survey participants and non-participants\n",
      "  footer_text: \n",
      "    unique: PMC2562775/PMC2562775.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Results for model for use of bronchodilator\n",
      "  footer_text: Notes: Model A (Exchangeable single latent trait) Model B (General single latent trait) Model C (Direct dependence single latent trait) Model D (Direct dependence bivariate latent trait) Model E (Correlated bivariate latent trait)\n",
      "    unique: PMC2562775/PMC2562775.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Simulations based on n = 1500 subjects, two informants based on a 5-item questionnaire with correlation of 0.4. Rows correspond to different patterns of missingness, with logit(Pr(missing))=γ0+γ1∑Xvib+γ2Y. Table entries show the mean of the estimates of α1 from 1000 simulated datasets fit using model (E), where the true value of α1 is 2. Corresponding standard errors are given in parentheses.\n",
      "  footer_text: \n",
      "    unique: PMC2562775/PMC2562775.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562777.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected EPR Parameters for [Fe2(S2CXH2x)(CO)2L(dppv)]BF4 (Spectra Recorded at 110 K (−163 °C), in 1:1 CH2Cl2:toluene).\n",
      "  footer_text: a For [3pdt]+ the two species are present in at 2:1 ratio.\n",
      "    unique: PMC2562777/PMC2562777.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Selected EPR Parameters for [Fe2(S2CxH2x)(μ-CO)(CO)3L(dppv)]BF4 (Spectra Recorded at 110K (−163 °C), in 1:1 CH2Cl2:toluene)\n",
      "  footer_text: \n",
      "    unique: PMC2562777/PMC2562777.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562775.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562775.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562775.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562777.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562777.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Selected Structural Parameters for [Fe2(S2C3H6)(CO)3L(dppv)]BF4 (L = PMe3, Pi-Pr3).\n",
      "  footer_text: \n",
      "    unique: PMC2562777/PMC2562777.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Details of Data Collection and Structure Refinement for Crystallography.\n",
      "  footer_text: a R1 = Σ|Fo|−|Fc|/Σ|Fo| b wR2 = {[w(|Fo|−|Fc|)2]/Σ[wFo2]}1/2, where w = 1/σ2 (Fo)\n",
      "    unique: PMC2562777/PMC2562777.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562863.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic characteristics by mortality status for respective final testing wave.\n",
      "  footer_text: Note. Standard deviations appear in parentheses. Subjects rated their health relative to same-aged peers on a 5-point scale ranging from (0) very good to (4) very poor.\n",
      "    unique: PMC2562863/PMC2562863.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mortality and age group mean T-scores for cognitive and speeded measures at last testing wave.\n",
      "  footer_text: Note. Standard deviations appear in parentheses; ISD = intraindividual standard deviation. a These measures were not administered for Sample 1Wave 1.\n",
      "    unique: PMC2562863/PMC2562863.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Cox regressions predicting mortality status as a function of demographic and cognitive covariates.\n",
      "  footer_text: Note. Statistical models were computed separately for each cognitive and speed measure. Results represent parameters for a given regression block controlling for the influence of all predictors on previous blocks. Values in column 1 reflect unstandardized regression coefficients, whereas values in column 2 are odds ratios denoting percentage change in morality risk per unit increase of predictor. Mortality status was coded as either 0 (survivor) or 1 (decedent). CI = confidence interval; CVD = cardiovascular disease. Subjects rated their health relative to same-aged peers on a 5-point scale ranging from (0) very good to (4) very poor. CVD conditions represent self-reported presence of 4 cardiovascular diseases (heart disease, stroke, arteriosclerosis, and high blood pressure) for any wave of testing. CVD medications reflect self-reported use of medications prescribed to treat cardiovascular symptoms and disease (heart, chest, high blood pressure, and anticoagulants). ISD = intraindividual standard deviation. a N = 707 for all measures except computation span and listening span (n = 635).\n",
      "    unique: PMC2562863/PMC2562863.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562777.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562777.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562863.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562863.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562863.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562863.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562869.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562875.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562875.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Change in speed inconsistency as a function of time to death, age group, and mortality status.\n",
      "  footer_text: a Note. A significant Age group × Mortality status interaction was observed for this variable; associated mortality coefficients are reported separately for each age group in the text. γ00 = inconsistency for Young-Old Survivors at the sample midpoint for time remaining in the study (M = 7.24 years); γ10 = average rate of change in inconsistency per additional year of time remaining in the study for Young-Old Survivors; γ11 = average difference in rate of inconsistency change between Young-Old and Old-Old adults partialing the influence of mortality status; γ12 = average difference in rate of inconsistency change between Survivors and Decedents controlling for age group differences; SE = standard error; t = t-ratio. * p< .05 ** p < .01\n",
      "    unique: PMC2562863/PMC2562863.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562869.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: ELISA and Multiplex Cost Comparison\n",
      "  footer_text: * Cost per sample is determined by taking the commercial cost of the immunoassay kit and dividing by the number of samples that can be analyzed in duplicate (between 36 and 40, for most immunoassays depending on number of standards and controls run in each assay).\n",
      "    unique: PMC2562869/PMC2562869.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562875.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of study population\n",
      "  footer_text: * Louisiana, Arkansas, Oklahoma, New Mexico\n",
      "    unique: PMC2562875/PMC2562875.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Risk factors of pancreatic cancer and association of pancreatic cancer with HBV or HCV infection: Univariable and Multivariable logistic regression analysis\n",
      "  footer_text: Abbreviations: +, positive; −, negative; anti-HBc, anti-hepatitis B core antigen; anti-HBs, anti-hepatitis B surface antigen; HBV, hepatitis B virus; HBsAg, hepatitis B surface antigen; HCV, hepatitis C virus. * Never exposed; † Chronic carrier of HBV; ‡ Past exposure to HBV; § Past exposure to HBV with natural immunity; ¶ Possible chronically infected and have an undetectable level of HBsAg present in the serum.\n",
      "    unique: PMC2562875/PMC2562875.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562879.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562884.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Pre- and Postsupplementation Debrisoquine Urinary Recovery Ratios\n",
      "  footer_text: CI = confidence interval, HDEB = 6-hydroxydebrisoquine, DEB = debrisoquine † = geometric mean of postsupplementation/presupplementation ratios and 90% CI * = p < 0.05\n",
      "    unique: PMC2562884/PMC2562884.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Phytochemical analysis and disintegration times for botanical dosage forms.\n",
      "  footer_text: \n",
      "    unique: PMC2562884/PMC2562884.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562890.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562891.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of demographic variables, vascular risk factors, and Hamilton Depression Rating Scale (HAM) score between persons with and without incident Alzheimer’s disease. Washington Heights- Inwood Columbia Aging Project 1992-2003.\n",
      "  footer_text: \n",
      "    unique: PMC2562891/PMC2562891.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562884.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562884.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562891.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562891.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of Hamilton Dementia Rating Scale Scores by the presence of vascular risk factors. Washington Heights- Inwood Columbia Aging Project 1992-2003.\n",
      "  footer_text: \n",
      "    unique: PMC2562891/PMC2562891.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Hazard ratios (HR) and 95% confidence intervals relating Hamilton Depression Rating Scale (HAM) scores to incident Alzheimer’s disease (AD). Model 1 is adjusted for age and gender. Model 2 is adjusted also for ethnic group, education, APOEε4, diabetes, hypertension, heart disease, current smoking, and stroke. Washington Heights- Inwood Columbia Aging Project 1992-2003.\n",
      "  footer_text: \n",
      "    unique: PMC2562891/PMC2562891.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562897.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562898.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Digoxin pharmacokinetic parameters before and after supplementation/drug phases (mean ± s.d.)\n",
      "  footer_text: AUC = area under the concentration-time curve, Cmax = maximum serum concentration, * = p < 0.05\n",
      "    unique: PMC2562898/PMC2562898.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Phytochemical analysis and disintegration times for botanical dosage forms (n = 6).\n",
      "  footer_text: \n",
      "    unique: PMC2562898/PMC2562898.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562902.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562891.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562898.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562898.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562902.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562902.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1:\n",
      "  caption_text: Mean Knowledge Score by Occupation\n",
      "  footer_text: a. * ANOVA p.= < 0.000 b. ** Dunnet test p= <0.0000 Doctor was the control\n",
      "    unique: PMC2562902/PMC2562902.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2:\n",
      "  caption_text: Beliefs and Attitudes About Diagnosis and Care of HIV/AIDS Patients and Health Care Settings\n",
      "  footer_text: \n",
      "    unique: PMC2562902/PMC2562902.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3:\n",
      "  caption_text: Feelings and Views Towards Care of HIV/AIDS Patients in Different Health Care Settings *Total score on feelings and views towards care of HIV/AIDS patient item.\n",
      "  footer_text: * Maximum score = 48.\n",
      "    unique: PMC2562902/PMC2562902.xml::T3-0\n",
      "{'tfoot'}\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4:\n",
      "  caption_text: Comparison of Means of Total Knowledge Score by Source of Information\n",
      "  footer_text: a. School > radio p< 0.05 b. School > health talks/seminars p < 0.05\n",
      "    unique: PMC2562902/PMC2562902.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562904.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient information for cases included in this study.\n",
      "  footer_text: * The numeric values here represent a rank ordering of fracture severity amongst the twenty cases from 1 (indicating least severe) to 20 (most severe).\n",
      "    unique: PMC2562904/PMC2562904.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562902.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562904.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562912.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562912.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Correlations between observed empathic and nonempathic responses, self-reported spouse responses to pain, marital satisfaction, and perceived spousal support\n",
      "  footer_text: Note: * p < .05 ** p < .01. I-S = Invalidation by Spouse, V-S = Validation by Spouse, V-PP = Validation by Person with Pain, I-PP = Invalidation by Person with Pain. For the following variables, PP = Report by Person with Pain; S = Spouse Report: PR = Punishing Responses, SR = Solicitous Responses, DR = Distracting Responses, DAS = Dyadic Adjustment Scale, PSS = Perceived Spousal Support\n",
      "    unique: PMC2562912/PMC2562912.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Pattern Matrix for Exploratory Factor Analysis on Self-Report Spouse Responses and Observations of Validation and Invalidation\n",
      "  footer_text: a Solution accounts for 57% of the variance. The correlation between factors = -.17. PP = Person with Pain. * p < .05. ** p < .01.\n",
      "    unique: PMC2562912/PMC2562912.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562926.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Estimates of β for the ACTG 175 data. Unadjusted is Ȳ(1)−Ȳ(0); Change scores is Ȳ(1)−Ȳ(0) − the difference in sample mean baseline CD4; Forward-1 is β̂ in (17) with treatment-specific regression models developed using forward selection allowing linear terms only in elements of X; Forward-2 is the same but allowing linear, quadratic, and two-way interaction terms; ANCOVA is β̂ANCOV A1 using all elements of X; and Koch is β̂KOCH using all elements of X. SE is estimated standard error calculated as described in the text. Test Stat. is the Wald test statistic; and Rel. Eff. is (SE for the Unadjusted estimator)2 divided by (SE for the indicated estimator)2.\n",
      "  footer_text: * Row calculated using usual least squares SE\n",
      "    unique: PMC2562926/PMC2562926.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Results for the simulation scenario based on the ACTG 175 data using 5000 Monte Carlo data sets. Estimators are as in Table I; Benchmark is the proposed estimator found by fitting the true treatment-specific regressions to obtain predicted values. MC SD is Monte Carlo standard deviation, Ave SE is the average of standard error estimates, Cov. Prob. is coverage probability of a 95% Wald confidence interval, and Rel. Eff. is the Monte Carlo mean square error for the Unadjusted estimator divided by that for the indicated estimator."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562912.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562926.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562926.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562926.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562926.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "  footer_text: * Row calculated using usual least squares SE\n",
      "    unique: PMC2562926/PMC2562926.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Results for the second simulation scenario data using 5000 Monte Carlo data sets. All entries are as in Table II.\n",
      "  footer_text: \n",
      "    unique: PMC2562926/PMC2562926.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table IV\n",
      "  caption_text: Proportion of 5000 Monte Carlo data sets for which the null hypothesis β = 0 is rejected in favor of the alternative β > 0 using the test statistic based on each estimator and level of significance 0.025. Each of the two simulation scenarios in the text is considered with n = 400, δ = 0.50, and the intercept of the true E(Ȳ |Z = k, X), k = 0, 1, adjusted so that the true value of β is that indicated.\n",
      "  footer_text: \n",
      "    unique: PMC2562926/PMC2562926.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562929.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Effect on the BRCAPRO carrier probability of excluding vs. including all non-breast/ovary cancers into BRCAPRO for families based on figure 1. Non-breast/ovary cancers are included as per the methods of section 2.4.\n",
      "  footer_text: \n",
      "    unique: PMC2562929/PMC2562929.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Characteristics of the BRCAPRO predictions that exclude surviving all non-breast/ovary cancers versus including surviving all non-breast/ovary cancers. The first three columns are for predicting BRCA1 or BRCA2 mutations, the second for predicting solely BRCA1 mutations, the third column for predicting solely BRCA2 mutations. The postive/negative predictive values (PPV,NPV), sensitivity, and specificity are all for a 10% carrier probability threshold. The p-value column tests if the difference between including and excluding is zero for each statistic.\n",
      "  footer_text: \n",
      "    unique: PMC2562929/PMC2562929.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table III\n",
      "  caption_text: Calibration of the BRCAPRO predictions that exclude surviving all non-breast/ovary cancers versus including surviving all non-breast/ovary cancers. The first two columns are for predicting BRCA1 or BRCA2 mutations, the second for predicting solely BRCA1 mutations, the third for predicting solely BRCA2 mutations. The rows are for all the data (overall), those with carrier probability < 10%, and with carrier probability > 10%, computing the ratio of observed to expected (O/E) and the left/right endpoints of its 95% confidence interval.\n",
      "  footer_text: \n",
      "    unique: PMC2562929/PMC2562929.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562929.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562929.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562929.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562935.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562935.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2562934.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562935.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Frequency of CD4+ and CD8+ T-Cell Responses to Gag, Pol, Nef, Reg, and Env Among 71 HIV-1–Infected Women in South Africa\n",
      "  footer_text: \n",
      "    unique: PMC2562935/PMC2562935.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Peptide Pools Eliciting CD4+ and CD8+ T-Cell Responses Among 71 HIV-1–Infected Women in South Africa\n",
      "  footer_text: \n",
      "    unique: PMC2562935/PMC2562935.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562941.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2562949.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Test set of coiled-coil dimers of known orientation\n",
      "  footer_text: Data for seven bZIP coiled coils without structures not included in averages. a Fraction exposed is the ratio of the solvent-accessible surface area (SASA) of the coiled coil as observed in the crystal structure to the SASA of the isolated coiled coil. SASA calculated using NACCESS.68 b RMSD to the closest ideal Crick backbone is the difference between the crystal structure and the best-fitting Crick ideal structure. Data for all structure is shown in Supplemental Figure S1.\n",
      "    unique: PMC2562949/PMC2562949.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Summary of pair terms used in ISM models\n",
      "  footer_text: A prime (’) designates a residue on the opposite helix. All interaction pairs listed involve structurally adjacent sites on opposite helices. For edge interactions where there may be some ambiguity as to what pair is indicated, the interactions are as follows: g-e’ pairs in parallel coiled coils are between a g residue and the e residue of the next (mor C-terminal) heptad of the opposite helix; n antiparallel coiled coils g-g’ pairs are between a g residue and the g residue of the previous (more N-terminal) heptad of the opposit helix and e-e’ pairs are between an e residue and the e residue of the next (more C-terminal) heptad of the opposite helix.\n",
      "    unique: PMC2562949/PMC2562949.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2562989.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562949.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2562949.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563044.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563044.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2563044.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Summary of UO2 Synthesis and Main Characteristics of the Study Materials\n",
      "  footer_text: a Courtesy of Dan Schwarz and David Clark, Los Alamos National Laboratory. b Multipoint N2-BET. c Calculated from particle morphology visualized by scanning electron microscopy (SEM). d Derived from high-resolution transmission electron microscopy (HR-TEM) and synchrotron-based powder diffraction (SR-PD). e Lower value calculated from SSA, higher value taken from SEM images (upper limit). f Based on SEM images.\n",
      "    unique: PMC2563044/PMC2563044.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Comparison of UO2 Dissolution Rates Measured under Reducing, Carbonate-Free Conditions (Rm and Rn are means Calculated by eqs 2 and 3; the Rate Constant k is Obtained from the Dissolution Model (eq 5) Assuming Steady-State Was Reached; All Rates Account for the Time-Dependent Loss of Solid while Dissolution)\n",
      "  footer_text: a A thin film of 0.5 g UO2.0 was used in this work. b If no reference given, data are from the present work.\n",
      "    unique: PMC2563044/PMC2563044.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2563421.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of recommendations\n",
      "  footer_text: \n",
      "    unique: PMC2563421/PMC2563421.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2563799.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563421.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563799.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563799.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563957.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sequences of the Ib-IX-derived TM peptides used in this study\n",
      "  footer_text: a The TM sequence is indicated in bold and Cys residues underlined. b Residue C148 in the Ibβ cytoplasmic domain was changed to Ser (the residue in the rat sequence). c Residue C154 in the IX cytoplasmic domain was changed to Asn (the residue in the rat sequence).\n",
      "    unique: PMC2563799/PMC2563799.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The effect of detergents on production of the αβ2 peptide complex in thiol-disulfide exchange reactions. The molar ratio of Ibα-TM, Ibβ-TM, IX-TM (if included) and detergent was 1:2:1:200 in all reactions. The amount of αβ2 is measured through integration of the corresponding HPLC peak and expressed as the percentage of total Ibβ-TM in the reaction\n",
      "  footer_text: \n",
      "    unique: PMC2563799/PMC2563799.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2563805.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2563957.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Modification of Composition of Flora Within Biofilm With the Introduction of P. gingivalis to Healthy Tissues, Baseline Periodontitis, and Subsequent RvE1 Treatment in a Rabbit Model of Experimental Periodontitis\n",
      "  footer_text: Copyright 2007. The American Association of Immunologists.7 * Chemiluminescence units (scale of 0 to 5 with 0 indicating no signal): 1 = undetectable <104 bacteria; 2 = 105 bacteria; 3 = >105 but <106 bacteria; 4 = 106 bacteria; and 5 = >106 bacteria. † P <0.05 compared to healthy. ‡ Previously T. forsythensis.\n",
      "    unique: PMC2563957/PMC2563957.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2563959.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2563962.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: MAD Data Collection Statistics\n",
      "  footer_text: \n",
      "    unique: PMC2563962/PMC2563962.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563962.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2563962.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564287.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Data Collection and Refinement Statistics\n",
      "  footer_text: Numbers in the parentheses represent the statistics of the highest resolution shell.\n",
      "    unique: PMC2563962/PMC2563962.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564285.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2564287.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and Lifestyle Characteristics of men with and without Benign Prostate Hyperplasia in the Prostate Cancer Prevention Trial (PCPT)†.\n",
      "  footer_text: a) from t-test b) from chi-square test † All men in this study were in the placebo arm of the PCPT.\n",
      "    unique: PMC2564287/PMC2564287.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Definitions of BPH events in the Prostate Cancer Prevention Trial (n=704)\n",
      "  footer_text: a) Events were selected hierarchically in the order listed. b) International Prostate Symptom Score’s range from 0 to 30, with 0–7 indicating mildly symptomatic, 8–19 moderately symptomatic; and 20–35 severely symptomatic. * Included in definition of “clinical BPH”.\n",
      "    unique: PMC2564287/PMC2564287.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Associations of IGF1, IGFBP3 and IGF1:IGFBP3 with risk of BPH in the PCPT\n",
      "  footer_text: 1 Adjusted Adjusted for age at baseline and race 2 Adjusted for age at baseline, race and BMI 3 Polytomous logistic regression adjusting for age at baseline, race and BMI € The models for “all BPH” include all cases; the models for “clinical BPH” and “symptomatic progression BPH” are models for the two subtypes of BPH Quartile 1 is the referent for all models\n",
      "    unique: PMC2564287/PMC2564287.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Associations of IGF1, IGFBP3 and IGF1:IGFBP3 with Risk of Clinical BPH, stratified by age in the PCPT\n",
      "  footer_text: 1 Adjusted for BMI and race. Quartile 1 is the referent for all models\n",
      "    unique: PMC2564287/PMC2564287.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Associations of IGF1, IGFBP3 and IGF1:IGFBP3 with Risk of Clinical BPH, stratified by Body Mass Index (BMI) in the PCPT\n",
      "  footer_text: 1 Restricted to clinical cases only. 2 Adjusted for age and race Quartile 1 is referent for all models\n",
      "    unique: PMC2564287/PMC2564287.xml::T5-0"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564287.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564287.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564287.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564287.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564288.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564288.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564288.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564288.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "PMC0025XXXXX/PMC2564288.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Study 1. Factor Loadings for the 3-Factor Structure\n",
      "  footer_text: N = 412; factor loadings over .40 in bold; Disengage = Disengagement Factor, Negative/Dissatis. = Negativity/Dissatisfaction Factor, Positiv./Closeness = Positivity/Closeness Factor. Items reordered for ease of presentation.\n",
      "    unique: PMC2564288/PMC2564288.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Study 2. Descriptive Statistics: Means and Standard Deviations across Samples\n",
      "  footer_text: Note. RDI = Relational Distance Index. Dating sample: N = 203 for all variables except RDI; for RDI scales, n = 63. Marital sample (husbands and wives): N = 77 couples. Physically victimized women: N = 42. All means and standard deviations were converted to their respective response scales.\n",
      "    unique: PMC2564288/PMC2564288.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Study 2. Factor Loadings on the 1-Factor Romantic Disengagement Scale across Samples\n",
      "  footer_text: Note. A complete list of items included in initial and revised item pools may be obtained from first author. Based on our proposed conceptualization, items 1–7 represent emotional distance, items 8 and 9 represent cognitive distance, and items 10–18 represent behavioral distance; however, factor analyses did not distinguish components. All factor loadings were significant at p < .001. Items were reordered in this table for ease of presentation.\n",
      "    unique: PMC2564288/PMC2564288.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Study 2. Bivariate Correlations between the Romantic Disengagement Scale and Variables Included to Support Discriminant and Incremental Validity\n",
      "  footer_text: Note. RDI = Relational Distance Index. * p < .05. ** p < .01.\n",
      "    unique: PMC2564288/PMC2564288.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Study 2. Hierarchical Regression Analyses Demonstrating Incremental Validity of the Romantic Disengagement Scale (RDS) over Hess’s Relational Distancing Index (RDI)\n",
      "  footer_text: Note. RDI = Relational Distancing Index. + p < .10. * p < .05. ** p < .01.\n",
      "    unique: PMC2564288/PMC2564288.xml::T5-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564288.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564601.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564601.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564605.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2564601.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of Abused and Non-abused Low-Income Rural Pregnant Women (N = 50)\n",
      "  footer_text: \n",
      "    unique: PMC2564601/PMC2564601.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Stressors Reported by Abused and Non Abused Rural Women to Research Nurses (N = 50)\n",
      "  footer_text: \n",
      "    unique: PMC2564601/PMC2564601.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564605.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Stability of Affinity Grids in the presence of detergents as assessed by visual inspection of the specimens in the electron microscope.\n",
      "  footer_text: * according to Anatrace Inc. (Maumee, OH); **according to Sigma Chemical Company (St. Louis, MO) ++ No apparent degradation of the lipid monolayer (continuous lipid monolayer) + Minor degradation of the lipid monolayer (appearance of some small holes in the lipid monolayer and of some small lipid vesicles) − Major degradation of the lipid monolayer (presence of many small holes in the lipid monolayer and of many small lipid vesicles) − − Almost complete degradation of the lipid monolayer (large grid areas without lipid monolayer\n",
      "    unique: PMC2564605/PMC2564605.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Stability of Affinity Grids in the presence of glycerol as assessed by visual inspection of the specimens in the electron microscope.\n",
      "  footer_text: ++ No apparent degradation of the lipid monolayer (continuous lipid monolayer) + Minor degradation of the lipid monolayer (appearance of some small holes in the lipid monolayer and of some small lipid vesicles) − Major degradation of the lipid monolayer (presence of many small holes in the lipid monolayer and of many small lipid vesicles) −− Almost complete degradation of the lipid monolayer (large grid areas without lipid monolayer)\n",
      "    unique: PMC2564605/PMC2564605.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564794.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of physiological parameters between QC and SC fibers and their responses to pruritic stimuli (cowhage and histamine)\n",
      "  footer_text: QC fibers had significantly smaller mechanical and heat thresholds, and the responses to cowhage were significantly larger, with a significantly higher intraburst frequency and a significantly smaller interburst interval. Histamine responses did not differ between QC and SC fibers. * p < 0.01, Mann-Whitney U test (QC vs SC); ** p < 0.01, unpaired t test (QC vs SC); *** p < 0.001, unpaired t test (QC vs SC).\n",
      "    unique: PMC2564794/PMC2564794.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564798.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Clinical characteristics of POH and other GNAS-based disorders of superficial heterotopic ossification (HO).\n",
      "  footer_text: All (+) or no (−) patients within the diagnostic category displayed the indicated characteristic. a Deep HO refers to the extension of superficial (dermal) HO to deep tissues as described in the text. b AHO features included the following characteristics: short stature, obesity, round face, brachydactaly, and neurobehavioral abnormalities. The presence of heterotopic ossification was common to all presentations and excluded here. c An endocrine evaluation included a survey of calcium, phosphorus, TSH, and intact PTH blood levels. Two POH patients had endocrine abnormalities: one had a high TSH, and another had high calcium and phosphate levels. One POH/AHO patient had a high phosphate level. All POH/PHP 1a/1c and PHP 1a/1c patients had PTH resistance or PTH and thyroid hormone resistance. Differences found to be statistically significant when compared to POH are d p value < 0.0001 e p value = 0.0002. Abbreviations are as defined in the text.\n",
      "    unique: PMC2564798/PMC2564798.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: AHO features among patients with POH and other GNAS-based disorders of superficial heterotopic ossification.\n",
      "  footer_text: \n",
      "    unique: PMC2564798/PMC2564798.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564605.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564794.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564798.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564798.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2564809.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2564811.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: An example of the overall fit indices inferred by the SEM for Model 1 at two voxels (with Talaraich coordinates in units of mm) in one subject’s brain. Voxel A was selected from the red region of Figure 4, and Voxel B was selected from outside that region.\n",
      "  footer_text: \n",
      "    unique: PMC2564811/PMC2564811.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The region of peak activation value (determined by AGFI) detected by six models at two hemispheres. The Talaraich coordinates of regions, standardized path weights and AGFIs in the models, were inter-subject averaged. The numbers of subjects represent how many of the seven subjects’ data showed activity in this area. The values are presented as: Mean ± Standard Deviation. The unit of coordinates in Talaraich space is mm.\n",
      "  footer_text: \n",
      "    unique: PMC2564811/PMC2564811.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: The ROIs used in SEM analysis at two hemispheres. The ROIs were defined by a series of t-tests and anatomic landmarks, as described in Methods and Materials. Talaraich coordinates of peak activity (determined by p-value) in the region were averaged among seven subjects. The time courses of each ROI were tested by a corresponding structural equation model (referred as “Tested Model” in table). The last column is the inter-subject averaged AGFI derived from the model. The values are presented as: Mean ± Standard Deviation. The unit of coordinates in Talaraich space is mm.\n",
      "  footer_text: \n",
      "    unique: PMC2564811/PMC2564811.xml::T3-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564811.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564811.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564811.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564827.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2564827.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Biographical data for the subjects tested.\n",
      "  footer_text: \n",
      "    unique: PMC2564827/PMC2564827.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Percent correct agreement between the channels selected by ACE and the channels selected by IdBM in different background conditions.\n",
      "  footer_text: \n",
      "    unique: PMC2564827/PMC2564827.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Average performance, in terms of hits and false alarm rates6, of three SNR estimation algorithms that were used to compute the binary mask.\n",
      "  footer_text: \n",
      "    unique: PMC2564827/PMC2564827.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564854.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Intended and Unintended Damage for All Experimental Groups\n",
      "  footer_text: Data are the estimated percentage of normal volume as assessed from MR images. Areas 11, 12, 13, and 14 – cytoarchitectonic subregions of the macaque frontal lobe as defined by Carmichael and Price (1994); Ia – agranular insular area as defined by Carmichael and Price (1994); ERh – entorhinal cortex and PRh – perirhinal cortex as defined by Amaral and colleagues (Amaral et al., 1987; Insausti et al., 1987); L – percentage of damage to the left hemisphere; R – percentage of damage to the right hemisphere; Avg – average of L and R; W = (L × R)/100 [weighted index as defined by Hodos and Bobko (1984)]; X – group mean.\n",
      "    unique: PMC2564854/PMC2564854.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564827.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564827.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564854.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564854.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564857.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564857.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564857.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Behavioral Ethogram\n",
      "  footer_text: List of all behaviors recorded along with brief definitions. All behaviors were analyzed for frequency (total number of occurrences). * Behavior for which total duration was also measured. NOTE: An additional general category of Facial Expressions was also analyzed, which included Lipsmack, Fear Grimace, Mild Threat and Full Threat.\n",
      "    unique: PMC2564854/PMC2564854.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564857.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographics of Doxycycline Treated and Control Study Populations\n",
      "  footer_text: \n",
      "    unique: PMC2564857/PMC2564857.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Study Parameters at Baseline\n",
      "  footer_text: \n",
      "    unique: PMC2564857/PMC2564857.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Procedural parameters and peri-procedural demographic data\n",
      "  footer_text: \n",
      "    unique: PMC2564857/PMC2564857.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564858.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Effects of the various treatment conditions on food-reinforced responding Rats were trained to respond under a multiple, alternating schedule of cocaine (0.25 mg/kg/infusion) and food reinforcement. Values represent the means (± S.E.M.) for N=8. None of the treatments (except for extinction conditions) significantly altered food-maintained responding from values seen when the vehicle was administered.\n",
      "  footer_text: * p<0.05 compared to all other treatment conditions.\n",
      "    unique: PMC2564858/PMC2564858.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564858.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564858.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564858.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564858.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564867.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Effects of the various treatment conditions on food-reinforced responding Rats were trained to respond under a multiple, alternating schedule of food and either 0.125 mg/kg/infusion (left side) or 0.5 mg/kg/infusion (right side) cocaine reinforcement. Values represent the means (± S.E.M.) for N=8. None of the treatments (except for extinction conditions) significantly altered food-maintained responding from values seen when the vehicle was administered.\n",
      "  footer_text: * p<0.05 compared to all other treatment conditions.\n",
      "    unique: PMC2564858/PMC2564858.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Effects of the various treatment conditions on plasma corticosterone (ng/ml) Rats were trained to respond under a multiple, alternating schedule of cocaine (0.25 mg/kg/infusion) and food reinforcement. Values represent the means (± S.E.M.) for N=8.\n",
      "  footer_text: \n",
      "    unique: PMC2564858/PMC2564858.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Effects of the various treatment conditions on plasma corticosterone (ng/ml) Rats were trained to respond under a multiple, alternating schedule of food and either 0.125 mg/kg/infusion (left side) or 0.5 mg/kg/infusion (right side) cocaine reinforcement. Values represent the means (± S.E.M.) for N=8. None of the treatments significantly altered plasma corticosterone from values seen when the vehicle was administered.\n",
      "  footer_text: \n",
      "    unique: PMC2564858/PMC2564858.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564867.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient demographic characteristics\n",
      "  footer_text: * Data presented is not mutually exclusive; patients may have reported more than one work status. FM, fibromyalgia\n",
      "    unique: PMC2564867/PMC2564867.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Patient clinical characteristics\n",
      "  footer_text: * One participant reported being diagnosed six months ago. FM, fibromyalgia\n",
      "    unique: PMC2564867/PMC2564867.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564867.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564871.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564891.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Key fibromyalgia domains identified by patients with fibromyalgia*\n",
      "  footer_text: * Not ranked by order of importance\n",
      "    unique: PMC2564867/PMC2564867.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564871.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Probabilities for formation of structural elements within subsets of conformations sampled during kinetic folding simulations\n",
      "  footer_text: a The average fraction of native contacts formed for various structural elements was computed for conformations in kinetic simulations with 18 Å < Rg < 20 Å and 0.29 < QTotal < 0.42= 0.37, and similarly for the regions of conformational space specified in columns 3–5. b Dashes are used to denote native contacts between two elements of secondary structure: e.g., between strands 1 and 2 (β1–β2) or between helix 1 and the rest of the protein (α1–rest).\n",
      "    unique: PMC2564871/PMC2564871.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564891.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Data-collection, phasing and refinement statistics Values in parentheses are for the outermost shell.\n",
      "  footer_text: † Rmerge = Σhkl Σi |Ii(hkl) − 〈I(hkl)〉|/Σhkl Σi Ii, where 〈I(hkl)〉 is the mean intensity of all symmetry-related reflections Ii(hkl). ‡ Rcryst = (Σ|Fobs −Fcalc|)/ΣFobs. Rfree as for Rcryst but using a random subset of the data (5%) not included in the refinement.\n",
      "    unique: PMC2564891/PMC2564891.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2564994.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2564996.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinical characteristics of patients enrolled on study\n",
      "  footer_text: \n",
      "    unique: PMC2564996/PMC2564996.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Summary of median time to low T and normalization of T and DHT\n",
      "  footer_text: \n",
      "    unique: PMC2564996/PMC2564996.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Cox proportional hazards model† for factors associated with T rise in cycle 1 and cycle 2\n",
      "  footer_text: * Groups with delayed T rise listed first; † The final model, resulting from backward elimination, is presented. Factors included in the initial models, but which did not meet criteria to remain in the final model were baseline T in cycle 1 (for both models in cycle 1), and prior RT (for both models in cycle 2).\n",
      "    unique: PMC2564996/PMC2564996.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Clinical characteristics of patients proceeding to cycle 2\n",
      "  footer_text: \n",
      "    unique: PMC2564996/PMC2564996.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565494.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564996.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564996.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2564996.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2565496.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565508.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565509.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sociodemographic Characteristics of the Sample (N=126)\n",
      "  footer_text: \n",
      "    unique: PMC2565509/PMC2565509.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Means of Symptom Control for Variables for Each Class\n",
      "  footer_text: Notes: - WCA Well-Controlled Accurate Group; IG Inaccurate Group; PWC Poorly-Controlled Accurate Group - a=WCA vs. IG-1; b=WCA vs. IG-2; c=WCA vs. PCA; d=IG-1 vs. IG-2; e=IG-1 vs. PCA; f=IG-2 vs. PCA\n",
      "    unique: PMC2565509/PMC2565509.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Means of Psychosocial Factors in Three Groups\n",
      "  footer_text: Notes: - WCA Well-Controlled Accurate Group; IG Inaccurate Group; PWC Poorly-Controlled Accurate Group - a=WCA vs. IG; b=WCA vs. PCA; c=IG vs PCA\n",
      "    unique: PMC2565509/PMC2565509.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565510.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Clinical data (mean and standard deviation) of the subjects\n",
      "  footer_text: \n",
      "    unique: PMC2565510/PMC2565510.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Prevalence and high load (≥106/sample) of the periodontopathogens Aggregatibacter actinomycetemcomitans, Porphyromonas gingivalis, Tannerella forsythia, Treponema denticola, and Prevotella intermedia in gingival crevicular fluid obtained from periodontitis patients as well as healthy controls\n",
      "  footer_text: \n",
      "    unique: PMC2565510/PMC2565510.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565509.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565509.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565510.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565510.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565510.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Correlation coefficients (Spearman test) between periodontopathogenic species and myeloperoxidase (MPO) as well as antimicrobial peptides\n",
      "  footer_text: * P < 0.05 ** P < 0.01.\n",
      "    unique: PMC2565510/PMC2565510.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565570.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565585.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: EXAFS Curve Fitting Results for 3H and DCH Mutantsa\n",
      "  footer_text: a Distances (angstroms) and disorder parameters [in parentheses, σ2 (×10−3 Å2)] derive from integer coordination number fits to Fourier-filtered EXAFS data. b Multiple scattering paths represent combined scattering paths described in Experimental Procedures. c Goodness of fit (Rf for fits to filtered data, Ru for fits to unfiltered data) defined as 1000Σi=1N {[Re(χicalc)]2 + [Im(χicalc)]2}/| Σi=1N {[Re(χiobs)]2 + [Im(χiobs)]2}, where N is the number of data points. d For 3H-BcII at pH 6, k = [1.2, 13.5]; R = [0.8, 2.0] for first-shell fits; and R = [0.1, 4.4] for ms fits. e For 3H-BcII at pH 8, k = [1.2, 13.1]; R = [0.8, 2.0] for first-shell fits; and R = [0.1, 4.4] for ms fits. f For DCH-BcII, k = [1.2, 13.5]; R = [0.8, 2.2] for first-shell fits; and R = [0.1, 4.4] for ms fits.\n",
      "    unique: PMC2565585/PMC2565585.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Kinetic Parameters for Hydrolysis of Nitrocefin, Cefotaxime, Benzylpenicillin, and Imipenem by WT BcII, 3H-BcII, and DCH-BcIIa\n",
      "  footer_text: a Parameters were determined by fitting initial reaction rates against substrate concentration. The experiments were carried out in 10 mM HEPES (pH 7.5), 200 mM NaCl, 50 μg/mL BSA, and 20 μM Zn(II) in the case of WT BcII and DCH-BcII. In the case of 3H-BcII, the experiments were carried out in 10 mM HEPES (pH 7.5), 200 mM NaCl, and 50 μg/mL BSA. For 3H-BcII, the same kcat/KM values were obtained with increasing Zn(II) concentrations (from 20 to 100 μM).\n",
      "    unique: PMC2565585/PMC2565585.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565587.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565601.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565585.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565585.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565601.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565601.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Botanicals Tested with Globo H-KLH vaccines, Source, Dose, and Active Ingredients\n",
      "  footer_text: \n",
      "    unique: PMC2565601/PMC2565601.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: ELISA Antibody Titers after Vaccination with Globo H-KLH plus or minus Botanicals\n",
      "  footer_text: * Reciprocal antibody titer. Number in parentheses is the number of mice with that titer, if more than 1. ** Groups in bold are significantly (p<0.05) higher than the no adjuvant control group.\n",
      "    unique: PMC2565601/PMC2565601.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Summary of ELISA Results of Individual Experiments and of Meta-analysis\n",
      "  footer_text: * Positive experiment = p value for higher titers induced by indicated botanical compared to no adjuvant or low dose GPI-0100 controls <0.05. ** Compared to mice receiving no adjuvant *** Botanical induced antibody titers that were significantly lower than the controls in these individual experiments.\n",
      "    unique: PMC2565601/PMC2565601.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: ELISA Antibody Titers after Vaccination with GD3-KLH plus or minus Botanicals\n",
      "  footer_text: * Reciprocal antibody titer. Number in parentheses is the number of mice with that titer, if more than 1. ** Groups in bold are significantly (p<0.05) higher than the no adjuvant control group. *** Extraction in the indicated percent Ethanol was performed and the precipitate (ppt) and solute (sol) tested.\n",
      "    unique: PMC2565601/PMC2565601.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565601.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565601.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565802.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565802.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2565602.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565685.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565802.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic Variables\n",
      "  footer_text: Differences in age and education were evaluated using an ANOVA; differences in gender and race were evaluated using a chi-square analysis. a F3,184=3.4, p=.02; SCZ>CON-SIB (p=.005), CON (p=.01). b χ2(3) = 34.9, p<.001; c F3,184=3.0, p=.03; SCZ<CON-SIB (p=.026), CON (p=.004).\n",
      "    unique: PMC2565802/PMC2565802.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mean Scores on the Dimensions of the TCI\n",
      "  footer_text: Values are least square means (standard errors) as assessed in a mixed model with age and gender as covariates. a F3,73=12.7, p<.001; SCZ>SCZ-SIB (p=.05), CON-SIB (p<.001), CON (p<.001). SCZ>CON-SIB (p=.002), CON (p<.001). b F3,67=7.9, p<.001; SCZ<SCZ-SIB (p<.001), CON-SIB (p=.02), CON (p=.02). c F3,76=7.5, p<.001; SCZ<SCZ-SIB (p<.001), CON-SIB (p=.004), CON (p<.001). d F3,72=3.8, p=.01; SCZ<SCZ-SIB (p=.002), CON-SIB (p=.04), CON (p=.02). e F3,67=5.7, p<.01; SCZ>CON-SIB (p=.006), CON (p<.001); SCZ-SIB>CON-SIB (p=.02), CON (p=.004).\n",
      "    unique: PMC2565802/PMC2565802.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Analysis of Temperament and Character Configurations\n",
      "  footer_text: Generalized linear model estimated marginal means (standard errors); these means can be interpreted as an expected fraction after controlling for age, gender, and sibling correlation. a χ2(3) = 24.9, p<.001; SCZ>SCZ-SIB (p=.017), CON-SIB (p<.001), CON (p<.001). b χ2(3) = 12.3, p=.006; SCZ<CON-SIB (p=.001), CON (p<.001); SCZ-SIB<CONSIB (p=.027). c χ2(3) = 7.9, p=.047; SCZ<CON (p=.02). d χ2(3) = 16.3, p=.001; SCZ<SCZ-SIB (p=.013), CON-SIB (p=.004), CON (p<.001). e The model does not converge due to n=0 in one or more cells. f χ2(3) = 15.7, p=.001; SCZ<SCZ-SIB (p=.001), CON-SIB (p<.001), CON (p<.001).\n",
      "    unique: PMC2565802/PMC2565802.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565861.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2565862.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565802.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565862.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2565862.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Predominant features of acute and chronic graft-versus-host disease (Filipovich et al, 2005)\n",
      "  footer_text: \n",
      "    unique: PMC2565862/PMC2565862.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Histopathologic studies of minor glands and clinical findings of GVHD salivary gland disease\n",
      "  footer_text: \n",
      "    unique: PMC2565862/PMC2565862.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2565865.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566523.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "tw(T1): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 1\n",
      "  caption_text: Single SNP association results for single nucleotide polymorphisms selected for replication from Finnish case–control cohort\n",
      "  footer_text: \n",
      "    unique: PMC2566523/PMC2566523.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Demographic factors of the Finnish case and control groups\n",
      "  footer_text: Other; migraine aura symptoms not fulfilling the International Headache Society aura criteria, but classified as migraine with aura by a clinical neurologist.\n",
      "    unique: PMC2566523/PMC2566523.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "tw(T3): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 3\n",
      "  caption_text: Replication cohort sizes for individuals successfully genotyped\n",
      "  footer_text: \n",
      "    unique: PMC2566523/PMC2566523.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "tw(T4): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 4\n",
      "  caption_text: Replication association analysis results\n",
      "  footer_text: \n",
      "    unique: PMC2566523/PMC2566523.xml::T4-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566523.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566531.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "tw(T5): {'id', 'orientation', 'position'}\n",
      "  label_text: Table 5\n",
      "  caption_text: Allele frequency data for rs1431656 and rs7076100 interaction\n",
      "  footer_text: \n",
      "    unique: PMC2566523/PMC2566523.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Breakdown of candidate gene categories\n",
      "  footer_text: \n",
      "    unique: PMC2566523/PMC2566523.xml::T6-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566531.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected baseline characteristics (mean (standard deviation), or percent) of the ARIC study cohort in relation to baseline exposure status\n",
      "  footer_text: \n",
      "    unique: PMC2566531/PMC2566531.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Baseline characteristics (mean (standard deviation), or percent) of the ARIC cohort stratified by menopausal and exposure status\n",
      "  footer_text: \n",
      "    unique: PMC2566531/PMC2566531.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566531.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566531.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566546.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566546.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Association of HDL-cholesterol and breast cancer incidence. ARIC study cohort, 1987–2000\n",
      "  footer_text: HR: hazard ratio, 95% CI: 95 percent confidence interval, SD: standard deviation, P-years: person-years * Regression model adjusted for age, race_by_ARIC center, age at menarche, body mass index, cigarette smoking status, reproductive hormone use, age at menopause (total sample and post-menopausal women only).\n",
      "    unique: PMC2566531/PMC2566531.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566540.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566546.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Average (+/−SEM) Associative Decision Times (ms) on Correct & Incorrect Trials\n",
      "  footer_text: \n",
      "    unique: PMC2566546/PMC2566546.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Design of Experiment 2\n",
      "  footer_text: \n",
      "    unique: PMC2566546/PMC2566546.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566547.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566585.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566635.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566738.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Primary Antibodies Used in This Study\n",
      "  footer_text: \n",
      "    unique: PMC2566738/PMC2566738.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566740.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of single point energies (kcal/mol) calculated for reactants, transition states, intermediates and product structures of reference uncatalyzed reaction (geometries were optimized at IEFPCM(εr=78.4)/B3LYP/6-31G(d,p) level, see Methods for details and Supporting information for structures labeled R, TS1, I1, TS2, I2, TS3, P, and P′ endo/exo) at MPW1K/6-31+G(d,p) and extrapolated CCSD(T)/CBS levels, corresponding errors of the MPW1K functional, solvation terms and the corrections to Gibbs energies at IEFPCM(εr=78.4)/B3LYP/6-31G(d,p) level, and extrapolated total Gibbs energies in water solution at the CCSD(T)/CBS level.\n",
      "  footer_text: a differences between CCSD(T)/CBS and MPW1K/6-31+G(d,p) energies. b solvation term calculated as the difference between IEFPCM(εr=78.4)/B3LYP/6-31G(d,p) and gas phase B3LYP/6-31G(d,p) SCF energies. c the corrections to Gibbs energies were calculated at the IEFPCM(εr=78.4)/B3LYP/6-31G(d,p) level.\n",
      "    unique: PMC2566740/PMC2566740.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The distances (in Å) between C75(N4) and G1(O1P) for various structures. Precursor, transition, and product states are labeled R, TS, and P, respectively. The labels PD, HD, and HS stand for different structural arrangements in the HDV ribozyme active site (see text and Figure 4).\n",
      "  footer_text: \n",
      "    unique: PMC2566740/PMC2566740.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566746.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566750.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566738.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566740.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566740.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566750.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic and behavioral characteristics of the sample (N = 30)\n",
      "  footer_text: a On average, 62.4% of participants' male partners were aware of their bisexuality b On average, 43.5% of participants' female partners were aware of their bisexuality\n",
      "    unique: PMC2566750/PMC2566750.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Participant relationship characteristics within the past 12 months (N = 30)\n",
      "  footer_text: a M = male relationship, F = female relationship b Participant 23 did not specify a number for casual male partners. He used “many” to describe them c Participant 30 had not had sex with his current male partner\n",
      "    unique: PMC2566750/PMC2566750.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566776.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant baseline characteristics by baseline depression status stratified by gender\n",
      "  footer_text: a p<0.05 comparing difference between depressed and not depressed for overall sample based on χ2 test or t-test b p<0.05 comparing difference between depressed and not depressed for males based on χ2 test or t-test c p<0.05 comparing difference between depressed and not depressed for females based on χ2 test or t-test\n",
      "    unique: PMC2566776/PMC2566776.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566750.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566776.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566776.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566776.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Regression coefficients for self-reported BMI (kg/m2) at 12-month visit, stratified by gender.\n",
      "  footer_text: a Income categories: < $25,000, $25,000–34,999, $35,000–49,999, $50,000–74,999, $75,000–99,999, ≥$100,000) b Education categories: less than high school graduate, high school graduate, some college, college graduate or further c Race/ethnic categories: Black or Pacific Islander, other d Pubertal stage categories: Stage 1 or 2, Stage 3, Stage 4 or 5 e Based on self-reported height and weight; modeled as a continuous variable\n",
      "    unique: PMC2566776/PMC2566776.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean (SD) baseline and 12-month BMI (kg/m2) calculated from self-reported and measured height and weight values, stratified by baseline depressive symptom status and gender\n",
      "  footer_text: a Boys b Girls\n",
      "    unique: PMC2566776/PMC2566776.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Regression coefficients for measured BMI at 12-months stratified by gender\n",
      "  footer_text: a Income categories: < $25,000, $25,000–34,999, $35,000–49,999, $50,000–74,999, $75,000–99,999, ≥$100,000) b Education categories: less than high school graduate, high school graduate, some college, college graduate or further c Race/ethnic categories: Black or Pacific Islander, other d Pubertal stage categories: Stage 1 or 2, Stage 3, Stage 4 or 5 e Based on measured height and weight; modeled as a continuous variable\n",
      "    unique: PMC2566776/PMC2566776.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566783.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566784.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Real-time PCR primers: The primer sequences are reported in 5’→3’ orientation.\n",
      "  footer_text: * The accession number used for designing primers for each gene is indicated in the first column in parenthesis.\n",
      "    unique: PMC2566784/PMC2566784.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566776.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566784.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566790.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2566790.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Patient demographics\n",
      "  footer_text: \n",
      "    unique: PMC2566790/PMC2566790.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Differentially expressed genes in three overrepresented biological processes associated with celecoxib pre-treatment\n",
      "  footer_text: NOTE: Fold change represents the difference in expression in celecoxib pre-treated patients versus patients receiving no drug pretreatment. * P-value represents the likelihood that the number of differentially expressed genes belonging to the biological process occurred by chance as determined by the Fischer’s exact test.\n",
      "    unique: PMC2566790/PMC2566790.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566819.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean subjective effects and psychomotor values in which a significant Group (light [LD] vs. moderate drinking [MD] history) x Nitrous Oxide Concentration (0, 20, 30, 40%) x Sample Drug (placebo [PLC], nitrous oxide [N2O]) interaction was found.\n",
      "  footer_text: For the sake of clarity the 0% nitrous oxide condition is omitted. Numbers in bold font indicate that values differ significantly from placebo as determined by Tukey post hoc testing. a Endpoints of 100-mm VAS: 0=not at all; 100=extremely. b Measures assessed at 5 min into the sampling period in which subjects were asked to rate their response based on how they currently felt. c Tukey post hoc testing revealed significant differences existed between light drinkers (LD) and moderate drinkers (MD). d Responses were made on a 100-mm bipolar VAS. For “drug liking” and “odor liking,” 0=dislike a lot; 50=neutral; 100=like a lot, and for “inhale again,” 0=definitely would not; 50=neutral (don’t care); 100=definitely would. e Dependent measure assessed 5 min after sampling periods; numbers represent mean number of statements endorsed out of a possible 10.\n",
      "    unique: PMC2566819/PMC2566819.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566790.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566819.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566821.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2566821.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean (SD) Paw Withdrawal Latencies (in seconds) by Group at Baseline and Following Exercise\n",
      "  footer_text: * Notes: p < .05 ** p < .01.\n",
      "    unique: PMC2566821/PMC2566821.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566849.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566857.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Demographic information about the three CAH subgroups and the control group.\n",
      "  footer_text: BMI = Body Mass Index.\n",
      "    unique: PMC2566857/PMC2566857.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566897.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sources, Types, and Examples of Chemicals Identified as Potential Endocrine Disruptors (Adapted [2])\n",
      "  footer_text: \n",
      "    unique: PMC2566897/PMC2566897.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566905.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566857.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566897.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566905.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Probe list with numbers of in situ hybridization tests per developmental stage group. Genes (accession number and sequence) for Wnt ligands, Frizzled receptors, and Wnt antagonists analyzed in this study on inner ear tissue from chicken embryos ranging from stage (s)24 to 41 (according to Hamburger and Hamilton, 1951) embryonic day (E)4 to 15.\n",
      "  footer_text: Shaded in gray are genes that were not detected in ventral inner ear tissue (Wnt1, Wnt2b, and Cer). Those three probes were not tested beyond s38 (E12).\n",
      "    unique: PMC2566905/PMC2566905.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566937.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Percent inhibition of [3H]muscimol binding to well-washed membranes from rat cortex and cerebellum by 100 μM and 1 mM of test compounds. Given are the means ± S.D. with the number of experiments in brackets. Compound 17a was insoluble at 1 mM and therefore not tested at this concentration. Structure I applies except were II is specifically indicated. Compounds for which an IC50 is reported in Table 2 are shaded grey.\n",
      "  footer_text: * inactive.\n",
      "    unique: PMC2566937/PMC2566937.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T5-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T6-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566937.xml::T7-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Binding parameters of selected compounds against 6 nM [3H]muscimol in crude membrane preparations of rat cortex and cerebellum determined by nonlinear regression. Given are the means ± S.E.M. of the decadic logarithm of the IC50 in μM, the corresponding pseudo-Hill coefficient η and the number of experiments n.\n",
      "  footer_text: \n",
      "    unique: PMC2566937/PMC2566937.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: GABA concentrations and the corresponding relative effective concentration that induces a response around the EC20 when applied to GABAA receptors αiβ3γ2 (i=1–6) transiently expressed in HEK 293 cells. Given are the mean ± S.E.M.\n",
      "  footer_text: \n",
      "    unique: PMC2566937/PMC2566937.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Maximal intrinsic activity of 1 mM 5d normalized to the approximate maximal GABA-induced current (I/ImaxGABA) and the 5d EC50 and Hill coefficient on αiβ3γ2 (i=1–6) GABAA receptors. Data are means ± S.E.M.\n",
      "  footer_text: * marks values of an extrapolated dose-response curve.\n",
      "    unique: PMC2566937/PMC2566937.xml::T4-0\n",
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Summary of dose-response data and 6a competition data using a GABA EC20 concentration from cysteine-substituted and wt α1β2 GABAA receptors.\n",
      "  footer_text: \n",
      "    unique: PMC2566937/PMC2566937.xml::T5-0\n",
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Extent of inhibition of GABA-induced currents after reaction with MTS reagent.\n",
      "  footer_text: Inhibition (% ± S.E.M. (n)) for GABA EC50 currents after complete reaction with MTSES− and MTSEA-biotin on GABAA receptors containing the cited cys-engineered α1 subunit together with the β2 subunit expressed in Xenopus oocytes. n.d. = not determined.\n",
      "    unique: PMC2566937/PMC2566937.xml::T6-0\n",
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: MTS-reagent reaction rates.\n",
      "  footer_text: \n",
      "    unique: PMC2566937/PMC2566937.xml::T7-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566943.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566947.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566953.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Behavioral data from the pilot study (n = 15). Mean RT (ms) and mean accuracy (% correct) with standard deviations (SD) for all experimental conditions\n",
      "  footer_text: \n",
      "    unique: PMC2566953/PMC2566953.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Behavioral data from the fMRI study (n = 15). Mean RT (ms) and mean accuracy (% correct) with standard deviations (SD) for all experimental conditions\n",
      "  footer_text: \n",
      "    unique: PMC2566953/PMC2566953.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Areas exhibiting significant activation differences in planned comparisons, thresholded at voxel level p < .025, cluster level p < .05 (≥ 51 contiguous voxels)\n",
      "  footer_text: Note. Coordinates, indicating the maximum intensity voxel for each cluster, are in Talairach and Tournoux space. L = left. R = Right. BA = Brodmann’s area.\n",
      "    unique: PMC2566953/PMC2566953.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Appendix\n",
      "  caption_text: \n",
      "  footer_text: \n",
      "    unique: PMC2566953/PMC2566953.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566957.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2566960.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The set of labeled cells chosen for analysis, with their PSD95 densities.\n",
      "  footer_text: \n",
      "    unique: PMC2566960/PMC2566960.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566953.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566953.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566953.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566953.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566960.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2566962.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Properties and Assumptions of Continuous Autoregulation Monitoring Modalities\n",
      "  footer_text: Licox indicates parenchymal oxygen tension monitor; CMRO2, cerebral metabolic rate of oxygen; and HCT, hematocrit.\n",
      "    unique: PMC2566962/PMC2566962.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Physiologic Parameters (±SD) of the Piglets at Normal and Low CPP\n",
      "  footer_text: \n",
      "    unique: PMC2566962/PMC2566962.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2566963.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567021.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline subject characteristics (n = 132)\n",
      "  footer_text: BMI, body mass index; NYHA, New York Heart Association; LV, left ventricle; ACE, angiotensin converting enzyme; ARB, angiotensin II receptor blocker; NT-proBNP, aminoterminal pro-B-type natriuretic peptide; eGFR, estimated glomerular filtration rate; ADMA, asymmetric dimethylarginine; SDMA, symmetric dimethylarginine; MMA, N-mono-methylarginine; IQR, inter-quartile range.\n",
      "    unique: PMC2567021/PMC2567021.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Univariate correlations between plasma levels of methylated arginine and lysine metabolites and clinical and echocardiographic indices\n",
      "  footer_text: ADMA, asymmetric dimethylarginine; SDMA, symmetric dimethylarginine; MMA, N-mono-methylarginine; BMI, body mass index; NT-proBNP, aminoterminal pro-B-type natriuretic peptide; eGFR, estimated glomerular filtration rate; LV, left ventricle; LA, left atrial; PV, pulmonary vein; RV, right ventricular.\n",
      "    unique: PMC2567021/PMC2567021.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566962.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2566962.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567021.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567021.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567021.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Arginine metabolite levels in patients taking vs. not taking beta-blocker at baseline\n",
      "  footer_text: Data expressed in median (inter-quartile range). ADMA, asymmetric dimethylarginine; SDMA, symmetric dimethylarginine; MMA, N-mono-methylarginine.\n",
      "    unique: PMC2567021/PMC2567021.xml::T3-0\n",
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Univariate and multivariable hazard ratios for predicting death, transplant, or heart failure hospitalization for plasma levels of methylated arginine metabolites\n",
      "  footer_text: a Hazard ratios per 1-SD increment [1-SD for ADMA = 0.14 μM; 1-SD for SDMA = 0.12 μM; 1-SD for MMA = 14 nM; 1-SD for all methylarginines (ADMA+SDMA+MMA) = 0.24 μM; 1-SD for methyl-lysine = 1.81 μM]. b Multivariable adjustments including age, eGFR, and LV ejection fraction (1-SD for LVEF = 5.99%; 1-SD for age = 13.3 years; 1-SD for eGFR = 20.9 mL/min per 1.73 m2). Hazard ratios per 1-SD increment ADMA = 0.14 μM.\n",
      "    unique: PMC2567021/PMC2567021.xml::T4-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567049.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table I\n",
      "  caption_text: Association of mtDNA markers with multiple sclerosis (MS) and systemic lupus erythematosus SLE. All p<0.05 are indicated. After correction for multiple testing, significant associations are observed for the mtDNA nt9055 variant of the ATP6 gene (in haplogroup K) in both MS and SLE, and for the nt4719 variant of the ND2 gene (in haplogroup T) in MS.\n",
      "  footer_text: \n",
      "    unique: PMC2567049/PMC2567049.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table II\n",
      "  caption_text: Association of mtDNA haplotypes with multiple sclerosis (MS) and systemic lupus erythematosus (SLE).\n",
      "  footer_text: \n",
      "    unique: PMC2567049/PMC2567049.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567051.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of the genomes of a central African strain of monkeypox virus (Zaire-1979) and a west African strain of monkeypox (USA-2003)\n",
      "  footer_text: Abbreviations: aa: amino acids; BR: cowpox virus strain Brighton Red; C-term: C-terminal fragment; COP: vaccinia virus, strain Copenhagen; CPV: cowpox virus; IL: Interleukin; MPV-Zar_1979_005: central African strain of monkeypox virus; MPV-USA_2003_039: west African strain of monkeypox virus; N-term: N-terminal fragment.\n",
      "    unique: PMC2567051/PMC2567051.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567021.xml::T4-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567049.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567049.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567051.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of monkeypox virus and variola virus genomes A. Genes with known function present in monkeypox virus and absent/fragmented in variola virus\n",
      "  footer_text: Abbreviations: aa: amino acids; BR: Cowpox virus strain Brighton Red; C-term: C-terminal fragment; COP: vaccinia virus, strain Copenhagen; CPV: cowpox virus; eIF: eukaryotic initiation factor; ER: endoplasmic reticulum; IFN: interferon; IL-1: Interleukin-1; MPV: central African strain of monkeypox virus Zaire_1979_005; N-term: N-terminal fragment; ORF: open reading frame; Ser/Thr: serine/threonine; VARV: variola virus strain Bangladesh75maj.\n",
      "    unique: PMC2567051/PMC2567051.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of monkeypox virus and vaccinia virus genomes A. Genes with known function present in monkeypox virus and absent/fragmented in vaccinia virus (ACAM2000)\n",
      "  footer_text: Abbreviations: aa: amino acids; ACAM2000: vaccine strain of vaccinia virus; BR: cowpox virus strain Brighton Red; C-term: C-terminal fragment; COP: vaccinia virus, strain, Copenhagen; CPV: cowpox virus; eIF: eukaryotic initiation factor; IFN: interferon; IL: interleukin; MPV: central African strain of monkeypox virus Zaire_1979_005; N-term: N-terminal fragment; ORF: open reading frame; TNF: tumor necrosis factor.\n",
      "    unique: PMC2567051/PMC2567051.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567053.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567056.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567058.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567051.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567051.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567058.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567058.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Culture Medium\n",
      "  footer_text: \n",
      "    unique: PMC2567058/PMC2567058.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison Between Mesenchymal Stem Cells (MSCs) and Nucleus Pulposus (NP) Cells\n",
      "  footer_text: \n",
      "    unique: PMC2567058/PMC2567058.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567059.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567102.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Signal intensities of different tissues in the No-Tuning (NT), Anatomy (ANAT), and High-Tuning (HT) C-SENC images. High = signal intensities greater than 0.8 in normalized images; and Low = signal intensities less than 0.2 in normalized images.\n",
      "  footer_text: \n",
      "    unique: PMC2567102/PMC2567102.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567103.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567118.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Comparison of cryoablation procedures for AVNRT with the 4 mm versus 6 mm cryoablation catheter tips\n",
      "  footer_text: AVNRT: Atrioventricular nodal reentrant tachcyardia * Data encompasses all patients with intent to treat, including patients with cryoablation lesions lasting < 240 sec\n",
      "    unique: PMC2567118/PMC2567118.xml::T1-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567102.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567118.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567118.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567123.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Comparison of patients with and without AVNRT recurrence following an initially successful cryoablation.\n",
      "  footer_text: AV: atrioventricular AVNRT: atrioventricular nodal reentrant tachycardia\n",
      "    unique: PMC2567118/PMC2567118.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567123.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: The Nine Items Translated “Without Complication”\n",
      "  footer_text: \n",
      "    unique: PMC2567123/PMC2567123.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: The Fifteen Items Translated “With Complication”\n",
      "  footer_text: \n",
      "    unique: PMC2567123/PMC2567123.xml::T2-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567124.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567128.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Participant Characteristics at Enrollment\n",
      "  footer_text: \n",
      "    unique: PMC2567128/PMC2567128.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Mixed Linear Regression Results for Drinking Outcomes and Secondary Outcomes of Sleep Quality and Quality of Life†\n",
      "  footer_text: † Mixed linear model including linear and quadratic change in the outcomes over time (measured at baseline 1-, 3-, and 6-month intervals), intervention difference at baseline, and intervention by linear and quadratic time interactions. *P <0.05 ** P < 0.01 *** P <0.001\n",
      "    unique: PMC2567128/PMC2567128.xml::T2-0\n",
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Comparison of Study Medication Adherence and Side Effects, by Study Condition\n",
      "  footer_text: * From daily sleep log data over the first 4 weeks (total N=121) † From 2-week follow-up data (total N=128) ‡ Log-transformed variable\n",
      "    unique: PMC2567128/PMC2567128.xml::T3-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567130.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567132.xml\n",
      "\n",
      "PMC0025XXXXX/PMC2567141.xml\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567123.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567128.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567128.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567128.xml::T3-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0025XXXXX/PMC2567239.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: A comparison of the mass spectrometry-based phosphoproteomics methods that have been applied to quantify temporal dynamics in the ErbB signaling network.\n",
      "  footer_text: \n",
      "    unique: PMC2567239/PMC2567239.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567241.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Subject characteristics and cell yield for calcium imaging experiments\n",
      "  footer_text: \n",
      "    unique: PMC2567241/PMC2567241.xml::T1-0\n",
      "\n",
      "PMC0025XXXXX/PMC2567282.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Baseline Characteristics of the Sample\n",
      "  footer_text: IMED = Interim Medical History questionnaire; MAP = Mean Arterial Pressure; HAM-D = Hamilton Depression scale; CR = Care Recipient; CG = Caregiver\n",
      "    unique: PMC2567282/PMC2567282.xml::T1-0\n",
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Parameter Estimates for Variables Predicting (log10) Beta Receptor Sensitivity\n",
      "  footer_text: IMED = Interim Medical History questionnaire; MAP = Mean Arterial Pressure.\n",
      "    unique: PMC2567282/PMC2567282.xml::T2-0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567239.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567241.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567282.xml::T1-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n",
      "/home/kimlab1/strokach/anaconda3/lib/python3.6/site-packages/tables/path.py:112: NaturalNameWarning: object name is not a valid Python identifier: 'PMC2567282.xml::T2-0'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though\n",
      "  NaturalNameWarning)\n"
     ]
    }
   ],
   "source": [
    "with pd.HDFStore('pmc-tables-1.h5', 'w') as store:\n",
    "\n",
    "    for i, row in enumerate(manuscript_files.itertuples()):\n",
    "        print('\\n' + f\"{row.File}\")\n",
    "\n",
    "        results = {}\n",
    "        xml_file = MANUSCRIPT_DATA_DIR.joinpath(row.File)\n",
    "\n",
    "        count += 1\n",
    "        if count > 1_000:\n",
    "            break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best results</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Neutral Model I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>33.0</td>\n",
       "      <td>24.8</td>\n",
       "      <td>43.7</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>17.2</td>\n",
       "      <td>9.63</td>\n",
       "      <td>27.4</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.7</td>\n",
       "      <td>7.39</td>\n",
       "      <td>18.2</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.9</td>\n",
       "      <td>10.3</td>\n",
       "      <td>20.4</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Neutral Model II</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>34.0</td>\n",
       "      <td>26.5</td>\n",
       "      <td>45.1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>18.5</td>\n",
       "      <td>10.7</td>\n",
       "      <td>29.1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.3</td>\n",
       "      <td>6.86</td>\n",
       "      <td>18.4</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.3</td>\n",
       "      <td>10.9</td>\n",
       "      <td>21.4</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Neutral Model III</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>13.2</td>\n",
       "      <td>7.00</td>\n",
       "      <td>19.5</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>13.2</td>\n",
       "      <td>6.84</td>\n",
       "      <td>20.3</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Predictor            Average            Minimum            Maximum     % best results\n",
       "0               NaN    Neutral Model I    Neutral Model I    Neutral Model I    Neutral Model I\n",
       "1         Raw rates               33.0               24.8               43.7                  0\n",
       "2   Empirical Bayes               17.2               9.63               27.4                  4\n",
       "3   Poisson kriging               12.7               7.39               18.2                 16\n",
       "4    Statistic Δsαβ               15.9               10.3               20.4                  0\n",
       "5               NaN   Neutral Model II   Neutral Model II   Neutral Model II   Neutral Model II\n",
       "6         Raw rates               34.0               26.5               45.1                  0\n",
       "7   Empirical Bayes               18.5               10.7               29.1                  0\n",
       "8   Poisson kriging               12.3               6.86               18.4                 68\n",
       "9    Statistic Δsαβ               15.3               10.9               21.4                  0\n",
       "10              NaN  Neutral Model III  Neutral Model III  Neutral Model III  Neutral Model III\n",
       "11  Poisson kriging               13.2               7.00               19.5                  8\n",
       "12   Statistic Δsαβ               13.2               6.84               20.3                  4"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_df_dev"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = table_df_dev\n",
    "\n",
    "\n",
    "def add_section_index(df):\n",
    "    df = df.copy()\n",
    "\n",
    "    index_ = []\n",
    "    skiprows = []\n",
    "    for i, row in enumerate(df.itertuples()):\n",
    "        row_items = [v for v in row[1:] if pd.notnull(v)]\n",
    "        print(row_items)\n",
    "        if len(set(row_items)) == 1:\n",
    "            index_.append(row_items[0])\n",
    "            skiprows.append(row.Index)\n",
    "        elif i > 0:\n",
    "            assert index_\n",
    "            index_.append(index_[-1])\n",
    "        else:\n",
    "            return df\n",
    "    columns = ['section_index'] + list(df.columns)\n",
    "    df['index_'] = index_\n",
    "    df = df.drop(pd.Index(skiprows))\n",
    "    return df[['index_'] + [c for c in df.columns if c != 'index_']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "run_control": {
     "marked": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Neutral Model I', 'Neutral Model I', 'Neutral Model I', 'Neutral Model I']\n",
      "['Raw rates', '33.0', '24.8', '43.7', '0']\n",
      "['Empirical Bayes', '17.2', '9.63', '27.4', '4']\n",
      "['Poisson kriging', '12.7', '7.39', '18.2', '16']\n",
      "['Statistic Δsαβ', '15.9', '10.3', '20.4', '0']\n",
      "['Neutral Model II', 'Neutral Model II', 'Neutral Model II', 'Neutral Model II']\n",
      "['Raw rates', '34.0', '26.5', '45.1', '0']\n",
      "['Empirical Bayes', '18.5', '10.7', '29.1', '0']\n",
      "['Poisson kriging', '12.3', '6.86', '18.4', '68']\n",
      "['Statistic Δsαβ', '15.3', '10.9', '21.4', '0']\n",
      "['Neutral Model III', 'Neutral Model III', 'Neutral Model III', 'Neutral Model III']\n",
      "['Poisson kriging', '13.2', '7.00', '19.5', '8']\n",
      "['Statistic Δsαβ', '13.2', '6.84', '20.3', '4']\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index_</th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best results</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Raw rates</td>\n",
       "      <td>33.0</td>\n",
       "      <td>24.8</td>\n",
       "      <td>43.7</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>17.2</td>\n",
       "      <td>9.63</td>\n",
       "      <td>27.4</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.7</td>\n",
       "      <td>7.39</td>\n",
       "      <td>18.2</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.9</td>\n",
       "      <td>10.3</td>\n",
       "      <td>20.4</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Raw rates</td>\n",
       "      <td>34.0</td>\n",
       "      <td>26.5</td>\n",
       "      <td>45.1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>18.5</td>\n",
       "      <td>10.7</td>\n",
       "      <td>29.1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.3</td>\n",
       "      <td>6.86</td>\n",
       "      <td>18.4</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.3</td>\n",
       "      <td>10.9</td>\n",
       "      <td>21.4</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>13.2</td>\n",
       "      <td>7.00</td>\n",
       "      <td>19.5</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>13.2</td>\n",
       "      <td>6.84</td>\n",
       "      <td>20.3</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               index_        Predictor Average Minimum Maximum % best results\n",
       "1     Neutral Model I        Raw rates    33.0    24.8    43.7              0\n",
       "2     Neutral Model I  Empirical Bayes    17.2    9.63    27.4              4\n",
       "3     Neutral Model I  Poisson kriging    12.7    7.39    18.2             16\n",
       "4     Neutral Model I   Statistic Δsαβ    15.9    10.3    20.4              0\n",
       "6    Neutral Model II        Raw rates    34.0    26.5    45.1              0\n",
       "7    Neutral Model II  Empirical Bayes    18.5    10.7    29.1              0\n",
       "8    Neutral Model II  Poisson kriging    12.3    6.86    18.4             68\n",
       "9    Neutral Model II   Statistic Δsαβ    15.3    10.9    21.4              0\n",
       "11  Neutral Model III  Poisson kriging    13.2    7.00    19.5              8\n",
       "12  Neutral Model III   Statistic Δsαβ    13.2    6.84    20.3              4"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "add_section_index(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [],
   "source": [
    "table_df\n",
    "table_df_copy = table_df.copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'pmc_tables' from '/home/kimlab1/strokach/working/pmc_tables/pmc_tables/__init__.py'>"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "importlib.reload(pmc_tables)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index_</th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best results</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Raw rates</td>\n",
       "      <td>33.0</td>\n",
       "      <td>24.80</td>\n",
       "      <td>43.7</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>17.2</td>\n",
       "      <td>9.63</td>\n",
       "      <td>27.4</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.7</td>\n",
       "      <td>7.39</td>\n",
       "      <td>18.2</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.9</td>\n",
       "      <td>10.30</td>\n",
       "      <td>20.4</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Raw rates</td>\n",
       "      <td>34.0</td>\n",
       "      <td>26.50</td>\n",
       "      <td>45.1</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>18.5</td>\n",
       "      <td>10.70</td>\n",
       "      <td>29.1</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.3</td>\n",
       "      <td>6.86</td>\n",
       "      <td>18.4</td>\n",
       "      <td>68.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.3</td>\n",
       "      <td>10.90</td>\n",
       "      <td>21.4</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>13.2</td>\n",
       "      <td>7.00</td>\n",
       "      <td>19.5</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>13.2</td>\n",
       "      <td>6.84</td>\n",
       "      <td>20.3</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               index_        Predictor Average  Minimum  Maximum  % best results\n",
       "1     Neutral Model I        Raw rates    33.0    24.80     43.7             0.0\n",
       "2     Neutral Model I  Empirical Bayes    17.2     9.63     27.4             4.0\n",
       "3     Neutral Model I  Poisson kriging    12.7     7.39     18.2            16.0\n",
       "4     Neutral Model I   Statistic Δsαβ    15.9    10.30     20.4             0.0\n",
       "6    Neutral Model II        Raw rates    34.0    26.50     45.1             0.0\n",
       "7    Neutral Model II  Empirical Bayes    18.5    10.70     29.1             0.0\n",
       "8    Neutral Model II  Poisson kriging    12.3     6.86     18.4            68.0\n",
       "9    Neutral Model II   Statistic Δsαβ    15.3    10.90     21.4             0.0\n",
       "11  Neutral Model III  Poisson kriging    13.2     7.00     19.5             8.0\n",
       "12  Neutral Model III   Statistic Δsαβ    13.2     6.84     20.3             4.0"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pmc_tables.process_dataframe(table_df_copy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "MultiIndex(levels=[['Model-1: σ', 'Model-2: c', 'Perf. Measures vs. noise level'], ['1', '10', '150', '30', '50', 'Unnamed: 0_level_1']],\n",
       "           labels=[[2, 0, 0, 1, 1, 1], [5, 4, 2, 0, 1, 3]])"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_df_dev.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('Perf. Measures vs. noise level', 'Unnamed: 0_level_1'),\n",
       " ('Model-1: σ', '50'),\n",
       " ('Model-1: σ', '150'),\n",
       " ('Model-2: c', '1'),\n",
       " ('Model-2: c', '10'),\n",
       " ('Model-2: c', '30')]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(table_df_dev.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df = table_df_dev"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "def parse_string_column(col):\n",
    "    if col.startswith('Unnamed:'):\n",
    "        return ''\n",
    "    else:\n",
    "        return col\n",
    "\n",
    "    \n",
    "def process_column_names(df):\n",
    "    \n",
    "    new_columns = []\n",
    "\n",
    "    for column in df.columns:\n",
    "        if isinstance(column, str):\n",
    "            new_column = parse_string_column(column)\n",
    "        else:\n",
    "            new_column = \"\"\n",
    "            for col in column:\n",
    "                new_col = parse_string_column(col)\n",
    "                if new_col:\n",
    "                    new_column += ' / ' + new_col\n",
    "            new_column = new_column.strip('/ ')\n",
    "        new_columns.append(new_column)\n",
    "        \n",
    "    return new_columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Perf. Measures vs. noise level',\n",
       " 'Model-1: σ / 50',\n",
       " 'Model-1: σ / 150',\n",
       " 'Model-2: c / 1',\n",
       " 'Model-2: c / 10',\n",
       " 'Model-2: c / 30']"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "process_column_names(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0021XXXXX/PMC2136438.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Mean absolute error (MAE) of prediction for four alternate predictors. The average, minimum and maximum value calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the MAE for the geostatistical methods is lower than for the local empirical Bayes smoother.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% best prediction</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6984</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6413</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.7467</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2584</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2252</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2916</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2345</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1961</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2740</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">92</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">E-type estimate</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2348</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1967</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2750</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">92</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best prediction</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.6984</td>\n",
       "      <td>0.6413</td>\n",
       "      <td>0.7467</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.2584</td>\n",
       "      <td>0.2252</td>\n",
       "      <td>0.2916</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.2345</td>\n",
       "      <td>0.1961</td>\n",
       "      <td>0.2740</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>E-type estimate</td>\n",
       "      <td>0.2348</td>\n",
       "      <td>0.1967</td>\n",
       "      <td>0.2750</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Predictor  Average  Minimum  Maximum  % best prediction\n",
       "0        Raw rates   0.6984   0.6413   0.7467                NaN\n",
       "1  Empirical Bayes   0.2584   0.2252   0.2916                NaN\n",
       "2  Poisson kriging   0.2345   0.1961   0.2740               92.0\n",
       "3  E-type estimate   0.2348   0.1967   0.2750               92.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Average of the reference risk values and the estimates obtained using four alternate approaches. The average, minimum and maximum value of the statistic calculated over 25 simulated rate maps are reported.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Reference risk</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.209</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.210</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.083</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.295</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.303</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.227</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.364</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.242</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.165</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.300</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">E-type estimate</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.242</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.164</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.300</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Reference risk</td>\n",
       "      <td>7.209</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>7.210</td>\n",
       "      <td>7.083</td>\n",
       "      <td>7.295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>7.303</td>\n",
       "      <td>7.227</td>\n",
       "      <td>7.364</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>7.242</td>\n",
       "      <td>7.165</td>\n",
       "      <td>7.300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>E-type estimate</td>\n",
       "      <td>7.242</td>\n",
       "      <td>7.164</td>\n",
       "      <td>7.300</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Predictor  Average  Minimum  Maximum\n",
       "0   Reference risk    7.209      NaN      NaN\n",
       "1        Raw rates    7.210    7.083    7.295\n",
       "2  Empirical Bayes    7.303    7.227    7.364\n",
       "3  Poisson kriging    7.242    7.165    7.300\n",
       "4  E-type estimate    7.242    7.164    7.300"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean absolute error (MAE) of prediction of the boundary statistic by five alternate estimators. The average, minimum and maximum MAE calculated over 25 simulated rate maps are reported. The last column gives the percentage of realizations where the MAE for the geostatistical methods is lower than for the local empirical Bayes smoother.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% best prediction</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4556</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4178</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4907</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1027</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0830</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1199</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0890</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0809</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0985</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">96</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>E</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0890</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0807</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0982</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">96</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1074</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0925</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1287</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">28</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best prediction</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.4556</td>\n",
       "      <td>0.4178</td>\n",
       "      <td>0.4907</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.1027</td>\n",
       "      <td>0.0830</td>\n",
       "      <td>0.1199</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.0890</td>\n",
       "      <td>0.0809</td>\n",
       "      <td>0.0985</td>\n",
       "      <td>96.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Statistic ΔEαβ</td>\n",
       "      <td>0.0890</td>\n",
       "      <td>0.0807</td>\n",
       "      <td>0.0982</td>\n",
       "      <td>96.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.1074</td>\n",
       "      <td>0.0925</td>\n",
       "      <td>0.1287</td>\n",
       "      <td>28.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Predictor  Average  Minimum  Maximum  % best prediction\n",
       "0        Raw rates   0.4556   0.4178   0.4907                NaN\n",
       "1  Empirical Bayes   0.1027   0.0830   0.1199                NaN\n",
       "2  Poisson kriging   0.0890   0.0809   0.0985               96.0\n",
       "3   Statistic ΔEαβ   0.0890   0.0807   0.0982               96.0\n",
       "4   Statistic Δsαβ   0.1074   0.0925   0.1287               28.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Average of the reference boundary statistic values and the estimates obtained using five alternate approaches. The average, minimum and maximum value of the statistic calculated over 25 simulated rate maps are reported.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Reference value</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1019</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5263</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4826</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5664</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1200</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0954</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1476</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0937</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0742</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1187</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>E</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0938</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0743</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1189</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1329</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0990</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1750</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Reference value</td>\n",
       "      <td>0.1019</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.5263</td>\n",
       "      <td>0.4826</td>\n",
       "      <td>0.5664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.1200</td>\n",
       "      <td>0.0954</td>\n",
       "      <td>0.1476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.0937</td>\n",
       "      <td>0.0742</td>\n",
       "      <td>0.1187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic ΔEαβ</td>\n",
       "      <td>0.0938</td>\n",
       "      <td>0.0743</td>\n",
       "      <td>0.1189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.1329</td>\n",
       "      <td>0.0990</td>\n",
       "      <td>0.1750</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Predictor  Average  Minimum  Maximum\n",
       "0  Reference value   0.1019      NaN      NaN\n",
       "1        Raw rates   0.5263   0.4826   0.5664\n",
       "2  Empirical Bayes   0.1200   0.0954   0.1476\n",
       "3  Poisson kriging   0.0937   0.0742   0.1187\n",
       "4   Statistic ΔEαβ   0.0938   0.0743   0.1189\n",
       "5   Statistic Δsαβ   0.1329   0.0990   0.1750"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Average proportion of false positives necessary to the detection of all 67 target edges using three types of neutral models and four estimators for the boundary statistic. The average, minimum and maximum proportions calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the particular method yields the smallest proportion of false positives.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T5-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% best results</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model I</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">43.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.63</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.39</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model II</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">34.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">26.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">29.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.86</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">68</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model III</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">19.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.84</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best results</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>33.0</td>\n",
       "      <td>24.80</td>\n",
       "      <td>43.7</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>17.2</td>\n",
       "      <td>9.63</td>\n",
       "      <td>27.4</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.7</td>\n",
       "      <td>7.39</td>\n",
       "      <td>18.2</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.9</td>\n",
       "      <td>10.30</td>\n",
       "      <td>20.4</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>34.0</td>\n",
       "      <td>26.50</td>\n",
       "      <td>45.1</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>18.5</td>\n",
       "      <td>10.70</td>\n",
       "      <td>29.1</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>12.3</td>\n",
       "      <td>6.86</td>\n",
       "      <td>18.4</td>\n",
       "      <td>68.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>15.3</td>\n",
       "      <td>10.90</td>\n",
       "      <td>21.4</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>13.2</td>\n",
       "      <td>7.00</td>\n",
       "      <td>19.5</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>13.2</td>\n",
       "      <td>6.84</td>\n",
       "      <td>20.3</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Predictor            Average  Minimum  Maximum  % best results\n",
       "0               NaN    Neutral Model I      NaN      NaN             NaN\n",
       "1         Raw rates               33.0    24.80     43.7             0.0\n",
       "2   Empirical Bayes               17.2     9.63     27.4             4.0\n",
       "3   Poisson kriging               12.7     7.39     18.2            16.0\n",
       "4    Statistic Δsαβ               15.9    10.30     20.4             0.0\n",
       "5               NaN   Neutral Model II      NaN      NaN             NaN\n",
       "6         Raw rates               34.0    26.50     45.1             0.0\n",
       "7   Empirical Bayes               18.5    10.70     29.1             0.0\n",
       "8   Poisson kriging               12.3     6.86     18.4            68.0\n",
       "9    Statistic Δsαβ               15.3    10.90     21.4             0.0\n",
       "10              NaN  Neutral Model III      NaN      NaN             NaN\n",
       "11  Poisson kriging               13.2     7.00     19.5             8.0\n",
       "12   Statistic Δsαβ               13.2     6.84     20.3             4.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Goodness statistic measuring the agreement between the actual proportion of false positives and the significance level used in tests based on three types of neutral models and four estimators for the boundary statistic. The average, minimum and maximum values calculated over 25 simulated rate maps are reported. The last column gives the percentage of simulations where the particular method yields the largest goodness statistic.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T6-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Average</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Minimum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Maximum</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% best results</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model I</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.898</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.873</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.919</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.676</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.605</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.713</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.554</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.450</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.628</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.561</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.472</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.629</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model II</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.822</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.718</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.945</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.877</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.814</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.927</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.772</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.691</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.817</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.605</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.532</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.656</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\">Neutral Model III</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.969</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.883</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.994</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">96</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.914</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.857</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.953</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best results</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.898</td>\n",
       "      <td>0.873</td>\n",
       "      <td>0.919</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.676</td>\n",
       "      <td>0.605</td>\n",
       "      <td>0.713</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.554</td>\n",
       "      <td>0.450</td>\n",
       "      <td>0.628</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.561</td>\n",
       "      <td>0.472</td>\n",
       "      <td>0.629</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.822</td>\n",
       "      <td>0.718</td>\n",
       "      <td>0.945</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.877</td>\n",
       "      <td>0.814</td>\n",
       "      <td>0.927</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.772</td>\n",
       "      <td>0.691</td>\n",
       "      <td>0.817</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.605</td>\n",
       "      <td>0.532</td>\n",
       "      <td>0.656</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.969</td>\n",
       "      <td>0.883</td>\n",
       "      <td>0.994</td>\n",
       "      <td>96.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.914</td>\n",
       "      <td>0.857</td>\n",
       "      <td>0.953</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Predictor            Average  Minimum  Maximum  % best results\n",
       "0               NaN    Neutral Model I      NaN      NaN             NaN\n",
       "1         Raw rates              0.898    0.873    0.919             0.0\n",
       "2   Empirical Bayes              0.676    0.605    0.713             0.0\n",
       "3   Poisson kriging              0.554    0.450    0.628             0.0\n",
       "4    Statistic Δsαβ              0.561    0.472    0.629             0.0\n",
       "5               NaN   Neutral Model II      NaN      NaN             NaN\n",
       "6         Raw rates              0.822    0.718    0.945             0.0\n",
       "7   Empirical Bayes              0.877    0.814    0.927             0.0\n",
       "8   Poisson kriging              0.772    0.691    0.817             0.0\n",
       "9    Statistic Δsαβ              0.605    0.532    0.656             0.0\n",
       "10              NaN  Neutral Model III      NaN      NaN             NaN\n",
       "11  Poisson kriging              0.969    0.883    0.994            96.0\n",
       "12   Statistic Δsαβ              0.914    0.857    0.953             4.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Average p-value for the tests conducted over the sets of target and non-target edges. The average, minimum and maximum values calculated over 25 simulated rate maps are reported. The last column reports the ratio of average p-values for non-target versus target edges.\n",
      "  footer_text: \n",
      "    unique: PMC2136438/PMC2136438.xml::T7-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"2\">Predictor</th>\n",
       "              <th align=\"center\" colspan=\"3\" rowspan=\"1\">Target edges</th>\n",
       "              <th align=\"center\" colspan=\"3\" rowspan=\"1\">Non-target edges</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"2\">Contrast</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Aver.</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Min.</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Max.</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Aver.</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Min.</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Max.</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">Neutral Model I</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.432</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.255</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.575</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.599</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.209</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.918</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.38</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.496</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.216</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.883</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.801</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.381</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.966</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.62</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.573</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.275</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.872</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.856</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.516</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.961</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.49</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.596</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.312</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.858</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.808</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.558</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.921</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.36</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">Neutral Model II</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.251</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.106</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.433</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.420</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.084</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.843</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.67</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.308</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.049</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.756</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.657</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.206</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.920</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.13</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.351</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.088</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.727</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.723</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.292</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.883</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.06</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.544</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.273</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.813</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.772</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.505</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.894</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.42</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">Neutral Model III</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.141</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.578</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.532</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.037</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.855</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.76</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Statistic &#916;<italic><sup>s</sup></italic><sub>&#945;&#946;</sub></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.159</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.428</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.399</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.079</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.494</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.51</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Target edges</th>\n",
       "      <th>Non-target edges</th>\n",
       "      <th>Contrast</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Aver.</th>\n",
       "      <th>Min.</th>\n",
       "      <th>Max.</th>\n",
       "      <th>Aver.</th>\n",
       "      <th>Min.</th>\n",
       "      <th>Max.</th>\n",
       "      <th>Unnamed: 6_level_1</th>\n",
       "      <th>Unnamed: 7_level_1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model I</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.432</td>\n",
       "      <td>0.255</td>\n",
       "      <td>0.575</td>\n",
       "      <td>0.599</td>\n",
       "      <td>0.209</td>\n",
       "      <td>0.918</td>\n",
       "      <td>1.38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.496</td>\n",
       "      <td>0.216</td>\n",
       "      <td>0.883</td>\n",
       "      <td>0.801</td>\n",
       "      <td>0.381</td>\n",
       "      <td>0.966</td>\n",
       "      <td>1.62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.573</td>\n",
       "      <td>0.275</td>\n",
       "      <td>0.872</td>\n",
       "      <td>0.856</td>\n",
       "      <td>0.516</td>\n",
       "      <td>0.961</td>\n",
       "      <td>1.49</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.596</td>\n",
       "      <td>0.312</td>\n",
       "      <td>0.858</td>\n",
       "      <td>0.808</td>\n",
       "      <td>0.558</td>\n",
       "      <td>0.921</td>\n",
       "      <td>1.36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model II</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.251</td>\n",
       "      <td>0.106</td>\n",
       "      <td>0.433</td>\n",
       "      <td>0.420</td>\n",
       "      <td>0.084</td>\n",
       "      <td>0.843</td>\n",
       "      <td>1.67</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.308</td>\n",
       "      <td>0.049</td>\n",
       "      <td>0.756</td>\n",
       "      <td>0.657</td>\n",
       "      <td>0.206</td>\n",
       "      <td>0.920</td>\n",
       "      <td>2.13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.351</td>\n",
       "      <td>0.088</td>\n",
       "      <td>0.727</td>\n",
       "      <td>0.723</td>\n",
       "      <td>0.292</td>\n",
       "      <td>0.883</td>\n",
       "      <td>2.06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.544</td>\n",
       "      <td>0.273</td>\n",
       "      <td>0.813</td>\n",
       "      <td>0.772</td>\n",
       "      <td>0.505</td>\n",
       "      <td>0.894</td>\n",
       "      <td>1.42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Neutral Model III</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.141</td>\n",
       "      <td>0.001</td>\n",
       "      <td>0.578</td>\n",
       "      <td>0.532</td>\n",
       "      <td>0.037</td>\n",
       "      <td>0.855</td>\n",
       "      <td>3.76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Statistic Δsαβ</td>\n",
       "      <td>0.159</td>\n",
       "      <td>0.001</td>\n",
       "      <td>0.428</td>\n",
       "      <td>0.399</td>\n",
       "      <td>0.079</td>\n",
       "      <td>0.494</td>\n",
       "      <td>2.51</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Predictor       Target edges Non-target edges Contrast Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0\n",
       "              Aver.               Min.             Max.    Aver.               Min.               Max. Unnamed: 6_level_1 Unnamed: 7_level_1\n",
       "0               NaN    Neutral Model I              NaN      NaN                NaN                NaN                NaN                NaN\n",
       "1         Raw rates              0.432            0.255    0.575              0.599              0.209              0.918               1.38\n",
       "2   Empirical Bayes              0.496            0.216    0.883              0.801              0.381              0.966               1.62\n",
       "3   Poisson kriging              0.573            0.275    0.872              0.856              0.516              0.961               1.49\n",
       "4    Statistic Δsαβ              0.596            0.312    0.858              0.808              0.558              0.921               1.36\n",
       "5               NaN   Neutral Model II              NaN      NaN                NaN                NaN                NaN                NaN\n",
       "6         Raw rates              0.251            0.106    0.433              0.420              0.084              0.843               1.67\n",
       "7   Empirical Bayes              0.308            0.049    0.756              0.657              0.206              0.920               2.13\n",
       "8   Poisson kriging              0.351            0.088    0.727              0.723              0.292              0.883               2.06\n",
       "9    Statistic Δsαβ              0.544            0.273    0.813              0.772              0.505              0.894               1.42\n",
       "10              NaN  Neutral Model III              NaN      NaN                NaN                NaN                NaN                NaN\n",
       "11  Poisson kriging              0.141            0.001    0.578              0.532              0.037              0.855               3.76\n",
       "12   Statistic Δsαβ              0.159            0.001    0.428              0.399              0.079              0.494               2.51"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0021XXXXX/PMC2185066.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE I\n",
      "  caption_text: Estimation Performance of the Rate Parameters When Different Noise Levels cin Are Added to the Input Function. The Results Are Calculated From 100 Simulation Runs\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">c<sub>in</sub></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>bias</italic>(<italic>k</italic><sub>2<italic>f</italic></sub>)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>bias</italic>(<italic>k</italic><sub>2<italic>s</italic></sub>)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>CV</italic>(<italic>K</italic><sub>2<italic>f</italic></sub>)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>CV</italic>(<italic>K</italic><sub>2<italic>s</italic></sub>)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0015</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0033</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0176</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0191</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0338</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0299</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0721</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.1070</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.118</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.039</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.083</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.110</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cin</th>\n",
       "      <th>bias(k2f)</th>\n",
       "      <th>bias(k2s)</th>\n",
       "      <th>CV(K2f)</th>\n",
       "      <th>CV(K2s)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.01</td>\n",
       "      <td>0.0015</td>\n",
       "      <td>0.0033</td>\n",
       "      <td>0.0176</td>\n",
       "      <td>0.0191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.05</td>\n",
       "      <td>0.0338</td>\n",
       "      <td>0.0299</td>\n",
       "      <td>0.0721</td>\n",
       "      <td>0.1070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.10</td>\n",
       "      <td>0.1180</td>\n",
       "      <td>0.0390</td>\n",
       "      <td>0.0830</td>\n",
       "      <td>0.1100</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    cin  bias(k2f)  bias(k2s)  CV(K2f)  CV(K2s)\n",
       "0  0.01     0.0015     0.0033   0.0176   0.0191\n",
       "1  0.05     0.0338     0.0299   0.0721   0.1070\n",
       "2  0.10     0.1180     0.0390   0.0830   0.1100"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE II\n",
      "  caption_text: Estimation Performance of the Rate Parameters When Different Noise Level Is Added to the Voxel TACs. The Results Are Calculated From 100 Simulation Runs\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"2\" valign=\"top\">Perf. Measures vs. noise level</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Model-1: <italic>&#963;</italic></th>\n",
       "              <th align=\"center\" colspan=\"3\" rowspan=\"1\" valign=\"top\">Model-2: c</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">50</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">150</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">30</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>bias</italic>(<italic>k</italic><sub>2<italic>f</italic></sub>)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0219</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0617</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0017</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0038</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0311</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>CV</italic>(<italic>k</italic><sub>2<italic>f</italic></sub>)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0460</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0797</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0182</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0367</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0706</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>bias</italic>(<italic>k</italic><sub>2<italic>s</italic></sub>)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0169</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0345</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0128</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0356</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0563</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>CV</italic>(<italic>k</italic><sub>2<italic>s</italic></sub>)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0860</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.1781</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0258</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0816</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.1312</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Perf. Measures vs. noise level</th>\n",
       "      <th>Model-1: σ</th>\n",
       "      <th>Model-2: c</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>50</th>\n",
       "      <th>150</th>\n",
       "      <th>1</th>\n",
       "      <th>10</th>\n",
       "      <th>30</th>\n",
       "      <th>Unnamed: 5_level_1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>bias(k2f)</td>\n",
       "      <td>0.0219</td>\n",
       "      <td>0.0617</td>\n",
       "      <td>0.0017</td>\n",
       "      <td>0.0038</td>\n",
       "      <td>0.0311</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>CV(k2f)</td>\n",
       "      <td>0.0460</td>\n",
       "      <td>0.0797</td>\n",
       "      <td>0.0182</td>\n",
       "      <td>0.0367</td>\n",
       "      <td>0.0706</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>bias(k2s)</td>\n",
       "      <td>0.0169</td>\n",
       "      <td>0.0345</td>\n",
       "      <td>0.0128</td>\n",
       "      <td>0.0356</td>\n",
       "      <td>0.0563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>CV(k2s)</td>\n",
       "      <td>0.0860</td>\n",
       "      <td>0.1781</td>\n",
       "      <td>0.0258</td>\n",
       "      <td>0.0816</td>\n",
       "      <td>0.1312</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Perf. Measures vs. noise level Model-1: σ Model-2: c Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0\n",
       "                              50        150          1                 10                 30 Unnamed: 5_level_1\n",
       "0                      bias(k2f)     0.0219     0.0617             0.0017             0.0038             0.0311\n",
       "1                        CV(k2f)     0.0460     0.0797             0.0182             0.0367             0.0706\n",
       "2                      bias(k2s)     0.0169     0.0345             0.0128             0.0356             0.0563\n",
       "3                        CV(k2s)     0.0860     0.1781             0.0258             0.0816             0.1312"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE III\n",
      "  caption_text: Performance of Estimating the Factor Images, in Terms of CC and PM. in Each Parenthesis, the First Number Is the Mean, and the Second Number Is the Corresponding Standard Deviation of the Performance Measure CC or PM. Here, the Noise Level σ2 = 30\n",
      "  footer_text: \n",
      "    unique: PMC2185066/PMC2185066.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">factor</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">fast</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">slow</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">input</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">CC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.962,0.041)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.996,0.005)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.980,0.016)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <italic>PM</italic>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.388,0.113)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.110,0.079)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(0.207,0.064)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>factor</th>\n",
       "      <th>fast</th>\n",
       "      <th>slow</th>\n",
       "      <th>input</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>CC</td>\n",
       "      <td>(0.962,0.041)</td>\n",
       "      <td>(0.996,0.005)</td>\n",
       "      <td>(0.980,0.016)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PM</td>\n",
       "      <td>(0.388,0.113)</td>\n",
       "      <td>(0.110,0.079)</td>\n",
       "      <td>(0.207,0.064)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  factor           fast           slow          input\n",
       "0     CC  (0.962,0.041)  (0.996,0.005)  (0.980,0.016)\n",
       "1     PM  (0.388,0.113)  (0.110,0.079)  (0.207,0.064)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0022XXXXX/PMC2268633.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Distribution of deaths by sex and place of birth, NHW residents, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">Male</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">Female</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">Total</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">769,405</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">710,942</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,480,347</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">19,753</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">23,644</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">43,397</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,543</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,685</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,228</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,840</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">8,135</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13,975</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,109</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,879</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10,988</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,305</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,283</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10,588</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4,072</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,595</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">9,667</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7,755</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,834</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13,589</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,801</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,796</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4,597</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4,954</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4,038</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">8,992</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Total</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">827,537</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">773,831</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,601,368</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Total</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>769405</td>\n",
       "      <td>710942</td>\n",
       "      <td>1480347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>19753</td>\n",
       "      <td>23644</td>\n",
       "      <td>43397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>2543</td>\n",
       "      <td>2685</td>\n",
       "      <td>5228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>5840</td>\n",
       "      <td>8135</td>\n",
       "      <td>13975</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>5109</td>\n",
       "      <td>5879</td>\n",
       "      <td>10988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>5305</td>\n",
       "      <td>5283</td>\n",
       "      <td>10588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>4072</td>\n",
       "      <td>5595</td>\n",
       "      <td>9667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>7755</td>\n",
       "      <td>5834</td>\n",
       "      <td>13589</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>2801</td>\n",
       "      <td>1796</td>\n",
       "      <td>4597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>4954</td>\n",
       "      <td>4038</td>\n",
       "      <td>8992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Total</td>\n",
       "      <td>827537</td>\n",
       "      <td>773831</td>\n",
       "      <td>1601368</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Place of birth    Male  Female    Total\n",
       "0            NHW US-born  769405  710942  1480347\n",
       "1            Anglo-Saxon   19753   23644    43397\n",
       "2        Northern Europe    2543    2685     5228\n",
       "3         Western Europe    5840    8135    13975\n",
       "4        Southern Europe    5109    5879    10988\n",
       "5         Eastern Europe    5305    5283    10588\n",
       "6            Former USSR    4072    5595     9667\n",
       "7            Middle East    7755    5834    13589\n",
       "8       Arab Middle East    2801    1796     4597\n",
       "9   Non-Arab Middle East    4954    4038     8992\n",
       "10                 Total  827537  773831  1601368"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Coronary heart disease</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">241,743</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">234,780</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">6,571</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6628.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.97&#8211;1.02)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7,960</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8039.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.97&#8211;1.01)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">775</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">841.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.92 (0.86&#8211;0.99)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">886</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">920.08</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.96 (0.90&#8211;1.03)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,870</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1790.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.04 (1.00&#8211;1.09)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,524</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2595.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.97 (0.93&#8211;1.01)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,673</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1691.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.94&#8211;1.04)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,257</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2167.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.04(1.00&#8211;1.09)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,957</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1762.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.11 (1.06&#8211;1.16)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,964</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1835.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.07 (1.02&#8211;1.12)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,556</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1353.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.15 (1.09&#8211;1.21)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,314</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2008.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.15 (1.11&#8211;1.20)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,883</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2414.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.19 (1.15&#8211;1.24)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,275</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1827.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.25 (1.19&#8211;1.30)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,051</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">823.41</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.28 (1.20&#8211;1.36)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">642</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">487.31</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.32 (1.22&#8211;1.42)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,832</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1590.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.15 (1.10&#8211;1.21)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,633</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1339.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.22 (1.16&#8211;1.28)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Coronary heart disease</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>241743</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>234780</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>6571</td>\n",
       "      <td>6628.10</td>\n",
       "      <td>0.99 (0.97–1.02)</td>\n",
       "      <td>7960</td>\n",
       "      <td>8039.60</td>\n",
       "      <td>0.99 (0.97–1.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>775</td>\n",
       "      <td>841.10</td>\n",
       "      <td>0.92 (0.86–0.99)</td>\n",
       "      <td>886</td>\n",
       "      <td>920.08</td>\n",
       "      <td>0.96 (0.90–1.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>1870</td>\n",
       "      <td>1790.70</td>\n",
       "      <td>1.04 (1.00–1.09)</td>\n",
       "      <td>2524</td>\n",
       "      <td>2595.30</td>\n",
       "      <td>0.97 (0.93–1.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>1673</td>\n",
       "      <td>1691.20</td>\n",
       "      <td>0.99 (0.94–1.04)</td>\n",
       "      <td>2257</td>\n",
       "      <td>2167.70</td>\n",
       "      <td>1.04(1.00–1.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>1957</td>\n",
       "      <td>1762.50</td>\n",
       "      <td>1.11 (1.06–1.16)</td>\n",
       "      <td>1964</td>\n",
       "      <td>1835.90</td>\n",
       "      <td>1.07 (1.02–1.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>1556</td>\n",
       "      <td>1353.30</td>\n",
       "      <td>1.15 (1.09–1.21)</td>\n",
       "      <td>2314</td>\n",
       "      <td>2008.80</td>\n",
       "      <td>1.15 (1.11–1.20)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>2883</td>\n",
       "      <td>2414.20</td>\n",
       "      <td>1.19 (1.15–1.24)</td>\n",
       "      <td>2275</td>\n",
       "      <td>1827.20</td>\n",
       "      <td>1.25 (1.19–1.30)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>1051</td>\n",
       "      <td>823.41</td>\n",
       "      <td>1.28 (1.20–1.36)</td>\n",
       "      <td>642</td>\n",
       "      <td>487.31</td>\n",
       "      <td>1.32 (1.22–1.42)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>1832</td>\n",
       "      <td>1590.80</td>\n",
       "      <td>1.15 (1.10–1.21)</td>\n",
       "      <td>1633</td>\n",
       "      <td>1339.90</td>\n",
       "      <td>1.22 (1.16–1.28)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Coronary heart disease    Male   Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "          Place of birth     OBS      EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0            NHW US-born  241743      NaN                NaN             234780                NaN                NaN\n",
       "1            Anglo-Saxon    6571  6628.10   0.99 (0.97–1.02)               7960            8039.60   0.99 (0.97–1.01)\n",
       "2        Northern Europe     775   841.10   0.92 (0.86–0.99)                886             920.08   0.96 (0.90–1.03)\n",
       "3         Western Europe    1870  1790.70   1.04 (1.00–1.09)               2524            2595.30   0.97 (0.93–1.01)\n",
       "4        Southern Europe    1673  1691.20   0.99 (0.94–1.04)               2257            2167.70    1.04(1.00–1.09)\n",
       "5         Eastern Europe    1957  1762.50   1.11 (1.06–1.16)               1964            1835.90   1.07 (1.02–1.12)\n",
       "6            Former USSR    1556  1353.30   1.15 (1.09–1.21)               2314            2008.80   1.15 (1.11–1.20)\n",
       "7            Middle East    2883  2414.20   1.19 (1.15–1.24)               2275            1827.20   1.25 (1.19–1.30)\n",
       "8       Arab Middle East    1051   823.41   1.28 (1.20–1.36)                642             487.31   1.32 (1.22–1.42)\n",
       "9   Non-Arab Middle East    1832  1590.80   1.15 (1.10–1.21)               1633            1339.90   1.22 (1.16–1.28)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">All cancers</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">189,613</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">168,336</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,207</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5,032.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.03 (1.01&#8211;1.06)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,960</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5,558.4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.07 (1.05&#8211;1.10)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">791</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">693.66</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.14 (1.06&#8211;1.22)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">753</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">621.27</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.21 (1.13&#8211;1.30)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,600</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,516.4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.06 (1.00&#8211;1.11)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,442</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2,128</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.15 (1.10&#8211;1.19)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,529</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,336.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.14 (1.09&#8211;1.20)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,183</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,155.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.02 (0.97&#8211;1.08)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,470</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,423.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.03 (0.98&#8211;1.09)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,295</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,190.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.09 (1.03&#8211;1.15)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,117</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,002</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.11 (1.05&#8211;1.18)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,338</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,154.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.16 (1.10&#8211;1.22)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,014</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,947.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.03 (0.99&#8211;1.08)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,494</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,500.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.95&#8211;1.05)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">720</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">746.62</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.96 (0.90&#8211;1.04)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">534</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">560.09</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.95 (0.87&#8211;1.04)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,294</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,201.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.08 (1.02&#8211;1.14)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">960</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">940.69</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.02 (0.96&#8211;1.09)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>All cancers</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>189613</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>168336</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>5207</td>\n",
       "      <td>5032.60</td>\n",
       "      <td>1.03 (1.01–1.06)</td>\n",
       "      <td>5960</td>\n",
       "      <td>5558.40</td>\n",
       "      <td>1.07 (1.05–1.10)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>791</td>\n",
       "      <td>693.66</td>\n",
       "      <td>1.14 (1.06–1.22)</td>\n",
       "      <td>753</td>\n",
       "      <td>621.27</td>\n",
       "      <td>1.21 (1.13–1.30)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>1600</td>\n",
       "      <td>1516.40</td>\n",
       "      <td>1.06 (1.00–1.11)</td>\n",
       "      <td>2442</td>\n",
       "      <td>2128.00</td>\n",
       "      <td>1.15 (1.10–1.19)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>1529</td>\n",
       "      <td>1336.90</td>\n",
       "      <td>1.14 (1.09–1.20)</td>\n",
       "      <td>1183</td>\n",
       "      <td>1155.70</td>\n",
       "      <td>1.02 (0.97–1.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>1470</td>\n",
       "      <td>1423.90</td>\n",
       "      <td>1.03 (0.98–1.09)</td>\n",
       "      <td>1295</td>\n",
       "      <td>1190.20</td>\n",
       "      <td>1.09 (1.03–1.15)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>1117</td>\n",
       "      <td>1002.00</td>\n",
       "      <td>1.11 (1.05–1.18)</td>\n",
       "      <td>1338</td>\n",
       "      <td>1154.60</td>\n",
       "      <td>1.16 (1.10–1.22)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>2014</td>\n",
       "      <td>1947.90</td>\n",
       "      <td>1.03 (0.99–1.08)</td>\n",
       "      <td>1494</td>\n",
       "      <td>1500.80</td>\n",
       "      <td>1.00 (0.95–1.05)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>720</td>\n",
       "      <td>746.62</td>\n",
       "      <td>0.96 (0.90–1.04)</td>\n",
       "      <td>534</td>\n",
       "      <td>560.09</td>\n",
       "      <td>0.95 (0.87–1.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>1294</td>\n",
       "      <td>1201.30</td>\n",
       "      <td>1.08 (1.02–1.14)</td>\n",
       "      <td>960</td>\n",
       "      <td>940.69</td>\n",
       "      <td>1.02 (0.96–1.09)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "            All cancers    Male   Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth     OBS      EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  189613      NaN                NaN             168336                NaN                NaN\n",
       "1           Anglo-Saxon    5207  5032.60   1.03 (1.01–1.06)               5960            5558.40   1.07 (1.05–1.10)\n",
       "2       Northern Europe     791   693.66   1.14 (1.06–1.22)                753             621.27   1.21 (1.13–1.30)\n",
       "3        Western Europe    1600  1516.40   1.06 (1.00–1.11)               2442            2128.00   1.15 (1.10–1.19)\n",
       "4       Southern Europe    1529  1336.90   1.14 (1.09–1.20)               1183            1155.70   1.02 (0.97–1.08)\n",
       "5        Eastern Europe    1470  1423.90   1.03 (0.98–1.09)               1295            1190.20   1.09 (1.03–1.15)\n",
       "6           Former USSR    1117  1002.00   1.11 (1.05–1.18)               1338            1154.60   1.16 (1.10–1.22)\n",
       "7           Middle East    2014  1947.90   1.03 (0.99–1.08)               1494            1500.80   1.00 (0.95–1.05)\n",
       "8      Arab Middle East     720   746.62   0.96 (0.90–1.04)                534             560.09   0.95 (0.87–1.04)\n",
       "9  Non-Arab Middle East    1294  1201.30   1.08 (1.02–1.14)                960             940.69   1.02 (0.96–1.09)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: * Chronic obstructive pulmonary disease\n",
      "    unique: PMC2268633/PMC2268633.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">COPD<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">43,590</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">43,964</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,097</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,239</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.89 (0.83&#8211;0.94)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,441</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,534.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.94 (0.89&#8211;0.99)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">106</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">158.82</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.67 (0.55&#8211;0.81)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">96</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">163.44</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.59 (0.48&#8211;0.72)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">164</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">322.76</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.51 (0.43&#8211;0.59)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">310</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">515.03</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.60 (0.54&#8211;0.67)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">186</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">311.31</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.60 (0.51&#8211;0.69)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">146</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">336.12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.43 (0.37&#8211;0.51)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">146</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">330.93</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.44 (0.37&#8211;0.52)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">142</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">330.92</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.43 (0.36&#8211;0.51)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">101</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">246.25</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.41 (0.33&#8211;0.50)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">132</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">335.55</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.39 (0.33&#8211;0.47)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">269</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">424.41</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.63 (0.56&#8211;0.71)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">135</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">367.42</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.37 (0.31&#8211;0.43)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">89</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">143.07</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.62 (0.50&#8211;0.77)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">28</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">119.35</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.23 (0.16&#8211;0.34)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">180</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">281.34</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.64 (0.55&#8211;0.74)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">107</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">248.06</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.43 (0.35&#8211;0.52)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>COPD*</th>\n",
       "      <th>Male\\n</th>\n",
       "      <th>Female\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>43590</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>43964</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>1097</td>\n",
       "      <td>1239.00</td>\n",
       "      <td>0.89 (0.83–0.94)</td>\n",
       "      <td>1441</td>\n",
       "      <td>1534.60</td>\n",
       "      <td>0.94 (0.89–0.99)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>106</td>\n",
       "      <td>158.82</td>\n",
       "      <td>0.67 (0.55–0.81)</td>\n",
       "      <td>96</td>\n",
       "      <td>163.44</td>\n",
       "      <td>0.59 (0.48–0.72)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>164</td>\n",
       "      <td>322.76</td>\n",
       "      <td>0.51 (0.43–0.59)</td>\n",
       "      <td>310</td>\n",
       "      <td>515.03</td>\n",
       "      <td>0.60 (0.54–0.67)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>186</td>\n",
       "      <td>311.31</td>\n",
       "      <td>0.60 (0.51–0.69)</td>\n",
       "      <td>146</td>\n",
       "      <td>336.12</td>\n",
       "      <td>0.43 (0.37–0.51)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>146</td>\n",
       "      <td>330.93</td>\n",
       "      <td>0.44 (0.37–0.52)</td>\n",
       "      <td>142</td>\n",
       "      <td>330.92</td>\n",
       "      <td>0.43 (0.36–0.51)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>101</td>\n",
       "      <td>246.25</td>\n",
       "      <td>0.41 (0.33–0.50)</td>\n",
       "      <td>132</td>\n",
       "      <td>335.55</td>\n",
       "      <td>0.39 (0.33–0.47)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>269</td>\n",
       "      <td>424.41</td>\n",
       "      <td>0.63 (0.56–0.71)</td>\n",
       "      <td>135</td>\n",
       "      <td>367.42</td>\n",
       "      <td>0.37 (0.31–0.43)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>89</td>\n",
       "      <td>143.07</td>\n",
       "      <td>0.62 (0.50–0.77)</td>\n",
       "      <td>28</td>\n",
       "      <td>119.35</td>\n",
       "      <td>0.23 (0.16–0.34)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>180</td>\n",
       "      <td>281.34</td>\n",
       "      <td>0.64 (0.55–0.74)</td>\n",
       "      <td>107</td>\n",
       "      <td>248.06</td>\n",
       "      <td>0.43 (0.35–0.52)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                  COPD* Male\\n Female\\n Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth    OBS      EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  43590      NaN                NaN              43964                NaN                NaN\n",
       "1           Anglo-Saxon   1097  1239.00   0.89 (0.83–0.94)               1441            1534.60   0.94 (0.89–0.99)\n",
       "2       Northern Europe    106   158.82   0.67 (0.55–0.81)                 96             163.44   0.59 (0.48–0.72)\n",
       "3        Western Europe    164   322.76   0.51 (0.43–0.59)                310             515.03   0.60 (0.54–0.67)\n",
       "4       Southern Europe    186   311.31   0.60 (0.51–0.69)                146             336.12   0.43 (0.37–0.51)\n",
       "5        Eastern Europe    146   330.93   0.44 (0.37–0.52)                142             330.92   0.43 (0.36–0.51)\n",
       "6           Former USSR    101   246.25   0.41 (0.33–0.50)                132             335.55   0.39 (0.33–0.47)\n",
       "7           Middle East    269   424.41   0.63 (0.56–0.71)                135             367.42   0.37 (0.31–0.43)\n",
       "8      Arab Middle East     89   143.07   0.62 (0.50–0.77)                 28             119.35   0.23 (0.16–0.34)\n",
       "9  Non-Arab Middle East    180   281.34   0.64 (0.55–0.74)                107             248.06   0.43 (0.35–0.52)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T5-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Cerebrovascular accidents</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">41,050</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">62,443</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,192</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,196.4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.94&#8211;1.05)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2,074</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2,217.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.94 (0.90&#8211;0.98)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">142</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">141.97</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.84&#8211;1.18)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">207</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">253.49</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.82 (0.71&#8211;0.94)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">277</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">291.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.95 (0.84&#8211;1.07)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">634</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">700.55</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.91 (0.84&#8211;0.98)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">343</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">302.58</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.13 (1.02&#8211;1.26)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">501</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">506.65</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.90&#8211;1.08)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">352</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">293.38</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.20 (1.08&#8211;1.33)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">659</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">604.12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.09 (1.01&#8211;1.18)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">250</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">246.54</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.01 (0.89&#8211;1.15)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">510</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">560.22</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.91 (0.83&#8211;0.99)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">397</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">406.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.98 (0.88&#8211;1.08)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">444</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">499.36</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.89 (0.81&#8211;0.98)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">125</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">123.06</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.02 (0.85&#8211;1.21)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">127</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">127.99</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.83&#8211;1.18)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">272</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">283.13</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.96 (0.85&#8211;1.08)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">317</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">371.36</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.85 (0.76&#8211;0.95)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Cerebrovascular accidents</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>41050</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>62443</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>1192</td>\n",
       "      <td>1196.40</td>\n",
       "      <td>1.00 (0.94–1.05)</td>\n",
       "      <td>2074</td>\n",
       "      <td>2217.60</td>\n",
       "      <td>0.94 (0.90–0.98)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>142</td>\n",
       "      <td>141.97</td>\n",
       "      <td>1.00 (0.84–1.18)</td>\n",
       "      <td>207</td>\n",
       "      <td>253.49</td>\n",
       "      <td>0.82 (0.71–0.94)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>277</td>\n",
       "      <td>291.30</td>\n",
       "      <td>0.95 (0.84–1.07)</td>\n",
       "      <td>634</td>\n",
       "      <td>700.55</td>\n",
       "      <td>0.91 (0.84–0.98)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>343</td>\n",
       "      <td>302.58</td>\n",
       "      <td>1.13 (1.02–1.26)</td>\n",
       "      <td>501</td>\n",
       "      <td>506.65</td>\n",
       "      <td>0.99 (0.90–1.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>352</td>\n",
       "      <td>293.38</td>\n",
       "      <td>1.20 (1.08–1.33)</td>\n",
       "      <td>659</td>\n",
       "      <td>604.12</td>\n",
       "      <td>1.09 (1.01–1.18)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>250</td>\n",
       "      <td>246.54</td>\n",
       "      <td>1.01 (0.89–1.15)</td>\n",
       "      <td>510</td>\n",
       "      <td>560.22</td>\n",
       "      <td>0.91 (0.83–0.99)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>397</td>\n",
       "      <td>406.20</td>\n",
       "      <td>0.98 (0.88–1.08)</td>\n",
       "      <td>444</td>\n",
       "      <td>499.36</td>\n",
       "      <td>0.89 (0.81–0.98)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>125</td>\n",
       "      <td>123.06</td>\n",
       "      <td>1.02 (0.85–1.21)</td>\n",
       "      <td>127</td>\n",
       "      <td>127.99</td>\n",
       "      <td>0.99 (0.83–1.18)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>272</td>\n",
       "      <td>283.13</td>\n",
       "      <td>0.96 (0.85–1.08)</td>\n",
       "      <td>317</td>\n",
       "      <td>371.36</td>\n",
       "      <td>0.85 (0.76–0.95)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Cerebrovascular accidents   Male   Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "             Place of birth    OBS      EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0               NHW US-born  41050      NaN                NaN              62443                NaN                NaN\n",
       "1               Anglo-Saxon   1192  1196.40   1.00 (0.94–1.05)               2074            2217.60   0.94 (0.90–0.98)\n",
       "2           Northern Europe    142   141.97   1.00 (0.84–1.18)                207             253.49   0.82 (0.71–0.94)\n",
       "3            Western Europe    277   291.30   0.95 (0.84–1.07)                634             700.55   0.91 (0.84–0.98)\n",
       "4            Eastern Europe    343   302.58   1.13 (1.02–1.26)                501             506.65   0.99 (0.90–1.08)\n",
       "5           Southern Europe    352   293.38   1.20 (1.08–1.33)                659             604.12   1.09 (1.01–1.18)\n",
       "6               Former USSR    250   246.54   1.01 (0.89–1.15)                510             560.22   0.91 (0.83–0.99)\n",
       "7               Middle East    397   406.20   0.98 (0.88–1.08)                444             499.36   0.89 (0.81–0.98)\n",
       "8          Arab Middle East    125   123.06   1.02 (0.85–1.21)                127             127.99   0.99 (0.83–1.18)\n",
       "9      Non-Arab Middle East    272   283.13   0.96 (0.85–1.08)                317             371.36   0.85 (0.76–0.95)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T6-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Pneumonia/influenza</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">33,417</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">40,822</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">941</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">936.97</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.94&#8211;1.07)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,394</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1,387.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.95&#8211;1.06)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">105</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">105.98</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (0.81&#8211;1.20)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">172</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">161.58</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.06 (0.91&#8211;1.24)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">170</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">215.76</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.79 (0.67&#8211;0.92)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">403</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">437.68</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.92 (0.83&#8211;1.02)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">231</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">228.35</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.01 (0.89&#8211;1.15)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">323</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">322.06</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00 (0.90&#8211;1.12)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">206</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">225.82</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.91 (0.79&#8211;1.05)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">369</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">396.26</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.93 (0.84&#8211;1.03)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">185</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">198.03</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.93 (0.80&#8211;1.08)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">326</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">361.35</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.90 (0.81&#8211;1.01)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">294</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">312.69</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.94 (0.84&#8211;1.05)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">291</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">309.34</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.94 (0.84&#8211;1.06)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">84</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">86.56</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.97 (0.77&#8211;1.20)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">75.26</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.81 (0.62&#8211;1.04)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">210</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">226.13</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.93 (0.81&#8211;1.06)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">230</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">234.09</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.98 (0.86&#8211;1.12)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Pneumonia/influenza</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>33417</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>40822</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>941</td>\n",
       "      <td>936.97</td>\n",
       "      <td>1.00 (0.94–1.07)</td>\n",
       "      <td>1394</td>\n",
       "      <td>1387.60</td>\n",
       "      <td>1.00 (0.95–1.06)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>105</td>\n",
       "      <td>105.98</td>\n",
       "      <td>0.99 (0.81–1.20)</td>\n",
       "      <td>172</td>\n",
       "      <td>161.58</td>\n",
       "      <td>1.06 (0.91–1.24)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>170</td>\n",
       "      <td>215.76</td>\n",
       "      <td>0.79 (0.67–0.92)</td>\n",
       "      <td>403</td>\n",
       "      <td>437.68</td>\n",
       "      <td>0.92 (0.83–1.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>231</td>\n",
       "      <td>228.35</td>\n",
       "      <td>1.01 (0.89–1.15)</td>\n",
       "      <td>323</td>\n",
       "      <td>322.06</td>\n",
       "      <td>1.00 (0.90–1.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>206</td>\n",
       "      <td>225.82</td>\n",
       "      <td>0.91 (0.79–1.05)</td>\n",
       "      <td>369</td>\n",
       "      <td>396.26</td>\n",
       "      <td>0.93 (0.84–1.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>185</td>\n",
       "      <td>198.03</td>\n",
       "      <td>0.93 (0.80–1.08)</td>\n",
       "      <td>326</td>\n",
       "      <td>361.35</td>\n",
       "      <td>0.90 (0.81–1.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>294</td>\n",
       "      <td>312.69</td>\n",
       "      <td>0.94 (0.84–1.05)</td>\n",
       "      <td>291</td>\n",
       "      <td>309.34</td>\n",
       "      <td>0.94 (0.84–1.06)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>84</td>\n",
       "      <td>86.56</td>\n",
       "      <td>0.97 (0.77–1.20)</td>\n",
       "      <td>61</td>\n",
       "      <td>75.26</td>\n",
       "      <td>0.81 (0.62–1.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>210</td>\n",
       "      <td>226.13</td>\n",
       "      <td>0.93 (0.81–1.06)</td>\n",
       "      <td>230</td>\n",
       "      <td>234.09</td>\n",
       "      <td>0.98 (0.86–1.12)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Pneumonia/influenza   Male  Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth    OBS     EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  33417     NaN                NaN              40822                NaN                NaN\n",
       "1           Anglo-Saxon    941  936.97   1.00 (0.94–1.07)               1394            1387.60   1.00 (0.95–1.06)\n",
       "2       Northern Europe    105  105.98   0.99 (0.81–1.20)                172             161.58   1.06 (0.91–1.24)\n",
       "3        Western Europe    170  215.76   0.79 (0.67–0.92)                403             437.68   0.92 (0.83–1.02)\n",
       "4        Eastern Europe    231  228.35   1.01 (0.89–1.15)                323             322.06   1.00 (0.90–1.12)\n",
       "5       Southern Europe    206  225.82   0.91 (0.79–1.05)                369             396.26   0.93 (0.84–1.03)\n",
       "6           Former USSR    185  198.03   0.93 (0.80–1.08)                326             361.35   0.90 (0.81–1.01)\n",
       "7           Middle East    294  312.69   0.94 (0.84–1.05)                291             309.34   0.94 (0.84–1.06)\n",
       "8      Arab Middle East     84   86.56   0.97 (0.77–1.20)                 61              75.26   0.81 (0.62–1.04)\n",
       "9  Non-Arab Middle East    210  226.13   0.93 (0.81–1.06)                230             234.09   0.98 (0.86–1.12)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T7-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">HIV/AIDS</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">26,940</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">983</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">465</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">410.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.13 (1.03&#8211;1.24)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.834</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.84 (0.47&#8211;1.39)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">50.92</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.18 (0.90&#8211;1.52)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.315</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.02 (1.21&#8211;6.20)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">285</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">243.59</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.17 (1.04&#8211;1.31)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.881</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.19 (0.64&#8211;2.04)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">117.41</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.77 (0.62&#8211;0.94)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.302</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.30 (0.00&#8211;1.53)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">49</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">105.99</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.46 (0.34&#8211;0.61)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.84</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.52 (0.06&#8211;1.82)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">12</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">88.64</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.14 (0.07&#8211;0.24)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.143</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">122</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">298.09</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.41 (0.34&#8211;0.49)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.698</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.56 (0.20&#8211;1.21)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">70</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">137.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.51 (0.40&#8211;0.64)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.837</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.83 (0.22&#8211;2.09)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">160.39</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.32 (0.24&#8211;0.43)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.86</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.34 (0.04&#8211;1.19)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>HIV/AIDS</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>26940</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>983</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>465</td>\n",
       "      <td>410.01</td>\n",
       "      <td>1.13 (1.03–1.24)</td>\n",
       "      <td>15</td>\n",
       "      <td>17.834</td>\n",
       "      <td>0.84 (0.47–1.39)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>60</td>\n",
       "      <td>50.92</td>\n",
       "      <td>1.18 (0.90–1.52)</td>\n",
       "      <td>7</td>\n",
       "      <td>2.315</td>\n",
       "      <td>3.02 (1.21–6.20)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>285</td>\n",
       "      <td>243.59</td>\n",
       "      <td>1.17 (1.04–1.31)</td>\n",
       "      <td>13</td>\n",
       "      <td>10.881</td>\n",
       "      <td>1.19 (0.64–2.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>90</td>\n",
       "      <td>117.41</td>\n",
       "      <td>0.77 (0.62–0.94)</td>\n",
       "      <td>1</td>\n",
       "      <td>3.302</td>\n",
       "      <td>0.30 (0.00–1.53)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>49</td>\n",
       "      <td>105.99</td>\n",
       "      <td>0.46 (0.34–0.61)</td>\n",
       "      <td>2</td>\n",
       "      <td>3.840</td>\n",
       "      <td>0.52 (0.06–1.82)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>12</td>\n",
       "      <td>88.64</td>\n",
       "      <td>0.14 (0.07–0.24)</td>\n",
       "      <td>0</td>\n",
       "      <td>4.143</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>122</td>\n",
       "      <td>298.09</td>\n",
       "      <td>0.41 (0.34–0.49)</td>\n",
       "      <td>6</td>\n",
       "      <td>10.698</td>\n",
       "      <td>0.56 (0.20–1.21)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>70</td>\n",
       "      <td>137.70</td>\n",
       "      <td>0.51 (0.40–0.64)</td>\n",
       "      <td>4</td>\n",
       "      <td>4.837</td>\n",
       "      <td>0.83 (0.22–2.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>52</td>\n",
       "      <td>160.39</td>\n",
       "      <td>0.32 (0.24–0.43)</td>\n",
       "      <td>2</td>\n",
       "      <td>5.860</td>\n",
       "      <td>0.34 (0.04–1.19)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               HIV/AIDS   Male  Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth    OBS     EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  26940     NaN                NaN                983                NaN                NaN\n",
       "1           Anglo-Saxon    465  410.01   1.13 (1.03–1.24)                 15             17.834   0.84 (0.47–1.39)\n",
       "2       Northern Europe     60   50.92   1.18 (0.90–1.52)                  7              2.315   3.02 (1.21–6.20)\n",
       "3        Western Europe    285  243.59   1.17 (1.04–1.31)                 13             10.881   1.19 (0.64–2.04)\n",
       "4       Southern Europe     90  117.41   0.77 (0.62–0.94)                  1              3.302   0.30 (0.00–1.53)\n",
       "5        Eastern Europe     49  105.99   0.46 (0.34–0.61)                  2              3.840   0.52 (0.06–1.82)\n",
       "6           Former USSR     12   88.64   0.14 (0.07–0.24)                  0              4.143                NaN\n",
       "7           Middle East    122  298.09   0.41 (0.34–0.49)                  6             10.698   0.56 (0.20–1.21)\n",
       "8      Arab Middle East     70  137.70   0.51 (0.40–0.64)                  4              4.837   0.83 (0.22–2.09)\n",
       "9  Non-Arab Middle East     52  160.39   0.32 (0.24–0.43)                  2              5.860   0.34 (0.04–1.19)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table 8\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T8-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Suicide</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">20,441</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,483</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">426</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">75.42</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.65 (5.12&#8211;6.21)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">145</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.97</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.27 (3.60&#8211;5.02)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.75</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6.25 (4.78&#8211;8.03)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.16</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.81 (2.94&#8211;7.43)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">236</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42.85</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.51 (4.83&#8211;6.26)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">106</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.84</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.63 (4.61&#8211;6.80)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.51</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.81 (3.03&#8211;4.73)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.07</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.56 (0.78&#8211;2.78)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">143</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20.31</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7.04 (5.93&#8211;8.29)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.51</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.73 (4.15&#8211;7.72)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">95</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.48</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.42 (3.58&#8211;5.41)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">28</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.68</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.23 (2.14&#8211;4.66)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">143</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">58.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.44 (2.06&#8211;2.88)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">38</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.11</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.80 (1.27&#8211;2.47)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">50</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25.46</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.96 (1.46&#8211;2.59)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.53</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.52 (0.81&#8211;2.60)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">93</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.04</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.81 (2.27&#8211;3.45)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">25</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.58</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.99 (1.29&#8211;2.93)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Suicide</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>20441</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5483</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>426</td>\n",
       "      <td>75.42</td>\n",
       "      <td>5.65 (5.12–6.21)</td>\n",
       "      <td>145</td>\n",
       "      <td>33.97</td>\n",
       "      <td>4.27 (3.60–5.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>61</td>\n",
       "      <td>9.75</td>\n",
       "      <td>6.25 (4.78–8.03)</td>\n",
       "      <td>20</td>\n",
       "      <td>4.16</td>\n",
       "      <td>4.81 (2.94–7.43)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>236</td>\n",
       "      <td>42.85</td>\n",
       "      <td>5.51 (4.83–6.26)</td>\n",
       "      <td>106</td>\n",
       "      <td>18.84</td>\n",
       "      <td>5.63 (4.61–6.80)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>82</td>\n",
       "      <td>21.51</td>\n",
       "      <td>3.81 (3.03–4.73)</td>\n",
       "      <td>11</td>\n",
       "      <td>7.07</td>\n",
       "      <td>1.56 (0.78–2.78)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>143</td>\n",
       "      <td>20.31</td>\n",
       "      <td>7.04 (5.93–8.29)</td>\n",
       "      <td>43</td>\n",
       "      <td>7.51</td>\n",
       "      <td>5.73 (4.15–7.72)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>95</td>\n",
       "      <td>21.48</td>\n",
       "      <td>4.42 (3.58–5.41)</td>\n",
       "      <td>28</td>\n",
       "      <td>8.68</td>\n",
       "      <td>3.23 (2.14–4.66)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>143</td>\n",
       "      <td>58.50</td>\n",
       "      <td>2.44 (2.06–2.88)</td>\n",
       "      <td>38</td>\n",
       "      <td>21.11</td>\n",
       "      <td>1.80 (1.27–2.47)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>50</td>\n",
       "      <td>25.46</td>\n",
       "      <td>1.96 (1.46–2.59)</td>\n",
       "      <td>13</td>\n",
       "      <td>8.53</td>\n",
       "      <td>1.52 (0.81–2.60)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>93</td>\n",
       "      <td>33.04</td>\n",
       "      <td>2.81 (2.27–3.45)</td>\n",
       "      <td>25</td>\n",
       "      <td>12.58</td>\n",
       "      <td>1.99 (1.29–2.93)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                Suicide   Male Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth    OBS    EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  20441    NaN                NaN               5483                NaN                NaN\n",
       "1           Anglo-Saxon    426  75.42   5.65 (5.12–6.21)                145              33.97   4.27 (3.60–5.02)\n",
       "2       Northern Europe     61   9.75   6.25 (4.78–8.03)                 20               4.16   4.81 (2.94–7.43)\n",
       "3        Western Europe    236  42.85   5.51 (4.83–6.26)                106              18.84   5.63 (4.61–6.80)\n",
       "4       Southern Europe     82  21.51   3.81 (3.03–4.73)                 11               7.07   1.56 (0.78–2.78)\n",
       "5        Eastern Europe    143  20.31   7.04 (5.93–8.29)                 43               7.51   5.73 (4.15–7.72)\n",
       "6           Former USSR     95  21.48   4.42 (3.58–5.41)                 28               8.68   3.23 (2.14–4.66)\n",
       "7           Middle East    143  58.50   2.44 (2.06–2.88)                 38              21.11   1.80 (1.27–2.47)\n",
       "8      Arab Middle East     50  25.46   1.96 (1.46–2.59)                 13               8.53   1.52 (0.81–2.60)\n",
       "9  Non-Arab Middle East     93  33.04   2.81 (2.27–3.45)                 25              12.58   1.99 (1.29–2.93)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T9\n",
      "  label_text: Table 9\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T9-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Diabetes</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">12,057</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">12,344</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">280</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">314.72</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.89 (0.79&#8211;1.00)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">318</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">405.81</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.78 (0.70&#8211;0.87)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">28</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.67 (0.44&#8211;0.96)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">44.96</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.73 (0.51&#8211;1.03)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">93.93</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.86 (0.68&#8211;1.07)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">139</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">145.41</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.96 (0.80&#8211;1.13)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">107</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">82.34</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.30 (1.06&#8211;1.57)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">174</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">90.47</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.92 (1.65&#8211;2.23)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">106</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">86.98</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.22 (1.00&#8211;1.47)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">103</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">88.05</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.17 (0.95&#8211;1.42)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Former USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">75</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">62.78</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.19 (0.94&#8211;1.50)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">101</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">89.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.13 (0.92&#8211;1.38)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">195</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">121.38</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.61 (1.39&#8211;1.85)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">190</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">104.33</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.82(1.57&#8211;2.10)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">86</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45.48</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.89 (1.51&#8211;2.34)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">73</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35.74</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.04 (1.60&#8211;2.57)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">109</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">75.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.44 (1.18&#8211;1.73)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">117</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">68.58</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.71 (1.41&#8211;2.04)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Diabetes</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>12057</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>12344</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>280</td>\n",
       "      <td>314.72</td>\n",
       "      <td>0.89 (0.79–1.00)</td>\n",
       "      <td>318</td>\n",
       "      <td>405.81</td>\n",
       "      <td>0.78 (0.70–0.87)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>28</td>\n",
       "      <td>42.10</td>\n",
       "      <td>0.67 (0.44–0.96)</td>\n",
       "      <td>33</td>\n",
       "      <td>44.96</td>\n",
       "      <td>0.73 (0.51–1.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>81</td>\n",
       "      <td>93.93</td>\n",
       "      <td>0.86 (0.68–1.07)</td>\n",
       "      <td>139</td>\n",
       "      <td>145.41</td>\n",
       "      <td>0.96 (0.80–1.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>107</td>\n",
       "      <td>82.34</td>\n",
       "      <td>1.30 (1.06–1.57)</td>\n",
       "      <td>174</td>\n",
       "      <td>90.47</td>\n",
       "      <td>1.92 (1.65–2.23)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>106</td>\n",
       "      <td>86.98</td>\n",
       "      <td>1.22 (1.00–1.47)</td>\n",
       "      <td>103</td>\n",
       "      <td>88.05</td>\n",
       "      <td>1.17 (0.95–1.42)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Former USSR</td>\n",
       "      <td>75</td>\n",
       "      <td>62.78</td>\n",
       "      <td>1.19 (0.94–1.50)</td>\n",
       "      <td>101</td>\n",
       "      <td>89.20</td>\n",
       "      <td>1.13 (0.92–1.38)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>195</td>\n",
       "      <td>121.38</td>\n",
       "      <td>1.61 (1.39–1.85)</td>\n",
       "      <td>190</td>\n",
       "      <td>104.33</td>\n",
       "      <td>1.82(1.57–2.10)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>86</td>\n",
       "      <td>45.48</td>\n",
       "      <td>1.89 (1.51–2.34)</td>\n",
       "      <td>73</td>\n",
       "      <td>35.74</td>\n",
       "      <td>2.04 (1.60–2.57)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>109</td>\n",
       "      <td>75.90</td>\n",
       "      <td>1.44 (1.18–1.73)</td>\n",
       "      <td>117</td>\n",
       "      <td>68.58</td>\n",
       "      <td>1.71 (1.41–2.04)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               Diabetes   Male  Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth    OBS     EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  12057     NaN                NaN              12344                NaN                NaN\n",
       "1           Anglo-Saxon    280  314.72   0.89 (0.79–1.00)                318             405.81   0.78 (0.70–0.87)\n",
       "2       Northern Europe     28   42.10   0.67 (0.44–0.96)                 33              44.96   0.73 (0.51–1.03)\n",
       "3        Western Europe     81   93.93   0.86 (0.68–1.07)                139             145.41   0.96 (0.80–1.13)\n",
       "4       Southern Europe    107   82.34   1.30 (1.06–1.57)                174              90.47   1.92 (1.65–2.23)\n",
       "5        Eastern Europe    106   86.98   1.22 (1.00–1.47)                103              88.05   1.17 (0.95–1.42)\n",
       "6           Former USSR     75   62.78   1.19 (0.94–1.50)                101              89.20   1.13 (0.92–1.38)\n",
       "7           Middle East    195  121.38   1.61 (1.39–1.85)                190             104.33    1.82(1.57–2.10)\n",
       "8      Arab Middle East     86   45.48   1.89 (1.51–2.34)                 73              35.74   2.04 (1.60–2.57)\n",
       "9  Non-Arab Middle East    109   75.90   1.44 (1.18–1.73)                117              68.58   1.71 (1.41–2.04)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T10\n",
      "  label_text: Table 10\n",
      "  caption_text: Proportionate mortality ratio in first generation NHW immigrants, selected area of birth, California, 1989–1999\n",
      "  footer_text: \n",
      "    unique: PMC2268633/PMC2268633.xml::T10-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Homicide</th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Male\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Female\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Place of birth</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">OBS</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">EXP</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">PMR (95% CI)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NHW US-born</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5,519</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1,971</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Anglo-Saxon</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">375.35</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.16 (0.12&#8211;0.21)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">126.05</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.34 (0.25&#8211;0.46)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Northern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">49.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.10 (0.03&#8211;0.23)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16.17</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.12 (0.01&#8211;0.43)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Western Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">57</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">170.49</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.33 (0.25&#8211;0.43)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">65.84</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.47 (0.32&#8211;0.67)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Southern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">25</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">104.56</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.24 (0.15&#8211;0.35)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.93</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.40 (0.19&#8211;0.74)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eastern Europe</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">103.18</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.29 (0.20&#8211;0.41)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27.29</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.48 (0.25&#8211;0.81)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">USSR</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">88.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.61 (0.46&#8211;0.80)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27.73</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.25 (0.10&#8211;0.52)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">157</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">224.91</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.70(0.59&#8211;0.82)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">58.95</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.59 (0.41&#8211;0.83)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">83</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">96.13</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.86 (0.69&#8211;1.07)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">12</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25.05</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.48 (0.25&#8211;0.84)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-Arab Middle East</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">74</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">128.78</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.57 (0.45&#8211;0.72)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.68 (0.43&#8211;1.02)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Homicide</th>\n",
       "      <th>Male</th>\n",
       "      <th>Female</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Place of birth</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "      <th>OBS</th>\n",
       "      <th>EXP</th>\n",
       "      <th>PMR (95% CI)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NHW US-born</td>\n",
       "      <td>5519</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1971</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Anglo-Saxon</td>\n",
       "      <td>60</td>\n",
       "      <td>375.35</td>\n",
       "      <td>0.16 (0.12–0.21)</td>\n",
       "      <td>43</td>\n",
       "      <td>126.05</td>\n",
       "      <td>0.34 (0.25–0.46)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Northern Europe</td>\n",
       "      <td>5</td>\n",
       "      <td>49.80</td>\n",
       "      <td>0.10 (0.03–0.23)</td>\n",
       "      <td>2</td>\n",
       "      <td>16.17</td>\n",
       "      <td>0.12 (0.01–0.43)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Western Europe</td>\n",
       "      <td>57</td>\n",
       "      <td>170.49</td>\n",
       "      <td>0.33 (0.25–0.43)</td>\n",
       "      <td>31</td>\n",
       "      <td>65.84</td>\n",
       "      <td>0.47 (0.32–0.67)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Southern Europe</td>\n",
       "      <td>25</td>\n",
       "      <td>104.56</td>\n",
       "      <td>0.24 (0.15–0.35)</td>\n",
       "      <td>10</td>\n",
       "      <td>24.93</td>\n",
       "      <td>0.40 (0.19–0.74)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Eastern Europe</td>\n",
       "      <td>30</td>\n",
       "      <td>103.18</td>\n",
       "      <td>0.29 (0.20–0.41)</td>\n",
       "      <td>13</td>\n",
       "      <td>27.29</td>\n",
       "      <td>0.48 (0.25–0.81)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>USSR</td>\n",
       "      <td>54</td>\n",
       "      <td>88.10</td>\n",
       "      <td>0.61 (0.46–0.80)</td>\n",
       "      <td>7</td>\n",
       "      <td>27.73</td>\n",
       "      <td>0.25 (0.10–0.52)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Middle East</td>\n",
       "      <td>157</td>\n",
       "      <td>224.91</td>\n",
       "      <td>0.70(0.59–0.82)</td>\n",
       "      <td>35</td>\n",
       "      <td>58.95</td>\n",
       "      <td>0.59 (0.41–0.83)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Arab Middle East</td>\n",
       "      <td>83</td>\n",
       "      <td>96.13</td>\n",
       "      <td>0.86 (0.69–1.07)</td>\n",
       "      <td>12</td>\n",
       "      <td>25.05</td>\n",
       "      <td>0.48 (0.25–0.84)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Arab Middle East</td>\n",
       "      <td>74</td>\n",
       "      <td>128.78</td>\n",
       "      <td>0.57 (0.45–0.72)</td>\n",
       "      <td>23</td>\n",
       "      <td>33.90</td>\n",
       "      <td>0.68 (0.43–1.02)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               Homicide  Male  Female Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "         Place of birth   OBS     EXP       PMR (95% CI)                OBS                EXP       PMR (95% CI)\n",
       "0           NHW US-born  5519     NaN                NaN               1971                NaN                NaN\n",
       "1           Anglo-Saxon    60  375.35   0.16 (0.12–0.21)                 43             126.05   0.34 (0.25–0.46)\n",
       "2       Northern Europe     5   49.80   0.10 (0.03–0.23)                  2              16.17   0.12 (0.01–0.43)\n",
       "3        Western Europe    57  170.49   0.33 (0.25–0.43)                 31              65.84   0.47 (0.32–0.67)\n",
       "4       Southern Europe    25  104.56   0.24 (0.15–0.35)                 10              24.93   0.40 (0.19–0.74)\n",
       "5        Eastern Europe    30  103.18   0.29 (0.20–0.41)                 13              27.29   0.48 (0.25–0.81)\n",
       "6                  USSR    54   88.10   0.61 (0.46–0.80)                  7              27.73   0.25 (0.10–0.52)\n",
       "7           Middle East   157  224.91    0.70(0.59–0.82)                 35              58.95   0.59 (0.41–0.83)\n",
       "8      Arab Middle East    83   96.13   0.86 (0.69–1.07)                 12              25.05   0.48 (0.25–0.84)\n",
       "9  Non-Arab Middle East    74  128.78   0.57 (0.45–0.72)                 23              33.90   0.68 (0.43–1.02)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0022XXXXX/PMC2287164.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Sequence variants\n",
      "  footer_text: \n",
      "    unique: PMC2287164/PMC2287164.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Homozygous</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Heterozygous</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Synonymous</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-synonymous</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Synonymous</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Non-synonymous</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Number of alterations</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">26</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">25</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">11</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">16</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Homozygous</th>\n",
       "      <th>Heterozygous</th>\n",
       "      <th>Unnamed: 3</th>\n",
       "      <th>Unnamed: 4</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Synonymous</td>\n",
       "      <td>Non-synonymous</td>\n",
       "      <td>Synonymous</td>\n",
       "      <td>Non-synonymous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Number of alterations</td>\n",
       "      <td>26</td>\n",
       "      <td>25</td>\n",
       "      <td>11</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "              Unnamed: 0  Homozygous    Heterozygous  Unnamed: 3      Unnamed: 4\n",
       "0                    NaN  Synonymous  Non-synonymous  Synonymous  Non-synonymous\n",
       "1  Number of alterations          26              25          11              16"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0022XXXXX/PMC2288569.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Oral health characteristics of cohort at age 32 years, by sex and SES\n",
      "  footer_text: a One participant unable to be classified for the age-32 SES variable. * P < 0.05; ** P < 0.01; *** P < 0.001.\n",
      "    unique: PMC2288569/PMC2288569.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"2\" rowspan=\"1\">Sex\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"2\" rowspan=\"1\">SES at age 32 years <xref ref-type=\"table-fn\" rid=\"TFN1\">a</xref><hr /></th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Characteristic</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Male (<italic>n</italic> = 472) <italic>n</italic> (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Female (<italic>n</italic> = 452) <italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">High/medium (<italic>n</italic> = 765) <italic>n</italic> (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Low (<italic>n</italic> = 158) <italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">All combined (<italic>n</italic> = 924) <italic>n</italic> (%)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"6\" rowspan=\"1\">Dental caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DMFS 12+ (median)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">260 (55.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">221 (48.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">318 (49.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">162 (58.5)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">481 (52.1)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ Decayed surfaces</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">284 (60.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">203 (44.9)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">308 (47.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">178 (64.3)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">487 (52.7)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"6\" rowspan=\"1\">Periodontal disease</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">161 (34.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">108 (23.9)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">154 (23.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">114 (41.2)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">269 (29.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;2+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">108 (22.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">73 (16.2)<xref ref-type=\"table-fn\" rid=\"TFN3\">**</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">101 (15.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">80 (28.9)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">181 (19.6)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"6\" rowspan=\"1\">Tooth loss due to caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ teeth missing</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">126 (26.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">89 (19.7)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">116 (18.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">98 (35.4)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">215 (23.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Self-rated oral health worse than average compared with others of the same age</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">231 (48.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">216 (47.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">282 (43.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">164 (59.2)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">447 (48.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Self-rated oral health worsened since age 26 years</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">161 (34.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">147 (32.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">199 (30.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">108 (39.1)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">308 (33.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"6\" rowspan=\"1\">Usual reason for visiting a dentist</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Check-up</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">192 (40.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">242 (53.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">305 (47.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">93 (33.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">434 (47.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Problem</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">280 (59.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">210 (46.5)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">341 (52.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">184 (66.4)<xref ref-type=\"table-fn\" rid=\"TFN4\">***</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">490 (53.0)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Sex\\n</th>\n",
       "      <th>SES at age 32 years a</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Characteristic</th>\n",
       "      <th>Male (n = 472) n (%)</th>\n",
       "      <th>Female (n = 452) n (%)</th>\n",
       "      <th>High/medium (n = 765) n (%)</th>\n",
       "      <th>Low (n = 158) n (%)</th>\n",
       "      <th>All combined (n = 924) n (%)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Dental caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DMFS 12+ (median)</td>\n",
       "      <td>260 (55.1)</td>\n",
       "      <td>221 (48.9)</td>\n",
       "      <td>318 (49.2)</td>\n",
       "      <td>162 (58.5)*</td>\n",
       "      <td>481 (52.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1+ Decayed surfaces</td>\n",
       "      <td>284 (60.2)</td>\n",
       "      <td>203 (44.9)***</td>\n",
       "      <td>308 (47.7)</td>\n",
       "      <td>178 (64.3)***</td>\n",
       "      <td>487 (52.7)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Periodontal disease</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1+ sites with CAL of 4+ mm</td>\n",
       "      <td>161 (34.1)</td>\n",
       "      <td>108 (23.9)***</td>\n",
       "      <td>154 (23.8)</td>\n",
       "      <td>114 (41.2)*</td>\n",
       "      <td>269 (29.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>2+ sites with CAL of 4+ mm</td>\n",
       "      <td>108 (22.9)</td>\n",
       "      <td>73 (16.2)**</td>\n",
       "      <td>101 (15.6)</td>\n",
       "      <td>80 (28.9)***</td>\n",
       "      <td>181 (19.6)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Tooth loss due to caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1+ teeth missing</td>\n",
       "      <td>126 (26.7)</td>\n",
       "      <td>89 (19.7)*</td>\n",
       "      <td>116 (18.0)</td>\n",
       "      <td>98 (35.4)***</td>\n",
       "      <td>215 (23.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Self-rated oral health worse than average compared with others of the same age</td>\n",
       "      <td>231 (48.9)</td>\n",
       "      <td>216 (47.8)</td>\n",
       "      <td>282 (43.7)</td>\n",
       "      <td>164 (59.2)***</td>\n",
       "      <td>447 (48.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Self-rated oral health worsened since age 26 years</td>\n",
       "      <td>161 (34.1)</td>\n",
       "      <td>147 (32.6)</td>\n",
       "      <td>199 (30.8)</td>\n",
       "      <td>108 (39.1)*</td>\n",
       "      <td>308 (33.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Usual reason for visiting a dentist</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Check-up</td>\n",
       "      <td>192 (40.7)</td>\n",
       "      <td>242 (53.5)</td>\n",
       "      <td>305 (47.2)</td>\n",
       "      <td>93 (33.6)</td>\n",
       "      <td>434 (47.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Problem</td>\n",
       "      <td>280 (59.3)</td>\n",
       "      <td>210 (46.5)***</td>\n",
       "      <td>341 (52.8)</td>\n",
       "      <td>184 (66.4)***</td>\n",
       "      <td>490 (53.0)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                  \\n                              Sex\\n  SES at age 32 years a              \\n              Unnamed: 4_level_0           Unnamed: 5_level_0\n",
       "                                                                    Characteristic Male (n = 472) n (%) Female (n = 452) n (%) High/medium (n = 765) n (%) Low (n = 158) n (%) All combined (n = 924) n (%)\n",
       "0                                                                    Dental caries                  NaN                    NaN                         NaN                 NaN                          NaN\n",
       "1                                                                DMFS 12+ (median)           260 (55.1)             221 (48.9)                  318 (49.2)         162 (58.5)*                   481 (52.1)\n",
       "2                                                              1+ Decayed surfaces           284 (60.2)          203 (44.9)***                  308 (47.7)       178 (64.3)***                   487 (52.7)\n",
       "3                                                              Periodontal disease                  NaN                    NaN                         NaN                 NaN                          NaN\n",
       "4                                                       1+ sites with CAL of 4+ mm           161 (34.1)          108 (23.9)***                  154 (23.8)         114 (41.2)*                   269 (29.0)\n",
       "5                                                       2+ sites with CAL of 4+ mm           108 (22.9)            73 (16.2)**                  101 (15.6)        80 (28.9)***                   181 (19.6)\n",
       "6                                                         Tooth loss due to caries                  NaN                    NaN                         NaN                 NaN                          NaN\n",
       "7                                                                 1+ teeth missing           126 (26.7)             89 (19.7)*                  116 (18.0)        98 (35.4)***                   215 (23.3)\n",
       "8   Self-rated oral health worse than average compared with others of the same age           231 (48.9)             216 (47.8)                  282 (43.7)       164 (59.2)***                   447 (48.4)\n",
       "9                               Self-rated oral health worsened since age 26 years           161 (34.1)             147 (32.6)                  199 (30.8)         108 (39.1)*                   308 (33.4)\n",
       "10                                             Usual reason for visiting a dentist                  NaN                    NaN                         NaN                 NaN                          NaN\n",
       "11                                                                        Check-up           192 (40.7)             242 (53.5)                  305 (47.2)           93 (33.6)                   434 (47.0)\n",
       "12                                                                         Problem           280 (59.3)          210 (46.5)***                  341 (52.8)       184 (66.4)***                   490 (53.0)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Distribution of responses to individual OHIP-14 items and mean item scores (n = 924)\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Distribution of responses (%)\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Dimension and description of item &#8216;Because of trouble with your teeth, mouth or dentures during the last 4 weeks, &#8230;&#8217;</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Never (0)/hardly ever (<xref ref-type=\"bibr\" rid=\"R1\">1</xref>)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">Occasionally (<xref ref-type=\"bibr\" rid=\"R2\">2</xref>)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Fairly often (<xref ref-type=\"bibr\" rid=\"R3\">3</xref>)/very often (<xref ref-type=\"bibr\" rid=\"R4\">4</xref>)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Mean (SD)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Functional limitation</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you had trouble <italic>pronouncing any words</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">92.5</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">6.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.26 (0.63)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you felt that your <italic>sense of taste</italic> has worsened?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">90.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.37 (0.72)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Physical pain</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you had <italic>painful aching</italic> in your mouth?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">72.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">22.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.86 (0.95)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you found it <italic>uncomfortable to eat any foods</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">72.7</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">21.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.86 (1.00)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Psychological discomfort</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you been <italic>self-conscious</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">73.6</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">17.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">9.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.85 (1.08)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you <italic>felt tense</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">83.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">12.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.58 (0.89)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Physical disability</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Has your <italic>diet been unsatisfactory</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">65.6</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">25.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">8.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.99 (1.08)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you had to <italic>interrupt meals</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">85.2</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.53 (0.89)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Psychological disability</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you found it <italic>difficult to relax</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">83.2</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.60 (0.85)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you been a bit <italic>embarrassed</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">80.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">14.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.70 (0.94)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Social disability</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you been a bit <italic>irritable with other people</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">83.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13.6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.55 (0.85)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you had <italic>difficulty doing your usual jobs</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">93.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.29 (0.65)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">Handicap</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you felt that life in general was <italic>less satisfying</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">89.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7.4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.39 (0.76)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Have you been totally <italic>unable to function</italic>?</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">96.5</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.17 (0.53)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Distribution of responses (%)\\n</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Dimension and description of item ‘Because of trouble with your teeth, mouth or dentures during the last 4 weeks, …’</th>\n",
       "      <th>Never (0)/hardly ever (1)</th>\n",
       "      <th>Occasionally (2)</th>\n",
       "      <th>Fairly often (3)/very often (4)</th>\n",
       "      <th>Mean (SD)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Functional limitation</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Have you had trouble pronouncing any words?</td>\n",
       "      <td>92.5</td>\n",
       "      <td>6.0</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.26 (0.63)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Have you felt that your sense of taste has worsened?</td>\n",
       "      <td>90.4</td>\n",
       "      <td>7.7</td>\n",
       "      <td>1.9</td>\n",
       "      <td>0.37 (0.72)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Physical pain</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Have you had painful aching in your mouth?</td>\n",
       "      <td>72.8</td>\n",
       "      <td>22.9</td>\n",
       "      <td>4.2</td>\n",
       "      <td>0.86 (0.95)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Have you found it uncomfortable to eat any foods?</td>\n",
       "      <td>72.7</td>\n",
       "      <td>21.5</td>\n",
       "      <td>5.7</td>\n",
       "      <td>0.86 (1.00)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Psychological discomfort</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Have you been self-conscious?</td>\n",
       "      <td>73.6</td>\n",
       "      <td>17.3</td>\n",
       "      <td>9.1</td>\n",
       "      <td>0.85 (1.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Have you felt tense?</td>\n",
       "      <td>83.9</td>\n",
       "      <td>12.1</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.58 (0.89)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Physical disability</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Has your diet been unsatisfactory?</td>\n",
       "      <td>65.6</td>\n",
       "      <td>25.6</td>\n",
       "      <td>8.8</td>\n",
       "      <td>0.99 (1.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Have you had to interrupt meals?</td>\n",
       "      <td>85.2</td>\n",
       "      <td>10.8</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.53 (0.89)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Psychological disability</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Have you found it difficult to relax?</td>\n",
       "      <td>83.2</td>\n",
       "      <td>13.7</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.60 (0.85)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Have you been a bit embarrassed?</td>\n",
       "      <td>80.8</td>\n",
       "      <td>14.5</td>\n",
       "      <td>4.7</td>\n",
       "      <td>0.70 (0.94)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Social disability</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Have you been a bit irritable with other people?</td>\n",
       "      <td>83.4</td>\n",
       "      <td>13.6</td>\n",
       "      <td>2.9</td>\n",
       "      <td>0.55 (0.85)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Have you had difficulty doing your usual jobs?</td>\n",
       "      <td>93.9</td>\n",
       "      <td>4.5</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.29 (0.65)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Handicap</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Have you felt that life in general was less satisfying?</td>\n",
       "      <td>89.9</td>\n",
       "      <td>7.4</td>\n",
       "      <td>2.7</td>\n",
       "      <td>0.39 (0.76)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Have you been totally unable to function?</td>\n",
       "      <td>96.5</td>\n",
       "      <td>2.2</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.17 (0.53)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                       \\n               Distribution of responses (%)\\n   \\n                          Unnamed: 3_level_0 Unnamed: 4_level_0\n",
       "   Dimension and description of item ‘Because of trouble with your teeth, mouth or dentures during the last 4 weeks, …’       Never (0)/hardly ever (1) Occasionally (2) Fairly often (3)/very often (4)          Mean (SD)\n",
       "0                                                                                                 Functional limitation                             NaN              NaN                             NaN                NaN\n",
       "1                                                                           Have you had trouble pronouncing any words?                            92.5              6.0                             1.5        0.26 (0.63)\n",
       "2                                                                  Have you felt that your sense of taste has worsened?                            90.4              7.7                             1.9        0.37 (0.72)\n",
       "3                                                                                                         Physical pain                             NaN              NaN                             NaN                NaN\n",
       "4                                                                            Have you had painful aching in your mouth?                            72.8             22.9                             4.2        0.86 (0.95)\n",
       "5                                                                     Have you found it uncomfortable to eat any foods?                            72.7             21.5                             5.7        0.86 (1.00)\n",
       "6                                                                                              Psychological discomfort                             NaN              NaN                             NaN                NaN\n",
       "7                                                                                         Have you been self-conscious?                            73.6             17.3                             9.1        0.85 (1.08)\n",
       "8                                                                                                  Have you felt tense?                            83.9             12.1                             4.0        0.58 (0.89)\n",
       "9                                                                                                   Physical disability                             NaN              NaN                             NaN                NaN\n",
       "10                                                                                   Has your diet been unsatisfactory?                            65.6             25.6                             8.8        0.99 (1.08)\n",
       "11                                                                                     Have you had to interrupt meals?                            85.2             10.8                             4.0        0.53 (0.89)\n",
       "12                                                                                             Psychological disability                             NaN              NaN                             NaN                NaN\n",
       "13                                                                                Have you found it difficult to relax?                            83.2             13.7                             3.0        0.60 (0.85)\n",
       "14                                                                                     Have you been a bit embarrassed?                            80.8             14.5                             4.7        0.70 (0.94)\n",
       "15                                                                                                    Social disability                             NaN              NaN                             NaN                NaN\n",
       "16                                                                     Have you been a bit irritable with other people?                            83.4             13.6                             2.9        0.55 (0.85)\n",
       "17                                                                       Have you had difficulty doing your usual jobs?                            93.9              4.5                             1.5        0.29 (0.65)\n",
       "18                                                                                                             Handicap                             NaN              NaN                             NaN                NaN\n",
       "19                                                              Have you felt that life in general was less satisfying?                            89.9              7.4                             2.7        0.39 (0.76)\n",
       "20                                                                            Have you been totally unable to function?                            96.5              2.2                             1.3        0.17 (0.53)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Prevalence, extent and severity of impacts by OHIP-14 subscale and total score (n = 924)\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Dimension</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>Prevalence</italic>: no. reporting 1+ impacts fairly/very often (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>Extent</italic>: mean no. of items reported fairly/very often (SD)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>Severity</italic>: mean OHIP-14 score (SD)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Functional limitation</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">29 (3.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.03 (0.20)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.63 (1.12)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Physical pain</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">66 (7.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.10 (0.38)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.72 (1.78)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Psychological discomfort</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">95 (10.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.13 (0.41)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.43 (1.77)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Physical disability</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">99 (10.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.13 (0.39)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.52 (1.69)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Psychological disability</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">59 (6.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.08 (0.31)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.29 (1.59)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Social disability</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">34 (3.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.04 (0.24)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.85 (1.39)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Handicap</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27 (2.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.04 (0.25)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.56 (1.16)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Total OHIP-14 score</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">216 (23.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.55 (1.44)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">8.00 (8.08)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Dimension</th>\n",
       "      <th>Prevalence: no. reporting 1+ impacts fairly/very often (%)</th>\n",
       "      <th>Extent: mean no. of items reported fairly/very often (SD)</th>\n",
       "      <th>Severity: mean OHIP-14 score (SD)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Functional limitation</td>\n",
       "      <td>29 (3.1)</td>\n",
       "      <td>0.03 (0.20)</td>\n",
       "      <td>0.63 (1.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Physical pain</td>\n",
       "      <td>66 (7.1)</td>\n",
       "      <td>0.10 (0.38)</td>\n",
       "      <td>1.72 (1.78)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Psychological discomfort</td>\n",
       "      <td>95 (10.3)</td>\n",
       "      <td>0.13 (0.41)</td>\n",
       "      <td>1.43 (1.77)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Physical disability</td>\n",
       "      <td>99 (10.7)</td>\n",
       "      <td>0.13 (0.39)</td>\n",
       "      <td>1.52 (1.69)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Psychological disability</td>\n",
       "      <td>59 (6.4)</td>\n",
       "      <td>0.08 (0.31)</td>\n",
       "      <td>1.29 (1.59)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Social disability</td>\n",
       "      <td>34 (3.7)</td>\n",
       "      <td>0.04 (0.24)</td>\n",
       "      <td>0.85 (1.39)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Handicap</td>\n",
       "      <td>27 (2.9)</td>\n",
       "      <td>0.04 (0.25)</td>\n",
       "      <td>0.56 (1.16)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Total OHIP-14 score</td>\n",
       "      <td>216 (23.4)</td>\n",
       "      <td>0.55 (1.44)</td>\n",
       "      <td>8.00 (8.08)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                  Dimension Prevalence: no. reporting 1+ impacts fairly/very often (%) Extent: mean no. of items reported fairly/very often (SD) Severity: mean OHIP-14 score (SD)\n",
       "0     Functional limitation                                                   29 (3.1)                                               0.03 (0.20)                       0.63 (1.12)\n",
       "1             Physical pain                                                   66 (7.1)                                               0.10 (0.38)                       1.72 (1.78)\n",
       "2  Psychological discomfort                                                  95 (10.3)                                               0.13 (0.41)                       1.43 (1.77)\n",
       "3       Physical disability                                                  99 (10.7)                                               0.13 (0.39)                       1.52 (1.69)\n",
       "4  Psychological disability                                                   59 (6.4)                                               0.08 (0.31)                       1.29 (1.59)\n",
       "5         Social disability                                                   34 (3.7)                                               0.04 (0.24)                       0.85 (1.39)\n",
       "6                  Handicap                                                   27 (2.9)                                               0.04 (0.25)                       0.56 (1.16)\n",
       "7       Total OHIP-14 score                                                 216 (23.4)                                               0.55 (1.44)                       8.00 (8.08)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Prevalence of impacts (fairly/very often), by sex, oral disease prevalence, self-rated oral health and usual reason for visiting a dentist\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Males (<italic>n</italic> = 472)\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Females (<italic>n</italic> = 452)\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Odds ratio (95% CI)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Odds ratio (95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Dental caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DMFS &lt; 12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">34 (16.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">44 (19.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DMFS 12+</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">74 (28.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.08 (1.32&#8211;3.28)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">64 (29.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.73 (1.12&#8211;2.69)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.014</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DS = 0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">22 (11.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">38 (15.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DS 1+</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">86 (30.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.28 (1.97&#8211;5.47)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">70 (34.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.92 (1.86&#8211;4.59)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Periodontal disease</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;0 sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">62 (19.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">67 (19.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">46 (28.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.61 (1.03&#8211;2.50)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.034</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">41 (38.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.53 (1.58&#8211;4.05)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&lt;2 sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">75 (20.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">76 (20.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;2+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">33 (30.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.70 (1.05&#8211;2.75)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.031</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">32 (43.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.11 (1.84&#8211;5.27)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Tooth loss due to caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;0 teeth missing</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">57 (16.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">68 (18.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ teeth missing</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">51 (40.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.45 (2.19&#8211;5.44)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">40 (44.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.54 (2.16&#8211;5.80)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Self-rated oral health compared with others of the same age</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Better than average</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">31 (12.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">31 (13.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Worse than average</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">77 (33.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.39 (2.13&#8211;5.40)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">77 (35.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.75 (1.76&#8211;4.30)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Self-rated oral health since age 26 years</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Improved/stayed the same</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">47 (15.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">53 (17.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Got worse</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">61 (37.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.43 (2.20&#8211;5.35)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">54 (36.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.76 (1.74&#8211;4.38)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Usual reason for visiting a dentist</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Check-up</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">23 (12.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">31 (12.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Problem</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">85 (30.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.20 (1.93&#8211;5.31)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">77 (36.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.94 (2.46&#8211;6.31)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Males (n = 472)\\n</th>\n",
       "      <th>Females (n = 452)\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>n (%)</th>\n",
       "      <th>Odds ratio (95% CI)</th>\n",
       "      <th>P-value</th>\n",
       "      <th>n (%)</th>\n",
       "      <th>Odds ratio (95% CI)</th>\n",
       "      <th>P-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Dental caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DMFS &lt; 12</td>\n",
       "      <td>34 (16.0)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>44 (19.0)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DMFS 12+</td>\n",
       "      <td>74 (28.5)</td>\n",
       "      <td>2.08 (1.32–3.28)</td>\n",
       "      <td>0.001</td>\n",
       "      <td>64 (29.0)</td>\n",
       "      <td>1.73 (1.12–2.69)</td>\n",
       "      <td>0.014</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DS = 0</td>\n",
       "      <td>22 (11.7)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>38 (15.3)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DS 1+</td>\n",
       "      <td>86 (30.3)</td>\n",
       "      <td>3.28 (1.97–5.47)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>70 (34.5)</td>\n",
       "      <td>2.92 (1.86–4.59)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Periodontal disease</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0 sites with CAL of 4+ mm</td>\n",
       "      <td>62 (19.9)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>67 (19.5)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1+ sites with CAL of 4+ mm</td>\n",
       "      <td>46 (28.6)</td>\n",
       "      <td>1.61 (1.03–2.50)</td>\n",
       "      <td>0.034</td>\n",
       "      <td>41 (38.0)</td>\n",
       "      <td>2.53 (1.58–4.05)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>&lt;2 sites with CAL of 4+ mm</td>\n",
       "      <td>75 (20.6)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>76 (20.1)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>2+ sites with CAL of 4+ mm</td>\n",
       "      <td>33 (30.6)</td>\n",
       "      <td>1.70 (1.05–2.75)</td>\n",
       "      <td>0.031</td>\n",
       "      <td>32 (43.8)</td>\n",
       "      <td>3.11 (1.84–5.27)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Tooth loss due to caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0 teeth missing</td>\n",
       "      <td>57 (16.5)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>68 (18.7)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1+ teeth missing</td>\n",
       "      <td>51 (40.5)</td>\n",
       "      <td>3.45 (2.19–5.44)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>40 (44.9)</td>\n",
       "      <td>3.54 (2.16–5.80)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Self-rated oral health compared with others of the same age</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Better than average</td>\n",
       "      <td>31 (12.9)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>31 (13.1)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Worse than average</td>\n",
       "      <td>77 (33.3)</td>\n",
       "      <td>3.39 (2.13–5.40)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>77 (35.6)</td>\n",
       "      <td>2.75 (1.76–4.30)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Self-rated oral health since age 26 years</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Improved/stayed the same</td>\n",
       "      <td>47 (15.1)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>53 (17.4)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Got worse</td>\n",
       "      <td>61 (37.9)</td>\n",
       "      <td>3.43 (2.20–5.35)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>54 (36.7)</td>\n",
       "      <td>2.76 (1.74–4.38)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Usual reason for visiting a dentist</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Check-up</td>\n",
       "      <td>23 (12.0)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>31 (12.8)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Problem</td>\n",
       "      <td>85 (30.4)</td>\n",
       "      <td>3.20 (1.93–5.31)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>77 (36.7)</td>\n",
       "      <td>3.94 (2.46–6.31)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               \\n               Males (n = 472)\\n Females (n = 452)\\n Unnamed: 3_level_0 Unnamed: 4_level_0  Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "                                               \\n                           n (%) Odds ratio (95% CI)            P-value              n (%) Odds ratio (95% CI)            P-value\n",
       "0                                                 Dental caries               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "1                                                     DMFS < 12         34 (16.0)                1.00                NaN          44 (19.0)                1.00                NaN\n",
       "2                                                      DMFS 12+         74 (28.5)    2.08 (1.32–3.28)              0.001          64 (29.0)    1.73 (1.12–2.69)              0.014\n",
       "3                                                        DS = 0         22 (11.7)                1.00                NaN          38 (15.3)                1.00                NaN\n",
       "4                                                         DS 1+         86 (30.3)    3.28 (1.97–5.47)             <0.001          70 (34.5)    2.92 (1.86–4.59)             <0.001\n",
       "5                                           Periodontal disease               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "6                                     0 sites with CAL of 4+ mm         62 (19.9)                1.00                NaN          67 (19.5)                1.00                NaN\n",
       "7                                    1+ sites with CAL of 4+ mm         46 (28.6)    1.61 (1.03–2.50)              0.034          41 (38.0)    2.53 (1.58–4.05)             <0.001\n",
       "8                                    <2 sites with CAL of 4+ mm         75 (20.6)                1.00                NaN          76 (20.1)                1.00                NaN\n",
       "9                                    2+ sites with CAL of 4+ mm         33 (30.6)    1.70 (1.05–2.75)              0.031          32 (43.8)    3.11 (1.84–5.27)             <0.001\n",
       "10                                     Tooth loss due to caries               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "11                                              0 teeth missing         57 (16.5)                1.00                NaN          68 (18.7)                1.00                NaN\n",
       "12                                             1+ teeth missing         51 (40.5)    3.45 (2.19–5.44)             <0.001          40 (44.9)    3.54 (2.16–5.80)             <0.001\n",
       "13  Self-rated oral health compared with others of the same age               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "14                                          Better than average         31 (12.9)                1.00                NaN          31 (13.1)                1.00                NaN\n",
       "15                                           Worse than average         77 (33.3)    3.39 (2.13–5.40)             <0.001          77 (35.6)    2.75 (1.76–4.30)             <0.001\n",
       "16                    Self-rated oral health since age 26 years               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "17                                     Improved/stayed the same         47 (15.1)                1.00                NaN          53 (17.4)                1.00                NaN\n",
       "18                                                    Got worse         61 (37.9)    3.43 (2.20–5.35)             <0.001          54 (36.7)    2.76 (1.74–4.38)             <0.001\n",
       "19                          Usual reason for visiting a dentist               NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "20                                                     Check-up         23 (12.0)                1.00                NaN          31 (12.8)                1.00                NaN\n",
       "21                                                      Problem         85 (30.4)    3.20 (1.93–5.31)             <0.001          77 (36.7)    3.94 (2.46–6.31)             <0.001"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T5\n",
      "  label_text: Table 5\n",
      "  caption_text: Prevalence of impacts (fairly/very often), by SES at age 32 years, oral disease prevalence, self-rated oral health and usual reason for visiting a dentist\n",
      "  footer_text: \n",
      "    unique: PMC2288569/PMC2288569.xml::T5-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">High/medium SES (<italic>n</italic> = 765)\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"3\" rowspan=\"1\">Low SES (<italic>n</italic> = 158)\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Odds ratio (95% CI)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>n</italic> (%)</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Odds ratio (95% CI)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Dental caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DMFS &lt; 12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">52 (15.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">26 (22.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DMFS 12+ (median)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">69 (21.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.47 (0.99&#8211;2.19)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.06</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">68 (42.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.48 (1.45&#8211;4.24)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DS = 0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">42 (12.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">18 (18.2)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;DS 1+</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">79 (25.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.43 (1.61&#8211;3.67)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">76 (42.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.35 (1.86&#8211;6.05)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Periodontal disease</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;0 sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">82 (16.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">47 (28.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">39 (25.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.70 (1.10&#8211;2.62)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.02</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">47 (41.2)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.73 (1.05&#8211;2.87)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.03</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&lt;2 sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">92 (16.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">58 (29.4)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;2+ sites with CAL of 4+ mm</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">29 (28.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.98 (1.22&#8211;3.22)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.005</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">36 (45.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.96 (1.15&#8211;3.35)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.013</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Tooth loss because of caries</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;0 teeth missing</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">80 (15.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">45 (25.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;1+ teeth missing</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">41 (35.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.08 (1.96&#8211;4.82)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">49 (50.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.98 (1.77&#8211;5.01)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Self-rated oral health compared with others of the same age</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Better than average</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">42 (11.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">20 (17.7)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Worse than average</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">79 (28.0)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.98 (1.97&#8211;4.51)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">74 (45.1)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.82 (2.16&#8211;6.78)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Self-rated oral health since age 26 years</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Improved/stayed the same</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">57 (12.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">43 (25.6)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Got worse</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">64 (32.2)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.24 (2.16&#8211;4.87)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">50 (46.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.51 (1.50&#8211;4.19)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">Usual reason for visiting a dentist</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Check-up</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">37 (10.9)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">17 (18.3)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Problem</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">84 (27.5)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.12 (2.04&#8211;4.77)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">77 (41.8)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.22 (1.76&#8211;5.87)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>High/medium SES (n = 765)\\n</th>\n",
       "      <th>Low SES (n = 158)\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>n (%)</th>\n",
       "      <th>Odds ratio (95% CI)</th>\n",
       "      <th>P-value</th>\n",
       "      <th>n (%)</th>\n",
       "      <th>Odds ratio (95% CI)</th>\n",
       "      <th>P-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Dental caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DMFS &lt; 12</td>\n",
       "      <td>52 (15.9)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>26 (22.6)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DMFS 12+ (median)</td>\n",
       "      <td>69 (21.7)</td>\n",
       "      <td>1.47 (0.99–2.19)</td>\n",
       "      <td>0.06</td>\n",
       "      <td>68 (42.0)</td>\n",
       "      <td>2.48 (1.45–4.24)</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DS = 0</td>\n",
       "      <td>42 (12.4)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>18 (18.2)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DS 1+</td>\n",
       "      <td>79 (25.6)</td>\n",
       "      <td>2.43 (1.61–3.67)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>76 (42.7)</td>\n",
       "      <td>3.35 (1.86–6.05)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Periodontal disease</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0 sites with CAL of 4+ mm</td>\n",
       "      <td>82 (16.7)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>47 (28.8)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1+ sites with CAL of 4+ mm</td>\n",
       "      <td>39 (25.3)</td>\n",
       "      <td>1.70 (1.10–2.62)</td>\n",
       "      <td>0.02</td>\n",
       "      <td>47 (41.2)</td>\n",
       "      <td>1.73 (1.05–2.87)</td>\n",
       "      <td>0.03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>&lt;2 sites with CAL of 4+ mm</td>\n",
       "      <td>92 (16.9)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>58 (29.4)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>2+ sites with CAL of 4+ mm</td>\n",
       "      <td>29 (28.7)</td>\n",
       "      <td>1.98 (1.22–3.22)</td>\n",
       "      <td>0.005</td>\n",
       "      <td>36 (45.0)</td>\n",
       "      <td>1.96 (1.15–3.35)</td>\n",
       "      <td>0.013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Tooth loss because of caries</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0 teeth missing</td>\n",
       "      <td>80 (15.1)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>45 (25.1)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1+ teeth missing</td>\n",
       "      <td>41 (35.3)</td>\n",
       "      <td>3.08 (1.96–4.82)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>49 (50.0)</td>\n",
       "      <td>2.98 (1.77–5.01)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Self-rated oral health compared with others of the same age</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Better than average</td>\n",
       "      <td>42 (11.5)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>20 (17.7)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Worse than average</td>\n",
       "      <td>79 (28.0)</td>\n",
       "      <td>2.98 (1.97–4.51)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>74 (45.1)</td>\n",
       "      <td>3.82 (2.16–6.78)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Self-rated oral health since age 26 years</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Improved/stayed the same</td>\n",
       "      <td>57 (12.8)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>43 (25.6)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Got worse</td>\n",
       "      <td>64 (32.2)</td>\n",
       "      <td>3.24 (2.16–4.87)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>50 (46.3)</td>\n",
       "      <td>2.51 (1.50–4.19)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Usual reason for visiting a dentist</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Check-up</td>\n",
       "      <td>37 (10.9)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>17 (18.3)</td>\n",
       "      <td>1.00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Problem</td>\n",
       "      <td>84 (27.5)</td>\n",
       "      <td>3.12 (2.04–4.77)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>77 (41.8)</td>\n",
       "      <td>3.22 (1.76–5.87)</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               \\n               High/medium SES (n = 765)\\n Low SES (n = 158)\\n Unnamed: 3_level_0 Unnamed: 4_level_0  Unnamed: 5_level_0 Unnamed: 6_level_0\n",
       "                                               \\n                                     n (%) Odds ratio (95% CI)            P-value              n (%) Odds ratio (95% CI)            P-value\n",
       "0                                                 Dental caries                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "1                                                     DMFS < 12                   52 (15.9)                1.00                NaN          26 (22.6)                1.00                NaN\n",
       "2                                             DMFS 12+ (median)                   69 (21.7)    1.47 (0.99–2.19)               0.06          68 (42.0)    2.48 (1.45–4.24)              0.001\n",
       "3                                                        DS = 0                   42 (12.4)                1.00                NaN          18 (18.2)                1.00                NaN\n",
       "4                                                         DS 1+                   79 (25.6)    2.43 (1.61–3.67)             <0.001          76 (42.7)    3.35 (1.86–6.05)             <0.001\n",
       "5                                           Periodontal disease                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "6                                     0 sites with CAL of 4+ mm                   82 (16.7)                1.00                NaN          47 (28.8)                1.00                NaN\n",
       "7                                    1+ sites with CAL of 4+ mm                   39 (25.3)    1.70 (1.10–2.62)               0.02          47 (41.2)    1.73 (1.05–2.87)               0.03\n",
       "8                                    <2 sites with CAL of 4+ mm                   92 (16.9)                1.00                NaN          58 (29.4)                1.00                NaN\n",
       "9                                    2+ sites with CAL of 4+ mm                   29 (28.7)    1.98 (1.22–3.22)              0.005          36 (45.0)    1.96 (1.15–3.35)              0.013\n",
       "10                                 Tooth loss because of caries                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "11                                              0 teeth missing                   80 (15.1)                1.00                NaN          45 (25.1)                1.00                NaN\n",
       "12                                             1+ teeth missing                   41 (35.3)    3.08 (1.96–4.82)             <0.001          49 (50.0)    2.98 (1.77–5.01)             <0.001\n",
       "13  Self-rated oral health compared with others of the same age                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "14                                          Better than average                   42 (11.5)                1.00                NaN          20 (17.7)                1.00                NaN\n",
       "15                                           Worse than average                   79 (28.0)    2.98 (1.97–4.51)             <0.001          74 (45.1)    3.82 (2.16–6.78)             <0.001\n",
       "16                    Self-rated oral health since age 26 years                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "17                                     Improved/stayed the same                   57 (12.8)                1.00                NaN          43 (25.6)                1.00                NaN\n",
       "18                                                    Got worse                   64 (32.2)    3.24 (2.16–4.87)             <0.001          50 (46.3)    2.51 (1.50–4.19)             <0.001\n",
       "19                          Usual reason for visiting a dentist                         NaN                 NaN                NaN                NaN                 NaN                NaN\n",
       "20                                                     Check-up                   37 (10.9)                1.00                NaN          17 (18.3)                1.00                NaN\n",
       "21                                                      Problem                   84 (27.5)    3.12 (2.04–4.77)             <0.001          77 (41.8)    3.22 (1.76–5.87)             <0.001"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T6\n",
      "  label_text: Table 6\n",
      "  caption_text: Logistic regression model for prevalence of impacts (‘fairly often’ or ‘very often’) at age 32 years (n = 923) Coeff. Odds ratio 95% CI for OR P-value\n",
      "  footer_text: −2 Log-likelihood = 896.69; Cox & Snell R2 = 0.11; Nagelkerke R2 = 0.16; Hosmer and Lemeshow chi-squared test = 3.01, d.f. = 8, P = 0.93.\n",
      "    unique: PMC2288569/PMC2288569.xml::T6-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">Coeff.</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Odds ratio</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">95% CI for OR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Constant</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&#8722;2.611</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.313</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.37</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.98&#8211;1.91</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.065</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Low SES</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.451</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.57</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.12&#8211;2.21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.01</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Episodic dental user</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.840</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.32</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.60&#8211;3.36</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Any decayed surface (DS) at 32 years</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.666</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.95</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.35&#8211;2.81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">No. of teeth missing (due to caries) by 32 years</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.153</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.17</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.05&#8211;1.29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.003</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.401</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.49</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.02&#8211;2.19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.04</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th>Coeff.</th>\n",
       "      <th>Odds ratio</th>\n",
       "      <th>95% CI for OR</th>\n",
       "      <th>P-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Constant</td>\n",
       "      <td>−2.611</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Female</td>\n",
       "      <td>0.313</td>\n",
       "      <td>1.37</td>\n",
       "      <td>0.98–1.91</td>\n",
       "      <td>0.065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Low SES</td>\n",
       "      <td>0.451</td>\n",
       "      <td>1.57</td>\n",
       "      <td>1.12–2.21</td>\n",
       "      <td>0.01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Episodic dental user</td>\n",
       "      <td>0.840</td>\n",
       "      <td>2.32</td>\n",
       "      <td>1.60–3.36</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Any decayed surface (DS) at 32 years</td>\n",
       "      <td>0.666</td>\n",
       "      <td>1.95</td>\n",
       "      <td>1.35–2.81</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>No. of teeth missing (due to caries) by 32 years</td>\n",
       "      <td>0.153</td>\n",
       "      <td>1.17</td>\n",
       "      <td>1.05–1.29</td>\n",
       "      <td>0.003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)</td>\n",
       "      <td>0.401</td>\n",
       "      <td>1.49</td>\n",
       "      <td>1.02–2.19</td>\n",
       "      <td>0.04</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                    \\n                Coeff.  Odds ratio 95% CI for OR P-value\n",
       "0                                                           Constant  −2.611         NaN           NaN     NaN\n",
       "1                                                             Female   0.313        1.37     0.98–1.91   0.065\n",
       "2                                                            Low SES   0.451        1.57     1.12–2.21    0.01\n",
       "3                                               Episodic dental user   0.840        2.32     1.60–3.36  <0.001\n",
       "4                               Any decayed surface (DS) at 32 years   0.666        1.95     1.35–2.81  <0.001\n",
       "5                   No. of teeth missing (due to caries) by 32 years   0.153        1.17     1.05–1.29   0.003\n",
       "6  Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)   0.401        1.49     1.02–2.19    0.04"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T7\n",
      "  label_text: Table 7\n",
      "  caption_text: Poisson regression models for the extent and severity of impacts (n = 923)\n",
      "  footer_text: a Ratio of geometric mean values.\n",
      "    unique: PMC2288569/PMC2288569.xml::T7-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\"><italic>Extent</italic>: mean no. of OHIP items reported fairly/very often (pseudo-<italic>R</italic><sup>2</sup> = 0.13)\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\"><italic>Severity</italic>: mean total OHIP-14 score(pseudo-<italic>R</italic><sup>2</sup> = 0.09)\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Coeff.</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">IRR<xref ref-type=\"table-fn\" rid=\"TFN6\">a</xref></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">95% CI for IRR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Coeff.</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">IRR<xref ref-type=\"table-fn\" rid=\"TFN6\">a</xref></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">95% CI for IRR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\"><italic>P</italic>-value</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Constant</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;1.93</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.54</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8211;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.30</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.12&#8211;1.64</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.002</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.04</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.99&#8211;1.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.148</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Low SES</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.28</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.09&#8211;1.62</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.005</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.11</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.06&#8211;1.17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Episodic dental user</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.76</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.68&#8211;2.71</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.40</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.49</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.41&#8211;1.57</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Any decayed surface (DS) at 32 years</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.63</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.88</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.50&#8211;2.36</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.29</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.34</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.27&#8211;1.41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">No. of teeth missing (because of caries) by 32 years</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.12</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.09&#8211;1.15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.06</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.07</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.06&#8211;1.08</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.14</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.86&#8211;1.52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.349</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.13</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.05&#8211;1.23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.002</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Extent: mean no. of OHIP items reported fairly/very often (pseudo-R2 = 0.13)\\n</th>\n",
       "      <th>Severity: mean total OHIP-14 score(pseudo-R2 = 0.09)\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Coeff.</th>\n",
       "      <th>IRRa</th>\n",
       "      <th>95% CI for IRR</th>\n",
       "      <th>P-value</th>\n",
       "      <th>Coeff.</th>\n",
       "      <th>IRRa</th>\n",
       "      <th>95% CI for IRR</th>\n",
       "      <th>P-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Constant</td>\n",
       "      <td>−1.93</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Female</td>\n",
       "      <td>0.30</td>\n",
       "      <td>1.35</td>\n",
       "      <td>1.12–1.64</td>\n",
       "      <td>0.002</td>\n",
       "      <td>0.04</td>\n",
       "      <td>1.04</td>\n",
       "      <td>0.99–1.09</td>\n",
       "      <td>0.148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Low SES</td>\n",
       "      <td>0.28</td>\n",
       "      <td>1.33</td>\n",
       "      <td>1.09–1.62</td>\n",
       "      <td>0.005</td>\n",
       "      <td>0.11</td>\n",
       "      <td>1.11</td>\n",
       "      <td>1.06–1.17</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Episodic dental user</td>\n",
       "      <td>0.76</td>\n",
       "      <td>2.14</td>\n",
       "      <td>1.68–2.71</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>0.40</td>\n",
       "      <td>1.49</td>\n",
       "      <td>1.41–1.57</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Any decayed surface (DS) at 32 years</td>\n",
       "      <td>0.63</td>\n",
       "      <td>1.88</td>\n",
       "      <td>1.50–2.36</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>0.29</td>\n",
       "      <td>1.34</td>\n",
       "      <td>1.27–1.41</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>No. of teeth missing (because of caries) by 32 years</td>\n",
       "      <td>0.12</td>\n",
       "      <td>1.12</td>\n",
       "      <td>1.09–1.15</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>0.06</td>\n",
       "      <td>1.07</td>\n",
       "      <td>1.06–1.08</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)</td>\n",
       "      <td>0.14</td>\n",
       "      <td>1.14</td>\n",
       "      <td>0.86–1.52</td>\n",
       "      <td>0.349</td>\n",
       "      <td>0.13</td>\n",
       "      <td>1.14</td>\n",
       "      <td>1.05–1.23</td>\n",
       "      <td>0.002</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                    \\n               Extent: mean no. of OHIP items reported fairly/very often (pseudo-R2 = 0.13)\\n Severity: mean total OHIP-14 score(pseudo-R2 = 0.09)\\n Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0\n",
       "                                                    \\n                                                                                       Coeff.                                                   IRRa     95% CI for IRR            P-value             Coeff.               IRRa     95% CI for IRR            P-value\n",
       "0                                                           Constant                                                                          −1.93                                                    NaN                NaN                NaN               1.54                NaN                NaN                NaN\n",
       "1                                                             Female                                                                           0.30                                                   1.35          1.12–1.64              0.002               0.04               1.04          0.99–1.09              0.148\n",
       "2                                                            Low SES                                                                           0.28                                                   1.33          1.09–1.62              0.005               0.11               1.11          1.06–1.17             <0.001\n",
       "3                                               Episodic dental user                                                                           0.76                                                   2.14          1.68–2.71             <0.001               0.40               1.49          1.41–1.57             <0.001\n",
       "4                               Any decayed surface (DS) at 32 years                                                                           0.63                                                   1.88          1.50–2.36             <0.001               0.29               1.34          1.27–1.41             <0.001\n",
       "5               No. of teeth missing (because of caries) by 32 years                                                                           0.12                                                   1.12          1.09–1.15             <0.001               0.06               1.07          1.06–1.08             <0.001\n",
       "6  Case of periodontal disease (2+ sites with 4+ mm CAL at 32 years)                                                                           0.14                                                   1.14          0.86–1.52              0.349               0.13               1.14          1.05–1.23              0.002"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T8\n",
      "  label_text: Table 8\n",
      "  caption_text: International comparisons of prevalence, extent and severity of overall impacts and selected dimensions among dentate adults\n",
      "  footer_text: a Slade et al. (10). b Estimated for the UK and Australian samples by scrutinizing the data distributions.\n",
      "    unique: PMC2288569/PMC2288569.xml::T8-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">United Kingdom<xref ref-type=\"table-fn\" rid=\"TFN7\">a</xref></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Australia<xref ref-type=\"table-fn\" rid=\"TFN7\">a</xref></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">New Zealand</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">No. of participants</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5270</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3909</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">924</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">% aged 30&#8211;39 years</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Median age (years)<xref ref-type=\"table-fn\" rid=\"TFN8\">b</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">40</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">40&#8211;42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">32</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">% Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">50</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">51</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">49</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>Prevalence</italic>: % of people reporting 1+ impacts fairly/very often (95% CI)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.9 (14.9&#8211;16.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.2 (16.2&#8211;20.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">23.4 (22.0&#8211;24.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\"><italic>Extent</italic>: mean no. of items reported fairly/very often (95% CI)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.36 (0.32&#8211;0.40)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.46 (0.39&#8211;0.52)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.55 (0.46&#8211;0.65)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"4\" rowspan=\"1\"><italic>Severity</italic>: mean OHIP-14 score (95% CI)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"4\" rowspan=\"1\">&#8195;All OHIP items</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;Both sexes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.1 (4.8&#8211;5.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.5 (7.1&#8211;7.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.0 (7.5&#8211;8.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;Male aged 25&#8211;34 years</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.1 (4.8&#8211;5.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.9 (6.4&#8211;7.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.3 (7.6&#8211;9.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;Female aged 25&#8211;34 years</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.3 (5.1&#8211;5.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.9 (7.5&#8211;8.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.7 (6.9&#8211;8.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Physical pain/physical disability (both sexes)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1 (2.0&#8211;2.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.4 (3.3&#8211;3.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2 (3.0&#8211;3.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Other dimensions (both sexes)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.9 (2.8&#8211;3.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.0 (3.8&#8211;4.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.8 (4.4&#8211;5.1)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th>United Kingdoma</th>\n",
       "      <th>Australiaa</th>\n",
       "      <th>New Zealand</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>No. of participants</td>\n",
       "      <td>5270</td>\n",
       "      <td>3909</td>\n",
       "      <td>924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>% aged 30–39 years</td>\n",
       "      <td>24</td>\n",
       "      <td>21</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Median age (years)b</td>\n",
       "      <td>40</td>\n",
       "      <td>40–42</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>% Female</td>\n",
       "      <td>50</td>\n",
       "      <td>51</td>\n",
       "      <td>49</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Prevalence: % of people reporting 1+ impacts fairly/very often (95% CI)</td>\n",
       "      <td>15.9 (14.9–16.8)</td>\n",
       "      <td>18.2 (16.2–20.2)</td>\n",
       "      <td>23.4 (22.0–24.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Extent: mean no. of items reported fairly/very often (95% CI)</td>\n",
       "      <td>0.36 (0.32–0.40)</td>\n",
       "      <td>0.46 (0.39–0.52)</td>\n",
       "      <td>0.55 (0.46–0.65)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Severity: mean OHIP-14 score (95% CI)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>All OHIP items</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Both sexes</td>\n",
       "      <td>5.1 (4.8–5.3)</td>\n",
       "      <td>7.5 (7.1–7.9)</td>\n",
       "      <td>8.0 (7.5–8.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Male aged 25–34 years</td>\n",
       "      <td>5.1 (4.8–5.4)</td>\n",
       "      <td>6.9 (6.4–7.4)</td>\n",
       "      <td>8.3 (7.6–9.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Female aged 25–34 years</td>\n",
       "      <td>5.3 (5.1–5.5)</td>\n",
       "      <td>7.9 (7.5–8.3)</td>\n",
       "      <td>7.7 (6.9–8.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Physical pain/physical disability (both sexes)</td>\n",
       "      <td>2.1 (2.0–2.2)</td>\n",
       "      <td>3.4 (3.3–3.6)</td>\n",
       "      <td>3.2 (3.0–3.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Other dimensions (both sexes)</td>\n",
       "      <td>2.9 (2.8–3.1)</td>\n",
       "      <td>4.0 (3.8–4.3)</td>\n",
       "      <td>4.8 (4.4–5.1)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                           \\n                 United Kingdoma        Australiaa       New Zealand\n",
       "0                                                       No. of participants              5270              3909               924\n",
       "1                                                        % aged 30–39 years                24                21               100\n",
       "2                                                       Median age (years)b                40             40–42                32\n",
       "3                                                                  % Female                50                51                49\n",
       "4   Prevalence: % of people reporting 1+ impacts fairly/very often (95% CI)  15.9 (14.9–16.8)  18.2 (16.2–20.2)  23.4 (22.0–24.8)\n",
       "5             Extent: mean no. of items reported fairly/very often (95% CI)  0.36 (0.32–0.40)  0.46 (0.39–0.52)  0.55 (0.46–0.65)\n",
       "6                                     Severity: mean OHIP-14 score (95% CI)               NaN               NaN               NaN\n",
       "7                                                            All OHIP items               NaN               NaN               NaN\n",
       "8                                                                Both sexes     5.1 (4.8–5.3)     7.5 (7.1–7.9)     8.0 (7.5–8.5)\n",
       "9                                                     Male aged 25–34 years     5.1 (4.8–5.4)     6.9 (6.4–7.4)     8.3 (7.6–9.0)\n",
       "10                                                  Female aged 25–34 years     5.3 (5.1–5.5)     7.9 (7.5–8.3)     7.7 (6.9–8.4)\n",
       "11                           Physical pain/physical disability (both sexes)     2.1 (2.0–2.2)     3.4 (3.3–3.6)     3.2 (3.0–3.4)\n",
       "12                                            Other dimensions (both sexes)     2.9 (2.8–3.1)     4.0 (3.8–4.3)     4.8 (4.4–5.1)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0022XXXXX/PMC2293303.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Lifetime prevalence associations of gambling types with lifetime onset and persistence of CIDI/DSM-IV pathological gambling (PG)\n",
      "  footer_text: * Significant at the .05 level, two-sided test. 1 Multivariate model controlling for sex, race-ethnicity, age of first gambling, years since first gambling, and 11 gambling types. 2 Multivariate model controlling for sex, race-ethnicity, aoo of first gambling, aoo of first problem, years since first problem, and 11 gambling types\n",
      "    unique: PMC2293303/PMC2293303.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"below\" rules=\"none\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"bottom\">Prevalence</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"bottom\">Lifetime Pathological Gambling (PG) among Gamblers<xref ref-type=\"table-fn\" rid=\"TFN2\">1</xref></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"bottom\">Lifetime problem Gambling among Gamblers<xref ref-type=\"table-fn\" rid=\"TFN2\">1</xref></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"bottom\">Lifetime Pathological Gambling (PG) among Problem gamblers<xref ref-type=\"table-fn\" rid=\"TFN3\">2</xref></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"bottom\">Termination of Problem gambling<xref ref-type=\"table-fn\" rid=\"TFN3\">2</xref></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">%</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(se)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(95% CI)<xref ref-type=\"table-fn\" rid=\"TFN3\">2</xref></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(95% CI)<sup>3</sup></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(95% CI)<sup>4</sup></th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(95% CI)<sup>5</sup></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"10\" rowspan=\"1\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">I. Sports betting</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Office sports pool</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">44.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;6.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;1.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;4.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;1.2)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Sports with bookie or parlay cards</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.3<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.3&#8211;8.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.7<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.7&#8211;8.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;11.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.5&#8211;5.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Betting on horse/dog races or cock/dog fights</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">25.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.2&#8211;8.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;3.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.8&#8211;9.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;2.6)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Gambling at a casino</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">44.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(2.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.5&#8211;4.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.9&#8211;6.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;6.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.8<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;15.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">&#8195;&#8195;II. Other types of gambling that involve some aspect of mental or physical skill</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Games involving mental skill (e.g., cards)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">35.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.2<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(3.1&#8211;47.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;8.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">23.6<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(2.3&#8211;239.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;114.6)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Games involving physical skill (e.g., pool)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">22.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;5.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.8&#8211;3.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;9.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.8&#8211;7.7)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Speculating on high risk investments</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">8.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.8&#8211;5.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;2.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.3<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.0&#8211;18.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;2.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Internet gambling</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;14.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;8.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">49.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.3&#8211;9595.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.5&#8211;9.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">III. Types of gambling that largely involve chance rather than skill</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Playing numbers/lotto</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">62.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;2.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;1.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;9.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;0.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Gambling machines (e.g., video poker)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">26.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.3<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.3&#8211;8.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.9<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.0&#8211;8.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.6<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(2.3&#8211;104.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;4.1)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Slot machines, bingo, or pull tabs</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">48.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(1.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;1.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;2.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.0&#8211;1.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.3<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;0.8)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Prevalence</th>\n",
       "      <th>Lifetime Pathological Gambling (PG) among Gamblers1</th>\n",
       "      <th>Lifetime problem Gambling among Gamblers1</th>\n",
       "      <th>Lifetime Pathological Gambling (PG) among Problem gamblers2</th>\n",
       "      <th>Termination of Problem gambling2</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>%</th>\n",
       "      <th>(se)</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)2</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)3</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)4</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)5</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 2_level_2</th>\n",
       "      <th>Unnamed: 3_level_2</th>\n",
       "      <th>Unnamed: 4_level_2</th>\n",
       "      <th>Unnamed: 5_level_2</th>\n",
       "      <th>Unnamed: 6_level_2</th>\n",
       "      <th>Unnamed: 7_level_2</th>\n",
       "      <th>Unnamed: 8_level_2</th>\n",
       "      <th>Unnamed: 9_level_2</th>\n",
       "      <th>Unnamed: 10_level_2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>I. Sports betting</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Office sports pool</td>\n",
       "      <td>44.3</td>\n",
       "      <td>(1.7)</td>\n",
       "      <td>1.7</td>\n",
       "      <td>(0.4–6.5)</td>\n",
       "      <td>0.6</td>\n",
       "      <td>(0.2–1.6)</td>\n",
       "      <td>0.6</td>\n",
       "      <td>(0.1–4.5)</td>\n",
       "      <td>0.3</td>\n",
       "      <td>(0.1–1.2)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Sports with bookie or parlay cards</td>\n",
       "      <td>5.8</td>\n",
       "      <td>(0.5)</td>\n",
       "      <td>3.3*</td>\n",
       "      <td>(1.3–8.6)</td>\n",
       "      <td>3.7*</td>\n",
       "      <td>(1.7–8.1)</td>\n",
       "      <td>2.8</td>\n",
       "      <td>(0.7–11.0)</td>\n",
       "      <td>1.6</td>\n",
       "      <td>(0.5–5.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Betting on horse/dog races or cock/dog fights</td>\n",
       "      <td>25.0</td>\n",
       "      <td>(1.3)</td>\n",
       "      <td>3.2*</td>\n",
       "      <td>(1.2–8.5)</td>\n",
       "      <td>1.2</td>\n",
       "      <td>(0.4–3.7)</td>\n",
       "      <td>2.7</td>\n",
       "      <td>(0.8–9.1)</td>\n",
       "      <td>1.0</td>\n",
       "      <td>(0.4–2.6)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Gambling at a casino</td>\n",
       "      <td>44.7</td>\n",
       "      <td>(2.1)</td>\n",
       "      <td>1.4</td>\n",
       "      <td>(0.5–4.3)</td>\n",
       "      <td>2.5</td>\n",
       "      <td>(0.9–6.7)</td>\n",
       "      <td>1.2</td>\n",
       "      <td>(0.2–6.8)</td>\n",
       "      <td>4.8*</td>\n",
       "      <td>(1.5–15.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>II. Other types of gambling that involve some aspect of mental or physical skill</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Games involving mental skill (e.g., cards)</td>\n",
       "      <td>35.8</td>\n",
       "      <td>(1.2)</td>\n",
       "      <td>12.2*</td>\n",
       "      <td>(3.1–47.7)</td>\n",
       "      <td>1.4</td>\n",
       "      <td>(0.2–8.9)</td>\n",
       "      <td>23.6*</td>\n",
       "      <td>(2.3–239.3)</td>\n",
       "      <td>8.9</td>\n",
       "      <td>(0.7–114.6)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Games involving physical skill (e.g., pool)</td>\n",
       "      <td>22.7</td>\n",
       "      <td>(1.1)</td>\n",
       "      <td>1.8</td>\n",
       "      <td>(0.6–5.3)</td>\n",
       "      <td>1.6</td>\n",
       "      <td>(0.8–3.4)</td>\n",
       "      <td>1.4</td>\n",
       "      <td>(0.2–9.1)</td>\n",
       "      <td>2.5</td>\n",
       "      <td>(0.8–7.7)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Speculating on high risk investments</td>\n",
       "      <td>8.4</td>\n",
       "      <td>(0.7)</td>\n",
       "      <td>2.1</td>\n",
       "      <td>(0.8–5.6)</td>\n",
       "      <td>1.3</td>\n",
       "      <td>(0.6–2.6)</td>\n",
       "      <td>4.3*</td>\n",
       "      <td>(1.0–18.0)</td>\n",
       "      <td>0.6</td>\n",
       "      <td>(0.2–2.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Internet gambling</td>\n",
       "      <td>1.0</td>\n",
       "      <td>(0.2)</td>\n",
       "      <td>3.2</td>\n",
       "      <td>(0.7–14.8)</td>\n",
       "      <td>2.3</td>\n",
       "      <td>(0.6–8.9)</td>\n",
       "      <td>49.2</td>\n",
       "      <td>(0.3–9595.7)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>(0.5–9.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>III. Types of gambling that largely involve chance rather than skill</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Playing numbers/lotto</td>\n",
       "      <td>62.2</td>\n",
       "      <td>(1.5)</td>\n",
       "      <td>0.6</td>\n",
       "      <td>(0.2–2.4)</td>\n",
       "      <td>0.5</td>\n",
       "      <td>(0.2–1.4)</td>\n",
       "      <td>1.2</td>\n",
       "      <td>(0.1–9.6)</td>\n",
       "      <td>0.2*</td>\n",
       "      <td>(0.1–0.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Gambling machines (e.g., video poker)</td>\n",
       "      <td>26.1</td>\n",
       "      <td>(1.3)</td>\n",
       "      <td>3.3*</td>\n",
       "      <td>(1.3–8.4)</td>\n",
       "      <td>2.9*</td>\n",
       "      <td>(1.0–8.4)</td>\n",
       "      <td>15.6*</td>\n",
       "      <td>(2.3–104.9)</td>\n",
       "      <td>1.3</td>\n",
       "      <td>(0.4–4.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Slot machines, bingo, or pull tabs</td>\n",
       "      <td>48.9</td>\n",
       "      <td>(1.8)</td>\n",
       "      <td>0.5</td>\n",
       "      <td>(0.2–1.2)</td>\n",
       "      <td>1.0</td>\n",
       "      <td>(0.4–2.1)</td>\n",
       "      <td>0.2*</td>\n",
       "      <td>(0.0–1.0)</td>\n",
       "      <td>0.3*</td>\n",
       "      <td>(0.1–0.8)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                    \\n                                       Prevalence Lifetime Pathological Gambling (PG) among Gamblers1 Lifetime problem Gambling among Gamblers1 Lifetime Pathological Gambling (PG) among Problem gamblers2 Termination of Problem gambling2 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0 Unnamed: 10_level_0\n",
       "                                                                    \\n                                                %                                                (se)                                        OR                                                   (95% CI)2                               OR          (95% CI)3                 OR          (95% CI)4                 OR           (95% CI)5\n",
       "                                                                    \\n               \\n                \\n                                                Unnamed: 2_level_2                        Unnamed: 3_level_2                                          Unnamed: 4_level_2               Unnamed: 5_level_2 Unnamed: 6_level_2 Unnamed: 7_level_2 Unnamed: 8_level_2 Unnamed: 9_level_2 Unnamed: 10_level_2\n",
       "0                                                                  I. Sports betting                                NaN                                                 NaN                                       NaN                                                         NaN                              NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "1                                                                 Office sports pool                               44.3                                               (1.7)                                       1.7                                                   (0.4–6.5)                              0.6          (0.2–1.6)                0.6          (0.1–4.5)                0.3           (0.1–1.2)\n",
       "2                                                 Sports with bookie or parlay cards                                5.8                                               (0.5)                                      3.3*                                                   (1.3–8.6)                             3.7*          (1.7–8.1)                2.8         (0.7–11.0)                1.6           (0.5–5.3)\n",
       "3                                      Betting on horse/dog races or cock/dog fights                               25.0                                               (1.3)                                      3.2*                                                   (1.2–8.5)                              1.2          (0.4–3.7)                2.7          (0.8–9.1)                1.0           (0.4–2.6)\n",
       "4                                                               Gambling at a casino                               44.7                                               (2.1)                                       1.4                                                   (0.5–4.3)                              2.5          (0.9–6.7)                1.2          (0.2–6.8)               4.8*          (1.5–15.5)\n",
       "5   II. Other types of gambling that involve some aspect of mental or physical skill                                NaN                                                 NaN                                       NaN                                                         NaN                              NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "6                                         Games involving mental skill (e.g., cards)                               35.8                                               (1.2)                                     12.2*                                                  (3.1–47.7)                              1.4          (0.2–8.9)              23.6*        (2.3–239.3)                8.9         (0.7–114.6)\n",
       "7                                        Games involving physical skill (e.g., pool)                               22.7                                               (1.1)                                       1.8                                                   (0.6–5.3)                              1.6          (0.8–3.4)                1.4          (0.2–9.1)                2.5           (0.8–7.7)\n",
       "8                                               Speculating on high risk investments                                8.4                                               (0.7)                                       2.1                                                   (0.8–5.6)                              1.3          (0.6–2.6)               4.3*         (1.0–18.0)                0.6           (0.2–2.0)\n",
       "9                                                                  Internet gambling                                1.0                                               (0.2)                                       3.2                                                  (0.7–14.8)                              2.3          (0.6–8.9)               49.2       (0.3–9595.7)                2.2           (0.5–9.3)\n",
       "10              III. Types of gambling that largely involve chance rather than skill                                NaN                                                 NaN                                       NaN                                                         NaN                              NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "11                                                             Playing numbers/lotto                               62.2                                               (1.5)                                       0.6                                                   (0.2–2.4)                              0.5          (0.2–1.4)                1.2          (0.1–9.6)               0.2*           (0.1–0.8)\n",
       "12                                             Gambling machines (e.g., video poker)                               26.1                                               (1.3)                                      3.3*                                                   (1.3–8.4)                             2.9*          (1.0–8.4)              15.6*        (2.3–104.9)                1.3           (0.4–4.1)\n",
       "13                                                Slot machines, bingo, or pull tabs                               48.9                                               (1.8)                                       0.5                                                   (0.2–1.2)                              1.0          (0.4–2.1)               0.2*          (0.0–1.0)               0.3*           (0.1–0.8)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Lifetime comorbidity of CIDI/DSM-IV pathological gambling (PG) with other lifetime CIDI/DSM-IV disorders and temporal priorities in age of onset (AOO)\n",
      "  footer_text: * Significant at the .05 level, two-sided test. 1 The percent of respondents with PG who also meet criteria for the other disorder. 2 The ORs are calculated in the total sample, including respondents who never gambled. The ORs were calculated controlling for age at interview so as to avoid confounding due to time at risk.\n",
      "    unique: PMC2293303/PMC2293303.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"below\" rules=\"none\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"6\" rowspan=\"1\">Temporal priority in onset</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Prevalence<xref ref-type=\"table-fn\" rid=\"TFN5\">1</xref></th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">PG first</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Other first</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Same year</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">%</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">(se)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR<xref ref-type=\"table-fn\" rid=\"TFN6\">2</xref></th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">(95% CI)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">%</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(se)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">%</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">(se)</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">%</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">(se)</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"10\" rowspan=\"1\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">I. Mood disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Major depressive disorder or dysthymia</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">38.6</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.5<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.1&#8211;5.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">20.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(10.6)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">73.5</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(11.1)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">6.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(5.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Bipolar disorder<sup>3</sup></td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">17.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(7.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.6<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;14.2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">29.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(16.4)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">46.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(21.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">24.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(14.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any mood disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">55.6</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.7<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;9.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">23.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(9.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">65.1</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(11.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">11.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(6.9)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">II. Anxiety disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Panic disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">21.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(6.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.9<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.2&#8211;10.8)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(9.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">81.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(16.1)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">7.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(6.6)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Generalized anxiety disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">16.6</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(7.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.8<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.0&#8211;7.9)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">9.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(9.4)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">79.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(14.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(10.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Phobia</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">52.2</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(8.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.4&#8211;7.2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;PTSD</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">14.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(7.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;8.4)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">49.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(27.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">50.5</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(27.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any anxiety disorder<sup>4</sup></td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">60.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.1<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.4&#8211;7.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(7.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">82.1</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(7.9)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">4.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(4.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">III. Impulse-control disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;ADHD</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">13.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(8.1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;7.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Oppositional-defiant disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">15.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(6.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;5.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Conduct disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">24.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(8.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.1<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.2&#8211;7.8)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Intermittent explosive disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">27.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.1<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.1&#8211;8.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any impulse control disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">42.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.9&#8211;5.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">100.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(0.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">IV. Substance use disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Alcohol or drug abuse</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">46.2</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.5<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.8&#8211;11.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">18.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(10.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">70.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(11.3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">10.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(7.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Alcohol or drug dependence</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">31.8</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.8<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.4&#8211;14.4)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">44.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(16.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">55.7</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(16.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Nicotine dependence</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">63.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(9.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.9<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.7&#8211;8.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">61.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(11.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">33.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.8)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">5.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(4.2)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any substance use disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">76.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(7.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.5<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.3&#8211;13.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">36.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(12.2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">57.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(11.6)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">6.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(4.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\">V. Number of disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">96.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(2.6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.4<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(4.2&#8211;73.0)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">23.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(10.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">74.3</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">(2.2)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Exactly one disorder</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">22.0</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.1<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;65.6)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Exactly two disorders</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">9.9</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(6.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.1<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.3&#8211;65.2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Three or more disorders</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">64.4</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">(10.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">30.0<xref ref-type=\"table-fn\" rid=\"TFN4\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(7.6&#8211;118.7)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th colspan=\"5\" halign=\"left\">\\n</th>\n",
       "      <th>Temporal priority in onset</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Prevalence1</th>\n",
       "      <th colspan=\"2\" halign=\"left\">\\n</th>\n",
       "      <th>PG first</th>\n",
       "      <th>Other first</th>\n",
       "      <th>Same year</th>\n",
       "      <th>Unnamed: 7_level_1</th>\n",
       "      <th>Unnamed: 8_level_1</th>\n",
       "      <th>Unnamed: 9_level_1</th>\n",
       "      <th>Unnamed: 10_level_1</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>%</th>\n",
       "      <th>(se)</th>\n",
       "      <th>OR2</th>\n",
       "      <th>(95% CI)</th>\n",
       "      <th>%</th>\n",
       "      <th>(se)</th>\n",
       "      <th>%</th>\n",
       "      <th>(se)</th>\n",
       "      <th>%</th>\n",
       "      <th>(se)</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 2_level_3</th>\n",
       "      <th>Unnamed: 3_level_3</th>\n",
       "      <th>Unnamed: 4_level_3</th>\n",
       "      <th>Unnamed: 5_level_3</th>\n",
       "      <th>Unnamed: 6_level_3</th>\n",
       "      <th>Unnamed: 7_level_3</th>\n",
       "      <th>Unnamed: 8_level_3</th>\n",
       "      <th>Unnamed: 9_level_3</th>\n",
       "      <th>Unnamed: 10_level_3</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>I. Mood disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Major depressive disorder or dysthymia</td>\n",
       "      <td>38.6</td>\n",
       "      <td>(9.1)</td>\n",
       "      <td>2.5*</td>\n",
       "      <td>(1.1–5.7)</td>\n",
       "      <td>20.5</td>\n",
       "      <td>(10.6)</td>\n",
       "      <td>73.5</td>\n",
       "      <td>(11.1)</td>\n",
       "      <td>6.1</td>\n",
       "      <td>(5.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Bipolar disorder3</td>\n",
       "      <td>17.0</td>\n",
       "      <td>(7.1)</td>\n",
       "      <td>4.6*</td>\n",
       "      <td>(1.5–14.2)</td>\n",
       "      <td>29.2</td>\n",
       "      <td>(16.4)</td>\n",
       "      <td>46.3</td>\n",
       "      <td>(21.5)</td>\n",
       "      <td>24.5</td>\n",
       "      <td>(14.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Any mood disorder</td>\n",
       "      <td>55.6</td>\n",
       "      <td>(9.7)</td>\n",
       "      <td>3.7*</td>\n",
       "      <td>(1.5–9.0)</td>\n",
       "      <td>23.1</td>\n",
       "      <td>(9.0)</td>\n",
       "      <td>65.1</td>\n",
       "      <td>(11.7)</td>\n",
       "      <td>11.7</td>\n",
       "      <td>(6.9)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>II. Anxiety disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Panic disorder</td>\n",
       "      <td>21.9</td>\n",
       "      <td>(6.7)</td>\n",
       "      <td>4.9*</td>\n",
       "      <td>(2.2–10.8)</td>\n",
       "      <td>10.7</td>\n",
       "      <td>(9.5)</td>\n",
       "      <td>81.8</td>\n",
       "      <td>(16.1)</td>\n",
       "      <td>7.5</td>\n",
       "      <td>(6.6)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Generalized anxiety disorder</td>\n",
       "      <td>16.6</td>\n",
       "      <td>(7.0)</td>\n",
       "      <td>2.8*</td>\n",
       "      <td>(1.0–7.9)</td>\n",
       "      <td>9.3</td>\n",
       "      <td>(9.4)</td>\n",
       "      <td>79.8</td>\n",
       "      <td>(14.3)</td>\n",
       "      <td>10.9</td>\n",
       "      <td>(10.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Phobia</td>\n",
       "      <td>52.2</td>\n",
       "      <td>(8.8)</td>\n",
       "      <td>3.2*</td>\n",
       "      <td>(1.4–7.2)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>PTSD</td>\n",
       "      <td>14.8</td>\n",
       "      <td>(7.8)</td>\n",
       "      <td>2.3</td>\n",
       "      <td>(0.6–8.4)</td>\n",
       "      <td>49.5</td>\n",
       "      <td>(27.3)</td>\n",
       "      <td>50.5</td>\n",
       "      <td>(27.3)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Any anxiety disorder4</td>\n",
       "      <td>60.3</td>\n",
       "      <td>(9.1)</td>\n",
       "      <td>3.1*</td>\n",
       "      <td>(1.4–7.0)</td>\n",
       "      <td>13.4</td>\n",
       "      <td>(7.3)</td>\n",
       "      <td>82.1</td>\n",
       "      <td>(7.9)</td>\n",
       "      <td>4.5</td>\n",
       "      <td>(4.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>III. Impulse-control disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>ADHD</td>\n",
       "      <td>13.4</td>\n",
       "      <td>(8.1)</td>\n",
       "      <td>1.8</td>\n",
       "      <td>(0.4–7.3)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Oppositional-defiant disorder</td>\n",
       "      <td>15.4</td>\n",
       "      <td>(6.8)</td>\n",
       "      <td>1.9</td>\n",
       "      <td>(0.7–5.7)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Conduct disorder</td>\n",
       "      <td>24.9</td>\n",
       "      <td>(8.2)</td>\n",
       "      <td>3.1*</td>\n",
       "      <td>(1.2–7.8)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Intermittent explosive disorder</td>\n",
       "      <td>27.0</td>\n",
       "      <td>(9.0)</td>\n",
       "      <td>3.1*</td>\n",
       "      <td>(1.1–8.3)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Any impulse control disorder</td>\n",
       "      <td>42.3</td>\n",
       "      <td>(10.5)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>(0.9–5.3)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "      <td>100.0</td>\n",
       "      <td>(0.0)</td>\n",
       "      <td>0.0</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>IV. Substance use disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Alcohol or drug abuse</td>\n",
       "      <td>46.2</td>\n",
       "      <td>(10.7)</td>\n",
       "      <td>4.5*</td>\n",
       "      <td>(1.8–11.0)</td>\n",
       "      <td>18.7</td>\n",
       "      <td>(10.0)</td>\n",
       "      <td>70.9</td>\n",
       "      <td>(11.3)</td>\n",
       "      <td>10.4</td>\n",
       "      <td>(7.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Alcohol or drug dependence</td>\n",
       "      <td>31.8</td>\n",
       "      <td>(9.4)</td>\n",
       "      <td>5.8*</td>\n",
       "      <td>(2.4–14.4)</td>\n",
       "      <td>44.3</td>\n",
       "      <td>(16.7)</td>\n",
       "      <td>55.7</td>\n",
       "      <td>(16.7)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Nicotine dependence</td>\n",
       "      <td>63.0</td>\n",
       "      <td>(9.0)</td>\n",
       "      <td>3.9*</td>\n",
       "      <td>(1.7–8.5)</td>\n",
       "      <td>61.3</td>\n",
       "      <td>(11.5)</td>\n",
       "      <td>33.0</td>\n",
       "      <td>(10.8)</td>\n",
       "      <td>5.7</td>\n",
       "      <td>(4.2)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Any substance use disorder</td>\n",
       "      <td>76.3</td>\n",
       "      <td>(7.9)</td>\n",
       "      <td>5.5*</td>\n",
       "      <td>(2.3–13.5)</td>\n",
       "      <td>36.2</td>\n",
       "      <td>(12.2)</td>\n",
       "      <td>57.4</td>\n",
       "      <td>(11.6)</td>\n",
       "      <td>6.4</td>\n",
       "      <td>(4.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>V. Number of disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Any disorder</td>\n",
       "      <td>96.3</td>\n",
       "      <td>(2.6)</td>\n",
       "      <td>17.4*</td>\n",
       "      <td>(4.2–73.0)</td>\n",
       "      <td>23.5</td>\n",
       "      <td>(10.7)</td>\n",
       "      <td>74.3</td>\n",
       "      <td>(10.5)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>(2.2)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Exactly one disorder</td>\n",
       "      <td>22.0</td>\n",
       "      <td>(10.2)</td>\n",
       "      <td>10.1*</td>\n",
       "      <td>(1.5–65.6)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Exactly two disorders</td>\n",
       "      <td>9.9</td>\n",
       "      <td>(6.5)</td>\n",
       "      <td>9.1*</td>\n",
       "      <td>(1.3–65.2)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Three or more disorders</td>\n",
       "      <td>64.4</td>\n",
       "      <td>(10.4)</td>\n",
       "      <td>30.0*</td>\n",
       "      <td>(7.6–118.7)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          \\n                                                                                                           Temporal priority in onset Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0 Unnamed: 10_level_0\n",
       "                          \\n                                      Prevalence1   \\n                                            PG first                Other first          Same year Unnamed: 7_level_1 Unnamed: 8_level_1 Unnamed: 9_level_1 Unnamed: 10_level_1\n",
       "                          \\n                                                %               (se)                OR2           (95% CI)                          %               (se)                  %               (se)                  %                (se)\n",
       "                          \\n               \\n                \\n               Unnamed: 2_level_3 Unnamed: 3_level_3 Unnamed: 4_level_3         Unnamed: 5_level_3 Unnamed: 6_level_3 Unnamed: 7_level_3 Unnamed: 8_level_3 Unnamed: 9_level_3 Unnamed: 10_level_3\n",
       "0                        I. Mood disorders                                NaN                NaN                NaN                NaN                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "1   Major depressive disorder or dysthymia                               38.6              (9.1)               2.5*          (1.1–5.7)                       20.5             (10.6)               73.5             (11.1)                6.1               (5.8)\n",
       "2                        Bipolar disorder3                               17.0              (7.1)               4.6*         (1.5–14.2)                       29.2             (16.4)               46.3             (21.5)               24.5              (14.4)\n",
       "3                        Any mood disorder                               55.6              (9.7)               3.7*          (1.5–9.0)                       23.1              (9.0)               65.1             (11.7)               11.7               (6.9)\n",
       "4                    II. Anxiety disorders                                NaN                NaN                NaN                NaN                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "5                           Panic disorder                               21.9              (6.7)               4.9*         (2.2–10.8)                       10.7              (9.5)               81.8             (16.1)                7.5               (6.6)\n",
       "6             Generalized anxiety disorder                               16.6              (7.0)               2.8*          (1.0–7.9)                        9.3              (9.4)               79.8             (14.3)               10.9              (10.8)\n",
       "7                                   Phobia                               52.2              (8.8)               3.2*          (1.4–7.2)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "8                                     PTSD                               14.8              (7.8)                2.3          (0.6–8.4)                       49.5             (27.3)               50.5             (27.3)                0.0                 (-)\n",
       "9                    Any anxiety disorder4                               60.3              (9.1)               3.1*          (1.4–7.0)                       13.4              (7.3)               82.1              (7.9)                4.5               (4.3)\n",
       "10          III. Impulse-control disorders                                NaN                NaN                NaN                NaN                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "11                                    ADHD                               13.4              (8.1)                1.8          (0.4–7.3)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "12           Oppositional-defiant disorder                               15.4              (6.8)                1.9          (0.7–5.7)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "13                        Conduct disorder                               24.9              (8.2)               3.1*          (1.2–7.8)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "14         Intermittent explosive disorder                               27.0              (9.0)               3.1*          (1.1–8.3)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "15            Any impulse control disorder                               42.3             (10.5)                2.2          (0.9–5.3)                        0.0                (-)              100.0              (0.0)                0.0                 (-)\n",
       "16             IV. Substance use disorders                                NaN                NaN                NaN                NaN                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "17                   Alcohol or drug abuse                               46.2             (10.7)               4.5*         (1.8–11.0)                       18.7             (10.0)               70.9             (11.3)               10.4               (7.0)\n",
       "18              Alcohol or drug dependence                               31.8              (9.4)               5.8*         (2.4–14.4)                       44.3             (16.7)               55.7             (16.7)                NaN                 (-)\n",
       "19                     Nicotine dependence                               63.0              (9.0)               3.9*          (1.7–8.5)                       61.3             (11.5)               33.0             (10.8)                5.7               (4.2)\n",
       "20              Any substance use disorder                               76.3              (7.9)               5.5*         (2.3–13.5)                       36.2             (12.2)               57.4             (11.6)                6.4               (4.5)\n",
       "21                  V. Number of disorders                                NaN                NaN                NaN                NaN                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "22                            Any disorder                               96.3              (2.6)              17.4*         (4.2–73.0)                       23.5             (10.7)               74.3             (10.5)                2.2               (2.2)\n",
       "23                    Exactly one disorder                               22.0             (10.2)              10.1*         (1.5–65.6)                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "24                   Exactly two disorders                                9.9              (6.5)               9.1*         (1.3–65.2)                        NaN                NaN                NaN                NaN                NaN                 NaN\n",
       "25                 Three or more disorders                               64.4             (10.4)              30.0*        (7.6–118.7)                        NaN                NaN                NaN                NaN                NaN                 NaN"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Reciprocal time-lagged associations of temporally primary problem gambling predicting the subsequent first onset of other disorders and of temporally primary other disorders predicting the subsequent first onset of CIDI/DSM-IV PG1\n",
      "  footer_text: * Significant at the .05 level, two-sided test. 1 Models are bivariate for each disorder and controlling for age, sex, and race-ethnicity. 2 There are no onsets of ADHD, ODD, CD that came after the PG onsets\n",
      "    unique: PMC2293303/PMC2293303.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"below\" rules=\"none\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Problem gambling predicting others<xref ref-type=\"table-fn\" rid=\"TFN8\">1</xref></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Others predicting PG<xref ref-type=\"table-fn\" rid=\"TFN8\">1</xref></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">(95% CI)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">OR</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">(95% CI)</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"4\" rowspan=\"1\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">I. Mood disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Major depressive disorder or dysthymia</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.0&#8211;10.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.6<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.2&#8211;19.2)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Bipolar disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6.3<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.4&#8211;28.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.1<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.4&#8211;33.9)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any mood disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;5.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.0<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.0&#8211;13.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">II. Anxiety disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Panic disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.3</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;42.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.6<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(5.4&#8211;29.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Generalized anxiety disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.4&#8211;11.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.4<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.8&#8211;29.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Phobia</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.0<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.9<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(3.1&#8211;15.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;PTSD</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7.2<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.7&#8211;31.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;20.5)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any anxiety disorder<sup>3</sup></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.8<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;9.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.0<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.3&#8211;11.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">III. Impulse-control disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;ADHD</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)<xref ref-type=\"table-fn\" rid=\"TFN9\">2</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.5&#8211;9.1)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Oppositional-defiant disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)<xref ref-type=\"table-fn\" rid=\"TFN9\">2</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.3<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.1&#8211;10.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Conduct disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)<xref ref-type=\"table-fn\" rid=\"TFN9\">2</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.4<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.6&#8211;12.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Intermittent explosive disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;23.4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.7<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.0&#8211;15.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any impulse control disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.1</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.1&#8211;10.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.6<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.5&#8211;8.7)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">IV. Substance use disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Alcohol or drug abuse</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">3.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.0&#8211;12.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.4<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.8&#8211;16.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Alcohol or drug dependence</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">9.8<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(3.7&#8211;25.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.8<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.5&#8211;31.1)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Nicotine dependence</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5.3<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(2.4&#8211;11.7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.6&#8211;6.0)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any substance use disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7.7<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(3.6&#8211;16.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.5<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.6&#8211;12.7)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"5\" rowspan=\"1\">V. Number of disorders</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Any disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7.1<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(3.8&#8211;13.2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.4<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(1.9&#8211;15.1)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Exactly one disorder</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.2&#8211;5.9)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Exactly two disorders</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(0.7&#8211;30.6)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Three or more disorders</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">-</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(-)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11.3<xref ref-type=\"table-fn\" rid=\"TFN7\">*</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">(4.5&#8211;28.7)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Problem gambling predicting others1</th>\n",
       "      <th>Others predicting PG1</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)</th>\n",
       "      <th>OR</th>\n",
       "      <th>(95% CI)</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 2_level_2</th>\n",
       "      <th>Unnamed: 3_level_2</th>\n",
       "      <th>Unnamed: 4_level_2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>I. Mood disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Major depressive disorder or dysthymia</td>\n",
       "      <td>3.2</td>\n",
       "      <td>(1.0–10.5)</td>\n",
       "      <td>6.6*</td>\n",
       "      <td>(2.2–19.2)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Bipolar disorder</td>\n",
       "      <td>6.3*</td>\n",
       "      <td>(1.4–28.2)</td>\n",
       "      <td>9.1*</td>\n",
       "      <td>(2.4–33.9)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Any mood disorder</td>\n",
       "      <td>1.1</td>\n",
       "      <td>(0.2–5.7)</td>\n",
       "      <td>5.0*</td>\n",
       "      <td>(2.0–13.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>II. Anxiety disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Panic disorder</td>\n",
       "      <td>5.3</td>\n",
       "      <td>(0.7–42.2)</td>\n",
       "      <td>12.6*</td>\n",
       "      <td>(5.4–29.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Generalized anxiety disorder</td>\n",
       "      <td>2.0</td>\n",
       "      <td>(0.4–11.0)</td>\n",
       "      <td>7.4*</td>\n",
       "      <td>(1.8–29.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Phobia</td>\n",
       "      <td>0.0*</td>\n",
       "      <td>(-)</td>\n",
       "      <td>6.9*</td>\n",
       "      <td>(3.1–15.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>PTSD</td>\n",
       "      <td>7.2*</td>\n",
       "      <td>(1.7–31.4)</td>\n",
       "      <td>3.5</td>\n",
       "      <td>(0.6–20.5)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Any anxiety disorder3</td>\n",
       "      <td>3.8*</td>\n",
       "      <td>(1.5–9.5)</td>\n",
       "      <td>5.0*</td>\n",
       "      <td>(2.3–11.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>III. Impulse-control disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>ADHD</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)2</td>\n",
       "      <td>2.2</td>\n",
       "      <td>(0.5–9.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Oppositional-defiant disorder</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)2</td>\n",
       "      <td>3.3*</td>\n",
       "      <td>(1.1–10.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Conduct disorder</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)2</td>\n",
       "      <td>4.4*</td>\n",
       "      <td>(1.6–12.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Intermittent explosive disorder</td>\n",
       "      <td>3.7</td>\n",
       "      <td>(0.6–23.4)</td>\n",
       "      <td>5.7*</td>\n",
       "      <td>(2.0–15.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Any impulse control disorder</td>\n",
       "      <td>1.1</td>\n",
       "      <td>(0.1–10.9)</td>\n",
       "      <td>3.6*</td>\n",
       "      <td>(1.5–8.7)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>IV. Substance use disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Alcohol or drug abuse</td>\n",
       "      <td>3.5</td>\n",
       "      <td>(1.0–12.3)</td>\n",
       "      <td>5.4*</td>\n",
       "      <td>(1.8–16.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Alcohol or drug dependence</td>\n",
       "      <td>9.8*</td>\n",
       "      <td>(3.7–25.7)</td>\n",
       "      <td>8.8*</td>\n",
       "      <td>(2.5–31.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Nicotine dependence</td>\n",
       "      <td>5.3*</td>\n",
       "      <td>(2.4–11.7)</td>\n",
       "      <td>1.9</td>\n",
       "      <td>(0.6–6.0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Any substance use disorder</td>\n",
       "      <td>7.7*</td>\n",
       "      <td>(3.6–16.3)</td>\n",
       "      <td>4.5*</td>\n",
       "      <td>(1.6–12.7)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>V. Number of disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Any disorder</td>\n",
       "      <td>7.1*</td>\n",
       "      <td>(3.8–13.2)</td>\n",
       "      <td>5.4*</td>\n",
       "      <td>(1.9–15.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Exactly one disorder</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)</td>\n",
       "      <td>1.0</td>\n",
       "      <td>(0.2–5.9)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Exactly two disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)</td>\n",
       "      <td>4.5</td>\n",
       "      <td>(0.7–30.6)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Three or more disorders</td>\n",
       "      <td>NaN</td>\n",
       "      <td>(-)</td>\n",
       "      <td>11.3*</td>\n",
       "      <td>(4.5–28.7)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          \\n               Problem gambling predicting others1 Others predicting PG1 Unnamed: 3_level_0 Unnamed: 4_level_0\n",
       "                          \\n                                                OR              (95% CI)                 OR           (95% CI)\n",
       "                          \\n                \\n                \\n                  Unnamed: 2_level_2 Unnamed: 3_level_2 Unnamed: 4_level_2\n",
       "0                        I. Mood disorders                                 NaN                   NaN                NaN                NaN\n",
       "1   Major depressive disorder or dysthymia                                 3.2            (1.0–10.5)               6.6*         (2.2–19.2)\n",
       "2                         Bipolar disorder                                6.3*            (1.4–28.2)               9.1*         (2.4–33.9)\n",
       "3                        Any mood disorder                                 1.1             (0.2–5.7)               5.0*         (2.0–13.0)\n",
       "4                    II. Anxiety disorders                                 NaN                   NaN                NaN                NaN\n",
       "5                           Panic disorder                                 5.3            (0.7–42.2)              12.6*         (5.4–29.5)\n",
       "6             Generalized anxiety disorder                                 2.0            (0.4–11.0)               7.4*         (1.8–29.3)\n",
       "7                                   Phobia                                0.0*                   (-)               6.9*         (3.1–15.5)\n",
       "8                                     PTSD                                7.2*            (1.7–31.4)                3.5         (0.6–20.5)\n",
       "9                    Any anxiety disorder3                                3.8*             (1.5–9.5)               5.0*         (2.3–11.0)\n",
       "10          III. Impulse-control disorders                                 NaN                   NaN                NaN                NaN\n",
       "11                                    ADHD                                 NaN                  (-)2                2.2          (0.5–9.1)\n",
       "12           Oppositional-defiant disorder                                 NaN                  (-)2               3.3*         (1.1–10.3)\n",
       "13                        Conduct disorder                                 NaN                  (-)2               4.4*         (1.6–12.0)\n",
       "14         Intermittent explosive disorder                                 3.7            (0.6–23.4)               5.7*         (2.0–15.8)\n",
       "15            Any impulse control disorder                                 1.1            (0.1–10.9)               3.6*          (1.5–8.7)\n",
       "16             IV. Substance use disorders                                 NaN                   NaN                NaN                NaN\n",
       "17                   Alcohol or drug abuse                                 3.5            (1.0–12.3)               5.4*         (1.8–16.3)\n",
       "18              Alcohol or drug dependence                                9.8*            (3.7–25.7)               8.8*         (2.5–31.1)\n",
       "19                     Nicotine dependence                                5.3*            (2.4–11.7)                1.9          (0.6–6.0)\n",
       "20              Any substance use disorder                                7.7*            (3.6–16.3)               4.5*         (1.6–12.7)\n",
       "21                  V. Number of disorders                                 NaN                   NaN                NaN                NaN\n",
       "22                            Any disorder                                7.1*            (3.8–13.2)               5.4*         (1.9–15.1)\n",
       "23                    Exactly one disorder                                 NaN                   (-)                1.0          (0.2–5.9)\n",
       "24                   Exactly two disorders                                 NaN                   (-)                4.5         (0.7–30.6)\n",
       "25                 Three or more disorders                                 NaN                   (-)              11.3*         (4.5–28.7)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0023XXXXX/PMC2323355.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Selected genes identified by comparison of NE from patients with EAC (N with T) vs. without EAC (N without T). Threshold value set at 2.7; N with T: gene score in the group of patients with esophageal adenocarcinoma; N without T: gene score in the group of patients without esophageal adenocarcinoma. Gene identifiers and gene names are shown in the two leftmost columns.\n",
      "  footer_text: \n",
      "    unique: PMC2323355/PMC2323355.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Gene ID</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Gene Name</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">N with T</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">N without T</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">AB003476</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">gravin</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&#8722;0.7322</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4707</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_005319</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">H1 histone family, member 2 (H1F2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.5907</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.3797</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">XM_004416</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">H2A histone family, member L (H2AFL)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.5384</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.3461</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_003519</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">H2B histone family, member C (H2BFC)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.5062</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.3254</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_002273</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">keratin 8 (KRT8)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.3834</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.2465</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_015902</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">progestin induced protein (DD5)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&#8722;0.3112</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_003516</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">H2A histone family, member O (H2AFO)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.2322</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.1493</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">XM_009572</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">transglutaminase 3 (TGM3)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&#8722;0.2078</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1336</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_019111</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">major histocompatibility complex, class II, DR alpha (HLA-DRA)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.1695</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.109</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">AF107297</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">mitotic checkpoint protein kinase BUB1B (BUB1B)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">&#8722;0.0626</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.0402</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">NM_002083</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">glutathione peroxidase 2 (gastrointestinal) (GPX2)</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0.0614</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;0.0395</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene ID</th>\n",
       "      <th>Gene Name</th>\n",
       "      <th>N with T</th>\n",
       "      <th>N without T</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>AB003476</td>\n",
       "      <td>gravin</td>\n",
       "      <td>−0.7322</td>\n",
       "      <td>0.4707</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NM_005319</td>\n",
       "      <td>H1 histone family, member 2 (H1F2)</td>\n",
       "      <td>0.5907</td>\n",
       "      <td>−0.3797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>XM_004416</td>\n",
       "      <td>H2A histone family, member L (H2AFL)</td>\n",
       "      <td>0.5384</td>\n",
       "      <td>−0.3461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NM_003519</td>\n",
       "      <td>H2B histone family, member C (H2BFC)</td>\n",
       "      <td>0.5062</td>\n",
       "      <td>−0.3254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NM_002273</td>\n",
       "      <td>keratin 8 (KRT8)</td>\n",
       "      <td>0.3834</td>\n",
       "      <td>−0.2465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NM_015902</td>\n",
       "      <td>progestin induced protein (DD5)</td>\n",
       "      <td>−0.3112</td>\n",
       "      <td>0.2001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NM_003516</td>\n",
       "      <td>H2A histone family, member O (H2AFO)</td>\n",
       "      <td>0.2322</td>\n",
       "      <td>−0.1493</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>XM_009572</td>\n",
       "      <td>transglutaminase 3 (TGM3)</td>\n",
       "      <td>−0.2078</td>\n",
       "      <td>0.1336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>NM_019111</td>\n",
       "      <td>major histocompatibility complex, class II, DR alpha (HLA-DRA)</td>\n",
       "      <td>0.1695</td>\n",
       "      <td>−0.109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>AF107297</td>\n",
       "      <td>mitotic checkpoint protein kinase BUB1B (BUB1B)</td>\n",
       "      <td>−0.0626</td>\n",
       "      <td>0.0402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NM_002083</td>\n",
       "      <td>glutathione peroxidase 2 (gastrointestinal) (GPX2)</td>\n",
       "      <td>0.0614</td>\n",
       "      <td>−0.0395</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Gene ID                                                       Gene Name N with T N without T\n",
       "0    AB003476                                                          gravin  −0.7322      0.4707\n",
       "1   NM_005319                              H1 histone family, member 2 (H1F2)   0.5907     −0.3797\n",
       "2   XM_004416                            H2A histone family, member L (H2AFL)   0.5384     −0.3461\n",
       "3   NM_003519                            H2B histone family, member C (H2BFC)   0.5062     −0.3254\n",
       "4   NM_002273                                                keratin 8 (KRT8)   0.3834     −0.2465\n",
       "5   NM_015902                                 progestin induced protein (DD5)  −0.3112      0.2001\n",
       "6   NM_003516                            H2A histone family, member O (H2AFO)   0.2322     −0.1493\n",
       "7   XM_009572                                       transglutaminase 3 (TGM3)  −0.2078      0.1336\n",
       "8   NM_019111  major histocompatibility complex, class II, DR alpha (HLA-DRA)   0.1695      −0.109\n",
       "9    AF107297                 mitotic checkpoint protein kinase BUB1B (BUB1B)  −0.0626      0.0402\n",
       "10  NM_002083              glutathione peroxidase 2 (gastrointestinal) (GPX2)   0.0614     −0.0395"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0023XXXXX/PMC2365750.xml\n",
      "\n",
      "PMC0023XXXXX/PMC2396881.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2423936.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Summary of mouse lines, their genotypes and designations\n",
      "  footer_text: a Wild type mice were homozygous for Trsp and albumin Cre. b Trsp liver knockout mice lacked Trsp in their liver. c A34 transgenic mice carrying 2 copies of the T34→A34 mutant transgene. d G37 transgenic mice carrying 2 copies of the A37→G37 mutant transgene. e G37 transgenic mice carrying 16 copies of the A37→G37 mutant transgene. f Designations used in the text to denote each mouse line.\n",
      "    unique: PMC2423936/PMC2423936.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Mice</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Genotype</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Designation <xref ref-type=\"table-fn\" rid=\"TFN6\">f</xref></th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Wild type <xref ref-type=\"table-fn\" rid=\"TFN1\">a</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp</italic><sup>+/+</sup>-<italic>AlbCre</italic><sup>+/+</sup></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <italic>Trsp</italic>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp</italic> liver knockout <xref ref-type=\"table-fn\" rid=\"TFN2\">b</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp<sup>fl/fl</sup></italic>-<italic>AlbCre</italic><sup>+/+</sup></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <italic>&#916;Trsp</italic>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>A34</italic> transgenic (2 copies) <xref ref-type=\"table-fn\" rid=\"TFN3\">c</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp<sup>fl/fl</sup></italic>-<italic>AlbCre</italic><sup>+/+</sup>-<italic>A34<sup>t/t</sup></italic></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <italic>A34</italic>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>G37</italic> transgenic (2 copies) <xref ref-type=\"table-fn\" rid=\"TFN4\">d</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp<sup>fl/fl</sup></italic>-<italic>AlbCre</italic><sup>+/+</sup>-<italic>G37<sup>t/t</sup></italic></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <italic>G37L</italic>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>G37</italic> transgenic (16 copies) <xref ref-type=\"table-fn\" rid=\"TFN5\">e</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\"><italic>Trsp<sup>fl/fl</sup></italic>-<italic>AlbCre</italic><sup>+/+</sup>-<italic>G37<sup>t/t</sup></italic></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <italic>G37H</italic>\n",
       "              </td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Mice</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Designation f</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Wild type a</td>\n",
       "      <td>Trsp+/+-AlbCre+/+</td>\n",
       "      <td>Trsp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Trsp liver knockout b</td>\n",
       "      <td>Trspfl/fl-AlbCre+/+</td>\n",
       "      <td>ΔTrsp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A34 transgenic (2 copies) c</td>\n",
       "      <td>Trspfl/fl-AlbCre+/+-A34t/t</td>\n",
       "      <td>A34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>G37 transgenic (2 copies) d</td>\n",
       "      <td>Trspfl/fl-AlbCre+/+-G37t/t</td>\n",
       "      <td>G37L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>G37 transgenic (16 copies) e</td>\n",
       "      <td>Trspfl/fl-AlbCre+/+-G37t/t</td>\n",
       "      <td>G37H</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           Mice                    Genotype Designation f\n",
       "0                   Wild type a           Trsp+/+-AlbCre+/+          Trsp\n",
       "1         Trsp liver knockout b         Trspfl/fl-AlbCre+/+         ΔTrsp\n",
       "2   A34 transgenic (2 copies) c  Trspfl/fl-AlbCre+/+-A34t/t           A34\n",
       "3   G37 transgenic (2 copies) d  Trspfl/fl-AlbCre+/+-G37t/t          G37L\n",
       "4  G37 transgenic (16 copies) e  Trspfl/fl-AlbCre+/+-G37t/t          G37H"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Primers for assessing real-time PCRa\n",
      "  footer_text: a Primers designed for determining real-time PCR of each mRNA examined are shown in the table.\n",
      "    unique: PMC2423936/PMC2423936.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Gene</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Forward Sequence</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">Reverse Sequence</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Aox1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GAAGCTGGACAACGCTTACA-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCACATTTGATTGCCACTTC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Cd36</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GATTGTACCTGGGAGTTGGC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CATGAGAATGCCTCCAAACA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ces1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CAGAAGACAGCTGCATCCAT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TCCAATCAAGTCCAGGAACA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ces2</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-ATGTGAGGCTATGGATTCCC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TCCTCAGATGCCAACAACTC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Cyp2A5</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GAGATTGATCGGGTGATTGG-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CGAAACTTGGTGTCCTTGGT-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ddc</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CTGAATGGTGTGGAGTTTGC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TGAATCCTGAGTCCTGGTGA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Dmpk</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CGTGTTCGCCTATGAGATGT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-ACGAATGAGGTCCTGAGCTT-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ephx1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GGGTCAAAGCCATCAGCCA-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCTCCAGAAGGACACCACTTT-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gsta1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CGCAGACCAGAGCCATTCTC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TTGCCCAATCATTTCAGTCAGA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gsta2</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCCCTTTCCCTCTGCTGAAG-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TGCAGCCACACTAAAACTTGA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gsta4</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TTGAAATCGATGGGATGATG-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-ATCATCATCAGGTCCTGGGT-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gstm1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCAAACACACAGGTCAGTCC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CGTCACCCATGGTGTATCTC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gstm2</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCTATGACACTAGGTTACTGG-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CACTGGCTTCGGTCATAGTCA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gstm3</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TATGACACTGGGCTATTGGAAC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GGGCATCCCCCATGACA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Gstt3</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GGCAGAAGATGATGTTCCCT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TCAGCCACAGAAATATGGGA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Hmox1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GCCACCAAGGAGGTACACAT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GCTTGTTGCGCTCTATCTCC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Htatip2</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GGCCAGGAGTCCTACTGTGT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GTTCAGCATCGCTCTAACCA-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ikbkg</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCTGGTAGCCAAACAGGAAT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCTTCTTCTCCACCAGCTTC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Lgals1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-GCAACAACCTGTGCCTACAC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TGATGCACACCTCTGTGATG-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Srxn1</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCAGGGTGGCGACTACTACT-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CAAGTCTGGTGTGGATGCTC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ugdh</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-TGCTGTCCAATCCTGAGTTC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-ACCCAGTGCTCATACACAGC-3&#8242;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">\n",
       "                <bold>Ugt2b35</bold>\n",
       "              </td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-AATGACCTTCTCGGTCATCC-3&#8242;</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"middle\">5&#8242;-CCACCATGTGTGCAATGTTA-3&#8242;</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Forward Sequence</th>\n",
       "      <th>Reverse Sequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Aox1</td>\n",
       "      <td>5′-GAAGCTGGACAACGCTTACA-3′</td>\n",
       "      <td>5′-CCACATTTGATTGCCACTTC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Cd36</td>\n",
       "      <td>5′-GATTGTACCTGGGAGTTGGC-3′</td>\n",
       "      <td>5′-CATGAGAATGCCTCCAAACA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Ces1</td>\n",
       "      <td>5′-CAGAAGACAGCTGCATCCAT-3′</td>\n",
       "      <td>5′-TCCAATCAAGTCCAGGAACA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Ces2</td>\n",
       "      <td>5′-ATGTGAGGCTATGGATTCCC-3′</td>\n",
       "      <td>5′-TCCTCAGATGCCAACAACTC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Cyp2A5</td>\n",
       "      <td>5′-GAGATTGATCGGGTGATTGG-3′</td>\n",
       "      <td>5′-CGAAACTTGGTGTCCTTGGT-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Ddc</td>\n",
       "      <td>5′-CTGAATGGTGTGGAGTTTGC-3′</td>\n",
       "      <td>5′-TGAATCCTGAGTCCTGGTGA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Dmpk</td>\n",
       "      <td>5′-CGTGTTCGCCTATGAGATGT-3′</td>\n",
       "      <td>5′-ACGAATGAGGTCCTGAGCTT-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Ephx1</td>\n",
       "      <td>5′-GGGTCAAAGCCATCAGCCA-3′</td>\n",
       "      <td>5′-CCTCCAGAAGGACACCACTTT-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Gsta1</td>\n",
       "      <td>5′-CGCAGACCAGAGCCATTCTC-3′</td>\n",
       "      <td>5′-TTGCCCAATCATTTCAGTCAGA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Gsta2</td>\n",
       "      <td>5′-CCCCTTTCCCTCTGCTGAAG-3′</td>\n",
       "      <td>5′-TGCAGCCACACTAAAACTTGA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Gsta4</td>\n",
       "      <td>5′-TTGAAATCGATGGGATGATG-3′</td>\n",
       "      <td>5′-ATCATCATCAGGTCCTGGGT-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Gstm1</td>\n",
       "      <td>5′-CCAAACACACAGGTCAGTCC-3′</td>\n",
       "      <td>5′-CGTCACCCATGGTGTATCTC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Gstm2</td>\n",
       "      <td>5′-CCTATGACACTAGGTTACTGG-3′</td>\n",
       "      <td>5′-CACTGGCTTCGGTCATAGTCA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Gstm3</td>\n",
       "      <td>5′-TATGACACTGGGCTATTGGAAC-3′</td>\n",
       "      <td>5′-GGGCATCCCCCATGACA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Gstt3</td>\n",
       "      <td>5′-GGCAGAAGATGATGTTCCCT-3′</td>\n",
       "      <td>5′-TCAGCCACAGAAATATGGGA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Hmox1</td>\n",
       "      <td>5′-GCCACCAAGGAGGTACACAT-3′</td>\n",
       "      <td>5′-GCTTGTTGCGCTCTATCTCC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Htatip2</td>\n",
       "      <td>5′-GGCCAGGAGTCCTACTGTGT-3′</td>\n",
       "      <td>5′-GTTCAGCATCGCTCTAACCA-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Ikbkg</td>\n",
       "      <td>5′-CCTGGTAGCCAAACAGGAAT-3′</td>\n",
       "      <td>5′-CCTTCTTCTCCACCAGCTTC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Lgals1</td>\n",
       "      <td>5′-GCAACAACCTGTGCCTACAC-3′</td>\n",
       "      <td>5′-TGATGCACACCTCTGTGATG-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Srxn1</td>\n",
       "      <td>5′-CCAGGGTGGCGACTACTACT-3′</td>\n",
       "      <td>5′-CAAGTCTGGTGTGGATGCTC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Ugdh</td>\n",
       "      <td>5′-TGCTGTCCAATCCTGAGTTC-3′</td>\n",
       "      <td>5′-ACCCAGTGCTCATACACAGC-3′</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Ugt2b35</td>\n",
       "      <td>5′-AATGACCTTCTCGGTCATCC-3′</td>\n",
       "      <td>5′-CCACCATGTGTGCAATGTTA-3′</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Gene              Forward Sequence              Reverse Sequence\n",
       "0      Aox1    5′-GAAGCTGGACAACGCTTACA-3′    5′-CCACATTTGATTGCCACTTC-3′\n",
       "1      Cd36    5′-GATTGTACCTGGGAGTTGGC-3′    5′-CATGAGAATGCCTCCAAACA-3′\n",
       "2      Ces1    5′-CAGAAGACAGCTGCATCCAT-3′    5′-TCCAATCAAGTCCAGGAACA-3′\n",
       "3      Ces2    5′-ATGTGAGGCTATGGATTCCC-3′    5′-TCCTCAGATGCCAACAACTC-3′\n",
       "4    Cyp2A5    5′-GAGATTGATCGGGTGATTGG-3′    5′-CGAAACTTGGTGTCCTTGGT-3′\n",
       "5       Ddc    5′-CTGAATGGTGTGGAGTTTGC-3′    5′-TGAATCCTGAGTCCTGGTGA-3′\n",
       "6      Dmpk    5′-CGTGTTCGCCTATGAGATGT-3′    5′-ACGAATGAGGTCCTGAGCTT-3′\n",
       "7     Ephx1     5′-GGGTCAAAGCCATCAGCCA-3′   5′-CCTCCAGAAGGACACCACTTT-3′\n",
       "8     Gsta1    5′-CGCAGACCAGAGCCATTCTC-3′  5′-TTGCCCAATCATTTCAGTCAGA-3′\n",
       "9     Gsta2    5′-CCCCTTTCCCTCTGCTGAAG-3′   5′-TGCAGCCACACTAAAACTTGA-3′\n",
       "10    Gsta4    5′-TTGAAATCGATGGGATGATG-3′    5′-ATCATCATCAGGTCCTGGGT-3′\n",
       "11    Gstm1    5′-CCAAACACACAGGTCAGTCC-3′    5′-CGTCACCCATGGTGTATCTC-3′\n",
       "12    Gstm2   5′-CCTATGACACTAGGTTACTGG-3′   5′-CACTGGCTTCGGTCATAGTCA-3′\n",
       "13    Gstm3  5′-TATGACACTGGGCTATTGGAAC-3′       5′-GGGCATCCCCCATGACA-3′\n",
       "14    Gstt3    5′-GGCAGAAGATGATGTTCCCT-3′    5′-TCAGCCACAGAAATATGGGA-3′\n",
       "15    Hmox1    5′-GCCACCAAGGAGGTACACAT-3′    5′-GCTTGTTGCGCTCTATCTCC-3′\n",
       "16  Htatip2    5′-GGCCAGGAGTCCTACTGTGT-3′    5′-GTTCAGCATCGCTCTAACCA-3′\n",
       "17    Ikbkg    5′-CCTGGTAGCCAAACAGGAAT-3′    5′-CCTTCTTCTCCACCAGCTTC-3′\n",
       "18   Lgals1    5′-GCAACAACCTGTGCCTACAC-3′    5′-TGATGCACACCTCTGTGATG-3′\n",
       "19    Srxn1    5′-CCAGGGTGGCGACTACTACT-3′    5′-CAAGTCTGGTGTGGATGCTC-3′\n",
       "20     Ugdh    5′-TGCTGTCCAATCCTGAGTTC-3′    5′-ACCCAGTGCTCATACACAGC-3′\n",
       "21  Ugt2b35    5′-AATGACCTTCTCGGTCATCC-3′    5′-CCACCATGTGTGCAATGTTA-3′"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Transcripts in elevated levels within liver of knockout mice relative to wild type mice a\n",
      "  footer_text: a Genes elevated ≥ 2.0 fold in ΔTrsp mice compared to Trsp mice with a P-value ≤ 0.05 were assessed and shown in the table. The corresponding transcript levels were also analyzed in A34, G37L and G37H replacement mice relative to Trsp mice. Gene Unigene Accession Number, gene description, fold change with P value, and gene function(s) are shown. Four control and four experimental animals were used for comparing ΔTrsp, A34 and G37L with Trsp, while three control and three experimental animals were used for comparing G37H with Trsp. b Genes are grouped into classes according to function.\n",
      "    unique: PMC2423936/PMC2423936.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>&#916;Trsp</italic> vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>A34</italic> vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>G37</italic>L vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>G37</italic>H vs <italic>Trsp</italic></th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">UniGene</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Description</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Function/Gene Ontology</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Defense stress and detoxification</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.28191</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">carboxylesterase 2 (Ces2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.07</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.53</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ester hydrolase activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.389848</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Cytochrome P450, family 2, subfamily a, polypeptide 5 (Cyp2a5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>8.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>5.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.51</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Degradation of environmental toxins and mutagens</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.32550</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">DnaJ (Hsp40) homolog, subfamily C, member 12 (Dnajc12)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.51</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Heat shock protein binding, protein folding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.218639</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">sulfiredoxin 1 homolog (S. cerevisiae) (Srxn1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.96</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Antioxidant activity, response to oxidative stress</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><bold>Metabolism</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.30085</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">aldo-keto reductase family 1, member A4 (aldehyde reductase) (Akr1a4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.18</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glycerolipid metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.26787</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">aldehyde oxidase 1 (Aox1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>6.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.73</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.24021</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Biliverdin reductase B (Blvrb)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Porphyrin and chlorophyll metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.22720</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">carboxylesterase 1 (Ces1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>8.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Alkaloid biosynthesis</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.12906</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">dopa decarboxylase (Ddc)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Amino acid and derivative metabolic process</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.22758</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">emopamil binding protein-like (Ebpl)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Sterol metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.10211</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">ectonucleoside triphosphate diphosphohydrolase 5, transcript variant 1 (Entpd5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Purine metabolism; pyrimidine metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.9075</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">epoxide hydrolase 1, microsomal (Ephx1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.252391</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">glycerol-3-phosphate dehydrogenase 1 (soluble) (Gpd1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.08</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Carbohydrate metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.283573</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione reductase 1 (Gsr)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.197422</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, alpha 1 (Ya) (Gsta1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>23.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>3.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.422778</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, alpha 2 (Yc2) (Gsta2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.001</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.45</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.55</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.06</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.2662</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, alpha 4 (Gsta4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>8.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.51</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.37199</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, mu 1 (Gstm1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.10</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.37199</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, mu 2 (Gstm2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.37199</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, mu 3 (Gstm3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>12.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.16</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.21</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.5731</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione S-transferase, theta 3 (Gstt3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.07</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.276389</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">heme oxygenase (decycling) 1 (Hmox1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>17.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.08</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Porphyrin and chlorophyll metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.299381</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">hypoxanthine guanine phosphoribosyl transferase 1 (Hprt1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Purine metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.218286</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">microsomal glutathione S-transferase 3 (Mgst3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutathione metabolism; xenobiotic metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.344831</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">UDP-glucose dehydrogenase (Ugdh)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>6.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.002</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.56</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.16</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Nucleotide sugar metabolism; starch and sucrose metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.312095</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">UDP glucuronosyltransferase 2 family, polypeptide B35 (Ugt2b35)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Sugar metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Intracellular communication/signal transduction</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.18628</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">CD36 antigen (Cd36)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>3.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>3.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Adipocytokine signaling pathway; PPAR signaling pathway</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.6529</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dystrophia myotonica-protein kinase (Dmpk)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.06</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Protein amino acid phosphorylation; regulation of small GTPase mediated signal transduction</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.12967</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">inhibitor of kappaB kinase gamma, transcript variant 2 (Ikbkg)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Activation of NF-kappaB-inducing kinase</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.294007</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">PREDICTED: proprotein convertase subtilisin/kexin type 6, transcript variant 4 (Pcsk6)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Determination of left/right symmetry; transmembrane receptor protein tyrosine kinase signaling pathway</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.308180</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">protein tyrosine phosphatase-like A domain containing 1 (Ptplad1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.64</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.41</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">I-kappaB kinase/NF-kappaB cascade; JNK cascade; Rho protein signal transduction</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Cell cycle/Growth and differentiation</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.20801</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">HIV-1 tat interactive protein 2, homolog (human) (Htatip2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.22</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.40</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Regulation of angiogenesis and apoptosis; cell differentiation</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.43831</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">lectin, galactose binding, soluble 1 (Lgals1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>6.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Myoblast differentiation; sugar binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.62876</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">ZW10 interactor (Zwint)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">cell cycle; cell division</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.290692</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">transketolase (Tkt)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.40</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Metal ion binding; regulation of growth</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Cellular transport and transport mechanism</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.103777</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Rhesus blood group-associated B glycoprotein (Rhbg)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ammonium transporter activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.222536</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">solute carrier family 25 (mitochondrial carrier, peroxisomal membrane protein), member 17 (Slc25a17)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>5.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">mitochondrial transport; transporter activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.281800</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Solute carrier family 35 (UDP-glucuronic acid/UDP-N-acetylgalactosamine dual transporter), member D1 (Slc35d1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.53</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.37</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Nucleotide-sugar transporter</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.218875</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">target of myb1-like 2 (chicken), transcript variant 1 (Tom1l2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.06</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Golgi vesicle-mediated transport; intracellular protein transport</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Transcription/Translation/Protein modification</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.298030</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">synovial sarcoma, X member B, breakpoint 1 (Ssxb1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Regulation of transcription</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.34483</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">leucine rich repeat containing 35 (Lrrc35)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.25</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.12</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Metal ion binding; protein modification</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Miscellaneous (mixed functions)</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.234247</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dimethylarginine dimethylaminohydrolase 1 (Ddah1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.08</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.28</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">nitric oxide biosynthesis; protein amino acid nitrosylation</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.26834</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">EF hand domain family A1 (Efha1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Calcium ion binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.182912</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">growth hormone inducible transmembrane protein (Ghitm)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.07</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.10</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Interacts with FtsH</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.41665</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Glutamate receptor, ionotropic, N-methyl D-asparate-associated protein 1 (glutamate binding) (Grina)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.69</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.54</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Receptor activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.228797</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">major vault protein (Mvp)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.57</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.62</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Calcium ion binding; ribonucleoprotein complex</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.252080</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">PREDICTED: phosphogluconate dehydrogenase, transcript variant 1 (Pgd)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">pentose-phosphate shunt, oxidative branch</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.293463</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">pirin (Pir)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.18</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.46</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">metal ion binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.173058</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">secreted phosphoprotein 2 (Spp2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.74</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Bone remodeling</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Unknown</bold> <xref ref-type=\"table-fn\" rid=\"TFN9\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.22109</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 2610204L23 gene (2610204L23Rik)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.34</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.100125</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">SH3 domain binding glutamic acid-rich protein like 2 (Sh3bgrl2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.85</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.55</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.62</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.102407</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">testis expressed gene 2 (Tex2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.39</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.289795</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">ubiquitin-associated protein 1 (Ubap1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.44</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.257762</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">PREDICTED: WD repeat domain 43, transcript variant 9 (Wdr43)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th colspan=\"2\" halign=\"left\">\\n</th>\n",
       "      <th>ΔTrsp vs Trsp</th>\n",
       "      <th>A34 vs Trsp</th>\n",
       "      <th>G37L vs Trsp</th>\n",
       "      <th>G37H vs Trsp</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>UniGene</th>\n",
       "      <th>Description</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Function/Gene Ontology</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Defense stress and detoxification b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Mm.28191</td>\n",
       "      <td>carboxylesterase 2 (Ces2)</td>\n",
       "      <td>3.8</td>\n",
       "      <td>0.020</td>\n",
       "      <td>-0.70</td>\n",
       "      <td>0.07</td>\n",
       "      <td>-0.5</td>\n",
       "      <td>0.17</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.53</td>\n",
       "      <td>Ester hydrolase activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Mm.389848</td>\n",
       "      <td>Cytochrome P450, family 2, subfamily a, polypeptide 5 (Cyp2a5)</td>\n",
       "      <td>8.2</td>\n",
       "      <td>0.010</td>\n",
       "      <td>3.00</td>\n",
       "      <td>0.30</td>\n",
       "      <td>-5.6</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.51</td>\n",
       "      <td>Degradation of environmental toxins and mutagens</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Mm.32550</td>\n",
       "      <td>DnaJ (Hsp40) homolog, subfamily C, member 12 (Dnajc12)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.40</td>\n",
       "      <td>0.51</td>\n",
       "      <td>5.5</td>\n",
       "      <td>0.11</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Heat shock protein binding, protein folding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Mm.218639</td>\n",
       "      <td>sulfiredoxin 1 homolog (S. cerevisiae) (Srxn1)</td>\n",
       "      <td>3.3</td>\n",
       "      <td>0.030</td>\n",
       "      <td>-0.20</td>\n",
       "      <td>0.19</td>\n",
       "      <td>0.9</td>\n",
       "      <td>0.96</td>\n",
       "      <td>-1.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Antioxidant activity, response to oxidative stress</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Metabolism b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Mm.30085</td>\n",
       "      <td>aldo-keto reductase family 1, member A4 (aldehyde reductase) (Akr1a4)</td>\n",
       "      <td>2.4</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.30</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.18</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Glycerolipid metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Mm.26787</td>\n",
       "      <td>aldehyde oxidase 1 (Aox1)</td>\n",
       "      <td>6.5</td>\n",
       "      <td>0.010</td>\n",
       "      <td>0.70</td>\n",
       "      <td>0.73</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Mm.24021</td>\n",
       "      <td>Biliverdin reductase B (Blvrb)</td>\n",
       "      <td>3.4</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-1.50</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-2.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Porphyrin and chlorophyll metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Mm.22720</td>\n",
       "      <td>carboxylesterase 1 (Ces1)</td>\n",
       "      <td>8.0</td>\n",
       "      <td>0.010</td>\n",
       "      <td>2.80</td>\n",
       "      <td>0.09</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Alkaloid biosynthesis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Mm.12906</td>\n",
       "      <td>dopa decarboxylase (Ddc)</td>\n",
       "      <td>5.1</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.50</td>\n",
       "      <td>0.15</td>\n",
       "      <td>-2.0</td>\n",
       "      <td>0.15</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Amino acid and derivative metabolic process</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Mm.22758</td>\n",
       "      <td>emopamil binding protein-like (Ebpl)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.48</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.09</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Sterol metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Mm.10211</td>\n",
       "      <td>ectonucleoside triphosphate diphosphohydrolase 5, transcript variant 1 (Entpd5)</td>\n",
       "      <td>5.4</td>\n",
       "      <td>0.020</td>\n",
       "      <td>-0.70</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Purine metabolism; pyrimidine metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Mm.9075</td>\n",
       "      <td>epoxide hydrolase 1, microsomal (Ephx1)</td>\n",
       "      <td>3.3</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.50</td>\n",
       "      <td>0.02</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.3</td>\n",
       "      <td>0.14</td>\n",
       "      <td>Xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Mm.252391</td>\n",
       "      <td>glycerol-3-phosphate dehydrogenase 1 (soluble) (Gpd1)</td>\n",
       "      <td>3.1</td>\n",
       "      <td>0.020</td>\n",
       "      <td>-0.10</td>\n",
       "      <td>0.24</td>\n",
       "      <td>1.4</td>\n",
       "      <td>0.02</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.08</td>\n",
       "      <td>Carbohydrate metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Mm.283573</td>\n",
       "      <td>Glutathione reductase 1 (Gsr)</td>\n",
       "      <td>3.2</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.42</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.02</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Glutathione metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Mm.197422</td>\n",
       "      <td>Glutathione S-transferase, alpha 1 (Ya) (Gsta1)</td>\n",
       "      <td>23.4</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.20</td>\n",
       "      <td>0.33</td>\n",
       "      <td>-3.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.3</td>\n",
       "      <td>0.14</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Mm.422778</td>\n",
       "      <td>Glutathione S-transferase, alpha 2 (Yc2) (Gsta2)</td>\n",
       "      <td>2.9</td>\n",
       "      <td>0.001</td>\n",
       "      <td>0.45</td>\n",
       "      <td>0.55</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.06</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Mm.2662</td>\n",
       "      <td>Glutathione S-transferase, alpha 4 (Gsta4)</td>\n",
       "      <td>8.6</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.51</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.8</td>\n",
       "      <td>0.05</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Mm.37199</td>\n",
       "      <td>Glutathione S-transferase, mu 1 (Gstm1)</td>\n",
       "      <td>3.2</td>\n",
       "      <td>0.010</td>\n",
       "      <td>1.40</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.20</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.10</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Mm.37199</td>\n",
       "      <td>Glutathione S-transferase, mu 2 (Gstm2)</td>\n",
       "      <td>4.8</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-0.60</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Mm.37199</td>\n",
       "      <td>Glutathione S-transferase, mu 3 (Gstm3)</td>\n",
       "      <td>12.0</td>\n",
       "      <td>0.010</td>\n",
       "      <td>0.40</td>\n",
       "      <td>0.54</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.16</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.21</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Mm.5731</td>\n",
       "      <td>Glutathione S-transferase, theta 3 (Gstt3)</td>\n",
       "      <td>5.1</td>\n",
       "      <td>0.040</td>\n",
       "      <td>-1.80</td>\n",
       "      <td>0.07</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.15</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Glutathione metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Mm.276389</td>\n",
       "      <td>heme oxygenase (decycling) 1 (Hmox1)</td>\n",
       "      <td>17.3</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.60</td>\n",
       "      <td>0.08</td>\n",
       "      <td>2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.02</td>\n",
       "      <td>Porphyrin and chlorophyll metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Mm.299381</td>\n",
       "      <td>hypoxanthine guanine phosphoribosyl transferase 1 (Hprt1)</td>\n",
       "      <td>2.8</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.54</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.12</td>\n",
       "      <td>Purine metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Mm.218286</td>\n",
       "      <td>microsomal glutathione S-transferase 3 (Mgst3)</td>\n",
       "      <td>5.8</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.50</td>\n",
       "      <td>0.05</td>\n",
       "      <td>-2.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.04</td>\n",
       "      <td>Glutathione metabolism; xenobiotic metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>Mm.344831</td>\n",
       "      <td>UDP-glucose dehydrogenase (Ugdh)</td>\n",
       "      <td>6.4</td>\n",
       "      <td>0.002</td>\n",
       "      <td>0.50</td>\n",
       "      <td>0.56</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.16</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Nucleotide sugar metabolism; starch and sucrose metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Mm.312095</td>\n",
       "      <td>UDP glucuronosyltransferase 2 family, polypeptide B35 (Ugt2b35)</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.010</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.61</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Sugar metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>Intracellular communication/signal transduction b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>Mm.18628</td>\n",
       "      <td>CD36 antigen (Cd36)</td>\n",
       "      <td>5.6</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-2.60</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-3.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-3.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Adipocytokine signaling pathway; PPAR signaling pathway</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>Mm.6529</td>\n",
       "      <td>Dystrophia myotonica-protein kinase (Dmpk)</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.97</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.06</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Protein amino acid phosphorylation; regulation of small GTPase mediated signal transduction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Mm.12967</td>\n",
       "      <td>inhibitor of kappaB kinase gamma, transcript variant 2 (Ikbkg)</td>\n",
       "      <td>5.1</td>\n",
       "      <td>0.020</td>\n",
       "      <td>1.20</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-1.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Activation of NF-kappaB-inducing kinase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>Mm.294007</td>\n",
       "      <td>PREDICTED: proprotein convertase subtilisin/kexin type 6, transcript variant 4 (Pcsk6)</td>\n",
       "      <td>3.7</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.90</td>\n",
       "      <td>0.94</td>\n",
       "      <td>-0.7</td>\n",
       "      <td>0.03</td>\n",
       "      <td>1.8</td>\n",
       "      <td>0.04</td>\n",
       "      <td>Determination of left/right symmetry; transmembrane receptor protein tyrosine kinase signaling pathway</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>Mm.308180</td>\n",
       "      <td>protein tyrosine phosphatase-like A domain containing 1 (Ptplad1)</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.70</td>\n",
       "      <td>0.64</td>\n",
       "      <td>1.6</td>\n",
       "      <td>0.09</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.41</td>\n",
       "      <td>I-kappaB kinase/NF-kappaB cascade; JNK cascade; Rho protein signal transduction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>Cell cycle/Growth and differentiation b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>Mm.20801</td>\n",
       "      <td>HIV-1 tat interactive protein 2, homolog (human) (Htatip2)</td>\n",
       "      <td>4.2</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-0.10</td>\n",
       "      <td>0.22</td>\n",
       "      <td>-0.7</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.40</td>\n",
       "      <td>Regulation of angiogenesis and apoptosis; cell differentiation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>Mm.43831</td>\n",
       "      <td>lectin, galactose binding, soluble 1 (Lgals1)</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-1.40</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Myoblast differentiation; sugar binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>Mm.62876</td>\n",
       "      <td>ZW10 interactor (Zwint)</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-0.50</td>\n",
       "      <td>0.04</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.43</td>\n",
       "      <td>cell cycle; cell division</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>Mm.290692</td>\n",
       "      <td>transketolase (Tkt)</td>\n",
       "      <td>2.6</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.50</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.40</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Metal ion binding; regulation of growth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>Cellular transport and transport mechanism b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>Mm.103777</td>\n",
       "      <td>Rhesus blood group-associated B glycoprotein (Rhbg)</td>\n",
       "      <td>4.8</td>\n",
       "      <td>0.020</td>\n",
       "      <td>0.10</td>\n",
       "      <td>0.24</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Ammonium transporter activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>Mm.222536</td>\n",
       "      <td>solute carrier family 25 (mitochondrial carrier, peroxisomal membrane protein), member 17 (Slc25a17)</td>\n",
       "      <td>2.4</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.90</td>\n",
       "      <td>0.02</td>\n",
       "      <td>-1.9</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-5.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>mitochondrial transport; transporter activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>Mm.281800</td>\n",
       "      <td>Solute carrier family 35 (UDP-glucuronic acid/UDP-N-acetylgalactosamine dual transporter), member D1 (Slc35d1)</td>\n",
       "      <td>2.8</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.53</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.37</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Nucleotide-sugar transporter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>Mm.218875</td>\n",
       "      <td>target of myb1-like 2 (chicken), transcript variant 1 (Tom1l2)</td>\n",
       "      <td>2.1</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-2.10</td>\n",
       "      <td>0.06</td>\n",
       "      <td>5.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.5</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Golgi vesicle-mediated transport; intracellular protein transport</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>Transcription/Translation/Protein modification b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>Mm.298030</td>\n",
       "      <td>synovial sarcoma, X member B, breakpoint 1 (Ssxb1)</td>\n",
       "      <td>5.1</td>\n",
       "      <td>0.020</td>\n",
       "      <td>-0.20</td>\n",
       "      <td>0.20</td>\n",
       "      <td>1.7</td>\n",
       "      <td>0.05</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Regulation of transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>Mm.34483</td>\n",
       "      <td>leucine rich repeat containing 35 (Lrrc35)</td>\n",
       "      <td>3.2</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.40</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.25</td>\n",
       "      <td>-0.4</td>\n",
       "      <td>0.12</td>\n",
       "      <td>Metal ion binding; protein modification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>Miscellaneous (mixed functions) b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>Mm.234247</td>\n",
       "      <td>Dimethylarginine dimethylaminohydrolase 1 (Ddah1)</td>\n",
       "      <td>4.5</td>\n",
       "      <td>0.010</td>\n",
       "      <td>-0.60</td>\n",
       "      <td>0.08</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.33</td>\n",
       "      <td>1.4</td>\n",
       "      <td>0.28</td>\n",
       "      <td>nitric oxide biosynthesis; protein amino acid nitrosylation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>Mm.26834</td>\n",
       "      <td>EF hand domain family A1 (Efha1)</td>\n",
       "      <td>2.4</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-1.30</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.4</td>\n",
       "      <td>0.15</td>\n",
       "      <td>-1.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Calcium ion binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>Mm.182912</td>\n",
       "      <td>growth hormone inducible transmembrane protein (Ghitm)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-0.80</td>\n",
       "      <td>0.07</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.09</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.10</td>\n",
       "      <td>Interacts with FtsH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>Mm.41665</td>\n",
       "      <td>Glutamate receptor, ionotropic, N-methyl D-asparate-associated protein 1 (glutamate binding) (Grina)</td>\n",
       "      <td>2.3</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.10</td>\n",
       "      <td>0.29</td>\n",
       "      <td>0.7</td>\n",
       "      <td>0.69</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.54</td>\n",
       "      <td>Receptor activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>Mm.228797</td>\n",
       "      <td>major vault protein (Mvp)</td>\n",
       "      <td>3.2</td>\n",
       "      <td>0.010</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.57</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.62</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Calcium ion binding; ribonucleoprotein complex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>Mm.252080</td>\n",
       "      <td>PREDICTED: phosphogluconate dehydrogenase, transcript variant 1 (Pgd)</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.000</td>\n",
       "      <td>-2.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.45</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>pentose-phosphate shunt, oxidative branch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>Mm.293463</td>\n",
       "      <td>pirin (Pir)</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.050</td>\n",
       "      <td>1.20</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.18</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.46</td>\n",
       "      <td>metal ion binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>Mm.173058</td>\n",
       "      <td>secreted phosphoprotein 2 (Spp2)</td>\n",
       "      <td>2.3</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.30</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.74</td>\n",
       "      <td>1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Bone remodeling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>Unknown b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>Mm.22109</td>\n",
       "      <td>RIKEN cDNA 2610204L23 gene (2610204L23Rik)</td>\n",
       "      <td>2.3</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.34</td>\n",
       "      <td>1.6</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.14</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>Mm.100125</td>\n",
       "      <td>SH3 domain binding glutamic acid-rich protein like 2 (Sh3bgrl2)</td>\n",
       "      <td>3.5</td>\n",
       "      <td>0.030</td>\n",
       "      <td>0.80</td>\n",
       "      <td>0.85</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.55</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.62</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>Mm.102407</td>\n",
       "      <td>testis expressed gene 2 (Tex2)</td>\n",
       "      <td>2.4</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.70</td>\n",
       "      <td>0.61</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.39</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>Mm.289795</td>\n",
       "      <td>ubiquitin-associated protein 1 (Ubap1)</td>\n",
       "      <td>2.6</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.50</td>\n",
       "      <td>0.44</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.43</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>Mm.257762</td>\n",
       "      <td>PREDICTED: WD repeat domain 43, transcript variant 9 (Wdr43)</td>\n",
       "      <td>4.1</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.43</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.03</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.04</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     \\n                                                                                                                               ΔTrsp vs Trsp A34 vs Trsp G37L vs Trsp G37H vs Trsp \\n             Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0                                                                                     Unnamed: 10_level_0\n",
       "                                              UniGene                                                                                                     Description         Folds     P value        Folds      P value          Folds            P value              Folds            P value                                                                                  Function/Gene Ontology\n",
       "0                 Defense stress and detoxification b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "1                                            Mm.28191                                                                                       carboxylesterase 2 (Ces2)           3.8       0.020        -0.70         0.07           -0.5               0.17                0.5               0.53                                                                                Ester hydrolase activity\n",
       "2                                           Mm.389848                                                  Cytochrome P450, family 2, subfamily a, polypeptide 5 (Cyp2a5)           8.2       0.010         3.00         0.30           -5.6               0.03                0.2               0.51                                                        Degradation of environmental toxins and mutagens\n",
       "3                                            Mm.32550                                                          DnaJ (Hsp40) homolog, subfamily C, member 12 (Dnajc12)           2.2       0.000         0.40         0.51            5.5               0.11               -1.5               0.00                                                             Heat shock protein binding, protein folding\n",
       "4                                           Mm.218639                                                                  sulfiredoxin 1 homolog (S. cerevisiae) (Srxn1)           3.3       0.030        -0.20         0.19            0.9               0.96               -1.6               0.00                                                      Antioxidant activity, response to oxidative stress\n",
       "5                                                 NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "6                                        Metabolism b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "7                                            Mm.30085                                           aldo-keto reductase family 1, member A4 (aldehyde reductase) (Akr1a4)           2.4       0.000         1.30         0.03           -0.1               0.18                1.1               0.01                                                                                 Glycerolipid metabolism\n",
       "8                                            Mm.26787                                                                                       aldehyde oxidase 1 (Aox1)           6.5       0.010         0.70         0.73           -1.8               0.00               -1.1               0.00                                                                                   Xenobiotic metabolism\n",
       "9                                            Mm.24021                                                                                  Biliverdin reductase B (Blvrb)           3.4       0.010        -1.50         0.00           -2.1               0.00               -1.7               0.00                                                                    Porphyrin and chlorophyll metabolism\n",
       "10                                           Mm.22720                                                                                       carboxylesterase 1 (Ces1)           8.0       0.010         2.80         0.09           -2.4               0.00                3.0               0.00                                                                                   Alkaloid biosynthesis\n",
       "11                                           Mm.12906                                                                                        dopa decarboxylase (Ddc)           5.1       0.000        -0.50         0.15           -2.0               0.15               -1.1               0.00                                                             Amino acid and derivative metabolic process\n",
       "12                                           Mm.22758                                                                            emopamil binding protein-like (Ebpl)           2.2       0.000         0.60         0.48            1.2               0.09                1.3               0.00                                                                                       Sterol metabolism\n",
       "13                                           Mm.10211                                 ectonucleoside triphosphate diphosphohydrolase 5, transcript variant 1 (Entpd5)           5.4       0.020        -0.70         0.03           -2.3               0.00               -1.6               0.00                                                                Purine metabolism; pyrimidine metabolism\n",
       "14                                            Mm.9075                                                                         epoxide hydrolase 1, microsomal (Ephx1)           3.3       0.000         1.50         0.02           -1.8               0.00               -0.3               0.14                                                                                   Xenobiotic metabolism\n",
       "15                                          Mm.252391                                                           glycerol-3-phosphate dehydrogenase 1 (soluble) (Gpd1)           3.1       0.020        -0.10         0.24            1.4               0.02                1.2               0.08                                                                                 Carbohydrate metabolism\n",
       "16                                          Mm.283573                                                                                   Glutathione reductase 1 (Gsr)           3.2       0.000         0.20         0.42            1.2               0.02               -1.1               0.00                                                                                  Glutathione metabolism\n",
       "17                                          Mm.197422                                                                 Glutathione S-transferase, alpha 1 (Ya) (Gsta1)          23.4       0.000        -0.20         0.33           -3.6               0.00               -0.3               0.14                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "18                                          Mm.422778                                                                Glutathione S-transferase, alpha 2 (Yc2) (Gsta2)           2.9       0.001         0.45         0.55           -2.3               0.06                1.2               0.00                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "19                                            Mm.2662                                                                      Glutathione S-transferase, alpha 4 (Gsta4)           8.6       0.000         0.60         0.51           -1.5               0.00                1.8               0.05                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "20                                           Mm.37199                                                                         Glutathione S-transferase, mu 1 (Gstm1)           3.2       0.010         1.40         0.00            0.0               0.20                1.5               0.10                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "21                                           Mm.37199                                                                         Glutathione S-transferase, mu 2 (Gstm2)           4.8       0.010        -0.60         0.03           -1.2               0.00               -1.8               0.00                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "22                                           Mm.37199                                                                         Glutathione S-transferase, mu 3 (Gstm3)          12.0       0.010         0.40         0.54            0.0               0.16                1.2               0.21                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "23                                            Mm.5731                                                                      Glutathione S-transferase, theta 3 (Gstt3)           5.1       0.040        -1.80         0.07           -2.4               0.15                1.3               0.01                                                                                  Glutathione metabolism\n",
       "24                                          Mm.276389                                                                            heme oxygenase (decycling) 1 (Hmox1)          17.3       0.000         1.60         0.08            2.4               0.00                2.2               0.02                                             Porphyrin and chlorophyll metabolism; xenobiotic metabolism\n",
       "25                                          Mm.299381                                                       hypoxanthine guanine phosphoribosyl transferase 1 (Hprt1)           2.8       0.000         0.60         0.54            1.3               0.00                1.1               0.12                                                                                       Purine metabolism\n",
       "26                                          Mm.218286                                                                  microsomal glutathione S-transferase 3 (Mgst3)           5.8       0.000        -0.50         0.05           -2.7               0.00                1.1               0.04                                                           Glutathione metabolism; xenobiotic metabolism\n",
       "27                                          Mm.344831                                                                                UDP-glucose dehydrogenase (Ugdh)           6.4       0.002         0.50         0.56           -0.1               0.16               -1.2               0.00                                              Nucleotide sugar metabolism; starch and sucrose metabolism\n",
       "28                                          Mm.312095                                                 UDP glucuronosyltransferase 2 family, polypeptide B35 (Ugt2b35)           5.0       0.010         0.60         0.61           -2.4               0.00               -1.4               0.00                                                                                        Sugar metabolism\n",
       "29                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "30  Intracellular communication/signal transduction b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "31                                           Mm.18628                                                                                             CD36 antigen (Cd36)           5.6       0.000        -2.60         0.00           -3.2               0.00               -3.8               0.00                                                 Adipocytokine signaling pathway; PPAR signaling pathway\n",
       "32                                            Mm.6529                                                                      Dystrophia myotonica-protein kinase (Dmpk)           5.0       0.000         1.00         0.97            1.3               0.06                1.5               0.00             Protein amino acid phosphorylation; regulation of small GTPase mediated signal transduction\n",
       "33                                           Mm.12967                                                  inhibitor of kappaB kinase gamma, transcript variant 2 (Ikbkg)           5.1       0.020         1.20         0.01           -1.6               0.00               -1.1               0.00                                                                 Activation of NF-kappaB-inducing kinase\n",
       "34                                          Mm.294007                          PREDICTED: proprotein convertase subtilisin/kexin type 6, transcript variant 4 (Pcsk6)           3.7       0.000         0.90         0.94           -0.7               0.03                1.8               0.04  Determination of left/right symmetry; transmembrane receptor protein tyrosine kinase signaling pathway\n",
       "35                                          Mm.308180                                               protein tyrosine phosphatase-like A domain containing 1 (Ptplad1)           3.0       0.000         0.70         0.64            1.6               0.09                0.3               0.41                         I-kappaB kinase/NF-kappaB cascade; JNK cascade; Rho protein signal transduction\n",
       "36                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "37            Cell cycle/Growth and differentiation b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "38                                           Mm.20801                                                      HIV-1 tat interactive protein 2, homolog (human) (Htatip2)           4.2       0.010        -0.10         0.22           -0.7               0.03                0.4               0.40                                          Regulation of angiogenesis and apoptosis; cell differentiation\n",
       "39                                           Mm.43831                                                                   lectin, galactose binding, soluble 1 (Lgals1)           6.0       0.010        -1.40         0.03           -1.1               0.00               -1.5               0.00                                                                 Myoblast differentiation; sugar binding\n",
       "40                                           Mm.62876                                                                                         ZW10 interactor (Zwint)           3.0       0.010        -0.50         0.04           -1.2               0.00                0.3               0.43                                                                               cell cycle; cell division\n",
       "41                                          Mm.290692                                                                                             transketolase (Tkt)           2.6       0.000        -0.50         0.03            0.5               0.40               -1.2               0.00                                                                 Metal ion binding; regulation of growth\n",
       "42                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "43       Cellular transport and transport mechanism b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "44                                          Mm.103777                                                             Rhesus blood group-associated B glycoprotein (Rhbg)           4.8       0.020         0.10         0.24           -0.8               0.03               -1.1               0.00                                                                           Ammonium transporter activity\n",
       "45                                          Mm.222536            solute carrier family 25 (mitochondrial carrier, peroxisomal membrane protein), member 17 (Slc25a17)           2.4       0.000        -0.90         0.02           -1.9               0.00               -5.8               0.00                                                           mitochondrial transport; transporter activity\n",
       "46                                          Mm.281800  Solute carrier family 35 (UDP-glucuronic acid/UDP-N-acetylgalactosamine dual transporter), member D1 (Slc35d1)           2.8       0.000         0.60         0.53            0.5               0.37               -1.2               0.00                                                                            Nucleotide-sugar transporter\n",
       "47                                          Mm.218875                                                  target of myb1-like 2 (chicken), transcript variant 1 (Tom1l2)           2.1       0.000        -2.10         0.06            5.4               0.00                2.5               0.01                                       Golgi vesicle-mediated transport; intracellular protein transport\n",
       "48                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "49   Transcription/Translation/Protein modification b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "50                                          Mm.298030                                                              synovial sarcoma, X member B, breakpoint 1 (Ssxb1)           5.1       0.020        -0.20         0.20            1.7               0.05               -1.4               0.00                                                                             Regulation of transcription\n",
       "51                                           Mm.34483                                                                      leucine rich repeat containing 35 (Lrrc35)           3.2       0.000         1.40         0.03           -0.1               0.25               -0.4               0.12                                                                 Metal ion binding; protein modification\n",
       "52                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "53                  Miscellaneous (mixed functions) b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "54                                          Mm.234247                                                               Dimethylarginine dimethylaminohydrolase 1 (Ddah1)           4.5       0.010        -0.60         0.08            0.2               0.33                1.4               0.28                                             nitric oxide biosynthesis; protein amino acid nitrosylation\n",
       "55                                           Mm.26834                                                                                EF hand domain family A1 (Efha1)           2.4       0.000        -1.30         0.00           -0.4               0.15               -1.3               0.00                                                                                     Calcium ion binding\n",
       "56                                          Mm.182912                                                          growth hormone inducible transmembrane protein (Ghitm)           2.2       0.000        -0.80         0.07            1.1               0.09                1.1               0.10                                                                                     Interacts with FtsH\n",
       "57                                           Mm.41665            Glutamate receptor, ionotropic, N-methyl D-asparate-associated protein 1 (glutamate binding) (Grina)           2.3       0.000         0.10         0.29            0.7               0.69                0.5               0.54                                                                                       Receptor activity\n",
       "58                                          Mm.228797                                                                                       major vault protein (Mvp)           3.2       0.010         0.60         0.57            1.5               0.62                1.1               0.00                                                          Calcium ion binding; ribonucleoprotein complex\n",
       "59                                          Mm.252080                                           PREDICTED: phosphogluconate dehydrogenase, transcript variant 1 (Pgd)           3.0       0.000        -2.00         0.00            0.5               0.45               -1.2               0.00                                                               pentose-phosphate shunt, oxidative branch\n",
       "60                                          Mm.293463                                                                                                     pirin (Pir)           2.2       0.050         1.20         0.01           -0.1               0.18                0.2               0.46                                                                                       metal ion binding\n",
       "61                                          Mm.173058                                                                                secreted phosphoprotein 2 (Spp2)           2.3       0.000         1.30         0.01            0.8               0.74                1.7               0.00                                                                                         Bone remodeling\n",
       "62                                                NaN                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "63                                          Unknown b                                                                                                             NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                                                     NaN\n",
       "64                                           Mm.22109                                                                      RIKEN cDNA 2610204L23 gene (2610204L23Rik)           2.3       0.000         0.20         0.34            1.6               0.01               -0.8               0.14                                                                                                 Unknown\n",
       "65                                          Mm.100125                                                 SH3 domain binding glutamic acid-rich protein like 2 (Sh3bgrl2)           3.5       0.030         0.80         0.85            0.6               0.55                0.5               0.62                                                                                                 Unknown\n",
       "66                                          Mm.102407                                                                                  testis expressed gene 2 (Tex2)           2.4       0.000         0.70         0.61            0.0               0.20                0.3               0.39                                                                                                 Unknown\n",
       "67                                          Mm.289795                                                                          ubiquitin-associated protein 1 (Ubap1)           2.6       0.000         0.50         0.44            0.6               0.43               -1.2               0.00                                                                                                 Unknown\n",
       "68                                          Mm.257762                                                    PREDICTED: WD repeat domain 43, transcript variant 9 (Wdr43)           4.1       0.000         0.60         0.43           -0.8               0.03                1.3               0.04                                                                                                 Unknown"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Transcripts in reduced levels within liver of knockout mice relative to wild type mice a\n",
      "  footer_text: a Genes depleted ≥ 2.0 fold in ΔTrsp mice as compared to Trsp mice with a P-value ≤ 0.05 were assessed and shown in the table. The corresponding transcript levels were also analyzed in A34, G37L and G37H replacement mice relative to Trsp mice. Gene Unigene Accession Number, gene description, fold change with P value, and gene function(s) are shown. Four control and four experimental animals were used for comparing ΔTrsp, A34 and G37L with Trsp, while three control and three experimental animals were used for comparing G37H with Trsp. b Genes are grouped into classes according to function.\n",
      "    unique: PMC2423936/PMC2423936.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>&#916;Trsp</italic> vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>A34</italic> vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>G37</italic>L vs <italic>Trsp</italic></th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\"><italic>G37</italic>H vs <italic>Trsp</italic></th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">UniGene</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Description</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Folds</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Function/Gene Ontology</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Cellular transport and transport mechanism</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.381860</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">coatomer protein complex, subunit gamma 2 (Copg2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Protein transport</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.261168</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Potassium inwardly-rectifying channel, subfamily J, member 9 (Kcnj9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.08</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ion transport; voltage-gated ion channel activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.283370</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Neurocalcin delta (Ncald)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Vesicle-mediated transport</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.296837</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">sideroflexin 2 (Sfxn2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>3.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.49</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ion transport</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.250738</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 3 (Slc13a3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dicarboxylic acid transport; ion transport; symporter activity;</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.255586</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">solute carrier organic anion transporter family, member 1a4 (Slco1a4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.98</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ion transport; organic anion transporter activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Intracellular communication/Signal transduction</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.376094</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Anti-Mullerian hormone (Amh)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.32</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Transforming growth factor beta receptor binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.271854</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Low density lipoprotein receptor-related protein 1 (Lrp1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.18</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Negative regulation of Wnt receptor signaling pathway</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.6595</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">mitogen activated protein kinase kinase kinase 5 (Map3k5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Activation of MAPK activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.3810</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">phosphatidylinositol 3-kinase, C2 domain containing, alpha polypeptide (Pik3c2a)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.58</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Intracellular signaling cascade</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Metabolism</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.46269</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">insulin I (Ins1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Carbohydrate metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.34459</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">junctophilin 2 (Jph2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.34</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Carbohydrate metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.19669</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 3 (Pfkfb3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Fructose and mannose metabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.9277</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">phospholipase A2, group VII (platelet-activating factor acetylhydrolase, plasma) (Pla2g7)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Lipid catabolism</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Cellular structural organization</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.49994</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">hook homolog 1 (Drosophila) (Hook1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>3.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.59</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Cytoskeleton organization and biogenesis; spermatid development</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.129840</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">phosphodiesterase 4D interacting protein transcript variant 1 (Pde4dip),</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.56</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.21</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Association with spindle pole body microtubules</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.9684</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">vasodilator-stimulated phosphoprotein (Vasp)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Actin cytoskeleton organization and biogenesis</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Transcription/Translation</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.174044</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 2700050L05 gene, transcript variant 2 (2700050L05Rik)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Transcription regulation</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.2645</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">eukaryotic translation elongation factor 1 alpha 2 (Eef1a2)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Protein biosynthesis; translational elongation</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.303355</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">One cut domain, family member 1 (Onecut1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>4.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Regulation of transcription</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.7916</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Paf1, RNA polymerase II associated factor, homolog (S. cerevisiae) (Paf1)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.77</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.16</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.31</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Regulation of transcription</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.244820</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">TATA box binding protein (Tbp)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Regulation of transcription</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Miscellaneous (Mixed functions)</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.28796</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 5730446C15 gene, (5730446C15Rik)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.74</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Peptidase activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.71924</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">ankyrin repeat domain 38 (Ankrd38)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Protein-protein interaction</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.119936</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">attractin (Atrn)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Inflammatory response; protein binding; sugar binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.212861</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">cullin 4A (Cul4a)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.62</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Cell cycle; induction of apoptosis by intracellular signals; ubiquitin cycle</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.171323</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">nuclear receptor coactivator 6 interacting protein (Ncoa6ip)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.03</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">S-adenosylmethionine-dependent<break />methyltransferase activity; protein binding</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.261818</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Ring finger protein 167 (Rnf167)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.8</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.91</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.5</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Metal ion binding; protein binding; peptidase activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.284592</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">SUMO/sentrin specific peptidase 3 (Senp3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.6</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.04</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.43</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Peptidase activity; protein metabolism; ubiquitin-specific protease activity</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"11\" rowspan=\"1\" valign=\"top\"><bold>Unknown</bold> <xref ref-type=\"table-fn\" rid=\"TFN11\">b</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.416885</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 2010109N18 gene</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.36</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.27</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.4</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.138091</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">PREDICTED: RIKEN cDNA 3110001I20 gene, transcript variant 1 (3110001I20Rik)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.05</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.13</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.395958</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 9130221J18 gene</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>4.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.01</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.395042</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">RIKEN cDNA 9430078K24 gene</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>0.3</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.425114</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">PREDICTED: similar to gonadotropin inducible ovarian transcription factor 1, transcript variant 3 (LOC664985)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.9</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.2</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mm.219946</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">SET and MYND domain containing 5 (Smyd5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.7</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>1.1</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">-<bold>2.0</bold></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.00</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Unknown</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th colspan=\"2\" halign=\"left\">\\n</th>\n",
       "      <th>ΔTrsp vs Trsp</th>\n",
       "      <th>A34 vs Trsp</th>\n",
       "      <th>G37L vs Trsp</th>\n",
       "      <th>G37H vs Trsp</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>UniGene</th>\n",
       "      <th>Description</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Folds</th>\n",
       "      <th>P value</th>\n",
       "      <th>Function/Gene Ontology</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Cellular transport and transport mechanism b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Mm.381860</td>\n",
       "      <td>coatomer protein complex, subunit gamma 2 (Copg2)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.05</td>\n",
       "      <td>-2.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Protein transport</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Mm.261168</td>\n",
       "      <td>Potassium inwardly-rectifying channel, subfamily J, member 9 (Kcnj9)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.5</td>\n",
       "      <td>0.08</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Ion transport; voltage-gated ion channel activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Mm.283370</td>\n",
       "      <td>Neurocalcin delta (Ncald)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Vesicle-mediated transport</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Mm.296837</td>\n",
       "      <td>sideroflexin 2 (Sfxn2)</td>\n",
       "      <td>-3.8</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.49</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.05</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.43</td>\n",
       "      <td>Ion transport</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Mm.250738</td>\n",
       "      <td>solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 3 (Slc13a3)</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.04</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Dicarboxylic acid transport; ion transport; symporter activity;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Mm.255586</td>\n",
       "      <td>solute carrier organic anion transporter family, member 1a4 (Slco1a4)</td>\n",
       "      <td>-2.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.98</td>\n",
       "      <td>-0.3</td>\n",
       "      <td>0.20</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Ion transport; organic anion transporter activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Intracellular communication/Signal transduction b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Mm.376094</td>\n",
       "      <td>Anti-Mullerian hormone (Amh)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>0.41</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.24</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>0.32</td>\n",
       "      <td>Transforming growth factor beta receptor binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Mm.271854</td>\n",
       "      <td>Low density lipoprotein receptor-related protein 1 (Lrp1)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.1</td>\n",
       "      <td>0.23</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.18</td>\n",
       "      <td>-1.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Negative regulation of Wnt receptor signaling pathway</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Mm.6595</td>\n",
       "      <td>mitogen activated protein kinase kinase kinase 5 (Map3k5)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.2</td>\n",
       "      <td>0.35</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.43</td>\n",
       "      <td>Activation of MAPK activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Mm.3810</td>\n",
       "      <td>phosphatidylinositol 3-kinase, C2 domain containing, alpha polypeptide (Pik3c2a)</td>\n",
       "      <td>-2.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.58</td>\n",
       "      <td>-0.7</td>\n",
       "      <td>0.23</td>\n",
       "      <td>1.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Intracellular signaling cascade</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Metabolism b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Mm.46269</td>\n",
       "      <td>insulin I (Ins1)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.7</td>\n",
       "      <td>0.02</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.1</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Carbohydrate metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Mm.34459</td>\n",
       "      <td>junctophilin 2 (Jph2)</td>\n",
       "      <td>-2.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.34</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Carbohydrate metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Mm.19669</td>\n",
       "      <td>6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 3 (Pfkfb3)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.05</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.19</td>\n",
       "      <td>0.7</td>\n",
       "      <td>0.60</td>\n",
       "      <td>2.1</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Fructose and mannose metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Mm.9277</td>\n",
       "      <td>phospholipase A2, group VII (platelet-activating factor acetylhydrolase, plasma) (Pla2g7)</td>\n",
       "      <td>-2.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.7</td>\n",
       "      <td>0.60</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.03</td>\n",
       "      <td>Lipid catabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Cellular structural organization b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Mm.49994</td>\n",
       "      <td>hook homolog 1 (Drosophila) (Hook1)</td>\n",
       "      <td>-3.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.59</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.19</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Cytoskeleton organization and biogenesis; spermatid development</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Mm.129840</td>\n",
       "      <td>phosphodiesterase 4D interacting protein transcript variant 1 (Pde4dip),</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.56</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.21</td>\n",
       "      <td>Association with spindle pole body microtubules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Mm.9684</td>\n",
       "      <td>vasodilator-stimulated phosphoprotein (Vasp)</td>\n",
       "      <td>-2.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Actin cytoskeleton organization and biogenesis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Transcription/Translation b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Mm.174044</td>\n",
       "      <td>RIKEN cDNA 2700050L05 gene, transcript variant 2 (2700050L05Rik)</td>\n",
       "      <td>-2.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Transcription regulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>Mm.2645</td>\n",
       "      <td>eukaryotic translation elongation factor 1 alpha 2 (Eef1a2)</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.02</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.03</td>\n",
       "      <td>Protein biosynthesis; translational elongation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Mm.303355</td>\n",
       "      <td>One cut domain, family member 1 (Onecut1)</td>\n",
       "      <td>-4.0</td>\n",
       "      <td>0.04</td>\n",
       "      <td>-1.8</td>\n",
       "      <td>0.09</td>\n",
       "      <td>-2.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-2.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Regulation of transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>Mm.7916</td>\n",
       "      <td>Paf1, RNA polymerase II associated factor, homolog (S. cerevisiae) (Paf1)</td>\n",
       "      <td>-2.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.7</td>\n",
       "      <td>0.77</td>\n",
       "      <td>-0.9</td>\n",
       "      <td>0.16</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.31</td>\n",
       "      <td>Regulation of transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>Mm.244820</td>\n",
       "      <td>TATA box binding protein (Tbp)</td>\n",
       "      <td>-2.6</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.45</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Regulation of transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>Miscellaneous (Mixed functions) b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Mm.28796</td>\n",
       "      <td>RIKEN cDNA 5730446C15 gene, (5730446C15Rik)</td>\n",
       "      <td>-2.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.7</td>\n",
       "      <td>0.74</td>\n",
       "      <td>-0.5</td>\n",
       "      <td>0.24</td>\n",
       "      <td>1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Peptidase activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>Mm.71924</td>\n",
       "      <td>ankyrin repeat domain 38 (Ankrd38)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.7</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-1.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Protein-protein interaction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>Mm.119936</td>\n",
       "      <td>attractin (Atrn)</td>\n",
       "      <td>-2.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.2</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Inflammatory response; protein binding; sugar binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>Mm.212861</td>\n",
       "      <td>cullin 4A (Cul4a)</td>\n",
       "      <td>-2.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.7</td>\n",
       "      <td>0.62</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Cell cycle; induction of apoptosis by intracellular signals; ubiquitin cycle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>Mm.171323</td>\n",
       "      <td>nuclear receptor coactivator 6 interacting protein (Ncoa6ip)</td>\n",
       "      <td>-2.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.8</td>\n",
       "      <td>0.03</td>\n",
       "      <td>-0.4</td>\n",
       "      <td>0.17</td>\n",
       "      <td>-0.6</td>\n",
       "      <td>0.14</td>\n",
       "      <td>S-adenosylmethionine-dependentmethyltransferase activity; protein binding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>Mm.261818</td>\n",
       "      <td>Ring finger protein 167 (Rnf167)</td>\n",
       "      <td>-2.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.2</td>\n",
       "      <td>0.91</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-1.5</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Metal ion binding; protein binding; peptidase activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>Mm.284592</td>\n",
       "      <td>SUMO/sentrin specific peptidase 3 (Senp3)</td>\n",
       "      <td>-2.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.35</td>\n",
       "      <td>-0.6</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.43</td>\n",
       "      <td>Peptidase activity; protein metabolism; ubiquitin-specific protease activity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>Unknown b</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>Mm.416885</td>\n",
       "      <td>RIKEN cDNA 2010109N18 gene</td>\n",
       "      <td>-2.9</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.36</td>\n",
       "      <td>3.5</td>\n",
       "      <td>0.27</td>\n",
       "      <td>-1.4</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>Mm.138091</td>\n",
       "      <td>PREDICTED: RIKEN cDNA 3110001I20 gene, transcript variant 1 (3110001I20Rik)</td>\n",
       "      <td>-2.3</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.9</td>\n",
       "      <td>0.05</td>\n",
       "      <td>-1.7</td>\n",
       "      <td>0.10</td>\n",
       "      <td>-0.9</td>\n",
       "      <td>0.13</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>Mm.395958</td>\n",
       "      <td>RIKEN cDNA 9130221J18 gene</td>\n",
       "      <td>-4.9</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.3</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.3</td>\n",
       "      <td>0.52</td>\n",
       "      <td>1.4</td>\n",
       "      <td>0.01</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>Mm.395042</td>\n",
       "      <td>RIKEN cDNA 9430078K24 gene</td>\n",
       "      <td>-2.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-0.3</td>\n",
       "      <td>0.13</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.8</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>Mm.425114</td>\n",
       "      <td>PREDICTED: similar to gonadotropin inducible ovarian transcription factor 1, transcript variant 3 (LOC664985)</td>\n",
       "      <td>-2.9</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.35</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.24</td>\n",
       "      <td>-1.2</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>Mm.219946</td>\n",
       "      <td>SET and MYND domain containing 5 (Smyd5)</td>\n",
       "      <td>-2.7</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.21</td>\n",
       "      <td>-1.1</td>\n",
       "      <td>0.00</td>\n",
       "      <td>-2.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>Unknown</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     \\n                                                                                                                              ΔTrsp vs Trsp A34 vs Trsp G37L vs Trsp G37H vs Trsp \\n             Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0                                                           Unnamed: 10_level_0\n",
       "                                              UniGene                                                                                                    Description         Folds     P value        Folds      P value          Folds            P value              Folds            P value                                                        Function/Gene Ontology\n",
       "0        Cellular transport and transport mechanism b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "1                                           Mm.381860                                                              coatomer protein complex, subunit gamma 2 (Copg2)          -2.3        0.00         -0.8         0.05           -2.5               0.00               -1.7               0.00                                                             Protein transport\n",
       "2                                           Mm.261168                                           Potassium inwardly-rectifying channel, subfamily J, member 9 (Kcnj9)          -2.3        0.00         -0.5         0.08            1.1               0.01               -1.7               0.00                             Ion transport; voltage-gated ion channel activity\n",
       "3                                           Mm.283370                                                                                      Neurocalcin delta (Ncald)          -2.3        0.00         -1.5         0.00            1.5               0.00                1.2               0.00                                                    Vesicle-mediated transport\n",
       "4                                           Mm.296837                                                                                         sideroflexin 2 (Sfxn2)          -3.8        0.01          0.1         0.49           -1.4               0.05                0.0               0.43                                                                 Ion transport\n",
       "5                                           Mm.250738                      solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 3 (Slc13a3)          -2.4        0.00         -0.8         0.04           -1.2               0.00                1.0               0.00               Dicarboxylic acid transport; ion transport; symporter activity;\n",
       "6                                           Mm.255586                                          solute carrier organic anion transporter family, member 1a4 (Slco1a4)          -2.5        0.00          1.0         0.98           -0.3               0.20               -1.2               0.00                             Ion transport; organic anion transporter activity\n",
       "7                                                 NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "8   Intracellular communication/Signal transduction b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "9                                           Mm.376094                                                                                   Anti-Mullerian hormone (Amh)          -2.3        0.00         -0.0         0.41           -0.1               0.24               -1.0               0.32                              Transforming growth factor beta receptor binding\n",
       "10                                          Mm.271854                                                      Low density lipoprotein receptor-related protein 1 (Lrp1)          -2.3        0.00         -0.1         0.23            0.0               0.18               -1.3               0.00                         Negative regulation of Wnt receptor signaling pathway\n",
       "11                                            Mm.6595                                                      mitogen activated protein kinase kinase kinase 5 (Map3k5)          -2.3        0.00         -0.2         0.35            1.2               0.00                0.1               0.43                                                   Activation of MAPK activity\n",
       "12                                            Mm.3810                               phosphatidylinositol 3-kinase, C2 domain containing, alpha polypeptide (Pik3c2a)          -2.2        0.00          0.4         0.58           -0.7               0.23                1.6               0.00                                               Intracellular signaling cascade\n",
       "13                                                NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "14                                       Metabolism b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "15                                           Mm.46269                                                                                               insulin I (Ins1)          -2.3        0.00          1.7         0.02           -1.1               0.00                2.1               0.01                                                       Carbohydrate metabolism\n",
       "16                                           Mm.34459                                                                                          junctophilin 2 (Jph2)          -2.7        0.00          0.3         0.34           -1.8               0.00               -1.2               0.00                                                       Carbohydrate metabolism\n",
       "17                                           Mm.19669                                                 6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 3 (Pfkfb3)          -2.3        0.05         -2.3         0.19            0.7               0.60                2.1               0.01                                               Fructose and mannose metabolism\n",
       "18                                            Mm.9277                      phospholipase A2, group VII (platelet-activating factor acetylhydrolase, plasma) (Pla2g7)          -2.8        0.00          2.2         0.14            0.7               0.60                1.1               0.03                                                              Lipid catabolism\n",
       "19                                                NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "20                 Cellular structural organization b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "21                                           Mm.49994                                                                            hook homolog 1 (Drosophila) (Hook1)          -3.1        0.00          0.4         0.59            0.0               0.19               -1.1               0.00               Cytoskeleton organization and biogenesis; spermatid development\n",
       "22                                          Mm.129840                                       phosphodiesterase 4D interacting protein transcript variant 1 (Pde4dip),          -2.4        0.00          0.5         0.56           -1.2               0.00                1.0               0.21                               Association with spindle pole body microtubules\n",
       "23                                            Mm.9684                                                                   vasodilator-stimulated phosphoprotein (Vasp)          -2.0        0.00          1.2         0.01           -1.2               0.00               -1.1               0.00                                Actin cytoskeleton organization and biogenesis\n",
       "24                                                NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "25                        Transcription/Translation b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "26                                          Mm.174044                                               RIKEN cDNA 2700050L05 gene, transcript variant 2 (2700050L05Rik)          -2.0        0.00         -1.2         0.00           -1.4               0.00               -1.8               0.00                                                      Transcription regulation\n",
       "27                                            Mm.2645                                                    eukaryotic translation elongation factor 1 alpha 2 (Eef1a2)          -2.4        0.00         -2.4         0.02           -1.8               0.00                1.3               0.03                                Protein biosynthesis; translational elongation\n",
       "28                                          Mm.303355                                                                      One cut domain, family member 1 (Onecut1)          -4.0        0.04         -1.8         0.09           -2.1               0.00               -2.5               0.00                                                   Regulation of transcription\n",
       "29                                            Mm.7916                                      Paf1, RNA polymerase II associated factor, homolog (S. cerevisiae) (Paf1)          -2.7        0.00          0.7         0.77           -0.9               0.16                0.0               0.31                                                   Regulation of transcription\n",
       "30                                          Mm.244820                                                                                 TATA box binding protein (Tbp)          -2.6        0.00          0.4         0.45            1.2               0.00               -1.4               0.00                                                   Regulation of transcription\n",
       "31                                                NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "32                  Miscellaneous (Mixed functions) b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "33                                           Mm.28796                                                                    RIKEN cDNA 5730446C15 gene, (5730446C15Rik)          -2.1        0.00          0.7         0.74           -0.5               0.24                1.7               0.00                                                            Peptidase activity\n",
       "34                                           Mm.71924                                                                             ankyrin repeat domain 38 (Ankrd38)          -2.3        0.00         -0.7         0.01           -1.3               0.00               -1.1               0.00                                                   Protein-protein interaction\n",
       "35                                          Mm.119936                                                                                               attractin (Atrn)          -2.0        0.00          2.2         0.01           -1.1               0.00               -1.7               0.00                         Inflammatory response; protein binding; sugar binding\n",
       "36                                          Mm.212861                                                                                              cullin 4A (Cul4a)          -2.2        0.00          1.7         0.62           -1.4               0.00               -1.1               0.00  Cell cycle; induction of apoptosis by intracellular signals; ubiquitin cycle\n",
       "37                                          Mm.171323                                                   nuclear receptor coactivator 6 interacting protein (Ncoa6ip)          -2.4        0.00         -0.8         0.03           -0.4               0.17               -0.6               0.14     S-adenosylmethionine-dependentmethyltransferase activity; protein binding\n",
       "38                                          Mm.261818                                                                               Ring finger protein 167 (Rnf167)          -2.8        0.00          1.2         0.91           -1.5               0.00               -1.5               0.00                        Metal ion binding; protein binding; peptidase activity\n",
       "39                                          Mm.284592                                                                      SUMO/sentrin specific peptidase 3 (Senp3)          -2.2        0.00          0.2         0.35           -0.6               0.04                0.3               0.43  Peptidase activity; protein metabolism; ubiquitin-specific protease activity\n",
       "40                                                NaN                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "41                                          Unknown b                                                                                                            NaN           NaN         NaN          NaN          NaN            NaN                NaN                NaN                NaN                                                                           NaN\n",
       "42                                          Mm.416885                                                                                     RIKEN cDNA 2010109N18 gene          -2.9        0.00          0.3         0.36            3.5               0.27               -1.4               0.00                                                                       Unknown\n",
       "43                                          Mm.138091                                    PREDICTED: RIKEN cDNA 3110001I20 gene, transcript variant 1 (3110001I20Rik)          -2.3        0.00         -0.9         0.05           -1.7               0.10               -0.9               0.13                                                                       Unknown\n",
       "44                                          Mm.395958                                                                                     RIKEN cDNA 9130221J18 gene          -4.9        0.00         -0.3         0.14            0.3               0.52                1.4               0.01                                                                       Unknown\n",
       "45                                          Mm.395042                                                                                     RIKEN cDNA 9430078K24 gene          -2.1        0.00         -0.3         0.13           -1.2               0.00                1.8               0.00                                                                       Unknown\n",
       "46                                          Mm.425114  PREDICTED: similar to gonadotropin inducible ovarian transcription factor 1, transcript variant 3 (LOC664985)          -2.9        0.00          0.2         0.35            0.1               0.24               -1.2               0.00                                                                       Unknown\n",
       "47                                          Mm.219946                                                                       SET and MYND domain containing 5 (Smyd5)          -2.7        0.00          0.0         0.21           -1.1               0.00               -2.0               0.00                                                                       Unknown"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2424084.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of the Study Sample: Community-dwelling Framingham Heart Study Original Cohort\n",
      "  footer_text: * Cardiovascular disease= coronary heart disease, stroke, transient ischemic attack, congestive heart failure, or intermittent claudication. Hypertension= blood pressure ≥ 140/90 or on antihypertensive medication treatment. Diabetes=causal blood glucose of 200 mg/dl or greater or treatment with oral hypoglycemic agent or insulin. Depressive symptoms assessed at examinations 22 and 25 with the Center for Epidemiologic Studies Depression Scale (CES-D). A score of 16 or greater defined the presence of depressive symptoms. Medical conditions included any of the following five conditions: diabetes, cardiovascular disease, cancer, chronic obstructive pulmonary disease, and hip fracture. NA= not available as data not collected. Rx=medication treatment\n",
      "    unique: PMC2424084/PMC2424084.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Characteristic<break />Mean or percent</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><break />Exam<break />15<break />(1977-<break />79)<break />N=177</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Wome<break />n<break />Exam<break />20<break />(1988-<break />90)<break />N=159</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><break />Exam<break />25<break />(1997<break />-99)<break />N=17<break />4</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><break />Exam<break />15<break />(1977-<break />79)<break />N=103</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Men<break />Exam<break />20<break />(1988-<break />90)<break />N=98</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><break />Exam<break />25<break />(1997-<break />99)<break />N=119</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"7\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mean age, years<break />82.2</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82.2</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">81.9</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">81.6</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82.1</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82.3</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Chronic medical conditions<xref ref-type=\"table-fn\" rid=\"TFN1\">*</xref></th>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Hypertension, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">76.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">81.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">83.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">56.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">85.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">76.5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Hypertension treatment, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">47.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">63.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">60.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">61.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">55.9</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Total cholesterol &#8805; 240 mg/dL or Rx,<break />%</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">38.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">24.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">32.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">12.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">22.9</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Diabetes, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Obesity (BMI &#8805; 30 kg/m<sup>2</sup>), %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Cardiovascular disease, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">37.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">28.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">29.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">44.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">55.5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Cancer, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">14.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">19.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Chronic obstructive pulmonary<break />disease, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">18.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">12.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Hip fracture, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Depressive symptoms, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">NA</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">NA</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">At least one medical condition, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">52.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">52.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">51.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">63.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">58.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">75.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Health behaviors</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Current cigarette smoker, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Alcohol intake, number drinks/week</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Socio-demographic factors</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Marital Status, Married, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">19.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">30.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">73.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">80.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">73.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;&#8195;&#8195;&#8195;&#8195;Widowed, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">69.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">63.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">59.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">22.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">14.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;&#8195;&#8195;&#8195;&#8195;Single, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;&#8195;&#8195;&#8195;&#8195;&#8195;Divorced, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Education, high school or greater, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">53.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">58.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">78.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">54.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">76.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Living situation, lives alone, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">NA</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">72.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">NA</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">19.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Work status, currently working, %</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">14.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Subjective health, good or excellent,<break />%</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">66.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">80.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">79.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">80.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">76.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">84.0</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>CharacteristicMean or percent</th>\n",
       "      <th>Exam15(1977-79)N=177</th>\n",
       "      <th>WomenExam20(1988-90)N=159</th>\n",
       "      <th>Exam25(1997-99)N=174</th>\n",
       "      <th>Exam15(1977-79)N=103</th>\n",
       "      <th>MenExam20(1988-90)N=98</th>\n",
       "      <th>Exam25(1997-99)N=119</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 1_level_1</th>\n",
       "      <th>Unnamed: 2_level_1</th>\n",
       "      <th>Unnamed: 3_level_1</th>\n",
       "      <th>Unnamed: 4_level_1</th>\n",
       "      <th>Unnamed: 5_level_1</th>\n",
       "      <th>Unnamed: 6_level_1</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Mean age, years82.2</th>\n",
       "      <th>82.2</th>\n",
       "      <th>81.9</th>\n",
       "      <th>\\n</th>\n",
       "      <th>81.6</th>\n",
       "      <th>82.1</th>\n",
       "      <th>82.3</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Chronic medical conditions*</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Hypertension, %</td>\n",
       "      <td>76.1</td>\n",
       "      <td>81.1</td>\n",
       "      <td>83.3</td>\n",
       "      <td>56.4</td>\n",
       "      <td>85.7</td>\n",
       "      <td>76.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Hypertension treatment, %</td>\n",
       "      <td>47.9</td>\n",
       "      <td>63.5</td>\n",
       "      <td>60.3</td>\n",
       "      <td>33.0</td>\n",
       "      <td>61.2</td>\n",
       "      <td>55.9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Total cholesterol ≥ 240 mg/dL or Rx,%</td>\n",
       "      <td>38.3</td>\n",
       "      <td>24.5</td>\n",
       "      <td>32.8</td>\n",
       "      <td>12.9</td>\n",
       "      <td>11.7</td>\n",
       "      <td>22.9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Diabetes, %</td>\n",
       "      <td>9.8</td>\n",
       "      <td>8.2</td>\n",
       "      <td>5.7</td>\n",
       "      <td>7.4</td>\n",
       "      <td>13.3</td>\n",
       "      <td>10.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Obesity (BMI ≥ 30 kg/m2), %</td>\n",
       "      <td>12.4</td>\n",
       "      <td>15.5</td>\n",
       "      <td>20.7</td>\n",
       "      <td>2.2</td>\n",
       "      <td>12.4</td>\n",
       "      <td>15.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Cardiovascular disease, %</td>\n",
       "      <td>37.9</td>\n",
       "      <td>28.3</td>\n",
       "      <td>29.3</td>\n",
       "      <td>41.7</td>\n",
       "      <td>44.9</td>\n",
       "      <td>55.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Cancer, %</td>\n",
       "      <td>11.3</td>\n",
       "      <td>14.5</td>\n",
       "      <td>19.0</td>\n",
       "      <td>15.5</td>\n",
       "      <td>16.3</td>\n",
       "      <td>24.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Chronic obstructive pulmonarydisease, %</td>\n",
       "      <td>9.9</td>\n",
       "      <td>9.6</td>\n",
       "      <td>5.7</td>\n",
       "      <td>18.5</td>\n",
       "      <td>12.5</td>\n",
       "      <td>10.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Hip fracture, %</td>\n",
       "      <td>5.1</td>\n",
       "      <td>8.8</td>\n",
       "      <td>6.9</td>\n",
       "      <td>1.9</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Depressive symptoms, %</td>\n",
       "      <td>NaN</td>\n",
       "      <td>16.7</td>\n",
       "      <td>10.4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6.1</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>At least one medical condition, %</td>\n",
       "      <td>52.7</td>\n",
       "      <td>52.2</td>\n",
       "      <td>51.1</td>\n",
       "      <td>63.0</td>\n",
       "      <td>58.7</td>\n",
       "      <td>75.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Health behaviors</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Current cigarette smoker, %</td>\n",
       "      <td>4.9</td>\n",
       "      <td>5.7</td>\n",
       "      <td>4.0</td>\n",
       "      <td>15.3</td>\n",
       "      <td>4.1</td>\n",
       "      <td>5.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Alcohol intake, number drinks/week</td>\n",
       "      <td>1.9</td>\n",
       "      <td>3.6</td>\n",
       "      <td>2.5</td>\n",
       "      <td>6.1</td>\n",
       "      <td>6.4</td>\n",
       "      <td>5.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Socio-demographic factors</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Marital Status, Married, %</td>\n",
       "      <td>17.8</td>\n",
       "      <td>19.0</td>\n",
       "      <td>30.2</td>\n",
       "      <td>73.4</td>\n",
       "      <td>80.9</td>\n",
       "      <td>73.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Widowed, %</td>\n",
       "      <td>69.9</td>\n",
       "      <td>63.3</td>\n",
       "      <td>59.3</td>\n",
       "      <td>22.3</td>\n",
       "      <td>14.9</td>\n",
       "      <td>21.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Single, %</td>\n",
       "      <td>10.4</td>\n",
       "      <td>12.0</td>\n",
       "      <td>8.7</td>\n",
       "      <td>2.1</td>\n",
       "      <td>2.1</td>\n",
       "      <td>3.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Divorced, %</td>\n",
       "      <td>1.9</td>\n",
       "      <td>5.7</td>\n",
       "      <td>1.8</td>\n",
       "      <td>2.2</td>\n",
       "      <td>2.1</td>\n",
       "      <td>1.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Education, high school or greater, %</td>\n",
       "      <td>53.2</td>\n",
       "      <td>58.3</td>\n",
       "      <td>78.5</td>\n",
       "      <td>46.5</td>\n",
       "      <td>54.3</td>\n",
       "      <td>76.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Living situation, lives alone, %</td>\n",
       "      <td>NaN</td>\n",
       "      <td>72.2</td>\n",
       "      <td>46.2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>17.3</td>\n",
       "      <td>19.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Work status, currently working, %</td>\n",
       "      <td>2.3</td>\n",
       "      <td>6.3</td>\n",
       "      <td>6.3</td>\n",
       "      <td>12.6</td>\n",
       "      <td>14.3</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Subjective health, good or excellent,%</td>\n",
       "      <td>66.9</td>\n",
       "      <td>80.5</td>\n",
       "      <td>79.2</td>\n",
       "      <td>80.0</td>\n",
       "      <td>76.5</td>\n",
       "      <td>84.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "              CharacteristicMean or percent Exam15(1977-79)N=177 WomenExam20(1988-90)N=159 Exam25(1997-99)N=174 Exam15(1977-79)N=103 MenExam20(1988-90)N=98 Exam25(1997-99)N=119\n",
       "         \\n                \\n                 Unnamed: 1_level_1        Unnamed: 2_level_1   Unnamed: 3_level_1   Unnamed: 4_level_1     Unnamed: 5_level_1   Unnamed: 6_level_1\n",
       "                        Mean age, years82.2                 82.2                      81.9     \\n                               81.6                   82.1                 82.3\n",
       "0               Chronic medical conditions*                  NaN                       NaN                  NaN                  NaN                    NaN                  NaN\n",
       "1                           Hypertension, %                 76.1                      81.1                 83.3                 56.4                   85.7                 76.5\n",
       "2                 Hypertension treatment, %                 47.9                      63.5                 60.3                 33.0                   61.2                 55.9\n",
       "3     Total cholesterol ≥ 240 mg/dL or Rx,%                 38.3                      24.5                 32.8                 12.9                   11.7                 22.9\n",
       "4                               Diabetes, %                  9.8                       8.2                  5.7                  7.4                   13.3                 10.1\n",
       "5               Obesity (BMI ≥ 30 kg/m2), %                 12.4                      15.5                 20.7                  2.2                   12.4                 15.3\n",
       "6                 Cardiovascular disease, %                 37.9                      28.3                 29.3                 41.7                   44.9                 55.5\n",
       "7                                 Cancer, %                 11.3                      14.5                 19.0                 15.5                   16.3                 24.4\n",
       "8   Chronic obstructive pulmonarydisease, %                  9.9                       9.6                  5.7                 18.5                   12.5                 10.3\n",
       "9                           Hip fracture, %                  5.1                       8.8                  6.9                  1.9                    1.0                  1.7\n",
       "10                   Depressive symptoms, %                  NaN                      16.7                 10.4                  NaN                    6.1                  7.0\n",
       "11        At least one medical condition, %                 52.7                      52.2                 51.1                 63.0                   58.7                 75.2\n",
       "12                         Health behaviors                  NaN                       NaN                  NaN                  NaN                    NaN                  NaN\n",
       "13              Current cigarette smoker, %                  4.9                       5.7                  4.0                 15.3                    4.1                  5.1\n",
       "14       Alcohol intake, number drinks/week                  1.9                       3.6                  2.5                  6.1                    6.4                  5.4\n",
       "15                Socio-demographic factors                  NaN                       NaN                  NaN                  NaN                    NaN                  NaN\n",
       "16               Marital Status, Married, %                 17.8                      19.0                 30.2                 73.4                   80.9                 73.7\n",
       "17                               Widowed, %                 69.9                      63.3                 59.3                 22.3                   14.9                 21.2\n",
       "18                                Single, %                 10.4                      12.0                  8.7                  2.1                    2.1                  3.4\n",
       "19                              Divorced, %                  1.9                       5.7                  1.8                  2.2                    2.1                  1.7\n",
       "20     Education, high school or greater, %                 53.2                      58.3                 78.5                 46.5                   54.3                 76.3\n",
       "21         Living situation, lives alone, %                  NaN                      72.2                 46.2                  NaN                   17.3                 19.3\n",
       "22        Work status, currently working, %                  2.3                       6.3                  6.3                 12.6                   14.3                 16.0\n",
       "23   Subjective health, good or excellent,%                 66.9                      80.5                 79.2                 80.0                   76.5                 84.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Self-reported Prevalence of Functional Limitation or Physical Disability: Framingham Heart Study Original Cohort, Mean Age 82 years\n",
      "  footer_text: Nagi physical performance scale limitation= some difficulty, a lot of difficulty, unable to do, do not do on physician order Rosow-Breslau functional health scale disability= unable to do. Modified Katz ADL scale disability=human assistance needed, minimally dependent; dependent † At exam 25 this question was asked as part of the modified Katz ADL Scale ‡ At Exams 20 and 25 the question was changed to transferring (getting in and out of a chair) and walking on a level surface about 50 yards\n",
      "    unique: PMC2424084/PMC2424084.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Scale and Scale Items*</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">WOMEN</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">MEN</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">(percent)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />15<break />N=177</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />20<break />N=159</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />25<break />N=174</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />15<break />N=103</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />20<break />N=98</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\">Exam<break />25<break />N=119</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"7\" rowspan=\"1\">\n",
       "                <bold>Nagi Physical Performance Scale</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Pulling or pushing large objects</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>40.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>31.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>18.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>14.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>11.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>6.7</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Either stooping, crouching, or kneeling</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>48.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>47.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>18.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>23.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>18.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>17.6</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Reaching or extending arms above shoulder<break />level</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>9.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>14.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>7.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>5.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>8.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.2</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Reaching or extending arms below shoulder<break />level</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>4.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>3.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>2.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Writing, handling, or fingering small objects</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>17.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>11.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>5.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>15.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>11.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>10.1</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Standing in place for long periods say 15<break />minutes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>39.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>29.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>15.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>22.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>16.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>12.2</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Sitting for long periods say 1 hour</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>9.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>5.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>4.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>9.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>6.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Rosow-Breslau Functional Health Scale</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Heavy work around the house</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>65.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>39.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>30.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>35.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>31.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>19.3</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Walks up and down stairs to second floor<xref ref-type=\"table-fn\" rid=\"TFN4\">&#8224;</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>26.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>7.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>6.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>6.8</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Walk one half mile</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>39.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>25.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>16.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>16.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>11.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>14.3</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Modified Katz Activities of Daily Living Scale</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Bathing/personal grooming</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>15.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>4.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Dressing</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.7</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>3.4</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>3.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Eating</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Getting from a bed to a chair<xref ref-type=\"table-fn\" rid=\"TFN5\">&#8225;</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>2.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>3.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Walking across small room<xref ref-type=\"table-fn\" rid=\"TFN5\">&#8225;</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>4.5</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.3</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.6</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>1.0</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>0.8</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"7\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Any special equipment or device use</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>7.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>13.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>28.2</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>3.9</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>4.1</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>17.6</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Scale and Scale Items*</th>\n",
       "      <th>\\n</th>\n",
       "      <th>WOMEN</th>\n",
       "      <th colspan=\"2\" halign=\"left\">\\n</th>\n",
       "      <th>MEN</th>\n",
       "      <th>\\n</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>(percent)</th>\n",
       "      <th>Exam15N=177</th>\n",
       "      <th>Exam20N=159</th>\n",
       "      <th>Exam25N=174</th>\n",
       "      <th>Exam15N=103</th>\n",
       "      <th>Exam20N=98</th>\n",
       "      <th>Exam25N=119</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Nagi Physical Performance Scale</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Pulling or pushing large objects</td>\n",
       "      <td>40.6</td>\n",
       "      <td>31.6</td>\n",
       "      <td>18.3</td>\n",
       "      <td>14.7</td>\n",
       "      <td>11.2</td>\n",
       "      <td>6.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Either stooping, crouching, or kneeling</td>\n",
       "      <td>48.3</td>\n",
       "      <td>47.8</td>\n",
       "      <td>18.5</td>\n",
       "      <td>23.5</td>\n",
       "      <td>18.4</td>\n",
       "      <td>17.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Reaching or extending arms above shoulderlevel</td>\n",
       "      <td>9.7</td>\n",
       "      <td>14.6</td>\n",
       "      <td>7.5</td>\n",
       "      <td>5.9</td>\n",
       "      <td>8.2</td>\n",
       "      <td>4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Reaching or extending arms below shoulderlevel</td>\n",
       "      <td>4.5</td>\n",
       "      <td>3.8</td>\n",
       "      <td>1.1</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Writing, handling, or fingering small objects</td>\n",
       "      <td>17.5</td>\n",
       "      <td>11.9</td>\n",
       "      <td>5.7</td>\n",
       "      <td>15.5</td>\n",
       "      <td>11.2</td>\n",
       "      <td>10.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Standing in place for long periods say 15minutes</td>\n",
       "      <td>39.0</td>\n",
       "      <td>29.7</td>\n",
       "      <td>15.9</td>\n",
       "      <td>22.3</td>\n",
       "      <td>16.3</td>\n",
       "      <td>12.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Sitting for long periods say 1 hour</td>\n",
       "      <td>9.7</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>9.9</td>\n",
       "      <td>6.1</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Rosow-Breslau Functional Health Scale</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Heavy work around the house</td>\n",
       "      <td>65.5</td>\n",
       "      <td>39.0</td>\n",
       "      <td>30.5</td>\n",
       "      <td>35.9</td>\n",
       "      <td>31.6</td>\n",
       "      <td>19.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Walks up and down stairs to second floor†</td>\n",
       "      <td>26.0</td>\n",
       "      <td>7.5</td>\n",
       "      <td>6.9</td>\n",
       "      <td>6.8</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Walk one half mile</td>\n",
       "      <td>39.5</td>\n",
       "      <td>25.2</td>\n",
       "      <td>16.7</td>\n",
       "      <td>16.5</td>\n",
       "      <td>11.2</td>\n",
       "      <td>14.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Modified Katz Activities of Daily Living Scale</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Bathing/personal grooming</td>\n",
       "      <td>15.3</td>\n",
       "      <td>2.5</td>\n",
       "      <td>2.9</td>\n",
       "      <td>4.9</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Dressing</td>\n",
       "      <td>1.7</td>\n",
       "      <td>1.9</td>\n",
       "      <td>3.4</td>\n",
       "      <td>2.9</td>\n",
       "      <td>3.1</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Eating</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.1</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Getting from a bed to a chair‡</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.3</td>\n",
       "      <td>2.9</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.1</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Walking across small room‡</td>\n",
       "      <td>4.5</td>\n",
       "      <td>1.3</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Any special equipment or device use</td>\n",
       "      <td>7.9</td>\n",
       "      <td>13.2</td>\n",
       "      <td>28.2</td>\n",
       "      <td>3.9</td>\n",
       "      <td>4.1</td>\n",
       "      <td>17.6</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                              Scale and Scale Items* \\n                     WOMEN \\n                                  MEN \\n            \n",
       "                                           (percent)      Exam15N=177 Exam20N=159      Exam25N=174 Exam15N=103 Exam20N=98    Exam25N=119\n",
       "0                    Nagi Physical Performance Scale              NaN         NaN              NaN         NaN        NaN            NaN\n",
       "1                   Pulling or pushing large objects             40.6        31.6             18.3        14.7       11.2            6.7\n",
       "2            Either stooping, crouching, or kneeling             48.3        47.8             18.5        23.5       18.4           17.6\n",
       "3     Reaching or extending arms above shoulderlevel              9.7        14.6              7.5         5.9        8.2            4.2\n",
       "4     Reaching or extending arms below shoulderlevel              4.5         3.8              1.1         1.0        2.0            0.8\n",
       "5      Writing, handling, or fingering small objects             17.5        11.9              5.7        15.5       11.2           10.1\n",
       "6   Standing in place for long periods say 15minutes             39.0        29.7             15.9        22.3       16.3           12.2\n",
       "7                Sitting for long periods say 1 hour              9.7         5.0              4.0         9.9        6.1            0.8\n",
       "8              Rosow-Breslau Functional Health Scale              NaN         NaN              NaN         NaN        NaN            NaN\n",
       "9                        Heavy work around the house             65.5        39.0             30.5        35.9       31.6           19.3\n",
       "10         Walks up and down stairs to second floor†             26.0         7.5              6.9         6.8        2.0            0.0\n",
       "11                                Walk one half mile             39.5        25.2             16.7        16.5       11.2           14.3\n",
       "12    Modified Katz Activities of Daily Living Scale              NaN         NaN              NaN         NaN        NaN            NaN\n",
       "13                         Bathing/personal grooming             15.3         2.5              2.9         4.9        2.0            0.0\n",
       "14                                          Dressing              1.7         1.9              3.4         2.9        3.1            0.8\n",
       "15                                            Eating              0.0         0.0              1.1         0.0        2.0            0.8\n",
       "16                    Getting from a bed to a chair‡              0.0         1.3              2.9         1.0        3.1            0.0\n",
       "17                        Walking across small room‡              4.5         1.3              0.6         0.0        1.0            0.8\n",
       "18                                               NaN              NaN         NaN              NaN         NaN        NaN            NaN\n",
       "19               Any special equipment or device use              7.9        13.2             28.2         3.9        4.1           17.6"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: TABLE 3\n",
      "  caption_text: Sex-specific Trends in Self-reported Functional Limitations and Physical Disability Adjusted for Age\n",
      "  footer_text: \n",
      "    unique: PMC2424084/PMC2424084.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"8\" rowspan=\"1\">Community-dwelling Framingham Heart Study Original Cohort, Mean Age 82 years</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"9\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Physical Function Scale</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Women</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\">Men</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mean or percent<break />95% confidence interval</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam 15<break />N=177</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam<break />20<break />N=159</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam<break />25<break />N=174</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">p-value</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam 15<break />N=103</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam<break />20<break />N=98</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam<break />25<break />N=119</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">p-value</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Functional limitation, Nagi Scale</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Number of items with limitation</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.67</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.70</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&lt;0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.93</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.73</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.53</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.03</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.46, 1.88</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.23,<break /><break />1.67</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.48,<break /><break />0.91</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.71,<break />1.14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.51,<break />0.96</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.32,<break />0.73</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Any difficulty on the scale</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">74.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">60.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">37.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">54.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">37.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">27.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">66.7, 81.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">51.5,<break /><break />68.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">29.9,<break /><break />46.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">43.1,<break />65.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">27.6,<break />49.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">19.6,<break />37.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">\n",
       "                <bold>Physical disability, Rosow-Breslau &amp; Katz ADL Scales</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Number of items with impairment</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.64</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.0<break />01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.71</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.55</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.02</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.34, 1.70</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.62,<break />1.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.46,<break />0.82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.52,<break />0.90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.36,<break />0.75</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.17,<break />0.52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Any difficulty on the scales</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">74.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">48.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">34.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.0<break />01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">42.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">33.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">22.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.009</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">66.7, 80.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">39.8,<break />57.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">27.0,<break />43.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">31.9,<break />53.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">23.7,<break />44.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">15.4,<break />32.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Summary, All three scales</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Number of items with limitation or</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3.19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2.26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.34</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.0<break />01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.64</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.87</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.009</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">impairment</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2.86, 3.52</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.91,<break />2.61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.00,<break />1.67</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.28,<break />2.00</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.92,<break />1.66</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.54,<break />1.20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"9\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <italic>Any difficulty on the scales</italic>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">88.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">69.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">49.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.0<break />01</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">63.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">48.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">36.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&lt;0.001</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">81.9, 92.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">60.8,<break />77.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">40.4,<break />57.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">52.9,<break />73.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">37.7,<break />59.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">27.1,<break />46.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Community-dwelling Framingham Heart Study Original Cohort, Mean Age 82 years</th>\n",
       "      <th>Unnamed: 2_level_0</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 1_level_1</th>\n",
       "      <th>Unnamed: 2_level_1</th>\n",
       "      <th>Unnamed: 3_level_1</th>\n",
       "      <th>Unnamed: 4_level_1</th>\n",
       "      <th>Unnamed: 5_level_1</th>\n",
       "      <th>Unnamed: 6_level_1</th>\n",
       "      <th>Unnamed: 7_level_1</th>\n",
       "      <th>Unnamed: 8_level_1</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Physical Function Scale</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Women</th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Men</th>\n",
       "      <th>\\n</th>\n",
       "      <th>Unnamed: 8_level_2</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Mean or percent95% confidence interval</th>\n",
       "      <th>Exam 15N=177</th>\n",
       "      <th>Exam20N=159</th>\n",
       "      <th>Exam25N=174</th>\n",
       "      <th>p-value</th>\n",
       "      <th>Exam 15N=103</th>\n",
       "      <th>Exam20N=98</th>\n",
       "      <th>Exam25N=119</th>\n",
       "      <th>p-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Functional limitation, Nagi Scale</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Number of items with limitation</td>\n",
       "      <td>1.67</td>\n",
       "      <td>1.45</td>\n",
       "      <td>0.70</td>\n",
       "      <td>&lt;0.0</td>\n",
       "      <td>0.93</td>\n",
       "      <td>0.73</td>\n",
       "      <td>0.53</td>\n",
       "      <td>0.03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>1.46, 1.88</td>\n",
       "      <td>1.231.67</td>\n",
       "      <td>0.480.91</td>\n",
       "      <td>01</td>\n",
       "      <td>0.711.14</td>\n",
       "      <td>0.510.96</td>\n",
       "      <td>0.320.73</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Any difficulty on the scale</td>\n",
       "      <td>74.6</td>\n",
       "      <td>60.5</td>\n",
       "      <td>37.9</td>\n",
       "      <td>&lt;0.0</td>\n",
       "      <td>54.2</td>\n",
       "      <td>37.8</td>\n",
       "      <td>27.8</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>66.7, 81.1</td>\n",
       "      <td>51.568.8</td>\n",
       "      <td>29.946.5</td>\n",
       "      <td>01</td>\n",
       "      <td>43.165.0</td>\n",
       "      <td>27.649.2</td>\n",
       "      <td>19.637.9</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Physical disability, Rosow-Breslau &amp; Katz ADL Scales</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Number of items with impairment</td>\n",
       "      <td>1.52</td>\n",
       "      <td>0.81</td>\n",
       "      <td>0.64</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>0.71</td>\n",
       "      <td>0.55</td>\n",
       "      <td>0.35</td>\n",
       "      <td>0.02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34, 1.70</td>\n",
       "      <td>0.621.00</td>\n",
       "      <td>0.460.82</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.520.90</td>\n",
       "      <td>0.360.75</td>\n",
       "      <td>0.170.52</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Any difficulty on the scales</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>NaN</td>\n",
       "      <td>74.5</td>\n",
       "      <td>48.5</td>\n",
       "      <td>34.6</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>42.3</td>\n",
       "      <td>33.3</td>\n",
       "      <td>22.8</td>\n",
       "      <td>0.009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>66.7, 80.9</td>\n",
       "      <td>39.857.2</td>\n",
       "      <td>27.043.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>31.953.3</td>\n",
       "      <td>23.744.4</td>\n",
       "      <td>15.432.3</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Summary, All three scales</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Number of items with limitation or</td>\n",
       "      <td>3.19</td>\n",
       "      <td>2.26</td>\n",
       "      <td>1.34</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>1.64</td>\n",
       "      <td>1.29</td>\n",
       "      <td>0.87</td>\n",
       "      <td>0.009</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>impairment</td>\n",
       "      <td>2.86, 3.52</td>\n",
       "      <td>1.912.61</td>\n",
       "      <td>1.001.67</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.282.00</td>\n",
       "      <td>0.921.66</td>\n",
       "      <td>0.541.20</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Any difficulty on the scales</td>\n",
       "      <td>88.1</td>\n",
       "      <td>69.6</td>\n",
       "      <td>49.1</td>\n",
       "      <td>&lt;0.001</td>\n",
       "      <td>63.9</td>\n",
       "      <td>48.8</td>\n",
       "      <td>36.4</td>\n",
       "      <td>&lt;0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>NaN</td>\n",
       "      <td>81.9, 92.4</td>\n",
       "      <td>60.877.0</td>\n",
       "      <td>40.457.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>52.973.5</td>\n",
       "      <td>37.759.8</td>\n",
       "      <td>27.146.9</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        \\n               Community-dwelling Framingham Heart Study Original Cohort, Mean Age 82 years Unnamed: 2_level_0 Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0\n",
       "                      \\n                \\n                                                                         Unnamed: 1_level_1 Unnamed: 2_level_1 Unnamed: 3_level_1 Unnamed: 4_level_1 Unnamed: 5_level_1 Unnamed: 6_level_1 Unnamed: 7_level_1 Unnamed: 8_level_1\n",
       "                                 Physical Function Scale                                                             \\n                            Women   \\n                 \\n                 \\n                              Men     \\n             Unnamed: 8_level_2\n",
       "                  Mean or percent95% confidence interval                                                                 Exam 15N=177        Exam20N=159        Exam25N=174            p-value       Exam 15N=103         Exam20N=98        Exam25N=119            p-value\n",
       "0                      Functional limitation, Nagi Scale                                                                          NaN                NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "1                        Number of items with limitation                                                                         1.67               1.45               0.70               <0.0               0.93               0.73               0.53               0.03\n",
       "2                                                    NaN                                                                   1.46, 1.88           1.231.67           0.480.91                 01           0.711.14           0.510.96           0.320.73                NaN\n",
       "3                            Any difficulty on the scale                                                                         74.6               60.5               37.9               <0.0               54.2               37.8               27.8             <0.001\n",
       "4                                                    NaN                                                                   66.7, 81.1           51.568.8           29.946.5                 01           43.165.0           27.649.2           19.637.9                NaN\n",
       "5   Physical disability, Rosow-Breslau & Katz ADL Scales                                                                          NaN                NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "6                        Number of items with impairment                                                                         1.52               0.81               0.64             <0.001               0.71               0.55               0.35               0.02\n",
       "7                                                    NaN                                                                   1.34, 1.70           0.621.00           0.460.82                NaN           0.520.90           0.360.75           0.170.52                NaN\n",
       "8                           Any difficulty on the scales                                                                          NaN                NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "9                                                    NaN                                                                         74.5               48.5               34.6             <0.001               42.3               33.3               22.8              0.009\n",
       "10                                                   NaN                                                                   66.7, 80.9           39.857.2           27.043.0                NaN           31.953.3           23.744.4           15.432.3                NaN\n",
       "11                             Summary, All three scales                                                                          NaN                NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "12                    Number of items with limitation or                                                                         3.19               2.26               1.34             <0.001               1.64               1.29               0.87              0.009\n",
       "13                                            impairment                                                                   2.86, 3.52           1.912.61           1.001.67                NaN           1.282.00           0.921.66           0.541.20                NaN\n",
       "14                                                   NaN                                                                          NaN                NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "15                          Any difficulty on the scales                                                                         88.1               69.6               49.1             <0.001               63.9               48.8               36.4             <0.001\n",
       "16                                                   NaN                                                                   81.9, 92.4           60.877.0           40.457.9                NaN           52.973.5           37.759.8           27.146.9                NaN"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Appendix 1\n",
      "  caption_text: Study Sample: Framingham Original Cohort, Aged 79 to 88, Attending Index Examinations 15, 20, and 25\n",
      "  footer_text: † Examinations off-site in participants’ homes or nursing homes were not available at examination 15. ‡ The Framingham Disability Study collected physical function data at examination 15.(19;43)\n",
      "    unique: PMC2424084/PMC2424084.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam 15<break />1977&#8211;1979<break />N</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam 20<break />1988&#8211;1990<break />N</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Exam 25<break />1997&#8211;1999<break />N</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Eligible: Age 79 to 88 years at the Index Exam Based on Date of Birth</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1435</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1727</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1969</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Death Prior to Index Exam</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">855</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1076</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1170</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Alive at Index Exam</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">580</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">651</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">799</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <bold>Exclusions, N (percent of living participants)</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Alive, did not attend the index exam</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">156 (26.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">269 (41.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">243 (30.4)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Alive with dementia</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5 (0.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10 (1.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6 (0.8)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Alive, examination took place outside the Framingham Study<break />&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;clinic<xref ref-type=\"table-fn\" rid=\"TFN6\">&#8224;</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">58 (8.9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">122 (15.3)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(nursing home, personal residence, or unknown site)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">57(8.8)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">135 (16.9)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Alive, missing data (&#8805;1 item on the 3 physical function scales)<xref ref-type=\"table-fn\" rid=\"TFN7\">&#8225;</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">139 (24.0)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"4\" rowspan=\"1\" valign=\"bottom\">\n",
       "                <hr />\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>Final Sample, N (percent of living participants)</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">280<break />(48.3)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">257<break />(39.5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">293<break />(36.7)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th>Exam 151977–1979N</th>\n",
       "      <th>Exam 201988–1990N</th>\n",
       "      <th>Exam 251997–1999N</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Eligible: Age 79 to 88 years at the Index Exam Based on Date of Birth</td>\n",
       "      <td>1435</td>\n",
       "      <td>1727</td>\n",
       "      <td>1969</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Death Prior to Index Exam</td>\n",
       "      <td>855</td>\n",
       "      <td>1076</td>\n",
       "      <td>1170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Alive at Index Exam</td>\n",
       "      <td>580</td>\n",
       "      <td>651</td>\n",
       "      <td>799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Exclusions, N (percent of living participants)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Alive, did not attend the index exam</td>\n",
       "      <td>156 (26.9)</td>\n",
       "      <td>269 (41.3)</td>\n",
       "      <td>243 (30.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Alive with dementia</td>\n",
       "      <td>5 (0.9)</td>\n",
       "      <td>10 (1.5)</td>\n",
       "      <td>6 (0.8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Alive, examination took place outside the Framingham Study clinic†</td>\n",
       "      <td>NaN</td>\n",
       "      <td>58 (8.9)</td>\n",
       "      <td>122 (15.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>(nursing home, personal residence, or unknown site)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>57(8.8)</td>\n",
       "      <td>135 (16.9)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Alive, missing data (≥1 item on the 3 physical function scales)‡</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>NaN</td>\n",
       "      <td>139 (24.0)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Final Sample, N (percent of living participants)</td>\n",
       "      <td>280(48.3)</td>\n",
       "      <td>257(39.5)</td>\n",
       "      <td>293(36.7)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                         \\n               Exam 151977–1979N Exam 201988–1990N Exam 251997–1999N\n",
       "0   Eligible: Age 79 to 88 years at the Index Exam Based on Date of Birth              1435              1727              1969\n",
       "1                                                                     NaN               NaN               NaN               NaN\n",
       "2                                               Death Prior to Index Exam               855              1076              1170\n",
       "3                                                                     NaN               NaN               NaN               NaN\n",
       "4                                                     Alive at Index Exam               580               651               799\n",
       "5                                                                     NaN               NaN               NaN               NaN\n",
       "6                          Exclusions, N (percent of living participants)               NaN               NaN               NaN\n",
       "7                                    Alive, did not attend the index exam        156 (26.9)        269 (41.3)        243 (30.4)\n",
       "8                                                     Alive with dementia           5 (0.9)          10 (1.5)           6 (0.8)\n",
       "9      Alive, examination took place outside the Framingham Study clinic†               NaN          58 (8.9)        122 (15.3)\n",
       "10                    (nursing home, personal residence, or unknown site)               NaN           57(8.8)        135 (16.9)\n",
       "11       Alive, missing data (≥1 item on the 3 physical function scales)‡               NaN               NaN               NaN\n",
       "12                                                                    NaN        139 (24.0)               NaN               NaN\n",
       "13                                                                    NaN               NaN               NaN               NaN\n",
       "14                       Final Sample, N (percent of living participants)         280(48.3)         257(39.5)         293(36.7)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2424085.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: TABLE 1\n",
      "  caption_text: Characteristics of Participants, by Age Group: Health and Retirement Study, 1998\n",
      "  footer_text: Note. ADL = activities of daily living. IADL = instrumental activities of daily living. ADL include bathing, dressing, eating, transferring, and toileting. IADL include shopping, preparing meals, using the telephone, managing medications, and managing finances.\n",
      "    unique: PMC2424085/PMC2424085.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"none\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 50&#8211;<break />59 Years<break /> (n = 5382),<break /> %</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 60&#8211;<break />69 Years<break /> (n = 6591),<break /> %</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 70&#8211;<break />79 Years<break /> (n = 4903),<break /> %</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 80&#8211;<break />89 Years<break /> (n = 2219),<break /> %</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 80&#8211;<break />89 Years<break /> (n = 335),<break /> %</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">All<break /> Participants<break /> (n = 19 430),<break /> %</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"left\" colspan=\"6\" rowspan=\"1\" valign=\"top\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Mortality</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">47</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">76</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">18</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Men</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">46</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">38</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">44</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"8\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>Chronic conditions</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Hypertension</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">38</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">53</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">55</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">53</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">47</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Diabetes mellitus</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">None</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">89</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">85</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">83</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">85</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">86</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Oral medications only</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Insulin</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Kidney disease</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Heart disease</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">None</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">87</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">75</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">69</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">68</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">79</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Angina</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">16</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">22</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">16</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Heart attack past 2 y</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Heart failure</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Cancer</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">None</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">85</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">86</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">89</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Localized</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Spread</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Lung disease</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">None</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">89</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">88</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">91</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No medications</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Medications</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Oxygen</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Stroke</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">None</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">95</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">91</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">87</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">84</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">93</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No current deficits</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Current deficits</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Any chronic conditions</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">70</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">83</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">88</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">92</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">90</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"8\" rowspan=\"1\" valign=\"top\">\n",
       "                <bold>Functional limitations</bold>\n",
       "              </td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Toileting</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">96</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">89</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">95</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dependent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Eating</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">98</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">98</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">92</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">88</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dependent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Bathing</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">95</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">92</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">68</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">93</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dependent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Using the telephone</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">98</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">87</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">78</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">95</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dependent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Managing finances</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">No difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">97</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">94</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">85</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">72</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">95</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dependent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Any difficulty in an ADL or IADL</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">46</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">68</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">21</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Walking</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Able to jog 1 mile</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">14</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Able to walk several blocks</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">57</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">57</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Able to walk 1 block</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">15</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Difficulty walking 1 block</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">11</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Can't walk 1 block</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Age 50–59 Years (n = 5382), %</th>\n",
       "      <th>Age 60–69 Years (n = 6591), %</th>\n",
       "      <th>Age 70–79 Years (n = 4903), %</th>\n",
       "      <th>Age 80–89 Years (n = 2219), %</th>\n",
       "      <th>Age 80–89 Years (n = 335), %</th>\n",
       "      <th>All Participants (n = 19 430), %</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n</th>\n",
       "      <th>\\n                \\n</th>\n",
       "      <th>Unnamed: 3_level_1</th>\n",
       "      <th>Unnamed: 4_level_1</th>\n",
       "      <th>Unnamed: 5_level_1</th>\n",
       "      <th>Unnamed: 6_level_1</th>\n",
       "      <th>Unnamed: 7_level_1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Mortality</td>\n",
       "      <td>5</td>\n",
       "      <td>11.0</td>\n",
       "      <td>24.0</td>\n",
       "      <td>47.0</td>\n",
       "      <td>76.0</td>\n",
       "      <td>18.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Men</td>\n",
       "      <td>42</td>\n",
       "      <td>46.0</td>\n",
       "      <td>45.0</td>\n",
       "      <td>38.0</td>\n",
       "      <td>30.0</td>\n",
       "      <td>44.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Chronic conditions</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Hypertension</td>\n",
       "      <td>38</td>\n",
       "      <td>48.0</td>\n",
       "      <td>53.0</td>\n",
       "      <td>55.0</td>\n",
       "      <td>53.0</td>\n",
       "      <td>47.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Diabetes mellitus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>89.0</td>\n",
       "      <td>85.0</td>\n",
       "      <td>83.0</td>\n",
       "      <td>85.0</td>\n",
       "      <td>90.0</td>\n",
       "      <td>86.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Oral medications only</td>\n",
       "      <td>7.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>11.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>10.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Insulin</td>\n",
       "      <td>3.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Kidney disease</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Heart disease</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>87.0</td>\n",
       "      <td>81.0</td>\n",
       "      <td>75.0</td>\n",
       "      <td>69.0</td>\n",
       "      <td>68.0</td>\n",
       "      <td>79.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Angina</td>\n",
       "      <td>10.0</td>\n",
       "      <td>16.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>22.0</td>\n",
       "      <td>20.0</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Heart attack past 2 y</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Heart failure</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Cancer</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>94.0</td>\n",
       "      <td>90.0</td>\n",
       "      <td>85.0</td>\n",
       "      <td>82.0</td>\n",
       "      <td>86.0</td>\n",
       "      <td>89.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Localized</td>\n",
       "      <td>6.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>13.0</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Spread</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.6</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Lung disease</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>94.0</td>\n",
       "      <td>90.0</td>\n",
       "      <td>89.0</td>\n",
       "      <td>88.0</td>\n",
       "      <td>94.0</td>\n",
       "      <td>91.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No medications</td>\n",
       "      <td>4.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>5.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Medications</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Oxygen</td>\n",
       "      <td>0.4</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.6</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Stroke</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>97.0</td>\n",
       "      <td>95.0</td>\n",
       "      <td>91.0</td>\n",
       "      <td>87.0</td>\n",
       "      <td>84.0</td>\n",
       "      <td>93.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No current deficits</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Current deficits</td>\n",
       "      <td>2.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>Any chronic conditions</td>\n",
       "      <td>70</td>\n",
       "      <td>83.0</td>\n",
       "      <td>88.0</td>\n",
       "      <td>92.0</td>\n",
       "      <td>90.0</td>\n",
       "      <td>82.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Functional limitations</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>Toileting</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No difficulty</td>\n",
       "      <td>97.0</td>\n",
       "      <td>96.0</td>\n",
       "      <td>94.0</td>\n",
       "      <td>89.0</td>\n",
       "      <td>81.0</td>\n",
       "      <td>95.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty</td>\n",
       "      <td>3.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Dependent</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.8</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Eating</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No difficulty</td>\n",
       "      <td>98.0</td>\n",
       "      <td>98.0</td>\n",
       "      <td>97.0</td>\n",
       "      <td>92.0</td>\n",
       "      <td>88.0</td>\n",
       "      <td>97.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty</td>\n",
       "      <td>0.8</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Dependent</td>\n",
       "      <td>0.7</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>Bathing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No difficulty</td>\n",
       "      <td>97.0</td>\n",
       "      <td>95.0</td>\n",
       "      <td>92.0</td>\n",
       "      <td>82.0</td>\n",
       "      <td>68.0</td>\n",
       "      <td>93.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty</td>\n",
       "      <td>2.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Dependent</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>11.0</td>\n",
       "      <td>26.0</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>Using the telephone</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No difficulty</td>\n",
       "      <td>98.0</td>\n",
       "      <td>97.0</td>\n",
       "      <td>94.0</td>\n",
       "      <td>87.0</td>\n",
       "      <td>78.0</td>\n",
       "      <td>95.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Dependent</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.5</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>Managing finances</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>NaN</td>\n",
       "      <td>No difficulty</td>\n",
       "      <td>97.0</td>\n",
       "      <td>97.0</td>\n",
       "      <td>94.0</td>\n",
       "      <td>85.0</td>\n",
       "      <td>72.0</td>\n",
       "      <td>95.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Dependent</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.8</td>\n",
       "      <td>2.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>11.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>Any difficulty in an ADL or IADL</td>\n",
       "      <td>15</td>\n",
       "      <td>19.0</td>\n",
       "      <td>26.0</td>\n",
       "      <td>46.0</td>\n",
       "      <td>68.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Walking</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Able to jog 1 mile</td>\n",
       "      <td>24.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>14.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Able to walk several blocks</td>\n",
       "      <td>57.0</td>\n",
       "      <td>61.0</td>\n",
       "      <td>60.0</td>\n",
       "      <td>45.0</td>\n",
       "      <td>30.0</td>\n",
       "      <td>57.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Able to walk 1 block</td>\n",
       "      <td>11.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>24.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>15.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Difficulty walking 1 block</td>\n",
       "      <td>7.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>13.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>29.0</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Can't walk 1 block</td>\n",
       "      <td>0.8</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    \\n               Age 50–59 Years (n = 5382), %      Age 60–69 Years (n = 6591), % Age 70–79 Years (n = 4903), % Age 80–89 Years (n = 2219), % Age 80–89 Years (n = 335), % All Participants (n = 19 430), % Unnamed: 7_level_0\n",
       "                    \\n                            \\n               \\n                \\n                          Unnamed: 3_level_1            Unnamed: 4_level_1           Unnamed: 5_level_1               Unnamed: 6_level_1 Unnamed: 7_level_1\n",
       "0                          Mortality                             5                               11.0                          24.0                          47.0                         76.0                             18.0                NaN\n",
       "1                                Men                            42                               46.0                          45.0                          38.0                         30.0                             44.0                NaN\n",
       "2                 Chronic conditions                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "3                       Hypertension                            38                               48.0                          53.0                          55.0                         53.0                             47.0                NaN\n",
       "4                  Diabetes mellitus                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "5                                NaN                          None                               89.0                          85.0                          83.0                         85.0                             90.0               86.0\n",
       "6                                NaN         Oral medications only                                7.0                          10.0                          11.0                         10.0                              8.0               10.0\n",
       "7                                NaN                       Insulin                                3.0                           3.0                           4.0                          3.0                              2.0                3.0\n",
       "8                                NaN                Kidney disease                                2.0                           2.0                           2.0                          2.0                              1.0                2.0\n",
       "9                      Heart disease                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "10                               NaN                          None                               87.0                          81.0                          75.0                         69.0                             68.0               79.0\n",
       "11                               NaN                        Angina                               10.0                          16.0                          19.0                         22.0                             20.0               16.0\n",
       "12                               NaN         Heart attack past 2 y                                1.0                           2.0                           3.0                          4.0                              3.0                2.0\n",
       "13                               NaN                 Heart failure                                1.0                           2.0                           4.0                          5.0                              9.0                3.0\n",
       "14                            Cancer                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "15                               NaN                          None                               94.0                          90.0                          85.0                         82.0                             86.0               89.0\n",
       "16                               NaN                     Localized                                6.0                          10.0                          14.0                         17.0                             13.0               11.0\n",
       "17                               NaN                        Spread                                0.4                           0.4                           0.8                          0.8                              0.6                1.0\n",
       "18                      Lung disease                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "19                               NaN                          None                               94.0                          90.0                          89.0                         88.0                             94.0               91.0\n",
       "20                               NaN                No medications                                4.0                           5.0                           5.0                          5.0                              3.0                5.0\n",
       "21                               NaN                   Medications                                2.0                           4.0                           4.0                          4.0                              3.0                3.0\n",
       "22                               NaN                        Oxygen                                0.4                           1.0                           2.0                          2.0                              0.6                1.0\n",
       "23                            Stroke                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "24                               NaN                          None                               97.0                          95.0                          91.0                         87.0                             84.0               93.0\n",
       "25                               NaN           No current deficits                                1.0                           3.0                           5.0                          7.0                              6.0                3.0\n",
       "26                               NaN              Current deficits                                2.0                           3.0                           5.0                          6.0                              9.0                3.0\n",
       "27            Any chronic conditions                            70                               83.0                          88.0                          92.0                         90.0                             82.0                NaN\n",
       "28            Functional limitations                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "29                         Toileting                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "30                               NaN                 No difficulty                               97.0                          96.0                          94.0                         89.0                             81.0               95.0\n",
       "31                               NaN                    Difficulty                                3.0                           3.0                           5.0                          7.0                             10.0                4.0\n",
       "32                               NaN                     Dependent                                0.6                           0.8                           2.0                          4.0                              8.0                1.0\n",
       "33                            Eating                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "34                               NaN                 No difficulty                               98.0                          98.0                          97.0                         92.0                             88.0               97.0\n",
       "35                               NaN                    Difficulty                                0.8                           1.0                           1.0                          3.0                              6.0                1.0\n",
       "36                               NaN                     Dependent                                0.7                           1.0                           2.0                          5.0                              6.0                2.0\n",
       "37                           Bathing                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "38                               NaN                 No difficulty                               97.0                          95.0                          92.0                         82.0                             68.0               93.0\n",
       "39                               NaN                    Difficulty                                2.0                           3.0                           4.0                          7.0                              6.0                3.0\n",
       "40                               NaN                     Dependent                                1.0                           2.0                           4.0                         11.0                             26.0                4.0\n",
       "41               Using the telephone                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "42                               NaN                 No difficulty                               98.0                          97.0                          94.0                         87.0                             78.0               95.0\n",
       "43                               NaN                    Difficulty                                2.0                           2.0                           4.0                          9.0                             14.0                4.0\n",
       "44                               NaN                     Dependent                                0.1                           0.5                           1.0                          3.0                              9.0                1.0\n",
       "45                 Managing finances                           NaN                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "46                               NaN                 No difficulty                               97.0                          97.0                          94.0                         85.0                             72.0               95.0\n",
       "47                               NaN                    Difficulty                                2.0                           2.0                           4.0                          8.0                             17.0                4.0\n",
       "48                               NaN                     Dependent                                0.4                           0.8                           2.0                          7.0                             11.0                2.0\n",
       "49  Any difficulty in an ADL or IADL                            15                               19.0                          26.0                          46.0                         68.0                             21.0                NaN\n",
       "50                               NaN                       Walking                                NaN                           NaN                           NaN                          NaN                              NaN                NaN\n",
       "51                               NaN            Able to jog 1 mile                               24.0                          15.0                           8.0                          3.0                              1.0               14.0\n",
       "52                               NaN   Able to walk several blocks                               57.0                          61.0                          60.0                         45.0                             30.0               57.0\n",
       "53                               NaN          Able to walk 1 block                               11.0                          14.0                          17.0                         24.0                             21.0               15.0\n",
       "54                               NaN    Difficulty walking 1 block                                7.0                           9.0                          13.0                         21.0                             29.0               11.0\n",
       "55                               NaN            Can't walk 1 block                                0.8                           2.0                           2.0                          7.0                             19.0                2.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: TABLE 2\n",
      "  caption_text: Harrell's C Statistic of Cox Mortality Models Including Different Risk Factors by Age Groups: Health and Retirement Study, 1998\n",
      "  footer_text: a Permutation test of whether slope of regression between age group and C statistic = 0.\n",
      "    unique: PMC2424085/PMC2424085.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Models</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 50&#8211;59 Years</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 60&#8211;69 Years</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 70&#8211;79 Years</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 80&#8211;89 Years</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Age 90&#8211;99 Years</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Test for Trend<xref ref-type=\"table-fn\" rid=\"TFN1\">a</xref></th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">A: Base (age and gender)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.62 (0.59&#8211;0.65)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.60 (0.58&#8211;0.62)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.61 (0.59&#8211;0.62)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.60 (0.58&#8211;0.62)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.58 (0.54&#8211;0.62)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> = .05</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">B: Base + functional limitations</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.73 (0.71&#8211;0.76)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.72 (0.70&#8211;0.73)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.70 (0.69&#8211;0.72)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.67 (0.66&#8211;0.69)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.67 (0.63&#8211;0.70)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> = .017</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">C: Base + chronic conditions</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.78 (0.75&#8211;0.81)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.72 (0.70&#8211;0.74)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.70 (0.68&#8211;0.71)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.65 (0.63&#8211;0.67)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.61 (0.58&#8211;0.65)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> = .016</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">D: Base + conditions + function</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.79 (0.77&#8211;0.82)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.75 (0.73&#8211;0.77)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.73 (0.72&#8211;0.74)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.69 (0.68&#8211;0.71)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.69 (0.66&#8211;0.72)</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> = .017</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"><italic>P</italic> value for model B/C comparison</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">.001</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">.48</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">.36</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">.006</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">.004</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Models</th>\n",
       "      <th>Age 50–59 Years</th>\n",
       "      <th>Age 60–69 Years</th>\n",
       "      <th>Age 70–79 Years</th>\n",
       "      <th>Age 80–89 Years</th>\n",
       "      <th>Age 90–99 Years</th>\n",
       "      <th>Test for Trenda</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>A: Base (age and gender)</td>\n",
       "      <td>0.62 (0.59–0.65)</td>\n",
       "      <td>0.60 (0.58–0.62)</td>\n",
       "      <td>0.61 (0.59–0.62)</td>\n",
       "      <td>0.60 (0.58–0.62)</td>\n",
       "      <td>0.58 (0.54–0.62)</td>\n",
       "      <td>P = .05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>B: Base + functional limitations</td>\n",
       "      <td>0.73 (0.71–0.76)</td>\n",
       "      <td>0.72 (0.70–0.73)</td>\n",
       "      <td>0.70 (0.69–0.72)</td>\n",
       "      <td>0.67 (0.66–0.69)</td>\n",
       "      <td>0.67 (0.63–0.70)</td>\n",
       "      <td>P = .017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>C: Base + chronic conditions</td>\n",
       "      <td>0.78 (0.75–0.81)</td>\n",
       "      <td>0.72 (0.70–0.74)</td>\n",
       "      <td>0.70 (0.68–0.71)</td>\n",
       "      <td>0.65 (0.63–0.67)</td>\n",
       "      <td>0.61 (0.58–0.65)</td>\n",
       "      <td>P = .016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>D: Base + conditions + function</td>\n",
       "      <td>0.79 (0.77–0.82)</td>\n",
       "      <td>0.75 (0.73–0.77)</td>\n",
       "      <td>0.73 (0.72–0.74)</td>\n",
       "      <td>0.69 (0.68–0.71)</td>\n",
       "      <td>0.69 (0.66–0.72)</td>\n",
       "      <td>P = .017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>P value for model B/C comparison</td>\n",
       "      <td>.001</td>\n",
       "      <td>.48</td>\n",
       "      <td>.36</td>\n",
       "      <td>.006</td>\n",
       "      <td>.004</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                             Models   Age 50–59 Years   Age 60–69 Years   Age 70–79 Years   Age 80–89 Years   Age 90–99 Years Test for Trenda\n",
       "0          A: Base (age and gender)  0.62 (0.59–0.65)  0.60 (0.58–0.62)  0.61 (0.59–0.62)  0.60 (0.58–0.62)  0.58 (0.54–0.62)         P = .05\n",
       "1  B: Base + functional limitations  0.73 (0.71–0.76)  0.72 (0.70–0.73)  0.70 (0.69–0.72)  0.67 (0.66–0.69)  0.67 (0.63–0.70)        P = .017\n",
       "2      C: Base + chronic conditions  0.78 (0.75–0.81)  0.72 (0.70–0.74)  0.70 (0.68–0.71)  0.65 (0.63–0.67)  0.61 (0.58–0.65)        P = .016\n",
       "3   D: Base + conditions + function  0.79 (0.77–0.82)  0.75 (0.73–0.77)  0.73 (0.72–0.74)  0.69 (0.68–0.71)  0.69 (0.66–0.72)        P = .017\n",
       "4  P value for model B/C comparison              .001               .48               .36              .006              .004             NaN"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2424090.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Characteristics of those who died compared to those who did not die during follow-up.\n",
      "  footer_text: a CES-D = Center for Epidemiologic Studies Depression Scale – Short Form\n",
      "    unique: PMC2424090/PMC2424090.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Average Characteristics</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Dead (N = 1168)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Alive (N = 2986)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mean age in years (SD)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">80.2 (7.34)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">76.0 (5.45)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Sex (%)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">26.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">73.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">31.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">69.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Race (%)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Black</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">26.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">73.4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;White</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">30.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">69.5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Lifetime SES</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#8722;0.04 (.76)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.09 (.76)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mean score CES-D<xref ref-type=\"table-fn\" rid=\"TFN1\">a</xref> (SD)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">2.31(2.34)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.58 (2.02)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Mean no. medical conditions</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.22 (1.04)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.99 (0.93)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Perceived discrimination</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.13 (1.72)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.99 (1.58)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Average Characteristics</th>\n",
       "      <th>Dead (N = 1168)</th>\n",
       "      <th>Alive (N = 2986)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Mean age in years (SD)</td>\n",
       "      <td>80.2 (7.34)</td>\n",
       "      <td>76.0 (5.45)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Sex (%)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Female</td>\n",
       "      <td>26.3</td>\n",
       "      <td>73.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Male</td>\n",
       "      <td>31.0</td>\n",
       "      <td>69.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Race (%)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Black</td>\n",
       "      <td>26.6</td>\n",
       "      <td>73.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>White</td>\n",
       "      <td>30.5</td>\n",
       "      <td>69.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Lifetime SES</td>\n",
       "      <td>−0.04 (.76)</td>\n",
       "      <td>0.09 (.76)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Mean score CES-Da (SD)</td>\n",
       "      <td>2.31(2.34)</td>\n",
       "      <td>1.58 (2.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Mean no. medical conditions</td>\n",
       "      <td>1.22 (1.04)</td>\n",
       "      <td>0.99 (0.93)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Perceived discrimination</td>\n",
       "      <td>1.13 (1.72)</td>\n",
       "      <td>0.99 (1.58)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        Average Characteristics Dead (N = 1168) Alive (N = 2986)\n",
       "0        Mean age in years (SD)     80.2 (7.34)      76.0 (5.45)\n",
       "1                       Sex (%)             NaN              NaN\n",
       "2                        Female            26.3             73.7\n",
       "3                          Male            31.0             69.0\n",
       "4                      Race (%)             NaN              NaN\n",
       "5                         Black            26.6             73.4\n",
       "6                         White            30.5             69.5\n",
       "7                  Lifetime SES     −0.04 (.76)       0.09 (.76)\n",
       "8        Mean score CES-Da (SD)      2.31(2.34)      1.58 (2.02)\n",
       "9   Mean no. medical conditions     1.22 (1.04)      0.99 (0.93)\n",
       "10     Perceived discrimination     1.13 (1.72)      0.99 (1.58)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Relative risk of mortality associated with perceived discrimination\n",
      "  footer_text: Hazard ratios and 95% confidence interval; Hazard ratios are estimated from separate models adjusted for age, sex, race, race*age, and lifetime SES.\n",
      "    unique: PMC2424090/PMC2424090.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Variable</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Full Cohort (N = 4101)</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination score (range = 0 &#8722; 9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.05 (1.01, 1.09)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.12 (1.04, 1.20)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination &#215; Black</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.92 (0.85, 0.99)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Race-Matched Cohort (N=3070)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#8195;<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination (range = 0 &#8722;9)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.07 (1.03, 1.12)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.11 (1.03, 1.20)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination &#215; Black</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.95 (0.87, 1.04)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Variable</th>\n",
       "      <th>Full Cohort (N = 4101)</th>\n",
       "      <th>Unnamed: 2_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>Model 1</th>\n",
       "      <th>Model 2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Discrimination score (range = 0 − 9)</td>\n",
       "      <td>1.05 (1.01, 1.09)</td>\n",
       "      <td>1.12 (1.04, 1.20)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Discrimination × Black</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.92 (0.85, 0.99)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Race-Matched Cohort (N=3070)</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Model 1</td>\n",
       "      <td>Model 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Discrimination (range = 0 −9)</td>\n",
       "      <td>1.07 (1.03, 1.12)</td>\n",
       "      <td>1.11 (1.03, 1.20)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Discrimination × Black</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.95 (0.87, 1.04)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                               Variable        Full Cohort (N = 4101) Unnamed: 2_level_0\n",
       "                     Unnamed: 0_level_1                       Model 1            Model 2\n",
       "0  Discrimination score (range = 0 − 9)             1.05 (1.01, 1.09)  1.12 (1.04, 1.20)\n",
       "1                Discrimination × Black                           NaN  0.92 (0.85, 0.99)\n",
       "2                                   NaN  Race-Matched Cohort (N=3070)                NaN\n",
       "3                                   NaN                       Model 1            Model 2\n",
       "4         Discrimination (range = 0 −9)             1.07 (1.03, 1.12)  1.11 (1.03, 1.20)\n",
       "5                Discrimination × Black                           NaN  0.95 (0.87, 1.04)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Relative risk of mortality associated with unfair treatment and personal rejection\n",
      "  footer_text: Hazard ratios and 95% confidence interval; Hazard ratios are estimated from separate models adjusted for age, sex, race, race*age, and lifetime SES.\n",
      "    unique: PMC2424090/PMC2424090.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Variable</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Unfair Treatment</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Personal Rejection</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Full Cohort (N = 4098)</th>\n",
       "              <th align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Full Cohort (N = 4099)</th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination score (range = 0 &#8722; 4)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.05 (0.99, 1.11)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.12 (1.00, 1.24)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.12 (1.05, 1.20)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.29 (1.12, 1.48)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination &#215; Black</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.92 (0.81, 1.04)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.84 (0.72, 0.99)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Race-Matched Cohort (N = 3069)</td>\n",
       "              <td align=\"center\" colspan=\"2\" rowspan=\"1\" valign=\"top\">Race-Matched Cohort (N = 3068)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">&#8195;<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 1<hr /></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Model 2<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination score (range = 0 &#8722; 5)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.09 (1.02, 1.16)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.14 (1.02, 1.28)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.16 (1.07, 1.25)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">1.22 (1.04, 1.43)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\">Discrimination &#215; Black</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.94 (0.82, 1.07)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" valign=\"top\">0.93 (0.78, 1.12)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Variable</th>\n",
       "      <th>Unfair Treatment</th>\n",
       "      <th>Personal Rejection</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>Full Cohort (N = 4098)</th>\n",
       "      <th>Full Cohort (N = 4099)</th>\n",
       "      <th>Unnamed: 3_level_1</th>\n",
       "      <th>Unnamed: 4_level_1</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_2</th>\n",
       "      <th>Model 1</th>\n",
       "      <th>Model 2</th>\n",
       "      <th>Model 1</th>\n",
       "      <th>Model 2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Discrimination score (range = 0 − 4)</td>\n",
       "      <td>1.05 (0.99, 1.11)</td>\n",
       "      <td>1.12 (1.00, 1.24)</td>\n",
       "      <td>1.12 (1.05, 1.20)</td>\n",
       "      <td>1.29 (1.12, 1.48)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Discrimination × Black</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.92 (0.81, 1.04)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.84 (0.72, 0.99)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Race-Matched Cohort (N = 3069)</td>\n",
       "      <td>Race-Matched Cohort (N = 3068)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Model 1</td>\n",
       "      <td>Model 2</td>\n",
       "      <td>Model 1</td>\n",
       "      <td>Model 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Discrimination score (range = 0 − 5)</td>\n",
       "      <td>1.09 (1.02, 1.16)</td>\n",
       "      <td>1.14 (1.02, 1.28)</td>\n",
       "      <td>1.16 (1.07, 1.25)</td>\n",
       "      <td>1.22 (1.04, 1.43)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Discrimination × Black</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.94 (0.82, 1.07)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.93 (0.78, 1.12)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                               Variable                Unfair Treatment              Personal Rejection Unnamed: 3_level_0 Unnamed: 4_level_0\n",
       "                     Unnamed: 0_level_1          Full Cohort (N = 4098)          Full Cohort (N = 4099) Unnamed: 3_level_1 Unnamed: 4_level_1\n",
       "                     Unnamed: 0_level_2                         Model 1                         Model 2            Model 1            Model 2\n",
       "0  Discrimination score (range = 0 − 4)               1.05 (0.99, 1.11)               1.12 (1.00, 1.24)  1.12 (1.05, 1.20)  1.29 (1.12, 1.48)\n",
       "1                Discrimination × Black                             NaN               0.92 (0.81, 1.04)                NaN  0.84 (0.72, 0.99)\n",
       "2                                   NaN  Race-Matched Cohort (N = 3069)  Race-Matched Cohort (N = 3068)                NaN                NaN\n",
       "3                                   NaN                         Model 1                         Model 2            Model 1            Model 2\n",
       "4  Discrimination score (range = 0 − 5)               1.09 (1.02, 1.16)               1.14 (1.02, 1.28)  1.16 (1.07, 1.25)  1.22 (1.04, 1.43)\n",
       "5                Discrimination × Black                             NaN               0.94 (0.82, 1.07)                NaN  0.93 (0.78, 1.12)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2430055.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Peripheral blood cell counts in 10-month post-transplant animals\n",
      "  footer_text: Data shown are mean (±SD) values from three to five mice per genotype * p<0.05, ** p<0.005 when compared to wild type VEX\n",
      "    unique: PMC2430055/PMC2430055.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">RBC (M/ul)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Hematocrit (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">WBC (K/ul)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Plt (K/ul)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Wild type VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.03 (&#177;0.48)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">43.68 (&#177;2.87)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17.72 (&#177;5.99)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">638 (&#177;257)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Wild type RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.26 (&#177;2.21)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">40.80 (&#177;5.55)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.13 (&#177;5.75)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">353 (&#177;132)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">p15KO VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.31 (&#177;0.54)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">43.96 (&#177;2.52)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11.79 (&#177;3.34)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">805 (&#177;126)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">p15KO RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.90 (&#177;0.94)<xref ref-type=\"table-fn\" rid=\"TFN3\">**</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">36.79 (&#177;3.68)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.64 (&#177;2.24)<xref ref-type=\"table-fn\" rid=\"TFN3\">**</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">789 (&#177;176)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">p16KO VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.22 (&#177;0.48)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45.13 (&#177;1.01)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.20 (&#177;4.85)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">602 (&#177;151)</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">p16KO RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.09 (&#177;1.04)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41.45 (&#177;10.73)</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.76 (&#177;3.05)<xref ref-type=\"table-fn\" rid=\"TFN2\">*</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">480 (&#177;212)</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th>RBC (M/ul)</th>\n",
       "      <th>Hematocrit (%)</th>\n",
       "      <th>WBC (K/ul)</th>\n",
       "      <th>Plt (K/ul)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Wild type VEX</td>\n",
       "      <td>9.03 (±0.48)</td>\n",
       "      <td>43.68 (±2.87)</td>\n",
       "      <td>17.72 (±5.99)</td>\n",
       "      <td>638 (±257)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Wild type RE</td>\n",
       "      <td>7.26 (±2.21)</td>\n",
       "      <td>40.80 (±5.55)</td>\n",
       "      <td>12.13 (±5.75)</td>\n",
       "      <td>353 (±132)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>p15KO VEX</td>\n",
       "      <td>9.31 (±0.54)</td>\n",
       "      <td>43.96 (±2.52)</td>\n",
       "      <td>11.79 (±3.34)</td>\n",
       "      <td>805 (±126)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>p15KO RE</td>\n",
       "      <td>6.90 (±0.94)**</td>\n",
       "      <td>36.79 (±3.68)*</td>\n",
       "      <td>7.64 (±2.24)**</td>\n",
       "      <td>789 (±176)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>p16KO VEX</td>\n",
       "      <td>9.22 (±0.48)</td>\n",
       "      <td>45.13 (±1.01)</td>\n",
       "      <td>13.20 (±4.85)</td>\n",
       "      <td>602 (±151)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>p16KO RE</td>\n",
       "      <td>7.09 (±1.04)*</td>\n",
       "      <td>41.45 (±10.73)</td>\n",
       "      <td>7.76 (±3.05)*</td>\n",
       "      <td>480 (±212)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  \\n                    RBC (M/ul)  Hematocrit (%)      WBC (K/ul)  Plt (K/ul)\n",
       "0    Wild type VEX    9.03 (±0.48)   43.68 (±2.87)   17.72 (±5.99)  638 (±257)\n",
       "1     Wild type RE    7.26 (±2.21)   40.80 (±5.55)   12.13 (±5.75)  353 (±132)\n",
       "2        p15KO VEX    9.31 (±0.54)   43.96 (±2.52)   11.79 (±3.34)  805 (±126)\n",
       "3         p15KO RE  6.90 (±0.94)**  36.79 (±3.68)*  7.64 (±2.24)**  789 (±176)\n",
       "4        p16KO VEX    9.22 (±0.48)   45.13 (±1.01)   13.20 (±4.85)  602 (±151)\n",
       "5         p16KO RE   7.09 (±1.04)*  41.45 (±10.73)   7.76 (±3.05)*  480 (±212)"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Differential counts of sorted VEX (+) myeloid cells from 10-month post-transplant animals\n",
      "  footer_text: Data shown are percentages of >250 cells counted per sample. Blast/Pro: myeloblast and promyelocytes; Myelo: myelocytes; Band: band-form granulocytes; Eos: eosinophils; Mono: monocytes\n",
      "    unique: PMC2430055/PMC2430055.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Blast/Pro (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Myelo(%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Band(%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Eos (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Mono (%)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Wild type VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">73.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">75.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.8</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Wild type RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">51.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.9</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">p15KO VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">47.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">30.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">36.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">39.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">p15KO RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">14.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">58.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">36.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">32.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">47.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">p16KO VEX</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">57.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">67.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">62.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.8</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"6\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">p16KO RE</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">56.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.6</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">23.3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">38.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">34.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Blast/Pro (%)</th>\n",
       "      <th>Myelo(%)</th>\n",
       "      <th>Band(%)</th>\n",
       "      <th>Eos (%)</th>\n",
       "      <th>Mono (%)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Wild type VEX</td>\n",
       "      <td>8.9</td>\n",
       "      <td>15.0</td>\n",
       "      <td>73.7</td>\n",
       "      <td>0.4</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>10.7</td>\n",
       "      <td>12.0</td>\n",
       "      <td>75.7</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Wild type RE</td>\n",
       "      <td>31.5</td>\n",
       "      <td>11.7</td>\n",
       "      <td>51.7</td>\n",
       "      <td>2.3</td>\n",
       "      <td>2.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>35.4</td>\n",
       "      <td>13.0</td>\n",
       "      <td>46.3</td>\n",
       "      <td>0.4</td>\n",
       "      <td>4.9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>35.0</td>\n",
       "      <td>15.4</td>\n",
       "      <td>46.2</td>\n",
       "      <td>1.1</td>\n",
       "      <td>1.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>p15KO VEX</td>\n",
       "      <td>33.8</td>\n",
       "      <td>12.1</td>\n",
       "      <td>47.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>6.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NaN</td>\n",
       "      <td>27.5</td>\n",
       "      <td>35.8</td>\n",
       "      <td>30.0</td>\n",
       "      <td>2.5</td>\n",
       "      <td>4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>36.0</td>\n",
       "      <td>24.2</td>\n",
       "      <td>39.3</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>p15KO RE</td>\n",
       "      <td>14.6</td>\n",
       "      <td>24.5</td>\n",
       "      <td>58.8</td>\n",
       "      <td>0.3</td>\n",
       "      <td>1.9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>NaN</td>\n",
       "      <td>25.4</td>\n",
       "      <td>35.7</td>\n",
       "      <td>36.4</td>\n",
       "      <td>0.4</td>\n",
       "      <td>2.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>32.2</td>\n",
       "      <td>12.4</td>\n",
       "      <td>47.2</td>\n",
       "      <td>1.9</td>\n",
       "      <td>6.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>p16KO VEX</td>\n",
       "      <td>15.0</td>\n",
       "      <td>25.0</td>\n",
       "      <td>57.0</td>\n",
       "      <td>2.3</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>NaN</td>\n",
       "      <td>13.4</td>\n",
       "      <td>9.1</td>\n",
       "      <td>67.2</td>\n",
       "      <td>9.1</td>\n",
       "      <td>1.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>18.4</td>\n",
       "      <td>10.4</td>\n",
       "      <td>62.0</td>\n",
       "      <td>4.4</td>\n",
       "      <td>4.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>p16KO RE</td>\n",
       "      <td>56.8</td>\n",
       "      <td>15.7</td>\n",
       "      <td>15.3</td>\n",
       "      <td>4.7</td>\n",
       "      <td>7.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>NaN</td>\n",
       "      <td>45.7</td>\n",
       "      <td>23.3</td>\n",
       "      <td>25.7</td>\n",
       "      <td>2.7</td>\n",
       "      <td>2.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>NaN</td>\n",
       "      <td>38.1</td>\n",
       "      <td>24.6</td>\n",
       "      <td>34.7</td>\n",
       "      <td>2.1</td>\n",
       "      <td>0.5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Unnamed: 0  Blast/Pro (%)  Myelo(%)  Band(%)  Eos (%)  Mono (%)\n",
       "0   Wild type VEX            8.9      15.0     73.7      0.4       2.0\n",
       "1             NaN           10.7      12.0     75.7      0.8       0.8\n",
       "2    Wild type RE           31.5      11.7     51.7      2.3       2.7\n",
       "3             NaN           35.4      13.0     46.3      0.4       4.9\n",
       "4             NaN           35.0      15.4     46.2      1.1       1.3\n",
       "5       p15KO VEX           33.8      12.1     47.0      1.0       6.1\n",
       "6             NaN           27.5      35.8     30.0      2.5       4.2\n",
       "7             NaN           36.0      24.2     39.3      0.0       0.6\n",
       "8        p15KO RE           14.6      24.5     58.8      0.3       1.9\n",
       "9             NaN           25.4      35.7     36.4      0.4       2.1\n",
       "10            NaN           32.2      12.4     47.2      1.9       6.2\n",
       "11      p16KO VEX           15.0      25.0     57.0      2.3       1.0\n",
       "12            NaN           13.4       9.1     67.2      9.1       1.2\n",
       "13            NaN           18.4      10.4     62.0      4.4       4.8\n",
       "14       p16KO RE           56.8      15.7     15.3      4.7       7.6\n",
       "15            NaN           45.7      23.3     25.7      2.7       2.7\n",
       "16            NaN           38.1      24.6     34.7      2.1       0.5"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2430763.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2435206.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2435220.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Counts of edge classification results\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">b-Statistic\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Polygon Womble\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">a</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">b</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">c</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">d</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">a</th>\n",
       "              <th align=\"right\" colspan=\"1\" rowspan=\"1\">b</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">c</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">d</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Model</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">36</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">117</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">36</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">117</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Simulation</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">21</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">15</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">113</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">12</td>\n",
       "              <td align=\"right\" colspan=\"1\" rowspan=\"1\">24</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">117</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_0</th>\n",
       "      <th>b-Statistic</th>\n",
       "      <th>Polygon Womble</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "      <th>d</th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "      <th>d</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Model</td>\n",
       "      <td>36</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>117</td>\n",
       "      <td>36</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Simulation</td>\n",
       "      <td>21</td>\n",
       "      <td>15</td>\n",
       "      <td>4</td>\n",
       "      <td>113</td>\n",
       "      <td>12</td>\n",
       "      <td>24</td>\n",
       "      <td>0</td>\n",
       "      <td>117</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Unnamed: 0_level_0 b-Statistic Polygon Womble Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0\n",
       "  Unnamed: 0_level_1           a              b                  c                  d                  a                  b                  c                  d\n",
       "0              Model          36              0                  0                117                 36                  0                  0                117\n",
       "1         Simulation          21             15                  4                113                 12                 24                  0                117"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Accuracy and error statistics for boundary classification using the little b and polygon wombling approaches\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">b-Statistic\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Polygon Womble\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Specificity</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type I</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type II</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Power</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Specificity</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type I</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type II</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Power</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Model</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Simulation</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.583</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.417</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.034</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.966</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.333</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.667</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_0</th>\n",
       "      <th>b-Statistic</th>\n",
       "      <th>Polygon Womble</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>Specificity</th>\n",
       "      <th>Type I</th>\n",
       "      <th>Type II</th>\n",
       "      <th>Power</th>\n",
       "      <th>Specificity</th>\n",
       "      <th>Type I</th>\n",
       "      <th>Type II</th>\n",
       "      <th>Power</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Model</td>\n",
       "      <td>1.000</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.000</td>\n",
       "      <td>1.000</td>\n",
       "      <td>0.000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Simulation</td>\n",
       "      <td>0.583</td>\n",
       "      <td>0.417</td>\n",
       "      <td>0.034</td>\n",
       "      <td>0.966</td>\n",
       "      <td>0.333</td>\n",
       "      <td>0.667</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Unnamed: 0_level_0 b-Statistic Polygon Womble Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0\n",
       "  Unnamed: 0_level_1 Specificity         Type I            Type II              Power        Specificity             Type I            Type II              Power\n",
       "0              Model       1.000          0.000              0.000              1.000              1.000              0.000                0.0                1.0\n",
       "1         Simulation       0.583          0.417              0.034              0.966              0.333              0.667                0.0                1.0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Counts from classification table for Big B, Ladder and Local Moran methods\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Big B\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Ladder\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Local Moran\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">a</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">b</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">c</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">d</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">a</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">b</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">c</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">d</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">a</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">b</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">c</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">d</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Model</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">70</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">10</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">76</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">2</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">8</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">69</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Simulation</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">71</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">10</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">72</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">70</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_0</th>\n",
       "      <th>Big B</th>\n",
       "      <th>Ladder</th>\n",
       "      <th>Local Moran</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "      <th>Unnamed: 11_level_0</th>\n",
       "      <th>Unnamed: 12_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "      <th>d</th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "      <th>d</th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "      <th>d</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Model</td>\n",
       "      <td>5</td>\n",
       "      <td>5</td>\n",
       "      <td>6</td>\n",
       "      <td>70</td>\n",
       "      <td>10</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>76</td>\n",
       "      <td>2</td>\n",
       "      <td>8</td>\n",
       "      <td>7</td>\n",
       "      <td>69</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Simulation</td>\n",
       "      <td>4</td>\n",
       "      <td>6</td>\n",
       "      <td>5</td>\n",
       "      <td>71</td>\n",
       "      <td>10</td>\n",
       "      <td>0</td>\n",
       "      <td>4</td>\n",
       "      <td>72</td>\n",
       "      <td>4</td>\n",
       "      <td>6</td>\n",
       "      <td>6</td>\n",
       "      <td>70</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Unnamed: 0_level_0 Big B Ladder Local Moran Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0 Unnamed: 10_level_0 Unnamed: 11_level_0 Unnamed: 12_level_0\n",
       "  Unnamed: 0_level_1     a      b           c                  d                  a                  b                  c                  d                  a                   b                   c                   d\n",
       "0              Model     5      5           6                 70                 10                  0                  0                 76                  2                   8                   7                  69\n",
       "1         Simulation     4      6           5                 71                 10                  0                  4                 72                  4                   6                   6                  70"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T4\n",
      "  label_text: Table 4\n",
      "  caption_text: Specificity, Type I and Type II error, and sensitivity of the Big B, Ladder and Local Moran methods\n",
      "  footer_text: \n",
      "    unique: PMC2435220/PMC2435220.xml::T4-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Big B\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Ladder\n",
       "<hr /></th>\n",
       "              <th align=\"left\" colspan=\"4\" rowspan=\"1\">Local Moran\n",
       "<hr /></th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Specificity</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type I</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type II</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Power</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Specificity</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type I</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type II</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Power</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Specificity</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type I</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Type II</th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">Power</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Model</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.500</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.500</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.079</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.921</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.200</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.800</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.092</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.908</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Simulation</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.400</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.600</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.066</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.934</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.000</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.053</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.947</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.400</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.600</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.079</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.921</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_0</th>\n",
       "      <th>Big B</th>\n",
       "      <th>Ladder</th>\n",
       "      <th>Local Moran</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "      <th>Unnamed: 9_level_0</th>\n",
       "      <th>Unnamed: 10_level_0</th>\n",
       "      <th>Unnamed: 11_level_0</th>\n",
       "      <th>Unnamed: 12_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0_level_1</th>\n",
       "      <th>Specificity</th>\n",
       "      <th>Type I</th>\n",
       "      <th>Type II</th>\n",
       "      <th>Power</th>\n",
       "      <th>Specificity</th>\n",
       "      <th>Type I</th>\n",
       "      <th>Type II</th>\n",
       "      <th>Power</th>\n",
       "      <th>Specificity</th>\n",
       "      <th>Type I</th>\n",
       "      <th>Type II</th>\n",
       "      <th>Power</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Model</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.079</td>\n",
       "      <td>0.921</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000</td>\n",
       "      <td>1.000</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.092</td>\n",
       "      <td>0.908</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Simulation</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.066</td>\n",
       "      <td>0.934</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.053</td>\n",
       "      <td>0.947</td>\n",
       "      <td>0.4</td>\n",
       "      <td>0.6</td>\n",
       "      <td>0.079</td>\n",
       "      <td>0.921</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Unnamed: 0_level_0       Big B Ladder Local Moran Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0 Unnamed: 9_level_0 Unnamed: 10_level_0 Unnamed: 11_level_0 Unnamed: 12_level_0\n",
       "  Unnamed: 0_level_1 Specificity Type I     Type II              Power        Specificity             Type I            Type II              Power        Specificity              Type I             Type II               Power\n",
       "0              Model         0.5    0.5       0.079              0.921                1.0                0.0              0.000              1.000                0.2                 0.8               0.092               0.908\n",
       "1         Simulation         0.4    0.6       0.066              0.934                1.0                0.0              0.053              0.947                0.4                 0.6               0.079               0.921"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2435384.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Average ± SD values (%) of degree of substitution (DS) and purity of the MPS-peptide-dextran conjugates (n = 3).\n",
      "  footer_text: \n",
      "    unique: PMC2435384/PMC2435384.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Conjugate</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">DS (%)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Purity (%)</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.4 &#177; 0.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">91</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.0 &#177; 0.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">91</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.5 &#177; 0.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">95</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.4 &#177; 0.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">90</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.9 &#177; 0.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">93</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Conjugate</th>\n",
       "      <th>DS (%)</th>\n",
       "      <th>Purity (%)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MPS-mG-Dex</td>\n",
       "      <td>9.4 ± 0.2</td>\n",
       "      <td>91</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MPS-mGG-Dex</td>\n",
       "      <td>7.0 ± 0.2</td>\n",
       "      <td>91</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MPS-mGGG-Dex</td>\n",
       "      <td>9.5 ± 0.9</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MPS-mGGGG-Dex</td>\n",
       "      <td>7.4 ± 0.1</td>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MPS-mGGGGG-Dex</td>\n",
       "      <td>6.9 ± 0.1</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        Conjugate     DS (%)  Purity (%)\n",
       "0      MPS-mG-Dex  9.4 ± 0.2          91\n",
       "1     MPS-mGG-Dex  7.0 ± 0.2          91\n",
       "2    MPS-mGGG-Dex  9.5 ± 0.9          95\n",
       "3   MPS-mGGGG-Dex  7.4 ± 0.1          90\n",
       "4  MPS-mGGGGG-Dex  6.9 ± 0.1          93"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Inter-run accuracy (%Error) and precision (%CV) of the reversed-phase assay (n =5)\n",
      "  footer_text: \n",
      "    unique: PMC2435384/PMC2435384.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Analyte</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Added Conc. (&#956;g/mL)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Calculated Conc. (&#956;g/mL)</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% Error</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">% CV</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MP</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.50</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.49</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;2.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.53</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;3.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.5</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;4.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.44</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;12.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;13.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.8</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MP</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.06</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;3.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.4</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.79</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;4.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.6</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.81</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;3.8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.0</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;3.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.82</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;3.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.87</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">&#8722;2.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.3</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MP</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">108</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.6</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">107</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.7</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">108</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">104</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.1</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">103</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.2</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">102</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.0</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.8</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGGG-OH</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">100</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">101</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3.5</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Analyte</th>\n",
       "      <th>Added Conc. (μg/mL)</th>\n",
       "      <th>Calculated Conc. (μg/mL)</th>\n",
       "      <th>% Error</th>\n",
       "      <th>% CV</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MP</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.50</td>\n",
       "      <td>0.5</td>\n",
       "      <td>6.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MPS</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.49</td>\n",
       "      <td>−2.9</td>\n",
       "      <td>3.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MPS-mG-OH</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.53</td>\n",
       "      <td>6.9</td>\n",
       "      <td>5.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MPS-mGG-OH</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.48</td>\n",
       "      <td>−3.6</td>\n",
       "      <td>3.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MPS-mGGG-OH</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.48</td>\n",
       "      <td>−4.7</td>\n",
       "      <td>8.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>MPS-mGGGG-OH</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.44</td>\n",
       "      <td>−12.8</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>MPS-mGGGGG-OH</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.43</td>\n",
       "      <td>−13.2</td>\n",
       "      <td>9.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>MP</td>\n",
       "      <td>5.0</td>\n",
       "      <td>5.06</td>\n",
       "      <td>1.1</td>\n",
       "      <td>1.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>MPS</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.82</td>\n",
       "      <td>−3.6</td>\n",
       "      <td>2.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>MPS-mG-OH</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.79</td>\n",
       "      <td>−4.2</td>\n",
       "      <td>1.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>MPS-mGG-OH</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.81</td>\n",
       "      <td>−3.8</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>MPS-mGGG-OH</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.82</td>\n",
       "      <td>−3.6</td>\n",
       "      <td>2.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>MPS-mGGGG-OH</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.82</td>\n",
       "      <td>−3.6</td>\n",
       "      <td>0.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>MPS-mGGGGG-OH</td>\n",
       "      <td>5.0</td>\n",
       "      <td>4.87</td>\n",
       "      <td>−2.6</td>\n",
       "      <td>1.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>MP</td>\n",
       "      <td>100.0</td>\n",
       "      <td>108.00</td>\n",
       "      <td>7.6</td>\n",
       "      <td>1.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>MPS</td>\n",
       "      <td>100.0</td>\n",
       "      <td>107.00</td>\n",
       "      <td>7.2</td>\n",
       "      <td>1.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>MPS-mG-OH</td>\n",
       "      <td>100.0</td>\n",
       "      <td>108.00</td>\n",
       "      <td>7.7</td>\n",
       "      <td>3.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>MPS-mGG-OH</td>\n",
       "      <td>100.0</td>\n",
       "      <td>104.00</td>\n",
       "      <td>3.9</td>\n",
       "      <td>3.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>MPS-mGGG-OH</td>\n",
       "      <td>100.0</td>\n",
       "      <td>103.00</td>\n",
       "      <td>3.2</td>\n",
       "      <td>3.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>MPS-mGGGG-OH</td>\n",
       "      <td>100.0</td>\n",
       "      <td>102.00</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>MPS-mGGGGG-OH</td>\n",
       "      <td>100.0</td>\n",
       "      <td>101.00</td>\n",
       "      <td>0.6</td>\n",
       "      <td>3.5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          Analyte  Added Conc. (μg/mL)  Calculated Conc. (μg/mL) % Error  % CV\n",
       "0              MP                  0.5                      0.50     0.5   6.3\n",
       "1             MPS                  0.5                      0.49    −2.9   3.1\n",
       "2       MPS-mG-OH                  0.5                      0.53     6.9   5.5\n",
       "3      MPS-mGG-OH                  0.5                      0.48    −3.6   3.5\n",
       "4     MPS-mGGG-OH                  0.5                      0.48    −4.7   8.7\n",
       "5    MPS-mGGGG-OH                  0.5                      0.44   −12.8  12.0\n",
       "6   MPS-mGGGGG-OH                  0.5                      0.43   −13.2   9.8\n",
       "7              MP                  5.0                      5.06     1.1   1.2\n",
       "8             MPS                  5.0                      4.82    −3.6   2.4\n",
       "9       MPS-mG-OH                  5.0                      4.79    −4.2   1.6\n",
       "10     MPS-mGG-OH                  5.0                      4.81    −3.8   1.0\n",
       "11    MPS-mGGG-OH                  5.0                      4.82    −3.6   2.2\n",
       "12   MPS-mGGGG-OH                  5.0                      4.82    −3.6   0.7\n",
       "13  MPS-mGGGGG-OH                  5.0                      4.87    −2.6   1.3\n",
       "14             MP                100.0                    108.00     7.6   1.6\n",
       "15            MPS                100.0                    107.00     7.2   1.7\n",
       "16      MPS-mG-OH                100.0                    108.00     7.7   3.2\n",
       "17     MPS-mGG-OH                100.0                    104.00     3.9   3.1\n",
       "18    MPS-mGGG-OH                100.0                    103.00     3.2   3.2\n",
       "19   MPS-mGGGG-OH                100.0                    102.00     2.0   2.8\n",
       "20  MPS-mGGGGG-OH                100.0                    101.00     0.6   3.5"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T3\n",
      "  label_text: Table 3\n",
      "  caption_text: Mean ± SD values of degradation half lives (h) of DMP conjugates in pH 7.4 buffer and rat blood (n = 3).\n",
      "  footer_text: a–j Means with similar letters are significantly different each other: ANOVA with post-hoc Scheffe test..\n",
      "    unique: PMC2435384/PMC2435384.xml::T3-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">pH 7.4</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Blood</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.7 &#177; 0.1<sup>a,b,c,d,e</sup></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24.5 &#177; 3.0<sup>e,g,h,i</sup></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.67 &#177; 0.03<xref ref-type=\"table-fn\" rid=\"TFN1\">a</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8.36 &#177; 0.45<xref ref-type=\"table-fn\" rid=\"TFN1\">g</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.87 &#177; 0.03<sup>b,f</sup></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9.80 &#177; 0.25<sup>f,h,j</sup></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.69 &#177; 0.03<xref ref-type=\"table-fn\" rid=\"TFN1\">c</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.89 &#177; 0.25<xref ref-type=\"table-fn\" rid=\"TFN1\">i</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">MPS-mGGGGG-Dex</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.27 &#177; 0.02<xref ref-type=\"table-fn\" rid=\"TFN1\">d</xref></td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.67 &#177; 0.21<sup>g,j</sup></td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th>pH 7.4</th>\n",
       "      <th>Blood</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MPS-mG-Dex</td>\n",
       "      <td>10.7 ± 0.1a,b,c,d,e</td>\n",
       "      <td>24.5 ± 3.0e,g,h,i</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MPS-mGG-Dex</td>\n",
       "      <td>5.67 ± 0.03a</td>\n",
       "      <td>8.36 ± 0.45g</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MPS-mGGG-Dex</td>\n",
       "      <td>5.87 ± 0.03b,f</td>\n",
       "      <td>9.80 ± 0.25f,h,j</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MPS-mGGGG-Dex</td>\n",
       "      <td>4.69 ± 0.03c</td>\n",
       "      <td>6.89 ± 0.25i</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MPS-mGGGGG-Dex</td>\n",
       "      <td>4.27 ± 0.02d</td>\n",
       "      <td>4.67 ± 0.21g,j</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  \\n                             pH 7.4              Blood\n",
       "0       MPS-mG-Dex  10.7 ± 0.1a,b,c,d,e  24.5 ± 3.0e,g,h,i\n",
       "1      MPS-mGG-Dex         5.67 ± 0.03a       8.36 ± 0.45g\n",
       "2     MPS-mGGG-Dex       5.87 ± 0.03b,f   9.80 ± 0.25f,h,j\n",
       "3    MPS-mGGGG-Dex         4.69 ± 0.03c       6.89 ± 0.25i\n",
       "4   MPS-mGGGGG-Dex         4.27 ± 0.02d     4.67 ± 0.21g,j"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2435391.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Human IBD patient description\n",
      "  footer_text: NOTE. CD; Crohn’s disease, UC; ulcerative colitis, Sm Intest; small intestine.\n",
      "    unique: PMC2435391/PMC2435391.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"void\" rules=\"none\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "                <hr />\n",
       "              </th>\n",
       "              <th align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "                <hr />\n",
       "              </th>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Patient #</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Sex</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Age</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Diagnosis</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Tissue</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Patient #</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Sex</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Age</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Diagnosis</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Tissue</th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "              <td align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">78</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">25</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">47</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Sm Intest</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">39</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">28</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">37</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">29</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">59</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">61</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Normal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">72</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">56</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">32</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">54</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Rectum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">9</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">72</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">51</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">34</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">36</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">56</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">12</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">37</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">36</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Rectum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">13</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">37</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">14</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">65</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">38</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">15</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">39</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">16</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Cecum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">40</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">68</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Rectum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">17</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Sm Intest</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Rectum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">18</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">64</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">42</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">60</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">19</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">31</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">43</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">33</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">44</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">27</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">20</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Ileum</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">45</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">57</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Rectum</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">22</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">46</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">35</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">23</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">47</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">30</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">24</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Male</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">41</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">CD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">48</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Female</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">26</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">UC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Colon</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "              <td align=\"center\" colspan=\"5\" rowspan=\"1\">\n",
       "<hr /></td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Patient #</th>\n",
       "      <th>Sex</th>\n",
       "      <th>Age</th>\n",
       "      <th>Diagnosis</th>\n",
       "      <th>Tissue</th>\n",
       "      <th>Patient #.1</th>\n",
       "      <th>Sex.1</th>\n",
       "      <th>Age.1</th>\n",
       "      <th>Diagnosis.1</th>\n",
       "      <th>Tissue.1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Male</td>\n",
       "      <td>78</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Colon</td>\n",
       "      <td>25</td>\n",
       "      <td>Male</td>\n",
       "      <td>29</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Male</td>\n",
       "      <td>47</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Colon</td>\n",
       "      <td>26</td>\n",
       "      <td>Male</td>\n",
       "      <td>42</td>\n",
       "      <td>CD</td>\n",
       "      <td>Sm Intest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3</td>\n",
       "      <td>Female</td>\n",
       "      <td>31</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Colon</td>\n",
       "      <td>27</td>\n",
       "      <td>Female</td>\n",
       "      <td>39</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>4</td>\n",
       "      <td>Male</td>\n",
       "      <td>54</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Colon</td>\n",
       "      <td>28</td>\n",
       "      <td>Male</td>\n",
       "      <td>54</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5</td>\n",
       "      <td>Male</td>\n",
       "      <td>37</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>29</td>\n",
       "      <td>Male</td>\n",
       "      <td>59</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>6</td>\n",
       "      <td>Female</td>\n",
       "      <td>61</td>\n",
       "      <td>Normal</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>30</td>\n",
       "      <td>Male</td>\n",
       "      <td>72</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>7</td>\n",
       "      <td>Male</td>\n",
       "      <td>56</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>31</td>\n",
       "      <td>Male</td>\n",
       "      <td>41</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>8</td>\n",
       "      <td>Male</td>\n",
       "      <td>33</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>32</td>\n",
       "      <td>Female</td>\n",
       "      <td>54</td>\n",
       "      <td>UC</td>\n",
       "      <td>Rectum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>9</td>\n",
       "      <td>Female</td>\n",
       "      <td>46</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>33</td>\n",
       "      <td>Female</td>\n",
       "      <td>72</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10</td>\n",
       "      <td>Female</td>\n",
       "      <td>51</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>34</td>\n",
       "      <td>Female</td>\n",
       "      <td>36</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>11</td>\n",
       "      <td>Female</td>\n",
       "      <td>31</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>35</td>\n",
       "      <td>Male</td>\n",
       "      <td>56</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>12</td>\n",
       "      <td>Male</td>\n",
       "      <td>37</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>36</td>\n",
       "      <td>Male</td>\n",
       "      <td>31</td>\n",
       "      <td>UC</td>\n",
       "      <td>Rectum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>13</td>\n",
       "      <td>Male</td>\n",
       "      <td>48</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>37</td>\n",
       "      <td>Male</td>\n",
       "      <td>23</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>14</td>\n",
       "      <td>Female</td>\n",
       "      <td>65</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>38</td>\n",
       "      <td>Male</td>\n",
       "      <td>42</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>15</td>\n",
       "      <td>Female</td>\n",
       "      <td>26</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>39</td>\n",
       "      <td>Male</td>\n",
       "      <td>42</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>16</td>\n",
       "      <td>Male</td>\n",
       "      <td>33</td>\n",
       "      <td>CD</td>\n",
       "      <td>Cecum</td>\n",
       "      <td>40</td>\n",
       "      <td>Male</td>\n",
       "      <td>68</td>\n",
       "      <td>UC</td>\n",
       "      <td>Rectum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>17</td>\n",
       "      <td>Female</td>\n",
       "      <td>45</td>\n",
       "      <td>CD</td>\n",
       "      <td>Sm Intest</td>\n",
       "      <td>41</td>\n",
       "      <td>Male</td>\n",
       "      <td>45</td>\n",
       "      <td>UC</td>\n",
       "      <td>Rectum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>18</td>\n",
       "      <td>Male</td>\n",
       "      <td>64</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>42</td>\n",
       "      <td>Female</td>\n",
       "      <td>60</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>19</td>\n",
       "      <td>Female</td>\n",
       "      <td>31</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>43</td>\n",
       "      <td>Male</td>\n",
       "      <td>33</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>20</td>\n",
       "      <td>Male</td>\n",
       "      <td>35</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>44</td>\n",
       "      <td>Female</td>\n",
       "      <td>27</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>21</td>\n",
       "      <td>Male</td>\n",
       "      <td>20</td>\n",
       "      <td>CD</td>\n",
       "      <td>Ileum</td>\n",
       "      <td>45</td>\n",
       "      <td>Male</td>\n",
       "      <td>57</td>\n",
       "      <td>UC</td>\n",
       "      <td>Rectum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>22</td>\n",
       "      <td>Male</td>\n",
       "      <td>41</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>46</td>\n",
       "      <td>Female</td>\n",
       "      <td>35</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>23</td>\n",
       "      <td>Female</td>\n",
       "      <td>21</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>47</td>\n",
       "      <td>Female</td>\n",
       "      <td>30</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>24</td>\n",
       "      <td>Male</td>\n",
       "      <td>41</td>\n",
       "      <td>CD</td>\n",
       "      <td>Colon</td>\n",
       "      <td>48</td>\n",
       "      <td>Female</td>\n",
       "      <td>26</td>\n",
       "      <td>UC</td>\n",
       "      <td>Colon</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Patient #     Sex  Age Diagnosis     Tissue  Patient #.1   Sex.1  Age.1 Diagnosis.1   Tissue.1\n",
       "0           1    Male   78    Normal      Colon           25    Male     29          CD      Ileum\n",
       "1           2    Male   47    Normal      Colon           26    Male     42          CD  Sm Intest\n",
       "2           3  Female   31    Normal      Colon           27  Female     39          CD      Ileum\n",
       "3           4    Male   54    Normal      Colon           28    Male     54          UC      Colon\n",
       "4           5    Male   37    Normal      Ileum           29    Male     59          UC      Colon\n",
       "5           6  Female   61    Normal      Ileum           30    Male     72          UC      Colon\n",
       "6           7    Male   56        CD      Colon           31    Male     41          UC      Colon\n",
       "7           8    Male   33        CD      Colon           32  Female     54          UC     Rectum\n",
       "8           9  Female   46        CD      Colon           33  Female     72          UC      Colon\n",
       "9          10  Female   51        CD      Colon           34  Female     36          UC      Colon\n",
       "10         11  Female   31        CD      Ileum           35    Male     56          UC      Colon\n",
       "11         12    Male   37        CD      Ileum           36    Male     31          UC     Rectum\n",
       "12         13    Male   48        CD      Ileum           37    Male     23          UC      Colon\n",
       "13         14  Female   65        CD      Ileum           38    Male     42          UC      Colon\n",
       "14         15  Female   26        CD      Ileum           39    Male     42          UC      Colon\n",
       "15         16    Male   33        CD      Cecum           40    Male     68          UC     Rectum\n",
       "16         17  Female   45        CD  Sm Intest           41    Male     45          UC     Rectum\n",
       "17         18    Male   64        CD      Ileum           42  Female     60          UC      Colon\n",
       "18         19  Female   31        CD      Ileum           43    Male     33          UC      Colon\n",
       "19         20    Male   35        CD      Colon           44  Female     27          UC      Colon\n",
       "20         21    Male   20        CD      Ileum           45    Male     57          UC     Rectum\n",
       "21         22    Male   41        CD      Colon           46  Female     35          UC      Colon\n",
       "22         23  Female   21        CD      Colon           47  Female     30          UC      Colon\n",
       "23         24    Male   41        CD      Colon           48  Female     26          UC      Colon"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  ----------------------------------------\n",
      "  id: T2\n",
      "  label_text: Table 2\n",
      "  caption_text: Gene expression analysis in the colon of Hnf4αF/F and Hnf4αΔIEpC mice\n",
      "  footer_text: NOTE. Quantitative real-time PCR of colon mRNA from 2-month-old mice (n=6). All the values are means ± SD. Significant difference compared to Hnf4αF/F mice: * , p< 0.05 ** , p< 0.01 *** , p< 0.001.\n",
      "    unique: PMC2435391/PMC2435391.xml::T2-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <thead>\n",
       "            <tr>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">Gene</th>\n",
       "              <th align=\"center\" colspan=\"1\" rowspan=\"1\">\n",
       "                <italic>Hnf4&#945;</italic>\n",
       "                <sup>F/F</sup>\n",
       "              </th>\n",
       "              <th align=\"left\" colspan=\"1\" rowspan=\"1\">\n",
       "                <italic>Hnf4&#945;</italic>\n",
       "                <sup>&#916;IEpC</sup>\n",
       "              </th>\n",
       "            </tr>\n",
       "          </thead>\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">HNF4&#945;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.16</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.01 &#177; 0.00<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.34</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">7.54 &#177; 1.42<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc2</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.49</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.88 &#177; 0.41</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc3</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.38</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.07 &#177; 0.05<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.34</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.64 &#177; 0.47<xref ref-type=\"table-fn\" rid=\"TFN3\">*</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc5AC</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.39</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.74 &#177; 0.46<xref ref-type=\"table-fn\" rid=\"TFN3\">*</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc5B</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.35</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.77 &#177; 0.51<xref ref-type=\"table-fn\" rid=\"TFN3\">*</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Muc6</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.36</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.54 &#177; 0.39<xref ref-type=\"table-fn\" rid=\"TFN3\">*</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Aqp1</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.43</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.32 &#177; 0.10<xref ref-type=\"table-fn\" rid=\"TFN4\">**</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Aqp4</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.33</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">1.50 &#177; 0.39<xref ref-type=\"table-fn\" rid=\"TFN3\">*</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Aqp8</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.21</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.15 &#177; 0.03<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Mep1&#945;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.30</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.12 &#177; 0.05<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">Mep1&#946;</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">1.00 &#177; 0.32</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">0.10 &#177; 0.04<xref ref-type=\"table-fn\" rid=\"TFN5\">***</xref></td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Hnf4α\n",
       "                F/F</th>\n",
       "      <th>Hnf4α\n",
       "                ΔIEpC</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>HNF4α</td>\n",
       "      <td>1.00 ± 0.16</td>\n",
       "      <td>0.01 ± 0.00***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Muc1</td>\n",
       "      <td>1.00 ± 0.34</td>\n",
       "      <td>7.54 ± 1.42***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Muc2</td>\n",
       "      <td>1.00 ± 0.49</td>\n",
       "      <td>0.88 ± 0.41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Muc3</td>\n",
       "      <td>1.00 ± 0.38</td>\n",
       "      <td>0.07 ± 0.05***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Muc4</td>\n",
       "      <td>1.00 ± 0.34</td>\n",
       "      <td>1.64 ± 0.47*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Muc5AC</td>\n",
       "      <td>1.00 ± 0.39</td>\n",
       "      <td>1.74 ± 0.46*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Muc5B</td>\n",
       "      <td>1.00 ± 0.35</td>\n",
       "      <td>1.77 ± 0.51*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Muc6</td>\n",
       "      <td>1.00 ± 0.36</td>\n",
       "      <td>1.54 ± 0.39*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Aqp1</td>\n",
       "      <td>1.00 ± 0.43</td>\n",
       "      <td>0.32 ± 0.10**</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Aqp4</td>\n",
       "      <td>1.00 ± 0.33</td>\n",
       "      <td>1.50 ± 0.39*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Aqp8</td>\n",
       "      <td>1.00 ± 0.21</td>\n",
       "      <td>0.15 ± 0.03***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Mep1α</td>\n",
       "      <td>1.00 ± 0.30</td>\n",
       "      <td>0.12 ± 0.05***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Mep1β</td>\n",
       "      <td>1.00 ± 0.32</td>\n",
       "      <td>0.10 ± 0.04***</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Gene \\n                Hnf4α\\n                F/F\\n               \\n                Hnf4α\\n                ΔIEpC\\n              \n",
       "0    HNF4α                                                  1.00 ± 0.16                                                 0.01 ± 0.00***\n",
       "1     Muc1                                                  1.00 ± 0.34                                                 7.54 ± 1.42***\n",
       "2     Muc2                                                  1.00 ± 0.49                                                    0.88 ± 0.41\n",
       "3     Muc3                                                  1.00 ± 0.38                                                 0.07 ± 0.05***\n",
       "4     Muc4                                                  1.00 ± 0.34                                                   1.64 ± 0.47*\n",
       "5   Muc5AC                                                  1.00 ± 0.39                                                   1.74 ± 0.46*\n",
       "6    Muc5B                                                  1.00 ± 0.35                                                   1.77 ± 0.51*\n",
       "7     Muc6                                                  1.00 ± 0.36                                                   1.54 ± 0.39*\n",
       "8     Aqp1                                                  1.00 ± 0.43                                                  0.32 ± 0.10**\n",
       "9     Aqp4                                                  1.00 ± 0.33                                                   1.50 ± 0.39*\n",
       "10    Aqp8                                                  1.00 ± 0.21                                                 0.15 ± 0.03***\n",
       "11   Mep1α                                                  1.00 ± 0.30                                                 0.12 ± 0.05***\n",
       "12   Mep1β                                                  1.00 ± 0.32                                                 0.10 ± 0.04***"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMC0024XXXXX/PMC2435511.xml\n",
      "\n",
      "PMC0024XXXXX/PMC2440269.xml\n",
      "  ----------------------------------------\n",
      "  id: T1\n",
      "  label_text: Table 1\n",
      "  caption_text: Percent of Potentially Eligible Subjects Rejected by Reason for Exclusion\n",
      "  footer_text: \n",
      "    unique: PMC2440269/PMC2440269.xml::T1-0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table frame=\"box\" rules=\"all\">\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">\n",
       "                <bold>Modifiable Reasons</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Lack of Consent</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">21.4%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">&#8195;Patient/Surrogate Refusal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">10.9%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">&#8195;Surrogate Non-available</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.0%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Enrollment Window</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">&#8195;Original</td>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Post -Revision</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">6.6%/2.3%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Delayed Notification of Study Personnel</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">4.4%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">MD Refusal</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.7%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Other Study</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">2.3%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">\n",
       "                <bold>Non-Modifiable Reasons</bold>\n",
       "              </td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">CMO/Moribund</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">11.8%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Chronic illness</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.8%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">CKD</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">7.6%</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"2\" rowspan=\"1\">Obesity</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">5.9%</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Modifiable Reasons</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Lack of Consent</td>\n",
       "      <td>21.4%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Patient/Surrogate Refusal</td>\n",
       "      <td>10.9%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Surrogate Non-available</td>\n",
       "      <td>7.0%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Enrollment Window</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Original</td>\n",
       "      <td>Post -Revision</td>\n",
       "      <td>6.6%/2.3%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Delayed Notification of Study Personnel</td>\n",
       "      <td>4.4%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>MD Refusal</td>\n",
       "      <td>2.7%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Other Study</td>\n",
       "      <td>2.3%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Non-Modifiable Reasons</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>CMO/Moribund</td>\n",
       "      <td>11.8%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Chronic illness</td>\n",
       "      <td>7.8%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>CKD</td>\n",
       "      <td>7.6%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Obesity</td>\n",
       "      <td>5.9%</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          0               1          2\n",
       "0                        Modifiable Reasons             NaN        NaN\n",
       "1                           Lack of Consent           21.4%        NaN\n",
       "2                 Patient/Surrogate Refusal           10.9%        NaN\n",
       "3                   Surrogate Non-available            7.0%        NaN\n",
       "4                         Enrollment Window             NaN        NaN\n",
       "5                                  Original  Post -Revision  6.6%/2.3%\n",
       "6   Delayed Notification of Study Personnel            4.4%        NaN\n",
       "7                                MD Refusal            2.7%        NaN\n",
       "8                               Other Study            2.3%        NaN\n",
       "9                    Non-Modifiable Reasons             NaN        NaN\n",
       "10                             CMO/Moribund           11.8%        NaN\n",
       "11                          Chronic illness            7.8%        NaN\n",
       "12                                      CKD            7.6%        NaN\n",
       "13                                  Obesity            5.9%        NaN"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import re\n",
    "\n",
    "count = 0\n",
    "\n",
    "\n",
    "def caption_to_string(element):\n",
    "    if element is None:\n",
    "        return \"\"\n",
    "    element_text = ET.tostring(element, encoding='utf8', method='text').decode('utf-8')\n",
    "    element_text = element_text.strip().replace('\\n', ' ')\n",
    "    element_text = re.sub(' +', ' ', element_text)\n",
    "    return element_text\n",
    "\n",
    "\n",
    "for i, row in enumerate(manuscript_files.itertuples()):\n",
    "    print('\\n' + f\"{row.File}\")\n",
    "\n",
    "    results = {}\n",
    "    \n",
    "    tree = ET.parse(MANUSCRIPT_DATA_DIR.joinpath(row.File))\n",
    "    tables = tree.findall('.//table')\n",
    "    table_wraps = tree.findall('.//table-wrap')\n",
    "    \n",
    "    for table_wrap in table_wraps:\n",
    "        print(\"  \" + \"-\" * 40)\n",
    "        id_ = table_wrap.get('id')\n",
    "        assert id_ is not None\n",
    "        print(f\"  id: {id_}\")\n",
    "        # Attrib\n",
    "        attrib_set = set(table_wrap.attrib)\n",
    "        if attrib_set - {'id', 'position'}:\n",
    "            print(f\"tw({id_}): {attrib_set}\")\n",
    "        # Label\n",
    "        label = table_wrap.find('label')\n",
    "        label_text = label.text if label is not None else \"\"\n",
    "        print(f\"  label_text: {label_text}\")\n",
    "        # Caption\n",
    "        caption = table_wrap.find('caption')\n",
    "        caption_text = caption_to_string(caption)\n",
    "        print(f\"  caption_text: {caption_text}\")\n",
    "        # Footer\n",
    "        footer = table_wrap.find('table-wrap-foot')\n",
    "        footer_text = caption_to_string(footer)\n",
    "        print(f\"  footer_text: {footer_text}\")\n",
    "        # Other children\n",
    "        other_children = [\n",
    "            c.tag for c in table_wrap.getchildren() \n",
    "            if c.tag not in ['label', 'caption', 'table', 'table-wrap-foot']\n",
    "        ]\n",
    "        if other_children:\n",
    "            print(other_children)\n",
    "        \n",
    "        # Tables\n",
    "        tables = table_wrap.findall('table')\n",
    "        for i, table in enumerate(tables):\n",
    "            unique = f\"{row.PMCID}/{op.basename(row.File)}::{id_}-{i}\"\n",
    "            assert unique not in results\n",
    "            print(f\"    unique: {unique}\")\n",
    "            # Attrib\n",
    "            attrib_set = {a for a in table.attrib if a not in ['frame', 'rules']}\n",
    "            if attrib_set:\n",
    "                print(attrib_set)\n",
    "            # Children\n",
    "            children_set = {c.tag for c in table.getchildren() if c.tag not in ['thead', 'tbody']}\n",
    "            if children_set:\n",
    "                print(children_set)\n",
    "            # DataFrame\n",
    "            table_bytes = ET.tostring(table)\n",
    "            display(HTML(table_bytes.decode('utf-8')))\n",
    "            table_dfs = pd.read_html(table_bytes, na_values=['', '-', '–'])\n",
    "            assert len(table_dfs) == 1\n",
    "            table_df = table_dfs[0]\n",
    "            display(table_df)\n",
    "        \n",
    "    root = tree.getroot()\n",
    "    a, b = get_tables(root)\n",
    "    \n",
    "    count += 1\n",
    "    if count > 20:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 192,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "table.remove(table.find('thead'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 193,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "table_html = (ET.tostring(table).decode('utf-8'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table frame=\"hsides\" rules=\"groups\">\n",
       "          <tbody>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Raw rates</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6984</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.6413</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.7467</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Empirical Bayes</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2584</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2252</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2916</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\" />\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">Poisson kriging</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2345</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1961</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2740</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">92</td>\n",
       "            </tr>\n",
       "            <tr>\n",
       "              <td align=\"left\" colspan=\"1\" rowspan=\"1\">E-type estimate</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2348</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.1967</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">0.2750</td>\n",
       "              <td align=\"center\" colspan=\"1\" rowspan=\"1\">92</td>\n",
       "            </tr>\n",
       "          </tbody>\n",
       "        </table>\n",
       "      "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "HTML(table_html)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.6984</td>\n",
       "      <td>0.6413</td>\n",
       "      <td>0.7467</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.2584</td>\n",
       "      <td>0.2252</td>\n",
       "      <td>0.2916</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.2345</td>\n",
       "      <td>0.1961</td>\n",
       "      <td>0.2740</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>E-type estimate</td>\n",
       "      <td>0.2348</td>\n",
       "      <td>0.1967</td>\n",
       "      <td>0.2750</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                 0       1       2       3     4\n",
       "0        Raw rates  0.6984  0.6413  0.7467   NaN\n",
       "1  Empirical Bayes  0.2584  0.2252  0.2916   NaN\n",
       "2  Poisson kriging  0.2345  0.1961  0.2740  92.0\n",
       "3  E-type estimate  0.2348  0.1967  0.2750  92.0"
      ]
     },
     "execution_count": 197,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_html(table_html)[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Working side\\n</th>\n",
       "      <th>Balancing side\\n</th>\n",
       "      <th>Unnamed: 3_level_0</th>\n",
       "      <th>Unnamed: 4_level_0</th>\n",
       "      <th>Unnamed: 5_level_0</th>\n",
       "      <th>Unnamed: 6_level_0</th>\n",
       "      <th>Unnamed: 7_level_0</th>\n",
       "      <th>Unnamed: 8_level_0</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>\\n</th>\n",
       "      <th>Max</th>\n",
       "      <th>Min</th>\n",
       "      <th>Shear</th>\n",
       "      <th>Direc</th>\n",
       "      <th>Max</th>\n",
       "      <th>Min</th>\n",
       "      <th>Shear</th>\n",
       "      <th>Direc</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Premaxilla (PM)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Sham</td>\n",
       "      <td>30.3 ± 30.6</td>\n",
       "      <td>−45.1 ± 52.9</td>\n",
       "      <td>75.4 ± 81.8</td>\n",
       "      <td>27.4 ± 6.4</td>\n",
       "      <td>27.3 ± 20.7</td>\n",
       "      <td>−37.9 ± 40.5</td>\n",
       "      <td>63.5 ± 60.5</td>\n",
       "      <td>29.1 ± 7.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Reduction</td>\n",
       "      <td>28.7 ± 20.0</td>\n",
       "      <td>−34.6 ± 42.1</td>\n",
       "      <td>62.4 ± 58.9</td>\n",
       "      <td>151.8 ± 35.0***</td>\n",
       "      <td>19.9 ± 15.3</td>\n",
       "      <td>−26.3 ± 28.3</td>\n",
       "      <td>46.2 ± 42.9</td>\n",
       "      <td>159.2 ± 40.6***</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Mandible-Incisor (MI)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Sham</td>\n",
       "      <td>68.3 ± 45.2</td>\n",
       "      <td>−37.4 ± 34.2</td>\n",
       "      <td>105.7 ± 71.5</td>\n",
       "      <td>79.2 ± 34.1</td>\n",
       "      <td>64.9 ± 45.5</td>\n",
       "      <td>−31.5 ± 25.5</td>\n",
       "      <td>96.5 ± 67.0</td>\n",
       "      <td>87.9 ± 41.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Reduction</td>\n",
       "      <td>46.9 ± 53.7</td>\n",
       "      <td>−30.0 ± 18.4</td>\n",
       "      <td>76.9 ± 64.2</td>\n",
       "      <td>126.7 ± 35.3**</td>\n",
       "      <td>29.0 ± 23.6*</td>\n",
       "      <td>−17.3 ± 5.4*</td>\n",
       "      <td>46.4 ± 22.5**</td>\n",
       "      <td>119.1 ± 32.5*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Mandible-Molar (MM)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Sham</td>\n",
       "      <td>80.0 ± 57.8</td>\n",
       "      <td>−40.5 ± 32.1</td>\n",
       "      <td>120.5 ± 67.6</td>\n",
       "      <td>50.1 ± 28.2</td>\n",
       "      <td>58.9 ± 30.3</td>\n",
       "      <td>−44.3 ± 24.2</td>\n",
       "      <td>103.2 ± 50.2</td>\n",
       "      <td>54.3 ± 23.8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Reduction</td>\n",
       "      <td>69.6 ± 56.6</td>\n",
       "      <td>−41.0 ± 40.3</td>\n",
       "      <td>110.6 ± 89.3</td>\n",
       "      <td>69.3 ± 24.7</td>\n",
       "      <td>48.4 ± 45.5</td>\n",
       "      <td>−36.7 ± 45.7</td>\n",
       "      <td>85.2 ± 84.6</td>\n",
       "      <td>63.1 ± 29.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>ANOVA/Tukey</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Sham</td>\n",
       "      <td>MM &gt; PM#</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MM &gt; PM#</td>\n",
       "      <td>MI &gt; PM##</td>\n",
       "      <td>MM &gt; PM#</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MI &gt; PM##</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Reduction</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PM &gt; MM##</td>\n",
       "      <td>MM &gt; PM#</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PM &gt; MI#</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MI &gt; MM#</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PM &gt; MM##</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         \\n               Working side\\n Balancing side\\n Unnamed: 3_level_0 Unnamed: 4_level_0 Unnamed: 5_level_0 Unnamed: 6_level_0 Unnamed: 7_level_0 Unnamed: 8_level_0\n",
       "         \\n                          Max              Min              Shear              Direc                Max                Min              Shear              Direc\n",
       "0         Premaxilla (PM)            NaN              NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "1                    Sham    30.3 ± 30.6     −45.1 ± 52.9        75.4 ± 81.8         27.4 ± 6.4        27.3 ± 20.7       −37.9 ± 40.5        63.5 ± 60.5         29.1 ± 7.2\n",
       "2               Reduction    28.7 ± 20.0     −34.6 ± 42.1        62.4 ± 58.9    151.8 ± 35.0***        19.9 ± 15.3       −26.3 ± 28.3        46.2 ± 42.9    159.2 ± 40.6***\n",
       "3   Mandible-Incisor (MI)            NaN              NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "4                    Sham    68.3 ± 45.2     −37.4 ± 34.2       105.7 ± 71.5        79.2 ± 34.1        64.9 ± 45.5       −31.5 ± 25.5        96.5 ± 67.0        87.9 ± 41.7\n",
       "5               Reduction    46.9 ± 53.7     −30.0 ± 18.4        76.9 ± 64.2     126.7 ± 35.3**       29.0 ± 23.6*       −17.3 ± 5.4*      46.4 ± 22.5**      119.1 ± 32.5*\n",
       "6     Mandible-Molar (MM)            NaN              NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "7                    Sham    80.0 ± 57.8     −40.5 ± 32.1       120.5 ± 67.6        50.1 ± 28.2        58.9 ± 30.3       −44.3 ± 24.2       103.2 ± 50.2        54.3 ± 23.8\n",
       "8               Reduction    69.6 ± 56.6     −41.0 ± 40.3       110.6 ± 89.3        69.3 ± 24.7        48.4 ± 45.5       −36.7 ± 45.7        85.2 ± 84.6        63.1 ± 29.6\n",
       "9                     NaN            NaN              NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "10            ANOVA/Tukey            NaN              NaN                NaN                NaN                NaN                NaN                NaN                NaN\n",
       "11                   Sham       MM > PM#              NaN           MM > PM#          MI > PM##           MM > PM#                NaN                NaN          MI > PM##\n",
       "12              Reduction            NaN              NaN                NaN          PM > MM##           MM > PM#                NaN                NaN           PM > MI#\n",
       "13                    NaN            NaN              NaN                NaN           MI > MM#                NaN                NaN                NaN          PM > MM##"
      ]
     },
     "execution_count": 180,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = table_df[0].copy()\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\\n                  \\n                  object\n",
       "Working side\\n      Max                 object\n",
       "Balancing side\\n    Min                 object\n",
       "Unnamed: 3_level_0  Shear               object\n",
       "Unnamed: 4_level_0  Direc               object\n",
       "Unnamed: 5_level_0  Max                 object\n",
       "Unnamed: 6_level_0  Min                 object\n",
       "Unnamed: 7_level_0  Shear               object\n",
       "Unnamed: 8_level_0  Direc               object\n",
       "dtype: object"
      ]
     },
     "execution_count": 168,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_df[0].dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "c = next(table_wrap.getiterator())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tables = table_wrap.findall('table')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "table = tables[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'frame': 'void', 'rules': 'none'}"
      ]
     },
     "execution_count": 113,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table.attrib"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<Element 'thead' at 0x7f16c64a7cc8>, <Element 'tbody' at 0x7f16c64a7ea8>]"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table.getchildren()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mean (SD) Neuropsychological Test Performance for the Three Age Groups in Experiment 1\n"
     ]
    }
   ],
   "source": [
    "caption = table_wrap.find('caption')\n",
    "\n",
    "print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'id': 'T1', 'position': 'float'}"
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_wrap.attrib"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tree = ET.parse(MANUSCRIPT_DATA_DIR.joinpath('PMC0021XXXXX/PMC2136438.xml'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "root = tree.getroot()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "```xml\n",
    "<table-wrap id=\"T1\" position=\"float\">\n",
    "<label>Table 1</label>\n",
    "<caption>\n",
    "  <p>Mean (SD) Neuropsychological Test Performance for the Three Age Groups in Experiment 1</p>\n",
    "</caption>\n",
    "<table frame=\"void\" rules=\"none\">\n",
    "  <thead>\n",
    "    <tr>\n",
    "      <th align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\"/>\n",
    "      <th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Young Adults</th>\n",
    "      <th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Older-Young Adults</th>\n",
    "      <th align=\"right\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Older Adults</th>\n",
    "    </tr>\n",
    "  </thead>\n",
    "  <tbody>\n",
    "    <tr>\n",
    "      <td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Test (Maximum Score)</td>\n",
    "      <td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\"/>\n",
    "      <td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\"/>\n",
    "      <td align=\"right\" valign=\"middle\" rowspan=\"1\" colspan=\"1\"/>\n",
    "    </tr>\n",
    "    <tr>\n",
    "    ...\n",
    "    </tr>\n",
    "  </tbody>\n",
    "</table>\n",
    "<table-wrap-foot>\n",
    "  <p><italic>Note</italic>. CVLT = California Verbal Learning Test</p>\n",
    "  <fn id=\"TFN1\">\n",
    "    <label>*</label>\n",
    "    <p id=\"P61\">= significant group differences</p>\n",
    "  </fn>\n",
    "  <fn id=\"TFN2\">\n",
    "    <label>†</label>\n",
    "    <p id=\"P62\">= significant group differences</p>\n",
    "  </fn>\n",
    "</table-wrap-foot>\n",
    "</table-wrap>\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "collapsed": true,
    "run_control": {
     "marked": false
    }
   },
   "outputs": [],
   "source": [
    "table_wraps = tree.findall('.//table-wrap')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<Element 'table' at 0x7f16c64a7c78>]"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_wrap = table_wraps[0]\n",
    "table_wrap.findall('table')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'id': 'T1', 'position': 'float'}"
      ]
     },
     "execution_count": 76,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_wrap.attrib"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\n        '"
      ]
     },
     "execution_count": 77,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_wrap.text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Element 'caption' at 0x7f16c64a7bd8>"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table_wrap.find('caption')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<Element 'table' at 0x7f16c64a7c78>,\n",
       " <Element 'table' at 0x7f16c5942d68>,\n",
       " <Element 'table' at 0x7f16c592f868>,\n",
       " <Element 'table' at 0x7f16c591e0e8>,\n",
       " <Element 'table' at 0x7f16c591ee58>,\n",
       " <Element 'table' at 0x7f16c6c57188>,\n",
       " <Element 'table' at 0x7f16c6c5adb8>,\n",
       " <Element 'table' at 0x7f16c6c48d18>,\n",
       " <Element 'table' at 0x7f16c6c4cd18>]"
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tree.findall('*/*/*/table')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "a, b = get_tables(root)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<Element 'table-wrap' at 0x7f16c6c2d0e8>,\n",
       " <Element 'table-wrap' at 0x7f16c6c2dc78>,\n",
       " <Element 'table-wrap' at 0x7f16c6c34868>,\n",
       " <Element 'table-wrap' at 0x7f16c6c3b818>,\n",
       " <Element 'table-wrap' at 0x7f16c6c42778>,\n",
       " <Element 'table-wrap' at 0x7f16c6bcf4a8>,\n",
       " <Element 'table-wrap' at 0x7f16c6bde1d8>]"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<Element 'table' at 0x7f16c6c2d228>,\n",
       " <Element 'table' at 0x7f16c6c2ddb8>,\n",
       " <Element 'table' at 0x7f16c6c349a8>,\n",
       " <Element 'table' at 0x7f16c6c3b958>,\n",
       " <Element 'table' at 0x7f16c6c428b8>,\n",
       " <Element 'table' at 0x7f16c6bcf5e8>,\n",
       " <Element 'table' at 0x7f16c6bde318>]"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "b"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<Element 'front' at 0x7f16bff4e8b8>\n",
      "[]\n",
      "<Element 'body' at 0x7f16bff5b368>\n",
      "[]\n",
      "<Element 'back' at 0x7f16c6c23f48>\n",
      "[<Element 'table-wrap' at 0x7f16c6c2d0e8>, <Element 'table-wrap' at 0x7f16c6c2dc78>, <Element 'table-wrap' at 0x7f16c6c34868>, <Element 'table-wrap' at 0x7f16c6c3b818>, <Element 'table-wrap' at 0x7f16c6c42778>, <Element 'table-wrap' at 0x7f16c6bcf4a8>, <Element 'table-wrap' at 0x7f16c6bde1d8>]\n"
     ]
    }
   ],
   "source": [
    "for child in root:\n",
    "    print(child)\n",
    "    print(child.findall('*/table-wrap'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<function Element.findall>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "child.findall"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'xml.etree.ElementTree.Element' object has no attribute 'tostring'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-27-83cb165fef6f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mtable\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtostring\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'xml.etree.ElementTree.Element' object has no attribute 'tostring'"
     ]
    }
   ],
   "source": [
    "table.tostring()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tables = child.findall('**/table')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "table = tables[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "makeelement expected 2 arguments, got 1",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-23-9f0f6c538d7f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mtable\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmakeelement\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m: makeelement expected 2 arguments, got 1"
     ]
    }
   ],
   "source": [
    "table.makeelement(0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df = pd.read_html(ET.tostring(table))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Predictor</th>\n",
       "      <th>Average</th>\n",
       "      <th>Minimum</th>\n",
       "      <th>Maximum</th>\n",
       "      <th>% best prediction</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Raw rates</td>\n",
       "      <td>0.6984</td>\n",
       "      <td>0.6413</td>\n",
       "      <td>0.7467</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Empirical Bayes</td>\n",
       "      <td>0.2584</td>\n",
       "      <td>0.2252</td>\n",
       "      <td>0.2916</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Poisson kriging</td>\n",
       "      <td>0.2345</td>\n",
       "      <td>0.1961</td>\n",
       "      <td>0.2740</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>E-type estimate</td>\n",
       "      <td>0.2348</td>\n",
       "      <td>0.1967</td>\n",
       "      <td>0.2750</td>\n",
       "      <td>92.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Predictor  Average  Minimum  Maximum  % best prediction\n",
       "0        Raw rates   0.6984   0.6413   0.7467                NaN\n",
       "1  Empirical Bayes   0.2584   0.2252   0.2916                NaN\n",
       "2  Poisson kriging   0.2345   0.1961   0.2740               92.0\n",
       "3  E-type estimate   0.2348   0.1967   0.2750               92.0"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[0]"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  },
  "toc": {
   "nav_menu": {},
   "number_sections": false,
   "sideBar": false,
   "skip_h1_title": false,
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": "block",
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
