{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Setting the PYTHON_VERSION environment variable.\n",
      "Setting the SPARK_MASTER environment variable.\n",
      "Setting the DB_TYPE environment variable.\n",
      "Setting the DB_PORT environment variable.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2017-11-23 13:41:13.874015\n"
     ]
    }
   ],
   "source": [
    "%run _imports.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import zipfile"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Parameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "DATA_DIR = Path('../downloads/pmc/suppl/')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[PosixPath('../downloads/pmc/suppl/NON-OA'),\n",
       " PosixPath('../downloads/pmc/suppl/OA')]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(DATA_DIR.iterdir())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {
    "collapsed": true,
    "run_control": {
     "marked": false
    }
   },
   "outputs": [],
   "source": [
    "zfile = zipfile.ZipFile(DATA_DIR.joinpath('OA', 'PMC1085900-PMC1089899', 'PMC1088961.zip'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "csv_extensions = ['.csv', '.csv.gz', '.tsv', '.tsv.gz']\n",
    "csv_files = []\n",
    "\n",
    "xml_extensions = ['.xls', '.xls.gz', '.xlsx', '.xlsx.gz']\n",
    "xml_files = []\n",
    "\n",
    "for file in zfile.filelist:\n",
    "    if any(file.filename.endswith(e) for e in csv_extensions):\n",
    "        csv_files.append(file.filename)\n",
    "    elif any(file.filename.endswith(e) for e in xml_extensions):\n",
    "        xml_files.append(file.filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "csv_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10"
      ]
     },
     "execution_count": 267,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = {'a': 10, 'b': 20}\n",
    "x.pop('a')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 263,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "RangeIndex(start=0, stop=3, step=1)"
      ]
     },
     "execution_count": 263,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 265,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>a</td>\n",
       "      <td>b</td>\n",
       "      <td>c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>d</td>\n",
       "      <td>e</td>\n",
       "      <td>f</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    0  1  2\n",
       "10  a  b  c\n",
       "11  d  e  f"
      ]
     },
     "execution_count": 265,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.sort_values(list(df.columns), axis=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 239,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "p = Path('.').absolute()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables')"
      ]
     },
     "execution_count": 257,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(p.parents)[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'notebooks'"
      ]
     },
     "execution_count": 258,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/_imports.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/process_pmc_supl.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/.ipynb_checkpoints'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/load_xml_files.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/downloaded_pdfs'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/download_sample.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/_db.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/_spark.ipynb'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/example2.parquet'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/download_sample'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/1088961.xml'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/tmpdir'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/geckodriver.log'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/output'),\n",
       " PosixPath('/home/kimlab2/database_data/datapkg/pmc_tables/notebooks/parse_to_parquet')]"
      ]
     },
     "execution_count": 238,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(p.absolute().glob('*'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['gb-2005-6-4-r33-S1.xls', 'gb-2005-6-4-r33-S2.xls']"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "xml_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "extracted_files = []\n",
    "\n",
    "tmpdir = tempfile.mkdtemp()\n",
    "#with tempfile.TemporaryDirectory() as tmpdir:\n",
    "for file in csv_files + xml_files:\n",
    "    extracted_file = zfile.extract(file, tmpdir)\n",
    "    extracted_files.append(extracted_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['/tmp/strokach/tmp8mpdyfn4/gb-2005-6-4-r33-S1.xls',\n",
       " '/tmp/strokach/tmp8mpdyfn4/gb-2005-6-4-r33-S2.xls']"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "extracted_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING *** OLE2 inconsistency: SSCS size is 0 but SSAT size is non-zero\n"
     ]
    }
   ],
   "source": [
    "df = pd.read_excel(extracted_files[0], None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "odict_keys(['Adrenal_gland', 'Amygdala', 'Caudate_nucleus', 'Cerebellum', 'Corpus_callosum', 'Cortex', 'Drg', 'Heart', 'Kidney', 'Liver', 'Lung', 'Ovary', 'Pancreas', 'Pituitary_gland', 'Placenta', 'Prostate', 'Salivary_gland', 'Spinal_cord', 'Spleen', 'Testis', 'Thalamus', 'Thymus', 'Thyroid', 'Trachea', 'Uterus'])"
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "for key, df in dfs.items():\n",
    "    df['pmc_id'] = \n",
    "    df['file_name'] = file\n",
    "    df['sheet_name'] = key"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>a</td>\n",
       "      <td>b</td>\n",
       "      <td>c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>d</td>\n",
       "      <td>e</td>\n",
       "      <td>f</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    0  1  2\n",
       "10  a  b  c\n",
       "11  d  e  f"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.DataFrame([['a', 'b', 'c'], ['d', 'e', 'f']], index=[10, 11])\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 196,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>a</th>\n",
       "      <th>b</th>\n",
       "      <th>c</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>d</td>\n",
       "      <td>e</td>\n",
       "      <td>f</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    a  b  c\n",
       "11  d  e  f"
      ]
     },
     "execution_count": 196,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.rename(columns=df.iloc[0]).drop(df.index[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 159,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df.columns = df.iloc[0]\n",
    "df = df[1:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   1\n",
       "0  2\n",
       "1  5"
      ]
     },
     "execution_count": 91,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.reindex(columns=df.index.drop(0))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Int64Index([1], dtype='int64')"
      ]
     },
     "execution_count": 95,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.index.drop(0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "filename = z1.extract(f.filename, path=tmpdir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/tmp/strokach/tmpvujfij2p/1088961.xml'"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "filename"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<!DOCTYPE article PUBLIC \"-//NLM//DTD Journal Archiving and Interchange DTD v2.1 20050630//EN\" \"archivearticle.dtd\">\r\n",
      "<article xmlns:xlink=\"http://www.w3.org/1999/xlink\" article-type=\"research-article\"><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">Genome Biol</journal-id><journal-title>Genome Biology</journal-title><issn pub-type=\"ppub\">1465-6906</issn><issn pub-type=\"epub\">1465-6914</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmcid\">1088961</article-id><article-id pub-id-type=\"publisher-id\">gb-2005-6-4-r33</article-id><article-id pub-id-type=\"pmid\">15833120</article-id><article-id pub-id-type=\"doi\">10.1186/gb-2005-6-4-r33</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Promoter features related to tissue specificity as measured by Shannon entropy</article-title></title-group><contrib-group><contrib id=\"A1\" corresp=\"yes\" contrib-type=\"author\"><name><surname>Schug</surname><given-names>Jonathan</given-names></name><xref ref-type=\"aff\" rid=\"I1\">1</xref><email>jschug@pcbi.upenn.edu</email></contrib><contrib id=\"A2\" contrib-type=\"author\"><name><surname>Schuller</surname><given-names>Winfried-Paul</given-names></name><xref ref-type=\"aff\" rid=\"I2\">2</xref></contrib><contrib id=\"A3\" contrib-type=\"author\"><name><surname>Kappen</surname><given-names>Claudia</given-names></name><xref ref-type=\"aff\" rid=\"I2\">2</xref></contrib><contrib id=\"A4\" contrib-type=\"author\"><name><surname>Salbaum</surname><given-names>J Michael</given-names></name><xref ref-type=\"aff\" rid=\"I2\">2</xref></contrib><contrib id=\"A5\" contrib-type=\"author\"><name><surname>Bucan</surname><given-names>Maja</given-names></name><xref ref-type=\"aff\" rid=\"I3\">3</xref></contrib><contrib id=\"A6\" contrib-type=\"author\"><name><surname>Stoeckert</surname><given-names>Christian J</given-names><suffix>Jr</suffix></name><xref ref-type=\"aff\" rid=\"I1\">1</xref></contrib></contrib-group><aff id=\"I1\"><label>1</label>Center for Bioinformatics, University of Pennsylvania, Philadelphia, PA 19104, USA</aff><aff id=\"I2\"><label>2</label>Department of Genetics, Cell Biology and Anatomy, University of Nebraska Medical Center, Omaha, NE 68198, USA</aff><aff id=\"I3\"><label>3</label>Department of Genetics, University of Pennsylvania, Philadelphia, PA 19104, USA</aff><pub-date pub-type=\"ppub\"><year>2005</year></pub-date><pub-date pub-type=\"epub\"><day>29</day><month>3</month><year>2005</year></pub-date><volume>6</volume><issue>4</issue><fpage>R33</fpage><lpage>R33</lpage><ext-link ext-link-type=\"uri\" xlink:href=\"http://genomebiology.com/2005/6/4/R33\"/><history><date date-type=\"received\"><day>16</day><month>11</month><year>2004</year></date><date date-type=\"rev-recd\"><day>27</day><month>1</month><year>2005</year></date><date date-type=\"accepted\"><day>16</day><month>2</month><year>2005</year></date></history><copyright-statement>Copyright &#x000a9; 2005 Schug et al.; licensee BioMed Central Ltd.</copyright-statement><license license-type=\"open-access\" xlink:href=\"http://creativecommons.org/licenses/by/2.0\"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by/2.0\"/>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></license><abstract abstract-type=\"short\"><p>A genome-wide analysis of promoters was carried out in the context of gene expression patterns in tissue surveys using human microarray and EST-based expression data. The study revealed that most genes show statistically significant tissue-dependent variations of expression level and identified components of promoters that distinguish tissue-specific from ubiquitous genes.</p></abstract><abstract><sec><title>Background</title><p>The regulatory mechanisms underlying tissue specificity are a crucial part of the development and maintenance of multicellular organisms. A genome-wide analysis of promoters in the context of gene-expression patterns in tissue surveys provides a means of identifying the general principles for these mechanisms.</p></sec><sec><title>Results</title><p>We introduce a definition of tissue specificity based on Shannon entropy to rank human genes according to their overall tissue specificity and by their specificity to particular tissues. We apply our definition to microarray-based and expressed sequence tag (EST)-based expression data for human genes and use similar data for mouse genes to validate our results. We show that most genes show statistically significant tissue-dependent variations in expression level. We find that the most tissue-specific genes typically have a TATA box, no CpG island, and often code for extracellular proteins. As expected, CpG islands are found in most of the least tissue-specific genes, which often code for proteins located in the nucleus or mitochondrion. The class of genes with no CpG island or TATA box are the most common mid-specificity genes and commonly code for proteins located in a membrane. Sp1 was found to be a weak indicator of less-specific expression. YY1 binding sites, either as initiators or as downstream sites, were strongly associated with the least-specific genes.</p></sec><sec><title>Conclusions</title><p>We have begun to understand the components of promoters that distinguish tissue-specific from ubiquitous genes, to identify associations that can predict the broad class of gene expression from sequence data alone.</p></sec></abstract></article-meta></front><body><sec><title>Background</title><p>The development of an adult from the single cell of a fertilized egg requires a complex orchestration of genes to be expressed at the right time, place, and level. Basic cellular functions require the expression of certain genes in all cells and tissues (that is, in a ubiquitous manner) while specialized functions require restricted expression of other genes in a single or small number of cells and tissues (that is, tissue specific). Both types of genes may be needed for embryonic development as well as for the function of adult cells and tissues. While the details of regulatory mechanisms will vary for individual genes, general features of promoters (and here we will restrict our focus to RNA polymerase II (Pol II) promoters) are likely to facilitate whether a gene will be expressed widely or in a restricted manner. For example, based on the limited number of genes available at the time of the analysis, promoters with CpG islands have been associated with housekeeping genes [<xref ref-type=\"bibr\" rid=\"B1\">1</xref>,<xref ref-type=\"bibr\" rid=\"B2\">2</xref>]. It is desirable to re-examine this finding in the context of complete genomes for human and mouse and to place it in context with subsequent findings such as the association of CpG islands with embryonic expression [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>].</p><p>Furthermore, it would also be informative to examine the relationship of CpG islands to the base composition of promoters, and the distribution of motifs thought to be bound by factors closely involved with (or part of) the basal transcription complex. The distribution of major components of the core promoter, the TATA box (TBP/TFIID binding site) and initiator element (Pol II binding site, Inr) [<xref ref-type=\"bibr\" rid=\"B4\">4</xref>], and proximal elements such as Yin-Yang 1 (YY1) site [<xref ref-type=\"bibr\" rid=\"B5\">5</xref>-<xref ref-type=\"bibr\" rid=\"B8\">8</xref>], among genes is not yet well understood. In addition, the functional correlations with tissue specificity and promoter structure are largely unknown beyond the CpG island association. Our goal is to place these components together in general models for tissue specificity using genome-wide surveys of expression in many tissues.</p><p>Investigators have searched for combinations of transcription-factor-binding sites that confer tissue-specific expression on particular cell types such as muscle [<xref ref-type=\"bibr\" rid=\"B9\">9</xref>] or liver [<xref ref-type=\"bibr\" rid=\"B10\">10</xref>] in mammals, or in body plan specification in the fruit fly [<xref ref-type=\"bibr\" rid=\"B11\">11</xref>,<xref ref-type=\"bibr\" rid=\"B12\">12</xref>] (see [<xref ref-type=\"bibr\" rid=\"B13\">13</xref>] for a review). In support of these efforts, analyses of genome-wide expression data have largely focused on identifying common patterns for particular tissues, disease states or signaling inputs. For microarray data, investigators have begun defining these patterns, largely through the application of clustering algorithms [<xref ref-type=\"bibr\" rid=\"B14\">14</xref>,<xref ref-type=\"bibr\" rid=\"B15\">15</xref>]. Our approach is to rank genes in the spectrum of tissue specificity that runs from expression restricted to one tissue to uniform ubiquitous expression. We can study in detail the distribution of human and mouse genes across the spectrum of tissue specificity and use this to identify commonalities and differences in their promoters with the available complete genome sequences [<xref ref-type=\"bibr\" rid=\"B16\">16</xref>], libraries enriched for full-length cDNAs [<xref ref-type=\"bibr\" rid=\"B17\">17</xref>-<xref ref-type=\"bibr\" rid=\"B19\">19</xref>] and genome-wide surveys of gene expression using microarrays [<xref ref-type=\"bibr\" rid=\"B14\">14</xref>,<xref ref-type=\"bibr\" rid=\"B20\">20</xref>-<xref ref-type=\"bibr\" rid=\"B24\">24</xref>], SAGE [<xref ref-type=\"bibr\" rid=\"B25\">25</xref>], mRNAs [<xref ref-type=\"bibr\" rid=\"B18\">18</xref>] and expressed sequence tags (ESTs) [<xref ref-type=\"bibr\" rid=\"B26\">26</xref>]. We validate patterns discovered in human sequence and expression data by comparison to similar mouse data.</p><p>Measures have been developed for overall tissue specificity [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>,<xref ref-type=\"bibr\" rid=\"B27\">27</xref>,<xref ref-type=\"bibr\" rid=\"B28\">28</xref>] that amount to counting the number of tissues that express a gene. These are really measuring tissue restriction, as they do not consider any bias in the expression levels across the tissues that express the gene. Most specificity measures for a particular tissue are equivalent to the relative expression in a tissue compared to the total expression in all tissues considered, (see, for example [<xref ref-type=\"bibr\" rid=\"B29\">29</xref>]). We assert that overall tissue specificity measures should take into account the levels of expression in different tissues, not just presence and absence, and that specificity measures for particular tissues should consider the distribution of expression among all tissues in addition to the tissue of interest. Such measures would enable the correct identification of genes as specific for a tissue when that tissue is not the primary site of expression but there are only a few other tissues where the gene is expressed.</p><p>A metric for characterizing the breadth and uniformity of the expression pattern of a gene that meets our criteria is the Shannon information theoretic measure entropy. Although entropy has been used previously to identify potential drug targets [<xref ref-type=\"bibr\" rid=\"B30\">30</xref>,<xref ref-type=\"bibr\" rid=\"B31\">31</xref>] by considering the entropy of the variation of expression levels and to cluster microarray data [<xref ref-type=\"bibr\" rid=\"B32\">32</xref>], our direct application of entropy to measuring tissue specificity is unique. Entropy (<italic>H</italic>) measures the degree of overall tissue specificity of a gene, but does not indicate whether it is specific to a particular tissue. To quantify categorical tissue specificity, we introduce a new statistic (<italic>Q</italic>) that incorporates overall tissue specificity and relative expression level. We demonstrate that <italic>H </italic>and <italic>Q </italic>are effective metrics for ranking and selecting genes according to tissue specificity and then proceed to use them to investigate promoter features (CpG islands, base composition, transcription factor motifs) that may be used distinguish tissue-specific genes from nonspecific genes. The association of promoter features with a quantitative assessment of tissue specificity using <italic>H </italic>and <italic>Q </italic>is an important step towards developing models for promoter function.</p></sec><sec><title>Results</title><sec><title>Defining tissue specificity</title><p>We begin by defining the measurement of two kinds of tissue specificity, 'overall' tissue specificity and 'categorical' tissue specificity. (To avoid confusion we will always use the words 'specificity' and 'specific' to refer to the degree of tissue-restricted expression a gene exhibits and never as a synonym for the word 'particular'.) Overall tissue specificity ranks a gene according to the degree to which its expression pattern differs from ubiquitous uniform expression. We use the term 'ubiquitous' expression to mean expression at any level above background in all tissues. Categorical tissue specificity places special emphasis on a particular tissue of interest and ranks a gene according to the degree to which its expression pattern is skewed toward expression in only that particular tissue. In both cases, a gene's specificity to a tissue, cell type or other condition is decreased as the gene is more uniformly expressed in a wider variety of conditions. In addition, the categorical tissue specificity should decrease as the tissue of interest becomes a smaller component of the overall expression pattern of the gene.</p><p>Given a static multi-tissue expression profile for a gene, there are at least two dimensions along which we can assess the profile to measure tissue specificity. The first dimension is the number of tissues that express the gene above some background level. It can be argued that this dimension measures tissue restriction, that is, a gene shows restricted expression if it is expressed in only a subset of tissues. The second dimension is the uniformity of expression over all tissues that express the gene. A gene that shows significant non-uniform expression is exhibiting tissue-dependent regulation, in addition to any tissue restriction that may be occurring. We assume that a gene that exhibits no tissue-specific regulation will be expressed at the same level in every tissue. We do not assert that such genes are not regulated, only that they are regulated in a way that is not sensitive to tissue.</p><p>The term 'most tissue-specific' will refer to the range of genes that are closer to the extreme of expression in a single tissue than to the extreme of ubiquitous uniform expression. We will refer to genes close to the uniform and ubiquitous end as either 'least tissue-specific' or 'nonspecific' though the latter term may not be strictly true. The range in the middle will be termed 'semi-tissue specific'. The term 'housekeeping' has been applied to genes that are widely expressed and may show little tissue-specific changes in expression level. We can use such genes as an example of genes that will tend to be ubiquitously and uniformly expressed and thus ought to be nonspecific on average. We will use the phrase 'gene sharing' to refer to the situation that occurs when a gene is tissue-specific, and is expressed in a small number of tissues that can be said to share the gene.</p></sec><sec><title>Measuring tissue specificity with entropy</title><p>We used two gene-expression datasets to evaluate our methods; Affymetrix-based data from the GNF Gene Expression Atlas (GNF-GEA) [<xref ref-type=\"bibr\" rid=\"B22\">22</xref>] and the distribution of source tissues for EST libraries in the clusters and assemblies of ESTs in the DoTS mouse and human gene index [<xref ref-type=\"bibr\" rid=\"B33\">33</xref>]. As described in Materials and methods, the GNF-GEA data were used as provided; EST counts in the DoTS gene index were adjusted with pseudocounts and normalized to account for the different number of ESTs sampled from each tissue across all libraries. Given expression levels of a gene in <italic>N </italic>tissues, we defined the relative expression of a gene <italic>g </italic>in a tissue <italic>t </italic>as <italic>p</italic><sub><italic>t</italic>|<italic>g </italic></sub>= <italic>w</italic><sub><italic>g</italic>,<italic>t</italic></sub>/&#x02211;<sub>1 &#x02264; <italic>t </italic>&#x02264; <italic>N</italic></sub><italic>w</italic><sub><italic>g</italic>,<italic>t </italic></sub>where <italic>w</italic><sub><italic>g</italic>,<italic>t </italic></sub>is the expression level of the gene in the tissue. The entropy [<xref ref-type=\"bibr\" rid=\"B34\">34</xref>] of a gene's expression distribution is <italic>H</italic><sub><italic>g </italic></sub>= &#x02211;<sub>1 &#x02264; <italic>t </italic>&#x02264; <italic>N </italic></sub>- <italic>p</italic><sub><italic>t</italic>|<italic>g </italic></sub>log<sub>2</sub>(<italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>). <italic>H</italic><sub><italic>g </italic></sub>has units of bits and ranges from zero for genes expressed in a single tissue to log<sub>2</sub>(<italic>N</italic>) for genes expressed uniformly in all tissues considered. The maximum value of <italic>H</italic><sub><italic>g </italic></sub>depends on the number of tissues considered so we will report this number when appropriate. Because we use relative expression the entropy of a gene is not sensitive to the absolute expression levels. To measure categorical tissue specificity we define <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>= <italic>H</italic><sub><italic>g </italic></sub>- log<sub>2</sub>(<italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>). The quantity -log<sub>2</sub>(<italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>) also has units of bits and has a minimum of zero that occurs when a gene is expressed in a single tissue and grows unboundedly as the relative expression level drops to zero. Thus <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>is near its minimum of zero bits when a gene is relatively highly expressed in a small number of tissues including the tissue of interest, and becomes higher as either the number of tissues expressing the gene becomes higher, or as the relative contribution of the tissue to the gene's overall pattern becomes smaller. By itself, the term -log<sub>2</sub>(<italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>) is equivalent to <italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>. Adding the entropy term serves to favor genes that are not expressed highly in the tissue of interest, but are expressed only in a small number of other tissues. As described earlier, we want to consider such genes as categorically tissue-specific since their expression pattern is very restricted. Figure <xref ref-type=\"fig\" rid=\"F1\">1</xref> shows examples of patterns of GNF-GEA expression data for different values of <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t</italic></sub>. The top five genes specific to mouse amygdala, lymph node, and liver as assessed by this data are listed in Table <xref ref-type=\"table\" rid=\"T1\">1</xref>. Tables of <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>values for all genes in all tissues in the GNF-GEA datasets are available in Additional data files 1 and 2.</p><p>To compare results from microarray and EST-based expression data we mapped the tissues from the GNF-GEA study to the hierarchical controlled vocabulary of anatomical terms used by DoTS and chose a set of 45 tissue terms grouped into 32 groups shown in Table <xref ref-type=\"table\" rid=\"T2\">2</xref>. In both cases, the vast majority of genes are widely expressed as measured by <italic>H</italic><sub><italic>g </italic></sub>as shown in Figure <xref ref-type=\"fig\" rid=\"F2\">2a</xref>. Of the 7,714 probe sets in the GNF-GEA data with an average normalized intensity value above 50 arbitrary units (AU), 6,167 (80%) of genes had <italic>H</italic><sub><italic>g </italic></sub>&#x02265; 4 bits, which implies expression in at least 16 tissues and typically corresponds to wider, but uneven, expression. Only 87 (2%) of genes had <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 1.5 bits, which corresponds to expression in as few as three tissues. Both microarray- and EST-based data yielded similar overall curves. The EST curve peaked at a lower <italic>H</italic><sub><italic>g </italic></sub>than the microarray curve. This was due to the small numbers of EST sequences in some of the tissues we considered; EST counts for tissues ranged from 1,933 in the adrenal gland to 331,582 in the central nervous system (CNS). Genes that are ubiquitously expressed may not have ESTs from several of the lightly sequenced tissues, making them appear to have more restricted expression, and hence a lower entropy, than they really do. Figure <xref ref-type=\"fig\" rid=\"F2\">2b</xref> shows the correlation between estimates of <italic>H</italic><sub><italic>g </italic></sub>derived from microarray and EST data. Visual inspection of the plot reveals that while there are no strong contradictions between the two methods, quantitative agreement is limited. Detailed analysis shows that the standard deviation of the difference of paired <italic>H</italic><sub><italic>g </italic></sub>values is 0.61 bits. Under the null hypothesis that the estimates from the two data sources are totally uncorrelated the average standard deviation was found to be 0.91 bits. We can reject the null hypothesis (<italic>P </italic>&#x0003c; 10<sup>-5 </sup>as estimated by Monte Carlo methods). The distribution of <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>for selected tissues is shown in Figure <xref ref-type=\"fig\" rid=\"F2\">2c</xref>. These curves can be used to characterize tissues in terms of the number of tissue-specific genes and the amount of gene sharing; for example, liver has a relatively large number of genes shared with a small number of other tissues. In contrast, there were no genes in this set that are uniquely expressed in the amygdala.</p><p>It is important to determine how well the <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>statistics can be estimated from a dataset to determine the smallest meaningful difference in scores and to guide interpretation of gene rankings. To assess the standard deviations of and <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t</italic></sub>, we sampled from the replicates in the GNF-GEA microarray data to compute a large number of <italic>H</italic><sub><italic>g </italic></sub>values for each probe set. We found that the standard deviation for <italic>H</italic><sub><italic>g </italic></sub>was less than 0.2 bits for 97% of genes. <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>was not estimated as well; the standard deviation was 1 bit or less for 95% of gene and tissue pairs. This was probably due to the high standard deviation of the -log<sub>2</sub>(<italic>p</italic><sub><italic>t</italic>|<italic>g</italic></sub>) term for low expressing gene-tissue pairs. We found much more variation when we measure reproducibility by considering genes that have two or more probe sets (and therefore two or more different transcripts) in the microarray data. In this case, the standard deviation of <italic>H</italic><sub><italic>g </italic></sub>estimates was as high as 1 bit for 97% of the genes but less than 0.3 bits for about 70-80% of the genes. We chose a minimum of 1 bit for <italic>H</italic><sub><italic>g </italic></sub>bins and 2 bits for <italic>Q </italic>bins in the rest of the analyses that require binning. This bin size ensured that most of the genes are in the proper bin and thus the bin could be reliably used to determine associations with the tissue specificity of a class of genes.</p></sec><sec><title>Evaluating a set of housekeeping genes</title><p>A test of the <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>statistics is to determine values for a set of nonspecific genes such as housekeeping genes. A list of 797 human housekeeping genes [<xref ref-type=\"bibr\" rid=\"B35\">35</xref>] was evaluated using these statistics based on the GNF-GEA dataset using RefSeq accession numbers to identify appropriate probe sets. The housekeeping genes had a mean <italic>H</italic><sub><italic>g </italic></sub>= 4.6 &#x000b1; 0.27 bits in a set of 27 tissues with a maximum <italic>H </italic>= lg(27) = 4.75 bits; thus they are nonspecific as expected. Interestingly, a small number of these genes did show some degree of tissue specificity yet were ubiquitously expressed. For example, the median expression of NM_021983 the major histocompatibility complex, class II DR beta 4 gene (32035_at) is approximately 200 AU, but it shows much higher expression in a small set of tissues (spleen, thymus, lung, heart and whole blood), which lowered its entropy. A more extreme case is NM_001502 glycoprotein 2 (zymogen granule membrane protein 2), which is expressed between 250 and 1,000 AU in all tissues except pancreas, where it is expressed at 34,183 AU. This is a ubiquitously expressed gene that entropy categorizes as specific since it showed such extreme tissue-specific induction. The housekeeping genes had a mean <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>= 9.5 &#x000b1; 0.14 bits in the same set of tissues. The expected <italic>Q </italic>value for a uniformly and ubiquitously expressed gene is 2 lg(27) = 9.5 bits. Thus, the <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>statistics successfully captured the expected expression properties of housekeeping genes.</p></sec><sec><title>Most genes are regulated in a tissue-dependent manner</title><p>Although the housekeeping genes assessed above have relatively high entropies, they do show some small degree of overall tissue specificity. We therefore sought to determine how many genes show evidence of tissue-dependent regulation. Since random biological and experimental variation introduce fluctuations in the expression levels of genes, we made a probability model of the effect of these fluctuations on the observed entropy. The experimental variability was estimated from the GNF-GEA data using all normal tissues. The random tissue-to-tissue biological variability was modeled by assuming that each gene has an average expression level across all tissues and that the log base 2 of the tissue-dependent fold changes from the average level follow a normal distribution with mean equal to zero and some unknown, but 'small', standard deviation(s). We obtain a conservative estimate of the number of genes showing evidence of tissue-dependent regulation by using <italic>s </italic>= 0.5, which allows for a relatively large amount of variation; up to 1.4-fold tissue-to-tissue variation around the mean expression level in about 63% of tissues and larger changes in the remaining tissues. As a threshold for selecting genes with tissue-dependent expression, we choose <italic>H</italic><sub><italic>g </italic></sub>= 4.52 bits which has a <italic>p</italic>-value of 0.005 under the null hypothesis that all genes are uniform. We then find that 5,837/8,703 (67%) of human genes have entropies less than this and so are probably regulated in a tissue-dependent manner. If we use a more stringent definition of uniform expression that allows half as much variation in tissue-to-tissue expression levels (<italic>s </italic>= 0.25), then the threshold is <italic>H</italic><sub><italic>g </italic></sub>= 4.62 bits and we find that 7,584/8,703 (87%) of human genes show evidence of tissue-dependent regulation. Similar results are found in mouse using all 42 distinct tissues, where the corresponding thresholds are <italic>H</italic><sub><italic>g </italic></sub>= 5.24 bits (<italic>s </italic>= 0.5) and <italic>H</italic><sub><italic>g </italic></sub>= 5.35 bits (<italic>s </italic>= 0.25) and the fractions of genes showing tissue-dependent expression are 5,467/7,913 (69%) and 7,482/7,913 (94%) respectively. Thus we conclude that most genes show evidence of tissue-dependent expression levels.</p></sec><sec><title>Clustering tissues using <italic>Q</italic></title><p>A test of <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>with respect to specific genes is to evaluate the tissues in which they rank highly (that is, have low <italic>Q</italic>) for consistency. This was accomplished by clustering tissues with similar tissue-specific genes and inspecting the clusters formed. We used 27 normal human tissues and, separately, 39 tissues from the GNF-GEA data for mouse and selected the genes (<italic>N </italic>= 3,768 human and <italic>N</italic> = 1786 mouse) that express at least 200 AU in at least one tissue and have <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>= 7 in at least one tissue. With these genes, we made a consensus hierarchical clustering of the tissues as shown in Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref>. We found that the tissues in the nervous system, reproductive structures (excluding testis), immune system, and digestive system reliably cluster together in both species. In addition, skeletal muscle and heart clustered in mouse; the human survey did not have skeletal muscle. These results suggest that <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>is correctly identifying tissue-specific genes. Interestingly, testis is an outlier in both trees, indicating that the collection of genes expressed in testis are distinct from any other tissue or organ. Furthermore, <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>can also be used in conjunction with a tissue hierarchy to answer more complex questions about the tissue distribution of genes such as 'what genes are specific to the brain but are widely expressed throughout the brain?' In Table <xref ref-type=\"table\" rid=\"T3\">3</xref> we list the top five mouse genes expressed specifically but uniformly across three of the highlighted groups in Figure <xref ref-type=\"fig\" rid=\"F3\">3b</xref>.</p></sec><sec><title>CpG islands are associated with the least tissue-specific genes</title><p>It has been proposed that CpG islands are predominantly associated with promoters of housekeeping genes [<xref ref-type=\"bibr\" rid=\"B2\">2</xref>]. We performed a quantitative test of this hypothesis using the GNF-GEA data and determining the frequency of CpG islands in promoters as a function of <italic>H</italic><sub><italic>g</italic></sub>. We considered only predicted CpG islands that span the start of transcription (see [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>] for a justification of this definition), and genes that expressed at least at the median level of 200 AU (that is, were moderately expressed) in at least one tissue, and were represented by a single probe set on the Affymetrix chip used in the GNF-GEA experiments. Promoter sequences were obtained from DBTSS and were based on the 5' ends of full-length transcripts [<xref ref-type=\"bibr\" rid=\"B17\">17</xref>]. We found that there is a strong, roughly linear, correlation between a gene's entropy <italic>H</italic><sub><italic>g </italic></sub>and the probability that the gene will have a predicted start CpG island as shown in Figure <xref ref-type=\"fig\" rid=\"F4\">4</xref>. Start CpG islands were associated with only nine of the 100 most tissue-specific human genes as compared to 80% of the least tissue-specific genes. Similar numbers were found for mouse (7% start CpG island frequency for the 100 most tissue-specific genes; about 64% for the least tissue-specific genes). A comparison of CpG islands from the most and least tissue-specific genes did not reveal any significant difference in the overall base composition, or ratio of observed to expected CpG dinucleotides. The distribution of the position of the 5' end point of CpG islands was also very similar for the most and least tissue-specific genes though CpG islands tend to start further upstream in the least tissue-specific genes (data not shown).</p><p>Another group of genes observed to be associated with CpG islands are those expressed in the early embryo [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>] from the fertilized egg to the blastocyst. The question arises as to whether there is an association of genes having start CpG islands and the developmental stage of expression (that is, embryonic versus adult) in addition to the one for tissue specificity. We investigated this possibility in the mouse using DoTS [<xref ref-type=\"bibr\" rid=\"B33\">33</xref>] EST and mRNA assemblies by tabulating the number of DoTS genes that contain at least two ESTs from a mouse early embryo library as shown in Table <xref ref-type=\"table\" rid=\"T4\">4</xref>. We considered 933 genes with start CpG islands (CGI+) and 1,007 genes without start CpG islands (CGI-) that were expressed in the adult. If there were no developmental bias, this distribution of CpG+ and CpG- genes should be maintained in genes expressed in the embryo. However, only 139 (14%) of the CGI- genes were expressed in the early embryo in contrast to 365 (39%) CGI+ genes (<italic>P </italic>= 3 &#x000d7; 10<sup>-70 </sup>exact binomial). Therefore, a gene expressed in the adult was 2.8 (= 0.39/0.14) times more likely to be expressed in the early embryo if it contained a start CpG island. Furthermore, the most tissue-specific genes expressed in the adult were four times more likely to have been expressed in the early embryo if their promoter contained a start CpG island. These results strongly suggest that CpG islands are promoter features for both embryonic and the least tissue-specific genes.</p></sec><sec><title>Base composition of promoters depends on specificity</title><p>Analysis of base-composition profiles of promoters provides clues to common features, including motifs associated with promoter categories. We examined the base composition profiles of human promoters of high (0 &#x02264; <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 3.5 bits) and low (4.4 &#x02264; <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 4.71 bits) tissue-specificity genes. We considered CGI+ and CGI- genes separately, as it is clear the presence of a CpG island will strongly influence the base composition and that the fraction of start CpG islands varies with entropy. In addition, the presence of a start CpG island may indicate a different regulation mechanism related to either tissue specificity or embryonic expression (or both). The number of promoters from DBTSS in these four classes that were used in the analysis were: 310 CGI- and 129 CGI+ high specificity; 342 CGI- and 1,501 CGI+ low specificity. Genes that have only non-start CpG islands represented a minor component and were not included in this analysis. We used the full set of normal tissues in the first GNF-GEA microarray study for human and mouse. Base composition profiles with 10 base-pair (bp) windows are shown in Figure <xref ref-type=\"fig\" rid=\"F5\">5</xref> for human genes. Each of the features we report were observed in human and mouse (unless noted otherwise) and compare G to C or A to T over spans of at least 10 positional bins; the probability of observing a feature at least this long by chance is less than 0.5<sup>10 </sup>which is equivalent to 0.001. Promoters of CGI+ genes (Figure <xref ref-type=\"fig\" rid=\"F5\">5a,b</xref>) shared features but could also be distinguished on the basis of tissue specificity. A common feature of CGI+ promoters was the increase in C+G content that starts at 1,000 bp upstream of the transcription start site and continues at 200 bp downstream. The C+G bias reached p(C+G) = 0.7 at the start of transcription and continued into the 5' UTR. Nonspecific (Figure <xref ref-type=\"fig\" rid=\"F5\">5c</xref>) and tissue-specific (Figure <xref ref-type=\"fig\" rid=\"F5\">5d</xref>) CGI- genes still showed a C+G bias around the start of transcription, but it was much smaller in magnitude at p(C+G) = 0.54. The low specificity CGI+ genes (Figure <xref ref-type=\"fig\" rid=\"F5\">5a</xref>) showed upstream base composition biases that were not found in any of the other three gene classes. There was a preference for C over G (p(C) &#x0003e; p(G)) in the (-350, -150) region and also a preference for p(A) &#x0003e; p(T) in the -600, -200 region in human (this region is located (-400, -150) in mouse). In tissue-specific CGI+ (Figure <xref ref-type=\"fig\" rid=\"F5\">5b</xref>) genes the strong C+G bias held but p(C) = p(G), except for the (+50, +100) region where p(C) &#x0003e; p(G). These base-composition differences observed between nonspecific and tissue-specific promoters over regions of hundreds of base-pairs, even in the context of a CpG island, suggest different structural features and regulatory mechanisms for these CGI+ classes.</p><p>Most striking were differences between nonspecific and tissue-specific promoters that are independent of the presence of a CpG island. A sharp spike in the proportion of A and T was seen in the (-50,-1) region for all classes but was most pronounced in the tissue-specific promoters (Figure <xref ref-type=\"fig\" rid=\"F5\">5b,d</xref>). These spikes correspond to the presence of a TATA box and suggest a correlation of this motif with tissue-specific genes (explored more fully later). Conversely, all low-specificity genes (Figure <xref ref-type=\"fig\" rid=\"F5\">5a,c</xref>) shared a common feature in the (+1, +200) region where p(G) &#x0003e; p(C) and p(T) &#x0003e; p(A) that was not seen in tissue-specific genes (Figure <xref ref-type=\"fig\" rid=\"F5\">5b,d</xref>). As shown later, this low-specificity feature could be partially explained by the presence of a YY1 motif. These base-composition differences observed between nonspecific and tissue-specific promoters are likely to indicate motifs that distinguish the two classes.</p></sec><sec><title>Selected transcription factor motifs in the core promoter</title><p>We next examined the distribution of basic core promoter features: the TATA box, the initiator element, and two binding sites for selected ubiquitous transcription factors, Sp1 and YY1, to see if their presence in the proximal promoter was correlated with the tissue specificity of a gene. Two approaches were taken using different datasets and motif-searching methods that gave similar results, providing independent confirmation of results. First, we searched for core motifs using weight matrix hits in promoters of genes selected using <italic>H</italic><sub><italic>g </italic></sub>calculated from the GNF-GEA data. Second, we searched for core motif consensus sites in promoters of genes selected using <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>calculated from EST data.</p></sec><sec><title>TATA boxes are associated with tissue-specific genes</title><p>We grouped the human genes that expressed at least 200 AU (average value) in the GNF-GEA data by entropy and start CpG island status. The number of genes in each category is shown in Table <xref ref-type=\"table\" rid=\"T5\">5</xref> along with a summary of results. We used alignments of position-specific scoring matrices and scoring thresholds included in the Eukaryotic Promoter Database (EPD) [<xref ref-type=\"bibr\" rid=\"B36\">36</xref>] to identify the TATA box and initiator element. Matches to these motifs were preferentially located at the expected positions relative to the transcription start site based on the ratio of the number of observed set to the expected number using a set of random sequences with the same position-dependent base composition as each of the promoters.</p><p>We searched for the TATA box in the (-45, -10) region where the average observed/expected ratio for the TATA box was 3.1. As shown in Table <xref ref-type=\"table\" rid=\"T5\">5</xref>, the most-specific CGI- genes were six times more likely to have a TATA box than the least-specific CGI+ genes (117/215 (54%) versus 183/2072 (9%), <italic>P </italic>&#x02248; 0 exact binomial). Similar numbers are found in mouse (52%/11% = 4.7) This trend also holds within CGI- genes and CGI+ genes. The most specific CGI- genes were three times more likely to have a TATA box than the least specific CGI- genes (117/215 versus 110/607, <italic>P </italic>&#x02248; 0 exact binomial). While less common in CGI+ genes, TATA boxes were still almost four times as likely to be found in the most specific CGI+ genes than the least specific CGI+ genes (19/56 versus 183/2,072, <italic>P </italic>= 2 &#x000d7; 10<sup>-7 </sup>exact binomial). Thus TATA boxes are clearly associated with tissue-specific genes and provide a second axis (with CpG islands) for distinguishing between the most and least specific genes.</p><p>In contrast, the frequency of occurrences of the initiator element (Pol II binding site) was roughly constant across all tissue-specificity classes for both CGI+ and CGI- genes. We searched for the initiator element in the (-10, +10) region. It occurred in 762 of 1,118 (68%) of CGI- genes and 1,273 of 2,434 (52%) of CGI+ genes. Similarly, it occurred in 149 of 215 (69%) of the most specific genes and 388 of 607 (64%) of CGI+ genes. The observed frequency of TATA+/Inr+ promoters was not significantly different from the expected rate assuming independence of the two individual features (data not shown).</p></sec><sec><title>Sp1-binding sites are weakly associated with the least tissue-specific genes</title><p>Sp1 [<xref ref-type=\"bibr\" rid=\"B37\">37</xref>,<xref ref-type=\"bibr\" rid=\"B38\">38</xref>] is a ubiquitous transcription factor with a G-rich binding site with consensus sequence GGGCGGG that might explain the observed G-richness of the 5' UTR in non-specific genes. We used the GC-box weight matrix and scoring threshold from EPD [<xref ref-type=\"bibr\" rid=\"B36\">36</xref>] to identify Sp1 sites. We found that Sp1 sites are preferentially located in the (-150, +1) region in all sets of genes where they occurred on average at twice the expected rate in agreement with previous findings [<xref ref-type=\"bibr\" rid=\"B36\">36</xref>]. In both human and mouse, Sp1 sites were rarely found in the 5' UTR despite the G-richness of this region; they occurred at the expected rate of between 2 and 5%. Thus Sp1 sites were not the cause of the G-richness in the 5' UTR.</p><p>Sp1 sites are associated with CpG islands but are an important component of GGI- promoters as well. Considering just the (-150, +1) region, Sp1 sites occurred in 1,105/2,434 (45%) of human CGI+ gene promoters, and 316/1,118 (28%) of CGI- genes at about 2.5 to 3.0 times the expected frequency in both cases. Frequencies in mouse are 927/2075 (45%) of CGI+ promoters and 464/1652 (28%) CGI- promoters. Sp1 sites were also weakly associated with the least specific genes occurring in 1,105/2,679 (41%) of these genes as compared to 94/271 (32%) in the most tissue-specific genes (<italic>P </italic>= 0.016). Similar numbers are found in the mouse; 38% of the least specific and 26% of the most specific promoters have Sp1 sites. Thus, although Sp1 shows a preference for the least tissue-specific promoters, it is not a strong predictor of the tissue specificity of a gene.</p></sec><sec><title>YY1 binding sites are associated with low-specificity genes</title><p>The transcription factor YY1 [<xref ref-type=\"bibr\" rid=\"B5\">5</xref>-<xref ref-type=\"bibr\" rid=\"B8\">8</xref>] is also ubiquitously expressed and is thought to bind close to [<xref ref-type=\"bibr\" rid=\"B39\">39</xref>] and downstream of the transcription start site. There is evidence that the function of YY1 depends on its orientation [<xref ref-type=\"bibr\" rid=\"B40\">40</xref>]. The location and G-richness of the reverse complement consensus sequence (AANATGGCG) make YY1 a candidate for explaining the prominent G &#x0003e; C feature in the (+1, +200) region of low-specificity genes. We consider YY1 because a YY1-like motif was frequently included among the most statistically significant motifs identified by the motif discovery programs AlignACE [<xref ref-type=\"bibr\" rid=\"B41\">41</xref>] and MEME [<xref ref-type=\"bibr\" rid=\"B42\">42</xref>] in the (+1, +60) region of nonspecific CGI+ promoters (Figure <xref ref-type=\"fig\" rid=\"F6\">6a</xref>). Our form is most similar to the activating form [<xref ref-type=\"bibr\" rid=\"B43\">43</xref>], which may be associated with low-specificity genes. Because of the demonstrated functional sensitivity to the orientation of binding sites we considered each orientation separately. Indeed, as shown in Figure <xref ref-type=\"fig\" rid=\"F6\">6b</xref> we found each orientation exhibits different position preferences. Sites in the reverse orientation (YY1<sub>r</sub>) were preferentially located in the (+1, +25) region but with some elevated levels to +80 bp. Start positions of sites in the forward orientation (YY1<sub>f</sub>) showed a very sharp preference for -3 bp, which probably represents a YY1-like initiator sequence reviewed elsewhere [<xref ref-type=\"bibr\" rid=\"B44\">44</xref>]. Both orientations were found predominantly in the least specific genes (Table <xref ref-type=\"table\" rid=\"T5\">5</xref>). YY1<sub>f </sub>initiator sites are rare; only 55/2,679 (2%) were found above background in human low-specificity genes. The rate in mouse, 22/2,832 (0.8%) of low-specificity promoters, is even lower. The YY1<sub>r </sub>sites are more common and were found above background in 217 (8%) of the 2,679 least specific genes. YY1<sub>r </sub>sites were more common in CGI+ genes than in CGI- genes (202/2,072 (10%) versus 15/607 (2%) <italic>P </italic>= 3.7 &#x000d7; 10<sup>-9 </sup>two-population binomial). The corresponding rates in mouse confirm these observations; 178/2,832 (6%) for all low-specificity genes and 152/1,779 (9%) in CGI+ and 26/1,053 (2%) of CGI- low-specificity promoters. These YY1-like sites therefore constitute a feature strongly associated with the least specific genes and may partially explain the observed G &#x0003e; C ratio in the (+1, +200) region.</p></sec><sec><title><italic>Q</italic>-based analysis of core promoter motifs</title><p>A second analysis of TATA box and Inr motifs was done to determine if the association of the TATA box with tissue-specific genes is also found in genes ranked by <italic>Q </italic>and is robust to using EST data as well as promoters that did not specifically rely on full-length cDNA clones. The definition of <italic>Q </italic>implies that genes with a particular <italic>Q</italic>-value can have a variety of <italic>H</italic><sub><italic>g </italic></sub>values and thus it may be more difficult to identify features related to tissue specificity. We tabulated all DoTS genes that contained at least two ESTs from an islet-cell library then ranked the genes by <italic>Q</italic><sub>pancreas </sub>computed using EST counts. We used <italic>Q</italic><sub>pancreas </sub>&#x02264; 7 bits as the criterion for selecting pancreas-specific genes which we grouped into 2-bit Q intervals. For comparison we selected 50 genes with <italic>Q</italic><sub>pancreas </sub>= 8.5 bits, and 50 genes with 10 &#x02264; <italic>Q</italic><sub>pancreas </sub>&#x02264; 10.6 bits. Genes with high specificity for the pancreas (0 &#x02264; <italic>Q</italic><sub>pancreas </sub>&#x02264; 2 bits, <italic>N </italic>= 9) preferentially had TATA boxes (8 of 9) with half of these also having an initiation element (4 of 9; Figure <xref ref-type=\"fig\" rid=\"F7\">7a</xref>). With decreasing specificity, the fraction of genes containing TATA boxes drops with only18 of 81 (2/9) genes with <italic>Q </italic>&#x0003e; 6 bits having TATA boxes. Thus, the strong correlation of TATA boxes with specific genes found with <italic>H</italic><sub><italic>g </italic></sub>and microarray data was also seen with <italic>Q </italic>and EST data for pancreas-expressed genes. Also consistent is the observation that initiator elements were found at similar frequencies (around 60%) across all specificity classes (Figure <xref ref-type=\"fig\" rid=\"F7\">7b</xref>). Similar patterns were observed in other tissues (data not shown).</p><p>The consistency of findings for the TATA box with human islet genes based on <italic>Q </italic>and ESTs was next tested with orthologous genes in mouse. This test provides a measure for whether the global pattern observed (TATA box with tissue-specific genes) is also found for the same set of genes in another mammal. We also added bins of genes with higher <italic>Q</italic>-values that represent more widely expressed genes. For each human gene, the orthologous mouse gene was determined (see Materials and methods for details) and analyzed as described above. Overall, 18.8% of the human genes and 22.9% of the mouse genes that were analyzed carry the TATA box motif. Except for the last group (<italic>Q </italic>&#x0003e;10 bits) the percentage of the genes with TATA box motifs decreases with the increase in the <italic>Q</italic>-value. This is to be expected since genes with high <italic>Q </italic>may be specific to other tissues and hence are more likely to have a TATA box. Discrepancies between human and mouse promoters were noted for only about 10% of all human-mouse pairs analyzed and may reflect sequence differences and possible annotation discrepancies for the transcription start site. Nevertheless, there is overall excellent agreement for the presence of TATA motifs in human and mouse genes. Thus, our assessment of preferential presence of transcription regulatory motifs in the human pancreas-expressed genes also applies to their mouse orthologs. We conclude that genes expressed with restricted tissue-distribution may be preferentially regulated via TATA-mediated transcription, and that genes with broader expression profiles are more likely to be regulated by non-TATA mediated mechanisms (such as YY1).</p></sec><sec><title>Promoter classes</title><p>Since the presence or absence of a start CpG island and a TATA box appear to be the primary sequence feature that correlate with tissue specificity, we consider them in more detail. We observe that CpG islands and TATA boxes are not mutually exclusive features of promoters and so we consider all possible combinations of these features.</p></sec><sec><title>Frequency of promoter classes</title><p>Figure <xref ref-type=\"fig\" rid=\"F8\">8</xref> shows the cumulative fraction of each class of promoter as a function of increasing <italic>H</italic><sub><italic>g </italic></sub>in human (Figure <xref ref-type=\"fig\" rid=\"F8\">8a</xref>) and mouse (Figure <xref ref-type=\"fig\" rid=\"F8\">8b</xref>). The data from human and mouse follow similar trends even though the mouse has a lower proportion of CGI+ genes. Overall, CGI+/TATA- genes are the most common, at 50-60% depending on the species. Interestingly, the CGI-/TATA- class is the second most common overall, comprising 20-30% of genes, depending on the species. Genes in this promoter class are roughly equally common across the entire entropy range and are the most common promoters in the mid-specificity range in both species. The classes CGI-/TATA+ and CGI+/TATA+ are the least common (8 to 12% overall). CGI-/TATA+ genes are concentrated in the most specific genes. CGI+/TATA+ are found relatively uniformly across all but the most specific genes. Although the TATA box and CpG islands are strongly predictive of a gene's entropy, Figure <xref ref-type=\"fig\" rid=\"F8\">8</xref> also illustrates the limitations of the promoter classes as an explanation for expression patterns. First, although the CGI-/TATA+ and CGI+/TATA- classes are strongly associated with the most and least tissue-specific genes (respectively), instances of genes in each class cover virtually the entire range of tissue specificities. Second, the CGI-/TATA- class is the second most common, illustrating that any degree of tissue specificity can be obtained without these sequence features.</p></sec><sec><title>Functional assessment of promoter classes using Gene Ontology terms</title><p>To try to understand the functional correlates of the four promoter classes, we looked for trends in the cellular localization and biological process of the products of genes from each promoter class. We used the DAVID system [<xref ref-type=\"bibr\" rid=\"B45\">45</xref>,<xref ref-type=\"bibr\" rid=\"B46\">46</xref>], which identifies over-represented Gene Ontology (GO) [<xref ref-type=\"bibr\" rid=\"B47\">47</xref>] terms in a set of genes. A summary of the results for human and mouse genes are shown in Table <xref ref-type=\"table\" rid=\"T6\">6</xref>. In each case the set of genes in each promoter class were compared to all genes on the corresponding Affymetrix chip.</p><p>Products of genes in the CGI-/TATA+ class were often (70/198) located extracellularly. Examples of such genes are the insulin-like growth factor family, serum albumin and chymotrypsin. Some extracellular CGI-/TATA+ genes, such as luteinizing hormone beta (LHB) and bone morphogenetic protein 10 (Bmp10) in the mouse, have a high <italic>H</italic><sub><italic>g </italic></sub>because they are not induced in the tissues or at the developmental stages surveyed, but otherwise fit the pattern of secreted proteins. Gene products that are secreted from the cell must be produced at high level to be effective. Indeed we found the maximum expression level of TATA+ genes is higher than TATA- genes; 454/745 (61%) of TATA+ genes express at least 1,000 AU in one or more tissues, whereas only 1,321/3,773 (35%) of TATA- genes express that highly (<italic>p</italic>-value = 0; two-sample binomial population). A second group of CGI-/TATA+ that is common, but with a <italic>p</italic>-value just over the <italic>p</italic>-value cutoff are the muscle contraction-related genes, actin, troponin and members of the myosin family. Products of these genes are also required in large amounts to create the contractile apparatus but are only produced in a few cell types. The biological processes that are enriched in the CGI-/TATA+ class differ between human and mouse, but nearly all of them are descendants of the GO term 'response to stimulus' (GO:0050896).</p><p>The CGI+/TATA- promoters produce proteins that are typically located in the cell, especially in the cytoplasm and mitochondrion. These locations are consistent with many housekeeping functions. The human results for biological process suggests a large number of housekeeping processes, but these were not confirmed in the mouse using all CGI+/TATA- genes. When we consider just the least specific CGI+/TATA- mouse genes (4.45 &#x02264; <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 5.57 bits), we find cellular locations (including the nucleus) and biological processes that match the human results.</p><p>No significant concentrations of cellular locations or biological processes were found among the CGI+/TATA+ genes. A manual examination of genes in both human and mouse identifies a number of heat-shock proteins, histones and ribosomal proteins although these are not statistically significant as a result of the multiple testing correction. Many of these genes fit the expected expression pattern in that they are widely expressed and at high levels.</p><p>Interestingly, the products of CGI-/TATA- genes are often located in the plasma membrane (244/499 of human genes with a cellular location) and support signaling and response to the environment. Such products, for example, bradykinin receptor B2, prolactin receptor or protocadherin 9, may be expressed in a tissue-specific pattern, but not at the high levels required for secreted proteins. The exact biological process GO terms that are statistically significant vary between mouse and human, but a common core includes defense response (GO:0006952), immune response (GO:0006955) and response to stimulus (GO:0050896). Thus these genes are similar to CGI-/TATA+ genes in that they are involved in response, but are not (typically) required to be expressed at such high levels.</p></sec></sec><sec><title>Discussion</title><p>We have applied Shannon entropy as a novel measure of overall tissue specificity of gene expression and have created a new statistic <italic>Q </italic>to assess the categorical specificity of a gene for a particular tissue. We have evaluated the performance of entropy on microarray-and EST-based estimates of tissue-specific expression and found that it correctly identifies both tissue-specific and housekeeping genes. Ranking and binning genes by entropy allowed us to begin to deconstruct core promoters into features directing when and where the gene will be expressed. We verified and extended previous observations [<xref ref-type=\"bibr\" rid=\"B2\">2</xref>] about the correlation of CpG islands with housekeeping genes and embryonic genes. We then identified differences in the base composition profile of promoters of tissue-specific and nonspecific genes. Next, we identified correlations between, on the one hand, the TATA box and tissue-specific genes, and on the other hand, the YY1 site and nonspecific genes. Finally, we identified trends in promoter classes based on CpG island and TATA box status and associated them with common cellular locations and biological processes. Similar observations were also observed for TATA box and Q-selected genes in pancreas.</p><p>The identification of an association between promoter type and cellular location and biological function, while an important step in a fundamental understanding of biology, also has practical significance, as the genes in the CGI-/TATA+ and CGI-/TATA- classes are enriched for tissue-specific extracellular and cell surface proteins. Such genes are likely to be useful drug targets. Thus entropy <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q </italic>have allowed us to discover fundamental properties of mammalian Pol II promoters and should allow serve to aid understanding of expression in particular tissues of interest.</p><p>The validity of our approach is supported by findings in other work and by the fact that they are robust with respect to the algorithm used to process the expression data. Our finding that most genes are regulated in a tissue-dependent manner is consistent with another analysis of gene expression [<xref ref-type=\"bibr\" rid=\"B14\">14</xref>], which found that housekeeping genes cluster in a tissue-specific manner. Thus, it appears, even the most basic biological functions are subject to regulation. The tissue trees we produced contain relationships similar to those in an analysis [<xref ref-type=\"bibr\" rid=\"B48\">48</xref>] of mid-specificity genes, including the close relation between lung, and the immune system-related organs spleen and thymus. That analysis is based on a different method and a different set of expression data gives us confidence that <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>is properly identifying genes that are specific to a tissue. The GNF-GEA expression data we analyzed was processed with the MAS4 [<xref ref-type=\"bibr\" rid=\"B49\">49</xref>] algorithm. We reanalyzed the data from this study after reprocessing it with the more recent Robust Multichip Average (RMA) algorithm [<xref ref-type=\"bibr\" rid=\"B50\">50</xref>]. This algorithm tends to suppress low-level signals and we found that most genes appeared to be more tissue specific, that is, had lower <italic>H</italic>, in the RMA-processed data compared to the reported values. Although this affects some of the precise values of numbers we have reported it does not alter any of the fundamental trends or results. We include tissue specificities based on both analyses in Additional data files 1 and 2.</p><p>Our analysis focused on only a few sequence features and although we found good correlations, two aspects of our results indicate that there are other regulatory mechanisms not yet identified. First, there is a gradual transition in the frequency of the TATA box and CpG islands between the most and least tissue-specific genes. Second, while these features are strong indicators of high and low specificity, they are far from perfect predictors. Indeed, the middle range of entropies contains a mix of all promoter classes in large numbers, indicating that it is possible to achieve tissue-specific expression with any promoter class. YY1 may be an example of such a supplementary mechanism. While occurring in only 16% of genes, it is very strictly confined to low-specificity genes and is a better indicator of low specificity than CpG islands. We expect that other such signals will be found.</p><p>Anatomical resolution is an issue with the datasets used in this study. For example, the pancreas consists of exocrine cells, ductal cells and islet cells of several types. The bulk pancreas was used to generate the GNF-GEA data, so the reported expression level is the average mRNA concentrations weighted by the cell-type count. This approximation reduces the maximum possible entropy and, more significantly, can make the apparent entropy different from the true entropy. Genes highly and specifically expressed in a cell type with a small population may currently appear to be ubiquitous with very low overall expression. Genes expressed in a few tissues may be revealed to be less tissue specific as more cell types are measured in detail. Genes that appear to be ubiquitously expressed may turn out to not to be expressed in a few cell types. It will be interesting to see whether data with higher anatomical resolution will help to increase the accuracy of the rules we have identified here for identifying tissue-specific and nonspecific promoters.</p><p>Our method can be also applied to other sources of expression data including SAGE, reverse transcription PCR (RT-PCR) and <italic>in situ </italic>hybridization data. SAGE has the advantage of sensitivity, as these studies generally sequence to much greater depths than EST libraries [<xref ref-type=\"bibr\" rid=\"B51\">51</xref>]. <italic>In situ </italic>hybridization data may increase the anatomical resolution of the data. Qualitative intensities, for example, '0', '+', or '+++', can be converted to representative numeric values as appropriate. Our method can also be applied to other collections of conditions beside normal tissues, for example, different types of cancers or samples of the same cancer from multiple patients. Modification of our method to account for temporal changes in tissue specificity represents another direction for future work.</p><p>The analysis presented here focuses on genes rather than on transcripts generated from different promoters from the same gene. The rate of the occurrence of alternative transcription start sites is at least 9% [<xref ref-type=\"bibr\" rid=\"B52\">52</xref>] and may be as high as 25% [<xref ref-type=\"bibr\" rid=\"B53\">53</xref>]. The promoters we used were specified by the DBTSS dataset but there may be alternative promoters with different characteristics and tissue-specific usage patterns. Analyses based on different RNA species can easily be incorporated into our approach and is an area of future research.</p><p>Our results for CpG island frequency in very tissue-specific genes are lower than recent reports [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>] that were based upon present/absent calls, that is, tissue counting, using ESTs to measure tissue specificity. This may be due to two reasons. First, as we described in Results, a significant fraction of genes will show no evidence of expression in poorly sampled tissues. A poorly sampled nonspecific gene will appear therefore more tissue specific than it actually is and this increases the number of apparently tissue-specific genes with CpG islands. Second, when we use microarray data and determine tissue specificity by counting tissues expressing above the median value of 200 AU, we see (data not shown) rates of CpG island occurrence in 'specific' genes similar to those reported in [<xref ref-type=\"bibr\" rid=\"B3\">3</xref>]. Thus, we conclude that including the variation of expression levels rather than mere presence/absence is important for identifying very tissue-specific genes as assessed by start CpG islands.</p><p>These results present an initial look at the correlation between tissue specificity, CpG islands and binding sites for selected transcription factors that interact with the basal transcription apparatus. Using a novel approach with entropy-based metrics, we have begun to lay out the framework for promoter function by identifying strong correlations between tissue-specific or ubiquitous expression and a number of these sequence features. We plan to extend this work in several ways. First, we plan to identify correlations with other known transcription-factor-binding sites and novel motifs identified as over-represented in promoter regions [<xref ref-type=\"bibr\" rid=\"B54\">54</xref>]. Second, these results will help to understand regulation by combinations of multiple upstream transcription factors in genes specific to particular tissues or clusters of tissues.</p></sec><sec><title>Conclusions</title><p>We have used Shannon entropy to quantify and rank the tissue specificity of genes using tissue-survey data. First, this has allowed us to assess the prevalence of tissue-specific regulation; we find that most genes show evidence of some degree of tissue-dependent variation in expression levels. It has also allowed us to find and evaluate associations between promoter features and tissue specificity. We have verified and extended understanding of known associations between, on the one hand, CpG islands and the least tissue-specific genes and, on the other hand, the TATA box and the most tissue-specific genes. However, they are not the sole determinants of tissue-specific expression, as indicated by mid-specificity genes that exhibit a mix of all promoter classes. The class of CGI-/TATA- promoters has emerged as the second most common class of promoter overall and the most common promoter class in mid-specificity genes. Therefore, additional determinants of tissue specificity remain to be found. We have identified one potential determinant, a downstream YY1 site, which is very strongly associated with the least tissue-specific genes but is a relatively rare feature of these promoters. Finally, we have also been able to associate trends in the localization and function of protein products of genes according to their promoter class. Many of the CGI-/TATA+ genes code for highly expressed, very tissue specific, extracellular proteins involved in a cell's response to the environment. CGI-/TATA- genes are also involved in response to the environment, but are found more uniformly across the spectrum of tissue specificity, are not as highly expressed as CGI-/TATA+ genes, and very often code for membrane-bound proteins. CGI+/TATA- genes are more likely to be located in the cytoplasm or nucleus and, as expected, carry out housekeeping functions. All of the results we report are found in both human and mouse and so may reflect general principles of all mammalian species.</p></sec><sec sec-type=\"materials|methods\"><title>Materials and methods</title><sec><title>Processing GNF-GEA [<xref ref-type=\"bibr\" rid=\"B22\">22</xref>] and DoTS [<xref ref-type=\"bibr\" rid=\"B33\">33</xref>] data</title><p>The GNF-GEA data are processed as described [<xref ref-type=\"bibr\" rid=\"B22\">22</xref>]. Given a set of <italic>N </italic>tissues we define <italic>p</italic><sub><italic>t</italic>|<italic>g </italic></sub>= <italic>w</italic><sub><italic>g</italic>,<italic>t</italic></sub>/&#x02211;<sub>1 &#x02264; <italic>t </italic>&#x02264; <italic>N</italic></sub><italic>w</italic><sub><italic>g</italic>,<italic>t </italic></sub>where <italic>w</italic><sub><italic>t </italic></sub>is the expression level of the gene <italic>g </italic>in tissue <italic>t</italic>. DoTS, available through the AllGenes [<xref ref-type=\"bibr\" rid=\"B33\">33</xref>] site, contains ESTs and mRNAs assembled into transcripts that are then clustered into genes. We did not consider any transcript that contains only one EST as this may represent a spurious sequence and did not consider any gene with fewer than five ESTs because they provide a poor estimate of H<sub><italic>g</italic></sub>. To accommodate the great disparity in sampling depth across tissues we normalized EST counts by tissue. To avoid artificially low entropies for genes that contain relatively few ESTs we used pseudocounts to smooth the data. The expression level of a gene in a tissue is computed as <italic>w</italic><sub><italic>g</italic>,<italic>t </italic></sub>= (<italic>n</italic><sub><italic>g</italic>,<italic>t </italic></sub>+ 1)/(N<sub><italic>t </italic></sub>+ <italic>N</italic><sub><italic>g</italic></sub>) where <italic>n</italic><sub><italic>g</italic>,<italic>t </italic></sub>is the number of ESTs from libraries for a tissue included in a gene, <italic>N</italic><sub><italic>t </italic></sub>is the total number of ESTs from a tissue assembled into genes, and <italic>N</italic><sub><italic>g </italic></sub>is the number of genes. We used different sets of tissues depending on the task. <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q </italic>measures in Figure <xref ref-type=\"fig\" rid=\"F1\">1</xref> used the full GNF-GEA mouse set with a few modifications; adipose tissue and brown fat were merged, epidermis and snout epidermis were merged, digits and tongue were not considered as they are both a combination of skeletal muscle and epidermis. The expression level for a set of merged tissues is the median of the individual tissue replicate medians. For comparison of microarray and EST data we used a set of 27 tissues that were common to both datasets and merged the CNS and peripheral nervous system tissues.</p></sec><sec><title>Estimating variance</title><p>To estimate the variance in <italic>H </italic>and Q, we took advantage of tissue replicates in the GNF-GEA data. Using the mouse dataset, we repeatedly sampled one of the measurements from each pair of replicates and computed <italic>H </italic>for each gene. We then computed the variance of the distribution of the estimates of <italic>H </italic>for each gene and show the survivor distribution function in Figure <xref ref-type=\"fig\" rid=\"F2\">2</xref>. The variance of <italic>Q </italic>was computed in a similar manner.</p></sec><sec><title>Clustering tissues</title><p>Clustering was based on the <italic>Q </italic>scores for the set of mouse genes with <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>&#x02264; 7 for at least one tissue and expressing at least 200 AU in at least one tissue in the GNF-GEA data. There were 1,786 Affymetrix probe sets selected. The tree in Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref> was built by sampling 5,000 sets of 1,000 probe sets and clustering tissues using Pearson correlation and a centered measure using the XCLUSTER [<xref ref-type=\"bibr\" rid=\"B55\">55</xref>] program. The consensus tree was built using the program CONSENSE in the PHYLIP [<xref ref-type=\"bibr\" rid=\"B56\">56</xref>] package with the default parameters.</p></sec><sec><title>Identifying genes specific to a set of tissues</title><p>The total entropy of all tissues under a node can be computed at each node in the hierarchy using a generalization of the grouping theorem [<xref ref-type=\"bibr\" rid=\"B57\">57</xref>]. If the entropy of a gene at a node is close to the maximum possible entropy for the number of tissues under the node, then we select it and compute a <italic>Q</italic><sub><italic>g</italic>,<italic>n </italic></sub>for the gene at the node. Using <italic>Q</italic><sub><italic>g</italic>,<italic>n </italic></sub>we can rank genes by specificity to a cluster of tissues just as we can for an individual tissue.</p></sec><sec><title>Predicting CpG islands</title><p>We predicted CpG islands using the program NEWCGREPORT in the EMBOSS [<xref ref-type=\"bibr\" rid=\"B58\">58</xref>] package with the default parameters which require a minimum length of 200 bp, C+G fraction of 0.6 and ratio of observed over expected CpG of 0.5.</p></sec><sec><title>Statistical significance in embryonic expressed genes</title><p>We computed statistical significance of differences between all embryonic-expressed genes and adult-specific rates using a hypergeometric distribution. We start with a collection of <italic>N </italic>CGI+ genes, <italic>n</italic><sub><italic>e </italic></sub>of which are expressed in the embryo, that is, marked as special. The <italic>N</italic><sub><italic>A </italic></sub>tissue-specific genes in the adult are considered a random sample from the original <italic>N </italic>and we compute the probability of finding that at least (or at most) <italic>n</italic><sub><italic>ae </italic></sub>of these were expressed in the embryo.</p></sec><sec><title>Modeling distribution of entropy from uniform genes</title><p>To model the effect of experimental variability, we computed the distribution of the difference between expression levels of individual replicates for each gene and tissue and the mean expression level across replicates as a function of the mean expression level. This distribution was well fit by an exponential distribution with a parameter that depends on the mean expression level. Thus, given an 'ideal' expression level, we can estimate what the experimental variability will be. To model a uniformly expressed gene, we assume that a gene has some average expression level across all tissues and then allow the expression levels in individual tissues to follow a narrow distribution of random fold changes from that level. Specifically, we assumed that the log base 2 of the fold changes is distributed according to a normal distribution with mean equal to 0 and a standard deviation (<italic>s</italic>). The standard deviation can be adjusted to control the amount of biological variation a 'uniformly' expressed gene is allowed to show. For example, setting <italic>s </italic>= 0.5 means that about 68% of the fold changes between a particular tissue and the nominal level are within 1.4 up or down from the nominal level, that is, a twofold change from the lowest to the highest levels. Larger fold changes are expected to occur in 32% of tissues. This model allows significant variation and so is arguably close to the upper limit of variation allowable for a gene that shows no tissue specificity. We also used <italic>s </italic>= 0.25 as a more stringent definition of uniform expression. We sampled mean expression levels from the distribution of observed mean expression levels and sampled entropy values from the probability model. An entropy threshold was estimated by sampling approximately 5,000 random expression profiles and determining the value for a <italic>p</italic>-value of 0.002. This process was repeated 10 times and the corresponding thresholds and fraction of genes were computed. The thresholds spanned a range of less than 0.01 bit. The tissue-dependent gene fractions never varied by more than one percentage point in either direction.</p></sec><sec><title>Statistical significance of co-occurrence</title><p>We estimated the statistical significance of the co-occurrence of motifs using the hypergeometric distribution. Given two motifs with occurrence counts <italic>n</italic><sub>1 </sub>and <italic>n</italic><sub>2</sub>, measured in the same set of <italic>N </italic>promoters, and a co-occurrence count of <italic>n</italic><sub>12</sub>, we compute the significance as the probability of finding no more than (or at least) <italic>n</italic><sub>12 </sub>hits in a random selection of <italic>n</italic><sub>2 </sub>promoters from a pool of <italic>N </italic>promoters where <italic>n</italic><sub>1 </sub>of them are 'special'.</p></sec><sec><title>Comparison of frequency on independent sets</title><p>Given two sets of size <italic>N</italic><sub>1 </sub>and <italic>N</italic><sub>2 </sub>and positive observations <italic>n</italic><sub>1 </sub>and <italic>n</italic><sub>2 </sub>in each, we computed the probability that the underlying rates are different using an exact calculation of the binomial distribution to compute the probability of seeing at least (or no more) than <italic>n</italic><sub><italic>i </italic></sub>matches in <italic>N</italic><sub><italic>i </italic></sub>trials where the rate is assumed to be <italic>r </italic>= <italic>n</italic><sub><italic>j</italic></sub>/<italic>N</italic><sub><italic>j</italic></sub>. We estimated <italic>r </italic>using the larger of the two sets.</p></sec><sec><title>Two binomial populations</title><p>We used the normal approximation to the difference of the proportions normalized by their variance to compute a <italic>z</italic>-score.</p></sec><sec><title>Promoter sequences</title><p>We obtained promoter sequence in two ways. The <italic>H</italic>-based set of analyses used links from Affymetrix probe sets to RefSeq identifiers to select alignments from the DBTSS promoter sequences covering the (-1000, 200) region downloaded from the DBTSS website [<xref ref-type=\"bibr\" rid=\"B59\">59</xref>]. The <italic>Q</italic>-based analyses of TATA box and initiator elements used genomic locations of DoTS genes on UCSC Golden Path release mm3 [<xref ref-type=\"bibr\" rid=\"B60\">60</xref>,<xref ref-type=\"bibr\" rid=\"B61\">61</xref>] to identify gene names. Promoter sequences consisting of the 350 bp of the upstream region were then extracted from Ensembl [<xref ref-type=\"bibr\" rid=\"B62\">62</xref>]. The mouse homologs were also used as annotated in Ensembl.</p></sec><sec><title>Core motifs</title><p>The <italic>H</italic>-based analysis used core promoter element models from EPD [<xref ref-type=\"bibr\" rid=\"B36\">36</xref>,<xref ref-type=\"bibr\" rid=\"B63\">63</xref>]. The fraction of promoters containing each matrix was determined as follows for each set of genes (with and without CpG islands in each entropy bin) individually. Having verified that the positional distribution of each motif was sharply peaked at the appropriate place in the promoter sequences ((-40, -20) region for TATA and (-20, +20) region for the initiator element) we considered only the predictions in these windows from all genes. We used the log-likelihood function to score each subsequence against each matrix using the published score cut-offs. The YY1 motif was found in essentially every run of AlignACE and MEME performed on the downstream regions of ubiquitous CGI+ promoters. We explored different motif widths and other settings and selected version that achieved a combination of good coverage and conservation. In all cases we estimated the background rate of random occurrence of motifs by repeatedly scrambling the individual sequences over a 10 bp window to create approximately 1,000 test sequences for each combination of CpG island status and specificity range. These sequences were scored in the same manner as the unscrambled sequences. We estimated the statistical significance of differences of observed frequencies using exact computation of the binomial distribution. The <italic>Q</italic>-based analyses of core motifs used the TATA box motif (TATAA) and initiator element (YYANWYY). Motif searches were carried out using the tool patternmatch from the biological workbench 3.2 [<xref ref-type=\"bibr\" rid=\"B64\">64</xref>]. Only the TATAA instance located closest to the start of the mRNA's alignment to the genome was used. Matches to the initiator element were required to be downstream of the TATAA box when present.</p></sec><sec><title>YY1 motif</title><p>We used an AlignACE-derived weight matrix (shown in Figure <xref ref-type=\"fig\" rid=\"F6\">6a</xref>) to assess the occurrence of YY1-like sites as it contained the YY1 consensus and was built using approximately 100 sites which is many more than previously published weight matrices [<xref ref-type=\"bibr\" rid=\"B43\">43</xref>,<xref ref-type=\"bibr\" rid=\"B65\">65</xref>] also shown in Figure <xref ref-type=\"fig\" rid=\"F6\">6a</xref>.</p></sec><sec><title>GO association analysis</title><p>We submitted Affymetrix probe set ids of interest to the DAVID website [<xref ref-type=\"bibr\" rid=\"B45\">45</xref>,<xref ref-type=\"bibr\" rid=\"B46\">46</xref>] and compared them either to all probe sets on the appropriate Affymetrix chips or to all genes in the selected entropy range. We compensated for multiple testing by requiring the reported <italic>p</italic>-values be better than either 0.05/1472 = 0.00003 (cellular component) or 0.05/8972 = 0.000006 (biological process) using the number of GO terms for the corresponding GO divisions in a Bonferroni correction.</p></sec><sec><title>RMA quantification</title><p>We obtained CEL files for the GNF-GEA study from and re-quantified them using the gcrma package [<xref ref-type=\"bibr\" rid=\"B66\">66</xref>] in the Bioconductor [<xref ref-type=\"bibr\" rid=\"B67\">67</xref>] project for the R statistical analysis program [<xref ref-type=\"bibr\" rid=\"B68\">68</xref>]. We use the gcrma options 'type=c('fullmodel')' and 'fast=T'.</p></sec></sec><sec><title>Additional data files</title><p>Two additional data files are available with the online version of this article. They contain H and Q values for all normal tissues in the GNF-GEA data set for both human (Additional data file <xref ref-type=\"supplementary-material\" rid=\"S1\">1</xref>) and mouse (Additional data file <xref ref-type=\"supplementary-material\" rid=\"S2\">2</xref>) using both the MAS4 and RMA quantification methods. The RMA data were normalized to yield a common median of 3.75 (human) and 3.22 (mouse) prior to the H and Q calculation. The files are in Excel format. The data for each tissue are placed in separate worksheets. Each worksheet contains H- and Q-values, the expression value of the gene in the worksheet's tissue, and its maximum expression across all tissues in the file, the gene symbol, RefSeq, SwissProt, and Unigene ID, and a description. The rows in each worksheet are sorted by increasing values of Q using the RMA data. Thus the top of each worksheet displays the genes most specific to that worksheet's tissue.</p></sec><sec sec-type=\"supplementary-material\"><title>Supplementary Material</title><supplementary-material content-type=\"local-data\" id=\"S1\"><caption><title>Additional File 1</title><p>A table showing H and Q values for all normal human tissues in the GNF-GEA dataset. H and Q values for all normal tissues in the GNF-GEA dataset for human using both the original MAS4 quantification and our RMA re-quantification. The RMA data were normalized to yield common medians of 3.75 prior to the H and Q calculation. The data for each tissue are placed in separate worksheets. Each worksheet contains H- and Q-values, the expression value of the gene in the worksheet's tissue, and its maximum expression across all tissues in the file, the gene symbol, RefSeq, SwissProt, and Unigene ID, and a description. The rows in each worksheet are sorted by increasing values of Q using the RMA data. Thus the top of each worksheet displays the genes most specific to that worksheet's tissue.</p></caption><media xlink:href=\"gb-2005-6-4-r33-S1.xls\" mimetype=\"application\" mime-subtype=\"vnd.ms-excel\"><caption><p>Click here for file</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"S2\"><caption><title>Additional File 2</title><p>A table showing H and Q values for all normal mouse tissues in the GNF-GEA dataset. H and Q values for all normal tissues in the GNF-GEA dataset for mouse using both the original MAS4 quantification and our RMA re-quantification. The RMA data were normalized to yield common medians of 3.22 prior to the H and Q calculation. The data for each tissue are placed in separate worksheets. Each worksheet contains H- and Q-values, the expression value of the gene in the worksheet's tissue, and its maximum expression across all tissues in the file, the gene symbol, RefSeq, SwissProt, and Unigene ID, and a description. The rows in each worksheet are sorted by increasing values of Q using the RMA data. Thus the top of each worksheet displays the genes most specific to that worksheet's tissue.</p></caption><media xlink:href=\"gb-2005-6-4-r33-S2.xls\" mimetype=\"application\" mime-subtype=\"vnd.ms-excel\"><caption><p>Click here for file</p></caption></media></supplementary-material></sec></body><back><ack><sec><title>Acknowledgements</title><p>J.S. thanks J. Mazzarelli, M. Mintz and S. Hannenhalli for many helpful discussions, E. Manduchi and H. He for help with R and RMA, J. Hogenesch and J. Walker at Novartis for providing timely access to the CEL files for the GNF-GEA data, and T. Kadesh for critical readings of the manuscript. C.S. acknowledges support from NIH R01HG001539. J.M.S. and W.-P.S. in C.K.'s lab were supported by an R01 grant 1R01DK63336.</p></sec></ack><ref-list><ref id=\"B1\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bird</surname><given-names>AP</given-names></name></person-group><article-title>DNA methylation - how important in gene control?</article-title><source>Nature</source><year>1984</year><volume>307</volume><fpage>503</fpage><lpage>504</lpage><pub-id pub-id-type=\"pmid\">6320010</pub-id><pub-id pub-id-type=\"doi\">10.1038/307503a0</pub-id></citation></ref><ref id=\"B2\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bird</surname><given-names>AP</given-names></name></person-group><article-title>DNA methylation versus gene expression.</article-title><source>J Embryol Exp Morphol</source><year>1984</year><volume>83</volume><issue>Suppl</issue><fpage>31</fpage><lpage>40</lpage><pub-id pub-id-type=\"pmid\">6598198</pub-id></citation></ref><ref id=\"B3\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ponger</surname><given-names>L</given-names></name><name><surname>Duret</surname><given-names>L</given-names></name><name><surname>Mouchiroud</surname><given-names>D</given-names></name></person-group><article-title>Determinants of CpG islands: expression in early embryo and isochore structure.</article-title><source>Genome Res</source><year>2001</year><volume>11</volume><fpage>1854</fpage><lpage>1860</lpage><pub-id pub-id-type=\"pmid\">11691850</pub-id></citation></ref><ref id=\"B4\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Smale</surname><given-names>ST</given-names></name><name><surname>Baltimore</surname><given-names>D</given-names></name></person-group><article-title>The 'initiator' as a transcription control element.</article-title><source>Cell</source><year>1989</year><volume>57</volume><fpage>103</fpage><lpage>113</lpage><pub-id pub-id-type=\"pmid\">2467742</pub-id><pub-id pub-id-type=\"doi\">10.1016/0092-8674(89)90176-1</pub-id></citation></ref><ref id=\"B5\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Seto</surname><given-names>E</given-names></name><name><surname>Chang</surname><given-names>LS</given-names></name><name><surname>Shenk</surname><given-names>T</given-names></name></person-group><article-title>Transcriptional repression by YY1, a human GLI-Kruppel-related protein, and relief of repression by adenovirus E1A protein.</article-title><source>Cell</source><year>1991</year><volume>67</volume><fpage>377</fpage><lpage>388</lpage><pub-id pub-id-type=\"pmid\">1655281</pub-id><pub-id pub-id-type=\"doi\">10.1016/0092-8674(91)90189-6</pub-id></citation></ref><ref id=\"B6\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Seto</surname><given-names>E</given-names></name><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Shenk</surname><given-names>T</given-names></name></person-group><article-title>YY1 is an initiator sequence-binding protein that directs and activates transcription in vitro.</article-title><source>Nature</source><year>1991</year><volume>354</volume><fpage>241</fpage><lpage>245</lpage><pub-id pub-id-type=\"pmid\">1720509</pub-id><pub-id pub-id-type=\"doi\">10.1038/354241a0</pub-id></citation></ref><ref id=\"B7\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Riggs</surname><given-names>KJ</given-names></name><name><surname>Saleque</surname><given-names>S</given-names></name><name><surname>Wong</surname><given-names>KK</given-names></name><name><surname>Merrell</surname><given-names>KT</given-names></name><name><surname>Lee</surname><given-names>JS</given-names></name><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Calame</surname><given-names>K</given-names></name></person-group><article-title>Yin-yang 1 activates the c-myc promoter.</article-title><source>Mol Cell Biol</source><year>1993</year><volume>13</volume><fpage>7487</fpage><lpage>7495</lpage><pub-id pub-id-type=\"pmid\">8246966</pub-id></citation></ref><ref id=\"B8\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Riggs</surname><given-names>KJ</given-names></name><name><surname>Merrell</surname><given-names>KT</given-names></name><name><surname>Wilson</surname><given-names>G</given-names></name><name><surname>Calame</surname><given-names>K</given-names></name></person-group><article-title>Common factor 1 is a transcriptional activator which binds in the c-myc promoter, the skeletal alpha-actin promoter, and the immunoglobulin heavy-chain enhancer.</article-title><source>Mol Cell Biol</source><year>1991</year><volume>11</volume><fpage>1765</fpage><lpage>1769</lpage><pub-id pub-id-type=\"pmid\">1899910</pub-id></citation></ref><ref id=\"B9\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wasserman</surname><given-names>WW</given-names></name><name><surname>Fickett</surname><given-names>JW</given-names></name></person-group><article-title>Identification of regulatory regions which confer muscle-specific gene expression.</article-title><source>J Mol Biol</source><year>1998</year><volume>278</volume><fpage>167</fpage><lpage>181</lpage><pub-id pub-id-type=\"pmid\">9571041</pub-id><pub-id pub-id-type=\"doi\">10.1006/jmbi.1998.1700</pub-id></citation></ref><ref id=\"B10\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Krivan</surname><given-names>W</given-names></name><name><surname>Wasserman</surname><given-names>WW</given-names></name></person-group><article-title>A predictive model for regulatory sequences directing liver-specific transcription.</article-title><source>Genome Res</source><year>2001</year><volume>11</volume><fpage>1559</fpage><lpage>1566</lpage><pub-id pub-id-type=\"pmid\">11544200</pub-id><pub-id pub-id-type=\"doi\">10.1101/gr.180601</pub-id></citation></ref><ref id=\"B11\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ringrose</surname><given-names>L</given-names></name><name><surname>Rehmsmeier</surname><given-names>M</given-names></name><name><surname>Dura</surname><given-names>JM</given-names></name><name><surname>Paro</surname><given-names>R</given-names></name></person-group><article-title>Genome-wide prediction of Polycomb/Trithorax response elements in <italic>Drosophila melanogaster</italic>.</article-title><source>Dev Cell</source><year>2003</year><volume>5</volume><fpage>759</fpage><lpage>771</lpage><pub-id pub-id-type=\"pmid\">14602076</pub-id><pub-id pub-id-type=\"doi\">10.1016/S1534-5807(03)00337-X</pub-id></citation></ref><ref id=\"B12\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Berman</surname><given-names>BP</given-names></name><name><surname>Pfeiffer</surname><given-names>BD</given-names></name><name><surname>Laverty</surname><given-names>TR</given-names></name><name><surname>Salzberg</surname><given-names>SL</given-names></name><name><surname>Rubin</surname><given-names>GM</given-names></name><name><surname>Eisen</surname><given-names>MB</given-names></name><name><surname>Celniker</surname><given-names>SE</given-names></name></person-group><article-title>Computational identification of developmental enhancers: conservation and function of transcription factor binding-site clusters in <italic>Drosophila melanogaster </italic>and <italic>Drosophila pseudoobscura</italic>.</article-title><source>Genome Biol</source><year>2004</year><volume>5</volume><fpage>R61</fpage><pub-id pub-id-type=\"pmid\">15345045</pub-id><pub-id pub-id-type=\"doi\">10.1186/gb-2004-5-9-r61</pub-id></citation></ref><ref id=\"B13\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wasserman</surname><given-names>WW</given-names></name><name><surname>Sandelin</surname><given-names>A</given-names></name></person-group><article-title>Applied bioinformatics for the identification of regulatory elements.</article-title><source>Nat Rev Genet</source><year>2004</year><volume>5</volume><fpage>276</fpage><lpage>287</lpage><pub-id pub-id-type=\"pmid\">15131651</pub-id><pub-id pub-id-type=\"doi\">10.1038/nrg1315</pub-id></citation></ref><ref id=\"B14\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hsiao</surname><given-names>LL</given-names></name><name><surname>Dangond</surname><given-names>F</given-names></name><name><surname>Yoshida</surname><given-names>T</given-names></name><name><surname>Hong</surname><given-names>R</given-names></name><name><surname>Jensen</surname><given-names>RV</given-names></name><name><surname>Misra</surname><given-names>J</given-names></name><name><surname>Dillon</surname><given-names>W</given-names></name><name><surname>Lee</surname><given-names>KF</given-names></name><name><surname>Clark</surname><given-names>KE</given-names></name><name><surname>Haverty</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>A compendium of gene expression in normal human tissues.</article-title><source>Physiol Genomics</source><year>2001</year><volume>7</volume><fpage>97</fpage><lpage>104</lpage><pub-id pub-id-type=\"pmid\">11773596</pub-id></citation></ref><ref id=\"B15\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Arbeitman</surname><given-names>MN</given-names></name><name><surname>Furlong</surname><given-names>EE</given-names></name><name><surname>Imam</surname><given-names>F</given-names></name><name><surname>Johnson</surname><given-names>E</given-names></name><name><surname>Null</surname><given-names>BH</given-names></name><name><surname>Baker</surname><given-names>BS</given-names></name><name><surname>Krasnow</surname><given-names>MA</given-names></name><name><surname>Scott</surname><given-names>MP</given-names></name><name><surname>Davis</surname><given-names>RW</given-names></name><name><surname>White</surname><given-names>KP</given-names></name></person-group><article-title>Gene expression during the life cycle of Drosophila melanogaster.</article-title><source>Science</source><year>2002</year><volume>297</volume><fpage>2270</fpage><lpage>2275</lpage><pub-id pub-id-type=\"pmid\">12351791</pub-id><pub-id pub-id-type=\"doi\">10.1126/science.1072152</pub-id></citation></ref><ref id=\"B16\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Waterston</surname><given-names>RH</given-names></name><name><surname>Lindblad-Toh</surname><given-names>K</given-names></name><name><surname>Birney</surname><given-names>E</given-names></name><name><surname>Rogers</surname><given-names>J</given-names></name><name><surname>Abril</surname><given-names>JF</given-names></name><name><surname>Agarwal</surname><given-names>P</given-names></name><name><surname>Agarwala</surname><given-names>R</given-names></name><name><surname>Ainscough</surname><given-names>R</given-names></name><name><surname>Alexandersson</surname><given-names>M</given-names></name><name><surname>An</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>Initial sequencing and comparative analysis of the mouse genome.</article-title><source>Nature</source><year>2002</year><volume>420</volume><fpage>520</fpage><lpage>562</lpage><pub-id pub-id-type=\"pmid\">12466850</pub-id><pub-id pub-id-type=\"doi\">10.1038/nature01262</pub-id></citation></ref><ref id=\"B17\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Suzuki</surname><given-names>Y</given-names></name><name><surname>Yamashita</surname><given-names>R</given-names></name><name><surname>Sugano</surname><given-names>S</given-names></name><name><surname>Nakai</surname><given-names>K</given-names></name></person-group><article-title>DBTSS, DataBase of Transcriptional Start Sites: progress report 2004.</article-title><source>Nucleic Acids Res</source><year>2004</year><volume>32</volume><issue>Database</issue><fpage>D78</fpage><lpage>D81</lpage><pub-id pub-id-type=\"pmid\">14681363</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkh076</pub-id></citation></ref><ref id=\"B18\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Carninci</surname><given-names>P</given-names></name><name><surname>Waki</surname><given-names>K</given-names></name><name><surname>Shiraki</surname><given-names>T</given-names></name><name><surname>Konno</surname><given-names>H</given-names></name><name><surname>Shibata</surname><given-names>K</given-names></name><name><surname>Itoh</surname><given-names>M</given-names></name><name><surname>Aizawa</surname><given-names>K</given-names></name><name><surname>Arakawa</surname><given-names>T</given-names></name><name><surname>Ishii</surname><given-names>Y</given-names></name><name><surname>Sasaki</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Targeting a complex transcriptome: the construction of the mouse full-length cDNA encyclopedia.</article-title><source>Genome Res</source><year>2003</year><volume>13</volume><fpage>1273</fpage><lpage>1289</lpage><pub-id pub-id-type=\"pmid\">12819125</pub-id><pub-id pub-id-type=\"doi\">10.1101/gr.1119703</pub-id></citation></ref><ref id=\"B19\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Strausberg</surname><given-names>RL</given-names></name><name><surname>Feingold</surname><given-names>EA</given-names></name><name><surname>Klausner</surname><given-names>RD</given-names></name><name><surname>Collins</surname><given-names>FS</given-names></name></person-group><article-title>The mammalian gene collection.</article-title><source>Science</source><year>1999</year><volume>286</volume><fpage>455</fpage><lpage>457</lpage><pub-id pub-id-type=\"pmid\">10521335</pub-id><pub-id pub-id-type=\"doi\">10.1126/science.286.5439.455</pub-id></citation></ref><ref id=\"B20\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gitton</surname><given-names>Y</given-names></name><name><surname>Dahmane</surname><given-names>N</given-names></name><name><surname>Baik</surname><given-names>S</given-names></name><name><surname>Ruiz</surname><given-names>i</given-names></name><name><surname>Altaba</surname><given-names>A</given-names></name><name><surname>Neidhardt</surname><given-names>L</given-names></name><name><surname>Scholze</surname><given-names>M</given-names></name><name><surname>Herrmann</surname><given-names>BG</given-names></name><name><surname>Kahlem</surname><given-names>P</given-names></name><name><surname>Benkahla</surname><given-names>A</given-names></name><name><surname>Schrinner</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>A gene expression map of human chromosome 21 orthologues in the mouse.</article-title><source>Nature</source><year>2002</year><volume>420</volume><fpage>586</fpage><lpage>590</lpage><pub-id pub-id-type=\"pmid\">12466855</pub-id><pub-id pub-id-type=\"doi\">10.1038/nature01270</pub-id></citation></ref><ref id=\"B21\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reymond</surname><given-names>A</given-names></name><name><surname>Marigo</surname><given-names>V</given-names></name><name><surname>Yaylaoglu</surname><given-names>MB</given-names></name><name><surname>Leoni</surname><given-names>A</given-names></name><name><surname>Ucla</surname><given-names>C</given-names></name><name><surname>Scamuffa</surname><given-names>N</given-names></name><name><surname>Caccioppoli</surname><given-names>C</given-names></name><name><surname>Dermitzakis</surname><given-names>ET</given-names></name><name><surname>Lyle</surname><given-names>R</given-names></name><name><surname>Banfi</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Human chromosome 21 gene expression atlas in the mouse.</article-title><source>Nature</source><year>2002</year><volume>420</volume><fpage>582</fpage><lpage>586</lpage><pub-id pub-id-type=\"pmid\">12466854</pub-id><pub-id pub-id-type=\"doi\">10.1038/nature01178</pub-id></citation></ref><ref id=\"B22\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Su</surname><given-names>AI</given-names></name><name><surname>Cooke</surname><given-names>MP</given-names></name><name><surname>Ching</surname><given-names>KA</given-names></name><name><surname>Hakak</surname><given-names>Y</given-names></name><name><surname>Walker</surname><given-names>JR</given-names></name><name><surname>Wiltshire</surname><given-names>T</given-names></name><name><surname>Orth</surname><given-names>AP</given-names></name><name><surname>Vega</surname><given-names>RG</given-names></name><name><surname>Sapinoso</surname><given-names>LM</given-names></name><name><surname>Moqrich</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Large-scale analysis of the human and mouse transcriptomes.</article-title><source>Proc Natl Acad Sci USA</source><year>2002</year><volume>99</volume><fpage>4465</fpage><lpage>4470</lpage><pub-id pub-id-type=\"pmid\">11904358</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.012025199</pub-id></citation></ref><ref id=\"B23\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Safran</surname><given-names>M</given-names></name><name><surname>Chalifa-Caspi</surname><given-names>V</given-names></name><name><surname>Shmueli</surname><given-names>O</given-names></name><name><surname>Olender</surname><given-names>T</given-names></name><name><surname>Lapidot</surname><given-names>M</given-names></name><name><surname>Rosen</surname><given-names>N</given-names></name><name><surname>Shmoish</surname><given-names>M</given-names></name><name><surname>Peter</surname><given-names>Y</given-names></name><name><surname>Glusman</surname><given-names>G</given-names></name><name><surname>Feldmesser</surname><given-names>E</given-names></name><etal></etal></person-group><article-title>Human gene-centric databases at the Weizmann Institute of Science: GeneCards, UDB, CroW 21 and HORDE.</article-title><source>Nucleic Acids Res</source><year>2003</year><volume>31</volume><fpage>142</fpage><lpage>146</lpage><pub-id pub-id-type=\"pmid\">12519968</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkg050</pub-id></citation></ref><ref id=\"B24\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hayashizaki</surname><given-names>Y</given-names></name></person-group><article-title>RIKEN mouse genome encyclopedia.</article-title><source>Mech Ageing Dev</source><year>2003</year><volume>124</volume><fpage>93</fpage><lpage>102</lpage><pub-id pub-id-type=\"pmid\">12618011</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0047-6374(02)00173-2</pub-id></citation></ref><ref id=\"B25\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wheeler</surname><given-names>DL</given-names></name><name><surname>Church</surname><given-names>DM</given-names></name><name><surname>Federhen</surname><given-names>S</given-names></name><name><surname>Lash</surname><given-names>AE</given-names></name><name><surname>Madden</surname><given-names>TL</given-names></name><name><surname>Pontius</surname><given-names>JU</given-names></name><name><surname>Schuler</surname><given-names>GD</given-names></name><name><surname>Schriml</surname><given-names>LM</given-names></name><name><surname>Sequeira</surname><given-names>E</given-names></name><name><surname>Tatusova</surname><given-names>TA</given-names></name><etal></etal></person-group><article-title>Database resources of the National Center for Biotechnology.</article-title><source>Nucleic Acids Res</source><year>2003</year><volume>31</volume><fpage>28</fpage><lpage>33</lpage><pub-id pub-id-type=\"pmid\">12519941</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkg033</pub-id></citation></ref><ref id=\"B26\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Boguski</surname><given-names>MS</given-names></name><name><surname>Lowe</surname><given-names>TM</given-names></name><name><surname>Tolstoshev</surname><given-names>CM</given-names></name></person-group><article-title>dbEST - database for 'expressed sequence tags'.</article-title><source>Nat Genet</source><year>1993</year><volume>4</volume><fpage>332</fpage><lpage>333</lpage><pub-id pub-id-type=\"pmid\">8401577</pub-id><pub-id pub-id-type=\"doi\">10.1038/ng0893-332</pub-id></citation></ref><ref id=\"B27\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huminiecki</surname><given-names>L</given-names></name><name><surname>Lloyd</surname><given-names>AT</given-names></name><name><surname>Wolfe</surname><given-names>KH</given-names></name></person-group><article-title>Congruence of tissue expression profiles from Gene Expression Atlas, SAGEmap and TissueInfo databases.</article-title><source>BMC Genomics</source><year>2003</year><volume>4</volume><fpage>31</fpage><pub-id pub-id-type=\"pmid\">12885301</pub-id><pub-id pub-id-type=\"doi\">10.1186/1471-2164-4-31</pub-id></citation></ref><ref id=\"B28\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vinogradov</surname><given-names>AE</given-names></name></person-group><article-title>Isochores and tissue-specificity.</article-title><source>Nucleic Acids Res</source><year>2003</year><volume>31</volume><fpage>5212</fpage><lpage>5220</lpage><pub-id pub-id-type=\"pmid\">12930973</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkg699</pub-id></citation></ref><ref id=\"B29\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Stanton</surname><given-names>JA</given-names></name><name><surname>Macgregor</surname><given-names>AB</given-names></name><name><surname>Green</surname><given-names>DP</given-names></name></person-group><article-title>Identifying tissue-enriched gene expression in mouse tissues using the NIH UniGene database.</article-title><source>Appl Bioinformatics</source><year>2003</year><volume>2</volume><fpage>S65</fpage><lpage>S73</lpage><pub-id pub-id-type=\"pmid\">15130819</pub-id></citation></ref><ref id=\"B30\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fuhrman</surname><given-names>S</given-names></name><name><surname>Cunningham</surname><given-names>MJ</given-names></name><name><surname>Wen</surname><given-names>X</given-names></name><name><surname>Zweiger</surname><given-names>G</given-names></name><name><surname>Seilhamer</surname><given-names>JJ</given-names></name><name><surname>Somogyi</surname><given-names>R</given-names></name></person-group><article-title>The application of shannon entropy in the identification of putative drug targets.</article-title><source>Biosystems</source><year>2000</year><volume>55</volume><fpage>5</fpage><lpage>14</lpage><pub-id pub-id-type=\"pmid\">10745103</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0303-2647(99)00077-5</pub-id></citation></ref><ref id=\"B31\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cunningham</surname><given-names>MJ</given-names></name><name><surname>Liang</surname><given-names>S</given-names></name><name><surname>Fuhrman</surname><given-names>S</given-names></name><name><surname>Seilhamer</surname><given-names>JJ</given-names></name><name><surname>Somogyi</surname><given-names>R</given-names></name></person-group><article-title>Gene expression microarray data analysis for toxicology profiling.</article-title><source>Ann NY Acad Sci</source><year>2000</year><volume>919</volume><fpage>52</fpage><lpage>67</lpage><pub-id pub-id-type=\"pmid\">11083097</pub-id></citation></ref><ref id=\"B32\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peterson</surname><given-names>LE</given-names></name></person-group><article-title>CLUSFAVOR 5.0: hierarchical cluster and principal-component analysis of microarray-based transcriptional profiles.</article-title><source>Genome Biol</source><year>2002</year><volume>3</volume><fpage>software0002.1</fpage><lpage>0002.8</lpage><pub-id pub-id-type=\"pmid\">12184816</pub-id><pub-id pub-id-type=\"doi\">10.1186/gb-2002-3-7-software0002</pub-id></citation></ref><ref id=\"B33\"><citation citation-type=\"other\"><article-title>DoTS</article-title><ext-link ext-link-type=\"uri\" xlink:href=\"http://www.allgenes.org\"/></citation></ref><ref id=\"B34\"><citation citation-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Shannon</surname><given-names>C</given-names></name></person-group><source>The Mathematical Theory of Communication</source><year>1949</year><publisher-name>Urbana: University of Illinois Press</publisher-name></citation></ref><ref id=\"B35\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Eisenberg</surname><given-names>E</given-names></name><name><surname>Levanon</surname><given-names>EY</given-names></name></person-group><article-title>Human housekeeping genes are compact.</article-title><source>Trends Genet</source><year>2003</year><volume>19</volume><fpage>362</fpage><lpage>365</lpage><pub-id pub-id-type=\"pmid\">12850439</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0168-9525(03)00140-9</pub-id></citation></ref><ref id=\"B36\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bucher</surname><given-names>P</given-names></name></person-group><article-title>Weight matrix descriptions of four eukaryotic RNA polymerase II promoter elements derived from 502 unrelated promoter sequences.</article-title><source>J Mol Biol</source><year>1990</year><volume>212</volume><fpage>563</fpage><lpage>578</lpage><pub-id pub-id-type=\"pmid\">2329577</pub-id><pub-id pub-id-type=\"doi\">10.1016/0022-2836(90)90223-9</pub-id></citation></ref><ref id=\"B37\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cook</surname><given-names>T</given-names></name><name><surname>Gebelein</surname><given-names>B</given-names></name><name><surname>Urrutia</surname><given-names>R</given-names></name></person-group><article-title>Sp1 and its likes: biochemical and functional predictions for a growing family of zinc finger transcription factors.</article-title><source>Ann NY Acad Sci</source><year>1999</year><volume>880</volume><fpage>94</fpage><lpage>102</lpage><pub-id pub-id-type=\"pmid\">10415854</pub-id></citation></ref><ref id=\"B38\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>S</given-names></name><name><surname>Sun</surname><given-names>JM</given-names></name><name><surname>Davie</surname><given-names>JR</given-names></name></person-group><article-title>Gene regulation by Sp1 and Sp3.</article-title><source>Biochem Cell Biol</source><year>2004</year><volume>82</volume><fpage>460</fpage><lpage>471</lpage><pub-id pub-id-type=\"pmid\">15284899</pub-id><pub-id pub-id-type=\"doi\">10.1139/o04-045</pub-id></citation></ref><ref id=\"B39\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>JS</given-names></name><name><surname>Galvin</surname><given-names>KM</given-names></name><name><surname>Shi</surname><given-names>Y</given-names></name></person-group><article-title>Evidence for physical interaction between the zinc-finger transcription factors YY1 and Sp1.</article-title><source>Proc Natl Acad Sci USA</source><year>1993</year><volume>90</volume><fpage>6145</fpage><lpage>6149</lpage><pub-id pub-id-type=\"pmid\">8327494</pub-id></citation></ref><ref id=\"B40\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Natesan</surname><given-names>S</given-names></name><name><surname>Gilman</surname><given-names>MZ</given-names></name></person-group><article-title>DNA bending and orientation-dependent function of YY1 in the c-<italic>fos </italic>promoter.</article-title><source>Genes Dev</source><year>1993</year><volume>7</volume><fpage>2497</fpage><lpage>2509</lpage><pub-id pub-id-type=\"pmid\">8276234</pub-id></citation></ref><ref id=\"B41\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McGuire</surname><given-names>AM</given-names></name><name><surname>Hughes</surname><given-names>JD</given-names></name><name><surname>Church</surname><given-names>GM</given-names></name></person-group><article-title>Conservation of DNA regulatory motifs and discovery of new motifs in microbial genomes.</article-title><source>Genome Res</source><year>2000</year><volume>10</volume><fpage>744</fpage><lpage>757</lpage><pub-id pub-id-type=\"pmid\">10854408</pub-id><pub-id pub-id-type=\"doi\">10.1101/gr.10.6.744</pub-id></citation></ref><ref id=\"B42\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bailey</surname><given-names>TL</given-names></name><name><surname>Baker</surname><given-names>ME</given-names></name><name><surname>Elkan</surname><given-names>CP</given-names></name></person-group><article-title>An artificial intelligence approach to motif discovery in protein sequences: application to steroid dehydrogenases.</article-title><source>J Steroid Biochem Mol Biol</source><year>1997</year><volume>62</volume><fpage>29</fpage><lpage>44</lpage><pub-id pub-id-type=\"pmid\">9366496</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0960-0760(97)00013-7</pub-id></citation></ref><ref id=\"B43\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shrivastava</surname><given-names>A</given-names></name><name><surname>Calame</surname><given-names>K</given-names></name></person-group><article-title>An analysis of genes regulated by the multi-functional transcriptional regulator Yin Yang-1.</article-title><source>Nucleic Acids Res</source><year>1994</year><volume>22</volume><fpage>5151</fpage><lpage>5155</lpage><pub-id pub-id-type=\"pmid\">7816599</pub-id></citation></ref><ref id=\"B44\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Smale</surname><given-names>ST</given-names></name></person-group><article-title>Transcription initiation from TATA-less promoters within eukaryotic protein-coding genes.</article-title><source>Biochim Biophys Acta</source><year>1997</year><volume>1351</volume><fpage>73</fpage><lpage>88</lpage><pub-id pub-id-type=\"pmid\">9116046</pub-id></citation></ref><ref id=\"B45\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hosack</surname><given-names>DA</given-names></name><name><surname>Dennis</surname><given-names>G</given-names><suffix>Jr</suffix></name><name><surname>Sherman</surname><given-names>BT</given-names></name><name><surname>Lane</surname><given-names>HC</given-names></name><name><surname>Lempicki</surname><given-names>RA</given-names></name></person-group><article-title>Identifying biological themes within lists of genes with EASE.</article-title><source>Genome Biol</source><year>2003</year><volume>4</volume><fpage>R70</fpage><pub-id pub-id-type=\"pmid\">14519205</pub-id><pub-id pub-id-type=\"doi\">10.1186/gb-2003-4-10-r70</pub-id></citation></ref><ref id=\"B46\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dennis</surname><given-names>G</given-names><suffix>Jr</suffix></name><name><surname>Sherman</surname><given-names>BT</given-names></name><name><surname>Hosack</surname><given-names>DA</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Lane</surname><given-names>HC</given-names></name><name><surname>Lempicki</surname><given-names>RA</given-names></name></person-group><article-title>DAVID: Database for Annotation, Visualization, and Integrated Discovery.</article-title><source>Genome Biol</source><year>2003</year><volume>4</volume><fpage>P3</fpage><pub-id pub-id-type=\"pmid\">12734009</pub-id><pub-id pub-id-type=\"doi\">10.1186/gb-2003-4-5-p3</pub-id></citation></ref><ref id=\"B47\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Harris</surname><given-names>MA</given-names></name><name><surname>Clark</surname><given-names>J</given-names></name><name><surname>Ireland</surname><given-names>A</given-names></name><name><surname>Lomax</surname><given-names>J</given-names></name><name><surname>Ashburner</surname><given-names>M</given-names></name><name><surname>Foulger</surname><given-names>R</given-names></name><name><surname>Eilbeck</surname><given-names>K</given-names></name><name><surname>Lewis</surname><given-names>S</given-names></name><name><surname>Marshall</surname><given-names>B</given-names></name><name><surname>Mungall</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>The Gene Ontology (GO) database and informatics resource.</article-title><source>Nucleic Acids Res</source><year>2004</year><volume>32</volume><issue>Database</issue><fpage>D258</fpage><lpage>D261</lpage><pub-id pub-id-type=\"pmid\">14681407</pub-id></citation></ref><ref id=\"B48\"><citation citation-type=\"other\"><person-group person-group-type=\"author\"><name><surname>Yanai</surname><given-names>I</given-names></name><name><surname>Benjamin</surname><given-names>H</given-names></name><name><surname>Shmoish</surname><given-names>M</given-names></name><name><surname>Chalifa-Caspi</surname><given-names>V</given-names></name><name><surname>Shklar</surname><given-names>M</given-names></name><name><surname>Ophir</surname><given-names>R</given-names></name><name><surname>Bar-Even</surname><given-names>A</given-names></name><name><surname>Horn-Saban</surname><given-names>S</given-names></name><name><surname>Safran</surname><given-names>M</given-names></name><name><surname>Domany</surname><given-names>E</given-names></name><etal></etal></person-group><article-title>Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification.</article-title><source>Bioinformatics</source><year>2004</year></citation></ref><ref id=\"B49\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lockhart</surname><given-names>DJ</given-names></name><name><surname>Dong</surname><given-names>H</given-names></name><name><surname>Byrne</surname><given-names>MC</given-names></name><name><surname>Folletti</surname><given-names>MT</given-names></name><name><surname>Gallo</surname><given-names>MV</given-names></name><name><surname>Chee</surname><given-names>MS</given-names></name><name><surname>Mittmann</surname><given-names>M</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Kobayashi</surname><given-names>M</given-names></name><name><surname>Horton</surname><given-names>H</given-names></name><name><surname>Brown</surname><given-names>EL</given-names></name></person-group><article-title>Expression monitoring by hybridization to high-density oligonucleotide arrays.</article-title><source>Nat Biotechnol</source><year>1996</year><volume>14</volume><fpage>1675</fpage><lpage>1680</lpage><pub-id pub-id-type=\"pmid\">9634850</pub-id><pub-id pub-id-type=\"doi\">10.1038/nbt1296-1675</pub-id></citation></ref><ref id=\"B50\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bolstad</surname><given-names>BM</given-names></name><name><surname>Irizarry</surname><given-names>RA</given-names></name><name><surname>Astrand</surname><given-names>M</given-names></name><name><surname>Speed</surname><given-names>TP</given-names></name></person-group><article-title>A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.</article-title><source>Bioinformatics</source><year>2003</year><volume>19</volume><fpage>185</fpage><lpage>193</lpage><pub-id pub-id-type=\"pmid\">12538238</pub-id><pub-id pub-id-type=\"doi\">10.1093/bioinformatics/19.2.185</pub-id></citation></ref><ref id=\"B51\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Boheler</surname><given-names>KR</given-names></name><name><surname>Stern</surname><given-names>MD</given-names></name></person-group><article-title>The new role of SAGE in gene discovery.</article-title><source>Trends Biotechnol</source><year>2003</year><volume>21</volume><fpage>55</fpage><lpage>57</lpage><pub-id pub-id-type=\"pmid\">12573851</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0167-7799(02)00031-8</pub-id></citation></ref><ref id=\"B52\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zavolan</surname><given-names>M</given-names></name><name><surname>van Nimwegen</surname><given-names>E</given-names></name><name><surname>Gaasterland</surname><given-names>T</given-names></name></person-group><article-title>Splice variation in mouse full-length cDNAs identified by mapping to the mouse genome.</article-title><source>Genome Res</source><year>2002</year><volume>12</volume><fpage>1377</fpage><lpage>1385</lpage><pub-id pub-id-type=\"pmid\">12213775</pub-id><pub-id pub-id-type=\"doi\">10.1101/gr.191702</pub-id></citation></ref><ref id=\"B53\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zavolan</surname><given-names>M</given-names></name><name><surname>Kondo</surname><given-names>S</given-names></name><name><surname>Schonbach</surname><given-names>C</given-names></name><name><surname>Adachi</surname><given-names>J</given-names></name><name><surname>Hume</surname><given-names>DA</given-names></name><name><surname>Hayashizaki</surname><given-names>Y</given-names></name><name><surname>Gaasterland</surname><given-names>T</given-names></name></person-group><article-title>Impact of alternative initiation, splicing, and termination on the diversity of the mRNA transcripts encoded by the mouse transcriptome.</article-title><source>Genome Res</source><year>2003</year><volume>13</volume><fpage>1290</fpage><lpage>1300</lpage><pub-id pub-id-type=\"pmid\">12819126</pub-id><pub-id pub-id-type=\"doi\">10.1101/gr.1017303</pub-id></citation></ref><ref id=\"B54\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marino-Ramirez</surname><given-names>L</given-names></name><name><surname>Spouge</surname><given-names>JL</given-names></name><name><surname>Kanga</surname><given-names>GC</given-names></name><name><surname>Landsman</surname><given-names>D</given-names></name></person-group><article-title>Statistical analysis of over-represented words in human promoter sequences.</article-title><source>Nucleic Acids Res</source><year>2004</year><volume>32</volume><fpage>949</fpage><lpage>958</lpage><pub-id pub-id-type=\"pmid\">14963262</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkh246</pub-id></citation></ref><ref id=\"B55\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sherlock</surname><given-names>G</given-names></name></person-group><article-title>Analysis of large-scale gene expression data.</article-title><source>Brief Bioinform</source><year>2001</year><volume>2</volume><fpage>350</fpage><lpage>362</lpage><pub-id pub-id-type=\"pmid\">11808747</pub-id></citation></ref><ref id=\"B56\"><citation citation-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Felsenstein</surname><given-names>J</given-names></name></person-group><article-title>PHYLIP.</article-title><source>Seattle</source><year>1993</year><edition>5</edition></citation></ref><ref id=\"B57\"><citation citation-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Ash</surname><given-names>RB</given-names></name></person-group><source>Information Theory</source><year>1965</year><publisher-name>Dover Publication, Inc., New York</publisher-name></citation></ref><ref id=\"B58\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rice</surname><given-names>P</given-names></name><name><surname>Longden</surname><given-names>I</given-names></name><name><surname>Bleasby</surname><given-names>A</given-names></name></person-group><article-title>EMBOSS: the European Molecular Biology Open Software Suite.</article-title><source>Trends Genet</source><year>2000</year><volume>16</volume><fpage>276</fpage><lpage>277</lpage><pub-id pub-id-type=\"pmid\">10827456</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0168-9525(00)02024-2</pub-id></citation></ref><ref id=\"B59\"><citation citation-type=\"other\"><article-title>Database of Transcriptional Start Sites</article-title><ext-link ext-link-type=\"uri\" xlink:href=\"http://dbtss.hgc.jp/index.html\"/></citation></ref><ref id=\"B60\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Karolchik</surname><given-names>D</given-names></name><name><surname>Hinrichs</surname><given-names>AS</given-names></name><name><surname>Furey</surname><given-names>TS</given-names></name><name><surname>Roskin</surname><given-names>KM</given-names></name><name><surname>Sugnet</surname><given-names>CW</given-names></name><name><surname>Haussler</surname><given-names>D</given-names></name><name><surname>Kent</surname><given-names>WJ</given-names></name></person-group><article-title>The UCSC Table Browser data retrieval tool.</article-title><source>Nucleic Acids Res</source><year>2004</year><volume>32</volume><issue>Database</issue><fpage>D493</fpage><lpage>D496</lpage><pub-id pub-id-type=\"pmid\">14681465</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkh103</pub-id></citation></ref><ref id=\"B61\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kent</surname><given-names>WJ</given-names></name><name><surname>Sugnet</surname><given-names>CW</given-names></name><name><surname>Furey</surname><given-names>TS</given-names></name><name><surname>Roskin</surname><given-names>KM</given-names></name><name><surname>Pringle</surname><given-names>TH</given-names></name><name><surname>Zahler</surname><given-names>AM</given-names></name><name><surname>Haussler</surname><given-names>D</given-names></name></person-group><article-title>The human genome browser at UCSC.</article-title><source>Genome Res</source><year>2002</year><volume>12</volume><fpage>996</fpage><lpage>1006</lpage><pub-id pub-id-type=\"pmid\">12045153</pub-id><comment>10.1101/gr.229102. Article published online before print in May 2002</comment></citation></ref><ref id=\"B62\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Birney</surname><given-names>E</given-names></name><name><surname>Andrews</surname><given-names>D</given-names></name><name><surname>Bevan</surname><given-names>P</given-names></name><name><surname>Caccamo</surname><given-names>M</given-names></name><name><surname>Cameron</surname><given-names>G</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Clarke</surname><given-names>L</given-names></name><name><surname>Coates</surname><given-names>G</given-names></name><name><surname>Cox</surname><given-names>T</given-names></name><name><surname>Cuff</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Ensembl 2004.</article-title><source>Nucleic Acids Res</source><year>2004</year><volume>32</volume><issue>Database</issue><fpage>D468</fpage><lpage>D470</lpage><pub-id pub-id-type=\"pmid\">14681459</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkh038</pub-id></citation></ref><ref id=\"B63\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Perier</surname><given-names>RC</given-names></name><name><surname>Praz</surname><given-names>V</given-names></name><name><surname>Junier</surname><given-names>T</given-names></name><name><surname>Bonnard</surname><given-names>C</given-names></name><name><surname>Bucher</surname><given-names>P</given-names></name></person-group><article-title>The eukaryotic promoter database (EPD).</article-title><source>Nucleic Acids Res</source><year>2000</year><volume>28</volume><fpage>302</fpage><lpage>303</lpage><pub-id pub-id-type=\"pmid\">10592254</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/28.1.302</pub-id></citation></ref><ref id=\"B64\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Subramaniam</surname><given-names>S</given-names></name></person-group><article-title>The Biology Workbench - a seamless database and analysis environment for the biologist.</article-title><source>Proteins</source><year>1998</year><volume>32</volume><fpage>1</fpage><lpage>2</lpage><pub-id pub-id-type=\"pmid\">9672036</pub-id><pub-id pub-id-type=\"doi\">10.1002/(SICI)1097-0134(19980701)32:1&#x0003c;1::AID-PROT1&#x0003e;3.0.CO;2-Q</pub-id></citation></ref><ref id=\"B65\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Matys</surname><given-names>V</given-names></name><name><surname>Fricke</surname><given-names>E</given-names></name><name><surname>Geffers</surname><given-names>R</given-names></name><name><surname>Gossling</surname><given-names>E</given-names></name><name><surname>Haubrock</surname><given-names>M</given-names></name><name><surname>Hehl</surname><given-names>R</given-names></name><name><surname>Hornischer</surname><given-names>K</given-names></name><name><surname>Karas</surname><given-names>D</given-names></name><name><surname>Kel</surname><given-names>AE</given-names></name><name><surname>Kel-Margoulis</surname><given-names>OV</given-names></name><etal></etal></person-group><article-title>TRANSFAC: transcriptional regulation, from patterns to profiles.</article-title><source>Nucleic Acids Res</source><year>2003</year><volume>31</volume><fpage>374</fpage><lpage>378</lpage><pub-id pub-id-type=\"pmid\">12520026</pub-id><pub-id pub-id-type=\"doi\">10.1093/nar/gkg108</pub-id></citation></ref><ref id=\"B66\"><citation citation-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>Z</given-names></name><name><surname>Irizarry</surname><given-names>R</given-names></name></person-group><source>gcrma.</source><year>2004</year><edition>1.1.3</edition><ext-link ext-link-type=\"uri\" xlink:href=\"http://www.bioconductor.org/\"/></citation></ref><ref id=\"B67\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gentleman</surname><given-names>RC</given-names></name><name><surname>Carey</surname><given-names>VJ</given-names></name><name><surname>Bates</surname><given-names>DM</given-names></name><name><surname>Bolstad</surname><given-names>B</given-names></name><name><surname>Dettling</surname><given-names>M</given-names></name><name><surname>Dudoit</surname><given-names>S</given-names></name><name><surname>Ellis</surname><given-names>B</given-names></name><name><surname>Gautier</surname><given-names>L</given-names></name><name><surname>Ge</surname><given-names>Y</given-names></name><name><surname>Gentry</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Bioconductor: open software development for computational biology and bioinformatics.</article-title><source>Genome Biol</source><year>2004</year><volume>5</volume><fpage>R80</fpage><pub-id pub-id-type=\"pmid\">15461798</pub-id><pub-id pub-id-type=\"doi\">10.1186/gb-2004-5-10-r80</pub-id></citation></ref><ref id=\"B68\"><citation citation-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Team</surname><given-names>RDC</given-names></name></person-group><source>R: A language and environment for statistical computing.</source><year>2004</year><edition>2.0.1</edition><publisher-name>Vienna, Austria: R Foundation for Statistical Computing</publisher-name></citation></ref><ref id=\"B69\"><citation citation-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Schneider</surname><given-names>TD</given-names></name><name><surname>Stephens</surname><given-names>RM</given-names></name></person-group><article-title>Sequence logos: a new way to display consensus sequences.</article-title><source>Nucleic Acids Res</source><year>1990</year><volume>18</volume><fpage>6097</fpage><lpage>6100</lpage><pub-id pub-id-type=\"pmid\">2172928</pub-id></citation></ref></ref-list><sec sec-type=\"display-objects\"><title>Figures and Tables</title><fig position=\"float\" id=\"F1\"><label>Figure 1</label><caption><p>Examples of GNF-GEA expression patterns for mouse genes at selected <italic>H</italic><sub><italic>g </italic></sub>and <italic>Q</italic>. Liver, indicated in red, is the tissue of interest for <italic>Q </italic>values. <bold>(a) </bold>Serum albumin (94777_at <italic>Alb1</italic>) shows very specific liver expression: <italic>H </italic>= 1.3 bits and <italic>Q</italic><sub>liver </sub>= 2.1 bits. <bold>(b) </bold>For liver-specific bHLH-Zip transcription factor (99452_at <italic>Lisch7</italic>), liver is a strong but not dominant part of the expression pattern: <italic>H </italic>= 3.7 bits and <italic>Q</italic><sub>liver </sub>= 6.8 bits. <bold>(c) </bold>For chloride channel 7 (104391_s_at <italic>Clcn7</italic>) there is near uniform expression: <italic>H </italic>= 4.3 bits and <italic>Q</italic><sub>liver </sub>= 10.2 bits. <bold>(d) </bold>Gelsolin (93750_at <italic>Gsn</italic>) is an otherwise widely expressed gene but is expressed at a very low level in the liver: <italic>H </italic>= 4.4 bits and <italic>Q</italic><sub>liver </sub>= 15.1 bits.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-1\"/></fig><fig position=\"float\" id=\"F2\"><label>Figure 2</label><caption><p>Distributions of <italic>H </italic>and <italic>Q </italic>for different data sources and tissues. <bold>(a) </bold>Distribution of <italic>H </italic>as estimated from GNF-GEA (red line) and DoTS (blue line). The DoTS curve was generated from genes with at least six ESTs. <bold>(b) </bold>Correlation of <italic>H </italic>estimates from GNF-GEA and DoTS. Genes with at least 30 ESTs are shown in red; those with more than 100 ESTs in blue. <bold>(c) </bold>Cumulative distribution of <italic>Q </italic>values for selected mouse tissues and the average for all 39 tissues. Mammary gland, liver, muscle and the amygdala have decreasing numbers of highly tissue-specific genes. Liver has a very large number of relatively specific genes. All distributions peak at 2 log<sub>2</sub>(39) = 10.6 bits and have a tail at high <italic>Q </italic>(not shown) that corresponds to genes that are ubiquitously expressed except in the tissue of interest.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-2\"/></fig><fig position=\"float\" id=\"F3\"><label>Figure 3</label><caption><p>Consensus tissue tree of tissues from human and mouse data. Trees are the consensus of trees created from 5,000 random samples of sets of 1,000 genes from <bold>(a) </bold>3,768 (human) or <bold>(b) </bold>1,786 (mouse) genes with <italic>Q</italic><sub><italic>g</italic>|<italic>t </italic></sub>&#x02264; 7 bits in at least one tissue. The length of the line leading into a node indicates how many trees did not include the set of tissues to the right of the node. The shortest lines correspond to unanimous subgroups. We have highlighted all maximal subgroups that occurred in at least half of the sampled trees. The nervous system is indicated in red, immune system in blue, reproductive tissue in yellow, digestive organs in purple and magenta, muscle tissue in cyan, and glandular tissue in brown. All maximal subgroups that occurred in at least half of the sampled trees. The tissues not included in a highlighted subgroup typically have statistically significant overlap with many of the highlighted tissues as estimated using the hypergeometric distribution.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-3\"/></fig><fig position=\"float\" id=\"F4\"><label>Figure 4</label><caption><p>The fraction of start CpG islands in genes ranked by entropy <italic>H</italic><sub><italic>g </italic></sub>increases with entropy. Each point represents the fraction of genes in consecutive groups of 100 genes ranked by entropy <italic>H</italic><sub><italic>g </italic></sub>computed from GNF-GEA data. Genes in this set are expressed above 200 AU in at least one tissue. The human dataset (diamonds) has 26 tissues (maximum <italic>H </italic>= 4.7 bits), the mouse dataset (squares) has 42 tissues (maximum <italic>H </italic>= 5.3 bits).</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-4\"/></fig><fig position=\"float\" id=\"F5\"><label>Figure 5</label><caption><p>Base-composition profiles for ubiquitous and tissue-specific genes with and without start CpG islands. Data is for human genes; similar patterns were observed in mouse. <bold>(a) </bold>Ubiquitous genes with a CpG island; <bold>(b) </bold>tissue-specific genes with a CpG island; <bold>(c) </bold>ubiquitous genes with no CpG island; and <bold>(d) </bold>tissue-specific genes with no CpG island. Note differences in upstream C+G content, peak sizes at TATA box (-35 bp) and initiator positions, and downstream C versus G differences.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-5\"/></fig><fig position=\"float\" id=\"F6\"><label>Figure 6</label><caption><p>YY1 motifs are found downstream of the transcription start site, depending on their orientation. <bold>(a) </bold>The top image shows a logo [69] representation of the YY1 motif in the (+10, +20) region of human CGI+ promoters identified using AlignACE. It is based on 102 sequences. The other two logos are for weight matrices contained in TRANSFAC v7.3 that represent activating and repressing YY1 binding sites. <bold>(b) </bold>Plot of the positional distribution of predicted YY1 sites and the fraction of genes with a predicted YY1 sites in the (+1, +60) region. YY1 sites were predicted using a weight matrix generated using AlignACE. YY1 sites are more than almost three times (<italic>P </italic>&#x02264; 2 &#x000d7; 10<sup>-7</sup>) as common in genes with nonspecific CGI+ genes (11%; <italic>N </italic>= 2,072) than in CGI- genes (4%; <italic>N </italic>= 607) and occur at more than 10 times the expected rate. Similar trends are observed in genes with 3 &#x02264; <italic>H </italic>&#x02264; 4 though with lower absolute and relative rates. The difference between CGI+ and CGI- genes is not statistically significant for genes in the 3 &#x02264; <italic>H </italic>&#x02264; 4 bin. Essentially no YY1 sites where observed in specific genes with <italic>H </italic>&#x02264; 3 bits whether or not they had a CpG island.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-6\"/></fig><fig position=\"float\" id=\"F7\"><label>Figure 7</label><caption><p>The distribution of TATA box and initiator element (Inr) in pancreas-specific genes. One hundred and sixty pancreas genes were divided into bins according to their <italic>Q</italic>-value. Genes that have a TATA box, an initiator with the motif YYANWYY, both, or none of these two motifs, are shown. <bold>(a) </bold>Absolute numbers of genes with core promoter motifs. Red bars, TATA only; blue bars, TATA and Inr; green bars, Inr only; purple bars, none. The <italic>p</italic>-values for pairwise comparison of distributions (TATA/total) are given below the graph. <italic>P</italic>-values were calculated for the sum of genes with TATA box (with and without initiator). <bold>(b) </bold>Results from (a) plotted as fractions of genes with each motif status within a bin. <bold>(c) </bold>Number of TATA boxes found in orthologous human and mouse gene pairs. Statistical significance of differences between <italic>Q </italic>bins are indicated.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-7\"/></fig><fig position=\"float\" id=\"F8\"><label>Figure 8</label><caption><p>The cumulative distribution of promoter classes as a function of entropy is similar in human and mouse. The cumulative fractions of genes with all possible combinations of CGI and TATA box features for <bold>(a) </bold>human and <bold>(b) </bold>mouse as a function of entropy <italic>H</italic><sub><italic>g </italic></sub>as computed from GNF-GEA data is shown. For example, in human about 50% of the genes with <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 2.5 have a CGI-/TATA+ promoter. The gray bars indicate the entropy range that is not significantly different from uniform ubiquitous expression. Curves are compiled from genes that express above 200 AU in at least one tissue. As expected, CGI+/TATA- genes are most common in less specific genes and CGI-/TATA+ genes are most common in tissue-specific genes. CGI-/TATA- genes are very common and are found nearly uniformly at every level of specificity. Furthermore, CGI+/TATA- and CGI-/TATA+ genes are both common in mid-specificity (3 &#x02264; <italic>H</italic><sub><italic>g </italic></sub>&#x02264; 4) genes showing that specificity is not determined by these features alone. The trends in human and mouse data are nearly identical despite the lower rate of CpG islands in mouse. The large variations in the graph at low entropy are due to the noise inherent in the small number of genes in this range.</p></caption><graphic xlink:href=\"gb-2005-6-4-r33-8\"/></fig><table-wrap position=\"float\" id=\"T1\"><label>Table 1</label><caption><p>The top five most tissue-specific genes for representative tissues</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td align=\"left\">Tissue</td><td align=\"left\">Probe set ID</td><td align=\"left\"><italic>H</italic></td><td align=\"left\"><italic>Q</italic></td><td align=\"left\">RefSeq</td><td align=\"left\">Description</td></tr></thead><tbody><tr><td align=\"left\">Amygdala</td><td align=\"left\">96055_at</td><td align=\"left\">3.2</td><td align=\"left\">5.8</td><td align=\"left\">NM_031161</td><td align=\"left\">Cholecystokinin</td></tr><tr><td></td><td align=\"left\">93178_at</td><td align=\"left\">2.7</td><td align=\"left\">5.8</td><td align=\"left\">NM_019867</td><td align=\"left\">Neuronal guanine nucleotide exchange factor</td></tr><tr><td></td><td align=\"left\">93273_at</td><td align=\"left\">3.7</td><td align=\"left\">5.8</td><td align=\"left\">NM_009221</td><td align=\"left\">Synuclein, alpha</td></tr><tr><td></td><td align=\"left\">92943_at</td><td align=\"left\">3.5</td><td align=\"left\">6.0</td><td align=\"left\">NM_008165</td><td align=\"left\">Glutamate receptor, ionotropic, AMPA1 (alpha 1)</td></tr><tr><td></td><td align=\"left\">95436_at</td><td align=\"left\">3.3</td><td align=\"left\">6.1</td><td align=\"left\">NM_009215</td><td align=\"left\">Somatostatin</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">Lymph node</td><td align=\"left\">98406_at</td><td align=\"left\">2.7</td><td align=\"left\">4.0</td><td align=\"left\">NM_013653</td><td align=\"left\">Chemokine (C-C motif) ligand 5</td></tr><tr><td></td><td align=\"left\">98063_at</td><td align=\"left\">1.6</td><td align=\"left\">4.1</td><td align=\"left\">-</td><td align=\"left\">Glycosylation dependent cell adhesion molecule 1</td></tr><tr><td></td><td align=\"left\">99446_at</td><td align=\"left\">2.5</td><td align=\"left\">4.1</td><td align=\"left\">NM_007641</td><td align=\"left\">Membrane-spanning 4-domains, subfamily A, member 1</td></tr><tr><td></td><td align=\"left\">92741_g_at</td><td align=\"left\">3.3</td><td align=\"left\">4.5</td><td align=\"left\">-</td><td align=\"left\">Immunoglobulin heavy chain 4 (serum IgG1)</td></tr><tr><td></td><td align=\"left\">102940_at</td><td align=\"left\">2.8</td><td align=\"left\">4.6</td><td align=\"left\">NM_008518</td><td align=\"left\">Lymphotoxin B</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">Liver</td><td align=\"left\">94777_at</td><td align=\"left\">1.3</td><td align=\"left\">2.1</td><td align=\"left\">-</td><td align=\"left\">Albumin 1</td></tr><tr><td></td><td align=\"left\">101287_s_at</td><td align=\"left\">1.6</td><td align=\"left\">2.2</td><td align=\"left\">NM_010005</td><td align=\"left\">Cytochrome P450, 2d10</td></tr><tr><td></td><td align=\"left\">99269_g_at</td><td align=\"left\">1.5</td><td align=\"left\">2.2</td><td align=\"left\">NM_019911</td><td align=\"left\">Tryptophan 2,3-dioxygenase</td></tr><tr><td></td><td align=\"left\">100329_at</td><td align=\"left\">1.4</td><td align=\"left\">2.3</td><td align=\"left\">NM_009246</td><td align=\"left\">Serine protease inhibitor 1-4</td></tr><tr><td></td><td align=\"left\">94318_at</td><td align=\"left\">1.6</td><td align=\"left\">2.3</td><td align=\"left\">NM_013475</td><td align=\"left\">Apolipoprotein H</td></tr></tbody></table><table-wrap-foot><p>Genes must express at 200 AU in one or more tissues. A full list of all genes is available in the <xref ref-type=\"supplementary-material\" rid=\"S1\">Additional data files 1</xref> and <xref ref-type=\"supplementary-material\" rid=\"S2\">2</xref>.</p></table-wrap-foot></table-wrap><table-wrap position=\"float\" id=\"T2\"><label>Table 2</label><caption><p>The list of tissues used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td align=\"left\">GNF+GEA tissues</td><td align=\"left\">Comparison to EST</td><td align=\"left\">Hierarchical clustering</td></tr></thead><tbody><tr><td align=\"left\">DRG</td><td align=\"left\">PNS</td><td align=\"left\">Nervous system</td></tr><tr><td align=\"left\">Trigeminal</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Hippocampus</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Amygdala</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Frontal_cortex</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Cortex</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Striatum</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Olfactory_bulb</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Hypothalamus</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Spinal_cord_lower</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Spinal_cord_upper</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Cerebellum</td><td align=\"left\">CNS</td><td></td></tr><tr><td align=\"left\">Eye</td><td align=\"left\">Eye</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Spleen</td><td align=\"left\">Spleen</td><td align=\"left\">Immune System + trachea</td></tr><tr><td align=\"left\">Lymph_node</td><td align=\"left\">Lymph_node</td><td></td></tr><tr><td align=\"left\">Trachea</td><td align=\"left\">Trachea</td><td></td></tr><tr><td align=\"left\">Thymus</td><td align=\"left\">Thymus</td><td></td></tr><tr><td align=\"left\">Bone_marrow</td><td align=\"left\">Bone</td><td></td></tr><tr><td align=\"left\">Bone</td><td align=\"left\">Bone</td><td></td></tr><tr><td align=\"left\">Lung</td><td align=\"left\">Lung</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Uterus</td><td align=\"left\">Uterus</td><td align=\"left\">Reproductive organs</td></tr><tr><td align=\"left\">Umbilical cord</td><td align=\"left\">Umbilical_cord</td><td></td></tr><tr><td align=\"left\">Placenta</td><td align=\"left\">Plancenta</td><td></td></tr><tr><td align=\"left\">Ovary</td><td align=\"left\">Ovary</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Epidermis, snout_epidermis</td><td align=\"left\">Epidermis</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Heart</td><td align=\"left\">Heart</td><td align=\"left\">Muscle</td></tr><tr><td align=\"left\">Skeletal_muscle</td><td align=\"left\">Skeletal_muscle</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Adipose_tissue, brown_fat</td><td align=\"left\">Fat</td><td></td></tr><tr><td align=\"left\">Adrenal_gland</td><td align=\"left\">Adrenal_gland</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Stomach</td><td align=\"left\">Stomach</td><td align=\"left\">Digestive tract</td></tr><tr><td align=\"left\">Bladder</td><td align=\"left\">Bladder</td><td></td></tr><tr><td align=\"left\">Small_intestine</td><td align=\"left\">Small_intestine</td><td></td></tr><tr><td align=\"left\">Large_intestine</td><td align=\"left\">Large_intestine</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Gall bladder</td><td align=\"left\">Gall_bladder</td><td align=\"left\">Gall bladder, liver, and kidney</td></tr><tr><td align=\"left\">Liver</td><td align=\"left\">Liver</td><td></td></tr><tr><td align=\"left\">Kidney</td><td align=\"left\">Kidney</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td align=\"left\">Salivary_gland</td><td align=\"left\">Salivary_gland</td><td></td></tr><tr><td align=\"left\">Thyroid</td><td align=\"left\">Thyroid</td><td></td></tr><tr><td align=\"left\">Mammary_gland</td><td align=\"left\">Mammary_gland</td><td></td></tr><tr><td align=\"left\">Prostate</td><td align=\"left\">Prostate</td><td></td></tr><tr><td align=\"left\">Testis</td><td align=\"left\">Testis</td><td></td></tr><tr><td align=\"left\">Tongue</td><td align=\"left\">Tongue</td><td></td></tr><tr><td align=\"left\">Digits</td><td align=\"left\">Digits</td><td></td></tr></tbody></table><table-wrap-foot><p>The list of tissues available in the mouse GNF+GEA survey, groupings of tissues used to compare microarray and EST-based entropy estimates, and tissue groups discovered by clustering tissues on the basis of genes expressed in common.</p></table-wrap-foot></table-wrap><table-wrap position=\"float\" id=\"T3\"><label>Table 3</label><caption><p>The top five most group-specific mouse genes for selected tissue groups</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td align=\"left\">Tissue cluster</td><td align=\"center\">Probe Set ID</td><td align=\"center\"><italic>H</italic></td><td align=\"center\"><italic>Q</italic></td><td align=\"left\">RefSeq</td><td align=\"left\">Description</td></tr></thead><tbody><tr><td align=\"left\">Nervous system</td><td align=\"center\">100047_at</td><td align=\"center\">3.3</td><td align=\"center\">3.4</td><td align=\"left\">NM_011428</td><td align=\"left\">Synaptosomal-associated protein, 25 kDa</td></tr><tr><td></td><td align=\"center\">103030_at</td><td align=\"center\">3.5</td><td align=\"center\">3.6</td><td></td><td align=\"left\">Dynamin</td></tr><tr><td></td><td align=\"center\">97983_s_at</td><td align=\"center\">3.7</td><td align=\"center\">3.8</td><td align=\"left\">NM_009295</td><td align=\"left\">Syntaxin binding protein 1</td></tr><tr><td></td><td align=\"center\">98339_at</td><td align=\"center\">3.7</td><td align=\"center\">3.8</td><td align=\"left\">NM_018804</td><td align=\"left\">Synaptotagmin 11</td></tr><tr><td></td><td align=\"center\">94545_at</td><td align=\"center\">3.7</td><td align=\"center\">3.8</td><td align=\"left\">NM_153457</td><td align=\"left\">Reticulon 1</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">Immune system</td><td align=\"center\">96648_at</td><td align=\"center\">2.807</td><td align=\"center\">2.882</td><td align=\"left\">NM_009898</td><td align=\"left\">Coronin, actin binding protein 1a</td></tr><tr><td></td><td align=\"center\">93584_at</td><td align=\"center\">3.373</td><td align=\"center\">3.622</td><td></td><td align=\"left\">Immunoglobulin heavy chain 6 (heavy chain of IgM)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td align=\"center\">101048_at</td><td align=\"center\">3.541</td><td align=\"center\">3.876</td><td align=\"left\">NM_011210</td><td align=\"left\">Protein tyrosine phosphatase, receptor type, C</td></tr><tr><td></td><td align=\"center\">94278_at</td><td align=\"center\">3.495</td><td align=\"center\">3.923</td><td align=\"left\">NM_008879</td><td align=\"left\">Lymphocyte cytosolic protein 1</td></tr><tr><td></td><td align=\"center\">100156_at</td><td align=\"center\">3.609</td><td align=\"center\">4.039</td><td align=\"left\">NM_008566</td><td align=\"left\">Mini chromosome maintenance deficient 5</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">Liver and gall bladder</td><td align=\"center\">94777_at</td><td align=\"center\">1.280</td><td align=\"center\">1.326</td><td></td><td align=\"left\">Albumin 1</td></tr><tr><td></td><td align=\"center\">100329_at</td><td align=\"center\">1.394</td><td align=\"center\">1.464</td><td align=\"left\">NM_009246</td><td align=\"left\">Serine protease inhibitor 1-4</td></tr><tr><td></td><td align=\"center\">99269_g_at</td><td align=\"center\">1.471</td><td align=\"center\">1.561</td><td align=\"left\">NM_019911</td><td align=\"left\">Tryptophan 2,3-dioxygenase</td></tr><tr><td></td><td align=\"center\">99862_at</td><td align=\"center\">1.503</td><td align=\"center\">1.595</td><td align=\"left\">NM_013465</td><td align=\"left\">Alpha-2-HS-glycoprotein</td></tr><tr><td></td><td align=\"center\">96846_at</td><td align=\"center\">1.515</td><td align=\"center\">1.607</td><td align=\"left\">NM_080844</td><td align=\"left\">Serine (or cysteine) proteinase inhibitor, clade C (antithrombin), member 1</td></tr></tbody></table><table-wrap-foot><p>The tissue groups were identified in a consensus clustering of tissues based on common tissue-specific genes. The <italic>Q </italic>value is for the gene and tissue group. To ensure uniform expression across the tissue group, genes were required to have an entropy on the tissue group that was 90% of the maximum possible for the group.</p></table-wrap-foot></table-wrap><table-wrap position=\"float\" id=\"T4\"><label>Table 4</label><caption><p>CpG islands are correlated with embryonic expression even for tissue-specific genes</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td align=\"left\">Gene type</td><td align=\"center\">CpG island state</td><td align=\"center\">Total genes considered</td><td align=\"center\">Expressed genes</td><td align=\"center\">Fraction</td><td align=\"center\">Fraction ratio</td></tr></thead><tbody><tr><td align=\"left\">Embryo</td><td align=\"center\">CGI+</td><td align=\"center\">933</td><td align=\"center\">365</td><td align=\"center\">39%</td><td align=\"center\">2.8</td></tr><tr><td></td><td align=\"center\">CGI-</td><td align=\"center\">1007</td><td align=\"center\">139</td><td align=\"center\">14%</td><td></td></tr><tr><td colspan=\"6\">&#x000a0;</td></tr><tr><td align=\"left\">Adult-specific</td><td align=\"center\">CGI+</td><td align=\"center\">29</td><td align=\"center\">8</td><td align=\"center\">29%</td><td align=\"center\">4</td></tr><tr><td></td><td align=\"center\">CGI-</td><td align=\"center\">180</td><td align=\"center\">12</td><td align=\"center\">7%</td><td></td></tr></tbody></table><table-wrap-foot><p>We determined the fraction of genes with (39%) and without (14%) start CpG islands that are expressed in the early embryo. A gene is 2.8 (= 0.39/0.14) times more likely to be expressed in the early embryo if it has a start CpG island. If we then consider genes that go on to be specific in the adult, we find the ratio of CGI+/CGI- genes is now 4 = 0.28/0.07. The differences in rates between CpG island status within each stage are significant (<italic>P </italic>&#x0003c; 0.0005; binomial). Of the between-stage comparisons, only the CGI- adult-specific/embryo change is significant (<italic>P </italic>= 0.0009; hypergeometric).</p></table-wrap-foot></table-wrap><table-wrap position=\"float\" id=\"T5\"><label>Table 5</label><caption><p>The most significant indicators of the degree of tissue specificity: start CpG island, TATA box, and YY1 site</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td align=\"left\" colspan=\"3\">Features</td><td align=\"left\">Total fraction</td><td align=\"left\"><italic>H </italic>0-3</td><td align=\"left\"><italic>H </italic>3-4</td><td align=\"left\"><italic>H </italic>4-5</td></tr><tr><td colspan=\"3\"><hr></hr></td><td></td><td colspan=\"3\"><hr></hr></td></tr><tr><td align=\"left\">CGI</td><td align=\"left\">TATA</td><td align=\"left\">YY1</td><td></td><td align=\"left\">Most specific</td><td align=\"left\">Semi-specific</td><td align=\"left\">Least specific</td></tr></thead><tbody><tr><td></td><td></td><td></td><td align=\"left\"><bold>3,552</bold></td><td align=\"left\"><bold>271</bold></td><td align=\"left\"><bold>602</bold></td><td align=\"left\"><bold>2679</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">1.00</td><td align=\"left\">0.08</td><td align=\"left\">0.17</td><td align=\"left\">0.75</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">CGI+</td><td></td><td></td><td align=\"left\"><bold>2,434</bold></td><td align=\"left\"><bold>56</bold></td><td align=\"left\"><bold>306</bold></td><td align=\"left\"><bold>2072</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.69</td><td align=\"left\">0.02</td><td align=\"left\">0.13</td><td align=\"left\">0.85</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.30</td><td align=\"left\">0.74</td><td align=\"left\">1.13</td></tr><tr><td align=\"left\">CGI-</td><td></td><td></td><td align=\"left\"><bold>1,118</bold></td><td align=\"left\"><bold>215</bold></td><td align=\"left\"><bold>296</bold></td><td align=\"left\"><bold>607</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.31</td><td align=\"left\">0.19</td><td align=\"left\">0.26</td><td align=\"left\">0.54</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">2.52</td><td align=\"left\">1.56</td><td align=\"left\">0.72</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td align=\"left\">TATA+</td><td></td><td align=\"left\"><bold>604</bold></td><td align=\"left\"><bold>136</bold></td><td align=\"left\"><bold>175</bold></td><td align=\"left\"><bold>293</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.17</td><td align=\"left\">0.23</td><td align=\"left\">0.29</td><td align=\"left\">0.49</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">2.95</td><td align=\"left\">1.71</td><td align=\"left\">0.64</td></tr><tr><td></td><td align=\"left\">TATA-</td><td></td><td align=\"left\"><bold>2,949</bold></td><td align=\"left\"><bold>135</bold></td><td align=\"left\"><bold>427</bold></td><td align=\"left\"><bold>2,387</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.83</td><td align=\"left\">0.05</td><td align=\"left\">0.14</td><td align=\"left\">0.81</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.60</td><td align=\"left\">0.85</td><td align=\"left\">1.07</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">CGI+</td><td align=\"left\">TATA+</td><td></td><td align=\"left\"><bold>284</bold></td><td align=\"left\"><bold>19</bold></td><td align=\"left\"><bold>82</bold></td><td align=\"left\"><bold>183</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.08</td><td align=\"left\">0.07</td><td align=\"left\">0.29</td><td align=\"left\">0.64</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.88</td><td align=\"left\">1.70</td><td align=\"left\">0.85</td></tr><tr><td align=\"left\">CGI-</td><td align=\"left\">TATA+</td><td></td><td align=\"left\"><bold>320</bold></td><td align=\"left\"><bold>117</bold></td><td align=\"left\"><bold>93</bold></td><td align=\"left\"><bold>110</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.09</td><td align=\"left\">0.37</td><td align=\"left\">0.29</td><td align=\"left\">0.34</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">4.79</td><td align=\"left\">1.71</td><td align=\"left\">0.46</td></tr><tr><td align=\"left\">CGI+</td><td align=\"left\">TATA-</td><td></td><td align=\"left\"><bold>2,150</bold></td><td align=\"left\"><bold>37</bold></td><td align=\"left\"><bold>224</bold></td><td align=\"left\"><bold>1,889</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.61</td><td align=\"left\">0.02</td><td align=\"left\">0.10</td><td align=\"left\">0.88</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.23</td><td align=\"left\">0.61</td><td align=\"left\">1.16</td></tr><tr><td align=\"left\">CGI-</td><td align=\"left\">TATA-</td><td></td><td align=\"left\"><bold>798</bold></td><td align=\"left\"><bold>98</bold></td><td align=\"left\"><bold>203</bold></td><td align=\"left\"><bold>497</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.22</td><td align=\"left\">0.12</td><td align=\"left\">0.25</td><td align=\"left\">0.62</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">1.61</td><td align=\"left\">1.50</td><td align=\"left\">0.83</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td align=\"left\">YY1+</td><td align=\"left\"><bold>293</bold></td><td align=\"left\"><bold>1</bold></td><td align=\"left\"><bold>16</bold></td><td align=\"left\"><bold>276</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.08</td><td align=\"left\">0.00</td><td align=\"left\">0.05</td><td align=\"left\">0.94</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.04</td><td align=\"left\">0.32</td><td align=\"left\">1.25</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">CGI+</td><td></td><td align=\"left\">YY1+</td><td align=\"left\"><bold>261</bold></td><td align=\"left\"><bold>1</bold></td><td align=\"left\"><bold>10</bold></td><td align=\"left\"><bold>250</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.07</td><td align=\"left\">0.00</td><td align=\"left\">0.04</td><td align=\"left\">0.96</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.05</td><td align=\"left\">0.23</td><td align=\"left\">1.27</td></tr><tr><td align=\"left\">CGI+</td><td></td><td align=\"left\">YY1-</td><td align=\"left\"><bold>2,173</bold></td><td align=\"left\"><bold>55</bold></td><td align=\"left\"><bold>296</bold></td><td align=\"left\"><bold>1,822</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.61</td><td align=\"left\">0.03</td><td align=\"left\">0.14</td><td align=\"left\">0.84</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.33</td><td align=\"left\">0.80</td><td align=\"left\">1.11</td></tr><tr><td align=\"left\">CGI-</td><td></td><td align=\"left\">YY1-</td><td align=\"left\"><bold>1,086</bold></td><td align=\"left\"><bold>215</bold></td><td align=\"left\"><bold>290</bold></td><td align=\"left\"><bold>581</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.31</td><td align=\"left\">0.20</td><td align=\"left\">0.27</td><td align=\"left\">0.53</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">2.59</td><td align=\"left\">1.58</td><td align=\"left\">0.71</td></tr><tr><td align=\"left\">CGI-</td><td></td><td align=\"left\">YY1+</td><td align=\"left\"><bold>32</bold></td><td align=\"left\"><bold>0</bold></td><td align=\"left\"><bold>6</bold></td><td align=\"left\"><bold>26</bold></td></tr><tr><td></td><td></td><td></td><td align=\"left\">0.01</td><td align=\"left\">0.00</td><td align=\"left\">0.19</td><td align=\"left\">0.81</td></tr><tr><td></td><td></td><td></td><td></td><td align=\"left\">0.00</td><td align=\"left\">1.11</td><td align=\"left\">1.08</td></tr></tbody></table><table-wrap-foot><p>The three columns on the left indicate the combination of features considered; empty cells indicate that the feature is not considered. The 'Total fraction' column indicates the number of promoters with each feature combination (in bold) and the corresponding fraction of all genes considered. The three columns on the right give statistics for matching genes in three bands of tissue specificity. The top two lines give the number and corresponding fraction of all genes considered for each band. For each feature combination, the numbers indicate the number (top, bold), fraction (middle), and enrichment ratio (bottom) of matching genes. The enrichment ratio is the fraction of promoters of genes in the entropy band that contain a feature divided by the band's fraction among all genes considered. For example, specific genes are best recognized by a combination of TATA box (TATA+) and lack of a CpG island (CGI-), which enriches the fraction of such genes from 8% to 37% - a factor of 4.79. Nonspecific genes are most specifically recognized by CpG islands and YY1 sites, which returns a set that is 96% nonspecific genes, but only matches 7%/75% = 10% of the nonspecific genes.</p></table-wrap-foot></table-wrap><table-wrap position=\"float\" id=\"T6\"><label>Table 6</label><caption><p>Over-represented Gene Ontology (GO) terms for cellular component and biological process of genes by promoter class</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td></td><td align=\"left\">Cellular component/biological process</td><td align=\"left\">Human only</td><td align=\"left\">Mouse only</td></tr></thead><tbody><tr><td align=\"left\">CGI-/TATA+</td><td align=\"left\">Extracellular, extracellular space</td><td align=\"left\"><bold>-</bold></td><td align=\"left\">Intermediate filament (cytoskeleton)</td></tr><tr><td></td><td align=\"left\">Response to stimulus</td><td align=\"left\">Cell-cell signaling, organismal physiological process, inflammatory response, innate immune response, response to pest/pathogen/parasite</td><td align=\"left\">-</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">CGI+/TATA-</td><td align=\"left\">Cell, cytoplasm, intracellular, mitochondrion</td><td align=\"left\">Nucleus, ribonucleoprotein complex</td><td align=\"left\"><bold>-</bold></td></tr><tr><td></td><td align=\"left\"><bold>-</bold></td><td align=\"left\">Nucleobase, nucleoside, nucleotide and nucleic acid metabolism, intracellular transport, metabolism, protein transport, intracellular protein transport, RNA processing, RNA metabolism, cell cycle, mitotic cell cycle</td><td align=\"left\"><bold>-</bold></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td align=\"left\">CGI-/TATA-</td><td align=\"left\">(Integral to) (plasma) membrane</td><td align=\"left\">-</td><td align=\"left\">Extracellular, extracellular space</td></tr><tr><td></td><td align=\"left\">Organismal physiological process, defense response, immune response, response to biotic stimulus, response to stimulus, response to external stimulus</td><td align=\"left\">Response to pest/pathogen/parasite, cell communication, response to wounding, cellular defense response, signal transduction</td><td align=\"left\">Complement activation, complement activation (classical pathway), humoral defense mechanism (<italic>sensu </italic>Vertebrata), humoral immune response</td></tr></tbody></table><table-wrap-foot><p>All terms were selected using a <italic>p</italic>-value &#x02264; 0.05 (corrected for multiple testing). Terms common to human and mouse are listed in the second column. The two columns on the right indicate any additional terms found in only one species. The CGI-/TATA+ terms are consistent with a model of strong condition-specific induction, CGI+/TATA- terms are consistent with housekeeping functions. CGI-/TATA- terms indicate support for cell sensing and communication functions. No significant results were found for CGI+/TATA+ genes.</p></table-wrap-foot></table-wrap></sec></back></article>"
     ]
    }
   ],
   "source": [
    "!head {filename}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "run_control": {
     "marked": false
    }
   },
   "outputs": [
    {
     "ename": "XLRDError",
     "evalue": "Unsupported format, or corrupt file: Expected BOF record; found b'<!DOCTYP'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mXLRDError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-40-9cb8d1ebe1ea>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mdf\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread_excel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/pandas/util/_decorators.py\u001b[0m in \u001b[0;36mwrapper\u001b[0;34m(*args, **kwargs)\u001b[0m\n\u001b[1;32m    116\u001b[0m                 \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    117\u001b[0m                     \u001b[0mkwargs\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mnew_arg_name\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnew_arg_value\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 118\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mfunc\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    119\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mwrapper\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    120\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0m_deprecate_kwarg\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/pandas/io/excel.py\u001b[0m in \u001b[0;36mread_excel\u001b[0;34m(io, sheet_name, header, skiprows, skip_footer, index_col, names, usecols, parse_dates, date_parser, na_values, thousands, convert_float, converters, dtype, true_values, false_values, engine, squeeze, **kwds)\u001b[0m\n\u001b[1;32m    228\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    229\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mio\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mExcelFile\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 230\u001b[0;31m         \u001b[0mio\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mExcelFile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mio\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mengine\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mengine\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    231\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    232\u001b[0m     return io._parse_excel(\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/pandas/io/excel.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, io, **kwds)\u001b[0m\n\u001b[1;32m    292\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbook\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mxlrd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mopen_workbook\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfile_contents\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    293\u001b[0m         \u001b[0;32melif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_io\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcompat\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstring_types\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 294\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbook\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mxlrd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mopen_workbook\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_io\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    295\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    296\u001b[0m             raise ValueError('Must explicitly set engine if not passing in'\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/xlrd/__init__.py\u001b[0m in \u001b[0;36mopen_workbook\u001b[0;34m(filename, logfile, verbosity, use_mmap, file_contents, encoding_override, formatting_info, on_demand, ragged_rows)\u001b[0m\n\u001b[1;32m    439\u001b[0m         \u001b[0mformatting_info\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mformatting_info\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    440\u001b[0m         \u001b[0mon_demand\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mon_demand\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 441\u001b[0;31m         \u001b[0mragged_rows\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mragged_rows\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    442\u001b[0m         )\n\u001b[1;32m    443\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mbk\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/xlrd/book.py\u001b[0m in \u001b[0;36mopen_workbook_xls\u001b[0;34m(filename, logfile, verbosity, use_mmap, file_contents, encoding_override, formatting_info, on_demand, ragged_rows)\u001b[0m\n\u001b[1;32m     89\u001b[0m         \u001b[0mt1\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtime\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mclock\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     90\u001b[0m         \u001b[0mbk\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload_time_stage_1\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mt1\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mt0\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 91\u001b[0;31m         \u001b[0mbiff_version\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbk\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgetbof\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mXL_WORKBOOK_GLOBALS\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     92\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mbiff_version\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     93\u001b[0m             \u001b[0;32mraise\u001b[0m \u001b[0mXLRDError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Can't determine file's BIFF version\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/xlrd/book.py\u001b[0m in \u001b[0;36mgetbof\u001b[0;34m(self, rqd_stream)\u001b[0m\n\u001b[1;32m   1228\u001b[0m             \u001b[0mbof_error\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Expected BOF record; met end of file'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1229\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mopcode\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mbofcodes\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1230\u001b[0;31m             \u001b[0mbof_error\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Expected BOF record; found %r'\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmem\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0msavpos\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0msavpos\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0;36m8\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1231\u001b[0m         \u001b[0mlength\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget2bytes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1232\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlength\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mMY_EOF\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/xlrd/book.py\u001b[0m in \u001b[0;36mbof_error\u001b[0;34m(msg)\u001b[0m\n\u001b[1;32m   1222\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mDEBUG\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"reqd: 0x%04x\"\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0mrqd_stream\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlogfile\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1223\u001b[0m         \u001b[0;32mdef\u001b[0m \u001b[0mbof_error\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmsg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1224\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mXLRDError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Unsupported format, or corrupt file: '\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mmsg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1225\u001b[0m         \u001b[0msavpos\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_position\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1226\u001b[0m         \u001b[0mopcode\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget2bytes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mXLRDError\u001b[0m: Unsupported format, or corrupt file: Expected BOF record; found b'<!DOCTYP'"
     ]
    }
   ],
   "source": [
    "df = pd.read_excel(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  },
  "toc": {
   "nav_menu": {},
   "number_sections": false,
   "sideBar": false,
   "skip_h1_title": false,
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": "block",
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
